<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Prenatal genetic evaluation of the fetus with anomalies or soft markers</title><link href="/z/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/z/d/css/icofont.min.css" rel="stylesheet"/><link href="/z/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/z/d/css/animate.css" rel="stylesheet"/><link href="/z/d/css/owl.carousel.css" rel="stylesheet"/><link href="/z/d/css/magnific-popup.css" rel="stylesheet"/><link href="/z/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/z/d/css/style.css" rel="stylesheet"/><link href="/z/d/css/rtl_edited.css" rel="stylesheet"/><link href="/z/d/css/responsive.css" rel="stylesheet"/><link href="https:///uptodate_assets/main.css" rel="stylesheet"/>
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>
<meta content="" name="description"/>
<link href="/z/d/img/favicon.png" rel="icon" type="image/png">
</link></head>
<body>
<!-- Start Main Menu Area -->
<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>
<section class="teacher-area ptb-10">
<div class="container">
</div>
<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Prenatal genetic evaluation of the fetus with anomalies or soft markers</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Prenatal genetic evaluation of the fetus with anomalies or soft markers</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Authors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Neeta Vora, MD</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Sarah Harris, MD, MS</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Louise Wilkins-Haug, MD, PhD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Vanessa A Barss, MD, FACOG</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Sep 14, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1778597631"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Approximately 3 percent of live births are affected by a major structural anomaly, many of which are now identified prenatally by ultrasound. The etiology of congenital anomalies is diverse and includes environmental influences, genetic factors, or a combination of both.</p><p>Identification of a fetal anomaly should prompt a discussion about fetal genetic evaluation because of the increased possibility of an underlying genetic alteration. However, the landscape of genetic testing is rapidly evolving and often leaves clinicians and patients with many questions about the most appropriate test(s) to choose when such testing is desired. Genetic testing is voluntary and may be chosen or declined by patients based upon their values, beliefs, and other factors that are important to them.</p><p>This topic will provide a reasonable approach to the genetic evaluation of a fetus with anomalies. Postnatal evaluation of the infant with anomalies is reviewed separately. (See  <a class="medical medical_review" href="/z/d/html/2940.html" rel="external">"Congenital anomalies: Approach to evaluation"</a>.)</p><p class="headingAnchor" id="H3955344228"><span class="h1">FREQUENCY OF CHROMOSOMAL ABNORMALITIES AND SIGNIFICANT GENETIC VARIANTS</span><span class="headingEndMark"> — </span>The frequency of informative genetic studies when the fetus has a structural anomaly depends on several factors: the specific anomaly, the number of anomalies, the combination of anomalies identified, and the type of genetic testing [<a href="#rid1">1</a>].</p><p>In several retrospective series of sonographically detected fetal anomalies that prompted fetal genetic studies, the frequency of an abnormal karyotype was [<a href="#rid1">1-6</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>2 to 18 percent among fetuses with an isolated anomaly</p><p class="bulletIndent1"><span class="glyph">●</span>13 to 35 percent among fetuses with multiple anomalies</p><p></p><p>The identification of causative DNA changes is even higher when chromosomal microarray analysis (CMA, which detects duplications or deletions in a region of the chromosomes [copy number variants]) and sequencing (which detects pathogenic variants within a specific gene sequence) are performed.</p><p class="headingAnchor" id="H3253529274"><span class="h1">APPROACH TO GENETIC EVALUATION OF THE FETUS WITH ANOMALIES</span></p><p class="headingAnchor" id="H2965367370"><span class="h2">Offer diagnostic genetic testing</span><span class="headingEndMark"> — </span>Standard clinical practice is to offer diagnostic genetic testing to patients with a structural fetal abnormality. Either chorionic villus sampling (CVS) or amniocentesis can be used to obtain a fetal specimen for this testing [<a href="#rid7">7</a>]. If the test results account for the observed abnormalities, then well-informed counseling about prognosis, reproductive options, obstetric and pediatric management, and recurrence risks is usually possible. Obstetric issues include whether in utero intervention is available and might be indicated, use of antenatal fetal surveillance (ultrasound, nonstress testing, biophysical profile), timing and route of delivery, and choice of facility for giving birth based on expected newborn needs (eg, level of care, palliative care, need for surgery).</p><p>The decision to undergo a procedure to obtain a fetal specimen for diagnosis is personal and must be based on the individual's values and goals. For patients to make informed decisions, pretest counseling should be provided by a clinician familiar with suspected fetal diagnoses, genetic testing options, spectrum of possible results, test limitations, risks of CVS and amniocentesis, and alternatives to prenatal diagnostic testing, such as the range of noninvasive prenatal screening options and postnatal diagnostic testing. Referral to a genetic counselor can aid these discussions. (See <a class="local">'Pretest counseling'</a> below.)</p><p>CVS is typically performed between 10 and 13 weeks of gestation (see  <a class="medical medical_review" href="/z/d/html/449.html" rel="external">"Chorionic villus sampling"</a>). Amniocentesis is optimally performed at ≥15 weeks of gestation (see  <a class="medical medical_review" href="/z/d/html/5390.html" rel="external">"Diagnostic amniocentesis"</a>). When performed at a high-volume, experienced center, the American College of Obstetricians and Gynecologists (ACOG) and Society for Maternal-Fetal Medicine (SMFM) estimated that the procedure-related pregnancy loss rate for a prenatal diagnostic procedure ranges from approximately 1 in 300 to 1 in 1000 (0.1 to 0.3 percent) in a 2016 practice bulletin [<a href="#rid8">8</a>]. Within this range, the observed pregnancy loss rate after CVS is thought to be higher than after amniocentesis because CVS is performed at an earlier gestational age when the background risk for spontaneous loss is higher. However, a 2019 meta-analysis has suggested that the procedure-related risk of pregnancy loss after amniocentesis and CVS is lower (less than 1 in 300) and may not be increased above the background risk of pregnancy loss [<a href="#rid9">9</a>].</p><p class="headingAnchor" id="H2179303794"><span class="h3">Choice of test when a common aneuploidy is suspected</span><span class="headingEndMark"> — </span>When one of the more common aneuploidies (eg, trisomy 21, 18, and 13; monosomy X; triploidy) is suspected based on ultrasound findings, we use the approach described in the algorithm  (<a class="graphic graphic_algorithm graphicRef108719" href="/z/d/graphic/108719.html" rel="external">algorithm 1</a>) and discussed below. These ultrasound findings are described in detail separately. (See  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Sonographic features of selected aneuploidies'</a>.):</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Begin with fluorescence in situ hybridization (FISH)</strong> – The authors' practice is to begin the genetic evaluation with interphase FISH for the major aneuploidies (chromosomes 13, 18, 21, X, and Y). The FISH results are typically available in 24 to 48 hours compared with the 7 to 10 days needed for karyotype or chromosomal microarray analysis (CMA). Although FISH provides a rapid result, it also adds to the cost of the fetal evaluation, so it is also reasonable to proceed directly to a karyotype or CMA. (See <a class="local">'Fluorescence in situ hybridization'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>If the FISH is consistent with aneuploidy</strong> – We recommend obtaining a karyotype both for confirmation and to determine if the aneuploidy is due to nondisjunction or secondary to an unbalanced translocation (which means a parent may have a balanced translocation). This determination is needed to calculate recurrence risks. The karyotype can be performed using cells from the same specimen obtained for FISH. (See <a class="local">'Chromosomal microarray and karyotype'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>If the FISH is not consistent with aneuploidy</strong> – We recommend CMA instead of a karyotype because CMA has a higher diagnostic yield. CMA can be performed on the same fetal specimen used for FISH. (See <a class="local">'Chromosomal microarray and karyotype'</a> below.)</p><p></p><p class="headingAnchor" id="H1452666696"><span class="h3">Choice of test when findings are not suggestive of a common aneuploidy</span><span class="headingEndMark"> — </span>We use the approach in the algorithm  (<a class="graphic graphic_algorithm graphicRef108719" href="/z/d/graphic/108719.html" rel="external">algorithm 1</a>) and discussed below when ultrasound findings are not suggestive of one of the more common aneuploidies (eg, trisomy 21, 18, and 13; monosomy X; triploidy), which are described separately. (See  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Sonographic features of selected aneuploidies'</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Begin with CMA </strong>– We begin with CMA because of its higher diagnostic yield compared with a karyotype. Our approach is the same whether an isolated or multiple structural anomalies are observed. While the yield of CMA is higher in fetuses with multiple anomalies, we believe that an isolated anomaly still warrants a thorough investigation when desired by the patient. Furthermore, an apparently isolated anomaly on prenatal ultrasound may not be isolated when the newborn is evaluated. (See <a class="local">'Chromosomal microarray and karyotype'</a> below.)</p><p></p><p class="bulletIndent1">CMA results are classified as benign, likely benign, variant of unknown significance (VUS), likely pathogenic, and pathogenic. (See  <a class="medical medical_review" href="/z/d/html/16683.html" rel="external">"Prenatal diagnosis of chromosomal imbalance: Chromosomal microarray"</a>.) </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>If CMA identifies a pathogenic or likely pathogenic variant</strong>, a qualified clinician should discuss the significance of the results with the patient. (See <a class="local">'Posttest counseling'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>If CMA is nondiagnostic</strong>, we review available molecular testing options with patients interested in pursuing additional genetic testing. (See <a class="local">'Advanced testing options'</a> below.) </p><p></p><p class="headingAnchor" id="H3061741056"><span class="h2">Options for patients who decline a diagnostic procedure</span><span class="headingEndMark"> — </span>Patients who decline to have a diagnostic procedure for genetic testing may choose to undergo cell-free DNA (cfDNA) screening and/or expanded carrier screening and use the results to inform further decision-making, or they may choose to undergo postnatal diagnostic genetic testing  (<a class="graphic graphic_algorithm graphicRef108719" href="/z/d/graphic/108719.html" rel="external">algorithm 1</a>).</p><p class="headingAnchor" id="H3150762859"><span class="h3">Cell-free DNA screening</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Standard prenatal cell-free DNA screening</strong> – A definitive prenatal genetic diagnosis can only be made if CVS or amniocentesis is performed to obtain a fetal sample for diagnostic testing. However, some patients decline to have an invasive procedure because they consider the risk of fetal loss unacceptable and the results would not impact their decision to carry the pregnancy to term. For these patients, we offer a cfDNA screening test to provide additional information about the fetal risk of aneuploidy, with appropriate pretest and posttest counseling. (See  <a class="medical medical_review" href="/z/d/html/458.html" rel="external">"Prenatal screening for common aneuploidies using cell-free DNA"</a>.) </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>cfDNA result:</strong> <strong>Low risk of trisomy 21, 18, 13, or a sex chromosome aneuploidy</strong>– Patients with a low-risk result should be counseled about the frequency and causes of false-negative results (ie, fetus is affected but the result indicates a low risk of 21, 18, 13 or sex chromosome aneuploidy). They also need to understand that the result does not eliminate the possibility of a fetal genetic condition other than trisomy 21, 18, 13, or a sex chromosome aneuploidy, especially in the setting of a fetal anomaly [<a href="#rid10">10,11</a>]. (See  <a class="medical medical_review" href="/z/d/html/458.html" rel="external">"Prenatal screening for common aneuploidies using cell-free DNA", section on 'False-negative cfDNA test results'</a>.)</p><p></p><p class="bulletIndent2">It has been estimated that, in the setting of a fetal anomaly, performing a cfDNA screening test alone will miss 8 percent of chromosomal abnormalities detectable by a karyotype and 16 percent of cytogenetic abnormalities diagnosed by CMA [<a href="#rid12">12,13</a>]. The frequency of missed genetic diagnosis varies by gestational age: it is &lt;10 percent in the first trimester but almost 20 percent in the late second or third trimester. The difference reflects the high frequency of sonographic abnormalities associated with trisomy 21, 18, and 13 that are detectable in early pregnancy (eg, enlarged nuchal translucency, cystic hygroma, cardiac anomalies); better detection of other structural anomalies later in gestation compared with the first trimester; and the association of these other anomalies with less common chromosome imbalances. </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>cfDNA result: Increased risk of trisomy 21, 18, 13, or a sex chromosome aneuploidy</strong> – Patients also need to understand that a cfDNA test result showing an increased risk of one of these aneuploidies may be a false positive (ie, the fetus is unaffected). (See  <a class="medical medical_review" href="/z/d/html/458.html" rel="external">"Prenatal screening for common aneuploidies using cell-free DNA", section on 'False-positive cfDNA test results'</a>.)</p><p></p><p class="bulletIndent2">All cfDNA results indicating an increased risk for aneuploidy should be confirmed with diagnostic testing, either pre- or postnatally. cfDNA results should be confirmed prenatally prior to a pregnancy termination or any other irreversible prenatal procedure. The positive predictive value of cfDNA for predicting an aneuploid fetus is higher in the setting of a comprehensive ultrasound examination by an experienced sonologist that clearly shows features of one of these disorders, but even in this setting, prediction of one of these aneuploidies is not 100 percent accurate. We counsel patients with positive cfDNA screening results on the positive predictive value based on their specific pretest probability. Although we do not recommend this, some patients with cfDNA results showing &gt;97 percent probability of trisomy have rejected diagnostic testing and moved directly to termination. (See  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Trisomy 18 (Edward syndrome)'</a> and  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Trisomy 13 (Patau syndrome)'</a> and  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Triploidy'</a>.)</p><p></p><p class="bulletIndent2">Confirmation of cfDNA results is also important because a trisomy that results from an unbalanced rearrangement in the fetus could be inherited from a parent who carries a balanced rearrangement, which is not detected with cfDNA screening but has implications for recurrence risk.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>cfDNA result: nonreportable</strong> – One to 5 percent of cfDNA tests will report "no result" for trisomy 21, 18, and 13, and sex chromosome aneuploidies. The reasons for nonreportable results (test failures) are multiple and depend on the testing platform, performing laboratory characteristics, fetal fraction, and patient characteristics. Further evaluation following a nonreportable result can include repeating cfDNA screening with a second sample or moving on to diagnostic testing as these patients are at increased risk for aneuploidy. (See  <a class="medical medical_review" href="/z/d/html/458.html" rel="external">"Prenatal screening for common aneuploidies using cell-free DNA", section on 'No call or no result'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Expanded cell-free DNA screening</strong> – cfDNA screening for genetic conditions other than trisomy 21, 18, 13, and sex chromosome aneuploidies is not recommended on a population-wide basis [<a href="#rid14">14-16</a>]. The use of expanded cfDNA screening (either for microdeletion/microduplication screening, monogenic disorders, or whole genome coverage) has not been clinically validated in the setting of a fetus with anomalies. However, this is an evolving area with new studies suggesting clinical value of more expanded screening strategies [<a href="#rid17">17,18</a>]. (See  <a class="medical medical_review" href="/z/d/html/134618.html" rel="external">"Cell-free DNA screening for fetal conditions other than the common aneuploidies"</a>.)</p><p></p><p class="bulletIndent1">Patients who desire this level of information regarding the genetic status of their fetus should opt for diagnostic testing on amniocytes or chorionic villi. However, the authors have encountered patients who consider expanded cfDNA screening, which is commercially available, an acceptable alternative to diagnostic testing. For these patients, the decision to obtain expanded cfDNA screening should only be made after appropriate counseling by a qualified clinician regarding the limitations of this testing. </p><p></p><p class="headingAnchor" id="H3563665985"><span class="h3">Expanded carrier screening</span><span class="headingEndMark"> — </span>Expanded carrier screening uses next-generation sequencing (NGS) to analyze select genes that are associated with autosomal recessive or X-linked conditions. Patients who initially decline diagnostic testing may consider this testing to better inform decision-making. One study found that 3.2 percent of newborns (7 of 222) with structural anomalies were diagnosed with a condition that would have been detected (ie, positive parental screening result) on an expanded carrier screening panel of 500 conditions [<a href="#rid19">19</a>]. This study suggests that when diagnostic testing is declined, expanded carrier screening can be considered after pretest counseling. If the expanded carrier screening is negative (ie, an inherited single gene disorder associated with the ultrasound anomalies is not identified), the family should not be overly reassured and postnatal testing for genetic etiologies is still recommended. (See  <a class="medical medical_review" href="/z/d/html/114595.html" rel="external">"Preconception and prenatal panethnic expanded carrier screening"</a>.)</p><p class="headingAnchor" id="H1845514527"><span class="h3">Postnatal testing</span><span class="headingEndMark"> — </span>Genetic testing after birth is another option. The approach to postnatal evaluation of the infant with congenital anomalies is reviewed separately. (See  <a class="medical medical_review" href="/z/d/html/2940.html" rel="external">"Congenital anomalies: Approach to evaluation"</a>.)</p><p class="headingAnchor" id="H1227339225"><span class="h2">Pretest and posttest counseling</span></p><p class="headingAnchor" id="H162674744"><span class="h3">Pretest counseling</span><span class="headingEndMark"> — </span>Appropriate pretest counseling (up-to-date, balanced, accurate information) that is patient-centered is critical for patients who are making the decision to undergo genetic testing. The goal is to help patients understand the benefits as well as the limitations of testing, discuss possible test results, and help patients make informed decisions consistent with their own goals and values [<a href="#rid20">20</a>]. Ideally, a certified genetic counselor or a knowledgeable obstetric provider should provide this counseling. Pretest counseling following the diagnosis of a fetal anomaly includes a discussion of the following:</p><p class="bulletIndent1"><span class="glyph">●</span>Patient values and goals:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>General attitudes toward prenatal testing and screening.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Desires regarding the level of information provided by testing.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Views about and availability of pregnancy termination.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Prenatal testing options and procedures, and the options of no prenatal testing or postnatal testing</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Possible results:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Aneuploidy or a pathologic variant with defined phenotype.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Copy number variants (CNVs) with variable phenotype.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Variants of uncertain significance.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Incidental findings, including nonpaternity, consanguinity, and variants in the fetus and/or parent associated with adult-onset disease.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Normal (ie, benign, likely benign) results. Normal results do not guarantee absence of a disorder as no test can detect all disorders.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Nonreportable result on cfDNA screening.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Potential psychosocial issues:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The meaning that the specific diagnosis has for the family, the sense of loss of a normal pregnancy or baby, any significant discord between parents or relatives.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Fears of coping with the challenges of a child with disabilities; coping strategies, referrals, and awareness of available resources when appropriate.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Pregnancy/postpartum management options:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Options of pregnancy termination versus continuing the pregnancy with possible changes in antepartum, intrapartum, and postpartum/neonatal care. For patients who would elect to continue a pregnancy with a life-limiting diagnosis (ie, a fetal demise, intrapartum fetal death, or neonatal/infant death is expected), discussing the option of perinatal palliative care can provide support and guidance for families throughout the pregnancy and birth. Additional resources for caregivers can be found at <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.perinatalhospice.org%2F&amp;token=kgWnuLuwd5ywHVQww%2BY9erUVa5jyC%2FmVCZuAjgcBXjQnWTgnBe8Wpk0%2Fo72h%2BhRb&amp;TOPIC_ID=104101" target="_blank">Perinatal Hospice and Palliative Care</a>.</p><p></p><p class="headingAnchor" id="H2583011505"><span class="h3">Posttest counseling</span><span class="headingEndMark"> — </span>Following genetic testing, it is important to give patients an opportunity to discuss the significance of their results with a qualified provider. This is important even when results are normal.</p><p class="bulletIndent1"><span class="glyph">●</span>Abnormal results: </p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Discuss the certainty of the diagnosis and whether further testing is indicated.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Discuss the significance of the results for the health of the fetus, before and after birth, including the limitations of prenatal phenotyping.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Review the patient’s goals and values and pregnancy/postpartum management options as discussed during pretest counseling.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Consider referral to a pediatric subspecialist with expertise in the identified condition.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Discuss recommended follow-up after birth.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Review recurrence risk and options for future pregnancies.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Refer to vetted online parent support resources.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Normal results:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Discuss that, while normal results are reassuring, they do not eliminate the possibility of an underlying genetic condition in the fetus. Discuss advanced testing options (sequencing), if appropriate.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Explain that variants of uncertain significance may be reclassified in the future and found to be disease causing.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>If the patient elected to have prenatal sequencing and it is uninformative, explain that additional features may be identified in the newborn or child, which may lead to a sequencing result becoming informative. </p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Review options for additional evaluation after birth, including consultation with a medical geneticist if appropriate.</p><p></p><p class="headingAnchor" id="H2706279128"><span class="h2">Testing options</span><span class="headingEndMark"> — </span>Testing options are described below and summarized in the table  (<a class="graphic graphic_table graphicRef129177" href="/z/d/graphic/129177.html" rel="external">table 1</a>).</p><p class="headingAnchor" id="H3850732324"><span class="h3">Initial testing options</span></p><p class="headingAnchor" id="H1704211276"><span class="h4">Fluorescence in situ hybridization</span><span class="headingEndMark"> — </span>Autosomal trisomies for 21, 18, and 13; sex chromosome aneuploidy; and triploidy account for 80 percent of clinically significant chromosomal conditions diagnosed prenatally [<a href="#rid21">21</a>]. FISH can be used for rapid and accurate detection of aneuploidies involving chromosomes 21, 18, 13, X, and Y, with results typically available in 24 to 48 hours [<a href="#rid21">21</a>]. FISH can also be used to detect other chromosome abnormalities when appropriate probes are available.</p><p>The concordance rates between FISH and karyotype of uncultured amniocytes are high, reportedly &gt;99 percent [<a href="#rid22">22</a>]. However, confirmatory testing with a karyotype is required to determine if a translocation is present. Normal FISH results also require additional testing, either through karyotype or CMA, because alternations in chromosome structure and less common aneuploidies would be missed if FISH was used in isolation. (See  <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">"Tools for genetics and genomics: Cytogenetics and molecular genetics", section on 'Fluorescence in situ hybridization'</a>.)</p><p class="headingAnchor" id="H1977182457"><span class="h4">Chromosomal microarray and karyotype</span><span class="headingEndMark"> — </span>Karyotyping has been the standard for prenatal diagnosis, but use of CMA is increasing. Advantages of CMA are:</p><p class="bulletIndent1"><span class="glyph">●</span>CMA detects small (typically 50 to 100 kb) gains and losses of genetic material (called copy number variants [CNVs]) that would not be identified by traditional karyotyping yet have the potential to lead to significant phenotypic abnormalities. By contrast, karyotyping typically has a resolution of only 5 to 10 Mb.</p><p></p><p class="bulletIndent1">In a systematic review of prenatal CMA, a clinically significant CNV was detected in 5.6 percent (95% CI 4.7-6.6 percent) of euploid fetuses with an ultrasound anomaly restricted to one anatomic system and in 9.1 percent (95% CI 7.5-10.8 percent) of euploid fetuses with multiple anomalies [<a href="#rid23">23</a>]. These estimates are similar to other reviews in which significant CNV in euploid fetuses with ultrasound anomalies ranged from 5.1 to 10.0 percent [<a href="#rid24">24,25</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>CMA does not require cell culture, thus reducing the turnaround time for results.</p><p></p><p>The greater depth of molecular analysis with CMA increases the likelihood of a diagnosis, but the likelihood of incidental findings is also increased, which increases the complexity of prenatal counseling. For example, CMA may detect variants of unknown significance, previously unsuspected genetic variants in one or both parents, or fetal genes associated with adult-onset diseases. In addition, while CMA has improved resolution over karyotype, CMA will not identify some clinically significant genetic findings, such as gene sequence changes that might affect gene function, balanced structural rearrangements, and unbalanced translocations. (See  <a class="medical medical_review" href="/z/d/html/16683.html" rel="external">"Prenatal diagnosis of chromosomal imbalance: Chromosomal microarray", section on 'Benefits and limitations of CMA'</a>.)</p><p>Many professional societies, including the American College of Obstetricians and Gynecologists (ACOG) and the Society for Maternal-Fetal Medicine (SMFM), have recommended use of CMA for prenatal diagnosis. The ACOG committee opinion states that CMA is recommended instead of a karyotype when the fetus has one or more major structural abnormality identified on ultrasound [<a href="#rid26">26</a>]. ACOG has also stated that CMA should be made available to any patient choosing to undergo invasive diagnostic testing. (See  <a class="medical medical_review" href="/z/d/html/16683.html" rel="external">"Prenatal diagnosis of chromosomal imbalance: Chromosomal microarray"</a> and  <a class="medical medical_review" href="/z/d/html/16647.html" rel="external">"Genomic disorders: An overview"</a>.)</p><p class="headingAnchor" id="H2558272566"><span class="h3">Advanced testing options</span><span class="headingEndMark"> — </span>When chromosomal analysis, including CMA, is nondiagnostic, the next step in the genetic evaluation is testing for specific genetic conditions through use of molecular genomic techniques. Advanced sequencing techniques such as NGS may be used to interrogate a single gene, a panel of selected genes, the exome (ES, the protein-coding genes that make up 1 to 2 percent of the genome), or the entire genome (GS). Sequencing options have the advantage of higher diagnostic yield and have gained support because of increasing knowledge that the defining features of a phenotype may not manifest until later in the gestation (after the time of diagnostic testing) and the fetal phenotype may be different from the classic pediatric presentation.</p><p>We recommend additional counseling with a prenatal genetic counselor or clinical geneticist prior to proceeding with this testing. The targeted approach has the advantage of reducing the chances of finding genomic variants of uncertain clinical significance or incidental findings, and the turnaround time is typically shorter than ES or GS. However, selection of a gene panel targeted to a fetal finding does not provide the opportunity to examine other clinically significant conditions [<a href="#rid27">27</a>].</p><p class="headingAnchor" id="H143400822"><span class="h4">General principles when considering advanced genetic testing</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Utilize appropriate expertise</strong> – Pretest counseling, interpretation of results, and posttest counseling are highly complex and are best conducted in consultation with a multidisciplinary team with expertise and experience in both the clinical and laboratory aspects of prenatal diagnosis and fetal sequencing.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Inform the laboratory about the possibility of advanced testing</strong> – Clinicians should inform the laboratory if advanced testing will be considered in cases of nondiagnostic cytogenetic results from CMA because the testing can be performed on fetal DNA from cultured chorionic villi or amniocytes obtained at the time of the diagnostic procedure. If an adequate initial sample is obtained, DNA can often be directly extracted from chorionic villi or amniocytes without culture and thus shorten the time to results. If an adequate cultured or direct sample from the original procedure is not available, then a second procedure may be required.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Address limitations of prenatal versus postnatal testing</strong> – While the presence of a fetal anomaly increases the risk for an underlying monogenic condition, the ability to test and the value of testing for specific conditions prenatally can be more challenging compared with the postnatal setting. Identification of a pathogenic variant is not a clinical diagnosis; clinical diagnosis requires correlation of the genetic test result, fetal findings (phenotype), and family history.</p><p></p><p class="bulletIndent1">Prenatal prognostic counseling is complicated because the resolution of ultrasound imaging of minor anatomic abnormalities is limited; functional features of a phenotype (eg, neurodevelopmental impairment) are generally not possible to detect prenatally; the characteristic phenotype may be poorly defined or incomplete prenatally; the phenotype may evolve after initial diagnosis and counseling; and disorders identified prenatally may have more benign or more deleterious outcomes than the same disorders identified postnatally [<a href="#rid28">28</a>]. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Discuss issues regarding time to make a diagnosis</strong> – Advanced genetic testing can be a long process, which can be a problem prenatally when time-sensitive decisions need to be made regarding possible pregnancy termination, pregnancy management, route of birth, site of birth, and level of neonatal care.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Discuss trio-based sequencing </strong>– Diagnostic sequencing for fetal indications is best done as a trio analysis (where fetal and both parental samples are sequenced and analyzed together); thus, patients should understand that parental blood samples are often required for confirmatory testing. The trio approach is more informative than duo analysis (one parent and the fetus) and helps to exclude uninformative variants, identify de novo fetal variants, and determine inheritance of recessive variants.</p><p></p><p class="bulletIndent1">Issues related to misattributed parentage (eg, mistaken paternity, nondisclosed relationship, nondisclosure of a donor) should be addressed during pretest counseling.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Discuss the potential for incidental findings</strong> – Ethical issues involving when to report secondary or incidental fetal and parental findings (variants for genetic disorders unrelated to the phenotype being investigated) have not been resolved. Medically actionable genes (eg, cardiovascular disease or cancer predisposition genes) are generally assessed and reported as secondary findings when established interventions are available for preventing or significantly reducing morbidity and mortality.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Discuss future reanalysis of results</strong> – There is ongoing need for reanalysis of results as information in database resources accrues after the initial analysis. Reanalysis may also be considered postnatally if a new phenotype is observed in the proband.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Discuss cost </strong>– Insurance coverage is limited, but the cost of advanced genetic testing continues to decrease.</p><p></p><p class="headingAnchor" id="H2807804420"><span class="h4">Targeted gene sequencing</span><span class="headingEndMark"> — </span>Targeted gene sequencing refers to testing a specific gene or genes known to be associated with a genetic condition. Single gene testing in the prenatal period often relies on a positive family history and a previously identified pathogenic variant [<a href="#rid29">29</a>]. For example, achondroplasia is caused by genetic alterations in <em>FGFR3</em>. When achondroplasia is suspected clinically, sequencing <em>FGFR3</em> can confirm the diagnosis [<a href="#rid30">30</a>].</p><p>Some genetic conditions are associated with genetic changes in multiple genes. For example, Noonan syndrome has been associated with pathogenic variants in <em>PTPN11</em>, <em>SOS1</em>, <em>KRAS</em>, <em>RAF1</em>, <em>NRAS</em>, <em>BRAF</em>, and <em>MAP2K1</em>. If Noonan syndrome is suspected clinically, then sequencing a panel of the genes associated with Noonan syndrome through NGS can confirm the diagnosis [<a href="#rid31">31</a>]. (See  <a class="medical medical_review" href="/z/d/html/13558.html" rel="external">"Noonan syndrome", section on 'Diagnosis'</a>.)</p><p>The decision regarding which test to perform and the interpretation of molecular genetic results can be complex, and usually should be done in consultation with a provider specializing in genetic testing, such as a certified genetic counselor or medical geneticist.</p><p class="headingAnchor" id="H3131806414"><span class="h4">Exome sequencing</span><span class="headingEndMark"> — </span>Exome sequencing (ES) uses NGS to sequence the exome (regions of the genome that are known to encode proteins). The exome includes approximately 1 percent of the genome but is thought to contain 85 percent of disease-causing variants [<a href="#rid32">32</a>]. ES platforms vary in their depth of sequencing (ie, the number of times a specific nucleotide is sequenced). The greater the depth, the higher the likelihood that an identified sequence alteration is truly present and the lower the risk that true sequence changes within the exomes will be missed. ES does not sequence the remainder of the DNA (99 percent); for this reason, some sequence abnormalities, such as those occurring in promoter regions, would not be detected by ES. In addition, relevant genes may be missed because of technical issues, and ES typically does not detect triplet repeat disorders, small insertions and deletions between 100 and 100,000 base pairs, structural variants (rearrangements and translocations), or some CNVs [<a href="#rid33">33</a>].</p><p>In a meta-analysis of the diagnostic yield of ES for prenatal diagnosis of fetal structural anomalies when karyotype/CMA was normal (72 reports from 2010 to 2021, 4350 anomalous fetuses), the pooled incremental yield of ES was 31 percent (95% CI 26-36) [<a href="#rid34">34</a>]. Diagnoses were strictly defined as variants classified as pathogenic/likely pathogenic and deemed to be causing the fetal phenotype; variants classified as possibly diagnostic, probably diagnostic, or potentially relevant diagnoses were excluded. The incremental diagnostic yield was affected by several factors; for example, it was higher for cases in which a multidisciplinary team with expertise in genetics suspected a monogenic etiology than in unselected cases (42 versus 15 percent). It also varied among phenotypic subgroups, ranging from 53 percent for isolated skeletal abnormalities, 29 percent for multisystem anomalies, and 11 percent for cardiac anomalies to 2 percent for isolated enlarged nuchal translucency.</p><p class="bulletIndent1">While the findings described in the meta-analysis are promising for the use of ES technology in the genetic evaluation of the fetus with anomalies, challenges remain for the routine clinical use [<a href="#rid35">35-38</a>]. Much of the phenotypic data for specific sequence changes are incomplete, and typically based on newborn/pediatric presentation, which makes interpretation of identified variants more difficult. ES typically identifies approximately 40,000 sequence variants. Bioinformatic algorithms are used for filtering, annotating, and prioritizing the genetic information from ES for clinical interpretation. This process narrows the number of variants down to those with a known (&gt;99 percent) or likely (90 to 99 percent) genotype‐pathologic phenotype association. When the probability of pathogenicity is &lt;90 percent but the variant is not clearly benign and without health consequences, it is considered a variant of uncertain significance. However, laboratories vary in their interpretation of gene variants, in part because of a lack of publicly available prenatal databases and standardized prenatal curation [<a href="#rid33">33</a>].</p><p></p><p>Given the complexities involved in interpretation of results, timeliness of results, and meaningful patient/family counseling, the decision to proceed with this type of testing should be made in consultation with a clinician specializing in genetic testing, such as a certified genetic counselor or medical geneticist. (See  <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications"</a>.)</p><p class="headingAnchor" id="H4073440780"><span class="h4">Whole genome sequencing</span><span class="headingEndMark"> — </span>Whole genome sequencing (WGS) analyzes both the exonic and intronic (noncoding) regions of the genome, thus it is a more comprehensive technique for detecting sequence variation than ES. It can identify even very small CNVs (below the resolution of microarray), other structural variants, expansions of short tandem repeats, mitochondrial disorders, and some coding exons not detected by ES [<a href="#rid33">33</a>]. Despite more comprehensive genomic assessment, turnaround time is often faster than for ES. However, ES analysis is more reliable for identifying mosaic sequence variants than WGS because of its higher sequencing depth [<a href="#rid39">39</a>].</p><p>The following statements reflect some of the key points of a 2022 International Society for Prenatal Diagnosis updated position statement endorsed by SMFM on the use of WGS for fetal diagnosis [<a href="#rid40">40</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>The routine use of prenatal sequencing as a diagnostic test on fetal tissue obtained from an invasive prenatal procedure for indications other than fetal anomalies (including upon parental request) is not currently supported due to insufficient validation data and knowledge about its benefits and pitfalls.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Prenatal sequencing may be beneficial in the following settings:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>A fetus with a single major anomaly or with multiple organ system anomalies that are suggestive of a possible genetic etiology, but no genetic diagnosis was found after microarray, or in select situations with no microarray result, following a multidisciplinary review and consensus, in which there is a fetus with a multiple anomaly 'pattern' that strongly suggests a single gene disorder.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>A personal (maternal or paternal) history of a prior undiagnosed fetus (or child) affected with a major single anomaly or multiple anomalies suggestive of a genetic etiology, and a recurrence of similar anomalies in the current pregnancy without a genetic diagnosis after karyotype or microarray. If such parents present for preconception counseling and no sample is available from the affected proband, or if a fetal sample cannot be obtained in an ongoing pregnancy, it is appropriate to offer expanded carrier screening for both biological parents to look for shared carrier status for autosomal recessive variants that might explain the fetal phenotype.</p><p></p><p>The document also discussed quality standards, analysis, variant interpretation, and reporting of results (including incidental findings) from diagnostic or research laboratories.</p><p class="headingAnchor" id="H651408341"><span class="h4">Methylation studies</span><span class="headingEndMark"> — </span>Methylation studies are useful when genetic disorders resulting from abnormal methylation or imprinting are suspected. For example, Beckwith-Wiedemann syndrome is in the differential diagnosis of omphalocele and is best detected with methylation studies. ES and GS will typically not detect disorders resulting from abnormal methylation. (See  <a class="medical medical_review" href="/z/d/html/13555.html" rel="external">"Beckwith-Wiedemann syndrome"</a>.)</p><p class="headingAnchor" id="H286557901"><span class="h1">APPROACH TO THE EVALUATION OF THE FETUS WITH "SOFT MARKERS" AND NO STRUCTURAL ANOMALIES</span><span class="headingEndMark"> — </span>A detailed description of soft markers is available separately. (See  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Soft markers'</a>.)</p><p class="headingAnchor" id="H3850733202"><span class="h2">Soft markers detected BEFORE aneuploidy screening</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Isolated soft marker</strong> – Our approach depends on the marker and is shown in the table  (<a class="graphic graphic_table graphicRef132712" href="/z/d/graphic/132712.html" rel="external">table 2</a>). A detailed ultrasound examination is important to ensure as technically possible that the soft marker is isolated (eg, the short long bones are morphologically normal and other signs suggestive of growth restriction are absent, the profile is normal except for the absent nasal bone).</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Two or more soft markers</strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Diagnostic testing approach</strong> – For patients with ≥2 soft markers who have not had aneuploidy screening, we suggest counseling and discussion of the option of diagnostic testing, as described in the algorithm  (<a class="graphic graphic_algorithm graphicRef108720" href="/z/d/graphic/108720.html" rel="external">algorithm 2</a>). If the patient elects this approach, we begin the evaluation with interphase fluorescence in situ hybridization (FISH) for the common aneuploidies. Patients are given the option of chromosomal microarray analysis (CMA) or karyotype if FISH is nondiagnostic. Other clinicians may omit FISH and go directly to CMA or karyotype. Patients may have a preference for one approach versus the other after pretest counseling [<a href="#rid8">8</a>].</p><p></p><p class="bulletIndent2">In contrast to pregnancies with fetal structural anomalies, data are limited on the value of CMA versus karyotyping when soft markers are identified on ultrasound. In a retrospective analysis that stratified CMA detection rates by specific ultrasound findings, CMA was abnormal in 2.6 percent of fetuses with isolated soft markers (2 of 77) [<a href="#rid41">41</a>]. Given the small number of cases in this study, it is not possible to draw definitive conclusions about the value of CMA when only soft markers are detected. However, from a large multicenter trial, we know that 1.7 percent of structurally normal fetuses with a normal karyotype have a clinically significant CMA finding [<a href="#rid42">42</a>].</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Cell-free DNA screening approach</strong> – For patients with ≥2 soft markers who have not had aneuploidy screening and decline diagnostic testing but are interested in further evaluation, we offer cell-free DNA (cfDNA) screening because this is the most sensitive single noninvasive test for the common aneuploidies.</p><p></p><p class="bulletIndent2">If results from cfDNA screening show no increased risk for trisomy 21, 18, 13, or sex chromosome anomalies, then patients can be reassured and typically continue with routine prenatal care. However, as discussed previously, we always stress that a normal screening result does not eliminate the possibility of a genetic condition in the fetus. (See <a class="local">'Cell-free DNA screening'</a> above.)</p><p></p><p class="headingAnchor" id="H128604733"><span class="h2">Soft markers detected AFTER aneuploidy screening</span><span class="headingEndMark"> — </span>By the time of the second-trimester anatomy ultrasound, many patients have already completed aneuploidy screening through first- or second-trimester maternal biochemical marker screening or cfDNA screening. The Society for Maternal-Fetal Medicine (SMFM) recommends not offering diagnostic testing solely for the indication of an isolated soft marker when prior aneuploidy screening shows no increased risk of the targeted aneuploidies [<a href="#rid43">43</a>]. We agree with SMFM that an isolated soft marker should not be overemphasized in these patients and that some soft markers should be described as a normal variant. However, we feel that it is important to ensure that guidelines do not disproportionately restrict subgroups of patients, such as those with a soft marker after cfDNA screening, from access to diagnostic genetic testing or devalue the utility of diagnostic genetic testing within that subgroup. Therefore, for patients whose cfDNA screening or serum screening shows no increased risk, our approach is as follows: </p><p class="bulletIndent1"><span class="glyph">●</span><strong>Isolated soft marker </strong>– Our approach depends on the soft marker, as shown in the table  (<a class="graphic graphic_table graphicRef132712" href="/z/d/graphic/132712.html" rel="external">table 2</a>).</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Two or more soft markers</strong> – We offer genetic counseling to discuss invasive diagnostic testing options, as shown in the algorithm  (<a class="graphic graphic_algorithm graphicRef112978" href="/z/d/graphic/112978.html" rel="external">algorithm 3</a>).</p><p></p><p class="headingAnchor" id="H2842335131"><span class="h2">Prenatal evaluation for other disorders</span><span class="headingEndMark"> — </span>Some soft markers are associated with disorders other than aneuploidy, which may require additional evaluation, including diagnostic testing. For example, echogenic bowel has been associated with blood in the bowel lumen, cystic fibrosis, growth restriction, infection, and gastrointestinal obstruction. Urinary tract dilation has been associated with kidney abnormalities. Evaluation of these fetuses depends on the specific marker.</p><p class="bulletIndent1"><span class="glyph">●</span>Urinary tract dilation – (See  <a class="medical medical_review" href="/z/d/html/6099.html" rel="external">"Fetal hydronephrosis: Etiology and prenatal management", section on 'Congenital anomalies of the kidney and urinary tract (CAKUT)'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Single umbilical artery – (See  <a class="medical medical_review" href="/z/d/html/15245.html" rel="external">"Single umbilical artery"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Echogenic bowel – (See  <a class="medical medical_review" href="/z/d/html/420.html" rel="external">"Fetal echogenic bowel"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Thickened nuchal fold – (See  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Thick nuchal fold'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Hypoplastic nasal bone – (See  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Second-trimester absent nasal bone'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Shortened long bones – (See  <a class="medical medical_review" href="/z/d/html/14209.html" rel="external">"Approach to prenatal diagnosis of the lethal (life-limiting) skeletal dysplasias"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Nonisolated choroid plexus cysts – (See  <a class="medical medical_review" href="/z/d/html/447.html" rel="external">"Sonographic findings associated with fetal aneuploidy", section on 'Choroid plexus cysts'</a>.)</p><p></p><p class="headingAnchor" id="H2466225954"><span class="h2">Role of postnatal genetic testing after detection of soft markers</span><span class="headingEndMark"> — </span>Most soft markers resolve by the third trimester. Prenatal detection of soft markers is not an indication for postnatal genetic evaluation unless the neonate has signs of aneuploidy or another genetic disorder on clinical examination.</p><p class="headingAnchor" id="H1779673659"><span class="h1">INFORMATION FOR PATIENTS</span><span class="headingEndMark"> — </span>UpToDate offers two types of patient education materials, "The Basics" and "Beyond the Basics." The Basics patient education pieces are written in plain language, at the 5<sup>th</sup> to 6<sup>th</sup> grade reading level, and they answer the four or five key questions a patient might have about a given condition. These articles are best for patients who want a general overview and who prefer short, easy-to-read materials. Beyond the Basics patient education pieces are longer, more sophisticated, and more detailed. These articles are written at the 10<sup>th</sup> to 12<sup>th</sup> grade reading level and are best for patients who want in-depth information and are comfortable with some medical jargon.</p><p>Here are the patient education articles that are relevant to this topic. We encourage you to print or e-mail these topics to your patients. (You can also locate patient education articles on a variety of subjects by searching on "patient info" and the keyword(s) of interest.)</p><p class="bulletIndent1"><span class="glyph">●</span>Basics topics (see  <a class="medical medical_basics" href="/z/d/html/117182.html" rel="external">"Patient education: Genetic testing (The Basics)"</a>)</p><p></p><p class="headingAnchor" id="H1421296933"><span class="h1">SUMMARY AND RECOMMENDATIONS</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Frequency of chromosomal abnormalities in the fetus with one or more anomalies</strong> – The frequency of a chromosomal abnormality depends on the specific fetal anomaly, the number of anomalies, and the combination of anomalies. In general:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>2 to 18 percent of fetuses with an isolated anomaly have a chromosomal abnormality</p><p class="bulletIndent2"><span class="glyph">•</span>13 to 35 percent of fetuses with multiple anomalies have a chromosomal abnormality </p><p></p><p class="bulletIndent1">The identification of causative DNA changes is even higher when genetic testing for copy number variants (CNVs; eg, chromosomal microarray analysis [CMA]) and sequence pathogenic variants are performed. (See <a class="local">'Frequency of chromosomal abnormalities and significant genetic variants'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Prenatal screening and testing</strong> – Prenatal screening and diagnostic tests are voluntary and such tests may be chosen or declined by patients based upon their values, beliefs, and the issues that they feel are most important. (See <a class="local">'Introduction'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Counseling</strong> – Appropriate pretest and posttest counseling is essential for all patients electing to proceed with genetic testing. (See <a class="local">'Pretest and posttest counseling'</a> above and <a class="local">'General principles when considering advanced genetic testing'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Approach to fetal genetic testing in</strong> <strong>patients who accept an invasive procedure</strong> </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Initial testing</strong> – We offer diagnostic genetic testing (which requires either chorionic villus sampling [CVS] or amniocentesis) to all patients with a structural fetal anomaly, as described in the algorithm  (<a class="graphic graphic_algorithm graphicRef108719" href="/z/d/graphic/108719.html" rel="external">algorithm 1</a>). The decision to proceed with an invasive procedure for diagnostic testing is personal and must take into account the individual patient's goals and values. Other options include noninvasive screening and postnatal testing. (See <a class="local">'Approach to genetic evaluation of the fetus with anomalies'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Advanced testing</strong> – If initial testing is nondiagnostic, we suggest referral to a genetics specialist to discuss possible additional genetic testing and, if additional molecular testing is desired, the choice of test  (<a class="graphic graphic_table graphicRef129177" href="/z/d/graphic/129177.html" rel="external">table 1</a>). Identification of a pathogenic variant is not a clinical diagnosis; clinical diagnosis requires correlation of the genetic test result, fetal findings (phenotype), and family history. (See <a class="local">'Advanced testing options'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Approach to fetal genetic testing in patients who decline an invasive procedure</strong> – For patients who decline an invasive procedure for diagnostic genetic testing, noninvasive screening via cell-free DNA (cfDNA) is an option. Patients should be counseled about the limitations of cfDNA screening in the setting of fetal anomalies: normal results can be falsely reassuring since the results are limited to trisomy 21, 18, 13 or sex chromosome aneuploidy, and abnormal results may be falsely positive. (See <a class="local">'Options for patients who decline a diagnostic procedure'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Approach to patients with fetal soft markers but no structural anomalies</strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>No previous aneuploidy screening</strong> –<strong> </strong>The approach to management of an isolated soft marker is summarized in the table  (<a class="graphic graphic_table graphicRef132712" href="/z/d/graphic/132712.html" rel="external">table 2</a>). For patients with two or more soft markers, we suggest genetic counseling to review options for diagnostic testing and cfDNA screening  (<a class="graphic graphic_algorithm graphicRef108720" href="/z/d/graphic/108720.html" rel="external">algorithm 2</a>). (See <a class="local">'Soft markers detected BEFORE aneuploidy screening'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Previous aneuploidy screening</strong> –<strong> </strong>For patients whose aneuploidy screening results show a low risk for the targeted aneuploidies but a subsequent ultrasound examination identifies a soft marker, our approach depends on the number of soft markers (see <a class="local">'Soft markers detected AFTER aneuploidy screening'</a> above):</p><p></p><p class="bulletIndent3"><span class="glyph">-</span><strong>An isolated soft marker</strong> – We offer reassurance that this is likely a normal variant and the chance the fetus has one of the common aneuploidies targeted by screening is very low, in line with recommendations from the Society for Maternal-Fetal Medicine (SMFM)  (<a class="graphic graphic_table graphicRef132712" href="/z/d/graphic/132712.html" rel="external">table 2</a>). However, soft markers in euploid fetuses may be associated with specific disorders (eg, urinary tract dilation may be a sign of congenital anomalies of the kidney and urinary tract [CAKUT]). This should be addressed during counseling and options for prenatal diagnosis discussed. Some isolated soft markers, such as thickened nuchal fold and absent/hypoplastic nasal bone, are more strongly associated with fetal aneuploidy. In such cases, additional genetic counseling and testing may be warranted.</p><p></p><p class="bulletIndent3"><span class="glyph">-</span><strong>Two or more soft markers</strong> – We offer additional genetic counseling and discussion of diagnostic testing  (<a class="graphic graphic_algorithm graphicRef112978" href="/z/d/graphic/112978.html" rel="external">algorithm 3</a>).</p><p></p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Staebler M, Donner C, Van Regemorter N, et al. Should determination of the karyotype be systematic for all malformations detected by obstetrical ultrasound? Prenat Diagn 2005; 25:567.</a></li><li><a class="nounderline abstract_t">Halliday J, Lumley J, Bankier A. Karyotype abnormalities in fetuses diagnosed as abnormal on ultrasound before 20 weeks' gestational age. Prenat Diagn 1994; 14:689.</a></li><li><a class="nounderline abstract_t">van Zalen-Sprock MM, van Vugt JM, Karsdorp VH, et al. Ultrasound diagnosis of fetal abnormalities and cytogenetic evaluation. Prenat Diagn 1991; 11:655.</a></li><li><a class="nounderline abstract_t">Nicolaides KH, Snijders RJ, Gosden CM, et al. Ultrasonographically detectable markers of fetal chromosomal abnormalities. Lancet 1992; 340:704.</a></li><li><a class="nounderline abstract_t">Rizzo N, Pittalis MC, Pilu G, et al. Distribution of abnormal karyotypes among malformed fetuses detected by ultrasound throughout gestation. Prenat Diagn 1996; 16:159.</a></li><li><a class="nounderline abstract_t">Bijok J, Massalska D, Kucińska-Chahwan A, et al. Complex malformations involving the fetal body wall - definition and classification issues. Prenat Diagn 2017; 37:1033.</a></li><li><a class="nounderline abstract_t">American College of Obstetricians and Gynecologists. ACOG Practice Bulletin No. 88, December 2007. Invasive prenatal testing for aneuploidy. Obstet Gynecol 2007; 110:1459.</a></li><li><a class="nounderline abstract_t">American College of Obstetricians and Gynecologists’ Committee on Practice Bulletins—Obstetrics, Committee on Genetics, Society for Maternal–Fetal Medicine. Practice Bulletin No. 162: Prenatal Diagnostic Testing for Genetic Disorders. Obstet Gynecol 2016; 127:e108.</a></li><li><a class="nounderline abstract_t">Salomon LJ, Sotiriadis A, Wulff CB, et al. Risk of miscarriage following amniocentesis or chorionic villus sampling: systematic review of literature and updated meta-analysis. Ultrasound Obstet Gynecol 2019; 54:442.</a></li><li><a class="nounderline abstract_t">Oneda B, Steindl K, Masood R, et al. Noninvasive prenatal testing: more caution in counseling is needed in high risk pregnancies with ultrasound abnormalities. Eur J Obstet Gynecol Reprod Biol 2016; 200:72.</a></li><li><a class="nounderline abstract_t">Beulen L, Faas BHW, Feenstra I, et al. Clinical utility of non-invasive prenatal testing in pregnancies with ultrasound anomalies. Ultrasound Obstet Gynecol 2017; 49:721.</a></li><li><a class="nounderline abstract_t">Benachi A, Letourneau A, Kleinfinger P, et al. Cell-free DNA analysis in maternal plasma in cases of fetal abnormalities detected on ultrasound examination. Obstet Gynecol 2015; 125:1330.</a></li><li><a class="nounderline abstract_t">Reimers RM, Mason-Suares H, Little SE, et al. When ultrasound anomalies are present: An estimation of the frequency of chromosome abnormalities not detected by cell-free DNA aneuploidy screens. Prenat Diagn 2018; 38:250.</a></li><li><a class="nounderline abstract_t">Committee Opinion No. 640: Cell-Free DNA Screening For Fetal Aneuploidy. Obstet Gynecol 2015; 126:e31. Reaffirmed 2017.</a></li><li><a class="nounderline abstract_t">Yaron Y, Jani J, Schmid M, Oepkes D. Current Status of Testing for Microdeletion Syndromes and Rare Autosomal Trisomies Using Cell-Free DNA Technology. Obstet Gynecol 2015; 126:1095.</a></li><li><a class="nounderline abstract_t">Dungan JS, Klugman S, Darilek S, et al. Noninvasive prenatal screening (NIPS) for fetal chromosome abnormalities in a general-risk population: An evidence-based clinical guideline of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2023; 25:100336.</a></li><li><a class="nounderline abstract_t">van Prooyen Schuurman L, Sistermans EA, Van Opstal D, et al. Clinical impact of additional findings detected by genome-wide non-invasive prenatal testing: Follow-up results of the TRIDENT-2 study. Am J Hum Genet 2022; 109:1140.</a></li><li><a class="nounderline abstract_t">Mohan P, Lemoine J, Trotter C, et al. Clinical experience with non-invasive prenatal screening for single-gene disorders. Ultrasound Obstet Gynecol 2022; 59:33.</a></li><li><a class="nounderline abstract_t">Stevens BK, Nunley PB, Wagner C, et al. Utility of expanded carrier screening in pregnancies with ultrasound abnormalities. Prenat Diagn 2022; 42:60.</a></li><li><a class="nounderline abstract_t">Fonda Allen J, Stoll K, Bernhardt BA. Pre- and post-test genetic counseling for chromosomal and Mendelian disorders. Semin Perinatol 2016; 40:44.</a></li><li><a class="nounderline abstract_t">Shaffer LG, Bui TH. Molecular cytogenetic and rapid aneuploidy detection methods in prenatal diagnosis. Am J Med Genet C Semin Med Genet 2007; 145C:87.</a></li><li><a class="nounderline abstract_t">Weise A, Liehr T. Rapid Prenatal Aneuploidy Screening by Fluorescence In Situ Hybridization (FISH). Methods Mol Biol 2019; 1885:129.</a></li><li><a class="nounderline abstract_t">de Wit MC, Srebniak MI, Govaerts LC, et al. Additional value of prenatal genomic array testing in fetuses with isolated structural ultrasound abnormalities and a normal karyotype: a systematic review of the literature. Ultrasound Obstet Gynecol 2014; 43:139.</a></li><li><a class="nounderline abstract_t">Hillman SC, Pretlove S, Coomarasamy A, et al. Additional information from array comparative genomic hybridization technology over conventional karyotyping in prenatal diagnosis: a systematic review and meta-analysis. Ultrasound Obstet Gynecol 2011; 37:6.</a></li><li><a class="nounderline abstract_t">Hillman SC, McMullan DJ, Hall G, et al. Use of prenatal chromosomal microarray: prospective cohort study and systematic review and meta-analysis. Ultrasound Obstet Gynecol 2013; 41:610.</a></li><li><a class="nounderline abstract_t">Committee on Genetics and the Society for Maternal-Fetal Medicine. Committee Opinion No.682: Microarrays and Next-Generation Sequencing Technology: The Use of Advanced Genetic Diagnostic Tools in Obstetrics and Gynecology. Obstet Gynecol 2016; 128:e262. Reaffirmed 2019.</a></li><li><a class="nounderline abstract_t">Gray KJ, Wilkins-Haug L. Special issue on "Feto-Maternal Genomic Medicine": a decade of incredible advances. Hum Genet 2020; 139:1119.</a></li><li><a class="nounderline abstract_t">Gray KJ, Wilkins-Haug LE, Herrig NJ, Vora NL. Fetal phenotypes emerge as genetic technologies become robust. Prenat Diagn 2019; 39:811.</a></li><li><a class="nounderline abstract_t">Krakow D, Lachman RS, Rimoin DL. Guidelines for the prenatal diagnosis of fetal skeletal dysplasias. Genet Med 2009; 11:127.</a></li><li><a class="nounderline abstract_t">Xue Y, Sun A, Mekikian PB, et al. FGFR3 mutation frequency in 324 cases from the International Skeletal Dysplasia Registry. Mol Genet Genomic Med 2014; 2:497.</a></li><li><a class="nounderline abstract_t">Croonen EA, Nillesen WM, Stuurman KE, et al. Prenatal diagnostic testing of the Noonan syndrome genes in fetuses with abnormal ultrasound findings. Eur J Hum Genet 2013; 21:936.</a></li><li><a class="nounderline abstract_t">Rabbani B, Tekin M, Mahdieh N. The promise of whole-exome sequencing in medical genetics. J Hum Genet 2014; 59:5.</a></li><li><a class="nounderline abstract_t">Vora NL, Norton ME. Prenatal exome and genome sequencing for fetal structural abnormalities. Am J Obstet Gynecol 2023; 228:140.</a></li><li><a class="nounderline abstract_t">Mellis R, Oprych K, Scotchman E, et al. Diagnostic yield of exome sequencing for prenatal diagnosis of fetal structural anomalies: A systematic review and meta-analysis. Prenat Diagn 2022; 42:662.</a></li><li><a class="nounderline abstract_t">Best S, Wou K, Vora N, et al. Promises, pitfalls and practicalities of prenatal whole exome sequencing. Prenat Diagn 2018; 38:10.</a></li><li><a class="nounderline abstract_t">Horn R, Parker M. Opening Pandora's box?: ethical issues in prenatal whole genome and exome sequencing. Prenat Diagn 2018; 38:20.</a></li><li><a class="nounderline abstract_t">Harris S, Gilmore K, Hardisty E, et al. Ethical and counseling challenges in prenatal exome sequencing. Prenat Diagn 2018; 38:897.</a></li><li><a class="nounderline abstract_t">Monaghan KG, Leach NT, Pekarek D, et al. The use of fetal exome sequencing in prenatal diagnosis: a points to consider document of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2020; 22:675.</a></li><li><a class="nounderline abstract_t">Miceikaite I, Fagerberg C, Brasch-Andersen C, et al. Comprehensive prenatal diagnostics: Exome versus genome sequencing. Prenat Diagn 2023; 43:1132.</a></li><li><a class="nounderline abstract_t">Van den Veyver IB, Chandler N, Wilkins-Haug LE, et al. International Society for Prenatal Diagnosis Updated Position Statement on the use of genome-wide sequencing for prenatal diagnosis. Prenat Diagn 2022; 42:796.</a></li><li><a class="nounderline abstract_t">Shaffer LG, Rosenfeld JA, Dabell MP, et al. Detection rates of clinically significant genomic alterations by microarray analysis for specific anomalies detected by ultrasound. Prenat Diagn 2012; 32:986.</a></li><li><a class="nounderline abstract_t">Wapner RJ, Martin CL, Levy B, et al. Chromosomal microarray versus karyotyping for prenatal diagnosis. N Engl J Med 2012; 367:2175.</a></li><li><a class="nounderline abstract_t">Society for Maternal-Fetal Medicine (SMFM). Electronic address: <span class="__cf_email__" data-cfemail="601015021320130d060d4e0f1207">[email protected]</span>, Prabhu M, Kuller JA, Biggio JR. Society for Maternal-Fetal Medicine Consult Series #57: Evaluation and management of isolated soft ultrasound markers for aneuploidy in the second trimester: (Replaces Consults #10, Single umbilical artery, October 2010; #16, Isolated echogenic bowel diagnosed on second-trimester ultrasound, August 2011; #17, Evaluation and management of isolated renal pelviectasis on second-trimester ultrasound, December 2011; #25, Isolated fetal choroid plexus cysts, April 2013; #27, Isolated echogenic intracardiac focus, August 2013). Am J Obstet Gynecol 2021; 225:B2.</a></li></ol></div><div id="topicVersionRevision">Topic 104101 Version 32.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16032766" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Should determination of the karyotype be systematic for all malformations detected by obstetrical ultrasound?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7991511" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Karyotype abnormalities in fetuses diagnosed as abnormal on ultrasound before 20 weeks' gestational age.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1837358" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Ultrasound diagnosis of fetal abnormalities and cytogenetic evaluation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1355807" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Ultrasonographically detectable markers of fetal chromosomal abnormalities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8650127" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Distribution of abnormal karyotypes among malformed fetuses detected by ultrasound throughout gestation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28809041" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Complex malformations involving the fetal body wall - definition and classification issues.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18055749" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : ACOG Practice Bulletin No. 88, December 2007. Invasive prenatal testing for aneuploidy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26938573" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Practice Bulletin No. 162: Prenatal Diagnostic Testing for Genetic Disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31124209" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Risk of miscarriage following amniocentesis or chorionic villus sampling: systematic review of literature and updated meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26989803" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Noninvasive prenatal testing: more caution in counseling is needed in high risk pregnancies with ultrasound abnormalities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27515011" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Clinical utility of non-invasive prenatal testing in pregnancies with ultrasound anomalies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26000504" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Cell-free DNA analysis in maternal plasma in cases of fetal abnormalities detected on ultrasound examination.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29436713" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : When ultrasound anomalies are present: An estimation of the frequency of chromosome abnormalities not detected by cell-free DNA aneuploidy screens.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26287791" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Committee Opinion No. 640: Cell-Free DNA Screening For Fetal Aneuploidy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26444108" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Current Status of Testing for Microdeletion Syndromes and Rare Autosomal Trisomies Using Cell-Free DNA Technology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/36524989" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Noninvasive prenatal screening (NIPS) for fetal chromosome abnormalities in a general-risk population: An evidence-based clinical guideline of the American College of Medical Genetics and Genomics (ACMG).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/35659929" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Clinical impact of additional findings detected by genome-wide non-invasive prenatal testing: Follow-up results of the TRIDENT-2 study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34358384" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Clinical experience with non-invasive prenatal screening for single-gene disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34792213" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Utility of expanded carrier screening in pregnancies with ultrasound abnormalities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26718445" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Pre- and post-test genetic counseling for chromosomal and Mendelian disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17290441" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Molecular cytogenetic and rapid aneuploidy detection methods in prenatal diagnosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30506195" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Rapid Prenatal Aneuploidy Screening by Fluorescence In Situ Hybridization (FISH).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23897843" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Additional value of prenatal genomic array testing in fetuses with isolated structural ultrasound abnormalities and a normal karyotype: a systematic review of the literature.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20658510" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Additional information from array comparative genomic hybridization technology over conventional karyotyping in prenatal diagnosis: a systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23512800" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Use of prenatal chromosomal microarray: prospective cohort study and systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27875474" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Committee Opinion No.682: Microarrays and Next-Generation Sequencing Technology: The Use of Advanced Genetic Diagnostic Tools in Obstetrics and Gynecology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32840692" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Special issue on "Feto-Maternal Genomic Medicine": a decade of incredible advances.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31330568" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Fetal phenotypes emerge as genetic technologies become robust.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19265753" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Guidelines for the prenatal diagnosis of fetal skeletal dysplasias.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25614871" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : FGFR3 mutation frequency in 324 cases from the International Skeletal Dysplasia Registry.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23321623" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Prenatal diagnostic testing of the Noonan syndrome genes in fetuses with abnormal ultrasound findings.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24196381" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : The promise of whole-exome sequencing in medical genetics.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/36027950" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Prenatal exome and genome sequencing for fetal structural abnormalities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/35170059" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Diagnostic yield of exome sequencing for prenatal diagnosis of fetal structural anomalies: A systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28654730" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Promises, pitfalls and practicalities of prenatal whole exome sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28695688" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Opening Pandora's box?: ethical issues in prenatal whole genome and exome sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30171820" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Ethical and counseling challenges in prenatal exome sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31911674" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : The use of fetal exome sequencing in prenatal diagnosis: a points to consider document of the American College of Medical Genetics and Genomics (ACMG).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/37355983" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : Comprehensive prenatal diagnostics: Exome versus genome sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/35583085" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : International Society for Prenatal Diagnosis Updated Position Statement on the use of genome-wide sequencing for prenatal diagnosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22847778" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Detection rates of clinically significant genomic alterations by microarray analysis for specific anomalies detected by ultrasound.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23215555" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Chromosomal microarray versus karyotyping for prenatal diagnosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34171388" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : Society for Maternal-Fetal Medicine Consult Series #57: Evaluation and management of isolated soft ultrasound markers for aneuploidy in the second trimester: (Replaces Consults #10, Single umbilical artery, October 2010; #16, Isolated echogenic bowel diagnosed on second-trimester ultrasound, August 2011; #17, Evaluation and management of isolated renal pelviectasis on second-trimester ultrasound, December 2011; #25, Isolated fetal choroid plexus cysts, April 2013; #27, Isolated echogenic intracardiac focus, August 2013).</p>
</a>
</div>
</div>
</div>
</div>
</section>
</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->
<!-- End Footer Area -->
<!-- Back to top -->
<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->
<!-- Prpper JS -->
<!-- Bootstrap Min JS -->
<!-- Classy Nav Min Js -->
<!-- Owl Carousel Min Js -->
<!-- Magnific Popup JS -->
<!-- CounterUp JS -->
<!-- Waypoints JS -->
<!-- Form Validator Min JS -->
<!-- Contact Form Min JS -->
<!-- Main JS -->
<!-- Global site tag (gtag.js) - Google Analytics -->
</body>
</html>
