<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Kidney transplantation in adults: Overview of HLA sensitization and crossmatch testing</title><link href="/z/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/z/d/css/icofont.min.css" rel="stylesheet"/><link href="/z/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/z/d/css/animate.css" rel="stylesheet"/><link href="/z/d/css/owl.carousel.css" rel="stylesheet"/><link href="/z/d/css/magnific-popup.css" rel="stylesheet"/><link href="/z/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/z/d/css/style.css" rel="stylesheet"/><link href="/z/d/css/rtl_edited.css" rel="stylesheet"/><link href="/z/d/css/responsive.css" rel="stylesheet"/><link href="https:///uptodate_assets/main.css" rel="stylesheet"/>
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>
<meta content="" name="description"/>
<link href="/z/d/img/favicon.png" rel="icon" type="image/png">
</link></head>
<body>
<!-- Start Main Menu Area -->
<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>
<section class="teacher-area ptb-10">
<div class="container">
</div>
<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Kidney transplantation in adults: Overview of HLA sensitization and crossmatch testing</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Kidney transplantation in adults: Overview of HLA sensitization and crossmatch testing</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Melissa Y Yeung, MD, FRCPC</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Daniel C Brennan, MD, FACP</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Albert Q Lam, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> May 09, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H2265181887"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>The goal of histocompatibility testing is to determine the immunologic risk of a transplant recipient in the context of a potential donor. If the transplant is between genetically different individuals, the allogeneic graft (allograft) is recognized as foreign predominantly because of differences between donor/recipient major histocompatibility complex (MHC) molecules, which are also known as the human leukocyte antigens (HLAs). The ensuing immune response occurs through two major mechanisms: T cell-mediated (cellular) responses and antibody-mediated (humoral) responses. Existing histocompatibility testing focuses primarily on predicting antibody-mediated alloimmune responses.</p><p>The figure (<a class="graphic graphic_figure graphicRef120933" href="/z/d/graphic/120933.html" rel="external">figure 1</a>) depicts how HLA testing is utilized in the evaluation of a potential transplant recipient. Prior to transplantation, HLA typing is performed to assess the degree of donor/recipient mismatching, and anti-HLA antibody screening and crossmatching are performed to evaluate the recipient's likelihood of rejecting a graft from this donor. If no potential living donors are available, pretransplant HLA testing is also used to determine a candidate's chance of receiving a transplant from a deceased donor and to set criteria to avoid transplantation when the recipient has preformed immunologic memory against a particular donor's HLA antigens (ie, listing of "unacceptable antigens"). Posttransplantation, monitoring for the presence of donor-specific anti-HLA antibodies (DSAs) is useful in the diagnosis of antibody-mediated rejection (ABMR) as well as evaluation of a patient's risk of rejection or response to therapy. (See  <a class="medical medical_review" href="/z/d/html/7326.html" rel="external">"Kidney transplantation in adults: Prevention and treatment of antibody-mediated rejection", section on 'Prevention'</a>.)</p><p>This topic will review the utility, methodology, and limitations of assays used in histocompatibility testing. Further implications of these tests on the management of patients are discussed elsewhere:</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/z/d/html/7355.html" rel="external">"Kidney transplantation in adults: HLA matching and outcomes"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/z/d/html/7326.html" rel="external">"Kidney transplantation in adults: Prevention and treatment of antibody-mediated rejection"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/z/d/html/7351.html" rel="external">"Kidney transplantation in adults: HLA-incompatible transplantation"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/z/d/html/113661.html" rel="external">"Kidney transplantation in adults: ABO-incompatible transplantation"</a>.)</p><p></p><p class="headingAnchor" id="H2986880598"><span class="h1">OVERVIEW OF HLA</span><span class="headingEndMark"> — </span>The recognition of a graft as foreign/non-self occurs through the host's immune system's ability to detect mismatched donor antigens, including differences in the major histocompatibility complex (MHC) molecules (also known as human leukocyte antigens [HLAs]) expressed on the graft. In humans, the MHC genes are encoded on the short arm of chromosome 6 and include the class I genes <em>HLA-A</em>, <em>HLA-B</em>, and <em>HLA-C</em>, and the class II genes <em>HLA-DPA1, HLA-DPB1</em>, <em>HLA-DQA1, HLA-DQB1</em>, <em>HLA-DRA</em>, <em>HLA</em>-<em>DRB1</em>, <em>HLA</em>-<em>DRB3</em>, <em>HLA</em>-<em>DRB4</em>, and <em>HLA</em>-<em>DRB5</em> (<a class="graphic graphic_figure graphicRef120935" href="/z/d/graphic/120935.html" rel="external">figure 2</a>). Class I molecules are expressed on all nucleated cells, whereas class II molecules are expressed primarily on antigen-presenting cells (such as B cells, dendritic cells, and macrophages) but can be expressed under inflammatory conditions on a variety of cell types including endothelial cells and epithelial cells. MHC molecules are highly polymorphic and each MHC locus has 1000 to 5000 allelic variants, encoding different molecules (<a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ebi.ac.uk%2Fipd%2Fimgt%2Fhla%2F&amp;token=q%2BtFDlZ3%2BJRQXZSWNuOsHGJTIuvsQSxdHQh6xWs%2BJxdWW%2F7JAKtYHhAKObtzNQyb&amp;TOPIC_ID=116878" target="_blank">IPD-IMGT/HLA database</a>). This extensive polymorphism is what creates a major obstacle to successful transplantation and the need for immunosuppression to avoid rejection.</p><p class="headingAnchor" id="H3409242854"><span class="h2">Molecular structure</span><span class="headingEndMark"> — </span>Class I MHC molecules are comprised of a polymorphic alpha chain, which consists of three external domains, a transmembrane region, and an intracellular domain, noncovalently associated with a non-polymorphic beta-2 microglobulin chain (<a class="graphic graphic_figure graphicRef120934" href="/z/d/graphic/120934.html" rel="external">figure 3</a>). Class II MHC molecules are heterodimers consisting of two polypeptide chains, an alpha chain and a beta chain, each with a transmembrane domain, two external domains, and a short intracellular domain. HLA-DR, -DQ, and -DP molecules have polymorphic beta chains. HLA-DR molecules have a conserved alpha chain, while HLA-DQ and HLA-DP have polymorphic alpha chains (<a class="graphic graphic_figure graphicRef120934" href="/z/d/graphic/120934.html" rel="external">figure 3</a>). Both classes of molecules contain a peptide-binding groove to which specific peptides bind for presentation to T cells.</p><p>Despite the high degree of polymorphism among the MHC genes, only certain regions of the MHC molecule appear to be immunogenic (ie, capable of eliciting an antibody response). For the class I molecules, this corresponds primarily to the region encoded by exons 2 and 3 of the alpha chain (the alpha 1 and alpha 2 subunits, which are farthest from the cell membrane). MHC molecules that differ only in regions outside of these antigenic sites tend to appear "serologically identical" and are considered serologic antigen equivalents (serotypes). For class II molecules, the most antigenic regions are encoded by exon 2 of the beta chain for the HLA-DR molecules and exon 2 of the alpha and beta chains for HLA-DQ and -DP. While the alpha chain of HLA-DQ and -DP molecules is also highly polymorphic, most identified antibodies appear to be directed against the beta chain. However, there are reports of anti-HLA antibodies directed against class II alpha chain alone or against the tertiary structure of the alpha and beta chains together. (See <a class="local">'Screening for anti-HLA antibodies'</a> below.)</p><p>The part of the molecule to which an antibody can bind is known as an epitope, and each HLA antigen may have more than one epitope. In addition, HLA antigens may share a common ("public") epitope and belong to a cross-reactive epitope group (CREG).</p><p class="headingAnchor" id="H3875869720"><span class="h2">Genetic structure</span><span class="headingEndMark"> — </span>The HLA genes are encoded on chromosome 6 and are inherited in Mendelian fashion, with one linked set of genes (haplotype) inherited intact from each parent. Thus, a child will express one representative set of antigens from each of the class I and class II loci of each parent since HLA molecules are codominantly expressed. By definition, a child is a one-haplotype match with each parent. Statistically, a child has a 25 percent chance of sharing two inherited haplotypes with a sibling, a 50 percent chance of sharing one haplotype with a sibling, and a 25 percent chance of being a zero-haplotype match with a sibling.</p><p>An individual who shares all HLA antigens with another is said to be phenotypically identical. If these individuals also share the same allelic variants that encode for these antigens, they are considered to be genotypically identical. However, if they are not known to be of defined common descent, such individuals then have shared genotype "by state" but not necessarily "by descent." This can be observed in the case of deceased donors who are HLA matched at all loci with a transplant candidate.</p><p class="headingAnchor" id="H502703427"><span class="h2">HLA mismatches</span><span class="headingEndMark"> — </span>In solid organ transplantation, a "mismatch" refers to an HLA antigen found on the cells of the donor allograft but not in the recipient. The greater the disparity between the donor and recipient, the more "foreign" the allograft appears and the higher the likelihood of developing an alloimmune response. For the purpose of kidney transplant allocation, only the <em>HLA-A</em>, <em>HLA-B</em>, and <em>HLA-DR</em> loci are compared between a donor and recipient. Thus, a zero-antigen mismatch refers to concordance at these loci but does not rule out disparities at the others (<em>HLA-C</em>, <em>HLA-DP</em>, or <em>HLA-DQ</em>). Since HLA genes are inherited as a set, related donor/recipient pairs are likely to also share the same antigens at the other loci. As an example, a two-haplotype match would indicate that the pair is not only a zero-antigen mismatch at the <em>HLA-A</em>, <em>HLA</em>-<em>B</em>, and <em>HLA-DR</em> loci but is also matched at the <em>HLA-C</em>, <em>HLA-DP</em>, and <em>HLA-DQ</em> loci. However, if the donor and recipient are unrelated, such conclusions cannot be drawn. The clinical implications of HLA matching on organ allocation and kidney allograft survival are discussed elsewhere. (See  <a class="medical medical_review" href="/z/d/html/7355.html" rel="external">"Kidney transplantation in adults: HLA matching and outcomes"</a>.)</p><p class="headingAnchor" id="H2628804050"><span class="h2">HLA nomenclature</span><span class="headingEndMark"> — </span>HLA typing data are reported at the antigen or allele level. For the purpose of matching for kidney transplantation, most HLA typing is reported at the antigen level. Serologic typing has been replaced with molecular typing with capability of reporting at allele level. HLA allele names are reported with the gene (locus) designation, followed by an asterisk to denote that it was typed by molecular methods (<a class="graphic graphic_figure graphicRef109457" href="/z/d/graphic/109457.html" rel="external">figure 4</a>); colons separate the remaining fields. A high-resolution, "full" HLA typing may include up to four fields:</p><p class="bulletIndent1"><span class="glyph">●</span>The numeric digits following the asterisk (first field) correspond to the HLA allele group. Historically, this first field was intended to identify the serologic (antigen) equivalent of the expressed molecule. While this is still largely true, the naming convention for this first field has undergone changes with the vast number of new HLA alleles for which reference cells may have not been serologically identified. Many of the newly discovered alleles have been given first-field identifiers based solely on apparent sequence homology to known serological variants.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The second set of digits (second field) corresponds to a unique HLA protein; alleles that encode a unique amino acid sequence will have differing digits in this field.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The third field discriminates among alleles with synonymous nucleotide substitutions that translate into the same amino acid sequence.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The fourth field denotes differences in the noncoding region. Finally, a suffix may be used to identify changes in cell surface expression (eg, N = null allele and L = low expression).</p><p></p><p>In solid organ transplantation, we typically focus only on differences that identify serologically distinct proteins. Occasionally, differences in the second field (representing differences in amino acid sequence and, therefore, a unique protein) are considered since these differences could potentially elicit differing immunologic responses. As an example, <em>HLA-A*02:01</em> and <em>HLA-A*02:05</em> belong to the same serologic group, and the antigen equivalent name would be HLA-A2. However, they do differ in amino acid sequence, and there are described cases in which an antibody found to react against one does not appear to react against the other. As distinct reactivities are discovered, the "parent" antigen is split into narrower specificities. As an example, <em>HLA-A*02:03</em> used to be considered as the same serological antigen equivalent (A2) but has been identified as distinct and is now reclassified as its own serologic antigen (A203).</p><p class="headingAnchor" id="H2186004628"><span class="h1">HLA TYPING</span><span class="headingEndMark"> — </span>Accurate typing of human leukocyte antigens (HLAs) is essential in determining the degree of mismatch between donor and recipient and in avoiding the transplantation of organs from donors expressing HLA antigens against which the recipient has preformed antibody. Historically, HLA typing of donors for kidney transplantation focused on <em>HLA-A</em>, <em>HLA-B</em>, <em>HLA-DR</em> and <em>HLA-DQ</em> loci. This was, in part, due to constraints in the availability of typing methodology but also because preformed antibodies directed against these loci were thought to elicit the strongest alloimmune responses and could result in hyperacute/accelerated rejection. HLA typing was initially performed using serologic-based assays, but this has since been replaced by the use of DNA-based molecular techniques, which allow for higher resolution and more accurate typing of all loci (<em>HLA-A</em>, <em>HLA</em>-<em>B</em>, <em>HLA</em>-<em>C</em>, <em>HLA</em>-<em>DRB1</em>, <em>HLA</em>-<em>DRB3/4/5</em>, <em>HLA</em>-<em>DQA</em>, <em>HLA</em>-<em>DQB</em>, and <em>HLA</em>-<em>DPB</em>).</p><p>As of October 2020, 27,599 classical HLA class I and class II alleles, encoding over 16,000 distinct HLA proteins have been recorded in the <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ebi.ac.uk%2Fipd%2Fimgt%2Fhla%2F&amp;token=q%2BtFDlZ3%2BJRQXZSWNuOsHGJTIuvsQSxdHQh6xWs%2BJxdWW%2F7JAKtYHhAKObtzNQyb&amp;TOPIC_ID=116878" target="_blank">IPD-IMGT/HLA database</a>. This remarkable allelic diversity makes high-resolution HLA typing very challenging. High-resolution typing involves a more precise determination of an allelic variant through methodology that can sequence the entire amino acid length of the protein. Low-resolution DNA typing refers to typing results that resolve differences at the antigen level. For solid organ transplantation, only low-resolution typing is required. (See <a class="local">'HLA nomenclature'</a> above.)</p><p>In the United States, HLA typing of all loci by molecular methods is now mandated by the United Network for Organ Sharing (UNOS). Although allocation of the organ is based primarily upon matching for <em>HLA-A</em>, <em>HLA-B </em>and <em>HLA-DR</em>, allocation algorithms for deceased-donor kidney transplants now take into consideration all loci when determining a patient's suitability for transplantation from a particular donor based upon the presence or absence of preformed donor-specific anti-HLA antibodies (DSAs). However, existing algorithms still only consider the <em>HLA-A</em>, <em>HLA-B</em>, and <em>HLA</em>-<em>DR</em> loci for providing priority points for zero-antigen mismatches. (See  <a class="medical medical_review" href="/z/d/html/7355.html" rel="external">"Kidney transplantation in adults: HLA matching and outcomes"</a>.)</p><p>The specific methodology used for HLA typing in solid organ transplantation differs between HLA laboratories. Considerations in choosing a particular method include the level of resolution, ability to resolve ambiguities, turnaround time, cost, and expertise. HLA typing of deceased donors is typically performed using real-time polymerase chain reaction (RT-PCR) or premade, reverse sequence-specific oligonucleotide probes (rSSO) trays. The rapid turnaround time required precludes the use of high-resolution typing methods (eg, sequence-based typing, next-generation sequencing [NGS]), which may be considered for use in typing of recipients or in living-donor evaluations. Serologic-based assays are still utilized by some laboratories outside of North America and Europe.</p><p class="headingAnchor" id="H3155166486"><span class="h2">Serologic methods</span><span class="headingEndMark"> — </span>Serologic-based typing was the first standard method used to obtain donor and recipient HLA typing and was employed for more than 30 years. This technique uses a panel of reference sera (usually from multiparous women) that are known to contain antibodies to various HLA antigens. Lymphocytes from the donor or recipient are added to several wells of plates containing different sera. Following an initial incubation step to allow binding between antibody and antigen, complement is added to wells, and a viability dye allows detection of cell lysis. The presence of dead cells is a positive test. Comparison of the serologic specificities of the different sera that reacted allows one to assign the HLA type. Major limitations of the assay included the following:</p><p class="bulletIndent1"><span class="glyph">●</span>Large panels of sera with sufficient antibody strength and specificity were required to reliably identify the large number of HLA specificities.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Antisera were seldom monospecific and contained antibodies directed against more than one specific HLA molecule, resulting in inconclusive patterns of reactivity.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Typing for antigens with decreased cell surface expression (such as HLA-C and HLA-DP antigens) was difficult.</p><p></p><p>For these reasons, most laboratories have moved to using molecular methods for HLA typing.</p><p class="headingAnchor" id="H1328711670"><span class="h2">DNA-based molecular methods</span><span class="headingEndMark"> — </span>Two major developments facilitated the use of DNA-based molecular methods to more accurately determine an individual's HLA typing: the extensive sequencing of the polymorphic regions of the HLA genes (compiled and curated in the <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ebi.ac.uk%2Fipd%2Fimgt%2Fhla%2F&amp;token=q%2BtFDlZ3%2BJRQXZSWNuOsHGJTIuvsQSxdHQh6xWs%2BJxdWW%2F7JAKtYHhAKObtzNQyb&amp;TOPIC_ID=116878" target="_blank">IPD-IMGT/HLA database</a>) and the advent of the PCR technique. DNA-based molecular methods include the following:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Sequence-specific primers (SSP) typing</strong> – This form of typing involves the use of primers designed to recognize a particular HLA sequence (allele) or group of similar alleles, such that the polymorphism to be detected is located at the 3' end of the primer. DNA is first extracted from a blood sample and amplified by PCR using these primers. If both primers are able to bind to the DNA, then amplification occurs and can be detected by gel electrophoresis. The pattern of amplicons that are present allows for the assignment of the HLA genotype.</p><p></p><p class="bulletIndent1">Commercially available typing kits for class I genes have primer sets that detect polymorphisms in exons 2 and 3, whereas kits for class II genes detect polymorphisms in exon 2. These regions have been selected because they cover most of the known polymorphisms among the various alleles. The SSP method can be used for either low-resolution typing, by identifying allele groups of a particular antigen, or for high-resolution typing, which identifies a specific allele. A major advantage of the SSP method is its rapid turnaround time of two to three hours, making it suitable for use when rapid results are required, such as in the typing of deceased donors. However, it is not well suited for typing large numbers of samples. In addition, the ever-increasing number of HLA alleles makes it difficult to resolve even major allele groups with a one-step process. Even with the use of additional primer sets, ambiguities may remain, making high-resolution typing increasingly difficult. Furthermore, polymorphisms outside of the sequenced exons cannot be resolved.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Sequence-specific oligonucleotide probes (SSOP) typing</strong> – With SSOP typing, DNA is amplified using a set of primers that recognize a particular HLA locus. As with SSP typing, primers are designed to amplify the most polymorphic regions of the HLA gene (exons 2 and 3 for class I genes and exon 2 for class II genes). However, primers used in SSOP for DNA amplification are less specific than those used in SSP and cannot distinguish between particular alleles or groups of similar alleles. To discriminate between alleles within a locus, the amplified DNA is blotted into a membrane, and labeled SSOPs that recognize specific sequences are then added. A chemiluminescent or colorimetric reaction is then used to reveal the presence of bound oligonucleotide, and the pattern of positive reactions reveals the HLA typing. Additional probes can be added to another membrane with the amplified DNA to resolve any ambiguities.</p><p></p><p class="bulletIndent1">SSOP typing is well suited for typing large numbers of samples, with each assay capable of testing 80 to 180 samples but takes approximately two days to obtain results. For fewer samples, a modified assay known as <strong>reverse-SSO (rSSO)</strong> can be used. In this rSSO technique, oligonucleotide probes are bound to the membranes, with each membrane having all the SSOs bound that are required for typing of a particular HLA locus or allele group. Amplified, and then biotinylated, DNA is added to these commercially available, premade membranes. Hybridized amplicons are then detected using a chemiluminescent or color-based detection system. Oligonucleotide probes can also be linked to beads (solid phase hybridization) to allow for multiplex analysis of HLA typing using the Luminex platform. rSSO typing still has the same drawbacks of ambiguity resolution as the SSO method but allows for a much faster turnaround time.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Real-time PCR (RT-PCR)-based typing</strong> – This form of typing is based upon the use of allele-specific PCR similar to SSP methods. However, instead of using gel electrophoresis, amplicons are detected in real time with the use of fluorescent dyes or probes. Each well contains sequence-specific primers such that if the allele is present, the DNA becomes amplified. The added cyanine dye binds to any amplified, double-stranded DNA and fluoresces. Fluorescent readings of each well are obtained at different temperatures. As the temperature increases, the DNA dissociates ("melts") and the fluorescence decreases. The resulting melt-curve analysis allows for easy visualization of the presence or absence of specific alleles, and the pattern of reactive wells determines the HLA typing.</p><p></p><p class="bulletIndent1">Alternatively, labeled SSOPs can be used instead of the less specific cyanine dye. These probes are designed to bind to a location between the primers and are labeled with a fluorescent reporting dye. If the primer is able to bind to the DNA present, DNA polymerase will begin to copy the DNA. Once it reaches the site of the probe bound to the same strand of DNA, the dye becomes released and is detected. With successive rounds of amplification, more dye is released. This fluorescence is monitored in real time and used to determine whether the reaction is "positive." The benefit of using a probe-based approach is that it allows for different fluorescent reporter molecules to be used in the same reaction well, allowing for a greater degree of multiplexing. In addition, since the specificity of the reaction is not only dependent upon the primer pairs but also upon the binding of the sequence-specific probe, this increases the typing resolution of the assay. Use of RT-PCR-based typing shortens turnaround time to approximately one hour and requires much less hands-on involvement of the technician. Automated data interpretation also simplifies the analysis significantly.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Sequence-based typing (SBT)</strong> – SBT, or Sanger-based sequencing, is based upon the direct amplification and sequencing of the relevant exons using fluorescently labeled dideoxynucleotides. Since this method deciphers the specific nucleotide sequence of the amplified region, it allows for a high resolution typing. This sequence is then compared with known sequences of HLA alleles in the IPD-MGT/HLA database to assign the HLA typing. However, in samples with heterozygous alleles, it can be difficult to assign the base calls to one allele or the other, leading to potential ambiguities.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Next-generation sequencing (NGS)</strong> – The advent of NGS techniques has enabled high-resolution typing with significantly reduced ambiguity as it allows for base calls to be assigned to the same (cis) or different (trans) alleles (also known as phasing). Although this technology has been increasingly adopted for use in hematopoietic stem cell transplantation, limitations in throughput, scalability, cost, and speed restrict its use in some solid organ transplantation programs.</p><p></p><p class="headingAnchor" id="H3799008790"><span class="h1">SCREENING FOR ANTI-HLA ANTIBODIES</span><span class="headingEndMark"> — </span>Nearly 30 percent of waitlisted patients are known to have antibodies directed against one or more human leukocyte antigens (HLAs) [<a href="#rid1">1</a>], as a result of sensitization from prior exposures to HLA antigens, such as pregnancies, blood transfusions, and previous transplantation. The purpose of screening patients for anti-HLA antibodies prior to transplantation is to detect preexisting anti-HLA antibodies, define their specificity, and determine their relative strength. This information then aids the clinician in assessing the patient's likelihood of receiving an HLA-compatible transplant and classifying transplant candidates at higher immunological risk who may benefit from a more aggressive immunosuppressive regimen and/or more intensive posttransplantation monitoring.</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/z/d/html/7351.html" rel="external">"Kidney transplantation in adults: HLA-incompatible transplantation", section on 'Approach to the sensitized transplant candidate'</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/z/d/html/7353.html" rel="external">"Kidney transplantation in adults: Induction immunosuppressive therapy", section on 'Assessment of immunologic risk'</a>.)</p><p></p><p>Donor-specific anti-HLA antibodies (DSAs) detected by cell-based cytotoxicity assays are considered an absolute contraindication to transplantation because of their propensity to cause hyperacute rejection [<a href="#rid2">2</a>]. DSAs detected by more sensitive assays (eg, enzyme-linked immunosorbent assay [ELISA], flow cytometry, or bead-based assays) represent varying degrees of risk [<a href="#rid3">3</a>]. Although preexisting DSA is associated with increased risk of rejection and graft loss, it remains controversial whether antibodies that are detected exclusively by solid phase techniques influence long-term graft outcome [<a href="#rid4">4-9</a>]. On the one hand, avoiding all antigens against which the patient has even low-level antibody will result in a better "matched" kidney for the recipient and may improve the likelihood of long-term graft survival since low-level antibody is indicative of a prior exposure and immunological memory against the antigen. However, this approach may also restrict a patient's access to organs, leading to longer wait times and increased risk of morbidity and mortality while on the waitlist.</p><p>Previously, only preformed antibodies directed against HLA-A, -B, and -DR antigens were considered risks for antibody-mediated rejection (ABMR) and reduced graft survival. However, all HLA proteins including HLA-C, -DQ, and -DP are now recognized as antigenic targets with the potential of eliciting antibody responses, although it remains unclear whether antibodies directed against these loci have a similar impact on graft outcome compared with those targeting the HLA-A, -B, or -DR antigens. (See  <a class="medical medical_review" href="/z/d/html/7355.html" rel="external">"Kidney transplantation in adults: HLA matching and outcomes"</a>.)</p><p class="headingAnchor" id="H1762785739"><span class="h2">Assays for antibody screening</span></p><p class="headingAnchor" id="H4130568781"><span class="h3">Cell-based cytotoxicity assays</span><span class="headingEndMark"> — </span>Historically, anti-HLA antibodies were identified by testing recipient sera against a panel of donor cells that are representative of the HLA antigen frequency within the donor population. Recipient serum is mixed with donor lymphocytes, along with exogenous complement and a viability dye. If the serum contains antibody capable of binding to the donor cells and fixing complement, cell death occurs. The pattern of reactivity is used to calculate the recipient's degree of sensitization and likelihood of transplantation. As an example, if cell death is observed in 45 out of the 60 different cell donors in the panel, the recipient has a panel of reactive antibody (PRA) of 75 percent and would be ineligible to receive a graft from 75 percent of the donor population on the basis of having preformed DSA that would likely result in a positive cytotoxic crossmatch against these donors.</p><p>A major limitation of this assay is the difficulty in defining the specificity of the anti-HLA antibody, particularly for highly sensitized individuals, as the statistical likelihood of being able to define specificity depends upon a representation of every target antigen exclusive of others. An even more daunting limitation of the cytotoxic panel for antibody screening is that broadly reactive antibodies against non-HLA antigens can make analysis impossible. False-positive results could arise from the presence of non-HLA antibodies or immunoglobulin M (IgM) HLA and non-HLA antibodies, whereas false-negative results can occur with low titer antibody. Each donor cell has up to 12 distinct HLA molecules (two from each of the <em>HLA-A</em>, <em>HLA-B</em>, <em>HLA-C</em>, <em>HLA-DP</em>, <em>HLA-DQ, HLA-DR</em> loci) against which the serum may be reacting, and recipients often have antibodies directed against multiple HLA antigens. Even with careful examination of the patterns of reactivity and larger panels of cells, it is often impossible to determine their precise specificities. For accurate determination of PRA, panels should contain cells from volunteers that are representative of the donor pool but often include only the most common phenotypes. For these reasons, most HLA laboratories worldwide have moved to using newer technologies for antibody screening. Since 2009, the United Network for Organ Sharing (UNOS) has mandated the use of solid phase assays to identify HLA antibodies in potential transplant recipients in the United States; however, as noted below, solid phase assays are a completely different method that may not be concordant with cytotoxicity results [<a href="#rid10">10</a>].</p><p class="headingAnchor" id="H2474555561"><span class="h3">Solid phase assays</span><span class="headingEndMark"> — </span>The advent of solid phase multiplex technology has enabled the identification of anti-HLA antibody specificities with high sensitivity and specificity. In these assays, recipient serum is added to a cocktail of polystyrene beads, to which purified HLA antigens are attached (<a class="graphic graphic_figure graphicRef120936" href="/z/d/graphic/120936.html" rel="external">figure 5</a>). A fluorochrome-conjugated anti-immunoglobulin G (IgG) detection antibody is then added, and the presence of anti-HLA IgG isotype antibody is identified by flow cytometric methods (Luminex).</p><p>Often, for cost-saving purposes, laboratories will first screen sera using pooled antigen or phenotype beads that are coated with multiple HLA antigens. If this screening test is positive, then the <strong>single-antigen bead (SAB) assay</strong> is used to determine the precise specificity of the HLA antigen against which the antibody is directed. A single assay allows for detection of antibodies directed against up to 100 distinct HLA molecules, each uniquely expressed on a particular bead impregnated with two fluorescent dyes. Bead specificities are determined by the combination of these identifying signals and the presence of alloantibody detected by a third "reporter" channel (<a class="graphic graphic_figure graphicRef120937" href="/z/d/graphic/120937.html" rel="external">figure 6</a>). The degree of fluorescence exhibited by the presence of alloantibody in this reporter channel is resulted in terms of its <strong>median fluorescence intensity (MFI)</strong> and can provide some clue as to the amount and strength of alloantibody present. Although results are provided as a numerical value, the MFI value cannot be used as a quantitative method. In addition, MFI thresholds above which an antibody is considered "positive" are not standardized. Clinicians should discuss how best to interpret results within their HLA laboratory.</p><p class="bulletIndent1"><span class="glyph">●</span>(See <a class="local">'Determining the amount/strength of antibody'</a> below.)</p><p class="bulletIndent1"><span class="glyph">●</span>(See <a class="local">'Defining a positive result'</a> below.)</p><p></p><p class="headingAnchor" id="H4037221021"><span class="h4">Specificity of solid phase assays</span><span class="headingEndMark"> — </span>Solid phase assays are clearly more sensitive than cytotoxic assays. However, traditional solid phase assays that utilize a secondary antibody that recognizes only IgG do not detect IgM antibodies against HLA even though IgM anti-HLA antibodies do cause a positive crossmatch. Solid phase assays may also demonstrate false-positive results because of reactivity against the latex beads, denatured HLA antigen, or non-HLA protein used to coat the beads. Precise determination of the specificity of anti-HLA antibodies present in a patient's serum can help to identify which HLA antigens to avoid when considering a potential donor. Existing commercially available kits allow for the detection of antibody against &gt;200 different HLA antigens, including the most common phenotypes in the United States population.</p><p>Most of the class I HLA molecules are represented by a single bead. Among these, some share common epitopes (known as public epitopes) against which a single antibody can react against all beads expressing that particular epitope. Public epitopes are those common to all members of a cross-reactive epitope group (CREG), whereas private epitopes delineate the individual, serologically defined antigens. As an example, B7 and B8 can be recognized by distinct antibodies (private epitopes) but also by a common antibody directed against the BW6 public epitope.</p><p>Class II HLA molecules are expressed as heterodimers consisting of two polymorphic chains (an alpha chain and a beta chain) (<a class="graphic graphic_figure graphicRef120934" href="/z/d/graphic/120934.html" rel="external">figure 3</a>) and are traditionally named according to their beta chain only. The DR-alpha chain is largely monomorphic, and anti-DR antibodies primarily target epitopes on the beta chain. By contrast, DQ and DP antigens are comprised of alpha chains that are highly polymorphic (<a class="graphic graphic_figure graphicRef120935" href="/z/d/graphic/120935.html" rel="external">figure 2</a>), and while most identified antibodies appear to target epitopes on the beta chain, there are reports that some can be directed against the alpha chain or require a particular alpha/beta combination (<a class="graphic graphic_figure graphicRef120934" href="/z/d/graphic/120934.html" rel="external">figure 3</a>). In fact, nearly 25 percent of antibodies that react to DQ antigens require the recognition of both subunits. Thus, in each individual, up to four distinct alpha/beta combinations must be recognized as potential immune targets. Of note, not all alpha/beta combinations of the class II antigens expressed within the United States population are represented in the commercially available kits. (See <a class="local">'Molecular structure'</a> above.)</p><p>Antibodies that react against all loci except for DPA can be listed as an "unacceptable antigen" for deceased-donor organ allocation in the United States. In the case of the DQ locus, DQA and DQB are considered distinct molecules and do not account for the possibility that DQ antibodies recognize epitopes that are dependent upon both chains and are unique to certain alpha and beta combinations. Clinicians should discuss with their HLA laboratories their criteria for identifying these situations. (See <a class="local">'Unacceptable antigens'</a> below.)</p><p class="headingAnchor" id="H3921966074"><span class="h4">Defining a positive result</span><span class="headingEndMark"> — </span>MFI threshold cutoffs are not standardized across HLA laboratories nor are they used in the same manner by individual transplant programs. Each HLA laboratory establishes its threshold cutoffs through validation of the assay using known negative and positive sera to determine a cutoff value that optimizes the true negative and true positive rates of the assay. Some laboratories may also choose to set their cutoffs to better correlate with crossmatch results. Transplant programs may modify thresholds based upon clinical risk and have different cutoffs depending upon the type of organ, the recipient's sensitization history, whether the donor is living or deceased, and if testing is being performed pre- or posttransplantation.</p><p>A study commissioned by Clinical Trials in Organ Transplantation to examine standardization of SAB testing concluded that MFI-positive cutoffs ranging from values 1000 to 1500 yielded a high level of agreement (&gt;90 percent) among HLA laboratories in determining the presence or absence of an HLA antibody [<a href="#rid11">11</a>].</p><p>MFI levels can be affected by a number of technical considerations to the assay, including the density of antigen expressed on the beads, the fluorochrome detection antibody used, and the setup of the flow cytometer or Luminex instrument. As with any other assay, determining the threshold cutoffs is a balance between the sensitivity of the assay and its false-positive rate. As an example, in our laboratory, testing of sera from 20 normal, healthy males with no prior HLA-sensitizing event revealed a 7 percent false-positive rate using an MFI threshold of 1000, which was reduced to 0.5 percent when the cutoff was raised to 3000. However, "true" anti-HLA antibody resulting from a prior exposure can be detected at MFI levels between 1000 and 3000. Examining a patient's screening results longitudinally often allows one to distinguish these true positives, which remain persistent, from false positives, which are sporadic.</p><p>Laboratories may also use different MFI cutoffs for antibodies directed against different HLA loci to account for differences in their ability to elicit a positive crossmatch. The density of HLA antigen expressed on individual beads may not correspond to its density found on cell surfaces. As an example, HLA-C and -DP have 13- to 18-fold less cell surface expression than antigens of the other loci [<a href="#rid12">12,13</a>]. Thus, a higher burden of alloantibody-directed HLA-C/DP is needed to elicit a positive crossmatch [<a href="#rid14">14,15</a>]. For this reason, many laboratories have implemented higher threshold values above which to call an HLA-C or HLA-DP antibody "positive." It remains unclear whether antibodies directed against these loci have a similar impact on graft outcome compared with those targeting the other HLA antigens. However, case reports have identified their ability to cause ABMR [<a href="#rid16">16</a>].</p><p>Clinicians should be aware that inter-assay variability can also affect whether an antibody is considered present or absent if it is detected at an MFI level that is close to the threshold cutoff. Variability has been shown among kits from different manufacturers, different lots of the same kit, and different runs using the same lots. Studies have reported inter-assay differences in MFI values of up to 25 to 50 percent, even when performed by a single laboratory with strict standard operating procedures [<a href="#rid17">17</a>]. In addition, MFI values may not be comparable between different laboratories testing the same sample; even with standardization of reagents and protocols, one study found a 25 percent variability reported by the participating laboratories [<a href="#rid11">11</a>].</p><p class="headingAnchor" id="H1051495851"><span class="h4">Determining the amount/strength of antibody</span><span class="headingEndMark"> — </span>MFI levels on the beads, using undiluted patient sera, represent a relative amount of antibody that is bound to the antigen on the bead and can vary between individual beads. Importantly, MFI values are <strong>not</strong> synonymous with concentration or titer of antibody, and numerous studies have shown their inaccuracy in assessing antibody strength and concentration. MFI levels have been used successfully to predict the likelihood of having a negative crossmatch below a certain threshold (negative predictive value), but their inability to predict a positive crossmatch is much more problematic [<a href="#rid18">18</a>]. Similarly, MFI values of pretransplant DSA are not consistently predictive of graft outcomes.</p><p>However, knowledge of alloantibody burden can often be clinically useful, such as in assessing the efficacy of antibody removal using plasmapheresis in desensitization protocols or in the treatment of ABMR. Ideally, an assay would be able to provide clinicians with a reliable quantification. Unfortunately, attempts to standardize MFI still do not address assay variability. Existing consensus guidelines suggest that quantification of antibody burden is best estimated by titration (serial dilution) studies [<a href="#rid18">18,19</a>].</p><p>Correlation between MFI values and the quantity of antibody present is also more problematic at the assay's upper limit of detection. Beads can become saturated at MFI values of approximately 10,000, such that any additional antibody present in the serum is unable to bind [<a href="#rid20">20</a>]. In these instances, the antibody burden may be underestimated, and two different antibodies with similar MFI values near this upper threshold may have vastly different serum concentrations. Titration studies can be performed in these instances to provide a better estimation of the antibody load. An antibody that is detected out past a 1:128 dilution is present at much higher concentrations than antibody that is detected only to the 1:8 dilution, even if they both have similar MFI values in the neat (undiluted) serum. Monitoring antibody load by titration has been shown to be more accurate and predictable than using MFI values in assessing the effectiveness of desensitization protocols [<a href="#rid21">21</a>] and can also be useful in predicting a patient's likelihood of response.</p><p>In addition, the quantity of bound antigen varies from bead to bead. MFI differences between beads in an individual assay may be due to differing amounts of target antigen that are present, rather than different amounts of antibody, particularly at near-saturation levels [<a href="#rid22">22</a>]. Furthermore, the relationship between antigen density on the beads compared with cell surface expression is not well defined; two different antibodies with the same MFI level may show disparate crossmatch results.</p><p>MFI values may also underestimate the amount of antibody present in the serum sample. This occurs in instances where the alloantibody reacts against a public/shared epitope. Binding of the antibody is distributed across all beads containing antigens expressing the common epitope, effectively "diluting out" the antibody. This leads to much lower MFI values than if there was only a single bead containing the particular epitope and an underestimation of the true antibody burden.</p><p class="headingAnchor" id="H3465296140"><span class="h4">False-positive results</span><span class="headingEndMark"> — </span>Occasionally, the process of generating and coupling of HLA antigens to the beads leads to improper protein conformation and/or denaturation, thereby unveiling epitopes that are not naturally found. Binding of antibody to these neoepitopes can result in a false-positive detection of antibody that has no clinical significance [<a href="#rid23">23-25</a>].</p><p>Binding of the patient's IgG antibody to the latex beads themselves, or to a non-HLA protein used in bead manufacture, can lead to a high background signal and obscure the true results of the assay [<a href="#rid26">26</a>]. This is often detected by the high MFI values of the negative control bead, which does not contain any bound antigen. To eliminate this, sera can be pretreated with adsorption beads to remove interfering factors.</p><p class="headingAnchor" id="H1632280301"><span class="h4">False-negative results</span><span class="headingEndMark"> — </span>The presence of interfering factors in the patient's serum can also lead to under-recognition of DSA. Inhibitors such as various complement components (including C1q and C3/C4 activation products) can bind to the anti-HLA antibody and stereotypically hinder the ability of the detection antibody to bind (the prozone effect) [<a href="#rid27">27,28</a>]. This leads to low MFI readings and the inaccurate conclusion that the alloantibody is absent or low level. The presence of intravenous <a class="drug drug_general" data-topicid="16471" href="/z/d/drug information/16471.html" rel="external">immune globulin</a> (IVIG) in a patient's serum and/or IgM antibody of the same HLA specificity [<a href="#rid29">29</a>] can also mask the recognition of IgG alloantibody. Verified methods [<a href="#rid17">17</a>] that prevent this phenomenon include pretreating the patient's serum with ethylene diamine tetraacetic acid (EDTA) to prevent C1q binding [<a href="#rid30">30,31</a>] and/or titration studies, which dilute out the effect of the inhibitor and reveal the presence of the alloantibody at higher dilutions [<a href="#rid22">22</a>]. Other methods such as dithiothreitol (DTT) treatment or heat inactivation have also been reported.</p><p>False negatives can also occur when a low-level antibody is directed against a public/shared epitope. Since binding of the antibody is dispersed across more than one bead, the MFI value of a single bead underestimates the true amount of the antibody. This scenario may result in a negative SAB test but a positive result when phenotype beads are used or when crossmatches are performed. Thorough examination of SAB histogram reports for alloantibody binding to CREGs can help to identify this phenomenon. (See <a class="local">'Determining the amount/strength of antibody'</a> above.)</p><p class="headingAnchor" id="H603305560"><span class="h3">C1q binding assay</span><span class="headingEndMark"> — </span>The standard SAB assay detects all IgG antibodies, irrespective of their subclass and ability to bind complement. Certain subclasses (IgG1 and IgG3) have been shown to more effectively activate complement and are therefore more likely to cause allograft injury [<a href="#rid32">32</a>]. This has led to the development of a modified SAB technique known as the C1q binding assay. Following addition of the patient's sera to the SAB mixture, exogenous complement is added. The presence of any bound complement is then detected using a fluorescent-conjugated anti-C1q antibody.</p><p>Studies have suggested that complement-fixing IgG DSAs are associated with higher rates of graft loss than non-complement-fixing DSAs, although the latter still portends a worse outcome when compared with recipients without any DSA [<a href="#rid32">32,33</a>]. However, it is unclear whether this is due to the inherent ability to distinguish between the antibody's ability to activate complement or whether positivity simply reflects the amount of antibody [<a href="#rid19">19,33,34</a>] and/or improved sensitivity of the assay to detect alloantibody that may be masked by interfering proteins that affect the standard SAB assay [<a href="#rid35">35</a>]. An antibody detected at MFI values &gt;10,000 on the standard assay correlates strongly with C1q positivity [<a href="#rid33">33</a>], and, when adjusted for MFI values, use of the C1q assay did not appear to distinguish functionally distinct DSA with clinical significance [<a href="#rid19">19,36</a>]. The added utility of the C1q binding assay in kidney transplantation remains unclear, and this test is not routinely used in clinical practice.</p><p class="headingAnchor" id="H730265465"><span class="h2">Determination of patient sensitization</span><span class="headingEndMark"> — </span>The <strong>panel of reactive antibody (PRA)</strong> is a tool that clinicians can use to assess a patient's degree of sensitization to HLA antigens and thereby their likelihood of transplantation. In the past, PRA was determined by the reactivity patterns of the patient's serum against a panel of cells from volunteers with HLA phenotypes that were representative of the transplant donor pool. (See <a class="local">'Cell-based cytotoxicity assays'</a> above.)</p><p>In December 2007, UNOS mandated that this practice be replaced by the <strong>calculated PRA (cPRA)</strong> to provide a more uniform and accountable method for assessing sensitization. The cPRA calculates the likelihood of transplantation by using results of the SAB assay to identify the specificity of the anti-HLA antibodies, in combination with the known frequencies of HLA antigens within the donor population. As an example, if the patient has an antibody against the HLA-A2 antigen, which is present in 48 percent of the United States donor population (phenotypic frequency of the A2 antigen), their cPRA level would be 48 percent, and they would be ineligible to receive 48 percent of the kidneys on the basis of having DSA against the A2 antigen. If the patient had an antibody against B44, which is present in 27 percent of the population, their cPRA would be 27 percent. If the patient had antibodies against <strong>both</strong> A2 and B44, their cPRA level would be 59 percent. This is less than the sum of the individual antigen frequencies because some donors would express both the A2 and B44 antigens (ie, cPRA = the percentage of donors expressing A2 alone + the percentage of donors expressing B44 alone + the percentage of donors expressing both A2 and B44). The cPRA calculator based upon HLA frequencies derived from the United States donor population can be found on the <a class="external" href="/external-redirect?target_url=http%3A%2F%2Foptn.transplant.hrsa.gov%2Fresources%2Fallocation-calculators%2Fcpra-calculator%2F&amp;token=4qF%2FEDfKbtfsUau2uLlIcTEfsD0ezTCOWGCOgi8SgNM9JjJF2rLRbKSyp9Je08Gmz1U%2FbDT8w6NrY6Pig8rekHRKNKnwQA4v6E10sC7bv%2B8%3D&amp;TOPIC_ID=116878" target="_blank">Organ Procurement and Transplantation Network (OPTN) website</a>.</p><p>Patients who are <strong>highly sensitized </strong>(cPRA ≥80 percent) have antibodies against numerous common HLA antigens that result in their ineligibility to &gt;80 percent of the organs. <strong>Broadly sensitized</strong> patients have antibodies against numerous different antigens, but, if these are rare in frequency, they may not have a high cPRA.</p><p>Under the current kidney allocation system in the United States, a candidate's cPRA is used in the algorithm for organ allocation to create greater parity between individuals on the waitlist, regardless of their degree of sensitization. Those with increasing cPRA levels are given additional waitlist "points" to try to minimize differences in waitlist times between those who were highly sensitized compared with those without any anti-HLA antibodies (see  <a class="medical medical_review" href="/z/d/html/7337.html" rel="external">"Kidney transplantation in adults: Organ sharing", section on 'Sensitized candidates'</a>). The match algorithm used by UNOS calculates a candidate's cPRA using the unacceptable antigens list for the patient. (See <a class="local">'Unacceptable antigens'</a> below.)</p><p class="headingAnchor" id="H1256329285"><span class="h2">Unacceptable antigens</span><span class="headingEndMark"> — </span>The term "unacceptable antigen" refers to a donor HLA antigen against which the patient has preformed antibody and should be avoided because of an increased risk of ABMR. Organs from donors who express these unacceptable HLA antigens will not be offered to the patient.</p><p>The presence of preformed DSA is determined by the SAB assay, but individual transplant programs use differing criteria and threshold values by which to call an antigen unacceptable. This decision takes into consideration the balance between the patient's access to transplantation and the degree of immunologic risk that is deemed acceptable. Since even low levels of preformed DSA (if associated with a positive flow crossmatch) can be associated with early and late ABMR (see <a class="local">'Common clinical scenarios'</a> below), listing unacceptable antigens using a stringent threshold will result in a better "matched" kidney for the recipient and may improve long-term graft survival. However, this approach may also restrict a recipient's access to organs, leading to longer wait times and increased risk of morbidity and mortality while on the waitlist. This is an important consideration particularly in highly sensitized candidates. Although the newly implemented kidney allocation system has increased the number of transplants performed in these candidates by giving them "priority points," it is predicted that 25 percent of individuals with cPRA of 100 percent are unlikely to receive a single offer based on their listed unacceptable antigens [<a href="#rid37">37</a>]. In addition, some studies have shown that antibodies that are detected exclusively in sensitive, solid phase assays do not influence long-term graft outcome [<a href="#rid4">4-9</a>]. (See <a class="local">'Common clinical scenarios'</a> below.)</p><p>Unacceptable antigens can now be entered for all HLA loci: <em>HLA-A</em>, <em>HLA-B</em>, <em>HLA-C</em>, <em>HLA-DRB1</em>, <em>HLA-DRB3/4/5</em>, <em>HLA-DQA</em>, <em>HLA-DQB</em>, <em>HLA-DPA</em> and <em>HLA-DPB</em>. In addition, UNOS/OPTN has also recently revised the cPRA calculation to include <em>HLA-DQA1</em>, <em>-DPA1</em>, <em>-DPB1</em>, and allele-level antibodies. The prior omission of these loci in the calculation was shown to disadvantage approximately 8 percent of candidates reporting unacceptable antigens for <em>DQA</em> and <em>DPB</em>. The policy also updated the HLA reference panel used to calculate these frequencies. The new reference panel now uses higher resolution typing that permits the use of genotype instead of haplotype calculations and expands the reference cohort to better approximate the rate of incompatible donors and improve more equitable allocation for the highly sensitized candidates [<a href="#rid38">38</a>]. However, the decision to list certain loci and the threshold cutoffs used to determine whether to consider them as "unacceptable" remains up to the clinical transplant program and can vary widely. (See <a class="local">'Defining a positive result'</a> above.)</p><p class="headingAnchor" id="H910658124"><span class="h1">CROSSMATCH TESTING FOR DONOR-SPECIFIC ANTIBODIES</span><span class="headingEndMark"> — </span>The purpose of crossmatch testing is to identify any preformed donor-specific anti-human leukocyte antigen (HLA) antibody (DSA) present in a patient's serum that is directed against a particular donor. In their seminal paper published in 1969, Patel and Terasaki demonstrated that patients with preformed DSA that rendered a complement-dependent cytotoxic (CDC) crossmatch positive had higher rates of hyperacute rejection and primary graft nonfunction [<a href="#rid2">2</a>]. This led to the implementation of the CDC crossmatch against donor T cells as a universal prerequisite, with a positive result being an absolute contraindication to transplantation. Since then, other more sensitive crossmatch assays (flow crossmatch and "virtual" crossmatch) have been implemented for clinical use. While positive results from these tests may not preclude moving forward with transplantation, they do highlight the presence of increased immunologic risk for graft injury. The table (<a class="graphic graphic_table graphicRef120941" href="/z/d/graphic/120941.html" rel="external">table 1</a>) provides a comparison among the different crossmatch assays.</p><p class="headingAnchor" id="H3322989191"><span class="h2">Assays for crossmatch testing</span></p><p class="headingAnchor" id="H1870179459"><span class="h3">Complement-dependent cytotoxic (CDC) crossmatch</span><span class="headingEndMark"> — </span>Crossmatch testing is used to identify the presence of preformed antibody in a recipient's serum that is specifically directed against their potential donor (known as DSA). This typically refers to the presence of anti-HLA antibody since HLA molecules exhibit great degrees of polymorphism and represent the primary antigenic source against which the immune system reacts. Ideally, crossmatch testing using the target donor graft tissue would enable one to detect any antibodies that could potentially react against the allograft. This would include HLA antibodies as well as antibodies directed against non-HLA antigens expressed on the target cell surfaces, such as endothelial antigens, the angiotensin II type 1 (AT1) receptor, and MHC class I polypeptide-related sequence A (<em>MICA</em>), which have been associated with instances of antibody-mediated rejection (ABMR). However, crossmatch testing with allograft-derived cells is impractical at present. As such, current crossmatch assays utilize donor lymphocytes as surrogates: T cells express class I HLA molecules, and a positive T cell crossmatch identifies the presence of a class I DSA; B cells express both class I and class II HLA molecules, and a positive B cell crossmatch identifies class I and class II DSAs. Donor lymphocytes are first separated (usually by magnetic bead isolation) into a CD3+ T cell fraction and a CD19+ B cell fraction. Serum from the recipient is then added to the cells, followed by the addition of complement and a viability dye. If DSA is present, cell lysis is observed, and the crossmatch is deemed positive.</p><p>The CDC crossmatch depends upon the titer/amount of DSA present in sera, immunoglobulin isotype, and cell surface density of the target HLA antigen. Low titer, but potentially relevant, antibody may not cluster in sufficient density to be able to crosslink complement and activate the membrane attack complex to induce cell lysis. To enhance the assay's sensitivity, anti-human globulin (AHG), a complement-fixing antibody capable of binding human immunoglobulin, is added following the addition of the recipient's sera. By binding any DSA already bound to the donor's lymphocytes, it increases the density of antibody present, thereby increasing the likelihood of activating complement. Furthermore, since AHG binds not only to complement-binding DSA but also to non-complement-binding DSA, its use enables the detection of non-complement-binding DSA that would not result in reactivity in the unenhanced assay. Both assays can result in false-positive results due to the presence of non-HLA IgG antibody (directed against antigens present on lymphocytes) or IgM HLA or non-HLA antibody. (See <a class="local">'Unexpected positive crossmatch results'</a> below.)</p><p class="headingAnchor" id="H2903013264"><span class="h3">Flow cytometry crossmatch</span><span class="headingEndMark"> — </span>The flow cytometry crossmatch assay differs from the CDC crossmatch in its greater degree of sensitivity and ability to detect IgG DSA regardless of its capability to activate complement. Recipient serum is added to the donor lymphocytes, followed by the addition of a secondary fluorochrome-conjugated antibody that detects human IgG. In this assay, donor lymphocytes do not need to be separated into their T/B cell fractions. Rather, additional detection antibodies (conjugated to different fluorochromes) are added to distinguish between the two subsets. Samples are analyzed on a flow cytometer and results read out in a quantitative fashion as a unit of median fluorescence intensity (MFI). Samples with the patient's sera incubated with the donor cells (allocrossmatch) are compared with a negative control containing pooled sera from normal, healthy, unsensitized individuals. A shift in fluorescence intensity above a predetermined cutoff signifies the presence of an alloantibody (<a class="graphic graphic_figure graphicRef120938" href="/z/d/graphic/120938.html" rel="external">figure 7</a>). The strength and amount of antibody present are reflected in the magnitude of the shift.</p><p>It is important for the clinician to be aware that flow cytometry crossmatch protocols are not standardized in their testing or reporting. Individual HLA laboratories establish their own threshold cutoffs through validation testing of known negative and positive sera and correlation with single-antigen bead (SAB) testing. Reporting of flow cytometry crossmatch results also has great variability and can be expressed as channel shifts of median fluorescence (median channel shifts [MCS]) above the baseline or normalized against molecules of equivalent soluble fluorescence (MESF), ratios, or as direct fluorescence units (DFU). In addition, fluorescence intensity measurements can vary depending upon the cytometer and its settings, differences in the detection antibody used (eg, manufacturer, fluorochrome, and concentration), modifications in protocol, and variations in cell concentrations [<a href="#rid39">39</a>]. These inconsistencies render it difficult to standardize threshold cutoffs among HLA laboratories and make it challenging to interpret studies or clinical protocols that utilize these results quantitatively to determine the strength/amount of antibody. Clinicians should communicate with their own HLA laboratories for a better understanding of how to interpret crossmatch reporting at their center.</p><p>Common unexpected positive results encountered in flow cytometry crossmatch testing include high background signal (particularly with B cell flow cytometry crossmatch), the presence of antibody that reacts against lymphocyte-specific antigens, donor cell viability, and a strict threshold cutoff. Individuals who have only anti-HLA IgM antibody may have a negative flow cytometric crossmatch since the fluorochrome-labeled detection antibody is selective for IgG. (See <a class="local">'Unexpected positive crossmatch results'</a> below.)</p><p class="headingAnchor" id="H1953056897"><span class="h3">Virtual crossmatch</span><span class="headingEndMark"> — </span>The term "virtual" crossmatch refers to a process in which the clinician or laboratorian uses the results of two actual laboratory tests (the anti-HLA screening results and the HLA type of the donor) to deduce what the result of an actual crossmatch might be, should one be performed. If a candidate is found to have antibody against an HLA antigen for which the donor is mismatched with the candidate (DSA), and if the "strength" of the antibody is thought to be sufficient, there is some predictive value with a positive or negative actual crossmatch. The correlation of this DSA or "virtual crossmatch" naturally depends upon what kind of crossmatch is anticipated. (See <a class="local">'Solid phase assays'</a> above.)</p><p>In a study performed by the United Network of Organ Sharing/Organ Procurement and Transplantation Network (UNOS/OPTN), the virtual crossmatch had a &gt;85 percent positive predictive value on a subsequently performed flow crossmatch [<a href="#rid40">40</a>]. However, its negative predictive value was only approximately 50 percent, presumed to be low because of an incomplete profile of the patient's HLA antibodies (see <a class="local">'Unexpected positive crossmatch results'</a> below). At the time of this study, antibodies against HLA-C, -DQ, and -DP antigens were not typically considered to be unacceptable. The predictive value of the virtual crossmatch is also dependent upon the threshold cutoff each HLA laboratory uses to determine the presence of an antibody. (See <a class="local">'Defining a positive result'</a> above.)</p><p class="headingAnchor" id="H1438170999"><span class="h2">Unexpected positive crossmatch results</span><span class="headingEndMark"> — </span>A positive CDC or flow crossmatch result in the absence of DSA (a negative virtual crossmatch) may be caused by a number of factors (<a class="graphic graphic_table graphicRef120942" href="/z/d/graphic/120942.html" rel="external">table 2</a>):</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Presence of "true" DSA</strong> – Occasionally, positive crossmatch results may be due to "true" but unrecognized DSA. The serum used to screen for HLA antibodies may be different than the serum used for the crossmatch assay. In these situations, if a sensitizing event has occurred in the interim, a positive crossmatch but a negative SAB screen may reflect newly formed DSA. In addition, antibody levels can fluctuate over time, either naturally or in response to treatment.</p><p></p><p class="bulletIndent1">Absence of a comprehensive profile of a patient's HLA antibodies and/or insufficient/ambiguous donor HLA typing can also lead to an incorrect virtual crossmatch assessment and a discrepancy between these results and the physical crossmatch tests. As an example, if HLA-DP antibodies are present in a patient's sera but not routinely reported by an HLA laboratory, or if the donor is not typed at the DP locus to enable identification of whether the antibody is donor specific, then the virtual crossmatch would be falsely interpreted as negative.</p><p></p><p class="bulletIndent1">Failure to consider shared epitopes can also lead to under-recognition of DSA identified by SAB testing. When a number of beads contain a shared epitope, it is possible for all of those beads to register MFI values below the threshold cutoff. Knowledge of shared epitopes is therefore essential for proper interpretation of SAB assays. (See <a class="local">'Determining the amount/strength of antibody'</a> above.)</p><p></p><p class="bulletIndent1">Other scenarios that lead to under-recognition of DSA by SAB testing (ie, false-negative result) are discussed elsewhere in this topic. (See <a class="local">'False-negative results'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Presence of non-HLA antibody </strong>– The presence of antibody directed against self-/non-HLA antigens present on T/B cells can lead to a positive CDC or flow crossmatch result. Since reactivity of the antibody is directed against a non-HLA antigen that is expressed on lymphocytes but not on allograft tissue, these antibodies are unlikely to cause graft injury. Testing of the patient's sera against their own lymphocytes (autocrossmatch) or against surrogate donors would reveal a repeatedly positive crossmatch result in the absence of DSA.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Presence of IgM antibodies</strong> – The presence of IgM antibodies can cause false-positive CDC crossmatch results, if directed against non-HLA antigens, or true-positive CDC crossmatch results if directed against HLA. IgM can be autoantibody, most often detected in sera of patients with autoimmune disorders, and is not thought to be pathogenic [<a href="#rid41">41,42</a>]. IgM autoantibody can be detected by performing an autocrossmatch to assess reactivity of a patient's serum to their own lymphocytes. This can be confirmed by adding dithiothreitol (DTT) or dithioerythritol (DTE) to the assay, which breaks the disulfide bonds of the IgM pentamer and renders the CDC crossmatch assay negative. DTT/DTE can also be added to the allocrossmatch; a positive result that is rendered negative after DTT/DTE treatment is due to IgM antibody, whereas it would remain positive in the presence of IgG antibody.</p><p></p><p class="bulletIndent1">However, not all IgM antibodies are benign; those with anti-HLA specificity can be associated with hyperacute and accelerated rejection. If a patient is exposed to foreign HLA through a recent sensitizing event, the patient will first mount an IgM anti-HLA antibody response before class-switching to the IgG isotype begins to occur (typically within two to four weeks). The use of DTT/DTE does not distinguish between benign IgM autoantibody and potentially pathogenic IgM alloantibody.</p><p></p><p class="bulletIndent1">IgM antibodies are not detected with traditional flow cytometry crossmatch or SAB testing. These tests employ a detection antibody that specifically recognizes only IgG. Theoretically, these assays can be modified to use a secondary antibody capable of detecting IgM, but this is not routinely performed. It is thought that anti-HLA IgM can sometimes interfere with a flow crossmatch through steric hindrance of IgG and cause a false-negative result.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Therapeutic humanized monoclonal antibodies </strong>– An increasing number of patients awaiting transplantation are being treated with humanized monoclonal antibodies (mAb) that can interfere with crossmatch testing. The most commonly encountered scenario is a patient who is treated with <a class="drug drug_general" data-topicid="9616" href="/z/d/drug information/9616.html" rel="external">rituximab</a>, an anti-CD20 mAb. If present in the recipient serum, rituximab binds in vitro to the donor B cells (which express CD20), leading to a false-positive B cell crossmatch (both CDC and flow cytometry crossmatch). T cell crossmatch results are not impacted, since T cells do not express CD20.</p><p></p><p>Other examples of therapeutic antibodies that are known to interfere with these assays include <a class="drug drug_general" data-topicid="8725" href="/z/d/drug information/8725.html" rel="external">alemtuzumab</a> (binds to CD52 on T and B cells), antithymocyte globulin (ATG; polyclonal antibody that can bind to antigens on T and B cells), and <a class="drug drug_general" data-topicid="105443" href="/z/d/drug information/105443.html" rel="external">daratumumab</a> (binds to CD38 on B cells and, to a lesser extent, T cells). Their effect on the assay depends upon the pharmacokinetics and pharmacodynamics of the drug and is strongest with higher concentrations of the mAb in the serum. In the case of <a class="drug drug_general" data-topicid="9616" href="/z/d/drug information/9616.html" rel="external">rituximab</a>, this interference can last up to a year postinfusion. The addition of pronase to the assay can help to eliminate the contribution of the mAb.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Use of pronase </strong>– B cell flow crossmatch results are sometimes difficult to interpret because of a high background in fluorescence due to binding of the fluorochrome-conjugated detection antibody to cell surface IgG (ie, the B cell receptor) and Fc receptors present on B cells. This high background fluorescence reduces the sensitivity of the assay to detect additional fluorescence that is due specifically to DSA. To reduce this background fluorescence and improve the specificity of the test, some labs use pronase to treat the cells [<a href="#rid43">43,44</a>]. Pronase is a cocktail of nonspecific proteases isolated from <em>Streptomyces griseus</em>, comprised of neutral proteases, chymotrypsin, trypsin, carboxypeptidase, aminopeptidase, and phosphatases. Commercially available pronase formulations can differ in their composition and levels of activity and can vary between companies and between lots. Each lot is tested for its rate of Folin-positive amino acids and peptides from casein, which is reported as a unit of reactivity to allow for some standardization.</p><p></p><p class="bulletIndent1">However, use of pronase can affect HLA expression and therefore reduce sensitivity to detect the presence of DSA [<a href="#rid44">44</a>]. It can also cause false-positive crossmatch results by unveiling cryptic (hidden) epitopes [<a href="#rid45">45,46</a>]. Incubation time, concentration, lot-to-lot variability, and products from different manufacturers are factors that need to be considered when HLA laboratories validate this modified assay. Thresholds for determining a positive crossmatch result may be modified when pronase is used.</p><p></p><p class="bulletIndent1">As discussed above, pronase treatment is also frequently used to avoid false-positive crossmatch results due to the presence of a therapeutic monoclonal antibody in the patient's sera. In these instances, incubation periods are much longer and/or the concentration of pronase is increased compared with that used in protocols for reducing background alone. However, the same considerations apply with respect to the unintended consequence of reducing sensitivity due to effects on HLA expression levels. Because of this, some clinicians prefer to determine immunologic risk of class II DSA using SAB testing results alone, without B cell crossmatch results.</p><p></p><p class="headingAnchor" id="H2404402290"><span class="h1">USE OF HLA TESTING TO ASSESS IMMUNOLOGIC RISK</span><span class="headingEndMark"> — </span>The results of all human leukocyte antigen (HLA) testing should be integrated to provide an immunologic risk assessment between a donor and recipient pair (<a class="graphic graphic_figure graphicRef120939" href="/z/d/graphic/120939.html" rel="external">figure 8</a>). In addition, the decision to decline or proceed with transplantation should also take into account clinical characteristics such as organ type, urgency, likelihood of receiving a compatible graft, availability, suitability of immunosuppression strategies for the recipient, and donor factors.</p><p class="headingAnchor" id="H2295090575"><span class="h2">Common clinical scenarios</span><span class="headingEndMark"> — </span>Common clinical scenarios of HLA testing results and their interpretation are presented below and summarized in the table (<a class="graphic graphic_table graphicRef120942" href="/z/d/graphic/120942.html" rel="external">table 2</a>). Potential differences in laboratory protocols and reporting should be considered when evaluating the correlation between the assay and graft outcome and may explain some of the disparate findings between studies. Unexpected discrepancies between the assays should prompt a discussion with the HLA laboratory. (See <a class="local">'False-positive results'</a> above and <a class="local">'False-negative results'</a> above and <a class="local">'Unexpected positive crossmatch results'</a> above.)</p><p class="bulletIndent1"><span class="glyph">●</span>Positive CDC crossmatch – A positive complement-dependent cytotoxic (CDC) crossmatch due to donor-specific anti-HLA antibody (DSA) is indicative of high antibody burden and is considered a <strong>contraindication</strong> to transplantation because of its association with hyperacute/accelerated rejection [<a href="#rid2">2</a>]. (See  <a class="medical medical_review" href="/z/d/html/7301.html" rel="external">"Kidney transplantation in adults: Evaluation and diagnosis of acute kidney allograft dysfunction", section on 'Hyperacute rejection'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Negative CDC crossmatch <strong>and </strong>positive flow cytometry crossmatch – A positive flow crossmatch with a negative CDC crossmatch represents an intermediate risk for antibody-mediated rejection (ABMR) but is not necessarily a contraindication to transplantation. Several studies have found that a positive flow crossmatch due to DSA is associated with higher rates of acute rejection and both early and late graft loss [<a href="#rid47">47-51</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Negative CDC and flow cytometry crossmatch <strong>and </strong>DSA detected by single-antigen bead (SAB) assay (positive "virtual crossmatch") – There continue to be conflicting views on the clinical impact of antibodies detected only by solid phase assays, with numerous studies demonstrating that these low-level antibodies have no clinical impact [<a href="#rid4">4-9</a>]. This may reflect the overly sensitive threshold used to determine whether an antibody is present. However, the presence of DSA does indicate a prior exposure to the donor-specific HLA antigen, and therefore the patient is at risk for a latent memory response. (See <a class="local">'Defining a positive result'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Positive crossmatch with no DSA – In the absence of DSA defined by SAB testing, a positive crossmatch does not appear to correlate with graft outcomes [<a href="#rid41">41,42</a>]. These likely represent scenarios in which the crossmatch is positive due to clinically irrelevant, non-HLA antibody (see <a class="local">'Unexpected positive crossmatch results'</a> above). The caveat to this occurs in cases where a "true" anti-HLA antibody is present in the recipient sera but goes undetected in the SAB assay (false-negative result). (See <a class="local">'False-negative results'</a> above.)</p><p></p><p class="headingAnchor" id="H308444656"><span class="h2">Other considerations</span></p><p class="headingAnchor" id="H1978950244"><span class="h3">Historical DSA</span><span class="headingEndMark"> — </span>The absence of pretransplant DSAs does not imply a lack of prior sensitization, as testing reflects only the antibody levels present in the serum sample being tested. A careful examination of longitudinal SAB screening and prior crossmatching results may reveal the presence of a historical DSA and immunologic memory against the donor antigen. While the presence of immunologic memory confers a higher degree of risk, it remains difficult to predict whether this will alter graft outcome. There are no existing, clinically validated assays to challenge a latent memory response or determine whether low-titer DSA will remain low or rapidly increase posttransplantation following repeat exposure to the antigen. However, awareness of this potential memory recall response may affect the clinician's decision to intensify the immunosuppressive regimen and/or posttransplantation monitoring strategy.</p><p class="headingAnchor" id="H3731246385"><span class="h3">Clinical significance of MFI values of pretransplant DSA</span><span class="headingEndMark"> — </span>Median fluorescence intensity (MFI) values of pretransplant donor-specific anti-HLA antibody (DSA) do not appear to consistently predict graft outcomes. In one study, higher rates of ABMR and graft loss were seen when pretransplant DSA had MFI values &gt;10,000 but were comparable between all other groups (moderate MFI [5000 to 10,000] versus low MFI [1000 to 5000]) [<a href="#rid52">52</a>]. Another study found that differences in graft survival were dependent upon identifying the presence of DSA (MFI &gt;1500) but not upon the actual level of MFI. Failure to discern differences between these groups of patients may reflect technical limitations of the assay (with MFI values themselves being a poor predictor of amount/strength of antibody (see <a class="local">'Determination of patient sensitization'</a> above)) as well as our inability to predict whether low-level DSA will remain low or rapidly increase posttransplantation upon repeat exposure to the antigen.</p><p class="headingAnchor" id="H4139961955"><span class="h3">Absence of pretransplant DSA on outcome</span><span class="headingEndMark"> — </span>Even in the absence of donor-specific anti-HLA antibody (DSA), some studies have shown that sensitized recipients (with non-DSA alloantibody) are at higher risk for graft failure [<a href="#rid53">53,54</a>]. However, this remains controversial, as other studies have shown no impact of having non-DSA on graft outcome [<a href="#rid55">55-57</a>]. These disparate findings may depend upon whether more or less aggressive immunosuppression is used [<a href="#rid58">58</a>] or could also reflect incomplete identification of DSA depending upon whether antibodies against all loci (including class II alpha chain) are accounted for.</p><p class="headingAnchor" id="H1234811680"><span class="h1">SOCIETY GUIDELINE LINKS</span><span class="headingEndMark"> — </span>Links to society and government-sponsored guidelines from selected countries and regions around the world are provided separately. (See  <a class="medical medical_society_guidelines" href="/z/d/html/110754.html" rel="external">"Society guideline links: Kidney transplantation"</a>.)</p><p class="headingAnchor" id="H3607911214"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Overview </strong>– The goal of histocompatibility testing is to determine the immunologic risk of a transplant recipient in the context of a potential donor. If the transplant is between genetically different individuals, the allogeneic graft (allograft) is recognized as foreign predominantly because of differences between donor/recipient major histocompatibility complex (MHC) molecules, which are also known as the human leukocyte antigens (HLAs). The ensuing immune response occurs through two major mechanisms: T cell-mediated (cellular) responses and antibody-mediated (humoral) responses. Existing histocompatibility testing focuses primarily on predicting antibody-mediated alloimmune responses. (See <a class="local">'Introduction'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Overview of HLA </strong>– In humans, the MHC genes are encoded on the short arm of chromosome 6 and include the class I genes, <em>HLA-A</em>, <em>HLA-B</em>, and <em>HLA-C</em>, and the class II genes, <em>HLA-DPA1</em>, <em>HLA-DPB1</em>, <em>HLA-DQA1</em>, <em>HLA-DQB1</em>, <em>HLA-DRA</em>, <em>HLA</em>-<em>DRB1</em>, <em>HLA</em>-<em>DRB3</em>, <em>HLA</em>-<em>DRB4</em>, and <em>HLA</em>-<em>DRB5</em>. Class I molecules are expressed on all nucleated cells, whereas class II molecules are expressed primarily on antigen-presenting cells (such as B cells, dendritic cells, and macrophages) but can be expressed under inflammatory conditions on a variety of cell types including endothelial cells and epithelial cells. (See <a class="local">'Overview of HLA'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>HLA mismatches </strong>– In solid organ transplantation, a "mismatch" refers to an HLA antigen found on the cells of the donor allograft but not in the recipient. The greater the disparity between the donor and recipient, the more "foreign" the allograft appears and the higher the likelihood of developing an alloimmune response. For the purpose of deceased-donor kidney transplant allocation, only the <em>HLA-A</em>, <em>HLA-B</em>, and <em>HLA-DR</em> loci are examined for their mismatch profile. (See <a class="local">'HLA mismatches'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>HLA typing </strong>– Accurate typing of a donor's and recipient's HLAs is essential in determining the degree of mismatch between them and in avoiding the transplantation of organs from donors expressing HLA antigens against which the recipient has preformed antibody. In some cases it is also desirable to avoid exposing a candidate to an organ that bears mismatched HLA antigens against which the candidate has previously made antibody. HLA typing was initially performed using serologic-based assays, but this has since been replaced by the use of DNA-based molecular techniques, which allow for higher resolution and more accurate typing of all loci. (See <a class="local">'HLA typing'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Screening for anti-HLA antibodies </strong>– Nearly 30 percent of waitlisted patients are known to have antibodies directed against one or more HLAs, as a result of sensitization from prior exposures such as pregnancies, blood transfusions, previous transplantation, and implants such as ventricular assist devices and homografts. The purpose of screening patients for anti-HLA antibodies prior to transplantation is to identify the presence of anti-HLA antibodies, define their specificity to particular HLA antigen(s), and determine their relative amount/strength. This information then aids the clinician in assessing the patient's likelihood of receiving an HLA-compatible transplant, identifying "unacceptable antigens" to avoid transplanting grafts from donors expressing these antigens, and classifying those at higher immunologic risk who may benefit from a more aggressive immunosuppressive regimen and/or more intensive posttransplantation monitoring. (See <a class="local">'Screening for anti-HLA antibodies'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Determination of patient sensitization </strong>– The <strong>calculated panel of reactive antibody (cPRA)</strong> is a tool that clinicians can use to assess a patient's degree of sensitization to HLA antigens and thereby their likelihood of transplantation. The cPRA calculates the likelihood of transplantation by using results of the single-antigen bead (SAB) assay to identify the specificity of the anti-HLA antibodies, in combination with the known frequencies of HLA antigens within the donor population. (See <a class="local">'Determination of patient sensitization'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Crossmatch testing </strong>– The purpose of crossmatch testing is to identify any preformed donor-specific anti-HLA antibody (DSA) present in a patient's serum that is directed against a particular donor. Cell-based assays such as the complement-dependent cytotoxic (CDC) crossmatch and flow crossmatches are performed prior to transplantation to determine whether it is suitable to move forward with transplantation. (See <a class="local">'Crossmatch testing for donor-specific antibodies'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Assessing immunologic risk </strong>– The results of all HLA testing should be integrated to provide an immunologic risk assessment between a donor and recipient pair. In addition, the decision to decline or proceed with transplantation should also take into account clinical characteristics such as organ type, urgency, likelihood of receiving a compatible graft, availability, suitability of immunosuppression strategies for the recipient, and donor factors. (See <a class="local">'Use of HLA testing to assess immunologic risk'</a> above.)</p><p></p></div><div class="headingAnchor" id="references"><ol id="reference"><li class="breakAll">https://optn.transplant.hrsa.gov/data/request-data/ (Accessed on June 06, 2018).</li><li><a class="nounderline abstract_t">Patel R, Terasaki PI. Significance of the positive crossmatch test in kidney transplantation. N Engl J Med 1969; 280:735.</a></li><li><a class="nounderline abstract_t">Gebel HM, Bray RA, Nickerson P. Pre-transplant assessment of donor-reactive, HLA-specific antibodies in renal transplantation: contraindication vs. risk. Am J Transplant 2003; 3:1488.</a></li><li><a class="nounderline abstract_t">Patel AM, Pancoska C, Mulgaonkar S, Weng FL. Renal transplantation in patients with pre-transplant donor-specific antibodies and negative flow cytometry crossmatches. Am J Transplant 2007; 7:2371.</a></li><li><a class="nounderline abstract_t">van den Berg-Loonen EM, Billen EV, Voorter CE, et al. Clinical relevance of pretransplant donor-directed antibodies detected by single antigen beads in highly sensitized renal transplant patients. Transplantation 2008; 85:1086.</a></li><li><a class="nounderline abstract_t">Ho EK, Vasilescu ER, Colovai AI, et al. Sensitivity, specificity and clinical relevance of different cross-matching assays in deceased-donor renal transplantation. Transpl Immunol 2008; 20:61.</a></li><li><a class="nounderline abstract_t">Aubert V, Venetz JP, Pantaleo G, Pascual M. Low levels of human leukocyte antigen donor-specific antibodies detected by solid phase assay before transplantation are frequently clinically irrelevant. Hum Immunol 2009; 70:580.</a></li><li><a class="nounderline abstract_t">Adebiyi OO, Gralla J, Klem P, et al. Clinical Significance of Pretransplant Donor-Specific Antibodies in the Setting of Negative Cell-Based Flow Cytometry Crossmatching in Kidney Transplant Recipients. Am J Transplant 2016; 16:3458.</a></li><li><a class="nounderline abstract_t">Süsal C, Ovens J, Mahmoud K, et al. No association of kidney graft loss with human leukocyte antigen antibodies detected exclusively by sensitive Luminex single-antigen testing: a Collaborative Transplant Study report. Transplantation 2011; 91:883.</a></li><li class="breakAll">https://optn.transplant.hrsa.gov/media/1200/optn_policies.pdf#nameddest=Policy_07 (Accessed on June 06, 2018).</li><li><a class="nounderline abstract_t">Reed EF, Rao P, Zhang Z, et al. Comprehensive assessment and standardization of solid phase multiplex-bead arrays for the detection of antibodies to HLA. Am J Transplant 2013; 13:1859.</a></li><li><a class="nounderline abstract_t">Apps R, Qi Y, Carlson JM, et al. Influence of HLA-C expression level on HIV control. Science 2013; 340:87.</a></li><li><a class="nounderline abstract_t">Kaur G, Gras S, Mobbs JI, et al. Structural and regulatory diversity shape HLA-C protein expression levels. Nat Commun 2017; 8:15924.</a></li><li><a class="nounderline abstract_t">Simmons DP, Kafetzi ML, Wood I, et al. Antibodies against HLA-DP recognize broadly expressed epitopes. Hum Immunol 2016; 77:1128.</a></li><li><a class="nounderline abstract_t">Hönger G, Krähenbühl N, Dimeloe S, et al. Inter-individual differences in HLA expression can impact the CDC crossmatch. Tissue Antigens 2015; 85:260.</a></li><li><a class="nounderline abstract_t">Bachelet T, Martinez C, Del Bello A, et al. Deleterious Impact of Donor-Specific Anti-HLA Antibodies Toward HLA-Cw and HLA-DP in Kidney Transplantation. Transplantation 2016; 100:159.</a></li><li><a class="nounderline abstract_t">Tambur AR, Campbell P, Claas FH, et al. Sensitization in Transplantation: Assessment of Risk (STAR) 2017 Working Group Meeting Report. Am J Transplant 2018; 18:1604.</a></li><li><a class="nounderline abstract_t">Tait BD, Süsal C, Gebel HM, et al. Consensus guidelines on the testing and clinical management issues associated with HLA and non-HLA antibodies in transplantation. Transplantation 2013; 95:19.</a></li><li><a class="nounderline abstract_t">Tambur AR, Herrera ND, Haarberg KM, et al. Assessing Antibody Strength: Comparison of MFI, C1q, and Titer Information. Am J Transplant 2015; 15:2421.</a></li><li><a class="nounderline abstract_t">Tambur AR, Wiebe C. HLA Diagnostics: Evaluating DSA Strength by Titration. Transplantation 2018; 102:S23.</a></li><li><a class="nounderline abstract_t">Tambur AR, Glotz D, Herrera ND, et al. Can solid phase assays be better utilized to measure efficacy of antibody removal therapies? Hum Immunol 2016; 77:624.</a></li><li><a class="nounderline abstract_t">Konvalinka A, Tinckam K. Utility of HLA Antibody Testing in Kidney Transplantation. J Am Soc Nephrol 2015; 26:1489.</a></li><li><a class="nounderline abstract_t">Cai J, Terasaki PI, Anderson N, et al. Intact HLA not beta2m-free heavy chain-specific HLA class I antibodies are predictive of graft failure. Transplantation 2009; 88:226.</a></li><li><a class="nounderline abstract_t">Pereira S, Perkins S, Lee JH, et al. Donor-specific antibody against denatured HLA-A1: clinically nonsignificant? Hum Immunol 2011; 72:492.</a></li><li><a class="nounderline abstract_t">Gombos P, Opelz G, Scherer S, et al. Influence of test technique on sensitization status of patients on the kidney transplant waiting list. Am J Transplant 2013; 13:2075.</a></li><li class="breakAll">One Lambda product insert. http://www.onelambda.com/content/dam/onelambda/en/TDX/Documents/securedocs/docs/PS-ENGLISH_LANGUAGE/LS-LSCN-PI-EN-00.pdf (Accessed on October 19, 2020).</li><li><a class="nounderline abstract_t">Schwaiger E, Wahrmann M, Bond G, et al. Complement component C3 activation: the leading cause of the prozone phenomenon affecting HLA antibody detection on single-antigen beads. Transplantation 2014; 97:1279.</a></li><li><a class="nounderline abstract_t">Visentin J, Vigata M, Daburon S, et al. Deciphering complement interference in anti-human leukocyte antigen antibody detection with flow beads assays. Transplantation 2014; 98:625.</a></li><li><a class="nounderline abstract_t">Kosmoliaptsis V, Bradley JA, Peacock S, et al. Detection of immunoglobulin G human leukocyte antigen-specific alloantibodies in renal transplant patients using single-antigen-beads is compromised by the presence of immunoglobulin M human leukocyte antigen-specific alloantibodies. Transplantation 2009; 87:813.</a></li><li><a class="nounderline abstract_t">Schnaidt M, Weinstock C, Jurisic M, et al. HLA antibody specification using single-antigen beads--a technical solution for the prozone effect. Transplantation 2011; 92:510.</a></li><li><a class="nounderline abstract_t">Carey BS, Boswijk K, Mabrok M, et al. A reliable method for avoiding false negative results with Luminex single antigen beads; evidence of the prozone effect. Transpl Immunol 2016; 37:23.</a></li><li><a class="nounderline abstract_t">Loupy A, Lefaucheur C, Vernerey D, et al. Complement-binding anti-HLA antibodies and kidney-allograft survival. N Engl J Med 2013; 369:1215.</a></li><li><a class="nounderline abstract_t">Crespo M, Torio A, Mas V, et al. Clinical relevance of pretransplant anti-HLA donor-specific antibodies: does C1q-fixation matter? Transpl Immunol 2013; 29:28.</a></li><li><a class="nounderline abstract_t">Schaub S, Hönger G, Koller MT, et al. Determinants of C1q binding in the single antigen bead assay. Transplantation 2014; 98:387.</a></li><li><a class="nounderline abstract_t">Taylor CJ, Kosmoliaptsis V, Martin J, et al. Technical Limitations of the C1q Single-Antigen Bead Assay to Detect Complement Binding HLA-Specific Antibodies. Transplantation 2017; 101:1206.</a></li><li><a class="nounderline abstract_t">Freitas MC, Rebellato LM, Ozawa M, et al. The role of immunoglobulin-G subclasses and C1q in de novo HLA-DQ donor-specific antibody kidney transplantation outcomes. Transplantation 2013; 95:1113.</a></li><li><a class="nounderline abstract_t">Gebel HM, Kasiske BL, Gustafson SK, et al. Allocating Deceased Donor Kidneys to Candidates with High Panel-Reactive Antibodies. Clin J Am Soc Nephrol 2016; 11:505.</a></li><li class="breakAll">Organ Procurement and Transplantation Network Histocompatibility Committee. Change Calculated Panel Reactive Antibody
(CPRA) Calculation. Available at: https://optn.transplant.hrsa.gov/media/x1bp10dl/histo_change-cpra-calculation_presentation_winter-2022-pc.pdf.</li><li><a class="nounderline abstract_t">Gratama JW, D'hautcourt JL, Mandy F, et al. Flow cytometric quantitation of immunofluorescence intensity: problems and perspectives. European Working Group on Clinical Cell Analysis. Cytometry 1998; 33:166.</a></li><li><a class="nounderline abstract_t">Nikaein A, Cherikh W, Nelson K, et al. Organ procurement and transplantation network/united network for organ sharing histocompatibility committee collaborative study to evaluate prediction of crossmatch results in highly sensitized patients. Transplantation 2009; 87:557.</a></li><li><a class="nounderline abstract_t">Kerman RH, Susskind B, Buyse I, et al. Flow cytometry-detected IgG is not a contraindication to renal transplantation: IgM may be beneficial to outcome. Transplantation 1999; 68:1855.</a></li><li><a class="nounderline abstract_t">Przybylowski P, Balogna M, Radovancevic B, et al. The role of flow cytometry-detected IgG and IgM anti-donor antibodies in cardiac allograft recipients. Transplantation 1999; 67:258.</a></li><li><a class="nounderline abstract_t">Lobo PI, Spencer CE, Stevenson WC, et al. The use of pronase-digested human leukocytes to improve specificity of the flow cytometric crossmatch. Transpl Int 1995; 8:472.</a></li><li><a class="nounderline abstract_t">Apithy MJ, Desoutter J, Gicquel A, et al. Pronase treatment improves flow cytometry crossmatching results. HLA 2017; 90:157.</a></li><li><a class="nounderline abstract_t">Park H, Lim YM, Han BY, et al. Frequent false-positive reactions in pronase-treated T-cell flow cytometric cross-match tests. Transplant Proc 2012; 44:87.</a></li><li><a class="nounderline abstract_t">Hetrick SJ, Schillinger KP, Zachary AA, Jackson AM. Impact of pronase on flow cytometric crossmatch outcome. Hum Immunol 2011; 72:330.</a></li><li><a class="nounderline abstract_t">Karpinski M, Rush D, Jeffery J, et al. Flow cytometric crossmatching in primary renal transplant recipients with a negative anti-human globulin enhanced cytotoxicity crossmatch. J Am Soc Nephrol 2001; 12:2807.</a></li><li><a class="nounderline abstract_t">Graff RJ, Buchanan PM, Dzebisashvili N, et al. The clinical importance of flow cytometry crossmatch in the context of CDC crossmatch results. Transplant Proc 2010; 42:3471.</a></li><li><a class="nounderline abstract_t">Lentine KL, Graff RJ, Xiao H, et al. Flow cytometry crossmatch before kidney transplantation in contemporary practice: target cell utilization, results patterns, and associated long-term graft survival. Clin Transpl 2008; :253.</a></li><li><a class="nounderline abstract_t">Mahoney RJ, Ault KA, Given SR, et al. The flow cytometric crossmatch and early renal transplant loss. Transplantation 1990; 49:527.</a></li><li><a class="nounderline abstract_t">Ogura K, Terasaki PI, Johnson C, et al. The significance of a positive flow cytometry crossmatch test in primary kidney transplantation. Transplantation 1993; 56:294.</a></li><li><a class="nounderline abstract_t">Gloor JM, Winters JL, Cornell LD, et al. Baseline donor-specific antibody levels and outcomes in positive crossmatch kidney transplantation. Am J Transplant 2010; 10:582.</a></li><li><a class="nounderline abstract_t">Süsal C, Döhler B, Opelz G. Presensitized kidney graft recipients with HLA class I and II antibodies are at increased risk for graft failure: a Collaborative Transplant Study report. Hum Immunol 2009; 70:569.</a></li><li><a class="nounderline abstract_t">Richter R, Süsal C, Köhler S, et al. Pretransplant human leukocyte antigen antibodies detected by single-antigen bead assay are a risk factor for long-term kidney graft loss even in the absence of donor-specific antibodies. Transpl Int 2016; 29:988.</a></li><li><a class="nounderline abstract_t">Caro-Oleas JL, González-Escribano MF, González-Roncero FM, et al. Clinical relevance of HLA donor-specific antibodies detected by single antigen assay in kidney transplantation. Nephrol Dial Transplant 2012; 27:1231.</a></li><li><a class="nounderline abstract_t">Lefaucheur C, Loupy A, Hill GS, et al. Preexisting donor-specific HLA antibodies predict outcome in kidney transplantation. J Am Soc Nephrol 2010; 21:1398.</a></li><li><a class="nounderline abstract_t">Otten HG, Verhaar MC, Borst HP, et al. Pretransplant donor-specific HLA class-I and -II antibodies are associated with an increased risk for kidney graft failure. Am J Transplant 2012; 12:1618.</a></li><li><a class="nounderline abstract_t">Malheiro J, Tafulo S, Dias L, et al. Impact on mid-term kidney graft outcomes of pretransplant anti-HLA antibodies detected by solid-phase assays: Do donor-specific antibodies tell the whole story? Hum Immunol 2017; 78:526.</a></li></ol></div><div id="topicVersionRevision">Topic 116878 Version 11.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://optn.transplant.hrsa.gov/data/request-data/" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : https://optn.transplant.hrsa.gov/data/request-data/ (Accessed on June 06, 2018).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/4886455" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Significance of the positive crossmatch test in kidney transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14629279" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Pre-transplant assessment of donor-reactive, HLA-specific antibodies in renal transplantation: contraindication vs. risk.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17845571" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Renal transplantation in patients with pre-transplant donor-specific antibodies and negative flow cytometry crossmatches.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18431226" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Clinical relevance of pretransplant donor-directed antibodies detected by single antigen beads in highly sensitized renal transplant patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18929659" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Sensitivity, specificity and clinical relevance of different cross-matching assays in deceased-donor renal transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19375474" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Low levels of human leukocyte antigen donor-specific antibodies detected by solid phase assay before transplantation are frequently clinically irrelevant.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27140940" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Clinical Significance of Pretransplant Donor-Specific Antibodies in the Setting of Negative Cell-Based Flow Cytometry Crossmatching in Kidney Transplant Recipients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21325993" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : No association of kidney graft loss with human leukocyte antigen antibodies detected exclusively by sensitive Luminex single-antigen testing: a Collaborative Transplant Study report.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21325993" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : No association of kidney graft loss with human leukocyte antigen antibodies detected exclusively by sensitive Luminex single-antigen testing: a Collaborative Transplant Study report.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23763485" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Comprehensive assessment and standardization of solid phase multiplex-bead arrays for the detection of antibodies to HLA.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23559252" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Influence of HLA-C expression level on HIV control.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28649982" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Structural and regulatory diversity shape HLA-C protein expression levels.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27664843" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Antibodies against HLA-DP recognize broadly expressed epitopes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25786570" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Inter-individual differences in HLA expression can impact the CDC crossmatch.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26262501" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Deleterious Impact of Donor-Specific Anti-HLA Antibodies Toward HLA-Cw and HLA-DP in Kidney Transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29603613" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Sensitization in Transplantation: Assessment of Risk (STAR) 2017 Working Group Meeting Report.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23238534" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Consensus guidelines on the testing and clinical management issues associated with HLA and non-HLA antibodies in transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25930984" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Assessing Antibody Strength: Comparison of MFI, C1q, and Titer Information.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29266059" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : HLA Diagnostics: Evaluating DSA Strength by Titration.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27267046" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Can solid phase assays be better utilized to measure efficacy of antibody removal therapies?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25804279" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Utility of HLA Antibody Testing in Kidney Transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19623018" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Intact HLA not beta2m-free heavy chain-specific HLA class I antibodies are predictive of graft failure.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21396421" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Donor-specific antibody against denatured HLA-A1: clinically nonsignificant?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23841891" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Influence of test technique on sensitization status of patients on the kidney transplant waiting list.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23841891" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Influence of test technique on sensitization status of patients on the kidney transplant waiting list.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24621535" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Complement component C3 activation: the leading cause of the prozone phenomenon affecting HLA antibody detection on single-antigen beads.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25058376" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Deciphering complement interference in anti-human leukocyte antigen antibody detection with flow beads assays.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19300182" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Detection of immunoglobulin G human leukocyte antigen-specific alloantibodies in renal transplant patients using single-antigen-beads is compromised by the presence of immunoglobulin M human leukocyte antigen-specific alloantibodies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21869744" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : HLA antibody specification using single-antigen beads--a technical solution for the prozone effect.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27109036" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : A reliable method for avoiding false negative results with Luminex single antigen beads; evidence of the prozone effect.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24066742" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Complement-binding anti-HLA antibodies and kidney-allograft survival.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23907088" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Clinical relevance of pretransplant anti-HLA donor-specific antibodies: does C1q-fixation matter?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24918611" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Determinants of C1q binding in the single antigen bead assay.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27306532" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Technical Limitations of the C1q Single-Antigen Bead Assay to Detect Complement Binding HLA-Specific Antibodies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23514959" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : The role of immunoglobulin-G subclasses and C1q in de novo HLA-DQ donor-specific antibody kidney transplantation outcomes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26839235" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Allocating Deceased Donor Kidneys to Candidates with High Panel-Reactive Antibodies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26839235" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Allocating Deceased Donor Kidneys to Candidates with High Panel-Reactive Antibodies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9773877" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : Flow cytometric quantitation of immunofluorescence intensity: problems and perspectives. European Working Group on Clinical Cell Analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19307794" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Organ procurement and transplantation network/united network for organ sharing histocompatibility committee collaborative study to evaluate prediction of crossmatch results in highly sensitized patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10628764" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Flow cytometry-detected IgG is not a contraindication to renal transplantation: IgM may be beneficial to outcome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10075590" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : The role of flow cytometry-detected IgG and IgM anti-donor antibodies in cardiac allograft recipients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8579739" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : The use of pronase-digested human leukocytes to improve specificity of the flow cytometric crossmatch.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28660746" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : Pronase treatment improves flow cytometry crossmatching results.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22310587" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : Frequent false-positive reactions in pronase-treated T-cell flow cytometric cross-match tests.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21262310" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : Impact of pronase on flow cytometric crossmatch outcome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11729251" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Flow cytometric crossmatching in primary renal transplant recipients with a negative anti-human globulin enhanced cytotoxicity crossmatch.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21094799" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : The clinical importance of flow cytometry crossmatch in the context of CDC crossmatch results.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19708460" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : Flow cytometry crossmatch before kidney transplantation in contemporary practice: target cell utilization, results patterns, and associated long-term graft survival.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2138366" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : The flow cytometric crossmatch and early renal transplant loss.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8356583" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : The significance of a positive flow cytometry crossmatch test in primary kidney transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20121740" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : Baseline donor-specific antibody levels and outcomes in positive crossmatch kidney transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19375472" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : Presensitized kidney graft recipients with HLA class I and II antibodies are at increased risk for graft failure: a Collaborative Transplant Study report.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27103066" id="rid53" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>54 : Pretransplant human leukocyte antigen antibodies detected by single-antigen bead assay are a risk factor for long-term kidney graft loss even in the absence of donor-specific antibodies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21810767" id="rid54" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>55 : Clinical relevance of HLA donor-specific antibodies detected by single antigen assay in kidney transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20634297" id="rid55" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>56 : Preexisting donor-specific HLA antibodies predict outcome in kidney transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22404993" id="rid56" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>57 : Pretransplant donor-specific HLA class-I and -II antibodies are associated with an increased risk for kidney graft failure.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28732720" id="rid57" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>58 : Impact on mid-term kidney graft outcomes of pretransplant anti-HLA antibodies detected by solid-phase assays: Do donor-specific antibodies tell the whole story?</p>
</a>
</div>
</div>
</div>
</div>
</section>
</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->
<!-- End Footer Area -->
<!-- Back to top -->
<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->
<!-- Prpper JS -->
<!-- Bootstrap Min JS -->
<!-- Classy Nav Min Js -->
<!-- Owl Carousel Min Js -->
<!-- Magnific Popup JS -->
<!-- CounterUp JS -->
<!-- Waypoints JS -->
<!-- Form Validator Min JS -->
<!-- Contact Form Min JS -->
<!-- Main JS -->
<!-- Global site tag (gtag.js) - Google Analytics -->
</body>
</html>
