<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Genetics of hemophilia A and B</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Genetics of hemophilia A and B</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Genetics of hemophilia A and B</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Authors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">W Keith Hoots, MD</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Meadow Heiman, MS, LCGC, CCRC</a></dd><dt><span> </span>Section Editors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Benjamin A Raby, MD, MPH</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Amy D Shapiro, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jennifer S Tirnauer, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> May 18, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Hemophilia A and B refer to factor VIII and factor IX deficiency, respectively. They are caused by pathogenic variants (eg, mutations) in the <em>F8 </em>or <em>F9</em> gene. These are X-linked bleeding disorders that predominantly affect males. The genetics of hemophilia has implications for disease severity, inhibitor development, and preconception testing and counseling.</p><p>This topic reviews the genetics of hemophilia A and B.</p><p>Diagnosis and management of hemophilia and other bleeding disorders are discussed separately.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hemophilia A and B</strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Diagnosis</strong> – (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia"</a>.)</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Treatment of bleeding and perioperative care (including inhibitor patients)</strong> – (See  <a class="medical medical_review" href="/d/html/1314.html" rel="external">"Treatment of bleeding and perioperative management in hemophilia A and B"</a>.)</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Routine care, factor prophylaxis, and investigational approaches including gene therapy</strong> – (See  <a class="medical medical_review" href="/d/html/107911.html" rel="external">"Hemophilia A and B: Routine management including prophylaxis"</a>.)</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Treatment of age-related conditions</strong> – (See  <a class="medical medical_review" href="/d/html/110258.html" rel="external">"Chronic complications and age-related comorbidities in people with hemophilia"</a>.)</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Inhibitors and immune tolerance induction</strong> – (See  <a class="medical medical_review" href="/d/html/1308.html" rel="external">"Inhibitors in hemophilia: Mechanisms, prevalence, diagnosis, and eradication"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Other bleeding disorders</strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Overview</strong> – (See  <a class="medical medical_review" href="/d/html/1367.html" rel="external">"Approach to the adult with a suspected bleeding disorder"</a>.)</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Factor XI deficiency (hemophilia C)</strong> – (See  <a class="medical medical_review" href="/d/html/1333.html" rel="external">"Factor XI (eleven) deficiency"</a>.)</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Von Willebrand disease</strong> – (See  <a class="medical medical_review" href="/d/html/1307.html" rel="external">"Clinical presentation and diagnosis of von Willebrand disease"</a> and  <a class="medical medical_review" href="/d/html/1372.html" rel="external">"von Willebrand disease (VWD): Treatment of major bleeding and major surgery"</a>.)</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Rare inherited clotting factor deficiencies </strong>– (See  <a class="medical medical_review" href="/d/html/1336.html" rel="external">"Rare inherited coagulation disorders"</a>.)</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Acquired inhibitors including acquired hemophilia A</strong> – (See  <a class="medical medical_review" href="/d/html/1305.html" rel="external">"Acquired hemophilia A (and other acquired coagulation factor inhibitors)"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Genetics glossary</strong> – (See  <a class="medical medical_review" href="/d/html/2898.html" rel="external">"Genetics: Glossary of terms"</a>.)</p><p></p><p class="headingAnchor" id="H1835953560"><span class="h1">TERMINOLOGY</span><span class="headingEndMark"> — </span>The following terminology is used herein:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hemophilia</strong> – Typically refers to factor VIII or factor IX deficiency (activity &lt;40 percent of normal) due to a genetic cause.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Hemophilia A</strong> – Factor VIII deficiency due to variation in the <em>F8 </em>gene.</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Hemophilia B</strong> – Factor IX deficiency due to variation in the <em>F9</em> gene.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Disease severity</strong> – When used for hemophilia A and B, disease severity is defined according to factor levels in the plasma.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Severe disease</strong> – Factor activity &lt;1 percent of normal</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Moderate disease</strong> – Factor activity 1 to 5 percent of normal</p><p class="bulletIndent2"><span class="glyph">•</span><strong>Mild disease</strong> – Factor activity &gt;5 percent (and &lt;40 percent) of normal</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Inhibitor</strong> – When used in the context of hemophilia A or B, an inhibitor refers to an autoantibody that arises after administration of exogenous factor and typically interferes with the function of endogenous factor and administered factor concentrates.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hemophilia C</strong> – Sometimes used to refer to factor XI deficiency due to a genetic cause.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Acquired hemophilia</strong> – Refers to hemophilia due to an acquired condition, most frequently an acquired autoantibody that acts as a factor inhibitor for one of the endogenous clotting factors. The most common is acquired factor VIII inhibitor, also called acquired hemophilia A.</p><p></p><p class="headingAnchor" id="H2793845548"><span class="h1">ROLE OF FACTORS VIII AND IX IN HEMOSTASIS</span><span class="headingEndMark"> — </span>The mechanisms of primary and secondary hemostasis are reviewed in a separate topic. (See  <a class="medical medical_review" href="/d/html/1371.html" rel="external">"Overview of hemostasis"</a>.)</p><p>Factor VIII and IX both contribute to secondary hemostasis (formation of a fibrin clot) via their role in the intrinsic pathway X-ase (ten-ase) complex, which activates factor X. This is illustrated in the overview figure (<a class="graphic graphic_figure graphicRef90873" href="/d/graphic/90873.html" rel="external">figure 1</a>) and the traditional depiction of the coagulation cascade (<a class="graphic graphic_figure graphicRef69920" href="/d/graphic/69920.html" rel="external">figure 2</a>).</p><p>The X-ase complex consists of activated factor IX (factor IXa) as the protease; factor X as the substrate; and activated factor VIII (factor VIIIa), calcium, and phospholipids as cofactors in factor X cleavage, as discussed in more detail separately. (See  <a class="medical medical_review" href="/d/html/1316.html" rel="external">"Biology and normal function of factor VIII and factor IX"</a>.)</p><p class="headingAnchor" id="H976720939"><span class="h1">PREVALENCE</span><span class="headingEndMark"> — </span>The prevalence of hemophilia is as follows:</p><p class="bulletIndent1"><span class="glyph">●</span>Hemophilia A (factor VIII deficiency) – 1 in 5000 to 10,000 males. Approximately 60 percent have severe disease (factor VIII activity &lt;1 percent of normal).</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Hemophilia B (factor IX deficiency) – 1 in 25,000 to 30,000 males. Approximately half have severe disease (factor IX activity &lt;1 percent of normal).</p><p></p><p>Additional information and supporting studies are discussed separately. (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Epidemiology'</a>.)</p><p class="headingAnchor" id="H785011230"><span class="h1">TRANSMISSION</span><span class="headingEndMark"> — </span>Hemophilia is usually inherited. However, sporadic cases are common. (See <a class="local">'Sporadic cases'</a> below.)</p><p class="headingAnchor" id="H3963126997"><span class="h2">X-linked recessive inheritance</span><span class="headingEndMark"> — </span>Hemophilia A and B are X-linked recessive disorders (<a class="graphic graphic_figure graphicRef55180" href="/d/graphic/55180.html" rel="external">figure 3</a>). The factor VIII and factor IX genes (<em>F8</em> and <em>F9</em>) are both located on the X chromosome, and males are hemizygous for most X-linked genes. Males with a variant (mutation such as a deletion or inversion) that interferes with factor activity are most likely to be affected. </p><p class="bulletIndent1"><span class="glyph">●</span><strong>Males</strong> – Males are predominantly affected because they have a single X chromosome that contains the <em>F8</em> or <em>F9</em> gene with the hemophilia variant (ie, they are hemizygous for the variant).</p><p></p><p class="bulletIndent1">Affected males can only transmit the disease-causing variant to their daughters, who are obligate carriers. This is because an affected male will transmit a Y chromosome to all of his sons and the affected X chromosome to all of his daughters.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Females</strong> – Although female carriers typically have one normal factor allele, they may experience bleeding symptoms similar to those seen in a patient with mild factor deficiency (eg, due to partially skewed X-inactivation [skewed Lyonization]). Therefore, their care and symptoms should be carefully evaluated. (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Bleeding in females/carriers'</a>.)</p><p></p><p class="bulletIndent1">Rarely, females may have more symptomatic hemophilia [<a href="#rid1">1-3</a>]. Possible explanations include coinheritance of pathogenic variants from both parents or more highly skewed X-inactivation [<a href="#rid2">2,4</a>].</p><p></p><p class="bulletIndent1">Other causes of more symptomatic disease in females include the following:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Loss of part of the normal X chromosome, as in Turner syndrome</p><p class="bulletIndent2"><span class="glyph">•</span>Skewed X-inactivation</p><p class="bulletIndent2"><span class="glyph">•</span>Coinheritance of hemophilia mutations from an affected father and a carrier mother</p><p class="bulletIndent2"><span class="glyph">•</span>Other rare genetic events such as chromosomal translocations or pathogenic variants in the <em>XIST</em> gene</p><p></p><p class="bulletIndent1">Female carriers have a 50 percent chance to transmit the disease-causing variant to each of their children. Sons have a 50 percent chance to be affected, and daughters have a 50 percent chance to be carriers. The other half of sons and daughters will receive the unaffected X chromosome.</p><p></p><p class="headingAnchor" id="H1283462643"><span class="h2">Sporadic cases</span><span class="headingEndMark"> — </span>Sporadic cases of hemophilia, in which an affected male is born to a noncarrier female with a negative family history, are sometimes encountered.</p><p>The prevalence of sporadic hemophilia was previously estimated as approximately one-third of cases [<a href="#rid5">5</a>]. However, subsequent studies have shown that the frequency of sporadic disease varies depending on the population tested and the severity of disease [<a href="#rid6">6</a>].</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hemophilia A</strong> – Sporadic cases account for approximately 55 percent of severe disease and approximately 30 percent of mild to moderate disease.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hemophilia B</strong> – Sporadic cases account for approximately 40 percent of severe disease and approximately 30 percent of mild to moderate disease.</p><p></p><p>Approximately 10 to 20 percent of de novo mutational events are likely to be explained by mosaicism, which is similar to the rates observed in other inherited diseases. This can occur when an individual has germline mosaicism, somatic mosaicism, or both. Studies have shown 13 percent of sporadic cases of hemophilia A were the result of somatic mosaicism and 11 percent of sporadic cases of hemophilia B were due to mosaicism [<a href="#rid7">7-9</a>].</p><p>The effect of mosaicism and the risk for transmission of the <em>F8</em> or <em>F9</em> variant that is mosaic in the individual's cells are dependent on the exact stage of embryonic development during which the mutational event occurred, and in turn, which tissues are involved. There have also been cases of men with sporadic mild hemophilia who went on to have a grandson with severe hemophilia due to the grandfather having somatic and germline mosaicism and the grandson having the variant in all of their cells [<a href="#rid8">8-10</a>]. </p><p>When providing genetic counseling to families with sporadic hemophilia, the less likely modes of transmission should be considered. Mosaicism is not often detected by commonly available clinical testing that is provided to such families, although more detection is increasing with newer methodologies. </p><p>Mothers without a family history of hemophilia who have negative carrier testing have an approximately 5 percent risk of having a second child with hemophilia or a daughter who is a carrier due to maternal germline mosaicism [<a href="#rid11">11</a>].</p><p class="headingAnchor" id="H1496685708"><span class="h1">OVERVIEW OF GENE VARIANTS</span></p><p class="headingAnchor" id="H3"><span class="h2">F8 gene (hemophilia A)</span></p><p class="headingAnchor" id="H3404936593"><span class="h3">Gene structure</span><span class="headingEndMark"> — </span>The <em>F8</em> gene is large, comprising approximately 0.1 percent of the X chromosome. It is divided into 26 exons that span 186,000 base pairs and encode a mature protein of 2332 amino acids [<a href="#rid12">12-16</a>].</p><p>The factor VIII protein contains several areas of internal homology, consisting of a heavy chain with A1 and A2 domains; a connecting region with a B domain; and a light chain with A3, C1, and C2 domains [<a href="#rid14">14,15</a>]. Different epitopes on the C2 domain are responsible for binding to phosphatidylserine, the procoagulant phospholipid on activated platelets, as well as to endothelial cells, von Willebrand factor (which importantly slows the catabolism of factor VIII), factor Xa, and thrombin. The A2 domain and the A1/A3-C1-C2 dimer contribute to the binding of factor IXa. (See  <a class="medical medical_review" href="/d/html/1316.html" rel="external">"Biology and normal function of factor VIII and factor IX", section on 'Factor VIII'</a>.)</p><p class="headingAnchor" id="H1050812437"><span class="h3">Spectrum of variants (F8)</span><span class="headingEndMark"> — </span>Examination of <em>F8</em> genetic variants in individuals with hemophilia A genes has demonstrated a wide range of alterations in multiple regions of the gene [<a href="#rid17">17-21</a>]. The large gene size and the presence of "hotspots" within the gene are thought to increase the likelihood of de novo mutations.</p><p>These include inversions, point mutations (missense and nonsense), small deletions and insertions, large deletions, and splice-site mutations [<a href="#rid16">16</a>]. The majority of variants are listed in searchable databases. (See <a class="local">'Database resources'</a> below.)</p><p>Information from genome sequencing has come from the My Life, Our Future (MLOF) project, which was established in 2012 to create a repository of hemophilia genetic information in the United States [<a href="#rid22">22</a>]. This builds on decades of previous work identifying <em>F8</em> variants using other genetics tools and characterizing the clinical implications. An analysis from the project in 2018 involving nearly 10,000 individuals (affected males and carrier females) identified 700 previously unreported variants and reclassified nondeleterious variants that had previously been reported as causative [<a href="#rid22">22</a>].</p><p>Common <em>F8</em> variants are illustrated in the figure (<a class="graphic graphic_figure graphicRef122164" href="/d/graphic/122164.html" rel="external">figure 4</a>) and include the following:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Intron 22 rearrangements</strong> – Intron 22 rearrangements (typically inversions, also referred to as IVS-22) are the most common type of hemophilia A variants. Approximately 40 to 45 percent of severe hemophilia A is caused by a major inversion of a section of the tip of the long arm of the X chromosome, one breakpoint of which is situated within intron 22 of the <em>F8</em> gene. A consortium study involving 22 laboratories from 14 countries evaluated 2093 patients with severe hemophilia A and found that 740 (35 percent) had a distal <em>F8</em> inversion and 140 (7 percent) had a proximal inversion [<a href="#rid23">23</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Intron 1 rearrangements</strong> – Intron 1 rearrangements (typically inversions, also referred to as IVS-1) also account for some hemophilia A variants. Approximately 1 to 5 percent of severe hemophilia A is caused by an inversion in intron 1 [<a href="#rid16">16,24</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Point mutations and small deletions/insertions</strong> – Other variations including point mutations (including nonsense mutations) and small deletions account for approximately 60 percent of hemophilia A, but no specific mutation or deletion predominates [<a href="#rid16">16</a>]. Mutational hotspots have been identified in the CpG sites (of which there are 70) and in one of two series of multiple adenines in exon 14. CpG island methylation and transcriptional slippage in strings of adenine are both well-characterized mechanisms for the introduction of new mutations. In an analysis of the first 3000 individuals entered in the MLOF repository, approximately 700 unique variants were identified, over 200 of which were new [<a href="#rid21">21</a>]. These were distributed throughout the <em>F8</em> gene.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Gene-disrupting variants are common in severe disease (factor activity &lt;1 percent) [<a href="#rid12">12,25</a>].</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Point mutations that cause an amino acid change (ie, missense mutations) are common in mild to moderate disease.</p><p></p><p class="bulletIndent2">In some cases, these amino acid changes interfere with factor activity but do not reduce protein levels. As a result, factor VIII antigen is normal (also referred to as cross-reacting material positive [CRM+]). This designation is less important since clinical assays measure factor VIII activity rather than factor VIII antigen. Many of these changes affect the A2 domain of the <em>F8</em> gene and some were shown to interfere with factor VIII binding to factor IXa [<a href="#rid26">26</a>].</p><p></p><p>Mechanisms for the different types of variants and aspects of male and female germ cell development that facilitate them have been proposed [<a href="#rid16">16,27-29</a>].</p><p class="headingAnchor" id="H4158545840"><span class="h3">Combined factor VIII and V deficiency (LMAN1 and MCFD2 genes)</span><span class="headingEndMark"> — </span>An uncommon cause of factor VIII deficiency is a mutation in the <em>LMAN1</em> gene, which causes a rare autosomal recessive condition of combined deficiency of factor VIII and factor V. (See  <a class="medical medical_review" href="/d/html/1336.html" rel="external">"Rare inherited coagulation disorders", section on 'Factor V and VIII combined deficiency (F5F8D)'</a>.)</p><p>The <em>LMAN1</em> gene encodes a protein that may act as a molecular chaperone of factors VIII and V and other secreted proteins from the endoplasmic reticulum to the Golgi apparatus [<a href="#rid30">30-32</a>]. Combined deficiency of factors VIII and V is associated with a moderate bleeding tendency, with plasma levels of 5 to 30 percent of normal for both factors. </p><p>In other kindreds, combined factor VIII and V deficiency is associated with mutations in a gene on chromosome 2 called multiple coagulation factor deficiency 2 gene (<em>MCFD2</em>) [<a href="#rid32">32</a>]. <em>MCFD2</em> encodes a protein that forms a Ca<sup>++</sup>-dependent stoichiometric complex with the LMAN1 protein and acts as a cofactor in the intracellular trafficking of factors V and VIII [<a href="#rid32">32</a>].</p><p class="headingAnchor" id="H1946651540"><span class="h3">Modifier genes</span><span class="headingEndMark"> — </span>Variants in genes other than <em>F8</em> have been reported to influence bleeding rates, although their contribution is substantially less than that of <em>F8 </em>genotype.</p><p class="bulletIndent1"><span class="glyph">●</span>Mediators of the inflammatory response may impact the extent of joint damage following joint bleeding [<a href="#rid33">33</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Thrombophilia mutations such as factor V Leiden and prothrombin G20210A may reduce bleeding [<a href="#rid34">34</a>].</p><p></p><p>We do not test for variants in modifier genes and we do not alter our management based on results outside of a research study.</p><p class="headingAnchor" id="H6"><span class="h2">F9 gene (hemophilia B)</span><span class="headingEndMark"> — </span>The <em>F9</em> gene is a 34 kb gene located near the terminus of the long arm of the X chromosome. It contains eight exons and seven introns. <em>F9</em> is the largest gene in the family of vitamin K-dependent coagulation factors; it contains highly conserved features (exon number, splice junction types) that are highly conserved in homologous vitamin K-dependent proteins.</p><p class="headingAnchor" id="H3840712592"><span class="h3">Spectrum of variants (F9)</span><span class="headingEndMark"> — </span>Variants in <em>F9</em> are highly heterogeneous [<a href="#rid21">21,22,35</a>]. They include deletions, duplications, insertions, splice-site variants, missense variants (which cause an amino acid substitution), and nonsense variants (which introduce a premature stop codon), as illustrated in the figure (<a class="graphic graphic_figure graphicRef122164" href="/d/graphic/122164.html" rel="external">figure 4</a>). Most affected families show a unique variant, and next-generation sequencing continues to identify new variants.</p><p>The majority of variants are listed in searchable databases. (See <a class="local">'Database resources'</a> below.)</p><p class="bulletIndent1"><span class="glyph">●</span>Missense mutations are the most common type of variant, accounting for 47 percent of variants in one analysis from the MLOF database [<a href="#rid21">21</a>]. The first person diagnosed with factor IX deficiency, Mr. Christmas, had an <em>F9</em> point mutation that resulted in a single amino acid change (C206S) and caused severe factor IX deficiency [<a href="#rid36">36</a>].</p><p></p><p class="bulletIndent1">Similar to hemophilia A, some hemophilia B families have antigen-positive (cross-reacting material positive [CRM+]) variants, with clinical severity ranging from mild to severe. These patients have antigenic levels of factor IX that are near normal, but they have much lower factor IX activity levels. Approximately one-third of hemophilia B cases fall within this group. Mutations have been described that affect post-translational protein processing, gamma carboxylation and lipid binding, epidermal growth factor (EGF) domain function, zymogen activation, substrate recognition, and/or enzymatic activity.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><em>F9</em> Cambridge and <em>F9</em> Oxford – Alterations in post-translational processing that prevent cleavage of the 18-amino-acid propeptide of factor IX [<a href="#rid37">37,38</a>].</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><em>F9 </em>Chapel Hill – Impaired activation of the zymogen due to interference with one of the factor IX cleavage sites [<a href="#rid39">39</a>].</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><em>F9</em> Vancouver – Interference with factor IX catalytic activity toward factor X due to a point mutation changes an isoleucine to a threonine at amino acid position 397, which affects hydrogen bonding between threonine 397 and the carbonyl oxygen of tryptophan 385, which decreases factor IX binding to factor X in a configuration-favoring catalysis [<a href="#rid40">40</a>].</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Hemophilia Bm – A dysfunctional protein that causes prolongation of the prothrombin time (PT) only when the PT reagent is derived from ox brain as the source of thromboplastin. The Bm phenotype can be caused by mutations in <em>F9</em> that affect amino acid residues 180, 181, or 182 near the amino terminus of the heavy chain or residues 311, 364, 368, 390, 396, or 397 near the beta cleavage site of factor IX [<a href="#rid41">41,42</a>]. Most affected individuals have clinically severe disease.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Complete gene deletions have been described but are rare. In a study of 70 families in France, only two had complete gene deletions [<a href="#rid43">43</a>].</p><p></p><p class="headingAnchor" id="H8"><span class="h3">Leyden phenotype</span><span class="headingEndMark"> — </span>Hemophilia B Leyden is a rare form of hemophilia B caused by a mutation in the <em>F9</em> promoter rather than in the coding region [<a href="#rid44">44-46</a>]. A dozen different point mutations that cause hemophilia B Leyden have been described [<a href="#rid17">17</a>]. These promoter mutations disrupt the binding of transcription factors that increase factor IX expression after puberty. In individuals with the Leyden phenotype, factor IX levels increase following puberty, and these individuals often convert from a more severe to a milder clinical phenotype by adulthood [<a href="#rid47">47</a>].</p><p>Leyden phenotype mutations cluster in a region of the <em>F9</em> promoter that spans from nucleotide c.-50 to c.-18, known as the Leyden-specific region (LSR) [<a href="#rid48">48</a>]. The normal LSR is regulated by transcription factors such as the liver-enriched hepatocyte nuclear factor 4 (HNF4/LF-A1) and Ets1 [<a href="#rid48">48</a>]. At puberty, growth hormone causes these transcription factors to increase. In one instructive case, a 29-year-old man with mild hemophilia B Leyden (baseline factor IX level of 1.2 international units/mL) developed a dramatic rise in factor IX levels (to 10.8 international units/mL) after using anabolic steroids for body building [<a href="#rid49">49</a>].</p><p>In contrast to patients with the Leyden phenotype, those with a point mutation at the -26 position of the <em>F9</em> promoter that disrupts an androgen-responsive element (hemophilia B Brandenburg) do not show improved factor IX levels after puberty [<a href="#rid50">50</a>].</p><p class="headingAnchor" id="H2101670752"><span class="h1">OVERVIEW OF AVAILABLE TOOLS</span></p><p class="headingAnchor" id="H3765858502"><span class="h2">Database resources</span><span class="headingEndMark"> — </span>Several searchable databases of hemophilia A and B mutations are available:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>EAHAD</strong> – The European Association for Haemophilia and Allied Disorders (<a class="external" href="/external-redirect?target_url=http%3A%2F%2Feahad.org%2F&amp;token=VgQ8sMc052PVoyKA%2BCD7Cepj%2BG3CwDDww%2BmwjtpdxhE%3D&amp;TOPIC_ID=1311" target="_blank">EAHAD</a>) hosts separate databases for <a class="external" href="/external-redirect?target_url=https%3A%2F%2Ff8-db.eahad.org%2F&amp;token=gpn2%2F8XRuRJoPr0ijhrk0EGAohza4S9S0ixD5PLyDKyvALUkLBSw0hah5bEtTSir&amp;TOPIC_ID=1311" target="_blank">F8 variants</a> and <a class="external" href="/external-redirect?target_url=https%3A%2F%2Ff9-db.eahad.org%2F&amp;token=AYGCWnssNVTBR6pwxkQdF%2Fu6810D56l76TAfdCEJvEyvALUkLBSw0hah5bEtTSir&amp;TOPIC_ID=1311" target="_blank">F9 variants</a>.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>MLOF repository</strong> The <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.hemophilia.org%2Fresearch%2Fresearch-projects%2Fmy-life-our-future-development-of-the-worlds-largest-genetic-research-repository-for-hemophilia&amp;token=fTSBT3px78ymS3j2gBFKniIv6yI7ij4cnQzvG%2F9ujSGG4Y1hxHhfxiqjKPBHsX1oHQaE5ZKU6pomDGjYXJzwiSBTqQjtns2IbLeTL7hvnxQ%2FNTDDd34CMuX7FmCNkaxenrrCsq8wZhMezii6E5BNgg%3D%3D&amp;TOPIC_ID=1311" target="_blank">My Life, Our future</a> (MLOF) project was established in 2012 to create a repository of hemophilia genetic information in the United States [<a href="#rid22">22</a>]. Data from this repository have identified numerous new mutations and have provided resources for research and clinical care.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>CDC</strong> – The Centers for Disease Control and Prevention (CDC) in the United States hosts the <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.cdc.gov%2Fncbddd%2Fhemophilia%2Fchamps.html&amp;token=a1rnp4ylEWj%2B55Hg2%2FMxBcfPgku8D4kXeKpcaphe0B3%2BluoUyRm34TNMcjUCqAlSfwYnrIT%2BKtCLKEFSkCX88g%3D%3D&amp;TOPIC_ID=1311" target="_blank">CDC Hemophilia Mutation Project</a> (CHAMP for hemophilia A and CHBMP for hemophilia B) [<a href="#rid51">51</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>HAMSTeRS</strong> – The Hemophilia A Mutation, Structure, Test, and Resources (HAMSTeRS) database was established in 1996 to facilitate identification of <em>F8</em> gene variants [<a href="#rid52">52</a>]. It has been discontinued and is no longer available.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hemobase</strong> – A <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fwww.hemobase.com%2FEN%2Findex.htm&amp;token=SzDUOnM2qUS6i9rhTg33MxwJDvnhcM88lx1hF8zLfzzbsKe6pCguqzIg4w40%2FXK2&amp;TOPIC_ID=1311" target="_blank">registry</a> of <em>F8</em> and <em>F9</em> variants characterized in Spanish hemophilia patients [<a href="#rid53">53</a>]. </p><p></p><p class="headingAnchor" id="H3253762954"><span class="h2">Genotyping methods</span><span class="headingEndMark"> — </span>There are several methods available for determining genotype in an individual with hemophilia; the choice among them may depend on whether the individual is a member of a known kindred or has an apparent new mutation.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>MLPA</strong> – Multiplex ligation-dependent probe amplification (MLPA) is a method for quantifying the copy number (dosage) of a genomic sequence and is especially useful for identifying gene deletions. MLPA has been used to identify <em>F8</em> and <em>F9</em> gene deletions, especially those that are missed by polymerase chain reaction (PCR), and in defining carrier status in females [<a href="#rid54">54</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>PCR</strong> – Polymerase chain reaction (PCR) can be used to amplify specific regions of a gene and determine if specific changes (point mutations, intron 22 or intron 1 inversions) are present (ie, the PCR reaction only generates a product if the DNA variation is present) [<a href="#rid55">55</a>]. This method has been used in many clinical laboratories.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Next-generation sequencing</strong> – More recent analyses have used next-generation sequencing (NGS) methods to perform whole genome or whole exome sequencing (WGS or WES), which can identify point mutations as well as more complex changes (deletions, inversions) and, from WGS, changes involving noncoding sequence (promoter regions, introns) [<a href="#rid21">21,22</a>]. In contrast to WGS, WES only analyzes coding regions and thus would not detect intron inversions. NGS methods are discussed in more detail separately. (See  <a class="medical medical_review" href="/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications", section on 'Whole genome, exome, or gene panel'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Sanger sequencing</strong> – Sanger sequencing was the method used to determine DNA sequence for many years before NGS methods became available, and it is considered the gold standard for determining sequence. Variants identified by NGS are confirmed by Sanger sequencing, and some laboratories may continue to use Sanger sequencing. (See  <a class="medical medical_review" href="/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications", section on 'Terminology and evolution of technologies'</a>.)</p><p></p><p class="headingAnchor" id="H1719467264"><span class="h2">Predicting pathogenicity</span><span class="headingEndMark"> — </span>Guidelines for characterizing pathogenicity are presented separately. (See  <a class="medical medical_review" href="/d/html/96539.html" rel="external">"Secondary findings from genetic testing", section on 'Definitions and classification of variants'</a>.)</p><p class="headingAnchor" id="H1845512691"><span class="h1">WHEN, WHERE, AND HOW TO ORDER GENETIC TESTING</span><span class="headingEndMark"> — </span>Genetic testing is appropriate in most patients with hemophilia and many of their female first-degree relatives, especially those who are expected to undergo a surgical procedure and/or may have children. Exceptions include individuals such as at-risk male relatives, who would have testing of their plasma factor activity level (factor VIII or factor IX, depending on the family) rather than genetic testing.</p><p>Ideally, testing is performed under the guidance of a local expert center such as a Comprehensive Care Centre (in Europe) or a Hemophilia Treatment Center (in the United States). These organizations can assist in determining the best methodologic approach for genetic testing and can provide comprehensive counseling and care for the entire family. Additional information about the diagnostic testing is presented separately. (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Diagnostic evaluation'</a>.)</p><p>Clinical genetic testing of the <em>F8</em> and <em>F9</em> genes will not identify the disease-causing variant in all individuals with hemophilia A or B, respectively. There are known to be cases of the bleeding disorder caused by variants not detected by the available methodology. Examples include deep intronic variants and large duplications [<a href="#rid56">56-58</a>]. Analysis of 3000 hemophilia patients through the My Life Our Future (MLOF) program detected a reportable variant in 99.3 percent of hemophilia A patients and 98.1 percent of hemophilia B patients [<a href="#rid21">21,22</a>].</p><p class="headingAnchor" id="H597839042"><span class="h1">CLINICAL IMPLICATIONS OF GENOTYPE</span></p><p class="headingAnchor" id="H2438557682"><span class="h2">Overview of clinical implications</span><span class="headingEndMark"> — </span>Information about the patient's genotype has the following uses:</p><p class="bulletIndent1"><span class="glyph">●</span>It can often be correlated with factor activity levels and disease severity. Some variants are associated with a wider range of factors levels, making it more challenging to predict disease severity. (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia"</a> and <a class="local">'Disease severity'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>It may clarify diagnosis. As an example, identification of an <em>F8</em> mutation in a patient with mild hemophilia A distinguishes the diagnosis of mild hemophilia from type 2N von Willebrand disease (VWD). (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Diagnostic evaluation'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>It helps predict the risk of inhibitor formation. (See  <a class="medical medical_review" href="/d/html/1308.html" rel="external">"Inhibitors in hemophilia: Mechanisms, prevalence, diagnosis, and eradication", section on 'Pathogenesis'</a> and <a class="local">'Risk of inhibitor development'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>It facilitates carrier identification in female family members. (See <a class="local">'Testing at-risk relatives'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>It may be useful for preconception or prenatal counseling and testing, especially if the information would result in a change in reproductive plans. (See <a class="local">'Preconception and prenatal testing and counseling'</a> below.)</p><p></p><p class="headingAnchor" id="H4127099453"><span class="h2">Disease severity</span><span class="headingEndMark"> — </span>The specific factor variant determines factor activity level, which is the major basis for disease severity. (See <a class="local">'Terminology'</a> above.)</p><p>Variants that substantially affect the production or levels of the clotting factor or prevent its major role in clotting are most likely to cause severe disease. Specific genotype-phenotype correlations are discussed above. (See <a class="local">'Overview of gene variants'</a> above.)</p><p>Clinical implications of disease severity are discussed in detail separately. (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Clinical manifestations'</a>.) </p><p class="headingAnchor" id="H3960046014"><span class="h2">Risk of inhibitor development</span><span class="headingEndMark"> — </span>Development of an inhibitor (antibody that blocks the function of infused [and endogenous] factor) is one of the most challenging complications of hemophilia. Individuals who develop an inhibitor typically can no longer use factor replacement to treat bleeding, prevent surgical bleeding, or prophylax against spontaneous bleeds. In hemophilia B, inhibitors are also associated with a risk of anaphylaxis or other severe allergic reactions to infused factor [<a href="#rid59">59</a>].</p><p>Genotype for the relevant factor gene (<em>F8</em> or <em>F9</em>) plays an important role in inhibitor development.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hemophilia A</strong> – Inhibitors develop in approximately one-third of individuals with hemophilia A.</p><p></p><p class="bulletIndent1">A risk score for the development of a factor VIII inhibitor weighted genotype as approximately one-third of the risk, with family history of an inhibitor and intensive treatment at first bleeding episode as the other two major contributors [<a href="#rid60">60</a>].</p><p></p><p class="bulletIndent1">Factor VIII genotypes can be divided into high risk and low risk for inhibitor development.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>High-risk genotypes include large deletions, nonsense mutations (introduction of a premature stop codon), or gene inversions such as intron 22 inversions; in one early study, these all carried similar risks of inhibitor development (between 34 and 38 percent) [<a href="#rid61">61</a>].</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Low-risk genotypes include missense mutations and small deletions; the risk of inhibitor development in individuals with these variants in one study was 4 to 7 percent [<a href="#rid61">61</a>].</p><p></p><p class="bulletIndent1">These differences are generally explained by whether the individual's immune system has been exposed to some endogenous factor VIII during early life and develops tolerance.</p><p></p><p class="bulletIndent1">In nonsevere hemophilia A patients, it has been shown that mutations in the exons coding for the light chain of the protein are known to confer a higher risk for inhibitor development. A few specific sequence variants known to be associated with a particularly high risk of inhibitor development include the following [<a href="#rid62">62-64</a>]:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>p.Tyr2124Cys (Tyr2105Cys) – Inhibitor rate of 44 percent</p><p class="bulletIndent2"><span class="glyph">•</span>p.Arg2169His (Arg2150His) – Inhibitor rate of 24 percent</p><p class="bulletIndent2"><span class="glyph">•</span>p.Trp2248Cys (Trp2229Cys) – Inhibitor rate of 40 percent</p><p class="bulletIndent2"><span class="glyph">•</span>p.Asp2169His (Arg2150His) – Inhibitor rate of 16 to 24 percent</p><p class="bulletIndent2"><span class="glyph">•</span>p.Arg612Cys (Arg592Cys) – Inhibitor rate of 14 to 18 percent</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hemophilia B</strong> – Inhibitors develop in approximately 3 to 5 percent of individuals with severe disease (factor IX activity &lt;1 percent); some reports quote higher rates [<a href="#rid22">22,65</a>]. Complete deletion of the <em>F9</em> gene is strongly associated with inhibitor development and anaphylaxis to infused factor [<a href="#rid59">59</a>].</p><p></p><p class="bulletIndent1">The much lower rate of inhibitor development has been explained by the lower antigenicity of factor IX (perhaps because it is more antigenically similar to other vitamin K-dependent proteins) and because F9 variants are less likely to cause complete absence of the protein (causing infused factor to appear foreign to the recipient's immune system) [<a href="#rid61">61</a>].</p><p></p><p>Additional risk factors for inhibitor development including replacement product, age at which prophylaxis is initiated, and immunologic considerations, as well as a risk score for predicting inhibitor development, are discussed separately. (See  <a class="medical medical_review" href="/d/html/1308.html" rel="external">"Inhibitors in hemophilia: Mechanisms, prevalence, diagnosis, and eradication"</a>.)</p><p class="headingAnchor" id="H730114762"><span class="h2">Genetic testing and counseling</span></p><p class="headingAnchor" id="H3335473354"><span class="h3">Testing at-risk relatives</span><span class="headingEndMark"> — </span>At-risk relatives and other family members of an individual with hemophilia may benefit from counseling and testing [<a href="#rid66">66</a>]. This is especially true for carrier detection in females who may have children or may undergo a surgical procedure. (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Bleeding in females/carriers'</a>.)</p><p class="headingAnchor" id="H689116880"><span class="h3">Preconception and prenatal testing and counseling</span><span class="headingEndMark"> — </span>All individuals with a positive family history of hemophilia (known affected men, known carrier women, and those who are unsure of their status) should have access to preconception testing and counseling. This is an integral part of hemophilia treatment centers (HTCs) and can be provided by a certified genetic counselor or other clinician with expertise in hemophilia genetics. (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Reproductive counseling and testing'</a>.)</p><p>Pregnancy management of a hemophilia carrier is based on her factor activity level; delivery and immediate care of the neonate depend on the sex of the child, which should be assessed prenatally. (See  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Obstetric considerations'</a>.)</p><p class="headingAnchor" id="H866877229"><span class="h1">SOCIETY GUIDELINE LINKS</span><span class="headingEndMark"> — </span>Links to society and government-sponsored guidelines from selected countries and regions around the world are provided separately. (See  <a class="medical medical_society_guidelines" href="/d/html/112393.html" rel="external">"Society guideline links: Hemophilia A and B"</a>.)</p><p class="headingAnchor" id="H302567146"><span class="h1">INFORMATION FOR PATIENTS</span><span class="headingEndMark"> — </span>UpToDate offers two types of patient education materials, "The Basics" and "Beyond the Basics." The Basics patient education pieces are written in plain language, at the 5<sup>th</sup> to 6<sup>th</sup> grade reading level, and they answer the four or five key questions a patient might have about a given condition. These articles are best for patients who want a general overview and who prefer short, easy-to-read materials. Beyond the Basics patient education pieces are longer, more sophisticated, and more detailed. These articles are written at the 10<sup>th</sup> to 12<sup>th</sup> grade reading level and are best for patients who want in-depth information and are comfortable with some medical jargon.</p><p>Here are the patient education articles that are relevant to this topic. We encourage you to print or email these topics to your patients. (You can also locate patient education articles on a variety of subjects by searching on "patient info" and the keyword(s) of interest.)</p><p class="bulletIndent1"><span class="glyph">●</span>Basics topic (see  <a class="medical medical_basics" href="/d/html/83610.html" rel="external">"Patient education: Hemophilia (The Basics)"</a>)</p><p></p><p class="headingAnchor" id="H21506591"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Biology of FVIII and FIX</strong> – Hemophilia A (factor VIII deficiency) and hemophilia B (factor IX deficiency) are due to pathogenic variants (mutations) of the <em>F8</em> and <em>F9</em> genes, respectively. Factor VIII and IX both contribute to formation of a fibrin clot by activating factor X (<a class="graphic graphic_figure graphicRef90873" href="/d/graphic/90873.html" rel="external">figure 1</a>). Activated factor IX (factor IXa) is a protease that cleaves factor X to Xa. Activated factor VIII (factor VIIIa) is a cofactor for this reaction, along with calcium and phospholipids. (See <a class="local">'Terminology'</a> above and  <a class="medical medical_review" href="/d/html/1371.html" rel="external">"Overview of hemostasis"</a> and  <a class="medical medical_review" href="/d/html/1316.html" rel="external">"Biology and normal function of factor VIII and factor IX"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Inheritance</strong> – Hemophilia A and B are X-linked recessive disorders. They predominantly affect males; males have a single X chromosome that contains the <em>F8</em> or <em>F9</em> gene with the disease variant. Affected males transmit the gene variant to all of their daughters and none of their sons. Females are generally unaffected carriers, but they may have mild bleeding due to loss of part of the normal X chromosome (Turner syndrome, skewed X-inactivation, coinheritance of a pathogenic variant from an affected father and a carrier mother). Female carriers transmit the hemophilia gene to approximately half of their children. Hemophilia is mostly inherited, but sporadic cases are common and are responsible for up to half of individuals with hemophilia A and 40 percent with hemophilia B. (See <a class="local">'Transmission'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Common variants</strong> – Common <em>F8</em> and <em>F9</em> variants are illustrated in the figure (<a class="graphic graphic_figure graphicRef122164" href="/d/graphic/122164.html" rel="external">figure 4</a>), and searchable databases are listed above, along with information on genotyping methods and information on test ordering. Computer-based modeling (in silico analysis) may help predict which variants are likely to cause bleeding. (See <a class="local">'Overview of available tools'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Genetic testing</strong> – Genetic testing is appropriate in most patients with hemophilia and many of their first-degree relatives. Ideally, testing is performed under the guidance of a local expert center such as a Comprehensive Care Centre in Europe or a Hemophilia Treatment Center (HTC) in the United States. These organizations can assist in determining the best approach for genetic testing and can provide comprehensive counseling and care for the entire family. (See <a class="local">'When, where, and how to order genetic testing'</a> above and  <a class="medical medical_review" href="/d/html/1310.html" rel="external">"Clinical manifestations and diagnosis of hemophilia", section on 'Diagnostic evaluation'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Clinical implications</strong> – The major clinical implications of hemophilia genotype for the patient are in correlating with disease severity and predicting the risk of inhibitor development. For first-degree relatives, genotype may facilitate carrier identification, which can be used for counseling predicting bleeding in females who require a surgical procedure. (See <a class="local">'Clinical implications of genotype'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Diagnosis and management</strong> – The diagnosis and management of hemophilia A and B and related bleeding disorders are discussed in separate topic reviews listed above. (See <a class="local">'Introduction'</a> above.)</p><p></p><p class="headingAnchor" id="H4167560059"><span class="h1">ACKNOWLEDGMENTS</span><span class="headingEndMark"> — </span>The UpToDate editorial staff acknowledges extensive contributions of Donald H Mahoney, Jr, MD, and Lawrence LK Leung, MD, to earlier versions of this topic review.</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">MERSKEY C. The occurrence of haemophilia in the human female. Q J Med 1951; 20:299.</a></li><li><a class="nounderline abstract_t">Panarello C, Acquila M, Caprino D, et al. Concomitant Turner syndrome and hemophilia A in a female with an idic(X)(p11) heterozygous at locus DXS52. Cytogenet Cell Genet 1992; 59:241.</a></li><li><a class="nounderline abstract_t">Pavlova A, Brondke H, Müsebeck J, et al. Molecular mechanisms underlying hemophilia A phenotype in seven females. J Thromb Haemost 2009; 7:976.</a></li><li><a class="nounderline abstract_t">Valleix S, Vinciguerra C, Lavergne JM, et al. Skewed X-chromosome inactivation in monochorionic diamniotic twin sisters results in severe and mild hemophilia A. Blood 2002; 100:3034.</a></li><li class="breakAll">Treatment of Haemophilia and Other Coagulation Disorders, Biggs R, Macfarlane RG (Eds), F.A. Davis Company, Philadelphia 1966.</li><li><a class="nounderline abstract_t">Kasper CK, Lin JC. Prevalence of sporadic and familial haemophilia. Haemophilia 2007; 13:90.</a></li><li><a class="nounderline abstract_t">Leuer M, Oldenburg J, Lavergne JM, et al. Somatic mosaicism in hemophilia A: a fairly common event. Am J Hum Genet 2001; 69:75.</a></li><li><a class="nounderline abstract_t">Costa C, Frances AM, Letourneau S, et al. Mosaicism in men in hemophilia: is it exceptional? Impact on genetic counselling. J Thromb Haemost 2009; 7:367.</a></li><li><a class="nounderline abstract_t">Kasper CK, Buzin CH. Mosaics and haemophilia. Haemophilia 2009; 15:1181.</a></li><li><a class="nounderline abstract_t">Lu Y, Wu X, Dai J, et al. The characteristics and spectrum of F9 mutations in Chinese sporadic haemophilia B pedigrees. Haemophilia 2019; 25:316.</a></li><li><a class="nounderline abstract_t">Ketterling RP, Vielhaber E, Li X, et al. Germline origins in the human F9 gene: frequent G:C--&gt;A:T mosaicism and increased mutations with advanced maternal age. Hum Genet 1999; 105:629.</a></li><li><a class="nounderline abstract_t">Gitschier J, Wood WI, Goralka TM, et al. Characterization of the human factor VIII gene. Nature 1984; 312:326.</a></li><li><a class="nounderline abstract_t">Wood WI, Capon DJ, Simonsen CC, et al. Expression of active human factor VIII from recombinant DNA clones. Nature 1984; 312:330.</a></li><li><a class="nounderline abstract_t">Vehar GA, Keyt B, Eaton D, et al. Structure of human factor VIII. Nature 1984; 312:337.</a></li><li><a class="nounderline abstract_t">Toole JJ, Knopf JL, Wozney JM, et al. Molecular cloning of a cDNA encoding human antihaemophilic factor. Nature 1984; 312:342.</a></li><li><a class="nounderline abstract_t">Oldenburg J, Ananyeva NM, Saenko EL. Molecular basis of haemophilia A. Haemophilia 2004; 10 Suppl 4:133.</a></li><li><a class="nounderline abstract_t">Goodeve AC. Hemophilia B: molecular pathogenesis and mutation analysis. J Thromb Haemost 2015; 13:1184.</a></li><li><a class="nounderline abstract_t">Edison E, Konkle BA, Goodeve AC. Genetic analysis of bleeding disorders. Haemophilia 2016; 22 Suppl 5:79.</a></li><li><a class="nounderline abstract_t">Li JN, Carrero IG, Dong JF, Yu FL. Complexity and diversity of F8 genetic variations in the 1000 genomes. J Thromb Haemost 2015; 13:2031.</a></li><li><a class="nounderline abstract_t">Bach JE, Wolf B, Oldenburg J, et al. Identification of deep intronic variants in 15 haemophilia A patients by next generation sequencing of the whole factor VIII gene. Thromb Haemost 2015; 114:757.</a></li><li><a class="nounderline abstract_t">Johnsen JM, Fletcher SN, Huston H, et al. Novel approach to genetic analysis and results in 3000 hemophilia patients enrolled in the My Life, Our Future initiative. Blood Adv 2017; 1:824.</a></li><li><a class="nounderline abstract_t">Konkle BA, Johnsen JM, Wheeler M, et al. Genotypes, phenotypes and whole genome sequence: Approaches from the My Life Our Future haemophilia project. Haemophilia 2018; 24 Suppl 6:87.</a></li><li><a class="nounderline abstract_t">Antonarakis SE, Rossiter JP, Young M, et al. Factor VIII gene inversions in severe hemophilia A: results of an international consortium study. Blood 1995; 86:2206.</a></li><li><a class="nounderline abstract_t">Bagnall RD, Waseem N, Green PM, Giannelli F. Recurrent inversion breaking intron 1 of the factor VIII gene is a frequent cause of severe hemophilia A. Blood 2002; 99:168.</a></li><li><a class="nounderline abstract_t">Lawn RM, Vehar GA. The molecular genetics of hemophilia. Sci Am 1986; 254:48.</a></li><li><a class="nounderline abstract_t">Amano K, Sarkar R, Pemberton S, et al. The molecular basis for cross-reacting material-positive hemophilia A due to missense mutations within the A2-domain of factor VIII. Blood 1998; 91:538.</a></li><li><a class="nounderline abstract_t">Ljung RC, Sjörin E. Origin of mutation in sporadic cases of haemophilia A. Br J Haematol 1999; 106:870.</a></li><li><a class="nounderline abstract_t">Mårtensson A, Ivarsson S, Letelier A, et al. Origin of mutation in sporadic cases of severe haemophilia A in Sweden. Clin Genet 2016; 90:63.</a></li><li><a class="nounderline abstract_t">Becker J, Schwaab R, Möller-Taube A, et al. Characterization of the factor VIII defect in 147 patients with sporadic hemophilia A: family studies indicate a mutation type-dependent sex ratio of mutation frequencies. Am J Hum Genet 1996; 58:657.</a></li><li><a class="nounderline abstract_t">Cunningham MA, Pipe SW, Zhang B, et al. LMAN1 is a molecular chaperone for the secretion of coagulation factor VIII. J Thromb Haemost 2003; 1:2360.</a></li><li><a class="nounderline abstract_t">Neerman-Arbez M, Johnson KM, Morris MA, et al. Molecular analysis of the ERGIC-53 gene in 35 families with combined factor V-factor VIII deficiency. Blood 1999; 93:2253.</a></li><li><a class="nounderline abstract_t">Zhang B, Cunningham MA, Nichols WC, et al. Bleeding due to disruption of a cargo-specific ER-to-Golgi transport complex. Nat Genet 2003; 34:220.</a></li><li><a class="nounderline abstract_t">Parhampour B, Dadgoo M, Vasaghi-Gharamaleki B, et al. The effects of six-week resistance, aerobic and combined exercises on the pro-inflammatory and anti-inflammatory markers in overweight patients with moderate haemophilia A: A randomized controlled trial. Haemophilia 2019; 25:e257.</a></li><li><a class="nounderline abstract_t">Tizzano EF, Cornet M, Domènech M, Baiget M. Modifier genes in haemophilia: their expansion in the human genome. Haemophilia 2002; 8:250.</a></li><li><a class="nounderline abstract_t">Lillicrap D. The molecular basis of haemophilia B. Haemophilia 1998; 4:350.</a></li><li><a class="nounderline abstract_t">Taylor SA, Duffin J, Cameron C, et al. Characterization of the original Christmas disease mutation (cysteine 206----serine): from clinical recognition to molecular pathogenesis. Thromb Haemost 1992; 67:63.</a></li><li><a class="nounderline abstract_t">Diuguid DL, Rabiet MJ, Furie BC, et al. Molecular basis of hemophilia B: a defective enzyme due to an unprocessed propeptide is caused by a point mutation in the factor IX precursor. Proc Natl Acad Sci U S A 1986; 83:5803.</a></li><li><a class="nounderline abstract_t">Bentley AK, Rees DJ, Rizza C, Brownlee GG. Defective propeptide processing of blood clotting factor IX caused by mutation of arginine to glutamine at position -4. Cell 1986; 45:343.</a></li><li><a class="nounderline abstract_t">Noyes CM, Griffith MJ, Roberts HR, Lundblad RL. Identification of the molecular defect in factor IX Chapel Hill: substitution of histidine for arginine at position 145. Proc Natl Acad Sci U S A 1983; 80:4200.</a></li><li><a class="nounderline abstract_t">Geddes VA, Le Bonniec BF, Louie GV, et al. A moderate form of hemophilia B is caused by a novel mutation in the protease domain of factor IXVancouver. J Biol Chem 1989; 264:4689.</a></li><li><a class="nounderline abstract_t">Taylor SA, Liddell MB, Peake IR, et al. A mutation adjacent to the beta cleavage site of factor IX (valine 182 to leucine) results in mild haemophilia Bm. Br J Haematol 1990; 75:217.</a></li><li><a class="nounderline abstract_t">Hamaguchi N, Roberts H, Stafford DW. Mutations in the catalytic domain of factor IX that are related to the subclass hemophilia Bm. Biochemistry 1993; 32:6324.</a></li><li><a class="nounderline abstract_t">Attali O, Vinciguerra C, Trzeciak MC, et al. Factor IX gene analysis in 70 unrelated patients with haemophilia B: description of 13 new mutations. Thromb Haemost 1999; 82:1437.</a></li><li><a class="nounderline abstract_t">Veltkamp JJ, Meilof J, Remmelts HG, et al. Another genetic variant of haemophilia B: haemophilia B Leyden. Scand J Haematol 1970; 7:82.</a></li><li><a class="nounderline abstract_t">Briët E, Bertina RM, van Tilburg NH, Veltkamp JJ. Hemophilia B Leyden: a sex-linked hereditary disorder that improves after puberty. N Engl J Med 1982; 306:788.</a></li><li><a class="nounderline abstract_t">Reijnen MJ, Peerlinck K, Maasdam D, et al. Hemophilia B Leyden: substitution of thymine for guanine at position -21 results in a disruption of a hepatocyte nuclear factor 4 binding site in the factor IX promoter. Blood 1993; 82:151.</a></li><li><a class="nounderline abstract_t">Hildyard C, Keeling D. Effect of age on factor IX levels in symptomatic carriers of Haemophila B Leyden. Br J Haematol 2015; 169:448.</a></li><li><a class="nounderline abstract_t">Kurachi S, Huo JS, Ameri A, et al. An age-related homeostasis mechanism is essential for spontaneous amelioration of hemophilia B Leyden. Proc Natl Acad Sci U S A 2009; 106:7921.</a></li><li><a class="nounderline abstract_t">Rimmer EK, Seftel MD, Israels SJ, Houston DS. Unintended benefit of anabolic steroid use in hemophilia B leiden. Am J Hematol 2012; 87:122.</a></li><li><a class="nounderline abstract_t">Crossley M, Ludwig M, Stowell KM, et al. Recovery from hemophilia B Leyden: an androgen-responsive element in the factor IX promoter. Science 1992; 257:377.</a></li><li><a class="nounderline abstract_t">Li T, Miller CH, Payne AB, Craig Hooper W. The CDC Hemophilia B mutation project mutation list: a new online resource. Mol Genet Genomic Med 2013; 1:238.</a></li><li><a class="nounderline abstract_t">Kemball-Cook G, Tuddenham EG. The Factor VIII Mutation Database on the World Wide Web: the haemophilia A mutation, search, test and resource site. HAMSTeRS update (version 3.0). Nucleic Acids Res 1997; 25:128.</a></li><li class="breakAll">http://www.hemobase.com/EN/index.htm (Accessed on April 30, 2021).</li><li><a class="nounderline abstract_t">Belvini D, Salviato R, Radossi P, Tagariello G. Multiplex ligation-dependent probe amplification as first mutation screening for large deletions and duplications in haemophilia. Haemophilia 2017; 23:e124.</a></li><li><a class="nounderline abstract_t">Dutta D, Gunasekera D, Ragni MV, Pratt KP. Accurate, simple, and inexpensive assays to diagnose F8 gene inversion mutations in hemophilia A patients and carriers. Blood Adv 2016; 1:231.</a></li><li><a class="nounderline abstract_t">Inaba H, Shinozawa K, Amano K, Fukutake K. Identification of deep intronic individual variants in patients with hemophilia A by next-generation sequencing of the whole factor VIII gene. Res Pract Thromb Haemost 2017; 1:264.</a></li><li><a class="nounderline abstract_t">Pezeshkpoor B, Zimmer N, Marquardt N, et al. Deep intronic 'mutations' cause hemophilia A: application of next generation sequencing in patients without detectable mutation in F8 cDNA. J Thromb Haemost 2013; 11:1679.</a></li><li><a class="nounderline abstract_t">Feng J, Liu Q, Drost J, Sommer SS. Deep intronic mutations are rarely a cause of hemophilia B. Hum Mutat 1999; 14:267.</a></li><li><a class="nounderline abstract_t">Warrier I, Ewenstein BM, Koerper MA, et al. Factor IX inhibitors and anaphylaxis in hemophilia B. J Pediatr Hematol Oncol 1997; 19:23.</a></li><li><a class="nounderline abstract_t">ter Avest PC, Fischer K, Mancuso ME, et al. Risk stratification for inhibitor development at first treatment for severe hemophilia A: a tool for clinical practice. J Thromb Haemost 2008; 6:2048.</a></li><li><a class="nounderline abstract_t">Schwaab R, Brackmann HH, Meyer C, et al. Haemophilia A: mutation type determines risk of inhibitor formation. Thromb Haemost 1995; 74:1402.</a></li><li><a class="nounderline abstract_t">Peerlinck K, Jacquemin M. Characteristics of inhibitors in mild/moderate haemophilia A. Haemophilia 2006; 12 Suppl 6:43.</a></li><li><a class="nounderline abstract_t">Eckhardt CL, van Velzen AS, Peters M, et al. Factor VIII gene (F8) mutation and risk of inhibitor development in nonsevere hemophilia A. Blood 2013; 122:1954.</a></li><li><a class="nounderline abstract_t">d'Oiron R, Pipe SW, Jacquemin M. Mild/moderate haemophilia A: new insights into molecular mechanisms and inhibitor development. Haemophilia 2008; 14 Suppl 3:138.</a></li><li><a class="nounderline abstract_t">DiMichele D. Inhibitor development in haemophilia B: an orphan disease in need of attention. Br J Haematol 2007; 138:305.</a></li><li class="breakAll">http://www1.wfh.org/publications/files/pdf-1160.pdf (Accessed on July 24, 2019).</li></ol></div><div id="topicVersionRevision">Topic 1311 Version 32.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14883304" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : The occurrence of haemophilia in the human female.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1544315" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Concomitant Turner syndrome and hemophilia A in a female with an idic(X)(p11) heterozygous at locus DXS52.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19302446" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Molecular mechanisms underlying hemophilia A phenotype in seven females.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12351418" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Skewed X-chromosome inactivation in monochorionic diamniotic twin sisters results in severe and mild hemophilia A.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12351418" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Skewed X-chromosome inactivation in monochorionic diamniotic twin sisters results in severe and mild hemophilia A.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17212731" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Prevalence of sporadic and familial haemophilia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11410838" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Somatic mosaicism in hemophilia A: a fairly common event.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19054320" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Mosaicism in men in hemophilia: is it exceptional? Impact on genetic counselling.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19473426" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Mosaics and haemophilia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30648777" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : The characteristics and spectrum of F9 mutations in Chinese sporadic haemophilia B pedigrees.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10647899" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Germline origins in the human F9 gene: frequent G:C--&gt;A:T mosaicism and increased mutations with advanced maternal age.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/6438525" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Characterization of the human factor VIII gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/6438526" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Expression of active human factor VIII from recombinant DNA clones.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/6438527" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Structure of human factor VIII.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/6438528" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Molecular cloning of a cDNA encoding human antihaemophilic factor.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15479386" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Molecular basis of haemophilia A.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25851415" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Hemophilia B: molecular pathogenesis and mutation analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27405681" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Genetic analysis of bleeding disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26383047" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Complexity and diversity of F8 genetic variations in the 1000 genomes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25948085" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Identification of deep intronic variants in 15 haemophilia A patients by next generation sequencing of the whole factor VIII gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29296726" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Novel approach to genetic analysis and results in 3000 hemophilia patients enrolled in the My Life, Our Future initiative.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29878652" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Genotypes, phenotypes and whole genome sequence: Approaches from the My Life Our Future haemophilia project.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7662970" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Factor VIII gene inversions in severe hemophilia A: results of an international consortium study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11756167" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Recurrent inversion breaking intron 1 of the factor VIII gene is a frequent cause of severe hemophilia A.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3006240" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : The molecular genetics of hemophilia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9427707" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : The molecular basis for cross-reacting material-positive hemophilia A due to missense mutations within the A2-domain of factor VIII.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10519986" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Origin of mutation in sporadic cases of haemophilia A.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26661908" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Origin of mutation in sporadic cases of severe haemophilia A in Sweden.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8644728" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Characterization of the factor VIII defect in 147 patients with sporadic hemophilia A: family studies indicate a mutation type-dependent sex ratio of mutation frequencies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14629470" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : LMAN1 is a molecular chaperone for the secretion of coagulation factor VIII.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10090934" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Molecular analysis of the ERGIC-53 gene in 35 families with combined factor V-factor VIII deficiency.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12717434" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Bleeding due to disruption of a cargo-specific ER-to-Golgi transport complex.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31131517" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : The effects of six-week resistance, aerobic and combined exercises on the pro-inflammatory and anti-inflammatory markers in overweight patients with moderate haemophilia A: A randomized controlled trial.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12010419" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Modifier genes in haemophilia: their expansion in the human genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9873754" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : The molecular basis of haemophilia B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1615485" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Characterization of the original Christmas disease mutation (cysteine 206----serine): from clinical recognition to molecular pathogenesis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3461460" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Molecular basis of hemophilia B: a defective enzyme due to an unprocessed propeptide is caused by a point mutation in the factor IX precursor.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3009023" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Defective propeptide processing of blood clotting factor IX caused by mutation of arginine to glutamine at position -4.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/6603618" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : Identification of the molecular defect in factor IX Chapel Hill: substitution of histidine for arginine at position 145.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2494175" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : A moderate form of hemophilia B is caused by a novel mutation in the protease domain of factor IXVancouver.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2372509" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : A mutation adjacent to the beta cleavage site of factor IX (valine 182 to leucine) results in mild haemophilia Bm.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8518277" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Mutations in the catalytic domain of factor IX that are related to the subclass hemophilia Bm.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10595634" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : Factor IX gene analysis in 70 unrelated patients with haemophilia B: description of 13 new mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/5450691" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : Another genetic variant of haemophilia B: haemophilia B Leyden.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7062952" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : Hemophilia B Leyden: a sex-linked hereditary disorder that improves after puberty.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8324220" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : Hemophilia B Leyden: substitution of thymine for guanine at position -21 results in a disruption of a hepatocyte nuclear factor 4 binding site in the factor IX promoter.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25382638" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Effect of age on factor IX levels in symptomatic carriers of Haemophila B Leyden.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19416882" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : An age-related homeostasis mechanism is essential for spontaneous amelioration of hemophilia B Leyden.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22038733" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : Unintended benefit of anabolic steroid use in hemophilia B leiden.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1631558" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : Recovery from hemophilia B Leyden: an androgen-responsive element in the factor IX promoter.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24498619" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : The CDC Hemophilia B mutation project mutation list: a new online resource.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9016520" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : The Factor VIII Mutation Database on the World Wide Web: the haemophilia A mutation, search, test and resource site. HAMSTeRS update (version 3.0).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9016520" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : The Factor VIII Mutation Database on the World Wide Web: the haemophilia A mutation, search, test and resource site. HAMSTeRS update (version 3.0).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28205367" id="rid53" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>54 : Multiplex ligation-dependent probe amplification as first mutation screening for large deletions and duplications in haemophilia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29296938" id="rid54" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>55 : Accurate, simple, and inexpensive assays to diagnose F8 gene inversion mutations in hemophilia A patients and carriers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30046696" id="rid55" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>56 : Identification of deep intronic individual variants in patients with hemophilia A by next-generation sequencing of the whole factor VIII gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23809411" id="rid56" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>57 : Deep intronic 'mutations' cause hemophilia A: application of next generation sequencing in patients without detectable mutation in F8 cDNA.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10477436" id="rid57" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>58 : Deep intronic mutations are rarely a cause of hemophilia B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9065715" id="rid58" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>59 : Factor IX inhibitors and anaphylaxis in hemophilia B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18983511" id="rid59" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>60 : Risk stratification for inhibitor development at first treatment for severe hemophilia A: a tool for clinical practice.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8772209" id="rid60" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>61 : Haemophilia A: mutation type determines risk of inhibitor formation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17123393" id="rid61" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>62 : Characteristics of inhibitors in mild/moderate haemophilia A.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23926300" id="rid62" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>63 : Factor VIII gene (F8) mutation and risk of inhibitor development in nonsevere hemophilia A.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18510534" id="rid63" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>64 : Mild/moderate haemophilia A: new insights into molecular mechanisms and inhibitor development.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17614818" id="rid64" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>65 : Inhibitor development in haemophilia B: an orphan disease in need of attention.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
