<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Tools for genetics and genomics: Gene expression profiling</title><link href="/z/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/z/d/css/icofont.min.css" rel="stylesheet"/><link href="/z/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/z/d/css/animate.css" rel="stylesheet"/><link href="/z/d/css/owl.carousel.css" rel="stylesheet"/><link href="/z/d/css/magnific-popup.css" rel="stylesheet"/><link href="/z/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/z/d/css/style.css" rel="stylesheet"/><link href="/z/d/css/rtl_edited.css" rel="stylesheet"/><link href="/z/d/css/responsive.css" rel="stylesheet"/><link href="https:///uptodate_assets/main.css" rel="stylesheet"/>
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>
<meta content="" name="description"/>
<link href="/z/d/img/favicon.png" rel="icon" type="image/png">
</link></head>
<body>
<!-- Start Main Menu Area -->
<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>
<section class="teacher-area ptb-10">
<div class="container">
</div>
<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Tools for genetics and genomics: Gene expression profiling</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Tools for genetics and genomics: Gene expression profiling</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Authors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Katrina Steiling, MD, MSc</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Stephanie Christenson, MD, MAS</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jennifer S Tirnauer, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Oct 03, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H3544017"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>The genetic basis for disease is determined by the specific variants within genes and noncoding deoxyribonucleic acid (DNA). Most genetic sequences are inherited, although some arise de novo. Gene expression is triggered through interactions with environmental signals, often in a cell-type or timing-specific manner, ultimately leading to synthesis of specific proteins.</p><p>Since messenger ribonucleic acid (mRNA) represents the functional bridge between DNA and protein, alterations in mRNA may serve as markers for the activation (expression) or inhibition (repression) of a particular gene. Analyses of gene expression, referred to as gene expression profiling, can be clinically useful for disease classification, diagnosis, prognosis, and tailoring treatment to underlying genetic or genomic determinants of pharmacologic response. It is also used as a research tool to identify the role of different genes in normal functioning or pathophysiology of disease.</p><p>This topic focuses on the role of mRNA in the cell, platforms for profiling mRNA expression, the challenges in interpreting the data from these analyses, and the emerging clinical applications of gene expression measurements. </p><p>Other molecular tools for evaluating genetic disorders are presented in separate topic reviews:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Cytogenetics </strong>– (See  <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">"Tools for genetics and genomics: Cytogenetics and molecular genetics"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Polymerase chain reaction (PCR)</strong> – (See  <a class="medical medical_review" href="/z/d/html/2896.html" rel="external">"Polymerase chain reaction (PCR)"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Next-generation sequencing</strong> – (See  <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications"</a>.)</p><p></p><p class="headingAnchor" id="H4107601532"><span class="h1">DEFINITIONS</span></p><p class="headingAnchor" id="H2681157443"><span class="h2">mRNA</span><span class="headingEndMark"> — </span>According to the central dogma of biology, ribonucleic acid (RNA) is transcribed from a deoxyribonucleic acid (DNA) template; messenger RNA (mRNA) is then translated into protein  (<a class="graphic graphic_figure graphicRef142009" href="/z/d/graphic/142009.html" rel="external">figure 1</a>). Transcription and translation underlie gene expression. (See  <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">"Basic genetics concepts: DNA regulation and gene expression", section on 'Gene expression'</a>.) </p><p>mRNA accounts for approximately 1 percent of the total RNA in a cell [<a href="#rid1">1</a>]. It is transcribed from approximately 20,000 to 25,000 protein-coding genes in the human genome [<a href="#rid2">2</a>]. </p><p>After mRNA is transcribed from DNA, it typically undergoes further modifications including the addition of a methyl-guanosine cap (5' cap), the addition of a series of adenines to the 3' end of RNA (poly-A tail), and the splicing out of introns  (<a class="graphic graphic_figure graphicRef70605" href="/z/d/graphic/70605.html" rel="external">figure 2</a>) [<a href="#rid3">3</a>]. mRNA is then transported from the nucleus to the cytoplasm where it is translated into protein. mRNA serves as a transient intermediate between DNA and protein and is degraded in minutes to hours [<a href="#rid3">3</a>].</p><p>At any point in time, approximately 3 to 5 percent of genes are active in a particular cell, even though all cells have the same information contained in their DNA. Most of the genome is selectively repressed, a property that is governed by the regulation of gene expression, mostly at the level of mRNA production from DNA (transcription). </p><p>In response to a cellular perturbation, changes in gene expression take place that result in the expression of hundreds of gene products and the suppression of others. This molecular heterogeneity can affect when and how a disease presents clinically in an individual with genetic predisposition to a condition, and it can determine how a given disease will respond to specific treatments in different individuals.</p><p class="headingAnchor" id="H2490430403"><span class="h2">Noncoding RNAs</span><span class="headingEndMark"> — </span>mRNA is the RNA that codes for and is translated into protein. (See <a class="local">'mRNA'</a> above.)</p><p>In addition to mRNA, there are several other classes of RNA that are not translated into protein and serve other functions in the cell. These noncoding RNAs can be measured using variations of the same technologies used to measure mRNA, and they may also have relevance to human disease. </p><p>Classes of noncoding RNAs include the following [<a href="#rid3">3-8</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>Transfer RNAs (tRNAs)</p><p class="bulletIndent1"><span class="glyph">●</span>Ribosomal RNAs (rRNAs)</p><p class="bulletIndent1"><span class="glyph">●</span>Small nuclear RNAs (snRNAs)</p><p class="bulletIndent1"><span class="glyph">●</span>Small nucleolar RNAs (snoRNAs)</p><p class="bulletIndent1"><span class="glyph">●</span>MicroRNAs (miRNAs)</p><p class="bulletIndent1"><span class="glyph">●</span>Piwi-interacting RNAs (piRNAs)</p><p class="bulletIndent1"><span class="glyph">●</span>Long noncoding RNAs (lncRNAs), which include the subclass large intergenic noncoding RNAs (lincRNAs)</p><p></p><p class="headingAnchor" id="H1062368910"><span class="h2">cDNA</span><span class="headingEndMark"> — </span>Complementary DNA (cDNA) is a DNA copy of an mRNA molecule. </p><p>Because mRNA only contains the coding regions of the DNA and not the introns, cDNA is a continuous piece of coding DNA, whereas genomic DNA that exists in vivo contains exons with interspersed introns.</p><p>cDNA is synthesized in a laboratory through an in vitro reverse transcription reaction. mRNA is susceptible to degradation by ribonucleases (RNases), and reverse transcription to cDNA is necessary to create a more stable analyte for genome-wide gene expression profiling. (See <a class="local">'Technical considerations'</a> below.)</p><p class="headingAnchor" id="H2380744261"><span class="h2">Array</span><span class="headingEndMark"> — </span>An array is an orderly arrangement of items. </p><p>In the context of genome-wide expression profiling, array is shorthand for microarray. Microarrays are one method for measuring gene expression on a genome-wide level. Microarrays are the physical substrate that is arrayed with microscopic quantities of nucleic acid probes. These nucleic acid probes each have known sequences and are laid out in known locations on the physical substrate. The nucleic acid probes bind complementary cDNA from a processed sample.</p><p class="headingAnchor" id="H3238884109"><span class="h2">Gene expression</span><span class="headingEndMark"> — </span>Gene expression is the process by which the information encoded in DNA is turned into mRNA. The expression of genes allows cells to adapt their phenotype by turning on (activating) or turning off (repressing) specific functions that the DNA encodes. (See  <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">"Basic genetics concepts: DNA regulation and gene expression", section on 'Gene expression'</a>.)</p><p>Gene expression is measured by assaying mRNA. </p><p class="headingAnchor" id="H1136001189"><span class="h2">Genome-wide expression profiling versus genome-wide genotyping and GWAS</span></p><p class="bulletIndent1"><span class="glyph">●</span>Genome-wide expression profiling is the process of measuring the expression of all genes in the genome. (See <a class="local">'Profiling genome-wide gene expression'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genome-wide genotyping is the process of measuring the allelic status (ie, the genotype) at hundreds of thousands (or millions) of single nucleotide polymorphisms (SNPs) across the genome. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genome-wide association studies (GWAS) are genetic mapping studies that assesses for evidence of association between genetic variants and heritable traits across the entire genome, individually testing for differences in genotype frequencies between trait-expressing individuals (cases) and individuals who do not express the trait (controls) across hundreds of thousands of common SNPs. </p><p></p><p>RNA sequencing platforms can provide genotype information for polymorphisms in mRNA, but this information cannot reliably be used to perform GWAS. (See <a class="local">'Transcriptome sequencing (RNA-seq)'</a> below.) </p><p class="headingAnchor" id="H3774110025"><span class="h2">Transcriptome</span><span class="headingEndMark"> — </span>Transcriptome is the full spectrum mRNA expressed in a cell or tissue. </p><p class="headingAnchor" id="H3544024"><span class="h1">METHODS</span></p><p class="headingAnchor" id="H2999683044"><span class="h2">Technical considerations</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>RNA degradation</strong> – A challenge in measuring RNA relates to the susceptibility of RNAs to degradation by ribonucleases (RNases). RNases are present in saliva, sweat, and other body secretions, and it can be very difficult for laboratory personnel to eliminate or control RNase contamination of laboratory assays [<a href="#rid9">9</a>]. Creation of the corresponding cDNA helps to mitigate this problem. (See <a class="local">'cDNA'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Choice of detection method</strong> – Methods of RNA detection take advantage of the single-stranded structure of RNA and its complementarity to the DNA from which it was transcribed. The method of RNA detection used requires consideration of the scientific question of interest, samples to be studied, experimental design, technical expertise, and availability of technologies. </p><p></p><p class="headingAnchor" id="H3544040"><span class="h2">Profiling small numbers of genes</span><span class="headingEndMark"> — </span>While microarray and sequencing technology may capture gene expression on a whole genome level, additional methods may be used to measure the expression of a small number of genes; the table compares these methods  (<a class="graphic graphic_table graphicRef76106" href="/z/d/graphic/76106.html" rel="external">table 1</a>). </p><p>Some of these methods have fallen out of favor with advancing technology but others, such as polymerase chain reaction (PCR), are commonly used: </p><p class="bulletIndent1"><span class="glyph">●</span>Real time (quantitative reverse-transcription polymerase chain reaction (RT-PCR) (see <a class="local">'Real-time RT-PCR'</a> below and  <a class="medical medical_review" href="/z/d/html/2896.html" rel="external">"Polymerase chain reaction (PCR)", section on 'PCR process'</a>)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>RNA in situ hybridization (see <a class="local">'RNA in situ hybridization'</a> below)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Custom microarrays (see <a class="local">'Custom microarrays'</a> below)</p><p></p><p>Other methods are no longer commonly used:</p><p class="bulletIndent1"><span class="glyph">●</span>Northern blot (see <a class="local">'Northern blot (mainly of historical interest)'</a> below)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Ribonuclease protection assay (see <a class="local">'Ribonuclease protection assay (mainly of historical interest)'</a> below)</p><p></p><p>These methods were originally developed using samples prepared from bulk tissues or samples composed of multiple cell types. Expression profiling at the single-cell level makes it possible to analyze differences in expression profiles of different cells. (See <a class="local">'Single-cell RNA sequencing'</a> below.)</p><p class="headingAnchor" id="H9716963"><span class="h3">Real-time RT-PCR</span><span class="headingEndMark"> — </span>Real-time reverse transcription polymerase chain reaction (RT-PCR) is a flexible and relatively inexpensive approach that can be used to assay small or large numbers of genes from a single sample [<a href="#rid10">10</a>]. (See  <a class="medical medical_review" href="/z/d/html/2896.html" rel="external">"Polymerase chain reaction (PCR)", section on 'Terminology'</a>.) </p><p>PCR is commonly used in clinical laboratories. (See  <a class="medical medical_review" href="/z/d/html/2896.html" rel="external">"Polymerase chain reaction (PCR)", section on 'Clinical applications'</a>.) </p><p>Examples include:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Infectious diseases</strong> – Testing for viruses such as SARS-CoV-2, influenza, and hepatitis C. </p><p class="bulletIndent1"><span class="glyph">●</span><strong>Cancer </strong>– Predictive markers such as for prostate cancer. (See  <a class="medical medical_review" href="/z/d/html/6922.html" rel="external">"Active surveillance for males with clinically localized prostate cancer", section on 'Tissue-based genomic prognostic markers'</a>.)</p><p></p><p>After isolating RNA from a sample, complementary DNAs (cDNAs) are synthesized by reverse transcription with an RNA-dependent DNA polymerase. This cDNA mixture is then combined with a DNA-dependent DNA polymerase and fluorescently-labeled oligonucleotide primers [<a href="#rid11">11</a>]. These primers are short sequences of nucleotides complementary to a portion of the cDNA and allow amplification. Fluorescence increases as the cDNA of interest is amplified with PCR  (<a class="graphic graphic_figure graphicRef71655" href="/z/d/graphic/71655.html" rel="external">figure 3</a>). The fluorescence intensity is monitored and the total number of PCR cycles is counted [<a href="#rid1">1</a>]. The point at which the PCR cycler can distinguish fluorescence related to gene amplification from background is the cycle threshold; this number can be used to estimate the relative starting quantity of the RNA of interest [<a href="#rid10">10</a>]. Careful primer selection is required to prevent amplification of related genes [<a href="#rid1">1</a>]. </p><p class="headingAnchor" id="H371739317"><span class="h3">RNA in situ hybridization</span><span class="headingEndMark"> — </span>In situ hybridization (ISH) uses a nucleic acid probe to detect nucleic acids in a tissue section. ISH can localize the RNA of interest at the anatomic or cellular level. The tissue section is fixed to preserve tissue morphology and nucleic acid integrity [<a href="#rid12">12,13</a>]. The sample is then treated with proteases to eliminate proteins bound to the RNA of interest [<a href="#rid12">12,13</a>]. A labeled probe is hybridized to the sample and detected using autoradiography or chemiluminescence [<a href="#rid13">13</a>]. In situ hybridization using a fluorescently labeled probe is also called fluorescence in situ hybridization (FISH) [<a href="#rid14">14</a>]. </p><p>The use of FISH to detect variations in DNA is discussed separately. (See  <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">"Tools for genetics and genomics: Cytogenetics and molecular genetics", section on 'Fluorescence in situ hybridization'</a>.)</p><p class="headingAnchor" id="H3544068"><span class="h3">Custom microarrays</span><span class="headingEndMark"> — </span>Microarrays can be used for whole genome measurements; custom microarrays can be constructed to analyze expression of a subset of the genome consisting of several hundred to several thousand genes. (See <a class="local">'Microarrays (oligonucleotide arrays)'</a> below.)</p><p>Generally, custom microarrays are used to measure specific sets of RNA that are not as easily assessed using whole transcriptome methods. For example, researchers may choose to use a custom microarray to measure tissue-specific noncoding RNAs from a particular tissue of interest [<a href="#rid15">15-17</a>]. </p><p>Examples of microarrays in clinical use are discussed below. (See <a class="local">'Clinical use'</a> below.)</p><p>Custom microarray methods typically use the same technology as those used for whole genome analyses, in which RNA isolated from one sample is processed on one microarray. (See <a class="local">'Microarrays (oligonucleotide arrays)'</a> below.)</p><p class="headingAnchor" id="H1194268760"><span class="h3">Digital expression profiling</span><span class="headingEndMark"> — </span>Digital expression profiling is a newer adaptation of expression profiling that provides an absolute count of the number of transcripts per gene, rather than abundance relative to other genes. Assays for certain cancer types (breast, pancreatic) are available or under study.<strong> </strong></p><p class="headingAnchor" id="H2245443066"><span class="h3">Northern blot (mainly of historical interest)</span><span class="headingEndMark"> — </span>Northern blots allow the determination of both the presence of an RNA molecule and its size [<a href="#rid18">18</a>]. Northern blotting is not used clinically, and its use in research has declined due to availability of less labor-intensive methods. </p><p>In Northern blotting, RNA molecules are first separated using gel electrophoresis and then transferred and cross-linked to a nylon membrane. The RNA of interest is detected by incubating the membrane with a labeled single-stranded DNA probe that is complementary to this RNA. Probes bound to the RNA of interest can then be detected using chemiluminescence or autoradiography.</p><p class="headingAnchor" id="H3859908472"><span class="h3">Ribonuclease protection assay (mainly of historical interest)</span><span class="headingEndMark"> — </span>Whereas the Northern blot uses complementary DNA probes, the ribonuclease protection assay (RPA) uses complementary RNA riboprobes (single-stranded radiolabeled antisense RNA probes) that are complementary to the RNA of interest [<a href="#rid1">1</a>]. RPA is not used clinically, and its use in research has declined due to availability of less labor-intensive methods.</p><p>In RPA, the riboprobe is incubated with sample RNA to form double-stranded RNA complexes, as well as ribonucleases to degrade excess unbound single-stranded RNA from both the sample and probe. The remaining double-stranded RNA complexes are size-separated by electrophoresis and detected by autoradiography.</p><p class="headingAnchor" id="H3544075"><span class="h2">Profiling genome-wide gene expression</span><span class="headingEndMark"> — </span>Platforms for profiling gene expression take advantage of increased knowledge of the sequence of the human genome and require smaller quantities of starting RNA. The table  (<a class="graphic graphic_table graphicRef76106" href="/z/d/graphic/76106.html" rel="external">table 1</a>) summarizes available methods, which include:</p><p class="bulletIndent1"><span class="glyph">●</span>Microarrays (oligonucleotide arrays) – (See <a class="local">'Microarrays (oligonucleotide arrays)'</a> below.)</p><p class="bulletIndent1"><span class="glyph">●</span>RNA (transcriptome) sequencing – (See <a class="local">'Transcriptome sequencing (RNA-seq)'</a> below.)</p><p class="bulletIndent1"><span class="glyph">●</span>In situ and spatial transcriptomic profiling – (See <a class="local">'In situ and spatial transcriptomics'</a> below.)<strong> </strong></p><p></p><p>Microarrays and sequencing can both assay large numbers of genes with relatively high throughput. As the cost of sequencing has declined, transcriptome sequencing has surpassed microarrays as the platform most often used for gene expression profiling of clinical specimens. </p><p class="headingAnchor" id="H95557895"><span class="h3">Microarrays (oligonucleotide arrays)</span><span class="headingEndMark"> — </span>In this method short probes are synthesized directly onto a slide, allowing for inclusion of a sufficient number of probes to assay RNA expression at a genome-wide level [<a href="#rid19">19,20</a>]. Depending on the commercial manufacturer, probes vary from approximately 20 to 60 base pairs in length. Several types of arrays are commercially available.</p><p>Sample preparation begins with the isolation of RNA from the tissue of interest, resulting in an extraction that contains all of the genes transcribed in the tissue at the time the RNA is isolated. The RNA is then reverse transcribed into cDNA and amplified using PCR. Finally, a <a class="drug drug_general" data-topicid="120386" href="/z/d/drug information/120386.html" rel="external">biotin</a> label is incorporated through an in vitro transcription process, which converts cDNA into labeled cRNA.</p><p>A single sample's labeled cRNA is applied to each array. Hybridization occurs between the labeled cRNA from the sample and complementary probes on the array. This is followed by binding to an avidin-conjugated fluorophore and a washing step that removes any unbound material. The fluorophore is excited by a laser scanner coupled to a computer that captures the fluorescence signals from probes hybridized to the cDNA of interest, thus enabling the detection of the expression of thousands of genes simultaneously.</p><p>In general, the greater the amount of mRNA from a particular gene (the higher that gene's expression), the more fluorescently-labeled material corresponding to that gene will bind to complementary probes on the array. Background fluorescence or nonspecific binding may limit detection of transcripts with very low expression levels. Probe-based detection for gene expression limits analysis to genes that are known.</p><p>Microarray-based assays were the first widely used platforms for whole-transcriptome profiling research, but they have been supplanted by more flexible and robust sequence-based platforms such as RNA sequencing. (See <a class="local">'Transcriptome sequencing (RNA-seq)'</a> below.)</p><p>However, microarray-based assays comprised of select sets of clinically informative genes remain in clinical use. (See <a class="local">'Custom microarrays'</a> above.)</p><p class="headingAnchor" id="H3544096"><span class="h3">Transcriptome sequencing (RNA-seq)</span><span class="headingEndMark"> — </span>An alternative for measuring gene expression is the direct sequencing and quantification of RNA molecules. Other names for transcriptome sequencing include:</p><p class="bulletIndent1"><span class="glyph">●</span>RNA sequencing (RNA-seq)</p><p class="bulletIndent1"><span class="glyph">●</span>Massively parallel RNA sequencing</p><p class="bulletIndent1"><span class="glyph">●</span>Bulk RNA sequencing</p><p class="bulletIndent1"><span class="glyph">●</span>Next-generation RNA sequencing (NGS)</p><p></p><p>Unlike microarray-based platforms (for which probes must be designed to amplify prespecified targets), sequencing-based profiling does not require prior knowledge of the sequences to be assayed. Hence, transcriptome sequencing allows improved detection of low abundance transcripts, as well as detection of novel transcripts and polymorphisms within a transcript's sequence.</p><p>Several commercial platforms are available. The details of each system vary. In general, the sample is prepared so that many sequencing reactions can occur simultaneously and can yield millions of RNA sequence reads obtained by laser scanning [<a href="#rid21">21</a>]. Advances in sample processing techniques also allow for preservation of the identity of the sense and antisense strands [<a href="#rid22">22</a>].</p><p>Next-generation DNA sequencing is discussed in more detail separately. (See  <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications"</a>.)</p><p class="headingAnchor" id="H1072005833"><span class="h3">Single-cell RNA sequencing</span><span class="headingEndMark"> — </span>Bulk RNA sequencing is performed on the pool of RNA extracted from a collection of cells. The resulting expression profile reflects the average transcript abundances across all cells. Thus, when analyzing heterogeneous samples with an admixture of various cell types, it is not clear whether gene expression differences observed between healthy tissue and diseased tissue are due to changes in the relative abundance of cell types with differing patterns of gene expression or to actual changes in the gene expression levels in a specific cell type. </p><p>In contrast, single-cell RNA sequencing (scRNA-seq) produces the transcriptome from a single cell. This approach overcomes the admixture-related limitations of bulk sequencing and allows for more accurate estimation of within-cell gene expression dynamics. scRNA-seq provides unique insights into how individual cells and cell types contribute to human health and disease beyond that which is possible with bulk sequencing. scRNA-seq is used almost exclusively on a genome-wide basis.</p><p>Multiple protocols for scRNA-seq are available. In general, these systems start with isolation of single cells separated manually. Separation methods include [<a href="#rid23">23-25</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>Serial or microwell dilution </p><p class="bulletIndent1"><span class="glyph">●</span>Fluorescence-activated cell sorting (FACS)</p><p class="bulletIndent1"><span class="glyph">●</span>Automated microfluidics-based technologies  </p><p></p><p>This is often followed by a confirmatory procedure such as microscopy to ensure that single cells were indeed isolated. This helps to prevent spurious conclusions based on the evaluation of chambers that are either empty or contain multiple cells. </p><p>After separation, cells are lysed, the RNA fraction is converted to cDNA by reverse transcription, and the cDNA is amplified and sequenced [<a href="#rid23">23,25</a>]. </p><p>Transcript barcoding tags the cell of origin with a unique nucleic acid sequence for identification. Barcoding, microfluidics, and other microwell-plate-based technologies allow for parallel sequencing of large numbers of individual cells [<a href="#rid26">26-28</a>].</p><p>scRNA-seq measures transcript abundance from the cell's cytoplasm and nucleus; it is highly dependent on tissue quality and is best used on fresh samples or those preserved by flash freezing. Because RNA in the nucleus is more stable, single-nuclear RNA-seq (snRNA-seq) has been developed to use cell nuclei (rather than whole cells); this method may have greater applicability for clinical use. </p><p class="headingAnchor" id="H3040993570"><span class="h3">In situ and spatial transcriptomics</span><span class="headingEndMark"> — </span>Spatial transcriptomics refers to expression profiling in different regions of a tissue, as a means of studying expression in local microenvironments [<a href="#rid29">29</a>]. Laser capture microdissection can be used to isolate specific regions within a tissue or tumor to investigate how changes differ from region to region. In situ hybridization of expression probes allows visualization of tissue histology alongside gene expression. These methods can be combined with scRNA-seq. (See <a class="local">'Single-cell RNA sequencing'</a> above.)</p><p>These methods are used for research into disease mechanisms, especially in cancer biology [<a href="#rid30">30,31</a>]. </p><p class="headingAnchor" id="H3399161024"><span class="h1">DATA ANALYSIS AND INTERPRETATION</span><span class="headingEndMark"> — </span>Typically, investigators involved with transcriptome profiling experiments are interested in comparing gene expression across different conditions [<a href="#rid32">32</a>]. While there are many approaches to the data analysis in order to accomplish this goal, there are generally several analytical steps that must first be taken  (<a class="graphic graphic_figure graphicRef71360" href="/z/d/graphic/71360.html" rel="external">figure 4</a>).</p><p>There are three general considerations:</p><p class="bulletIndent1"><span class="glyph">●</span>Preprocessing of raw data (see <a class="local">'Preprocessing of raw data'</a> below)</p><p class="bulletIndent1"><span class="glyph">●</span>Data storage and analysis (see <a class="local">'Data storage'</a> below and <a class="local">'Data analysis'</a> below)</p><p class="bulletIndent1"><span class="glyph">●</span>Biologic interpretation (see <a class="local">'Interpretation'</a> below)</p><p></p><p class="headingAnchor" id="H3544110"><span class="h2">Preprocessing of raw data</span><span class="headingEndMark"> — </span>Many statistical tests used to analyze genomic data require that the data exhibit Gaussian/normal or Poisson distributions with few outlying data points. Samples (or specific genes) that violate these properties often have undue influence on downstream analysis, frequently resulting in skewed, spurious findings that cannot be replicated. Preprocessing refers to the process of transforming raw data to a format suitable for statistical analysis. </p><p>Microarray-based and sequence-based approaches differ in their quantification of transcript abundance, but the general principles of data preprocessing are the same, consisting broadly of two approaches: normalization and quality assessment of the raw data.</p><p>Normalization transforms the raw data to reduce the impact of stochastic (random) variation and technical variation due to the platform, operator, or sample batch. This allows the software to generate per gene and per sample distributions that are more suitable for statistical inference testing. Quality assessment eliminates low-quality microarrays, poorly aligning sequencing reads, or outlier measurements, which, if not addressed, can lead to biased results. Sequence quality can impact normalization methods and, similarly, normalization can impact quality assessment; thus, quality assessment is typically performed both before and after normalization.</p><p class="headingAnchor" id="H3544117"><span class="h3">Quantification and normalization of expression</span><span class="headingEndMark"> — </span>For microarray data analysis, each microarray can be considered a separate experiment that contains slightly different amounts of starting RNA and different labeling efficiencies [<a href="#rid32">32</a>]. Data normalization adjusts the fluorescence intensities representing the amount of RNA bound to each probe so that these intensities are comparable across different arrays. The resulting values are intensities that are normalized to one of several reference distributions, depending on the normalization method used.</p><p>There are several methods for normalizing microarray data, including:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Scaling </strong>– Adjusts intensities by a constant factor so that the average expression level across microarrays is similar.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Quantile normalization</strong> – Adjusts the distribution of intensities across microarrays. As illustrated in the figure  (<a class="graphic graphic_figure graphicRef131176" href="/z/d/graphic/131176.html" rel="external">figure 5</a>), this is accomplished by ranking the probe intensities from highest to lowest for each array. A numerical value is assigned to represent this intensity on an individual array based on the behavior across all arrays and the rank of that probe on the individual array.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>LOWESS </strong>– Locally weighted scatterplot smoothing (LOWESS) adjusts the brightness or darkness of different fluorescent labels for two-color array experiments.</p><p></p><p>For transcriptome sequencing, each sample generates millions of sequencing reads that are used to estimate expression levels of each gene or isoform. To estimate gene expression, these reads must first be assigned to their respective transcript to allow for accurate abundance estimation. This is done by aligning high-quality sequencing reads to a reference genome using one of many available sequence aligners [<a href="#rid33">33</a>]. </p><p>Once aligned, the reads mapped to each transcript are counted to estimate each transcript's abundance, under the assumption that transcripts at higher abundance will yield a greater number of sequence reads compared with transcripts at low abundance. However, because sequencing can begin at any position along the length of the transcript, there are more opportunities for sequencing in larger transcripts than shorter ones. Without accounting for transcript length, RNA sequencing (RNA-seq) gene expression estimates would be biased towards longer genes. To address this, several methods are available, including:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Gene/transcript length normalization</strong> – For the most common of this family of methods, reads in a sample are first normalized for sequencing depth. The depth-normalized reads are then divided by the length of the corresponding gene or isoform in kilobases. This yields reads per kilobase per million reads (RPKM) [<a href="#rid33">33</a>]. Similar methods generate measures expressed as transcripts per million (TPM). While suitable for within-sample comparisons (comparing the relative abundance of two genes in the same sample), this method is not optimal for between-sample comparisons (differential gene expression analysis) and has largely fallen out of favor for most bulk-sequencing analyses [<a href="#rid34">34</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Trimmed mean of M-values (TMM)</strong> – This method uses the weighted average of log expression ratios for each gene calculated for all samples against one reference sample (M-values). Genes with outlier values are thrown out, and a weighted average for all M-values is set for each sample [<a href="#rid34">34,35</a>]. This approach provides superior between-sample normalization compared with TPM methods and is more reliable for downstream statistical inference. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>DESeq</strong> – The DESeq software for transcriptomic analysis uses a normalization procedure that calculates a per-sample scaling factor based on the median of the ratios of each gene's read count over its geometric mean across all samples [<a href="#rid36">36</a>]. Like TMM, this approach is more reliable than TPM methods for differential gene expression analysis.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Variance modeling at the observational level (voom)</strong> – In the voom method, log counts are first normalized for sequencing depth [<a href="#rid37">37</a>]. Then a precision weight incorporating the mean-variance trend for each normalized observation is generated and both the normalized counts and precision weights are entered into the analysis pipeline. This method is particularly useful for small sample sizes or datasets where the between-sample sequencing depth is highly variable.</p><p></p><p>Single-cell sequencing uses similar processes for expression quantification as bulk sequencing, with the caveat that the normalization procedure must account for the high proportion of zero read counts. This so-called "zero-inflation" is a result of two factors:</p><p class="bulletIndent1"><span class="glyph">●</span>Not all cells express the same genes.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Relatively low-abundance transcripts frequently are not always captured/sequenced in a given cell.</p><p></p><p>While many single-cell RNA sequencing (scRNA-seq) normalization methods use scaling factors as described for bulk sequencing and microarrays, additional methods have been developed to manage zero-inflation and the other biases inherent in single-cell sequencing data [<a href="#rid38">38</a>]. This is an active area of method development and study. (See <a class="local">'Single-cell RNA sequencing'</a> above.)</p><p class="headingAnchor" id="H3544124"><span class="h3">Quality assessment</span><span class="headingEndMark"> — </span>Quality assessment occurs both before and after data normalization.</p><p class="bulletIndent1"><span class="glyph">●</span>Pre-normalization quality assessment evaluates the quality of the raw data before preprocessing. For microarrays, the array itself is inspected to ensure there are no bubbles, scratches, or other artifacts on the array. Some commercial arrays also contain controls inserted during sample processing ("spike-in" controls) to ensure that all steps leading to the hybridization were successful. For transcriptome sequencing, each base pair call and individual sequencing read is considered a separate experiment that must be quality controlled [<a href="#rid33">33</a>]. This is performed with tools such as FastQC or NGSQC. Sequencing reads may be then "trimmed" to remove lower-quality base pairs at the ends of each sequencing read, along with the leading or tailing adapter (primer target) sequences that were added to conduct the sequencing reactions [<a href="#rid33">33,39</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Post-normalization quality assessment of either microarray or transcriptome sequencing data evaluates the normalized data from a sample relative to others in the experiment to identify outlier samples or differences in batches of microarrays or sequencing. Samples identified as significantly different from others can be adjusted statistically (ie, batch-level normalization) or excluded from the analysis.</p><p></p><p class="headingAnchor" id="H3544131"><span class="h3">Data transformation</span><span class="headingEndMark"> — </span>Many common statistical procedures assume a normal and continuous distribution of data. Gene expression levels from microarrays or transcriptome sequencing can be mathematically transformed, often using a logarithmic scale, so that they become normally distributed. Transcriptome sequencing data, which is comprised of read counts rather than continuous numerical values, can be filtered to include only higher read counts which may approximate continuous data. Alternatively, sequencing data can be modeled using a distribution more suitable for count data, such as the negative binomial distribution. Preprocessing can also include filtering out low-quality probe sets or genes with low variability across all samples in the experiment.</p><p class="headingAnchor" id="H95557425"><span class="h2">Data storage</span><span class="headingEndMark"> — </span>Microarray and transcriptome sequencing experiments require computational tools to store raw data, analyze gene expression, and ensure uniformity across different laboratories. Most scientific journals specify that raw data be made publicly available as a requirement for publication [<a href="#rid40">40</a>]. </p><p>The fluorescence intensities generated by scanning an oligonucleotide array or sequencing flow cell with a laser scanner results in an image file. </p><p class="bulletIndent1"><span class="glyph">●</span>A typical microarray raw data file, called a CEL file, is 0.1 to 1 gigabytes (GB) per array [<a href="#rid21">21</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>A typical sequencing text-based raw data file, called a FASTQ file, is approximately 1 to 5 GB per sample.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Beyond the storage of raw data, a full experimental dataset includes the final preprocessed expression data, a metadata file that describes the necessary technical information for each sample, and a file that includes the clinical or experimental variables (the phenotypes and covariates) associated with each sample. The metadata describes important experimental details for each sample such as the date of sample collection, measure of RNA quality, identity of the technician running the assay, and the well position of the sample on a 96-well plate.</p><p></p><p>Thus, these experiments generate a large storage requirement (often terabytes) and software to access, integrate, and analyze complex genomic and clinical datasets. As the size and complexity of these data steadily increase, commercial cloud storage and computing infrastructures are increasingly used, many of which have adopted sophisticated security measures to ensure compliance with privacy standards.  </p><p class="headingAnchor" id="H481273275"><span class="h2">Data analysis</span><span class="headingEndMark"> — </span>There are several possible levels of data analysis, ranging from simple statistical tests that can be performed with commercial software packages to advanced analyses and the development of novel algorithms. </p><p>In the clinical setting, the analysis is typically limited to reporting of technical quality and providing a quantitative readout of results. For single gene assays, the readout is either binary (the gene is expressed or not) or continuous (eg, estimated viral load). For multigene assays that estimate prognosis based on the collective expression pattern of a set number of genes, analysis consists of applying a trained statistical model that produces a prognostic score. Examples are listed separately. (See  <a class="medical medical_review" href="/z/d/html/94751.html" rel="external">"Molecular prognostic tests for prostate cancer", section on 'Tests based on molecular characteristics'</a> and  <a class="medical medical_review" href="/z/d/html/126631.html" rel="external">"Deciding when to use adjuvant chemotherapy for hormone receptor-positive, HER2-negative breast cancer"</a>.)</p><p>In research settings, advanced analyses and novel algorithms are implemented with a variety of programming languages, such as Perl and Python, and computational software, such as R and Matlab [<a href="#rid41">41,42</a>]. The flexibility to write, modify, and share algorithms using these tools makes them particularly well suited for gene expression data analysis.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Differential expression</strong> – One of the most common analyses performed on gene expression data is to determine which genes are altered in one condition as compared with another. This can be accomplished by performing a t-test, ANOVA, or linear model for continuous data, or binomial models for numerical data. (See  <a class="medical medical_review" href="/z/d/html/2759.html" rel="external">"Glossary of common biostatistical and epidemiological terms"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Multiple comparison problem</strong> – Statistical analyses of several thousand genes pose unique problems in the interpretation of the results due to the large number of tests performed. This is because every statistical test has a small possibility of leading to the conclusion that an association is present when no such association actually exists, and when thousands of genes are tested with a microarray, an unacceptably high number of false-positive associations may be produced. An overview of statistical principles relevant to the multiple comparison problem is presented separately. (See  <a class="medical medical_review" href="/z/d/html/2777.html" rel="external">"Hypothesis testing in clinical research: Proof, p-values, and confidence intervals"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Single-cell analysis</strong> – Single-cell RNA sequencing (scRNAseq) can identify specific cells or cell types and their functions by interrogating cell-specific molecular signatures [<a href="#rid23">23</a>]. Cell type identification is often done using clustering methods that exploit latent-class modeling to identify cells with similar gene expression patterns. Modified versions of tools for bulk sequencing enable differential expression and network analyses to characterize cell-specific differences in gene expression and function. Methods for scRNAseq analysis are still in their infancy and an active field of study. (See <a class="local">'Single-cell RNA sequencing'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Class prediction</strong> – In this type of analysis, samples from two conditions are split into a training set and a test set. A list of genes that distinguishes the two conditions is derived from the training set of samples, and the accuracy of this gene expression signature is assessed on the test set of samples.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Class discovery</strong> – In addition to these general types of analyses of gene expression data, transcriptome sequencing also allows for more advanced analyses, such as discovery of novel transcripts or isoforms, detection of alternative splicing, and de novo reconstruction of the transcriptome [<a href="#rid39">39</a>].</p><p></p><p class="bulletIndent1">By evaluating genes across all samples regardless of their clinical phenotype, it can be determined which samples are most closely correlated with each other based on gene expression alone. Samples that share similar patterns of gene expression may represent previously unrecognized subtypes of the disease.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Network analysis</strong> – The number of genes assayed by microarrays and transcriptome sequencing allows the entire dataset to be harnessed to make new predictions about how genes might interact. These approaches often operate on the premise that highly correlated genes in a network of gene-gene interactions are involved in the same or overlapping biologic pathways. One approach, weighted gene co-expression network analysis (WGCNA), works by clustering highly correlated genes, defined as "modules," within a gene network [<a href="#rid43">43</a>]. Genome-wide gene expression data can also be integrated with other data types, such as DNA methylation, proteomics, and metabolomics [<a href="#rid39">39</a>].</p><p></p><p class="headingAnchor" id="H3544180"><span class="h2">Interpretation</span><span class="headingEndMark"> — </span>The final step in gene expression data analysis is to interpret the results in a biologically meaningful context. </p><p>Making biologic sense of a whole transcriptome profiling-derived gene list is one of the more challenging aspects of the analysis. There are many strategies for accomplishing this goal. </p><p>Additional studies are often required to validate biologic predictions that are made from the microarray or sequencing data.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Comparison with other datasets</strong> – Several tools exist for comparing gene expression datasets, including large databases containing gene expression data to look for a shared gene expression signature [<a href="#rid44">44,45</a>], alternative gene probes [<a href="#rid46">46</a>], and analytic tools that incorporate phenotypic associations [<a href="#rid47">47-49</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Enrichment ranking</strong> – Gene set enrichment analysis (GSEA) is a method by which gene expression data is ranked by association with phenotypes and is used as a means to identify biologically-relevant pathways [<a href="#rid47">47,48</a>]. Other techniques provide other mechanisms to enrich pathways or functional categories [<a href="#rid50">50</a>] or to visualize previously published interactions between genes of interest [<a href="#rid51">51</a>]. Data visualization using heat maps, which organize samples by columns and genes by rows according to similarity in gene expression, are also useful for determining which groups of genes or samples share similar patterns of expression. Gene set variation analysis (GSVA) uses a similar approach to identify pathway enrichment in a gene-expression dataset [<a href="#rid52">52</a>].</p><p></p><p class="headingAnchor" id="H686018582"><span class="h1">CLINICAL AND RESEARCH APPLICATIONS</span><span class="headingEndMark"> — </span>Gene expression profiling is primarily used for research purposes, but a few expression profiling assays have been incorporated into clinical practice to assist with diagnosis and disease classification.</p><p class="headingAnchor" id="H2279827391"><span class="h2">Clinical use</span><span class="headingEndMark"> — </span>Gene expression profiling within clinical specimens has the potential to be used for improving diagnosis, clarifying prognosis, and helping to optimize treatment. As the platforms for measuring gene expression continue to evolve, personalized approaches to the diagnosis and treatment of complex human disease may increasingly find their place in routine clinical practice. (See  <a class="medical medical_review" href="/z/d/html/14599.html" rel="external">"Personalized medicine"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Breast cancer</strong> – The most advanced application of gene expression profiling is in predicting disease outcomes, which may help in treatment decisions for patients with certain cancers. The risk of certain therapies might be outweighed by the potential benefit for patients at high risk for relapse; in such cases, cancer therapy may be favored, whereas for those with a lower risk of relapse, it may be possible to avoid certain therapies. Gene expression profiling has been helpful in targeting appropriate therapy for patients with breast cancer and other cancers. </p><p></p><p class="bulletIndent1">A breast cancer classification assay such as the Oncotype DX can be performed on biopsy specimens to guide management decisions. (See  <a class="medical medical_review" href="/z/d/html/88415.html" rel="external">"Prognostic and predictive factors in metastatic breast cancer", section on 'Predictive factors'</a> and  <a class="medical medical_review" href="/z/d/html/126631.html" rel="external">"Deciding when to use adjuvant chemotherapy for hormone receptor-positive, HER2-negative breast cancer"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Heart transplant rejection </strong>– Routine endomyocardial biopsy surveillance is used for cardiac transplant patients to detect acute cellular rejection. Gene expression profiling on peripheral blood monocytes can then be used to determine the need for additional biopsies. In a 2006 study involving 107 heart transplant patients, a profile of the expression of 11 genes in blood monocytes correlated with rejection [<a href="#rid53">53</a>]. (See  <a class="medical medical_review" href="/z/d/html/3519.html" rel="external">"Heart transplantation in adults: Diagnosis of allograft rejection", section on 'Gene expression profiling'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Lung cancer </strong>– Individuals with an abnormal chest computed tomography (CT) scan and positive smoking history often require invasive diagnostic procedures beyond bronchoscopy to diagnose lung cancer. A gene expression profiling assay on bronchoscopy specimens is intended for patients with an intermediate probability of malignancy and a nondiagnostic bronchoscopy result [<a href="#rid54">54</a>]. (See  <a class="medical medical_review" href="/z/d/html/6986.html" rel="external">"Diagnostic evaluation of the incidental pulmonary nodule"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Interstitial lung disease classification </strong>– There are several forms of interstitial lung disease (ILD). Because treatments differ, it is important to differentiate usual interstitial pneumonia (UIP) from other forms of ILD. A gene expression profiling test performed on bronchoscopy specimens is able to distinguish UIP with a specificity of 92 percent and sensitivity of 60 percent compared with histopathology [<a href="#rid55">55</a>]. (See  <a class="medical medical_review" href="/z/d/html/4302.html" rel="external">"Idiopathic interstitial pneumonias: Classification and pathology"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Thyroid cancer</strong> – Thyroid nodules are frequently evaluated using fine-needle aspirates, but this approach sometimes yields indeterminate results and requirement for thyroid surgery to obtain a tissue sample for definitive diagnosis. An expression profiling test (Afirma) had a positive predictive value of 47 percent and a negative predictive value of 96 percent in patients with Bethesda III and IV thyroid nodules [<a href="#rid56">56</a>]. These findings suggest that patients with indeterminate results from a fine needle aspirate of a thyroid nodule can be monitored less invasively, potentially avoiding unnecessary surgery. (See  <a class="medical medical_review" href="/z/d/html/7890.html" rel="external">"Diagnostic approach to and treatment of thyroid nodules", section on 'Indeterminate cytology (Bethesda III and IV)'</a> and  <a class="medical medical_review" href="/z/d/html/116428.html" rel="external">"Evaluation and management of thyroid nodules with indeterminate cytology in adults"</a>.)</p><p></p><p class="headingAnchor" id="H1152649515"><span class="h2">Research applications</span><span class="headingEndMark"> — </span>Samples from human tissues and model systems can be studied in two main types of research:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Discovery-based (data-driven) analyses</strong> – In discovery-based analysis, researchers aim to identify novel biological insights into a disease or other biological state of interest. (See <a class="local">'Discovery'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Hypothesis-driven analyses</strong> – In hypothesis-driven analysis, the scientist has a prespecified question and uses gene expression profiling data to answer this question. (See <a class="local">'Hypothesis testing'</a> below.)</p><p></p><p class="headingAnchor" id="H4002984841"><span class="h3">Discovery</span><span class="headingEndMark"> — </span>Analytical tools discussed above may be used to identify novel biological insights. (See <a class="local">'Methods'</a> above.)</p><p>Differential gene expression and pathway analysis via enrichment ranking is used to identify previously unknown genes, and network analysis can be used to discover previously unappreciated biological pathways. Differential expression and pathway analysis tools broadly consider differences in genes or groups of genes between two groups. These tools may be used to identify differences between disease and control samples or samples that have or have not been treated with a medication of interest. </p><p>Class discovery, in which novel groupings of samples instead of genes are considered, is an alternative widely used discovery-based analytical approach. For example, class discovery may be used to identify subtypes of diseases with similar gene expression patterns in a tissue of interest. Examples of this discovery-based analytical approach include: </p><p class="bulletIndent1"><span class="glyph">●</span><strong>Bladder cancer </strong>– Maintenance treatment of locally advanced or metastatic bladder (urothelial) cancer with the immune checkpoint inhibitor <a class="drug drug_general" data-topicid="112481" href="/z/d/drug information/112481.html" rel="external">avelumab</a> following standard of care chemotherapy was found to prolong overall survival as compared with chemotherapy alone in a randomized trial [<a href="#rid57">57</a>]. Investigators next aimed to identify mechanisms and biomarkers of long-term avelumab treatment response using whole transcriptome profiling of tumor samples, done in combination with immunohistochemistry and whole-exome sequencing [<a href="#rid58">58</a>]. Using differential gene expression and pathway analyses that included network- and enrichment-based approaches, investigators found that avelumab survival benefit was associated with genes and pathways indicative of an enhanced innate and adaptive immune response, while worse prognosis was associated with genes and pathways indicative of tumor growth and angiogenesis. They found that a subset of these genes, when combined with a subset of identified tumor mutations, could predict better overall survival in response to avelumab. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Asthma endotyping</strong> – Asthma is a heterogeneous disease that responds to therapies such as glucocorticoids or biologic agents in a subset of individuals. In a 2007 study, airway samples were collected via research bronchoscopy from individuals with steroid-naïve asthma and healthy controls, and gene expression profiling was analyzed by microarray [<a href="#rid59">59</a>]. Class discovery analyses indicated that approximately 50 percent of the individuals with asthma had relatively high expression of the genes most altered between asthma and control conditions. In the other 50 percent, expression of these genes was similar to controls. Individuals with high expression of these genes were then found to have evidence of an enhanced type 2 inflammatory state, including a better response of lung function to glucocorticoids, than those individuals with low expression of these genes. Thus, the gene expression divided individuals with asthma into two endotypes, or groups of individuals with differing underlying biology, that differed by type 2 inflammation and response to the associated treatments. (See  <a class="medical medical_review" href="/z/d/html/544.html" rel="external">"Severe asthma phenotypes", section on 'Phenotyping based on biomarkers of inflammation'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>miRNAs</strong> <strong>in cancer</strong> – MicroRNAs (miRNAs) have also been found to be dysregulated in a number of solid tumors and hematologic malignancies [<a href="#rid60">60-72</a>]. (See <a class="local">'Noncoding RNAs'</a> above.)</p><p></p><p class="bulletIndent1">A 2012 systematic review of available studies examining associations between miRNA profiling and cancer prognosis found associations between certain miRNAs and poor outcomes, including decreased overall survival, but noted several potential sources of bias [<a href="#rid73">73</a>]. Further work will be needed to validate the use of these findings in clinical applications.</p><p></p><p class="headingAnchor" id="H1200055246"><span class="h3">Hypothesis testing</span><span class="headingEndMark"> — </span>Analytical tools such as differential gene expression and pathway analysis are also used to analyze the data when an investigator is considering a specific hypothesis. In this case, however, the investigator may be asking if a particular gene, or set of genes representing a biological pathway, is altered in one condition compared with another or related to an outcome of interest. By testing only one or a small number of genes in relation to a hypothesis, as opposed to all genes across the transcriptome, the multiple comparisons problem becomes less of an issue. (See <a class="local">'Data analysis'</a> above.)</p><p>The hypothesis-driven approach is often employed to determine if a gene or set of genes can be used as a biomarker to represent a particular biological subgroup of individuals or predict a treatment response. </p><p class="bulletIndent1"><span class="glyph">●</span><strong>Glucocorticoid-unresponsive inflammation in COPD</strong> – Inhaled glucocorticoids are a mainstay of treatment in chronic obstructive pulmonary disease (COPD). However, COPD often does not respond to this treatment. One possible explanation may be that interleukin (IL)-17 driven inflammation in the airways is glucocorticoid-unresponsive and may contribute to disease severity. In a 2019 study, our group generated an airway tissue IL-17 gene signature score, a composite of 11 IL-17-associated genes derived using a cell culture model stimulated with IL-17, to identify a COPD subgroup with enhanced IL-17-driven airway inflammation [<a href="#rid74">74</a>]. This signature was hypothesized to be a biomarker of IL-17-associated immune activity and was used to probe gene expression data in studies of psoriatic skin lesions before and after treatment with anti-IL-17 biologics. In these studies, the IL-17 gene signature decreased in association with clinically recognized treatment response to the biologics, suggesting that the signature did indeed mark IL-17 activity. The signature was then studied in a gene expression dataset of airway samples obtained from a randomized trial of inhaled glucocorticoids versus placebo for COPD. The signature was associated with lack of improvement in lung function with glucocorticoids. Thus, overall, the IL-17 gene signature was associated with anti-IL-17 treatment response in psoriasis and lack of response to glucocorticoids in COPD. </p><p></p><p class="headingAnchor" id="H3715747572"><span class="h2">Barriers</span><span class="headingEndMark"> — </span>Despite the promise of gene expression profiling for clinical and research applications, several barriers interfere with widespread use of these technologies. </p><p class="bulletIndent1"><span class="glyph">●</span><strong>Cost </strong>– The cost of transcriptome and single-cell sequencing is high. However, as cost declines, use is expected to increase. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Sample requirement</strong> – A sufficient quantity of high-quality RNA is required to run a valid assay. </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>RNA quantity </strong>– Needle biopsy specimens contain a relatively small number of cells, and some cells may be needed for initial pathology or cytology testing, leaving insufficient quantities for further analysis.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>RNA quality</strong> – Clinical specimens that are archived, paraffin embedded, and formalin fixed may have nucleic acid degradation, making transcriptome sequencing impossible. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Investigator or laboratory expertise</strong> – Expertise is required for experimental design and data analysis. Studies of microarray biomarker development have suggested that analytic expertise may be a key factor in obtaining reproducible and meaningful results [<a href="#rid75">75</a>]. </p><p></p><p class="headingAnchor" id="H3544264"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Definitions</strong> – Genome-wide expression profiling is a technique that uses data generated from microarrays or transcriptome sequencing. These technologies measure the full spectrum of RNA in a cell, including coding mRNA and noncoding RNAs. (See <a class="local">'Definitions'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Methods</strong> – A variety of methods are used to evaluate gene expression  (<a class="graphic graphic_table graphicRef76106" href="/z/d/graphic/76106.html" rel="external">table 1</a>). These methods all rely on the abundance of messenger ribonucleic acid (mRNA). They differ in their requirements for the amount of starting material, their sensitivity to detect the RNA of interest, technical requirements for running the experiment, and computational requirements for data storage, analysis, and interpretation. Small numbers of genes (100s to 1000s) can be profiled using reverse transcription polymerase chain reaction (RT-PCR), RNA in situ hybridization, and custom microarrays. Genome-wide gene expression can be profiled using microarrays or mRNA sequencing. This can be done on admixtures of cells from tissues or on single cells. Research applications may also incorporate in situ and spatial transcriptomics within a tissue. (See <a class="local">'Methods'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Analysis and interpretation</strong> – There are numerous challenges and pitfalls in the analysis and interpretation of the large volume of data generated, including considerations related to data processing, storage, analysis, and interpretation. All genome-wide gene expression studies should statistically account for the multiple-comparison problem inherent in these technologies. (See <a class="local">'Data analysis and interpretation'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Clinical use</strong> – Gene expression profiling is emerging as a potential approach for the diagnosis and prognosis of complex human disease. It is used in assays to help diagnose or prognosticate several conditions including breast cancer, heart transplant rejection, lung cancer, interstitial lung disease, and thyroid cancer. However, a number of important barriers remain to expanded use of gene expression profiling in clinical settings, including validation of biomarkers in prospective multicenter studies to demonstrate their reproducibility, accuracy, and cost-effectiveness across multiple sites and operators. (See <a class="local">'Clinical use'</a> above and <a class="local">'Barriers'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Research</strong> – Gene expression profiling methods can help to identify previously unknown genes and to discover previously unappreciated biological pathways. Examples include bladder cancer, asthma, and microRNAs (miRNAs). They can also be used to test hypotheses such as the role of interleukin (IL)-17-related genes in response to glucocorticoid treatment. (See <a class="local">'Research applications'</a> above.)</p><p></p><p class="headingAnchor" id="H2593110460"><span class="h1">ACKNOWLEDGMENT</span><span class="headingEndMark"> — </span>The UpToDate editorial staff acknowledges Avrum Spira, MD, MSc, who contributed to an earlier version of this topic review.</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Dvorák Z, Pascussi JM, Modrianský M. Approaches to messenger RNA detection - comparison of methods. Biomed Pap Med Fac Univ Palacky Olomouc Czech Repub 2003; 147:131.</a></li><li><a class="nounderline abstract_t">International Human Genome Sequencing Consortium. Finishing the euchromatic sequence of the human genome. Nature 2004; 431:931.</a></li><li class="breakAll">Brown TA. Genomes 3, 3rd ed, Garland Science, 2007.</li><li><a class="nounderline abstract_t">Lee RC, Feinbaum RL, Ambros V. The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell 1993; 75:843.</a></li><li><a class="nounderline abstract_t">Girard A, Sachidanandam R, Hannon GJ, Carmell MA. A germline-specific class of small RNAs binds mammalian Piwi proteins. Nature 2006; 442:199.</a></li><li><a class="nounderline abstract_t">Aravin A, Gaidatzis D, Pfeffer S, et al. A novel class of small RNAs bind to MILI protein in mouse testes. Nature 2006; 442:203.</a></li><li><a class="nounderline abstract_t">Khalil AM, Guttman M, Huarte M, et al. Many human large intergenic noncoding RNAs associate with chromatin-modifying complexes and affect gene expression. Proc Natl Acad Sci U S A 2009; 106:11667.</a></li><li><a class="nounderline abstract_t">Hon CC, Ramilowski JA, Harshbarger J, et al. An atlas of human long non-coding RNAs with accurate 5' ends. Nature 2017; 543:199.</a></li><li><a class="nounderline abstract_t">Blumberg DD. Creating a ribonuclease-free environment. Methods Enzymol 1987; 152:20.</a></li><li><a class="nounderline abstract_t">Nolan T, Hands RE, Bustin SA. Quantification of mRNA using real-time RT-PCR. Nat Protoc 2006; 1:1559.</a></li><li><a class="nounderline abstract_t">VanGuilder HD, Vrana KE, Freeman WM. Twenty-five years of quantitative PCR for gene expression analysis. Biotechniques 2008; 44:619.</a></li><li><a class="nounderline abstract_t">Gall JG, Pardue ML. Formation and detection of RNA-DNA hybrid molecules in cytological preparations. Proc Natl Acad Sci U S A 1969; 63:378.</a></li><li><a class="nounderline abstract_t">Jin L, Lloyd RV. In situ hybridization: methods and applications. J Clin Lab Anal 1997; 11:2.</a></li><li><a class="nounderline abstract_t">Young AP, Jackson DJ, Wyeth RC. A technical review and guide to RNA fluorescence in situ hybridization. PeerJ 2020; 8:e8806.</a></li><li><a class="nounderline abstract_t">Ali A, Jamieson NB, Khan IN, et al. Prognostic implications of microRNA-21 overexpression in pancreatic ductal adenocarcinoma: an international multicenter study of 686 patients. Am J Cancer Res 2022; 12:5668.</a></li><li><a class="nounderline abstract_t">Kappelhoff R, Puente XS, Wilson CH, et al. Overview of transcriptomic analysis of all human proteases, non-proteolytic homologs and inhibitors: Organ, tissue and ovarian cancer cell line expression profiling of the human protease degradome by the CLIP-CHIP™ DNA microarray. Biochim Biophys Acta Mol Cell Res 2017; 1864:2210.</a></li><li><a class="nounderline abstract_t">Hammond SM. RNAi, microRNAs, and human disease. Cancer Chemother Pharmacol 2006; 58 Suppl 1:s63.</a></li><li><a class="nounderline abstract_t">Alwine JC, Kemp DJ, Stark GR. Method for detection of specific RNAs in agarose gels by transfer to diazobenzyloxymethyl-paper and hybridization with DNA probes. Proc Natl Acad Sci U S A 1977; 74:5350.</a></li><li><a class="nounderline abstract_t">Pease AC, Solas D, Sullivan EJ, et al. Light-generated oligonucleotide arrays for rapid DNA sequence analysis. Proc Natl Acad Sci U S A 1994; 91:5022.</a></li><li><a class="nounderline abstract_t">Nuwaysir EF, Huang W, Albert TJ, et al. Gene expression analysis using oligonucleotide arrays produced by maskless photolithography. Genome Res 2002; 12:1749.</a></li><li><a class="nounderline abstract_t">Wilhelm BT, Landry JR. RNA-Seq-quantitative measurement of expression through massively parallel RNA-sequencing. Methods 2009; 48:249.</a></li><li><a class="nounderline abstract_t">Ozsolak F, Milos PM. RNA sequencing: advances, challenges and opportunities. Nat Rev Genet 2011; 12:87.</a></li><li><a class="nounderline abstract_t">Stegle O, Teichmann SA, Marioni JC. Computational and analytical challenges in single-cell transcriptomics. Nat Rev Genet 2015; 16:133.</a></li><li><a class="nounderline abstract_t">Hwang B, Lee JH, Bang D. Single-cell RNA sequencing technologies and bioinformatics pipelines. Exp Mol Med 2018; 50:1.</a></li><li><a class="nounderline abstract_t">Gawad C, Koh W, Quake SR. Single-cell genome sequencing: current state of the science. Nat Rev Genet 2016; 17:175.</a></li><li><a class="nounderline abstract_t">Islam S, Kjällquist U, Moliner A, et al. Characterization of the single-cell transcriptional landscape by highly multiplex RNA-seq. Genome Res 2011; 21:1160.</a></li><li><a class="nounderline abstract_t">Hashimshony T, Wagner F, Sher N, Yanai I. CEL-Seq: single-cell RNA-Seq by multiplexed linear amplification. Cell Rep 2012; 2:666.</a></li><li><a class="nounderline abstract_t">Macosko EZ, Basu A, Satija R, et al. Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets. Cell 2015; 161:1202.</a></li><li><a class="nounderline abstract_t">Zhang L, Chen D, Song D, et al. Clinical and translational values of spatial transcriptomics. Signal Transduct Target Ther 2022; 7:111.</a></li><li><a class="nounderline abstract_t">He B, Bergenstråhle L, Stenbeck L, et al. Integrating spatial gene expression and breast tumour morphology via deep learning. Nat Biomed Eng 2020; 4:827.</a></li><li><a class="nounderline abstract_t">Moncada R, Barkley D, Wagner F, et al. Integrating microarray-based spatial transcriptomics and single-cell RNA-seq reveals tissue architecture in pancreatic ductal adenocarcinomas. Nat Biotechnol 2020; 38:333.</a></li><li><a class="nounderline abstract_t">Quackenbush J. Microarray data normalization and transformation. Nat Genet 2002; 32 Suppl:496.</a></li><li><a class="nounderline abstract_t">Yang IS, Kim S. Analysis of Whole Transcriptome Sequencing Data: Workflow and Software. Genomics Inform 2015; 13:119.</a></li><li><a class="nounderline abstract_t">Robinson MD, Oshlack A. A scaling normalization method for differential expression analysis of RNA-seq data. Genome Biol 2010; 11:R25.</a></li><li><a class="nounderline abstract_t">Robinson MD, McCarthy DJ, Smyth GK. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 2010; 26:139.</a></li><li><a class="nounderline abstract_t">Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol 2014; 15:550.</a></li><li><a class="nounderline abstract_t">Law CW, Chen Y, Shi W, Smyth GK. voom: Precision weights unlock linear model analysis tools for RNA-seq read counts. Genome Biol 2014; 15:R29.</a></li><li><a class="nounderline abstract_t">Vallejos CA, Risso D, Scialdone A, et al. Normalizing single-cell RNA sequencing data: challenges and opportunities. Nat Methods 2017; 14:565.</a></li><li><a class="nounderline abstract_t">Conesa A, Madrigal P, Tarazona S, et al. A survey of best practices for RNA-seq data analysis. Genome Biol 2016; 17:13.</a></li><li><a class="nounderline abstract_t">Brazma A, Hingamp P, Quackenbush J, et al. Minimum information about a microarray experiment (MIAME)-toward standards for microarray data. Nat Genet 2001; 29:365.</a></li><li class="breakAll">R Development Core Team. R: A Language and Environment for Statistical Computing 2009. R Foundation for Statistical Computing. Available at: www.R-project.org (Accessed on December 14, 2009).</li><li class="breakAll">The Mathworks I. The MathWorks - MATLAB and Simulink for Technical Computing 2009. Available at: www.mathworks.com (Accessed on December 14, 2009).</li><li><a class="nounderline abstract_t">Langfelder P, Horvath S. WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics 2008; 9:559.</a></li><li><a class="nounderline abstract_t">Barrett T, Troup DB, Wilhite SE, et al. NCBI GEO: archive for high-throughput functional genomic data. Nucleic Acids Res 2009; 37:D885.</a></li><li><a class="nounderline abstract_t">Gower AC, Spira A, Lenburg ME. Discovering biological connections between experimental conditions based on common patterns of differential gene expression. BMC Bioinformatics 2011; 12:381.</a></li><li><a class="nounderline abstract_t">Dai M, Wang P, Boyd AD, et al. Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data. Nucleic Acids Res 2005; 33:e175.</a></li><li><a class="nounderline abstract_t">Subramanian A, Tamayo P, Mootha VK, et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci U S A 2005; 102:15545.</a></li><li><a class="nounderline abstract_t">Mootha VK, Lindgren CM, Eriksson KF, et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Nat Genet 2003; 34:267.</a></li><li><a class="nounderline abstract_t">Lamb J, Crawford ED, Peck D, et al. The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease. Science 2006; 313:1929.</a></li><li><a class="nounderline abstract_t">Dennis G Jr, Sherman BT, Hosack DA, et al. DAVID: Database for Annotation, Visualization, and Integrated Discovery. Genome Biol 2003; 4:P3.</a></li><li class="breakAll">Ingenuity Systems. Ingenuity Pathway Analysis Software 2009. Available at: www.ingenuity.com (Accessed on December 14, 2009).</li><li><a class="nounderline abstract_t">Hänzelmann S, Castelo R, Guinney J. GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics 2013; 14:7.</a></li><li><a class="nounderline abstract_t">Deng MC, Eisen HJ, Mehra MR, et al. Noninvasive discrimination of rejection in cardiac allograft recipients using gene expression profiling. Am J Transplant 2006; 6:150.</a></li><li><a class="nounderline abstract_t">Raval AA, Benn BS, Benzaquen S, et al. Reclassification of risk of malignancy with Percepta Genomic Sequencing Classifier following nondiagnostic bronchoscopy. Respir Med 2022; 204:106990.</a></li><li><a class="nounderline abstract_t">Richeldi L, Scholand MB, Lynch DA, et al. Utility of a Molecular Classifier as a Complement to High-Resolution Computed Tomography to Identify Usual Interstitial Pneumonia. Am J Respir Crit Care Med 2021; 203:211.</a></li><li><a class="nounderline abstract_t">Patel KN, Angell TE, Babiarz J, et al. Performance of a Genomic Sequencing Classifier for the Preoperative Diagnosis of Cytologically Indeterminate Thyroid Nodules. JAMA Surg 2018; 153:817.</a></li><li><a class="nounderline abstract_t">Powles T, Park SH, Voog E, et al. Avelumab Maintenance Therapy for Advanced or Metastatic Urothelial Carcinoma. N Engl J Med 2020; 383:1218.</a></li><li><a class="nounderline abstract_t">Powles T, Sridhar SS, Loriot Y, et al. Avelumab maintenance in advanced urothelial carcinoma: biomarker analysis of the phase 3 JAVELIN Bladder 100 trial. Nat Med 2021; 27:2200.</a></li><li><a class="nounderline abstract_t">Woodruff PG, Boushey HA, Dolganov GM, et al. Genome-wide profiling identifies epithelial cell genes associated with asthma and with treatment response to corticosteroids. Proc Natl Acad Sci U S A 2007; 104:15858.</a></li><li><a class="nounderline abstract_t">Calin GA, Sevignani C, Dumitru CD, et al. Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers. Proc Natl Acad Sci U S A 2004; 101:2999.</a></li><li><a class="nounderline abstract_t">Lovat F, Valeri N, Croce CM. MicroRNAs in the pathogenesis of cancer. Semin Oncol 2011; 38:724.</a></li><li><a class="nounderline abstract_t">Esquela-Kerscher A, Slack FJ. Oncomirs - microRNAs with a role in cancer. Nat Rev Cancer 2006; 6:259.</a></li><li><a class="nounderline abstract_t">Calin GA, Dumitru CD, Shimizu M, et al. Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia. Proc Natl Acad Sci U S A 2002; 99:15524.</a></li><li><a class="nounderline abstract_t">Calin GA, Ferracin M, Cimmino A, et al. A MicroRNA signature associated with prognosis and progression in chronic lymphocytic leukemia. N Engl J Med 2005; 353:1793.</a></li><li><a class="nounderline abstract_t">Garzon R, Volinia S, Liu CG, et al. MicroRNA signatures associated with cytogenetics and prognosis in acute myeloid leukemia. Blood 2008; 111:3183.</a></li><li><a class="nounderline abstract_t">Yanaihara N, Caplen N, Bowman E, et al. Unique microRNA molecular profiles in lung cancer diagnosis and prognosis. Cancer Cell 2006; 9:189.</a></li><li><a class="nounderline abstract_t">Yu SL, Chen HY, Chang GC, et al. MicroRNA signature predicts survival and relapse in lung cancer. Cancer Cell 2008; 13:48.</a></li><li><a class="nounderline abstract_t">Raponi M, Dossey L, Jatkoe T, et al. MicroRNA classifiers for predicting prognosis of squamous cell lung cancer. Cancer Res 2009; 69:5776.</a></li><li><a class="nounderline abstract_t">Fanini F, Vannini I, Amadori D, Fabbri M. Clinical implications of microRNAs in lung cancer. Semin Oncol 2011; 38:776.</a></li><li><a class="nounderline abstract_t">Boeri M, Pastorino U, Sozzi G. Role of microRNAs in lung cancer: microRNA signatures in cancer prognosis. Cancer J 2012; 18:268.</a></li><li><a class="nounderline abstract_t">Castañeda CA, Agullo-Ortuño MT, Fresno Vara JA, et al. Implication of miRNA in the diagnosis and treatment of breast cancer. Expert Rev Anticancer Ther 2011; 11:1265.</a></li><li><a class="nounderline abstract_t">Sandhu S, Garzon R. Potential applications of microRNAs in cancer diagnosis, prognosis, and treatment. Semin Oncol 2011; 38:781.</a></li><li><a class="nounderline abstract_t">Nair VS, Maeda LS, Ioannidis JP. Clinical outcome prediction by microRNAs in human cancer: a systematic review. J Natl Cancer Inst 2012; 104:528.</a></li><li><a class="nounderline abstract_t">Christenson SA, van den Berge M, Faiz A, et al. An airway epithelial IL-17A response signature identifies a steroid-unresponsive COPD patient subgroup. J Clin Invest 2019; 129:169.</a></li><li><a class="nounderline abstract_t">Shi L, Campbell G, Jones WD, et al. The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models. Nat Biotechnol 2010; 28:827.</a></li></ol></div><div id="topicVersionRevision">Topic 14602 Version 34.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15037893" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Approaches to messenger RNA detection - comparison of methods.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15496913" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Finishing the euchromatic sequence of the human genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15496913" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Finishing the euchromatic sequence of the human genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8252621" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16751776" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : A germline-specific class of small RNAs binds mammalian Piwi proteins.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16751777" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : A novel class of small RNAs bind to MILI protein in mouse testes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19571010" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Many human large intergenic noncoding RNAs associate with chromatin-modifying complexes and affect gene expression.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28241135" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : An atlas of human long non-coding RNAs with accurate 5' ends.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2443792" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Creating a ribonuclease-free environment.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17406449" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Quantification of mRNA using real-time RT-PCR.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18474036" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Twenty-five years of quantitative PCR for gene expression analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/4895535" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Formation and detection of RNA-DNA hybrid molecules in cytological preparations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9021518" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : In situ hybridization: methods and applications.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32219032" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : A technical review and guide to RNA fluorescence in situ hybridization.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/36628279" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Prognostic implications of microRNA-21 overexpression in pancreatic ductal adenocarcinoma: an international multicenter study of 686 patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28797648" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Overview of transcriptomic analysis of all human proteases, non-proteolytic homologs and inhibitors: Organ, tissue and ovarian cancer cell line expression profiling of the human protease degradome by the CLIP-CHIP™DNA microarray.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17093929" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : RNAi, microRNAs, and human disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/414220" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Method for detection of specific RNAs in agarose gels by transfer to diazobenzyloxymethyl-paper and hybridization with DNA probes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8197176" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Light-generated oligonucleotide arrays for rapid DNA sequence analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12421762" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Gene expression analysis using oligonucleotide arrays produced by maskless photolithography.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19336255" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : RNA-Seq-quantitative measurement of expression through massively parallel RNA-sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21191423" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : RNA sequencing: advances, challenges and opportunities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25628217" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Computational and analytical challenges in single-cell transcriptomics.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30089861" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Single-cell RNA sequencing technologies and bioinformatics pipelines.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26806412" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Single-cell genome sequencing: current state of the science.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21543516" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Characterization of the single-cell transcriptional landscape by highly multiplex RNA-seq.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22939981" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : CEL-Seq: single-cell RNA-Seq by multiplexed linear amplification.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26000488" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/35365599" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Clinical and translational values of spatial transcriptomics.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32572199" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Integrating spatial gene expression and breast tumour morphology via deep learning.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31932730" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Integrating microarray-based spatial transcriptomics and single-cell RNA-seq reveals tissue architecture in pancreatic ductal adenocarcinomas.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12454644" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Microarray data normalization and transformation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26865842" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Analysis of Whole Transcriptome Sequencing Data: Workflow and Software.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20196867" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : A scaling normalization method for differential expression analysis of RNA-seq data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19910308" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25516281" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24485249" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : voom: Precision weights unlock linear model analysis tools for RNA-seq read counts.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28504683" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Normalizing single-cell RNA sequencing data: challenges and opportunities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26813401" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : A survey of best practices for RNA-seq data analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11726920" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Minimum information about a microarray experiment (MIAME)-toward standards for microarray data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11726920" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Minimum information about a microarray experiment (MIAME)-toward standards for microarray data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11726920" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Minimum information about a microarray experiment (MIAME)-toward standards for microarray data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19114008" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : WGCNA: an R package for weighted correlation network analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18940857" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : NCBI GEO: archive for high-throughput functional genomic data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21951600" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : Discovering biological connections between experimental conditions based on common patterns of differential gene expression.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16284200" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : Evolving gene/transcript definitions significantly alter the interpretation of GeneChip data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16199517" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12808457" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17008526" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : The Connectivity Map: using gene-expression signatures to connect small molecules, genes, and disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12734009" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : DAVID: Database for Annotation, Visualization, and Integrated Discovery.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12734009" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : DAVID: Database for Annotation, Visualization, and Integrated Discovery.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23323831" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : GSVA: gene set variation analysis for microarray and RNA-seq data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16433769" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : Noninvasive discrimination of rejection in cardiac allograft recipients using gene expression profiling.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/36283245" id="rid53" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>54 : Reclassification of risk of malignancy with Percepta Genomic Sequencing Classifier following nondiagnostic bronchoscopy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32721166" id="rid54" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>55 : Utility of a Molecular Classifier as a Complement to High-Resolution Computed Tomography to Identify Usual Interstitial Pneumonia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29799911" id="rid55" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>56 : Performance of a Genomic Sequencing Classifier for the Preoperative Diagnosis of Cytologically Indeterminate Thyroid Nodules.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32945632" id="rid56" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>57 : Avelumab Maintenance Therapy for Advanced or Metastatic Urothelial Carcinoma.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34893775" id="rid57" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>58 : Avelumab maintenance in advanced urothelial carcinoma: biomarker analysis of the phase 3 JAVELIN Bladder 100 trial.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17898169" id="rid58" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>59 : Genome-wide profiling identifies epithelial cell genes associated with asthma and with treatment response to corticosteroids.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14973191" id="rid59" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>60 : Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22082758" id="rid60" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>61 : MicroRNAs in the pathogenesis of cancer.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16557279" id="rid61" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>62 : Oncomirs - microRNAs with a role in cancer.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12434020" id="rid62" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>63 : Frequent deletions and down-regulation of micro- RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16251535" id="rid63" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>64 : A MicroRNA signature associated with prognosis and progression in chronic lymphocytic leukemia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18187662" id="rid64" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>65 : MicroRNA signatures associated with cytogenetics and prognosis in acute myeloid leukemia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16530703" id="rid65" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>66 : Unique microRNA molecular profiles in lung cancer diagnosis and prognosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18167339" id="rid66" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>67 : MicroRNA signature predicts survival and relapse in lung cancer.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19584273" id="rid67" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>68 : MicroRNA classifiers for predicting prognosis of squamous cell lung cancer.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22082763" id="rid68" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>69 : Clinical implications of microRNAs in lung cancer.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22647364" id="rid69" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>70 : Role of microRNAs in lung cancer: microRNA signatures in cancer prognosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21916580" id="rid70" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>71 : Implication of miRNA in the diagnosis and treatment of breast cancer.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22082764" id="rid71" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>72 : Potential applications of microRNAs in cancer diagnosis, prognosis, and treatment.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22395642" id="rid72" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>73 : Clinical outcome prediction by microRNAs in human cancer: a systematic review.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30383540" id="rid73" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>74 : An airway epithelial IL-17A response signature identifies a steroid-unresponsive COPD patient subgroup.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20676074" id="rid74" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>75 : The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models.</p>
</a>
</div>
</div>
</div>
</div>
</section>
</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->
<!-- End Footer Area -->
<!-- Back to top -->
<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->
<!-- Prpper JS -->
<!-- Bootstrap Min JS -->
<!-- Classy Nav Min Js -->
<!-- Owl Carousel Min Js -->
<!-- Magnific Popup JS -->
<!-- CounterUp JS -->
<!-- Waypoints JS -->
<!-- Form Validator Min JS -->
<!-- Contact Form Min JS -->
<!-- Main JS -->
<!-- Global site tag (gtag.js) - Google Analytics -->
</body>
</html>
