<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Principles of complex trait genetics</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Principles of complex trait genetics</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Principles of complex trait genetics</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Authors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Juan C Celedón, MD, DrPH</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Gary M Hunninghake, MD, MPH</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jennifer S Tirnauer, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Nov 15, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H3540509"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Most human genetic traits can be classified as either monogenic or complex. Monogenic traits are strongly influenced by pathogenic variation within a single gene and are recognized by their classic patterns of inheritance within families. While monogenic traits formed the basis for "classic" genetics, it has become clear that conditions for which inheritance strictly conforms to Mendelian principles are relatively rare. (See  <a class="medical medical_review" href="/d/html/2897.html" rel="external">"Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)"</a> and  <a class="medical medical_review" href="/d/html/2898.html" rel="external">"Genetics: Glossary of terms"</a>.)</p><p>Complex traits are believed to result from variation within multiple genes and their interaction with behavioral and environmental factors. Complex traits do not follow readily predictable patterns of inheritance.</p><p>This topic will review the challenges related to the identification of complex trait susceptibility genes, the factors that contribute to phenotypic complexity, and current understanding of the genetic architecture of complex genetic traits. </p><p>Genetic traits with monogenic inheritance, either with a Mendelian or a non-Mendelian pattern, are discussed separately. (See  <a class="medical medical_review" href="/d/html/2897.html" rel="external">"Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)"</a>.)</p><p class="headingAnchor" id="H3540524"><span class="h1">SPECTRUM OF GENETIC VARIATION</span></p><p class="headingAnchor" id="H797497113"><span class="h2">Monogenic versus complex traits</span><span class="headingEndMark"> — </span>Most monogenic diseases are caused by pathogenic variants that reduce the function or stability of a single protein by altering its expression level and/or three-dimensional structure. These pathogenic variants include point mutations (single nucleotide changes that alter the amino acid sequence), insertions, or deletions in the DNA sequence that encodes the protein; or changes in the non-coding DNA that interfere with gene splicing. (See  <a class="medical medical_review" href="/d/html/2897.html" rel="external">"Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Broad and polygenic spectrum of variation</strong> – In complex genetic traits, the spectrum of genetic variation (ie, the types of and numbers of mutations, insertions, and deletions involved in a single disease) is broad and more complicated, and it is often difficult to identify a single causal genetic change. These traits tend to be polygenic. </p><p></p><p class="bulletIndent1">Further, although coding variants have been identified in complex diseases, most of the genetic variations mapped for complex genetic traits tend to be non-coding (located in non-genic, regulatory, or intronic regions) and likely act not by changing the amino acid sequence, but rather by altering one or more of the following:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The control of gene expression, such as by interfering with a transcription factor binding site</p><p class="bulletIndent2"><span class="glyph">•</span>The stability of the RNA or protein product</p><p class="bulletIndent2"><span class="glyph">•</span>Posttranslational modification of the protein product </p><p></p><p class="bulletIndent1">This distinction between monogenic and complex traits, while useful, can be overly simplistic. On the one hand, traits that appear to be monogenic can be influenced by variation in multiple genes ("modifier genes") [<a href="#rid1">1</a>]. Conversely, complex traits can be predominantly influenced by variation in a single gene [<a href="#rid2">2</a>]. (See <a class="local">'Reasons for weak genotype-phenotype correlations'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Challenges to analysis</strong> – The genetic variation in complex traits is also more difficult to analyze. Several factors contribute to difficulty in the recognition of functional regulatory variation:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Regulatory variants reside in noncoding sequences of the genome. Functional sequence annotation of noncoding sequences is frequently less comprehensive than annotation for coding regions.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Numerous genetic variants have been identified that are strongly associated with a complex disease, but are often not definitively known to be related to a gene (eg, the 9p21 locus in coronary artery disease) [<a href="#rid3">3,4</a>]. The mechanisms by which such variants affect complex traits are more difficult to uncover [<a href="#rid5">5</a>].</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Susceptibility loci for complex traits may influence the regulation of several genes within a genomic region [<a href="#rid6">6</a>]. This suggests that some regulatory regions of DNA, rather than affecting a single gene, could play a role in regulating the regional transcription patterns of multiple genes.</p><p></p><p class="headingAnchor" id="H3540531"><span class="h2">Reasons for weak genotype-phenotype correlations</span><span class="headingEndMark"> — </span>Whereas Mendelian traits are characterized by a strong genotype-phenotype correlation, complex genetic traits typically exhibit weak correlations between genotype and phenotype. Data from genome-wide association studies (GWAS) of complex traits demonstrate that many disease susceptibility variants by themselves confer only a modestly increased risk (ie, relative risk ranging between 1.1 and 1.5). (See <a class="local">'Approaches to identifying genetic determinants of complex traits'</a> below.)</p><p>The relatively weak genotype-phenotype correlations for complex traits may be explained by the following hypotheses: </p><p class="bulletIndent1"><span class="glyph">●</span><strong>Regulatory variation</strong> – Changes in the non-coding (ie, regulatory) regions of genes may alter the quantity of a protein without changing its function. The effect on cellular phenotype may be more subtle than the effect of coding pathogenic variants that affect protein structure.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Polygenic effects</strong> – Most complex genetic traits are likely the result of variation within multiple genes. Any individual genetic variant might only confer a relatively small effect; the cumulative effect of multiple genetic variants would result in a much larger overall effect on the trait [<a href="#rid7">7</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Gene-gene interactions</strong> – Some genetic variants may only have an effect that is contingent upon the presence of additional genetic variants at nearby or distant loci. Determining these interactions between genes presents an enormous mathematical challenge.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Gene-environment interactions</strong> – Environmental exposures (broadly defined as all non-genetic exposures) are likely to have a strong influence on the manifestation of most complex traits. These can include behavioral exposures and exposures about which the individual is unaware. The effect of a particular environmental exposure on the expression of a genetic determinant has been demonstrated for some conditions (ie, the effect of an MMP12 variant on chronic obstructive pulmonary disease [COPD] risk is dependent on cigarette smoke exposure) [<a href="#rid8">8</a>]. Determining how these gene-environment interactions are manifested also presents a substantial mathematical challenge.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Rare (uncommon) susceptibility variants</strong> – Rare (uncommon) susceptibility variants are uncommon genetic changes that may play a minor role in a disease phenotype in the population as a whole yet can have a large effect on the phenotype of single individual. These rare variants are difficult to incorporate into population-based genotype-phenotype correlations because of limited statistical power to detect their effect with usual sample size and challenges in developing models to examine their joint effects. (See <a class="local">'Common disease-rare variant hypothesis'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Somatic mutations</strong> – While most genetic studies have focused on inherited germline genetic mutations, there is growing evidence that somatic mutations (occurring in a person's DNA after conception, in a cell that is not a germ cell) can result in cell-specific phenotypic abnormalities that can contribute to the age-related risk of complex disease. An example is clonal hematopoiesis of indeterminate potential (CHIP) with somatic mutations in genes such as <em>DNMT3A</em> and <em>TET2</em> increasing the risk for hematologic malignancy [<a href="#rid9">9</a>]. The risk of developing somatic mutations may be related to inherited germline gene variants [<a href="#rid10">10</a>]. (See  <a class="medical medical_review" href="/d/html/107866.html" rel="external">"Clonal hematopoiesis of indeterminate potential (CHIP) and related disorders of clonal hematopoiesis"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Epigenetic mechanisms</strong> – Epigenetic mechanisms influence the expression of a gene without changing any DNA sequence. Such mechanisms may be heritable or post-natal, and include DNA methylation, histone modification, and non-coding RNA. Environmental or behavioral exposures may lead to human disease through epigenetics. For example, cigarette smoking can induce increased DNA methylation and thus reduced expression of genes acting as tumor suppressors. Moreover, epigenetic mechanisms could mask the identification of causative genetic loci. As an example, lack of awareness of epigenetic effects on the inheritance of telomere length would have obscured the identification of the causative <em>PARN</em> genetic loci in familial pulmonary fibrosis kindred [<a href="#rid11">11</a>]. </p><p></p><p class="headingAnchor" id="H1549795238"><span class="h2">Polygenic risk scores</span><span class="headingEndMark"> — </span>While most clinical genetic tests provide an assessment of a single gene or a small number of genes, most complex diseases are better explained by the combined effect of large numbers of genetic variants whose aggregated effect can approach that of risks seen with variants in monogenic syndromes [<a href="#rid12">12</a>]. These aggregated effects of large numbers of genetic variants are described by weighted scoring systems called polygenic risk scores (PGRs, PRRs) [<a href="#rid13">13</a>]. These scores can be used to assess individual risk in combination with other clinical information such as personal and family history, clinical findings, and disease biomarkers. </p><p class="headingAnchor" id="H3540573"><span class="h1">COMMON VERSUS RARE ALLELES</span><span class="headingEndMark"> — </span>The polygenic nature of complex traits is widely accepted, but controversy exists regarding the likely frequency of the most important susceptibility alleles for complex traits [<a href="#rid14">14,15</a>]. The relative contribution of common variants (minor allele frequency [MAF] ≥5 percent) or rare variants (MAF ≤1 percent) as critical determinants of complex traits is unclear.</p><p class="headingAnchor" id="H707825542"><span class="h2">Impact on disease</span><span class="headingEndMark"> — </span>The comparative impact of common or rare alleles likely depends on the complex trait of interest and whether "impact" is defined on the basis of population attributable risk (favoring common variants) or risk prediction (favoring rare variants with larger effects). Population attributable risk (PAR) refers to the proportion of subjects affected by a disease in a population that can be explained by exposure to a particular risk factor. Risk prediction relates to the likelihood that an individual with a particular risk factor will develop a disease. A classic example from epidemiology helps illustrate the differences between these two concepts:</p><p>Lung cancer risk is increased in both people who smoke and in people who have an occupational history of asbestos exposure [<a href="#rid16">16</a>]. The risk for lung cancer in an individual who worked in an asbestos factory is significantly greater than the risk for lung cancer in the average smoker. However, cigarette smoking is considerably more common than having worked in an asbestos factory. Thus, although exposure to asbestos confers a greater risk for lung cancer (better risk prediction) than a history of smoking at an individual level, reducing exposure to cigarette smoking would have a much greater impact in reducing the number of lung cancer cases than eliminating work in asbestos factories. Viewed through this framework, it is clear that there is value to identifying common and rare determinants of complex genetic traits, as they both affect the development of human diseases.</p><p class="headingAnchor" id="H3540580"><span class="h2">Common disease-common variant hypothesis</span><span class="headingEndMark"> — </span>The common disease-common variant hypothesis posits that common genetic variants underlie susceptibility to most common traits (eg, cardiovascular disease, diabetes, or asthma) [<a href="#rid17">17</a>]. This hypothesis is rooted in population genetics theory, which states that the current human population is the result of a global expansion from a smaller founding population from sub-Saharan Africa. Thus, the current human population shares common genetic variants that were present in the founding population, and common complex traits are the result of inheriting these variants [<a href="#rid18">18</a>].</p><p>To a certain extent, the identification of nearly &gt;71,000 highly replicable genetic loci from genome-wide association studies (GWAS) supports the common disease-common variant hypothesis [<a href="#rid19">19</a>]. However, rare variants may partly explain findings for common variants in GWAS [<a href="#rid20">20</a>] and explain some of the unexplained or "missing" heritability for most complex traits. (See <a class="local">'Approaches to identifying genetic determinants of complex traits'</a> below and  <a class="medical medical_review" href="/d/html/2902.html" rel="external">"Genetic association and GWAS studies: Principles and applications", section on 'Missing heritability'</a>.)</p><p class="headingAnchor" id="H3540587"><span class="h2">Common disease-rare variant hypothesis</span><span class="headingEndMark"> — </span>To date, more than &gt;71,000 common susceptibility variants have been mapped. For a given complex trait (eg, asthma), however, the variants identified explain only a small fraction of the total heritability (proportion of phenotypic variation explained by genetics). Moreover, they appear to be of limited value in predicting disease risk. As an example, while up to 80 percent of the variability in height is felt to be due to genetic factors [<a href="#rid21">21,22</a>], the combined effect of 20 genetic loci identified by GWAS explained less than 3 percent of the heritability of height [<a href="#rid23">23</a>].</p><p>Since the loci identified by an initial GWAS likely represent the loci with the greatest contribution to the heritability, it has been argued that further study of common genetic variation in increasingly larger samples is unlikely to uncover a substantial portion of the "missing heritability" of complex traits [<a href="#rid14">14</a>], which may be at least partly due to the effects of rare variants. Rare variants may be more valuable in predicting disease risk at an individual level than common variants [<a href="#rid24">24-26</a>].</p><p class="headingAnchor" id="H2886789"><span class="h2">Other hypotheses</span><span class="headingEndMark"> — </span>As stated above, the causes of a significant proportion of the heritability of complex traits have yet to be identified. There are plausible explanations for this, including gene-by-gene or gene-by-environment interactions and epigenetic<strong> </strong>mechanisms [<a href="#rid27">27</a>]. Most GWAS performed to date have analyzed data from the perspective of identifying those associations found most frequently, thereby detecting variants with the most significant p-values but not necessarily the strongest genetic effects. Similarly, virtually all published GWAS use a univariate modeling approach that tests for association one variant at a time, ignoring possible effects of two or more variants acting in combination. </p><p>In support of the importance of this concept, studies are beginning to demonstrate that scores based on the simultaneous effect of thousands of variants may help to improve risk prediction and may account for some of the "missing heritability" [<a href="#rid7">7</a>].</p><p>The inherent complexity of common diseases suggests that consideration of these issues will be fruitful. However, the required statistical methods needed for such analyses; the well-powered large studies and computational resources needed to search for complex gene-gene and gene-environment interactions; as well as resources to integrate epigenetic and genomics data at a genome-wide scale, are only now being developed.</p><p class="headingAnchor" id="H3540594"><span class="h1">APPROACHES TO IDENTIFYING GENETIC DETERMINANTS OF COMPLEX TRAITS</span></p><p class="headingAnchor" id="H3836004505"><span class="h2">Linkage analysis</span><span class="headingEndMark"> — </span>Linkage analysis, a statistical technique that successfully identified causal genes for over 2000 monogenic diseases, has had limited success for gene mapping of complex traits such as identification of maturity onset diabetes of the young (MODY) 1, 2, and 3 for diabetes [<a href="#rid28">28</a>]. Identifying pathogenic variants responsible for complex traits has proven to be very challenging. (See  <a class="medical medical_review" href="/d/html/2898.html" rel="external">"Genetics: Glossary of terms"</a>.)</p><p>Until the mid-1990s, the two main approaches to identify susceptibility variants for complex traits were positional cloning (genome-wide linkage analysis followed by fine mapping analyses of linkage and association) and candidate-gene association studies. The candidate gene approach (ie, making an educated guess about the role of a gene in a disease based on prior knowledge of the functions of that gene) has been occasionally successful (eg, identification of APOE4 for Alzheimer's disease) [<a href="#rid29">29</a>]. However, many findings have been inconsistently replicated and applications are limited to conditions with known biology.</p><p class="headingAnchor" id="H2720931801"><span class="h2">GWAS</span><span class="headingEndMark"> — </span>Genome-wide association studies (GWAS) involve the simultaneous testing of multiple genetic variants spread across the human genome for association with a particular disease or trait. Whereas candidate gene approaches require a priori knowledge about the functions of a gene, GWAS studies are a way to scan all the genes in the genome without bias. (See  <a class="medical medical_review" href="/d/html/2902.html" rel="external">"Genetic association and GWAS studies: Principles and applications"</a>.)</p><p>GWAS became feasible upon compilation of high resolution linkage disequilibrium maps of common genetic variants (through the International HapMap Project) [<a href="#rid30">30</a>], and with the advent of high throughput genome-wide genotyping technologies [<a href="#rid31">31</a>]. GWAS have catalogued associations for over 100,000 SNPs, including presentation of more than 71,000 highly-replicable genetic loci for thousands of human diseases and complex traits [<a href="#rid32">32</a>].</p><p>GWAS have identified susceptibility loci with [<a href="#rid33">33</a>] and without [<a href="#rid34">34</a>] known function. In addition, loci identified by GWAS are providing surprising genetic links among seemingly unrelated complex traits. As an example, a single nucleotide polymorphism (SNP) in the transcription factor 2 gene, also known as hepatocyte nuclear factor-1-beta (<em>HNF1B</em>), is associated with both type 2 diabetes and prostate cancer [<a href="#rid35">35</a>]. Although much work remains to be done with regard to functional characterization of well-replicated loci, GWAS have yielded novel insights into the pathogenesis of numerous complex traits. </p><p>The hallmark of a significant finding of genetic association for a complex trait is the ability to reproduce an original finding of association in independent populations. Demonstration that a statistically significant genotype-phenotype correlation can be found in an independent population is considered the most rigorous proof of genetic association and has become a standard in the field. In contrast, all singular findings of genetic association should be considered speculative. An important caveat to this rule is that certain susceptibility loci may only be relevant to certain ethnic groups. Thus, an observation about a genotype-phenotype association should first be tested in a cohort belonging to the same ethnic group as the initial analysis [<a href="#rid36">36</a>].</p><p>Although GWAS have demonstrated success in mapping the associations between common variants and complex traits, newer techniques such as next-generation sequencing, which can be used to sequence the entire exome (referred to as whole exome sequencing [WES]) or the entire genome (referred to as whole genome sequencing [WGS]) have improved our ability to assess the contribution of rare variants to the heritability of complex traits [<a href="#rid37">37</a>]. These studies require novel methodologies (eg, aggregation of data from multiple genetic variants across a single gene or pathway) and sample collection strategies (eg, large sample sizes, pedigree-based studies, or assessments of population isolates) to improve the statistical power to detect true associations [<a href="#rid38">38</a>]. (See  <a class="medical medical_review" href="/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications"</a>.)</p><p class="headingAnchor" id="H3540615"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Definition of complex traits</strong> – Most genetic traits can be classified as either monogenic or complex. Complex trait conditions are those that cannot be explained by alteration of a single gene. They are typically polygenic, with additional influences from gene-gene and gene-environment interactions, epigenetic phenomena, and unidentified rare disease susceptibility variants. The spectrum of genetic variation includes coding genes and regulatory genes. (See <a class="local">'Spectrum of genetic variation'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Polygenic risk scores</strong> – For complex traits, polygenic risk scores can be used to account for the contributions of different genes to the trait or phenotype. (See <a class="local">'Polygenic risk scores'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Contribution of different alleles</strong> – Complex genetic traits may be associated with commonly occurring allelic variants that individually have a weak effect or with rare variants that have a stronger effect. Common variants have a greater impact on population-attributable risk, while uncommon variants have a greater impact on risk prediction at the level of the individual. (See <a class="local">'Common versus rare alleles'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Methods for studying complex traits</strong> – Techniques to identify complex trait genes include linkage analyses, candidate gene testing, genome-wide association studies (GWAS), and next generation sequencing (NGS) of the entire exome or genome. Replication of findings in another cohort is an essential element to validate findings from an initial GWAS or NGS study. (See <a class="local">'Approaches to identifying genetic determinants of complex traits'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Monogenic traits</strong> – Monogenic traits are strongly influenced by pathogenic variation within a single gene and are recognized by their classic Mendelian patterns of inheritance within families. (See  <a class="medical medical_review" href="/d/html/2897.html" rel="external">"Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)"</a>.)</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Gu Y, Harley IT, Henderson LB, et al. Identification of IFRD1 as a modifier gene for cystic fibrosis lung disease. Nature 2009; 458:1039.</a></li><li><a class="nounderline abstract_t">Stefansson H, Rye DB, Hicks A, et al. A genetic risk factor for periodic limb movements in sleep. N Engl J Med 2007; 357:639.</a></li><li><a class="nounderline abstract_t">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature 2007; 447:661.</a></li><li><a class="nounderline abstract_t">Lo Sardo V, Chubukov P, Ferguson W, et al. Unveiling the Role of the Most Impactful Cardiovascular Risk Locus through Haplotype Editing. Cell 2018; 175:1796.</a></li><li><a class="nounderline abstract_t">Harismendy O, Notani D, Song X, et al. 9p21 DNA variants associated with coronary artery disease impair interferon-γ signalling response. Nature 2011; 470:264.</a></li><li><a class="nounderline abstract_t">Verlaan DJ, Berlivet S, Hunninghake GM, et al. Allele-specific chromatin remodeling in the ZPBP2/GSDMB/ORMDL3 locus associated with the risk of asthma and autoimmune disease. Am J Hum Genet 2009; 85:377.</a></li><li><a class="nounderline abstract_t">Moll M, Sakornsakolpat P, Shrine N, et al. Chronic obstructive pulmonary disease and related phenotypes: polygenic risk scores in population-based and case-control cohorts. Lancet Respir Med 2020; 8:696.</a></li><li><a class="nounderline abstract_t">Hunninghake GM, Cho MH, Tesfaigzi Y, et al. MMP12, lung function, and COPD in high-risk populations. N Engl J Med 2009; 361:2599.</a></li><li><a class="nounderline abstract_t">Jaiswal S, Fontanillas P, Flannick J, et al. Age-related clonal hematopoiesis associated with adverse outcomes. N Engl J Med 2014; 371:2488.</a></li><li><a class="nounderline abstract_t">Bick AG, Weinstock JS, Nandakumar SK, et al. Inherited causes of clonal haematopoiesis in 97,691 whole genomes. Nature 2020; 586:763.</a></li><li><a class="nounderline abstract_t">Xing C, Garcia CK. Epigenetic inheritance of telomere length obscures identification of causative PARN locus. J Med Genet 2016; 53:356.</a></li><li><a class="nounderline abstract_t">Khera AV, Chaffin M, Aragam KG, et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. Nat Genet 2018; 50:1219.</a></li><li><a class="nounderline abstract_t">Polygenic Risk Score Task Force of the International Common Disease Alliance. Responsible use of polygenic risk scores in the clinic: potential benefits, risks and gaps. Nat Med 2021; 27:1876.</a></li><li><a class="nounderline abstract_t">Goldstein DB. Common genetic variation and human traits. N Engl J Med 2009; 360:1696.</a></li><li><a class="nounderline abstract_t">Hirschhorn JN. Genomewide association studies--illuminating biologic pathways. N Engl J Med 2009; 360:1699.</a></li><li><a class="nounderline abstract_t">DOLL R. Mortality from lung cancer in asbestos workers. Br J Ind Med 1955; 12:81.</a></li><li><a class="nounderline abstract_t">Reich DE, Lander ES. On the allelic spectrum of human disease. Trends Genet 2001; 17:502.</a></li><li><a class="nounderline abstract_t">Cargill M, Daley GQ. Mining for SNPs: putting the common variants--common disease hypothesis to the test. Pharmacogenomics 2000; 1:27.</a></li><li><a class="nounderline abstract_t">Buniello A, MacArthur JAL, Cerezo M, et al. The NHGRI-EBI GWAS Catalog of published genome-wide association studies, targeted arrays and summary statistics 2019. Nucleic Acids Res 2019; 47:D1005.</a></li><li><a class="nounderline abstract_t">Dickson SP, Wang K, Krantz I, et al. Rare variants create synthetic genome-wide associations. PLoS Biol 2010; 8:e1000294.</a></li><li><a class="nounderline abstract_t">Carmichael CM, McGue M. A cross-sectional examination of height, weight, and body mass index in adult twins. J Gerontol A Biol Sci Med Sci 1995; 50:B237.</a></li><li><a class="nounderline abstract_t">Silventoinen K, Sammalisto S, Perola M, et al. Heritability of adult body height: a comparative study of twin cohorts in eight countries. Twin Res 2003; 6:399.</a></li><li><a class="nounderline abstract_t">Weedon MN, Lango H, Lindgren CM, et al. Genome-wide association analysis identifies 20 loci that influence adult height. Nat Genet 2008; 40:575.</a></li><li><a class="nounderline abstract_t">Meigs JB, Shrader P, Sullivan LM, et al. Genotype score in addition to common risk factors for prediction of type 2 diabetes. N Engl J Med 2008; 359:2208.</a></li><li><a class="nounderline abstract_t">Paynter NP, Chasman DI, Buring JE, et al. Cardiovascular disease risk prediction with and without knowledge of genetic variation at chromosome 9p21.3. Ann Intern Med 2009; 150:65.</a></li><li><a class="nounderline abstract_t">Kraft P, Hunter DJ. Genetic risk prediction--are we there yet? N Engl J Med 2009; 360:1701.</a></li><li><a class="nounderline abstract_t">Forno E, Wang T, Qi C, et al. DNA methylation in nasal epithelium, atopy, and atopic asthma in children: a genome-wide study. Lancet Respir Med 2019; 7:336.</a></li><li><a class="nounderline abstract_t">Bell GI, Polonsky KS. Diabetes mellitus and genetically programmed defects in beta-cell function. Nature 2001; 414:788.</a></li><li><a class="nounderline abstract_t">Strittmatter WJ, Saunders AM, Schmechel D, et al. Apolipoprotein E: high-avidity binding to beta-amyloid and increased frequency of type 4 allele in late-onset familial Alzheimer disease. Proc Natl Acad Sci U S A 1993; 90:1977.</a></li><li><a class="nounderline abstract_t">International HapMap Consortium. A haplotype map of the human genome. Nature 2005; 437:1299.</a></li><li><a class="nounderline abstract_t">Wang DG, Fan JB, Siao CJ, et al. Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome. Science 1998; 280:1077.</a></li><li><a class="nounderline abstract_t">Welter D, MacArthur J, Morales J, et al. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res 2014; 42:D1001.</a></li><li><a class="nounderline abstract_t">Kathiresan S, Melander O, Anevski D, et al. Polymorphisms associated with cholesterol and risk of cardiovascular events. N Engl J Med 2008; 358:1240.</a></li><li><a class="nounderline abstract_t">Duerr RH, Taylor KD, Brant SR, et al. A genome-wide association study identifies IL23R as an inflammatory bowel disease gene. Science 2006; 314:1461.</a></li><li><a class="nounderline abstract_t">Gudmundsson J, Sulem P, Steinthorsdottir V, et al. Two variants on chromosome 17 confer prostate cancer risk, and the one in TCF2 protects against type 2 diabetes. Nat Genet 2007; 39:977.</a></li><li><a class="nounderline abstract_t">Yasuda K, Miyake K, Horikawa Y, et al. Variants in KCNQ1 are associated with susceptibility to type 2 diabetes mellitus. Nat Genet 2008; 40:1092.</a></li><li><a class="nounderline abstract_t">Tennessen JA, Bigham AW, O'Connor TD, et al. Evolution and functional impact of rare coding variation from deep sequencing of human exomes. Science 2012; 337:64.</a></li><li><a class="nounderline abstract_t">Panoutsopoulou K, Tachmazidou I, Zeggini E. In search of low-frequency and rare variants affecting complex traits. Hum Mol Genet 2013; 22:R16.</a></li></ol></div><div id="topicVersionRevision">Topic 14600 Version 23.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19242412" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Identification of IFRD1 as a modifier gene for cystic fibrosis lung disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17634447" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : A genetic risk factor for periodic limb movements in sleep.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17554300" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30528432" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Unveiling the Role of the Most Impactful Cardiovascular Risk Locus through Haplotype Editing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21307941" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : 9p21 DNA variants associated with coronary artery disease impair interferon-γsignalling response.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19732864" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Allele-specific chromatin remodeling in the ZPBP2/GSDMB/ORMDL3 locus associated with the risk of asthma and autoimmune disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32649918" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Chronic obstructive pulmonary disease and related phenotypes: polygenic risk scores in population-based and case-control cohorts.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20018959" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : MMP12, lung function, and COPD in high-risk populations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25426837" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Age-related clonal hematopoiesis associated with adverse outcomes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/33057201" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Inherited causes of clonal haematopoiesis in 97,691 whole genomes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26908837" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Epigenetic inheritance of telomere length obscures identification of causative PARN locus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30104762" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34782789" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Responsible use of polygenic risk scores in the clinic: potential benefits, risks and gaps.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19369660" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Common genetic variation and human traits.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19369661" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Genomewide association studies--illuminating biologic pathways.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14363586" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Mortality from lung cancer in asbestos workers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11525833" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : On the allelic spectrum of human disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11258594" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Mining for SNPs: putting the common variants--common disease hypothesis to the test.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30445434" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : The NHGRI-EBI GWAS Catalog of published genome-wide association studies, targeted arrays and summary statistics 2019.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20126254" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Rare variants create synthetic genome-wide associations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7614237" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : A cross-sectional examination of height, weight, and body mass index in adult twins.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14624724" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Heritability of adult body height: a comparative study of twin cohorts in eight countries.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18391952" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Genome-wide association analysis identifies 20 loci that influence adult height.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19020323" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Genotype score in addition to common risk factors for prediction of type 2 diabetes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19153409" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Cardiovascular disease risk prediction with and without knowledge of genetic variation at chromosome 9p21.3.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19369656" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Genetic risk prediction--are we there yet?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30584054" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : DNA methylation in nasal epithelium, atopy, and atopic asthma in children: a genome-wide study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11742410" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Diabetes mellitus and genetically programmed defects in beta-cell function.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8446617" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Apolipoprotein E: high-avidity binding to beta-amyloid and increased frequency of type 4 allele in late-onset familial Alzheimer disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16255080" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : A haplotype map of the human genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9582121" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24316577" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : The NHGRI GWAS Catalog, a curated resource of SNP-trait associations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18354102" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Polymorphisms associated with cholesterol and risk of cardiovascular events.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17068223" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : A genome-wide association study identifies IL23R as an inflammatory bowel disease gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17603485" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Two variants on chromosome 17 confer prostate cancer risk, and the one in TCF2 protects against type 2 diabetes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18711367" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Variants in KCNQ1 are associated with susceptibility to type 2 diabetes mellitus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22604720" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Evolution and functional impact of rare coding variation from deep sequencing of human exomes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23922232" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : In search of low-frequency and rare variants affecting complex traits.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
