<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Genomic disorders: An overview
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Genomic disorders: An overview
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Genomic disorders: An overview
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Author:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Carlos A Bacino, MD, FACMG
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editors:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Helen V Firth, DM, FRCP, FMedSci
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Benjamin A Raby, MD, MPH
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Jennifer S Tirnauer, MD
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            Dec 14, 2023.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H16722192">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genomic disorders are diseases that result from the loss or gain of chromosomal/DNA material (copy number variations [CNVs]). There are a number of well-delineated genomic disorders that can be divided in two categories: those resulting from copy number losses (deletion syndromes) and copy number gains (duplication syndromes).
        </p>
        <p>
         An overview of genomic disorders is presented here. Specific syndromic disorders are reviewed separately, as indicated in the sections below.
        </p>
        <p>
         Separate topics also discuss:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Chromosomal abnormalities – (See
         <a class="medical medical_review" href="/z/d/html/4490.html" rel="external">
          "Chromosomal translocations, deletions, and inversions"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Cytogenetics – (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         DNA sequencing – (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Definitions of terms – (See
         <a class="medical medical_review" href="/z/d/html/2898.html" rel="external">
          "Genetics: Glossary of terms"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H16722199">
         <span class="h1">
          COPY NUMBER VARIATIONS
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Structural variation
         </strong>
         – Structural genetic variation refers to a class of sequence alterations spanning typically more than 1000 base pairs (one kilobase of DNA or kb) [
         <a href="#rid1">
          1
         </a>
         ]. However, studies have indicated that these genetic variations can be smaller in the range of up to 450 base pairs and that most individuals have at least 1000 such variations [
         <a href="#rid2">
          2
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         These structural genetic variations include:
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         Quantitative variants such as copy number variations (CNVs)
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         Sequence rearrangements (such as those observed among immunoglobulins)
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         Other less-common variations, including chromosomal rearrangements that may or may not alter the genome contents but may disrupt gene function and result in disease
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          CNVs
         </strong>
         – CNVs are the most prevalent type of structural variation. They are DNA segments spanning thousands to millions of bases whose copy number varies among different healthy individuals [
         <a href="#rid3">
          3,4
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         It is estimated that 50 percent of the genome is constituted by repeated sequences [
         <a href="#rid5">
          5
         </a>
         ]. These CNVs can be recurrent when mediated by regions where low copy repeats share more than 90 percent identity or where short-segmental duplications exceed 1 kilobase in size. Other CNVs can be nonrecurrent and associated with DNA blunt ends, micro-homologies, or even insertions that can lead to more complex rearrangements [
         <a href="#rid6">
          6
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         These submicroscopic genomic differences in the number of copies of one or more sections of DNA are the result of DNA gains or losses.
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Gains
         </strong>
         – Copy number gains can be the result of duplications, triplications, or even multiple copy number gains.
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Losses
         </strong>
         – Most deletions are the result of copy number loss at one locus (heterozygous deletions), but in some instances the loss can affect copies at both loci (homozygous deletions).
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Transmission
         </strong>
         – CNVs are commonly inherited but can occur de novo (as a new event). These were initially considered rare events resulting from sporadic mutation(s) and correlated with specific Mendelian diseases [
         <a href="#rid7">
          7,8
         </a>
         ]. These misperceptions about their rarity and absolute disease linkage were primarily due to technical limitations precluding genome-wide assessments in large cohorts. Advances in technology have shown that deviation from the diploid state is widespread and contributes substantially to genetic diversity.
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         Some studies have suggested that CNV differences in the human genome are as extensive as 20 percent, although this may be an overestimation [
         <a href="#rid9">
          9,10
         </a>
         ] and probably closer to 5 percent. It is estimated conservatively that most individuals carry an average of three large-scale CNVs [
         <a href="#rid4">
          4
         </a>
         ]. The number of known CNVs that contribute to disease pathogenesis continues to increase.
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Hotspots
         </strong>
         – The physical distribution of CNVs appears to be nonrandom across the genome, with both CNV hot and cold spots reported [
         <a href="#rid9">
          9,11
         </a>
         ]. CNV frequency is greatest in regions of segmental duplication (a 4- to 10-fold enrichment for CNVs), consistent with nonallelic homologous recombination as a primary mechanism for the causation of CNVs [
         <a href="#rid3">
          3,12-15
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         CNVs are more commonly observed in gene-rich regions. CNVs appear to be enriched in specific gene families, including immune and inflammatory response genes, cell signaling and cell adhesion molecules, structural proteins, and olfactory receptors [
         <a href="#rid9">
          9
         </a>
         ]. Most of these differences probably represent benign CNVs that reflect normal variation with no apparent clinical consequence [
         <a href="#rid16">
          16
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Pathogenicity
         </strong>
         – CNVs can be pathogenic if they involve dose-sensitive genes (haplo-insufficient or triplosensitive) or if they influence or disrupt genomic regions through regulatory elements [
         <a href="#rid17">
          17,18
         </a>
         ]. Gene dosage sensitivity information can be interrogated in curated databases such as ClinGen (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fsearch.clinicalgenome.org%2Fkb%2Fgene-dosage%3Fpage%3D1%26size%3D25%26search%3D%29&amp;token=huArPlKmeyU9jfruBDESZpzVGXUEGhU8SnP3aiZDG3smKYtC7PtbViZbKWXPA9UDSCV0MEYxrqx8v7eUTDhPF0%2FZ4bJDSSfuMvdkwki%2B2E0%3D&amp;TOPIC_ID=16647" target="_blank">
          ClinGen Dosage sensitivity page
         </a>
         ) [
         <a href="#rid19">
          19
         </a>
         ]. Some pathogenic CNVs cause syndromic disorders with consistent phenotypic features (eg, deletions of the elastin gene in Williams syndrome, duplications of the
         <em>
          PMP22
         </em>
         gene in Charcot-Marie-Tooth disease type 1A [CMT1A]), while others are associated with disease susceptibility or resistance (eg, cancer, HIV infection, autoimmune disorders, autism).
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H4084847380">
         <span class="h2">
          Mendelian disease associations
         </span>
         <span class="headingEndMark">
          —
         </span>
         CNVs can be responsible for Mendelian diseases associated with gains and losses of genetic material, even at the exonic level. Examples include the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Contiguous gene deletions or duplications as seen in Williams-Beuren syndrome, 22q11 deletion-syndrome, Smith-Magenis syndrome, and Potocki-Lupski syndrome. These are examples of recurrent CNVs and mediated by nonallelic homologous recombination (NAHR) occurring at sites of low-copy repeats. (See
         <a class="medical medical_review" href="/z/d/html/6127.html" rel="external">
          "Williams syndrome"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/16648.html" rel="external">
          "Microdeletion syndromes (chromosomes 12 to 22)", section on '17p11.2 deletion syndrome (Smith-Magenis syndrome)'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/13557.html" rel="external">
          "Microduplication syndromes", section on '17p11.2 duplication syndrome (Potocki-Lupski syndrome)'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Deletions of genes or portions of genes (exons) leading to Mendelian-inherited genetic disorders, including disorders that are autosomal dominant (eg, deletions of the
         <em>
          CREBBP
         </em>
         gene in Rubinstein-Taybi syndrome) and X-linked recessive (eg, deletions of the dystrophin gene in Duchenne muscular dystrophy) [
         <a href="#rid20">
          20
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/16648.html" rel="external">
          "Microdeletion syndromes (chromosomes 12 to 22)", section on '16p13.3 deletion syndrome (Rubinstein-Taybi syndrome)'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/116654.html" rel="external">
          "Duchenne and Becker muscular dystrophy: Genetics and pathogenesis", section on 'Genetics and pathogenesis'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H1693419053">
         <span class="h2">
          Associations with complex traits
         </span>
         <span class="headingEndMark">
          —
         </span>
         CNVs can also be associated with more complex traits or syndromes in which there are combinations of genetic and environmental factors [
         <a href="#rid2">
          2
         </a>
         ]. Examples include a number of disorders associated with auto-inflammatory disease. Some of these CNVs and their association with disease have been replicated by other studies but not always found to be significant and perhaps may be dependent of the population studied [
         <a href="#rid21">
          21
         </a>
         ].
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         A deletion at the complement component 4 (C4) locus that confers a 1.6- to 5.3-fold risk for systemic lupus erythematosus [
         <a href="#rid22">
          22
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4669.html" rel="external">
          "Epidemiology and pathogenesis of systemic lupus erythematosus"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Deletions in
         <em>
          FCGR3B
         </em>
         associated with granulomatosis with polyangiitis [
         <a href="#rid23">
          23
         </a>
         ] and other autoimmune/inflammatory conditions such as rheumatoid arthritis [
         <a href="#rid24">
          24,25
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/3074.html" rel="external">
          "Pathogenesis of antineutrophil cytoplasmic autoantibody-associated vasculitis"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         A deletion of defensin-beta 4 (
         <em>
          DEFB4
         </em>
         ) associated with increased risk of colonic Crohn disease [
         <a href="#rid26">
          26
         </a>
         ]. A deletion of 20 kb upstream of the
         <em>
          IRGM
         </em>
         gene has also been associated with Crohn disease and psoriasis [
         <a href="#rid21">
          21,27
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4062.html" rel="external">
          "Genetic factors in inflammatory bowel disease"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         A 32 kb deletion involving two genes that encode envelope proteins (
         <em>
          LCE3B
         </em>
         and
         <em>
          LCE3C
         </em>
         ) has been associated as a susceptibility factor for psoriasis [
         <a href="#rid28">
          28
         </a>
         ].
         <em>
          DEFB4
         </em>
         deletions have also been associated with psoriatic arthritis [
         <a href="#rid29">
          29
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Another example of complex disorders includes the increased frequency of de novo germ-line CNVs in patients with autism spectrum disorder (ASD) and schizophrenia [
         <a href="#rid30">
          30,31
         </a>
         ]. CNVs in multiple areas of the human genome potentially involved in autism pathogenesis have been described. (See
         <a class="medical medical_review" href="/z/d/html/595.html" rel="external">
          "Autism spectrum disorder (ASD) in children and adolescents: Terminology, epidemiology, and pathogenesis", section on 'Genetic factors'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         This line of research has also identified specific genes and pathways involved in ASD and related syndromes (eg, duplications of the 15q11-q13 region, deletions and duplications of 16p11.2, 1q21 duplications) [
         <a href="#rid32">
          32-34
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/16660.html" rel="external">
          "Microdeletion syndromes (chromosomes 1 to 11)"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/16648.html" rel="external">
          "Microdeletion syndromes (chromosomes 12 to 22)"
         </a>
         .)
        </p>
        <p>
        </p>
        <p>
         In addition, some conditions are associated with multiple CNVs, which may explain their variable phenotypes [
         <a href="#rid35">
          35
         </a>
         ]. As an example, in a retrospective study, array comparative genomic hybridization (CGH) was used to evaluate CNVs in 2312 children with developmental disabilities who already had one predefined CNV and in 8329 children without developmental disabilities [
         <a href="#rid36">
          36
         </a>
         ]. This study found that, in comparison to controls without developmental disabilities, individuals with developmental disabilities had an increased number of additional CNVs. This increase in CNVs may have played a causative role in the disabilities (eg, by causing disruption of a new gene or altering gene dosage), or may be an indirect marker of susceptibility to genomic damage. (See
         <a class="local">
          'Array comparative genomic hybridization'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H12440911">
         <span class="h2">
          Causes of CNVs
         </span>
         <span class="headingEndMark">
          —
         </span>
         Low copy repeats are stretches of repetitive DNA sequences (segmental duplications) approximately 10 to 300 kilobases in size that share ≥95 percent homology [
         <a href="#rid21">
          21
         </a>
         ]. Erroneous pairing of these highly homologous regions can cause misalignment and unequal recombination during meiosis. This can lead to duplication and deletion of chromosomal material resulting in CNVs. This process is known as nonallelic homologous recombination (NAHR)  (
         <a class="graphic graphic_figure graphicRef67644" href="/z/d/graphic/67644.html" rel="external">
          figure 1
         </a>
         ), the most common mechanism for the formation of genomic rearrangements [
         <a href="#rid37">
          37,38
         </a>
         ].
        </p>
        <p>
         NAHR can result in either deletions or duplication via the same mechanisms and due to low copy repeats mediated nonallelic homologous recombination. A classic example is the case of Charcot-Marie-Tooth Type I, a peripheral neuropathy caused by duplications of the PMP22 gene on chromosome 17p11.2. The same region when deleted leads to a different neuropathy known as Tomaculous neuropathy also known as hereditary neuropathy with liability to pressure palsies (HNPP). (See
         <a class="medical medical_review" href="/z/d/html/13557.html" rel="external">
          "Microduplication syndromes"
         </a>
         .)
        </p>
        <p>
         Other mechanisms leading to CNVs include nonhomologous end joining and microhomology-mediated break-induced replication, although discussion of these is beyond the scope of this chapter [
         <a href="#rid39">
          39
         </a>
         ].
        </p>
        <p>
         Despite a great deal of knowledge about the structural details of how CNVs occur, we do not know what predisposes certain individuals to develop these changes more than other individuals.
        </p>
        <p>
         Of interest, in a large study of patients with developmental disabilities, parental data provided information about whether CNVs were inherited or arose de novo [
         <a href="#rid36">
          36
         </a>
         ]. This study suggested that CNVs were more likely to arise de novo in the syndromic disorders (eg, Williams-Beuren syndrome), whereas CNVs were more likely to be inherited in the disorders with variable phenotype (eg, intellectual disability). A potential explanation may be that reproductive fitness is reduced in individuals with the more severe syndromic disorders. (See
         <a class="medical medical_review" href="/z/d/html/16660.html" rel="external">
          "Microdeletion syndromes (chromosomes 1 to 11)", section on '7q11.23 deletion syndrome (Williams syndrome)'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H12440918">
         <span class="h2">
          Interpreting CNVs
         </span>
         <span class="headingEndMark">
          —
         </span>
         The interpretation of CNVs has steadily improved due to use of large control databases that allow a direct comparison with apparently normal controls. Examples of these databases include the
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fdgv.tcag.ca%2Fdgv%2Fapp%2Fhome&amp;token=ZWwxzs9Qp%2FKuLWPOpqxKWBvxEZ9Fd%2FUNMTRzPXEfWVUohGXikFKQHoSXa2YrI39v&amp;TOPIC_ID=16647" target="_blank">
          Database of Genomic Variants
         </a>
         (DGV), large sequencing projects such as the
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fwww.internationalgenome.org%2F&amp;token=lmvGwDg43jbWTPc09l4hMGllUvRdVTvC18G4g6UyU6Htrk8rkwumt%2B55Atifj2nc&amp;TOPIC_ID=16647" target="_blank">
          1000 Genomes Project
         </a>
         , or
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fclinvar%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2Fy3%2BLj0shalBW%2BL1ZyL4trzGy12ipcVcj41059JRLkL3&amp;TOPIC_ID=16647" target="_blank">
          ClinVar
         </a>
         [
         <a href="#rid40">
          40-42
         </a>
         ]. Other databases that include phenotypic information, such as
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fdecipher.sanger.ac.uk%2F&amp;token=S4K3ySMKAR5ih4ZompinzIyNEaF8%2Bbf%2BqNX5W3tolf7P%2F%2FJZerg3tg2C5xbRFuvj&amp;TOPIC_ID=16647" target="_blank">
          DECIPHER
         </a>
         (
         <strong>
          D
         </strong>
         atabas
         <strong>
          E
         </strong>
         of
         <strong>
          C
         </strong>
         hromosomal
         <strong>
          I
         </strong>
         mbalance and
         <strong>
          P
         </strong>
         henotype in
         <strong>
          H
         </strong>
         umans using
         <strong>
          E
         </strong>
         nsembl
         <strong>
          R
         </strong>
         esources), are becoming valuable resources for findings detected by array-comparative genomic hybridization [
         <a href="#rid43">
          43
         </a>
         ]. DECIPHER is a web-based resource that contains array and clinical data deposited by over 247 member centers around the world. The Deciphering Developmental Disorders study performed in the United Kingdom has evaluated 14,000 children with severe developmental delay and their parents with microarray and exome sequencing [
         <a href="#rid44">
          44
         </a>
         ]. These large-scale studies are extremely valuable tools for the interpretation of CNV data. Another popular tool available for curation of CNVs and their gene content is
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fdosage.clinicalgenome.org%2Findex.shtml&amp;token=MImdFgeuXoMpkOhueTaaeWJtObzpYDSMe%2BGcOltNXFT3wJCChSh4A6MMWR9GayAGqlqCnZRTU1Eiosb5iTVhSg%3D%3D&amp;TOPIC_ID=16647" target="_blank">
          ClinGen
         </a>
         . This is an open clinical genome resource that establishes proper annotation of CNVs based on information assessed from multiple tiered resources and curated by experts [
         <a href="#rid45">
          45
         </a>
         ].
        </p>
        <p>
         Interpreting the pathogenicity of CNVs can be quite challenging in the presence of multiple CNVs in one individual. Some studies have shown that multiple rare CNVs, either inherited or de novo, can compound the clinical severity [
         <a href="#rid36">
          36,46
         </a>
         ]. It is also important to highlight the presence of common CNVs in the general population. It is unclear whether many of these common polymorphisms may play a role in common disorders [
         <a href="#rid47">
          47
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H16722227">
         <span class="h1">
          DISEASE MECHANISMS
         </span>
         <span class="headingEndMark">
          —
         </span>
         There are different potential mechanisms that can lead to disease in genomic disorders secondary to deletions and duplications. The main mechanism is related to changes in dosage sensitive genes. "Haploinsufficiency" ("haplo" means half) defines the concept where loss or gain of one allele of a gene may lead to abnormal protein production or function, thereby causing disease [
         <a href="#rid48">
          48
         </a>
         ].
        </p>
        <p>
         Deletions can interfere with the required amount of a protein dose, resulting in disease. One example is Williams-Beuren Syndrome, which is caused by a microdeletion in chromosome 7q11.23 involving multiple genes, including the elastin gene [
         <a href="#rid49">
          49,50
         </a>
         ]. Having only one-half of the normal dose of elastin is enough to cause disruption of the arterial elastic fibers, leading to aortic narrowing and multiple other arterial abnormalities. The deletions in Williams-Beuren syndrome, similar to what happens in many other genomic disorders, can be of different sizes. The loss typically encompasses 1.55 Mb, but in some cases the deletion can be more extensive or even smaller. The size difference is due to the different LCRs around the critical region that can be involved in mediating the rearrangements. (See
         <a class="medical medical_review" href="/z/d/html/4490.html" rel="external">
          "Chromosomal translocations, deletions, and inversions", section on 'Deletions'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/16660.html" rel="external">
          "Microdeletion syndromes (chromosomes 1 to 11)", section on '7q11.23 deletion syndrome (Williams syndrome)'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/5775.html" rel="external">
          "Valvar aortic stenosis in children"
         </a>
         .)
        </p>
        <p>
         The importance of gene dosage in determining the effect of CNVs was illustrated in the sex differences from a large study of CNVs in patients with developmental disabilities [
         <a href="#rid36">
          36
         </a>
         ]. When compared with females, males had more of the "variable phenotype genomic disorders" (eg, intellectual disability) but not syndromic disorders (eg, autism spectrum disorders). Females may be protected from these more genetically multifactorial disorders due to sex chromosome bias (ie, the protection of females from weakly deleterious mutations on one X chromosome by the normal corresponding genes on the other X chromosome).
        </p>
        <p>
         Duplications may cause disease if the genes involved in the duplications are triplosensitive or if the copy number gain disrupts the reading frame of a gene, leading to abnormal protein synthesis. Another mechanism includes unmasking of recessive alleles [
         <a href="#rid51">
          51-53
         </a>
         ]. When a recessive allele is deleted, it could potentially uncover a pathogenic variant in the remaining copy (gene) on the other chromosome. This would lead to disease because there is no normally functioning copy left for the affected gene.
        </p>
        <p>
         Other disease mechanisms include the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Interference with regulatory elements outside of genes, as in brachydactyly type A2 and duplications outside the
         <em>
          BMP2
         </em>
         gene [
         <a href="#rid54">
          54
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Interference with imprinted genes, as in the case of paternal duplications of 11p15 that lead to Beckwith-Wiedemann syndrome [
         <a href="#rid55">
          55
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/13555.html" rel="external">
          "Beckwith-Wiedemann syndrome", section on 'Genetics and pathogenesis'
         </a>
         .)
        </p>
        <p>
        </p>
        <p>
         Arrays that contain single nucleotide polymorphisms (SNPs) can further aid in the identification of imprinting disorders caused by uniparental disomy. One such example is Angelman syndrome and uniparental disomy (UPD) caused by isodisomy. Isodisomies result from either nondisjunction in meiosis II or by postzygotic duplication (monosomy rescue). (See
         <a class="medical medical_review" href="/z/d/html/129424.html" rel="external">
          "Basic genetics concepts: Chromosomes and cell division", section on 'Meiosis'
         </a>
         .)
        </p>
        <p>
         A small number of Angelman syndrome cases are the result of UPD. In those cases there is absence of the maternal contribution for a region of chromosome 15 (15q11-q13). These cases are typically associated with monosomy rescue (duplication of a chromosome from a monosomy 15 zygote), where there are two identical copies of the paternal chromosome 15 and no maternal contribution. Other examples of UPD can be seen in Prader-Willi syndrome. (See
         <a class="medical medical_review" href="/z/d/html/5875.html" rel="external">
          "Prader-Willi syndrome: Clinical features and diagnosis"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H174214015">
         <span class="h2">
          Contiguous gene syndromes
         </span>
         <span class="headingEndMark">
          —
         </span>
         Contiguous gene syndromes can occur when large CNVs affect several contiguous genes [
         <a href="#rid56">
          56,57
         </a>
         ].
        </p>
        <p>
         As examples:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Williams-Beuren Syndrome, also called Williams syndrome, is caused by a 1.5-1.8 Mb deletion on chromosome 7q11 that typically encompasses 26 to 28 genes. (See
         <a class="medical medical_review" href="/z/d/html/6127.html" rel="external">
          "Williams syndrome", section on 'Genetics'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         In WAGR syndrome (
         <strong>
          W
         </strong>
         ilms tumor,
         <strong>
          A
         </strong>
         niridia,
         <strong>
          G
         </strong>
         enitourinary anomalies, and mental
         <strong>
          R
         </strong>
         etardation), clinical features are attributable to the loss of individual genes by a large deletion: deletions of
         <em>
          WT1
         </em>
         are responsible for Wilms tumor, while
         <em>
          PAX6
         </em>
         deletions are responsible for the aniridia findings. Both genes are contiguously located within the short arm of chromosome 11. (See
         <a class="medical medical_review" href="/z/d/html/16660.html" rel="external">
          "Microdeletion syndromes (chromosomes 1 to 11)", section on '11p13 deletion syndrome (WAGR syndrome)'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H16722234">
         <span class="h1">
          DETECTION OF GENOMIC DISORDERS
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genomic disorders are typically investigated and detected by array comparative genomic hybridization (array CGH). Gains or losses detected on an array can be confirmed by an independent method such as fluorescence in situ hybridization (FISH), multiple ligation dependent probe amplification (MLPA), or quantitative PCR (Q-PCR). (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2896.html" rel="external">
          "Polymerase chain reaction (PCR)"
         </a>
         .)
        </p>
        <p>
         Parental FISH or array CGH testing may be considered in cases of selected genomic abnormalities that can be inherited or to interpret CNVs that are variants of uncertain significance and aid with the clinical interpretation. This is relevant for future pregnancies and family planning [
         <a href="#rid58">
          58
         </a>
         ]. Whereas the genomic position of a loss is clear by array testing, gains may be either tandem duplications or insertions. If the latter arise as a consequence of a parental insertional translocation (IT), this may have important implications for future pregnancies [
         <a href="#rid59">
          59
         </a>
         ]. In this case, FISH can be used to determine the nature and localization of the copy number gain.
        </p>
        <p>
         Next generation sequencing (NGS) technology used for exome sequencing is also suitable to identify CNVs but not widely used for this purpose. Whole exome sequencing (WES) can detect CNVs and sequence variations simultaneously [
         <a href="#rid60">
          60,61
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3390507">
         <span class="h2">
          Array comparative genomic hybridization
         </span>
         <span class="headingEndMark">
          —
         </span>
         Array comparative genomic hybridization (array CGH), also known as chromosome microarray or microarray-based comparative genomic hybridization, is the gold standard laboratory test for the detection of CNVs that cause genomic disorders  (
         <a class="graphic graphic_figure graphicRef58888" href="/z/d/graphic/58888.html" rel="external">
          figure 2
         </a>
         ). Array CGH allows detection of small losses or gains of genomic material down to several kilobases (kb) down to the exonic level. It is widely used in the evaluation of patients with intellectual disabilities and/or congenital malformations [
         <a href="#rid62">
          62-67
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics", section on 'Array comparative genomic hybridization'
         </a>
         .)
        </p>
        <p>
         The two main platforms currently used for CNV detection are oligonucleotide arrays (oligonucleotides are stretches of DNA ranging from 25 to 60 base pairs)  (
         <a class="graphic graphic_figure graphicRef58888" href="/z/d/graphic/58888.html" rel="external">
          figure 2
         </a>
         ), and single nucleotide polymorphism arrays (SNP arrays) [
         <a href="#rid68">
          68
         </a>
         ]. There are approximately 10 million polymorphic SNPs throughout the human genome. Both SNPs and oligonucleotide arrays can detect copy number variations, but SNP arrays can be used in addition to determine lack of heterozygosity or presence of homozygosity, as seen in cases of consanguinity [
         <a href="#rid69">
          69
         </a>
         ] and in cases of uniparental disomy [
         <a href="#rid70">
          70
         </a>
         ] when there is inheritance of regions or entire chromosomes from one single parent instead of the normal biparental contribution. SNPs can also detect loss of heterozygosity (LOH) typically seen in somatic cancer cell changes. SNPs can also be very useful for the detection of somatic mosaicism, a situation where two or more cell lines can be present in a single individual, and triploidies, a rare instance when there could be a total of 69 chromosomes (3 haploid sets) that can be detected in the prenatal setting. Some current platforms are combining the use of array CGH and SNPs integrated in one single platform.
        </p>
        <p class="headingAnchor" id="H1726088">
         <span class="h2">
          Other molecular diagnostic techniques
         </span>
         <span class="headingEndMark">
          —
         </span>
         Other molecular techniques used for the detection of genomic disorders include fluorescence in situ hybridization (FISH), PCR-based studies like Q-PCR (quantitative PCR), and MLPA (Multiple Ligation Dependent Probe Amplification). Protocols using sequencing of the whole genomes (WGS) or exome (WES, sequencing all regions of the genome that encode proteins) will likely become viable options for detecting CNV variations [
         <a href="#rid71">
          71,72
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          FISH
         </strong>
         – FISH uses larger stretches of DNA (approximately 50 to 200 kilobases of DNA) labeled with fluorescence reagents to target specific genome regions. The use of FISH requires knowledge of what specific area is being targeted and is dependent upon a clinical diagnosis [
         <a href="#rid73">
          73,74
         </a>
         ]. It is generally reserved for specific situations such as translocations, fusions, or amplifications. (See
         <a class="medical medical_review" href="/z/d/html/4482.html" rel="external">
          "General aspects of cytogenetic analysis in hematologic malignancies", section on 'Methods of detection'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          MLPA
         </strong>
         – MLPA uses a cocktail of multiple probes available in kits and targets specific chromosomes or disease regions. A single reaction allows simultaneous hybridization of multiple probes to multiple regions or even multiple exons within a gene.
        </p>
        <p>
        </p>
        <p>
         As the cost of microarrays and other molecular techniques is decreasing, FISH and MLPA are less frequently used.
        </p>
        <p>
         A copy number gain can be the result of a chromosome duplication immediately adjacent to the area of interest, a marker chromosome (an extra structurally abnormal chromosome), or the result of an insertion or translocation.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          WGS
         </strong>
         – Whole genome sequencing (WGS) is a more comprehensive tool than whole exome sequencing (WES) for detecting single nucleotide variants and for studying patients with genetic disorders. WGS is also a more efficient tool to assess the presence of CNVs than WES.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         As WGS cost lowers and analysis becomes more efficient, this technique may replace the use of chromosome microarray technology [
         <a href="#rid75">
          75,76
         </a>
         ]. The usefulness of this approach was demonstrated in a study that used sequencing from 849 individuals to identify areas of CNVs and their role in gene dosage [
         <a href="#rid71">
          71
         </a>
         ]. Genome sequencing has also been used to map breakpoints in the genome responsible for duplications of chromosomal regions [
         <a href="#rid77">
          77
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         Sequencing has the potential to provide improved resolution, but the sensitivity and specificity of sequencing for detecting CNVs is variable. WGS can allow simultaneous detection of single nucleotide and copy number variations, although the study cost is too high [
         <a href="#rid71">
          71,78
         </a>
         ]. While WGS can accurately achieve CNV detection, the use of microarrays continues to be the gold standard.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         Additional information about WGS and WES methods, collectively referred to as next-generation sequencing (NGS), is presented separately. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H1726355">
         <span class="h1">
          SUMMARY
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Genomic disorders
         </strong>
         – Genomic disorders are diseases that result from copy number variations (CNVs). The most common and better-delineated genomic disorders are divided in two main categories: those resulting from copy number losses (deletion syndromes) and copy number gains (duplication syndromes). (See
         <a class="local">
          'Introduction'
         </a>
         above and
         <a class="medical medical_review" href="/z/d/html/452.html" rel="external">
          "Congenital cytogenetic abnormalities"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Copy number variations
         </strong>
         – CNVs are generally submicroscopic genomic differences in the number of copies of one or more sections of DNA  (
         <a class="graphic graphic_figure graphicRef67644" href="/z/d/graphic/67644.html" rel="external">
          figure 1
         </a>
         ). Some pathogenic CNVs cause syndromic disorders with consistent phenotypic features (also known as genomic disorders). Other CNVs are associated with disease susceptibility or resistance. (See
         <a class="local">
          'Copy number variations'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          How they cause disease
         </strong>
         – The main mechanisms that leads to disease in genomic disorders are deletions (copy number losses) and duplications/triplications (copy number gains) in dosage-sensitive genes. Other disease mechanisms include interference with imprinted genes, unmasking of recessive disorders, or disruption of neighbor regulatory elements outside of genes. (See
         <a class="local">
          'Disease mechanisms'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Laboratory testing
         </strong>
         – Genomic disorders are typically detected by array comparative genomic hybridization (array CGH) also known as chromosome microarray. Next generation sequencing (NGS) is likely to become a popular way to detect CNVs, but chromosome microarray testing remains the standard of care. NGS may be done using whole exome sequencing (WES) or whole genome sequencing (WGS). Fluorescence in situ hybridization (FISH) is less commonly used, with a few exceptions such as in copy number gains and to rule out more complex chromosome abnormalities. (See
         <a class="local">
          'Detection of genomic disorders'
         </a>
         above.)
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li>
          <a class="nounderline abstract_t">
           Feuk L, Carson AR, Scherer SW. Structural variation in the human genome. Nat Rev Genet 2006; 7:85.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Conrad DF, Pinto D, Redon R, et al. Origins and functional impact of copy number variation in the human genome. Nature 2010; 464:704.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Redon R, Ishikawa S, Fitch KR, et al. Global variation in copy number in the human genome. Nature 2006; 444:444.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Tuzun E, Sharp AJ, Bailey JA, et al. Fine-scale structural variation of the human genome. Nat Genet 2005; 37:727.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lander ES, Linton LM, Birren B, et al. Initial sequencing and analysis of the human genome. Nature 2001; 409:860.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Harel T, Lupski JR. Genomic disorders 20 years on-mechanisms for clinical manifestations. Clin Genet 2018; 93:439.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ewart AK, Morris CA, Atkinson D, et al. Hemizygosity at the elastin locus in a developmental disorder, Williams syndrome. Nat Genet 1993; 5:11.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Magenis RE, Toth-Fejel S, Allen LJ, et al. Comparison of the 15q deletions in Prader-Willi and Angelman syndromes: specific regions, extent of deletions, parental origin, and clinical consequences. Am J Med Genet 1990; 35:333.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cooper GM, Nickerson DA, Eichler EE. Mutational and selective effects on copy-number variants in the human genome. Nat Genet 2007; 39:S22.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sharp AJ. Emerging themes and new challenges in defining the role of structural variation in human disease. Hum Mutat 2009; 30:135.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Perry GH, Tchinda J, McGrath SD, et al. Hotspots for copy number variation in chimpanzees and humans. Proc Natl Acad Sci U S A 2006; 103:8006.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sharp AJ, Cheng Z, Eichler EE. Structural variation of the human genome. Annu Rev Genomics Hum Genet 2006; 7:407.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sharp AJ, Locke DP, McGrath SD, et al. Segmental duplications and copy-number variation in the human genome. Am J Hum Genet 2005; 77:78.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sebat J, Lakshmi B, Troge J, et al. Large-scale copy number polymorphism in the human genome. Science 2004; 305:525.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wong KK, deLeeuw RJ, Dosanjh NS, et al. A comprehensive analysis of common copy-number variations in the human genome. Am J Hum Genet 2007; 80:91.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wellcome Trust Case Control Consortium, Craddock N, Hurles ME, et al. Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls. Nature 2010; 464:713.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Girirajan S, Eichler EE. Phenotypic variability and genetic susceptibility to genomic disorders. Hum Mol Genet 2010; 19:R176.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Fanciulli M, Petretto E, Aitman TJ. Gene copy number variation and common human disease. Clin Genet 2010; 77:201.
          </a>
         </li>
         <li class="breakAll">
          ClinGen Dosage Sensitivity. ClinGen. Available at: https://search.clinicalgenome.org/kb/gene-dosage?page=1&amp;size=25&amp;search=) (Accessed on December 06, 2023).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Boone PM, Bacino CA, Shaw CA, et al. Detection of clinically relevant exonic copy-number changes by array CGH. Hum Mutat 2010; 31:1326.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Carvalho CM, Zhang F, Lupski JR. Evolution in health and medicine Sackler colloquium: Genomic disorders: a window into human gene and genome evolution. Proc Natl Acad Sci U S A 2010; 107 Suppl 1:1765.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yang Y, Chung EK, Wu YL, et al. Gene copy-number variation and associated polymorphisms of complement component C4 in human systemic lupus erythematosus (SLE): low copy number is a risk factor for and high copy number is a protective factor against SLE susceptibility in European Americans. Am J Hum Genet 2007; 80:1037.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Fanciulli M, Norsworthy PJ, Petretto E, et al. FCGR3B copy number variation is associated with susceptibility to systemic, but not organ-specific, autoimmunity. Nat Genet 2007; 39:721.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rahbari R, Zuccherato LW, Tischler G, et al. Understanding the Genomic Structure of Copy-Number Variation of the Low-Affinity Fcγ Receptor Region Allows Confirmation of the Association of FCGR3B Deletion with Rheumatoid Arthritis. Hum Mutat 2017; 38:390.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Robinson JI, Carr IM, Cooper DL, et al. Confirmation of association of FCGR3B but not FCGR3A copy number with susceptibility to autoantibody positive rheumatoid arthritis. Hum Mutat 2012; 33:741.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Fellermann K, Stange DE, Schaeffeler E, et al. A chromosome 8 gene-cluster polymorphism with low human beta-defensin 2 gene copy number predisposes to Crohn disease of the colon. Am J Hum Genet 2006; 79:439.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           McCarroll SA, Huett A, Kuballa P, et al. Deletion polymorphism upstream of IRGM associated with altered IRGM expression and Crohn's disease. Nat Genet 2008; 40:1107.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           de Cid R, Riveira-Munoz E, Zeeuwen PL, et al. Deletion of the late cornified envelope LCE3B and LCE3C genes as a susceptibility factor for psoriasis. Nat Genet 2009; 41:211.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           O'Rielly DD, Rahman P. Genetics of psoriatic arthritis. Best Pract Res Clin Rheumatol 2014; 28:673.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sebat J, Lakshmi B, Malhotra D, et al. Strong association of de novo copy number mutations with autism. Science 2007; 316:445.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stefansson H, Rujescu D, Cichon S, et al. Large recurrent microdeletions associated with schizophrenia. Nature 2008; 455:232.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Girirajan S, Dennis MY, Baker C, et al. Refinement and discovery of new hotspots of copy-number variation associated with autism spectrum disorder. Am J Hum Genet 2013; 92:221.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Barber JC, Hall V, Maloney VK, et al. 16p11.2-p12.2 duplication syndrome; a genomic condition differentiated from euchromatic variation of 16p11.2. Eur J Hum Genet 2013; 21:182.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hogart A, Wu D, LaSalle JM, Schanen NC. The comorbidity of autism with the genomic disorders of chromosome 15q11.2-q13. Neurobiol Dis 2010; 38:181.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sanders SJ, Ercan-Sencicek AG, Hus V, et al. Multiple recurrent de novo CNVs, including duplications of the 7q11.23 Williams syndrome region, are strongly associated with autism. Neuron 2011; 70:863.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Girirajan S, Rosenfeld JA, Coe BP, et al. Phenotypic heterogeneity of genomic disorders and rare copy-number variants. N Engl J Med 2012; 367:1321.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stankiewicz P, Inoue K, Bi W, et al. Genomic disorders: genome architecture results in susceptibility to DNA rearrangements causing common human traits. Cold Spring Harb Symp Quant Biol 2003; 68:445.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stankiewicz P, Shaw CJ, Dapper JD, et al. Genome architecture catalyzes nonrecurrent chromosomal rearrangements. Am J Hum Genet 2003; 72:1101.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hastings PJ, Lupski JR, Rosenberg SM, Ira G. Mechanisms of change in gene copy number. Nat Rev Genet 2009; 10:551.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           1000 Genomes Project Consortium, Abecasis GR, Altshuler D, et al. A map of human genome variation from population-scale sequencing. Nature 2010; 467:1061.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zhang J, Feuk L, Duggan GE, et al. Development of bioinformatics resources for display and analysis of copy number and other structural variants in the human genome. Cytogenet Genome Res 2006; 115:205.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Landrum MJ, Lee JM, Riley GR, et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res 2014; 42:D980.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Firth HV, Richards SM, Bevan AP, et al. DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources. Am J Hum Genet 2009; 84:524.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wright CF, Fitzgerald TW, Jones WD, et al. Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data. Lancet 2015; 385:1305.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rehm HL, Berg JS, Brooks LD, et al. ClinGen--the Clinical Genome Resource. N Engl J Med 2015; 372:2235.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Girirajan S, Brkanac Z, Coe BP, et al. Relative burden of large CNVs on a range of neurodevelopmental phenotypes. PLoS Genet 2011; 7:e1002334.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gijsbers AC, Schoumans J, Ruivenkamp CA. Interpretation of array comparative genome hybridization data: a major challenge. Cytogenet Genome Res 2011; 135:222.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Seidman JG, Seidman C. Transcription factor haploinsufficiency: when half a loaf is not enough. J Clin Invest 2002; 109:451.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wu YQ, Sutton VR, Nickerson E, et al. Delineation of the common critical region in Williams syndrome and clinical correlation of growth, heart defects, ethnicity, and parental origin. Am J Med Genet 1998; 78:82.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Schubert C. The genomic basis of the Williams-Beuren syndrome. Cell Mol Life Sci 2009; 66:1178.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ropers HH. Genetics of intellectual disability. Curr Opin Genet Dev 2008; 18:241.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wong LJ, Dimmock D, Geraghty MT, et al. Utility of oligonucleotide array-based comparative genomic hybridization for detection of target gene deletions. Clin Chem 2008; 54:1141.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kurth I, Klopocki E, Stricker S, et al. Duplications of noncoding elements 5' of SOX9 are associated with brachydactyly-anonychia. Nat Genet 2009; 41:862.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Dathe K, Kjaer KW, Brehm A, et al. Duplications involving a conserved regulatory element downstream of BMP2 are associated with brachydactyly type A2. Am J Hum Genet 2009; 84:483.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Russo S, Finelli P, Recalcati MP, et al. Molecular and genomic characterisation of cryptic chromosomal alterations leading to paternal duplication of the 11p15.5 Beckwith-Wiedemann region. J Med Genet 2006; 43:e39.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ballabio A. Contiguous deletion syndromes. Curr Opin Genet Dev 1991; 1:25.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Punnett HH, Zakai EH. Old syndromes and new cytogenetics. Dev Med Child Neurol 1990; 32:824.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kang SH, Shaw C, Ou Z, et al. Insertional translocation detected using FISH confirmation of array-comparative genomic hybridization (aCGH) results. Am J Med Genet A 2010; 152A:1111.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Nowakowska BA, de Leeuw N, Ruivenkamp CA, et al. Parental insertional balanced translocations are an important cause of apparently de novo CNVs in patients with developmental anomalies. Eur J Hum Genet 2012; 20:166.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Chen Y, Zhao L, Wang Y, et al. SeqCNV: a novel method for identification of copy number variations in targeted next-generation sequencing data. BMC Bioinformatics 2017; 18:147.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gambin T, Akdemir ZC, Yuan B, et al. Homozygous and hemizygous CNV detection from exome sequencing data in a Mendelian disease cohort. Nucleic Acids Res 2017; 45:1633.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Bejjani BA, Theisen AP, Ballif BC, Shaffer LG. Array-based comparative genomic hybridization in clinical diagnosis. Expert Rev Mol Diagn 2005; 5:421.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cheung SW, Shaw CA, Yu W, et al. Development and validation of a CGH microarray for clinical cytogenetic diagnosis. Genet Med 2005; 7:422.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lu X, Shaw CA, Patel A, et al. Clinical implementation of chromosomal microarray analysis: summary of 2513 postnatal cases. PLoS One 2007; 2:e327.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lu XY, Phung MT, Shaw CA, et al. Genomic imbalances in neonates with birth defects: high detection rates by using chromosomal microarray analysis. Pediatrics 2008; 122:1310.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Miller DT, Adam MP, Aradhya S, et al. Consensus statement: chromosomal microarray is a first-tier clinical diagnostic test for individuals with developmental disabilities or congenital anomalies. Am J Hum Genet 2010; 86:749.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stankiewicz P, Beaudet AL. Use of array CGH in the evaluation of dysmorphology, malformations, developmental delay, and idiopathic mental retardation. Curr Opin Genet Dev 2007; 17:182.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cooper GM, Mefford HC. Detection of copy number variation using SNP genotyping. Methods Mol Biol 2011; 767:243.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Schaaf CP, Scott DA, Wiszniewska J, Beaudet AL. Identification of incestuous parental relationships by SNP-based DNA microarrays. Lancet 2011; 377:555.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Papenhausen P, Schwartz S, Risheg H, et al. UPD detection using homozygosity profiling with a SNP genotyping microarray. Am J Med Genet A 2011; 155A:757.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Handsaker RE, Van Doren V, Berman JR, et al. Large multiallelic copy number variations in humans. Nat Genet 2015; 47:296.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           de Ligt J, Boone PM, Pfundt R, et al. Detection of clinically relevant copy number variants with whole-exome sequencing. Hum Mutat 2013; 34:1439.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Shaffer LG, Kennedy GM, Spikes AS, Lupski JR. Diagnosis of CMT1A duplications and HNPP deletions by interphase FISH: implications for testing in the cytogenetics laboratory. Am J Med Genet 1997; 69:325.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Eijk-Van Os PG, Schouten JP. Multiplex Ligation-dependent Probe Amplification (MLPA®) for the detection of copy number variation in genomic sequences. Methods Mol Biol 2011; 688:97.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zhou B, Ho SS, Zhang X, et al. Whole-genome sequencing analysis of CNV using low-coverage and paired-end strategies is efficient and outperforms array-based CNV analysis. J Med Genet 2018; 55:735.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gross AM, Ajay SS, Rajan V, et al. Copy-number variants in clinical genome sequencing: deployment and interpretation for rare and undiagnosed disease. Genet Med 2019; 21:1121.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Newman S, Hermetz KE, Weckselblatt B, Rudd MK. Next-generation sequencing of duplication CNVs reveals that most are tandem and some create fusion genes at breakpoints. Am J Hum Genet 2015; 96:208.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Meienberg J, Bruggmann R, Oexle K, Matyas G. Clinical sequencing: is WGS the better WES? Hum Genet 2016; 135:359.
          </a>
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 16647 Version 21.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16418744" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : Structural variation in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19812545" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : Origins and functional impact of copy number variation in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17122850" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : Global variation in copy number in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15895083" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : Fine-scale structural variation of the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11237011" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : Initial sequencing and analysis of the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28950406" id="rid5" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          6 : Genomic disorders 20 years on-mechanisms for clinical manifestations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7693128" id="rid6" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          7 : Hemizygosity at the elastin locus in a developmental disorder, Williams syndrome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2309780" id="rid7" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          8 : Comparison of the 15q deletions in Prader-Willi and Angelman syndromes: specific regions, extent of deletions, parental origin, and clinical consequences.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17597777" id="rid8" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          9 : Mutational and selective effects on copy-number variants in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18837009" id="rid9" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          10 : Emerging themes and new challenges in defining the role of structural variation in human disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16702545" id="rid10" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          11 : Hotspots for copy number variation in chimpanzees and humans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16780417" id="rid11" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          12 : Structural variation of the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15918152" id="rid12" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          13 : Segmental duplications and copy-number variation in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15273396" id="rid13" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          14 : Large-scale copy number polymorphism in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17160897" id="rid14" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          15 : A comprehensive analysis of common copy-number variations in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20360734" id="rid15" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          16 : Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20807775" id="rid16" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          17 : Phenotypic variability and genetic susceptibility to genomic disorders.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20002459" id="rid17" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          18 : Gene copy number variation and common human disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20002459" id="rid18" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          19 : Gene copy number variation and common human disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20848651" id="rid19" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          20 : Detection of clinically relevant exonic copy-number changes by array CGH.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20080665" id="rid20" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          21 : Evolution in health and medicine Sackler colloquium: Genomic disorders: a window into human gene and genome evolution.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17503323" id="rid21" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          22 : Gene copy-number variation and associated polymorphisms of complement component C4 in human systemic lupus erythematosus (SLE): low copy number is a risk factor for and high copy number is a protective factor against SLE susceptibility in European Americans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17529978" id="rid22" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          23 : FCGR3B copy number variation is associated with susceptibility to systemic, but not organ-specific, autoimmunity.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27995740" id="rid23" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          24 : Understanding the Genomic Structure of Copy-Number Variation of the Low-Affinity FcγReceptor Region Allows Confirmation of the Association of FCGR3B Deletion with Rheumatoid Arthritis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22290871" id="rid24" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          25 : Confirmation of association of FCGR3B but not FCGR3A copy number with susceptibility to autoantibody positive rheumatoid arthritis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16909382" id="rid25" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          26 : A chromosome 8 gene-cluster polymorphism with low human beta-defensin 2 gene copy number predisposes to Crohn disease of the colon.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19165925" id="rid26" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          27 : Deletion polymorphism upstream of IRGM associated with altered IRGM expression and Crohn's disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19169253" id="rid27" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          28 : Deletion of the late cornified envelope LCE3B and LCE3C genes as a susceptibility factor for psoriasis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25488777" id="rid28" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          29 : Genetics of psoriatic arthritis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17363630" id="rid29" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          30 : Strong association of de novo copy number mutations with autism.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18668039" id="rid30" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          31 : Large recurrent microdeletions associated with schizophrenia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23375656" id="rid31" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          32 : Refinement and discovery of new hotspots of copy-number variation associated with autism spectrum disorder.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22828807" id="rid32" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          33 : 16p11.2-p12.2 duplication syndrome; a genomic condition differentiated from euchromatic variation of 16p11.2.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18840528" id="rid33" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          34 : The comorbidity of autism with the genomic disorders of chromosome 15q11.2-q13.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21658581" id="rid34" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          35 : Multiple recurrent de novo CNVs, including duplications of the 7q11.23 Williams syndrome region, are strongly associated with autism.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22970919" id="rid35" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          36 : Phenotypic heterogeneity of genomic disorders and rare copy-number variants.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15338647" id="rid36" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          37 : Genomic disorders: genome architecture results in susceptibility to DNA rearrangements causing common human traits.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12649807" id="rid37" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          38 : Genome architecture catalyzes nonrecurrent chromosomal rearrangements.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19597530" id="rid38" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          39 : Mechanisms of change in gene copy number.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20981092" id="rid39" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          40 : A map of human genome variation from population-scale sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17124402" id="rid40" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          41 : Development of bioinformatics resources for display and analysis of copy number and other structural variants in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24234437" id="rid41" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          42 : ClinVar: public archive of relationships among sequence variation and human phenotype.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19344873" id="rid42" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          43 : DECIPHER: Database of Chromosomal Imbalance and Phenotype in Humans Using Ensembl Resources.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25529582" id="rid43" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          44 : Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/26014595" id="rid44" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          45 : ClinGen--the Clinical Genome Resource.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22102821" id="rid45" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          46 : Relative burden of large CNVs on a range of neurodevelopmental phenotypes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22086107" id="rid46" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          47 : Interpretation of array comparative genome hybridization data: a major challenge.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11854316" id="rid47" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          48 : Transcription factor haploinsufficiency: when half a loaf is not enough.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9637430" id="rid48" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          49 : Delineation of the common critical region in Williams syndrome and clinical correlation of growth, heart defects, ethnicity, and parental origin.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19039520" id="rid49" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          50 : The genomic basis of the Williams-Beuren syndrome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18694825" id="rid50" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          51 : Genetics of intellectual disability.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18487280" id="rid51" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          52 : Utility of oligonucleotide array-based comparative genomic hybridization for detection of target gene deletions.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19639023" id="rid52" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          53 : Duplications of noncoding elements 5' of SOX9 are associated with brachydactyly-anonychia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19327734" id="rid53" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          54 : Duplications involving a conserved regulatory element downstream of BMP2 are associated with brachydactyly type A2.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16882733" id="rid54" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          55 : Molecular and genomic characterisation of cryptic chromosomal alterations leading to paternal duplication of the 11p15.5 Beckwith-Wiedemann region.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/1840874" id="rid55" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          56 : Contiguous deletion syndromes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2227145" id="rid56" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          57 : Old syndromes and new cytogenetics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20340098" id="rid57" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          58 : Insertional translocation detected using FISH confirmation of array-comparative genomic hybridization (aCGH) results.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21915152" id="rid58" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          59 : Parental insertional balanced translocations are an important cause of apparently de novo CNVs in patients with developmental anomalies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28253855" id="rid59" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          60 : SeqCNV: a novel method for identification of copy number variations in targeted next-generation sequencing data.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27980096" id="rid60" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          61 : Homozygous and hemizygous CNV detection from exome sequencing data in a Mendelian disease cohort.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15934818" id="rid61" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          62 : Array-based comparative genomic hybridization in clinical diagnosis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16024975" id="rid62" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          63 : Development and validation of a CGH microarray for clinical cytogenetic diagnosis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17389918" id="rid63" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          64 : Clinical implementation of chromosomal microarray analysis: summary of 2513 postnatal cases.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19047251" id="rid64" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          65 : Genomic imbalances in neonates with birth defects: high detection rates by using chromosomal microarray analysis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20466091" id="rid65" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          66 : Consensus statement: chromosomal microarray is a first-tier clinical diagnostic test for individuals with developmental disabilities or congenital anomalies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17467974" id="rid66" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          67 : Use of array CGH in the evaluation of dysmorphology, malformations, developmental delay, and idiopathic mental retardation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21822880" id="rid67" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          68 : Detection of copy number variation using SNP genotyping.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21315943" id="rid68" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          69 : Identification of incestuous parental relationships by SNP-based DNA microarrays.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21594998" id="rid69" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          70 : UPD detection using homozygosity profiling with a SNP genotyping microarray.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25621458" id="rid70" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          71 : Large multiallelic copy number variations in humans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23893877" id="rid71" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          72 : Detection of clinically relevant copy number variants with whole-exome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9096765" id="rid72" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          73 : Diagnosis of CMT1A duplications and HNPP deletions by interphase FISH: implications for testing in the cytogenetics laboratory.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20938835" id="rid73" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          74 : Multiplex Ligation-dependent Probe Amplification (MLPA®) for the detection of copy number variation in genomic sequences.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30061371" id="rid74" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          75 : Whole-genome sequencing analysis of CNV using low-coverage and paired-end strategies is efficient and outperforms array-based CNV analysis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30293986" id="rid75" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          76 : Copy-number variants in clinical genome sequencing: deployment and interpretation for rare and undiagnosed disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25640679" id="rid76" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          77 : Next-generation sequencing of duplication CNVs reveals that most are tandem and some create fusion genes at breakpoints.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/26742503" id="rid77" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          78 : Clinical sequencing: is WGS the better WES?
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
