<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Prenatal diagnosis of chromosomal imbalance: Chromosomal microarray</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Prenatal diagnosis of chromosomal imbalance: Chromosomal microarray</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Prenatal diagnosis of chromosomal imbalance: Chromosomal microarray</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">David T Miller, MD, PhD, FACMG</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Louise Wilkins-Haug, MD, PhD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Vanessa A Barss, MD, FACOG</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Jul 15, 2022.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H18810847"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Chromosomal microarray (CMA) is a common technique for diagnosis of fetal chromosomal imbalance (eg, aneuploidy, partial deletion or duplication) by analysis of amniocytes, chorionic villi, or fetal blood or tissue. It has several advantages over a conventional karyotype (ie, Giemsa [G]-banded metaphase chromosomes), including detection of submicroscopic gains and losses, faster turnaround time, and the ability to obtain results from cells that do not grow in culture. Identification of fetal chromosomal imbalance is important because it is a contributing factor to some pregnancy losses and the genetic basis for multiple syndromes. </p><p>A noninvasive prenatal screen (NIPS) that involves performing CMA on cell-free DNA in maternal blood may also be used to detect fetal chromosomal imbalance. NIPS is not a diagnostic test; therefore, results should be confirmed by follow-up cytogenetic testing.</p><p>This topic will discuss use of CMA for prenatal diagnosis, including fetal diagnostic evaluation after a demise. Other techniques for fetal genetic screening and diagnosis are reviewed separately:</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/d/html/426.html" rel="external">"Down syndrome: Overview of prenatal screening"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/d/html/458.html" rel="external">"Prenatal screening for common aneuploidies using cell-free DNA"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span>(See  <a class="medical medical_review" href="/d/html/104101.html" rel="external">"Prenatal genetic evaluation of the fetus with anomalies or soft markers"</a>.)</p><p></p><p class="headingAnchor" id="H1712376050"><span class="h1">WHAT IS CMA?</span><span class="headingEndMark"> — </span>CMA is an array-based molecular cytogenetic technique for the diagnosis of chromosomal imbalance. It is particularly useful for its ability to detect submicroscopic gains and losses on every chromosome and provide results from cells that do not grow in culture, such as after a fetal demise. "Array-based" describes the use of preselected segments of DNA, which are adhered to a surface  (<a class="graphic graphic_figure graphicRef139122" href="/d/graphic/139122.html" rel="external">figure 1</a>). Detecting gains or losses involves either comparison of the patient sample with that of a normal control individual or individuals or comparison of signal intensity at the various probes to expected intensity levels. The gains/losses that are detected can range in size from very large (eg, aneuploidy of entire chromosomes) to very small (eg, 200,000 base pairs [0.2 Mb]) and may correspond to identifiable structural changes (such as most unbalanced translocations). </p><p>Prenatal CMA typically assesses the whole genome, which may result in challenging counseling issues because variants of uncertain significance (VUS) and genetic abnormalities unrelated to the diagnostic goal are also detected. CMA can be targeted to regions of the genome known or highly suspected to cause specific phenotypes when deleted or duplicated. The copy number probe coverage on a targeted array is more densely clustered in the "known disease regions" caused by copy number changes, and coverage is less dense elsewhere; therefore, outside of the targeted disease regions, only the larger copy number changes would be detected (those approaching the size visible on a conventional karyotype). Although this reduces the chances of detecting VUS and incidental abnormalities, which is particularly desirable for prenatal arrays, it also increases the chances of missing some diagnoses. (See <a class="local">'Limitations compared with conventional karyotype'</a> below.)</p><p>Essentially all clinical laboratories now offer an array platform that uses a combination of copy number probes and single nucleotide polymorphism (SNP) probes. Interspersing SNP probes along with the copy number probes enables CMA to recognize triploidy as effectively as a conventional karyotype, detect some cases of uniparental disomy and consanguinity, and improve detection of low levels of mosaicism. The addition of SNP probes also enables detection of similarity in single nucleotide sequences (homozygosity). Regions of homozygosity may be a variety of lengths and could occur on any chromosome. Within a region of homozygosity, the allelic variants are exactly the same on both copies of the chromosome. One implication is that a pathogenic variant for a recessive disorder located within the region of homozygosity could result in a fetus affected by that recessive disorder. (See <a class="local">'Detection of some cases of uniparental disomy'</a> below and <a class="local">'Potential detection of consanguinity and other familial relationships'</a> below.)</p><p class="headingAnchor" id="H12423065"><span class="h1">BENEFITS AND LIMITATIONS OF CMA</span><span class="headingEndMark"> — </span>CMA has advantages and disadvantages compared with a conventional karyotype, as shown in the table  (<a class="graphic graphic_table graphicRef109837" href="/d/graphic/109837.html" rel="external">table 1</a>) and discussed below.</p><p class="headingAnchor" id="H18811338"><span class="h2">Benefits compared with conventional karyotype</span></p><p class="headingAnchor" id="H18811391"><span class="h3">Higher diagnostic yield</span><span class="headingEndMark"> — </span>Although the resolution of the array depends on the type of array used and the average spacing of the probes on the array, the minimum resolution is 50,000 to 200,000 base pairs compared with 3 to 10 million base pairs for a conventional karyotype. This ability of CMA to detect extremely small changes in the genome results in greater detection of abnormalities and higher diagnostic yield [<a href="#rid1">1-5</a>]. An abnormal phenotype in individuals with a normal conventional karyotype is likely due to the resolution limitations of conventional karyotyping and the possibility that balanced gains or losses of chromosomal material were not detected. </p><p>In systematic reviews of studies of the utility of fetal CMA when the conventional karyotype was normal, copy number variants (CNVs) of clinical significance were identified in: </p><p class="bulletIndent1"><span class="glyph">●</span>3.1 to 7.9 percent of fetuses tested because of an abnormal ultrasound [<a href="#rid6">6-8</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>4 percent of fetal losses ≥20 weeks of gestation (ie, stillbirths) [<a href="#rid9">9</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>2.4 percent of fetuses tested overall (eg, abnormal ultrasound, maternal age, other [parental anxiety, history of chromosomal abnormality, abnormal serum screening result]) [<a href="#rid6">6</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>0.84 percent of fetuses tested because of advanced maternal age or parental anxiety [<a href="#rid10">10</a>]. Of note, in contrast with aneuploidies, CNVs are independent of maternal age [<a href="#rid11">11,12</a>] and occur in approximately 0.4 percent of pregnancies. As with aneuploidies, the incidence of CNVs in offspring is independent of paternal age [<a href="#rid11">11,13,14</a>]. </p><p></p><p>A supernumerary marker chromosome refers to small chromosome fragments in addition to the normal diploid cell line. CMA is an efficient method to identify and characterize such markers, which may not be identified with a conventional karyotype because of the small size of markers. (See  <a class="medical medical_review" href="/d/html/452.html" rel="external">"Congenital cytogenetic abnormalities", section on 'Supernumerary marker chromosome'</a>.)</p><p class="headingAnchor" id="H2514550684"><span class="h3">Can be performed on uncultured cells</span><span class="headingEndMark"> — </span>CMA can be performed directly on uncultured cells, which shortens the time required to obtain results and increases diagnostic yield when only nondividing cells are available for analysis, such as in some stillborns. By contrast, conventional karyotype requires cell culture to obtain sufficient DNA for analysis from dividing cells.</p><p>In a meta-analysis of studies of stillborn fetuses with a normal karyotype, the test success rates (rates of informative results) for CMA and conventional karyotype were 90 and 75 percent, respectively [<a href="#rid9">9</a>]. (See <a class="local">'Fetal demise'</a> below.)</p><p class="headingAnchor" id="H18811433"><span class="h3">Faster turnaround time and increased testing efficiency</span><span class="headingEndMark"> — </span>CMA can be performed directly on high-quality DNA extracted from isolated cells; results are typically available within one week, but in the best circumstances, results can be available in as little as one to two days since time for culture is not required. CMA is also more amenable to automation in the laboratory, which enables objective interpretation and decreases labor costs.</p><p>By comparison, the cell cultures required to obtain sufficient DNA for conventional karyotype result in a turnaround time of one to two weeks and are more labor intensive.</p><p class="headingAnchor" id="H1482657944"><span class="h3">Potential detection of consanguinity and other familial relationships</span><span class="headingEndMark"> — </span>Single nucleotide polymorphism (SNP)-based CMA platforms may detect multiple regions of homozygosity suggestive of consanguinity. The SNP probes need to be located fairly uniformly across the whole genome to detect all areas that might have identical SNP alleles. Guidelines for reporting suspected consanguinity have been published by the American College of Medical Genetics and Genomics [<a href="#rid15">15</a>]. When SNP-based CMA is performed on the mother, purported father, and fetus (called a trio), nonpaternity may be suspected if the fetus and father share fewer similarities in their SNP markers than what would be expected for a first-degree relative.</p><p>By comparison, conventional karyotype cannot detect consanguinity or nonpaternity.</p><p class="headingAnchor" id="H2779430518"><span class="h3">Detection of some cases of uniparental disomy</span><span class="headingEndMark"> — </span>SNP-based CMA platforms can detect regions of homozygosity, which may indicate a subtype of uniparental disomy (UPD) termed isodisomy (two identical copies of the same chromosome are inherited from one parent due to a meiosis I error and no corresponding chromosome is inherited from the other parent). However, UPD is equally likely to occur through heterodisomy (two different copies of the same chromosome are inherited from one parent due to a meiosis II error or post-zygotic duplication of a chromosome and no corresponding chromosome is inherited from the other parent). UPD due to heterodisomy would not be detected. Therefore, relying on SNP-based CMA for detection of UPD has a high rate of false-negative results (approximately 50 percent). </p><p>Conventional karyotype will not detect any cases of UPD.</p><p class="headingAnchor" id="H18812072"><span class="h2">Limitations compared with conventional karyotype</span></p><p class="headingAnchor" id="H18812086"><span class="h3">Inability to detect balanced structural rearrangements</span><span class="headingEndMark"> — </span>In contrast to conventional karyotype,<strong> </strong>CMA is unable to detect balanced structural rearrangements (balanced translocations and inversions) because CMA only quantifies the amount of DNA present, it does not detect the physical location of the extra genetic material or structural chromosomal changes in which the total amount of DNA remains normal (no change in copy number). For example, CMA is unable to differentiate trisomy due to a full 21/22 translocation from trisomy 21 involving three free-lying chromosomes. For this reason, when fetal CMA shows trisomy, genetic counseling should discuss the possibility that a parent carries a balanced translocation, and parental (or fetal) karyotype should be performed to determine recurrence risks. </p><p>However, the inability to detect balanced structural rearrangements on a CMA is not always of clinical significance because:</p><p class="bulletIndent1"><span class="glyph">●</span>The likelihood that a truly balanced rearrangement would interrupt a gene is low. Even those that have breakpoints within a gene are not necessarily pathogenic [<a href="#rid16">16</a>]. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>When the parent and fetus have the same apparently balanced rearrangement, it is generally considered benign if the parent is unaffected. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>De novo balanced rearrangements are rare (&lt;0.1 percent [<a href="#rid17">17,18</a>]) and represent a small proportion of the abnormalities detected by conventional karyotype. </p><p></p><p class="bulletIndent1">Importantly, de novo rearrangements that appear balanced by conventional karyotype may actually have submicroscopic genomic differences in the number of copies of one or more sections of DNA that can be detected by CMA. These CNVs may be known to be benign or associated with phenotypic consequences, or their significance may be unknown. There is a 6 to 9 percent probability that an apparently balanced translocation identified on prenatal conventional karyotype has phenotypic consequences because it contains submicroscopic genomic differences and is not truly balanced [<a href="#rid18">18</a>]. Thus, the overall ability to predict abnormalities is more strongly correlated with findings on CMA, which detects submicroscopic genomic imbalance, than conventional karyotype, which has more limited resolution [<a href="#rid19">19-21</a>].</p><p></p><p class="bulletIndent1">The ability of a balanced structural rearrangement to cause phenotypic findings is likely due to disruption of a gene or important functional element. The 6 to 9 percent probability that an apparently balanced translocation identified on prenatal conventional karyotype has phenotypic consequences is based on phenotypic assessment of the newborn and thus biased toward phenotypes that manifest as physical anomalies. Neurodevelopmental or neuropsychiatric phenotypes cannot be assessed in the newborn period and may be a more common consequence of balanced structural rearrangements than physical anomalies. In one study in which 41 individuals with a balanced structural rearrangement were followed for a mean of 17 years, 27 percent showed a neurodevelopmental or neuropsychiatric phenotype [<a href="#rid22">22</a>].</p><p></p><p class="headingAnchor" id="H1547173431"><span class="h3">Detection of variants of uncertain significance</span><span class="headingEndMark"> — </span>Copy number variants of uncertain significance (VUS) are copy number changes that have not been reported and thus have an unknown phenotype. These small changes are not detectable by conventional karyotype but are identified by CMA in 1 to 2 percent of cases [<a href="#rid1">1,2,23,24</a>]. The additional information provided by CMA can be challenging to interpret, particularly if a phenotypically normal parent carries the same change. In the absence of clear prognostic information, parents may find it difficult to make a decision about continuing the pregnancy. (See  <a class="medical medical_review" href="/d/html/2900.html" rel="external">"Basic genetics concepts: DNA regulation and gene expression", section on 'Genetic variation'</a>.)</p><p>In a large prospective study performed from 2008 to mid-2011, CMA detected VUS in 3.4 percent (130 of 3822) of all cases that were normal by karyotype [<a href="#rid1">1</a>]. Although this degree of uncertainty is concerning, collective knowledge in this field continues to improve, thereby decreasing the frequency of VUS. If the VUS identified in this study were reanalyzed in 2021, the overall VUS rate would be approximately 1.5 percent (56 of 3822).</p><p>Detection of VUS has raised the question of whether it is ethically justifiable to withhold these test results from the patient [<a href="#rid25">25</a>]. VUS should become less problematic in the future as the International Standards for Cytogenomic Arrays (ISCA) is creating a database of array results and associated phenotypes for the National Institutes of Health (NIH). Some prenatal diagnostic centers have started to periodically compare VUS detected in past patients against updated information on phenotypes associated with CNVs and provide updates to the clinical reports for clinicians.</p><p>Laboratories can minimize the number of VUS through a combination of probe design (eg, eliminate results below a certain size threshold) and bioinformatic filtering (eg, only report variants above a certain size threshold or with evidence of clinical significance, regardless of size). These practices are generally similar across laboratories but may differ somewhat, leading to differences in results between and among clinical laboratories. Clinicians need to be familiar with these issues and how the laboratory reports results so they can have informed discussions with the laboratory providing the testing and, in turn, with their patients. </p><p class="headingAnchor" id="H837652650"><span class="h3">Detection of adult-onset disorders</span><span class="headingEndMark"> — </span>Prenatal diagnosis is typically performed to detect disorders manifesting early in life. However, copy number variants related to late- or adult-onset diseases may also be identified (so-called secondary findings [<a href="#rid26">26,27</a>]), which can have implications for both the parent and offspring. For example, a prenatal diagnostic procedure to detect trisomy 21 may also identify a <em>BRCA1</em> or <em>BRCA2</em> deletion inherited from a parent who is unaware of their carrier status. </p><p>The American College of Medical Genetics and Genomics (ACMG) guideline for reporting postnatal CMA results [<a href="#rid28">28</a>] makes a general recommendation to report deletions or duplications associated with presymptomatic (eg, late or adult onset) conditions to facilitate early access to medical care, but does not specify an exact list of genes and does not describe a policy related to reporting of incidental or secondary findings in prenatal samples. Generally, laboratories will describe their policy about reporting of incidental findings and may choose to report deletions or duplications related to the conditions on the ACMG list for secondary findings. Pretest counseling should include making the patient aware that incidental findings may be identified. </p><p class="headingAnchor" id="H2212754439"><span class="h3">Cost</span><span class="headingEndMark"> — </span>Costs for conventional karyotypes and molecular cytogenetics such as CMA vary considerably worldwide and across institutions and commercial laboratories. Providers ordering such testing should be knowledgeable as to the relative costs in their community and the availability of insurance coverage and need for prior authorizations.</p><p class="headingAnchor" id="H2163966987"><span class="h2">Limitations of both CMA and conventional karyotype</span></p><p class="headingAnchor" id="H3704877042"><span class="h3">Inability to detect low levels of mosaicism</span><span class="headingEndMark"> — </span>A typical 20-cell conventional karyotype detects mosaicism at approximately 14 percent of cells or more, and most current clinical CMA with SNP platforms typically detect mosaicism at comparable levels (approximately 10 to 15 percent). If there is suspicion of an even lower level mosaicism, a cytogenetics laboratory can be asked to assess additional cells by fluorescence in situ hybridization to improve the detection of mosaicism below 10 percent of cells. Very low-level mosaicism, however, can present a challenge in counseling as it generally results in a milder phenotypic abnormality than the same but nonmosaic genomic imbalance.</p><p class="headingAnchor" id="H3476934023"><span class="h3">Variable clinical sensitivity</span><span class="headingEndMark"> — </span>Clinical sensitivity refers to the proportion of patients who have the clinical symptoms of a disorder and are detectable by the testing method. This limitation is true of all genetic tests as there is no single test to detect all types of genetic aberration.</p><p>Neither CMA nor conventional karyotype detects autosomal recessive and autosomal dominant disorders, such as inborn errors of metabolism, skeletal dysplasias, and many syndromes. If a patient has a genetic condition in which all or a subset of cases are caused by DNA sequence changes, then DNA sequencing should be considered either in place of, or in addition to, CMA. As an example, among individuals with signs/symptoms of velocardiofacial syndrome, also known as 22q11.2 deletion syndrome, CMA will detect a microdeletion in over 90 percent. When the microdeletion is not detected, the patient usually does not have all of the symptoms and should be worked up for another similar diagnosis before resorting to genetic sequencing. By contrast, neurofibromatosis type 1 (NF1) can be caused by a genomic deletion that includes the <em>NF1</em> gene, but less than 5 percent of NF1 patients have a deletion; most of these patients have a sequence change in the gene that is undetectable by CMA and should be evaluated by DNA sequencing. </p><p class="headingAnchor" id="H1607843848"><span class="h3">Need for an invasive procedure</span><span class="headingEndMark"> — </span>Both CMA and conventional karyotype require an invasive procedure to obtain cells for prenatal diagnostic testing. Next-generation sequencing of cell-free DNA in maternal blood is a commonly used, noninvasive prenatal screen (NIPS) for fetal aneuploidy, most typically of chromosomes 13, 18, 21, X, and Y. Some clinical laboratories have refined the method to be able to assess for some of the common microdeletion syndromes (typically five syndromes), but NIPS for clinical use does not assess for microdeletions and microduplications throughout the genome in the same way as CMA. Although NIPS is being investigated as a screening test for fetal microdeletions/microduplications, it is not sufficiently sensitive to replace CMA as a diagnostic test for microdeletion or microduplication syndromes, particularly early in pregnancy. Currently, NIPS is able to assess DNA at the level approximately equal to the resolution of karyotype (&gt;7 Mb changes), a size much larger than the common microdeletion and microduplication syndromes. </p><p class="bulletIndent1"><span class="glyph">●</span>In a study involving 802 pregnant people who were tested in the second or third trimester, NIPS was able to identify approximately 70 percent of chromosomal gains or losses that were in the range of 5 to 20 Mb, but only 10 percent of those &lt;5 Mb, which is the range for common microdeletion and microduplication syndromes (eg, Williams syndrome and Prader-Willi syndrome) [<a href="#rid29">29</a>]. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>In a multicenter retrospective cohort analysis of 5541 low-risk pregnancies (ie, normal ultrasound) undergoing CMA from 2010 to 2016, 78 (1.4 percent) had clinically significant findings by CMA [<a href="#rid30">30</a>]. Of these, a conventional karyotype could have detected 31 (39.7 percent) since the variants had a size above 10 Mb and NIPS aimed at five common aneuploidies, the typical current offering for clinical NIPS, could have detected 28 (35.9 percent). Among the 47 submicroscopic results detectable by CMA only, 37 cases (78.7 percent) represented known microdeletion or microduplication syndromes.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>In a multicenter retrospective study of 943 singleton, high-risk pregnancies ascertained from 2015 through 2019, expanded genome-wide NIPS and CMA were both performed during pregnancy [<a href="#rid31">31</a>]. Concordance rates between NIPS and CMA for common aneuploidies varied by chromosome and were 82.3, 59.6, and 25.0 percent for trisomy 21, 18, and 13, respectively. For rare aneuploidies and segmental imbalances, NIPS and CMA results were concordant in only 7.5 and 33.3 percent of cases, respectively; the poor performance of NIPS is because the data analysis algorithms for NIPS are optimized for full chromosome aneuploidies and a small number of syndromes. CMA performed much better than NIPS for CNVs smaller than 5 Mb, detecting them in 6.5 percent of cases (61/943) versus 2.6 percent (24/943) for NIPS.</p><p></p><p class="headingAnchor" id="H1897463"><span class="h1">MOST COMMON USES OF CMA IN PRENATAL DIAGNOSIS</span></p><p class="headingAnchor" id="H12423086"><span class="h2">Fetus with anomalies</span><span class="headingEndMark"> — </span>The American College of Obstetricians and Gynecologists (ACOG) and the Society for Maternal-Fetal Medicine (SMFM) recommend use of CMA for genetic evaluation of fetuses with structural anomalies [<a href="#rid32">32,33</a>]. By comparison, for patients undergoing invasive prenatal diagnosis of a structurally normal viable fetus, ACOG concluded that either conventional karyotype or CMA is acceptable. Others have advocated CMA as a first-line test whenever fetal chromosomal analysis is planned [<a href="#rid34">34</a>]. (See  <a class="medical medical_review" href="/d/html/104101.html" rel="external">"Prenatal genetic evaluation of the fetus with anomalies or soft markers"</a>.)</p><p>In the setting of a fetus with structural abnormalities, some providers order fluorescence in situ hybridization (FISH) for chromosomes 13, 18, 21, X, and Y as a first-line test. This FISH panel will typically produce a result in 24 to 48 hours, which may be desirable in some cases since typical CMA can take up to one week. However, if the difference in turnaround times is not important to patient management, then FISH has no benefit. Essentially all clinical laboratories performing prenatal CMA are using a single nucleotide polymorphism platform and thus will be able to detect triploidy, so there is no increase in diagnostic yield with a FISH panel.</p><p>The preference for CMA over conventional karyotype in the setting of a fetal structural abnormality detectable by ultrasound is based on the high prevalence of genetic abnormalities in these cases and the increased diagnostic yield of CMA.</p><p class="bulletIndent1"><span class="glyph">●</span>In a study that karyotyped 2086 fetuses after ultrasonographic examination revealed fetal anomalies, growth restriction, or both, chromosomal abnormalities by conventional karyotype were detected in 301 cases (14 percent) and were more common among fetuses with multisystem anomalies (29 percent) than among those with an isolated anomaly (2 percent) [<a href="#rid35">35</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>In systematic reviews of CMA testing of anomalous fetuses with normal conventional karyotypes, the incremental diagnostic yield of CMA ranged from 3.1 to 7.9 percent [<a href="#rid6">6-8</a>]. The detection of abnormalities on CMA increases as the density of the array increases and when multiple anomalies are present.</p><p></p><p class="bulletIndent1">Congenital heart disease is the most common potentially severe fetal anomaly. In fetuses with congenital heart disease, a systematic review found that performing CMA had value even if the combination of conventional karyotype and 22q11 microdeletion analysis was normal [<a href="#rid36">36</a>]. In cases of isolated congenital heart disease with normal karyotype and normal 22q11 microdeletion analysis by FISH, the overall yield of additional pathogenic copy number variants (CNV) detected by CMA was approximately 3 percent but can be lower or higher for specific cardiac defects (eg, 1.4 percent for isolated ventricular septal defect in one study [<a href="#rid37">37</a>]). The overall incremental yield was 7 percent in fetuses with both cardiac and noncardiac anomalies.</p><p></p><p class="headingAnchor" id="H12423093"><span class="h2">Fetal demise</span><span class="headingEndMark"> — </span>ACOG and SMFM recommend use of CMA and state that CMA typically can replace the need for a conventional karyotype when genetic analysis is desired because of fetal demise/stillbirth [<a href="#rid32">32,33,38</a>]. (See  <a class="medical medical_review" href="/d/html/5402.html" rel="external">"Stillbirth: Maternal and fetal evaluation", section on 'Options for genetic testing'</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Late fetal loss</strong> – CMA is useful in evaluation of stillbirth (pregnancy loss ≥20 weeks of gestation) because both chromosomal abnormalities and culture failure are common in these cases. Chromosomal abnormalities identifiable by conventional karyotype are identified in approximately 5 percent of stillborn fetuses in the absence of an anatomic malformation and 35 to 40 percent of stillborns when structural abnormalities are present [<a href="#rid39">39,40</a>]. Culture failure is common when the fetus has died and thus prevents the accurate diagnosis of a karyotypic abnormality in these cases; CMA can overcome this difficulty.</p><p></p><p class="bulletIndent1">In a meta-analysis of seven studies involving 903 stillborn fetuses with a normal conventional karyotype, CMA had higher test success than the conventional cytogenetic analysis (90 versus 75 percent) [<a href="#rid9">9</a>]. The incremental yield of CMA over the conventional karyotype-based approach was 4 percent for pathogenic CNVs (6 percent for structurally abnormal fetuses and 3 percent for structurally normal fetuses); 8 percent had variants of uncertain significance (VUS). The pathogenic CNV found most commonly was 22q11.2 deletion syndrome.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Early fetal loss </strong>– CMA may also have some advantages over conventional karyotyping for diagnostic evaluation of miscarriages (pregnancy loss before 20 weeks). In a systematic review (23 studies, 5507 losses up to 20 weeks), when CMA and conventional karyotyping were performed concurrently, CMA had a higher yield of informative results (mean 95 versus 68 percent of cases) [<a href="#rid41">41</a>]. In this context, an informative result is one in which the test can be completed and provides a definitive positive or negative result. The incremental diagnostic yield of CMA over karyotyping was 2 percent for pathogenic CNVs, with a 4 percent frequency of VUS. The most commonly reported pathogenic CNVs were 22q11.21 and 1p36.33 deletion. </p><p></p><p class="headingAnchor" id="H1777982864"><span class="h2">Enlarged nuchal translucency</span><span class="headingEndMark"> — </span>Enlarged nuchal translucency (NT) is a marker for fetal genetic abnormalities and is used as a component of prenatal trisomy 21 screening in the first-trimester combined test and in first- and second-trimester integrated tests. Individuals with positive screening results may choose to undergo secondary cell-free DNA (cfDNA) screening followed by definitive fetal chromosomal analysis if secondary screening using cfDNA is also positive, or they may choose to proceed directly to definitive fetal chromosomal analysis without secondary cfDNA screening. (See  <a class="medical medical_review" href="/d/html/458.html" rel="external">"Prenatal screening for common aneuploidies using cell-free DNA", section on 'Secondary cfDNA screening'</a>.)</p><p>We offer CMA rather than a conventional karyotype to all patients undergoing invasive genetic studies for enlarged NT. In a systematic review of pooled data from 17 studies and 1696 pregnancies, the incremental yield of CMA over conventional karyotyping was 4 percent among fetuses with isolated enlarged NT and 7 percent among fetuses with enlarged NT associated with structural abnormalities diagnosed by first-trimester ultrasound [<a href="#rid42">42</a>]. The most common pathogenic CNVs detected by CMA were 22q11.2 deletion, 22q11.2 duplication, 10q26.12q26.3 deletion, and 12q21q22 deletion, and approximately 1 percent were VUS. Subsequent studies have confirmed these findings [<a href="#rid43">43-47</a>]. (See  <a class="medical medical_review" href="/d/html/435.html" rel="external">"Enlarged nuchal translucency and cystic hygroma", section on 'Genetic studies'</a>.)</p><p class="headingAnchor" id="H3204429187"><span class="h2">Confirmation of a microdeletion syndrome detected by NIPS</span><span class="headingEndMark"> — </span>Commercial companies have begun to offer noninvasive prenatal screening (NIPS) of cfDNA in maternal blood to detect common microdeletion syndromes such as 22q11.2 deletion (DiGeorge syndrome); however, they do not report the genomic coordinates or whether the deletion is maternal or fetal [<a href="#rid48">48</a>]. If microdeletion NIPS is performed, fetal or neonatal testing with CMA and appropriate parental studies are needed to confirm the diagnosis and determine precise genomic coordinates and inheritance.</p><p class="headingAnchor" id="H1897960"><span class="h2">Other potential uses</span><span class="headingEndMark"> — </span>In other settings where diagnostic fetal genetic studies may be desired, the incremental diagnostic yield of CMA over conventional karyotype is not well-established, but CMA may be considered. Such settings include:</p><p class="bulletIndent1"><span class="glyph">●</span>Genetic evaluation of fetal growth restriction (FGR). Although a meta-analysis including 10 small studies reported a 4 percent (95% CI 1-6 percent) incremental yield of CMA over karyotyping in nonanomalous FGR, the findings were limited by high heterogeneity of the included studies [<a href="#rid49">49</a>]. Of course, other factors, such as placental insufficiency and congenital infection, can impact fetal growth. If other causes of FGR have been ruled out, then chromosomal disorders should be considered as a possible explanation. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Diagnostic testing because of a screen-positive result on Down syndrome screening (maternal serum screening, cfDNA screening, and some cases of sonographic identification of soft markers without structural anomalies). (See  <a class="medical medical_review" href="/d/html/426.html" rel="external">"Down syndrome: Overview of prenatal screening", section on 'Screen-positive test result'</a> and  <a class="medical medical_review" href="/d/html/104101.html" rel="external">"Prenatal genetic evaluation of the fetus with anomalies or soft markers", section on 'Approach to the evaluation of the fetus with "soft markers" and no structural anomalies'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genetic evaluation of the fetus with oligohydramnios or polyhydramnios in the second trimester when other causes of abnormal amniotic fluid volume have been ruled out. (See  <a class="medical medical_review" href="/d/html/6781.html" rel="external">"Polyhydramnios: Etiology, diagnosis, and management in singleton gestations"</a> and  <a class="medical medical_review" href="/d/html/6777.html" rel="external">"Oligohydramnios: Etiology, diagnosis, and management in singleton gestations"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genetic evaluation of the fetus in pregnancies at low risk of a genetic abnormality.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>In pregnancies at low risk of a chromosomal abnormality (ie, maternal age &lt;35 years with NT &lt;3 mm, normal Down syndrome biochemical screening tests, and normal sonographic fetal survey [including absence of soft markers]), a retrospective study of over 2700 such pregnancies found that the overall rate of clinically significant chromosomal abnormalities was low at 0.76 percent, and additional diagnostic yield of CMA in fetuses with a normal karyotype was 0.55 percent [<a href="#rid30">30</a>]. Approximately 71 percent of the anomalies would have been missed by routine karyotyping, and 81 to 90 percent would have been missed by NIPS aimed at detecting 21/18/13 or sex chromosome aneuploidy and 21/18/13 aneuploidy, respectively.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>In another retrospective cohort of over 6400 low-risk pregnancies, the yield of CMA results considered clinically significant was 1.12 percent (72/6431), and among these positive results, 0.42 percent (27/6431) were pathogenic or likely pathogenic CNVs [<a href="#rid50">50</a>]. This number is comparable to the 0.55 percent yield of CMA in the presence of a normal karyotype in the above study. The 1.12 percent total includes 0.7 percent (45/6431) that had a susceptibility locus with more than 10 percent penetrance as the reported finding, and it is questionable whether those types of results would be routinely reported on a prenatal CMA.</p><p></p><p class="headingAnchor" id="H12423100"><span class="h1">PATIENT COUNSELING</span><span class="headingEndMark"> — </span>Patients considering CMA testing should receive pre- and posttest genetic counseling that is nondirective and clearly explains that testing is voluntary. Nondirective information enables patients to balance the risks, benefits, and limitations of this approach to prenatal diagnosis. Genetic counseling should involve a provider with specific expertise in the area of prenatal genetic testing, usually a genetic counselor, medical geneticist, or provider with relevant experience, given the range of potential results from CMA, the variable clinical effects of particular genomic imbalance disorders, and the disclosure of incidental findings of importance to their own or the offspring's adult health.</p><p><strong>Pretest counseling</strong> should provide the following information [<a href="#rid51">51</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>Potential psychological implications of prenatal diagnosis (eg, uncertainty, anxiety, need for consideration of pregnancy termination or other intervention). </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Risks associated with prenatal diagnosis (eg, procedure-related fetal loss).</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Implications of having a child with a genomic disorder. However, the patient should understand that the severity of disease often cannot be predicted by prenatal genetic studies and genetic studies may detect the gene for a disease that never manifests clinically.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Advantages of CMA versus conventional karyotype. (See <a class="local">'Benefits compared with conventional karyotype'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Limitations of CMA. The possibility and clinical implications of the following should be discussed (see <a class="local">'Limitations compared with conventional karyotype'</a> above):</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Variants of unknown significance. </p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Nondetection of a balanced rearrangement.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Potential CMA findings beyond a specific fetal diagnosis (see <a class="local">'Limitations compared with conventional karyotype'</a> above):</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Genes for adult-onset diseases may be identified, which have implications for the parents and the fetus.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Consanguinity or nonpaternity may be identified, potentially resulting in additional and unexpected concerns.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Information about the length of time necessary to obtain CMA results.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Results may be available one week after biopsy with direct cell preparation (eg, chorionic villus biopsy, biopsy of fetal tissue, amniocytes).</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>If amniocytes are cultured initially for conventional karyotype and then CMA is performed, results may be available three weeks after amniocentesis. </p><p></p><p><strong>Posttest counseling</strong> is best provided through consultation with a genetic counselor or a medical geneticist, particularly after a diagnosis of a genomic imbalance disorder, to provide the following information:</p><p class="bulletIndent1"><span class="glyph">●</span>Abnormal/positive results allow a better understanding of the etiology of the sonographic findings present in the fetus but not necessarily an ability to predict the medical or developmental outcome of a child born with that genomic disorder. The patient should be informed about:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The spectrum of medical and intellectual issues associated with the identified genomic disorder.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The difficulty in predicting phenotype due to variable penetrance and expressivity. Uncertainty about phenotypic expression can be particularly problematic for copy number changes associated with risk of neurocognitive or neuropsychiatric symptoms where the degree of symptom severity can be quite variable, even within the same family.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>How mutations arise (de novo versus inherited).</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Recurrence risks for future pregnancies. </p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Information about pregnancy termination. (See  <a class="medical medical_review" href="/d/html/3287.html" rel="external">"First-trimester pregnancy termination: Uterine aspiration"</a> and  <a class="medical medical_review" href="/d/html/14194.html" rel="external">"Overview of second-trimester pregnancy termination"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Normal/negative results do not rule out the presence of a genetic disorder because many disorders are not caused by microdeletions or microduplications. Other genetic testing may be warranted based on the clinical presentation.</p><p></p><p class="bulletIndent1">Detection of chromosomal imbalance is also possible based on analysis of fetal DNA by exome or genome sequencing (ES or GS). Copy number variants are detected more often by GS than ES, but ES may be able to detect large gains or losses (larger than 5 Mb). However, the bioinformatics capabilities to determine chromosomal imbalance from ES data vary among laboratories and have not become standard across all laboratories. This is in contrast to CMA where there is less likely to be variability in the detection of chromosomal imbalance among different clinical laboratories.</p><p></p><p class="bulletIndent1">ES or GS is a reasonable option in select cases, such as recurrent or lethal fetal anomalies in which a thorough evaluation has been noninformative. A systematic literature review documented the diagnostic yield for prenatal ES among unselected pregnancies complicated by fetal anomalies on ultrasound [<a href="#rid52">52</a>]. After a normal karyotype and CMA, prenatal ES had an overall diagnostic yield of 31 percent. Society guidelines regarding prenatal ES/GS were updated in 2022 to address best practices for the implementation of prenatal ES/GS [<a href="#rid53">53</a>].</p><p></p><p class="bulletIndent1">Before ES or GS testing, these patients should receive extensive counseling by an obstetrician-gynecologist or other health care provider with genetics expertise who is familiar with the new technology and its limitations. For example, patients should be informed that prenatal ES or GS would result in an even higher possibility of variants of uncertain significance and/or incidental findings compared with CMA.</p><p></p><p class="headingAnchor" id="H12423107"><span class="h1">SUMMARY AND RECOMMENDATIONS</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>General principles</strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Overview of chromosomal microarray</strong> – Prenatal chromosomal microarray (CMA) analysis is usually performed on DNA from uncultured amniocytes (amniotic fluid) or chorionic villus cells, but can be performed on cord blood and products of conception. (See <a class="local">'What is CMA?'</a> above.)</p><p></p><p class="bulletIndent2">Prenatal CMA is typically ordered with whole genome coverage, but may be targeted. It provides most of the information derived from conventional karyotype analysis (ie, presence/absence of aneuploidy, unbalanced structural changes, and triploidy) and higher sensitivity for detection of submicroscopic deletions and duplications. When performed with single nucleotide polymorphism (SNP) probes, it can detect runs of homozygosity between two copies of the same chromosome, thus enabling diagnosis of some cases of uniparental disomy and consanguinity. (See <a class="local">'What is CMA?'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Limitations</strong> – Balanced rearrangements are not detectable by CMA; however, these represent a smaller number of cases as compared with those with genomic imbalance. (See <a class="local">'Inability to detect balanced structural rearrangements'</a> above.) </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Counseling</strong> – Genetic counseling by a qualified provider should always be offered before and after prenatal CMA. Pretest counseling should include a discussion of the medical and psychological risks and the advantages and disadvantages of this approach compared with conventional karyotype. CMA detects variants of uncertain significance (VUS) in approximately 1 to 2 percent of cases. The concepts of VUS and variable expressivity and penetrance should be explained. Posttest counseling should include interpretation of the findings and an explanation of possible follow-up studies. (See <a class="local">'Patient counseling'</a> above and <a class="local">'Detection of variants of uncertain significance'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Clinical use</strong> </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Indications</strong> – In the obstetric setting, CMA is a preferred option for further evaluation of fetuses with structural abnormalities, after fetal demise (particularly when chromosomal analysis is desired but conventional karyotype is not possible due to failure of cell culture), and when a marker chromosome is identified. It is equivalent to conventional karyotype when the primary goal is to detect aneuploidy, as in Down syndrome screening. Some clinicians have advocated microarray as a first-line test whenever fetal chromosomal analysis is planned. (See <a class="local">'Most common uses of CMA in prenatal diagnosis'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Advantages</strong> – Advantages of CMA over conventional karyotype in the prenatal setting include higher diagnostic yield, faster turnaround time, and ability to perform the test without requiring cell culture. (See <a class="local">'Benefits compared with conventional karyotype'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Laboratory issues</strong> – Clinicians should understand the laboratory's general reporting practices, which vary among laboratories. Variations in the number and spacing of probes, or selection of a "targeted" prenatal CMA, can affect the number of VUS and how final results are reported. (See <a class="local">'What is CMA?'</a> above.)</p><p></p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Wapner RJ, Martin CL, Levy B, et al. Chromosomal microarray versus karyotyping for prenatal diagnosis. N Engl J Med 2012; 367:2175.</a></li><li><a class="nounderline abstract_t">Reddy UM, Page GP, Saade GR, et al. Karyotype versus microarray testing for genetic abnormalities after stillbirth. N Engl J Med 2012; 367:2185.</a></li><li><a class="nounderline abstract_t">Shaffer LG, Dabell MP, Fisher AJ, et al. Experience with microarray-based comparative genomic hybridization for prenatal diagnosis in over 5000 pregnancies. Prenat Diagn 2012; 32:976.</a></li><li><a class="nounderline abstract_t">Shaffer LG, Rosenfeld JA, Dabell MP, et al. Detection rates of clinically significant genomic alterations by microarray analysis for specific anomalies detected by ultrasound. Prenat Diagn 2012; 32:986.</a></li><li><a class="nounderline abstract_t">Scott F, Murphy K, Carey L, et al. Prenatal diagnosis using combined quantitative fluorescent polymerase chain reaction and array comparative genomic hybridization analysis as a first-line test: results from over 1000 consecutive cases. Ultrasound Obstet Gynecol 2013; 41:500.</a></li><li><a class="nounderline abstract_t">Callaway JL, Shaffer LG, Chitty LS, et al. The clinical utility of microarray technologies applied to prenatal cytogenetics in the presence of a normal conventional karyotype: a review of the literature. Prenat Diagn 2013; 33:1119.</a></li><li><a class="nounderline abstract_t">de Wit MC, Srebniak MI, Govaerts LC, et al. Additional value of prenatal genomic array testing in fetuses with isolated structural ultrasound abnormalities and a normal karyotype: a systematic review of the literature. Ultrasound Obstet Gynecol 2014; 43:139.</a></li><li><a class="nounderline abstract_t">Hillman SC, McMullan DJ, Hall G, et al. Use of prenatal chromosomal microarray: prospective cohort study and systematic review and meta-analysis. Ultrasound Obstet Gynecol 2013; 41:610.</a></li><li><a class="nounderline abstract_t">Martinez-Portilla RJ, Pauta M, Hawkins-Villarreal A, et al. Added value of chromosomal microarray analysis over conventional karyotyping in stillbirth work-up: systematic review and meta-analysis. Ultrasound Obstet Gynecol 2019; 53:590.</a></li><li><a class="nounderline abstract_t">Srebniak MI, Joosten M, Knapen MFCM, et al. Frequency of submicroscopic chromosomal aberrations in pregnancies without increased risk for structural chromosomal aberrations: systematic review and meta-analysis. Ultrasound Obstet Gynecol 2018; 51:445.</a></li><li><a class="nounderline abstract_t">Larroya M, Tortajada M, Mensión E, et al. Have maternal or paternal ages any impact on the prenatal incidence of genomic copy number variants associated with fetal structural anomalies? PLoS One 2021; 16:e0253866.</a></li><li><a class="nounderline abstract_t">Levy B, Wapner R. Prenatal diagnosis by chromosomal microarray analysis. Fertil Steril 2018; 109:201.</a></li><li><a class="nounderline abstract_t">Buizer-Voskamp JE, Blauw HM, Boks MP, et al. Increased paternal age and the influence on burden of genomic copy number variation in the general population. Hum Genet 2013; 132:443.</a></li><li><a class="nounderline abstract_t">Dviri M, Madjunkova S, Koziarz A, et al. Is there an association between paternal age and aneuploidy? Evidence from young donor oocyte-derived embryos: a systematic review and individual patient data meta-analysis. Hum Reprod Update 2021; 27:486.</a></li><li><a class="nounderline abstract_t">Rehder CW, David KL, Hirsch B, et al. American College of Medical Genetics and Genomics: standards and guidelines for documenting suspected consanguinity as an incidental finding of genomic testing. Genet Med 2013; 15:150.</a></li><li><a class="nounderline abstract_t">Baptista J, Prigmore E, Gribble SM, et al. Molecular cytogenetic analyses of breakpoints in apparently balanced reciprocal translocations carried by phenotypically normal individuals. Eur J Hum Genet 2005; 13:1205.</a></li><li><a class="nounderline abstract_t">Giardino D, Corti C, Ballarati L, et al. De novo balanced chromosome rearrangements in prenatal diagnosis. Prenat Diagn 2009; 29:257.</a></li><li><a class="nounderline abstract_t">Warburton D. De novo balanced chromosome rearrangements and extra marker chromosomes identified at prenatal diagnosis: clinical significance and distribution of breakpoints. Am J Hum Genet 1991; 49:995.</a></li><li><a class="nounderline abstract_t">Gribble SM, Prigmore E, Burford DC, et al. The complex nature of constitutional de novo apparently balanced translocations in patients presenting with abnormal phenotypes. J Med Genet 2005; 42:8.</a></li><li><a class="nounderline abstract_t">De Gregori M, Ciccone R, Magini P, et al. Cryptic deletions are a common finding in "balanced" reciprocal and complex chromosome rearrangements: a study of 59 patients. J Med Genet 2007; 44:750.</a></li><li><a class="nounderline abstract_t">Feenstra I, Hanemaaijer N, Sikkema-Raddatz B, et al. Balanced into array: genome-wide array analysis in 54 patients with an apparently balanced de novo chromosome rearrangement and a meta-analysis. Eur J Hum Genet 2011; 19:1152.</a></li><li><a class="nounderline abstract_t">Halgren C, Nielsen NM, Nazaryan-Petersen L, et al. Risks and Recommendations in Prenatally Detected De Novo Balanced Chromosomal Rearrangements from Assessment of Long-Term Outcomes. Am J Hum Genet 2018; 102:1090.</a></li><li><a class="nounderline abstract_t">Hillman SC, Pretlove S, Coomarasamy A, et al. Additional information from array comparative genomic hybridization technology over conventional karyotyping in prenatal diagnosis: a systematic review and meta-analysis. Ultrasound Obstet Gynecol 2011; 37:6.</a></li><li><a class="nounderline abstract_t">Breman A, Pursley AN, Hixson P, et al. Prenatal chromosomal microarray analysis in a diagnostic laboratory; experience with &gt;1000 cases and review of the literature. Prenat Diagn 2012; 32:351.</a></li><li><a class="nounderline abstract_t">McGillivray G, Rosenfeld JA, McKinlay Gardner RJ, Gillam LH. Genetic counselling and ethical issues with chromosome microarray analysis in prenatal testing. Prenat Diagn 2012; 32:389.</a></li><li><a class="nounderline abstract_t">Miller DT, Lee K, Chung WK, et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2021; 23:1381.</a></li><li><a class="nounderline abstract_t">Miller DT, Lee K, Gordon AS, et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2021 update: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2021; 23:1391.</a></li><li><a class="nounderline abstract_t">Kearney HM, Thorland EC, Brown KK, et al. American College of Medical Genetics standards and guidelines for interpretation and reporting of postnatal constitutional copy number variants. Genet Med 2011; 13:680.</a></li><li><a class="nounderline abstract_t">Zhu Y, Shan Q, Zheng J, et al. Comparison of Efficiencies of Non-invasive Prenatal Testing, Karyotyping, and Chromosomal Micro-Array for Diagnosing Fetal Chromosomal Anomalies in the Second and Third Trimesters. Front Genet 2019; 10:69.</a></li><li><a class="nounderline abstract_t">Sagi-Dain L, Cohen Vig L, Kahana S, et al. Chromosomal microarray vs. NIPS: analysis of 5541 low-risk pregnancies. Genet Med 2019; 21:2462.</a></li><li><a class="nounderline abstract_t">Zhu X, Chen M, Wang H, et al. Clinical utility of expanded non-invasive prenatal screening and chromosomal microarray analysis in high-risk pregnancy. Ultrasound Obstet Gynecol 2021; 57:459.</a></li><li><a class="nounderline abstract_t">Committee on Genetics and the Society for Maternal-Fetal Medicine. Committee Opinion No.682: Microarrays and Next-Generation Sequencing Technology: The Use of Advanced Genetic Diagnostic Tools in Obstetrics and Gynecology. Obstet Gynecol 2016; 128:e262. Reaffirmed 2019.</a></li><li><a class="nounderline abstract_t">Society for Maternal-Fetal Medicine (SMFM). Electronic address: <span class="__cf_email__" data-cfemail="2e5e5b4c5d6e5d43484300415c49">[email protected]</span>, Dugoff L, Norton ME, Kuller JA. The use of chromosomal microarray for prenatal diagnosis. Am J Obstet Gynecol 2016; 215:B2.</a></li><li><a class="nounderline abstract_t">Fiorentino F, Napoletano S, Caiazzo F, et al. Chromosomal microarray analysis as a first-line test in pregnancies with a priori low risk for the detection of submicroscopic chromosomal abnormalities. Eur J Hum Genet 2013; 21:725.</a></li><li><a class="nounderline abstract_t">Nicolaides KH, Snijders RJ, Gosden CM, et al. Ultrasonographically detectable markers of fetal chromosomal abnormalities. Lancet 1992; 340:704.</a></li><li><a class="nounderline abstract_t">Jansen FA, Blumenfeld YJ, Fisher A, et al. Array comparative genomic hybridization and fetal congenital heart defects: a systematic review and meta-analysis. Ultrasound Obstet Gynecol 2015; 45:27.</a></li><li><a class="nounderline abstract_t">Maya I, Singer A, Yonath H, et al. What have we learned from 691 prenatal chromosomal microarrays for ventricular septal defects? Acta Obstet Gynecol Scand 2020; 99:757.</a></li><li><a class="nounderline abstract_t">Management of Stillbirth: Obstetric Care Consensus No, 10. Obstet Gynecol 2020; 135:e110.</a></li><li><a class="nounderline abstract_t">Savage MS, Mourad MJ, Wapner RJ. Evolving applications of microarray analysis in prenatal diagnosis. Curr Opin Obstet Gynecol 2011; 23:103.</a></li><li><a class="nounderline abstract_t">Warburton D, Susser M, Stein Z, Kline J. Genetic and epidemiologic investigation of spontaneous abortion: relevance to clinical practice. Birth Defects Orig Artic Ser 1979; 15:127.</a></li><li><a class="nounderline abstract_t">Pauta M, Grande M, Rodriguez-Revenga L, et al. Added value of chromosomal microarray analysis over karyotyping in early pregnancy loss: systematic review and meta-analysis. Ultrasound Obstet Gynecol 2018; 51:453.</a></li><li><a class="nounderline abstract_t">Grande M, Jansen FA, Blumenfeld YJ, et al. Genomic microarray in fetuses with increased nuchal translucency and normal karyotype: a systematic review and meta-analysis. Ultrasound Obstet Gynecol 2015; 46:650.</a></li><li><a class="nounderline abstract_t">Egloff M, Hervé B, Quibel T, et al. Diagnostic yield of chromosomal microarray analysis in fetuses with isolated increased nuchal translucency: a French multicenter study. Ultrasound Obstet Gynecol 2018; 52:715.</a></li><li><a class="nounderline abstract_t">Leung TY, Au Yeung KC, Leung WC, et al. Prenatal diagnosis of pathogenic genomic imbalance in fetuses with increased nuchal translucency but normal karyotyping using chromosomal microarray. Hong Kong Med J 2019; 25 Suppl 5:30.</a></li><li><a class="nounderline abstract_t">Su L, Huang H, An G, et al. Clinical application of chromosomal microarray analysis in fetuses with increased nuchal translucency and normal karyotype. Mol Genet Genomic Med 2019; 7:e811.</a></li><li><a class="nounderline abstract_t">Xu LL, Li DZ. Diagnostic value of chromosomal microarray in fetuses with increased nuchal translucency. Ultrasound Obstet Gynecol 2019; 53:554.</a></li><li><a class="nounderline abstract_t">Chong HP, Hamilton S, Mone F, et al. Prenatal chromosomal microarray testing of fetuses with ultrasound structural anomalies: A prospective cohort study of over 1000 consecutive cases. Prenat Diagn 2019; 39:1064.</a></li><li><a class="nounderline abstract_t">Yatsenko SA, Peters DG, Saller DN, et al. Maternal cell-free DNA-based screening for fetal microdeletion and the importance of careful diagnostic follow-up. Genet Med 2015; 17:836.</a></li><li><a class="nounderline abstract_t">Borrell A, Grande M, Pauta M, et al. Chromosomal Microarray Analysis in Fetuses with Growth Restriction and Normal Karyotype: A Systematic Review and Meta-Analysis. Fetal Diagn Ther 2018; 44:1.</a></li><li><a class="nounderline abstract_t">Stern S, Hacohen N, Meiner V, et al. Universal chromosomal microarray analysis reveals high proportion of copy-number variants in low-risk pregnancies. Ultrasound Obstet Gynecol 2021; 57:813.</a></li><li><a class="nounderline abstract_t">Wapner RJ, Driscoll DA, Simpson JL. Integration of microarray technology into prenatal diagnosis: counselling issues generated during the NICHD clinical trial. Prenat Diagn 2012; 32:396.</a></li><li><a class="nounderline abstract_t">Mellis R, Oprych K, Scotchman E, et al. Diagnostic yield of exome sequencing for prenatal diagnosis of fetal structural anomalies: A systematic review and meta-analysis. Prenat Diagn 2022; 42:662.</a></li><li><a class="nounderline abstract_t">Van den Veyver IB, Chandler N, Wilkins-Haug LE, et al. International Society for Prenatal Diagnosis Updated Position Statement on the use of genome-wide sequencing for prenatal diagnosis. Prenat Diagn 2022; 42:796.</a></li></ol></div><div id="topicVersionRevision">Topic 16683 Version 41.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23215555" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Chromosomal microarray versus karyotyping for prenatal diagnosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23215556" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Karyotype versus microarray testing for genetic abnormalities after stillbirth.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22865506" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Experience with microarray-based comparative genomic hybridization for prenatal diagnosis in over 5000 pregnancies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22847778" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Detection rates of clinically significant genomic alterations by microarray analysis for specific anomalies detected by ultrasound.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23401365" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Prenatal diagnosis using combined quantitative fluorescent polymerase chain reaction and array comparative genomic hybridization analysis as a first-line test: results from over 1000 consecutive cases.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23983223" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : The clinical utility of microarray technologies applied to prenatal cytogenetics in the presence of a normal conventional karyotype: a review of the literature.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23897843" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Additional value of prenatal genomic array testing in fetuses with isolated structural ultrasound abnormalities and a normal karyotype: a systematic review of the literature.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23512800" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Use of prenatal chromosomal microarray: prospective cohort study and systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30549343" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Added value of chromosomal microarray analysis over conventional karyotyping in stillbirth work-up: systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28556491" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Frequency of submicroscopic chromosomal aberrations in pregnancies without increased risk for structural chromosomal aberrations: systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34242293" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Have maternal or paternal ages any impact on the prenatal incidence of genomic copy number variants associated with fetal structural anomalies?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29447663" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Prenatal diagnosis by chromosomal microarray analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23315237" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Increased paternal age and the influence on burden of genomic copy number variation in the general population.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/33355342" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Is there an association between paternal age and aneuploidy? Evidence from young donor oocyte-derived embryos: a systematic review and individual patient data meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23328890" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : American College of Medical Genetics and Genomics: standards and guidelines for documenting suspected consanguinity as an incidental finding of genomic testing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16118644" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Molecular cytogenetic analyses of breakpoints in apparently balanced reciprocal translocations carried by phenotypically normal individuals.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19248039" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : De novo balanced chromosome rearrangements in prenatal diagnosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1928105" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : De novo balanced chromosome rearrangements and extra marker chromosomes identified at prenatal diagnosis: clinical significance and distribution of breakpoints.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15635069" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : The complex nature of constitutional de novo apparently balanced translocations in patients presenting with abnormal phenotypes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17766364" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Cryptic deletions are a common finding in "balanced" reciprocal and complex chromosome rearrangements: a study of 59 patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21712853" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Balanced into array: genome-wide array analysis in 54 patients with an apparently balanced de novo chromosome rearrangement and a meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29805044" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Risks and Recommendations in Prenatally Detected De Novo Balanced Chromosomal Rearrangements from Assessment of Long-Term Outcomes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20658510" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Additional information from array comparative genomic hybridization technology over conventional karyotyping in prenatal diagnosis: a systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22467166" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Prenatal chromosomal microarray analysis in a diagnostic laboratory; experience with&gt;1000 cases and review of the literature.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22467169" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Genetic counselling and ethical issues with chromosome microarray analysis in prenatal testing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34012068" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34012069" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2021 update: a policy statement of the American College of Medical Genetics and Genomics (ACMG).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21681106" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : American College of Medical Genetics standards and guidelines for interpretation and reporting of postnatal constitutional copy number variants.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30915098" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Comparison of Efficiencies of Non-invasive Prenatal Testing, Karyotyping, and Chromosomal Micro-Array for Diagnosing Fetal Chromosomal Anomalies in the Second and Third Trimesters.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31123319" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Chromosomal microarray vs. NIPS: analysis of 5541 low-risk pregnancies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32198896" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Clinical utility of expanded non-invasive prenatal screening and chromosomal microarray analysis in high-risk pregnancy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27875474" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Committee Opinion No.682: Microarrays and Next-Generation Sequencing Technology: The Use of Advanced Genetic Diagnostic Tools in Obstetrics and Gynecology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27427470" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : The use of chromosomal microarray for prenatal diagnosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23211699" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Chromosomal microarray analysis as a first-line test in pregnancies with a priori low risk for the detection of submicroscopic chromosomal abnormalities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1355807" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Ultrasonographically detectable markers of fetal chromosomal abnormalities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25319878" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Array comparative genomic hybridization and fetal congenital heart defects: a systematic review and meta-analysis</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31424084" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : What have we learned from 691 prenatal chromosomal microarrays for ventricular septal defects?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32080052" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Management of Stillbirth: Obstetric Care Consensus No, 10.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21297472" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : Evolving applications of microarray analysis in prenatal diagnosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/508914" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Genetic and epidemiologic investigation of spontaneous abortion: relevance to clinical practice.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29055063" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Added value of chromosomal microarray analysis over karyotyping in early pregnancy loss: systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25900824" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Genomic microarray in fetuses with increased nuchal translucency and normal karyotype: a systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29027723" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : Diagnostic yield of chromosomal microarray analysis in fetuses with isolated increased nuchal translucency: a French multicenter study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31416984" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : Prenatal diagnosis of pathogenic genomic imbalance in fetuses with increased nuchal translucency but normal karyotyping using chromosomal microarray.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31209990" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : Clinical application of chromosomal microarray analysis in fetuses with increased nuchal translucency and normal karyotype.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30938479" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : Diagnostic value of chromosomal microarray in fetuses with increased nuchal translucency.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31393021" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Prenatal chromosomal microarray testing of fetuses with ultrasound structural anomalies: A prospective cohort study of over 1000 consecutive cases.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25569438" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : Maternal cell-free DNA-based screening for fetal microdeletion and the importance of careful diagnostic follow-up.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28889126" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : Chromosomal Microarray Analysis in Fetuses with Growth Restriction and Normal Karyotype: A Systematic Review and Meta-Analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32202684" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : Universal chromosomal microarray analysis reveals high proportion of copy-number variants in low-risk pregnancies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22467170" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : Integration of microarray technology into prenatal diagnosis: counselling issues generated during the NICHD clinical trial.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/35170059" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : Diagnostic yield of exome sequencing for prenatal diagnosis of fetal structural anomalies: A systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/35583085" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : International Society for Prenatal Diagnosis Updated Position Statement on the use of genome-wide sequencing for prenatal diagnosis.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
