<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Next-generation DNA sequencing (NGS): Principles and clinical applications</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Next-generation DNA sequencing (NGS): Principles and clinical applications</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Next-generation DNA sequencing (NGS): Principles and clinical applications</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Peter J Hulick, MD, MMSc, FACMG</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jennifer S Tirnauer, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Aug 10, 2022.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H747778294"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Technologies for sequencing DNA have improved dramatically, to the point that it has become practical to sequence an individual's entire genome. Next-generation sequencing (NGS) is a type of DNA sequencing technology that uses parallel sequencing of multiple small fragments of DNA to determine sequence. This "high-throughput" technology has allowed a dramatic increase in the speed (and a decrease in the cost) at which an individual's genome can be sequenced.</p><p>The ability to sequence an entire genome raises several challenging questions for the clinician, including the following:</p><p class="bulletIndent1"><span class="glyph">●</span>When should NGS be considered clinically?</p><p class="bulletIndent1"><span class="glyph">●</span>What is the best choice among several types of genetic testing available?</p><p class="bulletIndent1"><span class="glyph">●</span>What is the clinical significance of findings from sequencing of an entire genome?</p><p class="bulletIndent1"><span class="glyph">●</span>Which findings should be acted upon and/or conveyed to the patient?</p><p></p><p>This topic will review basic concepts and clinical uses of next-generation DNA sequencing (NGS). Genetic testing, counseling, and reporting of incidental findings from genome sequencing are discussed separately. (See  <a class="medical medical_review" href="/d/html/2901.html" rel="external">"Genetic testing"</a> and  <a class="medical medical_review" href="/d/html/96539.html" rel="external">"Secondary findings from genetic testing"</a>.)</p><p>Alternative methods for evaluating genetic and genomic disorders such as conventional sequencing, polymerase chain reaction (PCR), cytogenetics and fluorescence in situ hybridization (FISH), and gene expression profiling are also presented separately. (See  <a class="medical medical_review" href="/d/html/2904.html" rel="external">"Overview of pharmacogenomics"</a> and  <a class="medical medical_review" href="/d/html/2893.html" rel="external">"Tools for genetics and genomics: Cytogenetics and molecular genetics"</a> and  <a class="medical medical_review" href="/d/html/14602.html" rel="external">"Tools for genetics and genomics: Gene expression profiling"</a> and  <a class="medical medical_review" href="/d/html/14599.html" rel="external">"Personalized medicine"</a>.)</p><p class="headingAnchor" id="H22497283"><span class="h1">TERMINOLOGY AND EVOLUTION OF TECHNOLOGIES</span><span class="headingEndMark"> — </span>A large investment has been made in improving DNA sequencing technologies, to make them cheaper, faster, and more accurate. The following terms are used to distinguish sequencing methodologies:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Sanger sequencing</strong> – Manual or automated sequencing using the methods developed by Sanger, Maxam, and Gilbert is considered the "first" generation of DNA sequencing methods [<a href="#rid1">1</a>]. These types of sequencing are also commonly referred to as "conventional" or "traditional" sequencing. Sanger sequencing determines the sequence of large DNA fragments (up to approximately 500 to 900 bases), by collecting and aligning a series of different length products polymerized along the DNA template. The original Sanger method used radioactive markers for each nucleotide, and later adaptations have used fluorescently tagged versions.</p><p></p><p class="bulletIndent1">Sanger sequencing is used clinically when the sequence of a specific gene is being tested. As an example, conventional sequencing could be used to identify a mutation in factor IX in a patient with suspected hemophilia B, without examining the rest of the individual's genome. Sanger sequencing is preferred in this setting because mutations in the gene encoding factor IX are already known to cause the disease; gene mutations can be correlated with laboratory markers of the disease (eg, activated partial thromboplastin time [aPTT]); and mutations in other genes are exceedingly unlikely to cause hemophilia B. (See  <a class="medical medical_review" href="/d/html/1311.html" rel="external">"Genetics of hemophilia A and B"</a>.)</p><p></p><p class="bulletIndent1">In contrast, Sanger sequencing often cannot provide information about large portions of the genome (eg, multiple genes) at a practical cost and within a reasonable timeframe. One estimate predicted that sequencing an entire human genome using Sanger sequencing would take 60 years [<a href="#rid2">2</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Next-generation sequencing (NGS)</strong> – Next-generation sequencing (next-gen sequencing; NGS) uses sequencing of multiple DNA fragments, performed in parallel. This technology is also referred to by other terms (eg, short-read sequencing, high-throughput sequencing, deep sequencing, second-generation sequencing). In contrast to Sanger sequencing, the speed of sequencing and amounts of DNA sequence data generated with NGS are exponentially greater, and are produced at significantly reduced costs.</p><p></p><p class="bulletIndent1">Several "platforms" (ie, sequencing instruments and associated reagents) for NGS have been developed. Across NGS platforms, there is typically a sample preparation or "library preparation" step in which the patient's DNA, which serves as the template, is purified, amplified and fragmented, followed by physical isolation of DNA fragments by attachment to solid surfaces or small beads. Sequence data are generated on these small fragments, and the electronic results are computationally aligned against a "reference" genome or sequence (ie, a previously sequenced genome designated as a "normal" reference). An updated version of a commonly used human reference genome, GRCh38, was released by the National Center for Biotechnology Innovation (NCBI) on December 24, 2013.</p><p></p><p class="bulletIndent1">NGS is reserved for clinical scenarios in which it is considered useful to determine the sequences of multiple genes. (See <a class="local">'Clinical use of next-generation sequencing'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Third-generation sequencing</strong> – Third-generation sequencing (also known as long-read sequencing) uses parallel sequencing similar to NGS, third-generation sequencing uses single DNA molecules rather than amplified DNA as a template. Thus, third-generation sequencing potentially eliminates errors in DNA sequence introduced in the laboratory during the DNA amplification process.</p><p></p><p class="bulletIndent1">Third-generation methods are under development and generally are not clinically available. The technology of this approach is nearing the accuracy of short-read technology, which is used in NGS and is considered the clinical NGS gold standard. Third-generation sequencing allows for sequencing some of the most challenging areas of the genome (challenging due to the high percentage of guanine and cytosine bases, referred to as "GC content") and for providing better analysis of structural variation in the genome [<a href="#rid3">3</a>]. </p><p></p><p class="bulletIndent1">Additional terminology used to describe a variety of genetic and genomic concepts is presented separately. (See  <a class="medical medical_review" href="/d/html/2898.html" rel="external">"Genetics: Glossary of terms"</a>.)</p><p></p><p>In addition to evolving technical innovations, methods to optimize data interpretation continue to be studied, as large amounts of sequence data are becoming available before the clinical implications are known. (See <a class="local">'Interpretation'</a> below.)</p><p class="headingAnchor" id="H176018650"><span class="h1">TECHNICAL CONSIDERATIONS</span></p><p class="headingAnchor" id="H185155"><span class="h2">Source of DNA</span><span class="headingEndMark"> — </span>The starting material that provides a template for clinical NGS is double-stranded nuclear DNA. This can be obtained from a variety of cell types. In some cases, the DNA is further modified in the laboratory (eg, to remove non-coding regions when exome sequencing is performed). (See <a class="local">'Whole genome, exome, or gene panel'</a> below.)</p><p>DNA extracted from leukocytes in whole blood is a sterile source of DNA used for most clinical testing. When DNA is derived from non-sterile sources (eg, buccal swab, saliva sample), there is a potential risk for DNA sequence from bacterial flora to be confused for host sequence. However, non-sterile sources of DNA have the advantage of not requiring a blood draw, and less-invasive means for collecting DNA are increasingly used in large-scale genomic research studies. If needed, DNA can also be extracted from fixed tissues (eg, from formalin-fixed, paraffin-embedded pathology samples).</p><p>For patients who have undergone hematopoietic cell transplantation, leukocyte DNA from a blood sample is likely to contain donor rather than recipient genetic sequences. In this setting, DNA from a buccal swab, saliva, or hair follicles can be used [<a href="#rid4">4,5</a>]. Hair follicles are likely to be the most reliable source, because buccal and saliva samples have been reported to show chimerism for donor and recipient sequences [<a href="#rid6">6</a>].</p><p>Starting material DNA must be amplified prior to NGS to run the sequencing reactions. Amplification involves exponential in vitro replication of patient DNA using PCR-based methods. While the DNA polymerase rarely makes mistakes (eg, substituting the wrong nucleotide), the amplification step has the potential to introduce errors in sequence not present in the original DNA from the patient.</p><p>NGS methods can also be used to sequence RNA and mitochondrial DNA; however, the majority of known human disease genes reside in nuclear DNA; hence these sources of material are generally reserved for research studies or diagnosing specific rare disorders.</p><p>For research purposes, it may be informative to compare genome sequences using normal and diseased tissue (eg, a tumor) from the same individual. Such comparisons can help determine the timing during which mutations are acquired during cancer development, allowing an estimation of their relative contributions to carcinogenesis.</p><p class="headingAnchor" id="H22497411"><span class="h2">Whole genome, exome, or gene panel</span><span class="headingEndMark"> — </span>NGS can be used to sequence every nucleotide in an individual's DNA (ie, the whole genome), or limited to smaller portions of the genome such as the exome or a preselected subset of genes.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Whole genome sequencing</strong> – Whole genome sequencing is costlier than more limited sequencing, because the whole genome is equivalent to approximately 3.3 x 10<sup>9</sup> bases (3.3 gigabases [Gb]). Whole genome sequencing may become preferable to exome sequencing as cost decreases and more information about the role of non-coding DNA in human disease becomes available. Further, whole genome sequencing can be used to detect deletions and duplications typically detected only by array comparative genomic hybridization (aCGH) and thus is an opportunity to reduce the need for other supplemental testing. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Exome sequencing</strong> – The exome contains the portions of genes that encode proteins; it represents only 1.5 to 2.0 percent of the genome (ie, about 30 megabases [Mb]). The remaining (non-exomic) DNA consists of introns and regulatory regions that control other aspects of gene function such as splicing and gene expression levels.</p><p></p><p class="bulletIndent1">Exome sequencing is a reasonable approach for some clinical situations, because over 85 percent of known disease-causing mutations are found in exons. This approach substantially reduces cost and data storage requirements compared with whole genome sequencing. Exome sequencing also simplifies clinical reporting, because the significance of variants in exons is easier to interpret in most cases.</p><p></p><p class="bulletIndent1">The main disadvantage of exome sequencing over whole genome sequencing is that exome sequencing could potentially miss a pathogenic variant(s) in a non-coding region of the genome. Thus, whole genome sequencing may be used in selected cases if initial exome sequencing is not diagnostic.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Targeted gene panels</strong> – Gene panels provide sequence data for a limited subset of genes (typically 10 to 200 genes). Targeted gene panels are used in settings in which it would be appropriate to sequence many genes to make a diagnosis (eg, a Mendelian disorder for which the number of candidate genes is too large for traditional Sanger sequencing). As an example, targeted gene panels can be used to determine the genetic basis for hereditary early-onset and/or familial nonsyndromic hearing loss. Potential etiologies include pathogenic variants in over 60 genes; and screening for all of these individually would be impractical using traditional sequencing methods. Gene panels are becoming increasingly large (eg, &gt;1000 genes), particularly for neurologic indications such as ataxia and autism spectrum disorders.</p><p></p><p class="bulletIndent1">Targeted gene panels are also increasingly being used to evaluate potential hereditary cancer syndromes such as hereditary breast and ovarian cancer (HBOC) or hereditary colon cancer. In many cases, when testing of a single gene or a small number of genes is ordered, the laboratory will test the entire panel but only report on the genes that were ordered. In a series of 360 unselected patients with ovarian or fallopian tube cancer who were tested for abnormalities in a panel of tumor suppressor genes, 82 of the patients (23 percent) carried germline loss-of-function mutations in 1 of the 21 genes tested [<a href="#rid7">7</a>]. Of these 82 women, 25 (31 percent) had no family history of breast or ovarian cancer. In our experience, strength of the family history was not a reliable predictor for finding a pathogenic variant in a gene on further testing after initial <em>BRCA1</em> and <em>BRCA2</em> testing was negative [<a href="#rid8">8</a>]. Specific recommendations for HBOC are presented in detail separately. (See  <a class="medical medical_review" href="/d/html/753.html" rel="external">"Genetic testing and management of individuals at risk of hereditary breast and ovarian cancer syndromes"</a>.)</p><p></p><p class="bulletIndent1">Targeted gene panels may be preferable to exome sequencing due to the considerable cost advantage, the lower likelihood of identifying variants of unknown significance that are unrelated to the disease being evaluated, and a higher depth of coverage (number of independent "reads" of a region of DNA sequence) compared with whole exome sequencing. In practical terms, depth of coverage indicates how many times the DNA sequence is "proofread." Depth of coverage is important in helping to distinguish true SNPs or variation in the genome from errors introduced from the sequencing process. The utility of targeted gene panels over Sanger sequencing continues to increase as costs decline and as these gene panels offer a wide enough net of genes to assay for many conditions.</p><p></p><p class="bulletIndent1">The number of available panels also continues to increase, with validated panels available for hereditary cardiomyopathies, inherited cancer syndromes, lung diseases, ciliopathies, and other disorders in which molecular diagnosis is better facilitated by sequencing multiple genes known to be causative in the majority of cases. Additionally, the content of many disease-specific gene panels has been expanding to include genes that account for a smaller proportion of positive cases, and genes that have less available information on their mutation spectrum such as clinical outcomes and population-based data.</p><p></p><p class="bulletIndent1">A review of expanded gene panels in the testing for HBOC susceptibility noted that these panels may include genes for which the disease association and the degree of increased risk is unclear [<a href="#rid9">9</a>]. Consequently, some expanded gene panels may reveal variants that create dilemmas for medical management (eg, the appropriateness of prophylactic mastectomy may be substantially lower for a variant in a gene other than in <em>BRCA1</em> or <em>BRCA2</em>). The use of expanded panels may also increase the number of variants of unknown significance (VUS), for which data are very limited regarding the impact of mastectomy on clinical outcomes. Expanded gene panels are likely to be least beneficial in low-risk groups such as females in the general population, for whom some experts suggest offering <em>BRCA1</em> and <em>BRCA2</em> screening [<a href="#rid10">10</a>].</p><p></p><p class="bulletIndent1">For these reasons, caution should be exercised in choosing and interpreting results from expanded gene panels. The same expanded gene panel issues seen in HBOC testing (ie, inclusion of genes with limited data on the pathogenicity spectrum and/or questionable disease association) can be seen in other commonly-offered expanded gene panels, such as those for reproductive carrier screening, epilepsy, cardiomyopathy, and autism. </p><p></p><p class="bulletIndent1">Additional information about the relationship between genetic findings and disease phenotypes is available from <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fclinicalgenome.org%2F&amp;token=nbsXe3i4qfUea5RCPRHtOXAfmv8sxvNi1dzM5%2F2MCbBz%2Fjw073txTxlmmUrxvXA9&amp;TOPIC_ID=17039" target="_blank">ClinGen</a> [<a href="#rid11">11</a>]. (See <a class="local">'Variant databases'</a> below.)</p><p></p><p class="headingAnchor" id="H176018644"><span class="h2">Accuracy</span><span class="headingEndMark"> — </span>In general, one can expect usable data (adequate depth of coverage to provide confidence in the results) for over 92 to 95 percent of NGS sequence for whole genome and exome sequencing. Accuracy for targeted NGS gene panels is higher, since sequencing a smaller region of the genome allows for a greater degree of probe-template overlap (also called "probe tiling"). Sanger sequencing remains the "gold standard" for diagnosis based on gene sequencing, with &gt;99.99 percent accuracy reported for most genes sequenced.</p><p>Clinical laboratories (ie, those with Clinical Laboratory Improvement Amendments [CLIA] certification) generally perform Sanger sequencing to confirm any variant reported back to the ordering clinician as pathogenic, because of the greater accuracy of Sanger sequencing [<a href="#rid12">12,13</a>]. However given the continued improvement of NGS-based technology, the necessity of performing secondary validation with Sanger sequencing is being challenged [<a href="#rid14">14</a>]. (See <a class="local">'Interpretation'</a> below.)</p><p>Additional methods are used to limit the number of variants prioritized for validation and reporting, including the use of stringent bioinformatic filters and targeted sequencing in affected family members.</p><p>As noted below, re-analysis of prior sequencing data sometimes reveals a previously unrecognized diagnosis, such as reclassification of a variant of uncertain significance as either benign or pathogenic (see <a class="local">'Adults'</a> below). Many clinical laboratories offer a one-time "re-analysis of data" at a later time when the initial analysis of whole exome sequencing does not yield a diagnosis. Re-analysis of a whole exome or a genome sequence based on new insights into the human genome has the potential to impact care if it can be scaled.</p><p class="headingAnchor" id="H176019130"><span class="h2">Interpretation</span><span class="headingEndMark"> — </span>Clinical interpretation of data from NGS is more challenging than interpretation of traditional sequencing, because NGS provides data regarding "variants" in multiple genes, many of which are unexpected, outside the gene(s) of interest, and of unknown prognostic significance [<a href="#rid15">15</a>].</p><p>Results of NGS are generally reported as one of the following:</p><p class="bulletIndent1"><span class="glyph">●</span>"Pathogenic" – Pathogenic variants are variants previously reported in patients with disease and/or are strongly suspected of being pathogenic based on preclinical studies.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>"Likely pathogenic" – Likely pathogenic variants are those with sequence features that are likely to be implicated in disease pathogenesis but for which conclusive evidence of pathogenicity is not available.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>"Likely benign" – Likely benign variants are those for which weak data in the medical literature supporting pathogenicity may be available, but for which the majority of evidence suggests the effect of the variant is benign.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>"Benign" – Benign findings are genetic variants not predicted to alter gene expression or function.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>"Uncertain clinical significance" – Variants of uncertain significance (VUS; also called "variant of unknown significance" or "finding of unknown clinical significance") are variants that have some features suggestive of possible functional consequence, but for which there is insufficient evidence for either a pathogenic or benign role.</p><p></p><p>The clinician and patient must be aware that the likelihood of receiving a result of VUS is reasonably high because the clinical implications of many variants are unknown. As an example, a review on exome sequencing noted that an average sequence could identify over three million variants, of which the clinical implications might be uncharacterized for approximately 15 to 20 percent [<a href="#rid16">16</a>]. Thus clinicians ordering NGS testing and providing results of NGS to patients should receive proper training in how to discuss these issues. (See  <a class="medical medical_review" href="/d/html/96539.html" rel="external">"Secondary findings from genetic testing"</a> and  <a class="medical medical_review" href="/d/html/103304.html" rel="external">"Genetic counseling: Family history interpretation and risk assessment"</a>.)</p><p>Clinical NGS laboratories generally will be in agreement on their interpretation of a variant as pathogenic or benign. In contrast, significant laboratory-to-laboratory variability exists for the other categories. Consistency across laboratories continues to improve as more data (especially clinical outcomes data) become available for any given variant.</p><p>The challenges in interpreting NGS data are illustrated by following reports:</p><p class="bulletIndent1"><span class="glyph">●</span>A whole genome sequence analysis of an otherwise healthy individual with Charcot-Marie-Tooth (CMT) disease, in addition to facilitating the identification of the CMT-causing mutations, identified 159 mutations with known disease (or trait) associations, including 33 variants associated with cancers, 48 associated with complex diseases, and 21 associated with Mendelian (monogenic) diseases [<a href="#rid17">17</a>]. In the absence of significant family history for most of these conditions, the interpretation and management of these variants are unclear. However, longitudinal follow-up of the patient may be helpful as new data become available.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>A whole genome sequence analysis in a healthy 40-year-old man with a family history of coronary artery disease and sudden death in one of 27 relatives identified numerous variants associated with both rare and common conditions, including three mutations previously associated with sudden cardiac death [<a href="#rid18">18</a>]. Predictive models were applied for 32 conditions, with probability estimates suggesting substantially increased risk for the development of obesity, myocardial infarction (probability increased from 2.0 percent pre-testing to 8.9 percent post-testing), type 2 diabetes (probability increased from 27 to 54 percent), prostate cancer (probability increased from 16 to 23 percent), and depression, and a reduced risk for Alzheimer's disease (probability reduced from 9 to 1 percent). Based upon these results, in conjunction with the patient's strong family history of atherosclerosis, the patient's physician advised initiating lipid-lowering therapies despite a lipid profile for which treatment would not otherwise be recommended by established guidelines. Given the anecdotal nature of this report and the lack of prospective data, it is unclear whether any gains were achieved from the availability of the sequence data.</p><p></p><p>These cases illustrate the importance of discussing the likelihood of identifying unsuspected variants or variants of unknown significance prior to ordering testing, and the importance of having genetic counselors and other experienced subspecialists available to guide the patient once such variants are discovered. (See  <a class="medical medical_review" href="/d/html/96539.html" rel="external">"Secondary findings from genetic testing"</a> and  <a class="medical medical_review" href="/d/html/2901.html" rel="external">"Genetic testing", section on 'Obtaining informed consent'</a>.)</p><p class="headingAnchor" id="H1529255"><span class="h2">Variant databases</span><span class="headingEndMark"> — </span>The initial interpretation of the significance of a gene variant is performed by the genetics laboratory, and discussions of the implications generally are conducted by individuals with expertise in genetic counseling. However, it may be helpful (and important) for the ordering clinician to understand how pathogenicity is assessed, and in some cases to critically re-assess the interpretation. This is especially true in settings in which new information becomes available or when unique patient information (eg, personal or family history of disease) or values and preferences may alter clinical decision-making.</p><p>Interpretation of individual gene variants or mutations requires bioinformatics expertise. The process starts by checking the quality of the sequencing data. Sequence must be aligned to a normal reference genome. The interpretation of pathogenicity of gene variants incorporates several components: </p><p class="bulletIndent1"><span class="glyph">●</span>The degree of evolutionary conservation at the amino acid position, indicating a residue critical for protein function.</p><p class="bulletIndent1"><span class="glyph">●</span>The charge and hydrophobicity of the amino acid, indicating whether it is likely to be exposed on the protein surface or buried inside the protein. Information on clinical and disease manifestations associated with that specific variant from various databases.</p><p></p><p>A number of databases are available, including the National Institutes of Health (NIH) National Center for Biotechnology Information (NCBI) ClinVar effort (<a class="external" href="/external-redirect?target_url=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fclinvar%2F&amp;token=ODIqmPyH03K5IrNHL4oottVnYIHMebkQ75%2FQ3Mam14wGHZpSUYNdPv2Cp1abvDTd&amp;TOPIC_ID=17039" target="_blank">http://www.ncbi.nlm.nih.gov/clinvar/</a>).</p><p>ClinVar's mission is to foster communication and access to genomic information to establish evidence-based relationships between human disease and genomic variation. It aggregates data from multiple sources (eg, research and clinical laboratories, published literature, expert panels) to help resolve conflicting interpretations and provided necessary transparency and consistency necessary for variant calling and interpretation. Of note, however, none of the databases, including ClinVar, are completely comprehensive in gene and variant content.</p><p>Furthermore, variant assessment regarding pathogenicity is submitter-dependent, and conflicting assessments among multiple submitters are not uncommon. Another challenge in variant interpretation is the use of inconsistent nomenclature for variants across databases and published literature, resulting from the use of different gene transcripts (which are not always specified). Furthermore, variant assessments are not consistently shared among genetic testing laboratories; as a result, important information that could benefit patients can become siloed (ie, retained within the confines of individual laboratories rather than shared).</p><p>Ongoing efforts in ClinVar, ClinGen, Genome Connect, and other groups are actively addressing these issues [<a href="#rid19">19</a>]. Somatic variant databases in oncology are discussed below. (See <a class="local">'Cancer screening and management'</a> below.)</p><p>Even with the creation of large, publically available genomic databases and guidelines for assigning pathogenicity, interpreting genomic data and determining whether a change in an individual's DNA is pathogenic or normal human variation remains a challenge. Several studies have showed that concordance of variant calling by laboratories can be low, even amongst highly respected laboratories with significant expertise in the area being examined. There are multidiscipline efforts underway to help bring more consistency to variant calling [<a href="#rid20">20,21</a>]. This subject is discussed in more detail separately. (See  <a class="medical medical_review" href="/d/html/96539.html" rel="external">"Secondary findings from genetic testing"</a>.)</p><p class="headingAnchor" id="H16116013"><span class="h1">CLINICAL USE OF NEXT-GENERATION SEQUENCING</span></p><p class="headingAnchor" id="H1443116206"><span class="h2">Risks and benefits</span><span class="headingEndMark"> — </span>NGS is not always the most appropriate clinical genetic test. It is expensive, time-consuming, and often unnecessary for diagnosing genetic conditions for which the clinical evaluation has limited potential candidates to one or a few genes amenable to Sanger sequencing or other more traditional methods of detecting genetic defects. </p><p>However, it is appropriate to consider exome sequencing or targeted NGS gene panels when a large number of pathogenic genes need to be screened [<a href="#rid22">22,23</a>]. Similarly, exome sequencing or whole genome sequencing should be considered when a condition demonstrates high heritability in a family or is suspected to have a genetic basis, but the number of potential candidate genes is large, or responsible gene(s) are unknown. </p><p>Potential benefits of NGS were demonstrated in the 100,000 genomes project, in which whole genomes were sequenced in 4660 individuals with undiagnosed conditions (2183 probands plus 2477 affected and unaffected family members) [<a href="#rid24">24</a>]. The most common indications included neurologic disorders, ophthalmologic conditions, and tumor syndromes. Approximately three-fourths of the probands were adults. A genetic diagnosis was made in 25 percent of probands, and a variant of uncertain significance was identified in 10 percent. The diagnostic yield was higher for monogenic than for complex disorders (35 versus 11 percent). </p><p>In this study, benefits of genome sequencing included ending a diagnostic odyssey (often prolonged for years or decades), identifying new targets for research and drug development, and allowing testing of first-degree relatives, often children [<a href="#rid24">24</a>]. Of the 533 genetic diagnoses, 134 (25 percent) were considered to be of immediate clinical actionability, with several remarkable examples described. Only 11 (0.2 percent) were considered to be of no benefit. The clinical impact is significant, as over 80 percent of rare diseases are thought to have a genetic component, and these disorders affect as much as 6 percent of the population.</p><p>When used in diagnostic testing, gene sequencing or genome sequencing requires assessment of identified variants in the context of all of a patient's clinical features and the full spectrum of clinical manifestations that occur in the disease being considered. </p><p class="headingAnchor" id="H16116036"><span class="h2">Indications for NGS</span></p><p class="headingAnchor" id="H2312981551"><span class="h3">Diagnosis of complex diseases</span><span class="headingEndMark"> — </span>Consideration of NGS as a clinical tool (eg, for genetic diagnosis) is appropriate in individuals for whom sequencing of a single gene is unlikely to provide a diagnosis.</p><p>Examples include suspected genetic disorders in the following settings:</p><p class="bulletIndent1"><span class="glyph">●</span>One of many potential genes may be responsible, and/or the clinician does not know which gene(s) to test because many different genes cause the same phenotype (eg, due to genetic heterogeneity).</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Obvious candidate genes have been tested and were found to be normal. This is especially applicable when the percentage of disease attributed to these candidate genes is low, and other potentially causative genes for the disorder are thought to exist but have not yet been identified. Such analyses are often aided by comparison of NGS results from affected and unaffected family members.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>It would be less costly and more efficient to sequence the entire genome, exome, or gene panel than to sequence individual candidate genes sequentially.</p><p></p><p class="headingAnchor" id="H2870482126"><span class="h4">Children</span><span class="headingEndMark"> — </span>One of the most common medical indications for whole genome sequencing or whole exome sequencing is evaluation of severe intellectual disability or developmental delay believed to have a genetic etiology in a child with a negative initial evaluation. In some cases, evaluation of an affected child and both parents ("trio sequencing") is performed – especially when the inheritance pattern is dominant and a de novo mutation is suspected [<a href="#rid25">25,26</a>]. The value of NGS in this setting has been illustrated in several studies, in which the likelihood of reaching a molecular diagnosis is on the order of 25 percent [<a href="#rid25">25-29</a>]. NGS may thus be appropriate when an extensive evaluation including chromosomal microarray is negative for developmental delay with a suspected genetic etiology.</p><p>As examples:</p><p class="bulletIndent1"><span class="glyph">●</span>Exome sequencing was performed for 2000 individuals born with severe neurologic deficits unexplained by prior clinical evaluation; slightly fewer than half were younger than five years of age [<a href="#rid27">27,28</a>]. Molecular diagnosis was established in 504 of the patients (25 percent). Diagnosis was more likely in those with neurologic findings rather than anomalies of other organ systems; and the implicated genetic variant was more likely to be previously undescribed rather than a known variant. Almost one-third were due to genetic causes of disease discovered within the prior one to two years.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Exome sequencing was performed in 814 patients (half younger than five years of age) with a variety of unexplained syndromes, the most common of which were developmental delay in children and ataxia in adults [<a href="#rid25">25</a>]. Molecular diagnosis was established in 213 (26 percent). Diagnosis was more likely in patients who had trio sequencing rather than proband sequencing alone; patients younger than five years of age; and patients with retinal disorders (for which a larger fraction of disease genes may be known). Examples were presented in which identification of a variant previously not associated with disease coincided with publication of a case report demonstrating the association.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Whole genome sequencing was used to evaluate 50 children with severe intellectual disability who did not have a diagnosis after extensive genetic testing that included exome sequencing; a genetic diagnosis was made in 20 of these children (40 percent) [<a href="#rid29">29</a>]. This cohort was likely to have been enriched for de novo cases, since none of the children had a positive family history for intellectual disability. Rapid whole genome sequencing (rWGS) for critically ill newborns in neonatal and pediatric intensive care units when an underlying genetic etiology is suspected has been shown to improve clinical outcomes and to be cost effective [<a href="#rid30">30</a>].</p><p></p><p>Advances in methods and expansion of the information in databases that can be queried may improve diagnostic yield. This was demonstrated in a study involving a cohort of children with severe undiagnosed neurodevelopmental disorders and/or congenital anomalies, in which re-analysis of exome sequencing data four years after the initial analysis identified previously unrecognized genetic defects in an additional 182 individuals, raising the overall diagnostic yield from 27 to 40 percent [<a href="#rid31">31</a>]. (See  <a class="medical medical_review" href="/d/html/2940.html" rel="external">"Congenital anomalies: Approach to evaluation", section on 'Whole exome and whole genome sequencing'</a>.)</p><p>Diagnostic yield may be lower or higher in other clinical settings. In a study that used exome sequencing in 41 individuals with intellectual developmental disorder and metabolic abnormalities for which an underlying defect was not identified by previous extensive metabolic or genetic testing, whole exome sequencing helped to suggest nongenetic etiologies (eg, toxic or infectious exposures, autoimmune disorders) in four individuals and to identify a genetic diagnosis in 28 (68 percent) [<a href="#rid32">32</a>]. The high yield for a genetic diagnosis, relative to other whole exome sequencing studies, was likely due to the focus on patients with a high likelihood of having an enzyme defect that could be the result of a single gene defect, given the prevalence of autosomal recessive disorders with metabolic consequences. This highlights how the clinical scenario can have a significant impact on diagnostic yield and thus can drive decisions on how to incorporate NGS tests into clinical care.</p><p class="headingAnchor" id="H2159719047"><span class="h4">Adults</span><span class="headingEndMark"> — </span>NGS is also being incorporated into the National Institutes of Health "<a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.genome.gov%2FCurrent-NHGRI-Clinical-Studies%2FUndiagnosed-Diseases-Program-UDN&amp;token=uUhRqczo%2FC%2Bidu92LaED2TwzWNtdlqZGwMlP9ZpcQeHfUXQz5MtjR%2BaeHkaqExVugXcj9EArjVZ54Yq2ogwWIdb6YlX9FXL3O3vZVrFGTMfHp9%2FqHpiRMWw6g%2B%2BDpLAn&amp;TOPIC_ID=17039" target="_blank">Undiagnosed diseases program</a>,” which evaluates patients who have a longstanding medical condition that eludes diagnosis [<a href="#rid33">33</a>]. In some cases, molecular diagnoses may have been challenging prior to the availability of NGS if the phenotype was nonspecific (ie, intellectual disability without distinguishing syndromic features) and the candidate gene list too large for traditional Sanger sequencing to be practical.</p><p>This application of NGS was illustrated in a series of 1519 individuals with a variety of unexplained symptoms (neurologic, musculoskeletal, immunologic, rheumatologic, and others) who were referred to the Undiagnosed Disease Network (UDN, established in 2014) [<a href="#rid34">34</a>]. A total of 601 of these were accepted for evaluation, including 251 adults (median age of children, 8 years; median age of adults, 29 years). Approximately half had whole exome sequencing and half had whole genome sequencing; many of the latter group had previously undergone whole exome sequencing. Of the 382 individuals who had completed their evaluation at the time of publication, 132 (35 percent) received a diagnosis, the majority based on their sequencing results. Of these 132, the diagnosis for 11 (8 percent) was established based on the re-analysis of previously obtained sequencing data. Most of the diagnoses were recognized presentations of a known syndrome, but several were unusual presentations of known syndromes, or new syndromes related to either previously known or previously unknown genetic variants. Many of these diagnoses resulted in changes in medical care (eg, new treatments, revised monitoring, genetic counseling). This study highlights the discovery aspect of genome sequencing and the value of sequencing in adults.</p><p>NGS can also be used to diagnose genetic diseases if the mode of Mendelian inheritance is known and familial samples are also sequenced for comparison. The clinical availability of whole exome screening has increased and is a valuable tool for trying to end so-called "diagnostic odyssey" cases [<a href="#rid35">35</a>].</p><p class="headingAnchor" id="H3156709146"><span class="h3">Cancer screening and management</span><span class="headingEndMark"> — </span>For many types of cancer, the choices of screening, diagnostic testing, and therapy incorporate genomic information about the tumor (somatic changes), germline changes in inherited cancer genes (eg, <em>BRCA1</em> and <em>BRCA2</em>), and germline changes in genetic modifiers. Targeted gene panels have shown expanded usefulness across many cancer types, especially those for which more than one genetic variant may be responsible. </p><p>Multi-gene panels for certain inherited cancer syndromes based on National Comprehensive Cancer Network (NCCN) recommendations are becoming increasingly popular options for certain patients as the field moves away from single-gene testing to the panel approach [<a href="#rid36">36</a>]. This approach may be more efficient and cost-effective, given the decreasing cost associated with NGS technology and increasing indications for genetic testing.</p><p>In February of 2019, the American Society of Breast Surgeons recommended all patients with a diagnosis of breast cancer undergo germline genetic testing via a multigene panel that includes <em>BRCA1</em>, <em>BRCA2</em>, and <em>PALB2</em>. This guideline was based on an article that cited the limitations of NCCN guidelines for identifying patients who may harbor a pathogenic mutation in a clinically relevant breast cancer gene [<a href="#rid37">37</a>]. Criteria for genetic testing for hereditary breast and ovarian cancer gene variants is discussed separately. (See  <a class="medical medical_review" href="/d/html/744.html" rel="external">"Clinical features, diagnosis, and staging of newly diagnosed breast cancer", section on 'Postdiagnosis evaluation'</a> and  <a class="medical medical_review" href="/d/html/753.html" rel="external">"Genetic testing and management of individuals at risk of hereditary breast and ovarian cancer syndromes", section on 'Criteria for genetic risk evaluation'</a>.)</p><p>Examples of cancer gene panels used clinically to guide screening, preventive options, and cancer treatment include the following:</p><p class="bulletIndent1"><span class="glyph">●</span>Using a cancer gene panel that includes <em>BRCA1</em> and <em>BRCA2</em> if there is personal or family history of prostate and/or pancreatic cancer, even in the absence of breast or ovarian cancer. Other genes for which pathogenic variants may be associated increased breast cancer risk include <em>ATM</em>, <em>CDH1</em>, <em>CHEK2</em>, <em>NF1</em>, <em>PALB2</em>, and <em>TP53.</em> (See  <a class="medical medical_review" href="/d/html/753.html" rel="external">"Genetic testing and management of individuals at risk of hereditary breast and ovarian cancer syndromes", section on 'Criteria for genetic risk evaluation'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Screening for inherited causes of gastrointestinal cancers (eg, panels that include <em>APC</em>, <em>BMPR1A</em>, <em>EPCAM</em>, <em>MLH1</em>, <em>MSH2</em>, <em>MSH6</em>, <em>MUTYH</em>, <em>PMS2</em>, <em>PTEN</em>, <em>SMAD4</em>, <em>STK11</em>, <em>TP53</em>, <em>BLM</em>, <em>CHEK2</em>, <em>GALNT12</em>, <em>GREM1</em>, <em>POLD1</em>, and/or <em>POLE</em>). (See  <a class="medical medical_review" href="/d/html/7572.html" rel="external">"Screening for colorectal cancer in patients with a family history of colorectal cancer or advanced polyp"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Identification of familial acute leukemia syndromes. (See  <a class="medical medical_review" href="/d/html/93183.html" rel="external">"Familial disorders of acute leukemia and myelodysplastic syndromes", section on 'Diagnostic genetic testing'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Categorization of prognostic groups in acute myeloid leukemia. (See  <a class="medical medical_review" href="/d/html/4516.html" rel="external">"Acute myeloid leukemia: Risk factors and prognosis"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Classification of certain inherited bone marrow failure syndromes. (See  <a class="medical medical_review" href="/d/html/100663.html" rel="external">"Dyskeratosis congenita and other telomere biology disorders", section on 'Laboratory testing and bone marrow'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Analysis of tumor tissue or non-tumor tissue to identify genetic abnormalities that may be present in the germline and/or the tumor that could potentially match molecularly targeted therapies. As an example, a poly(ADP-ribose) polymerase (PARP) inhibitor could be considered in a patient with an identified <em>BRCA1</em> or <em>BRCA2</em> mutation. (See  <a class="medical medical_review" href="/d/html/14227.html" rel="external">"ER/PR negative, HER2-negative (triple-negative) breast cancer", section on 'Patients with previous exposure to chemotherapy'</a>.)</p><p></p><p>In 2017, the US Food and Drug Administration (FDA) approved two gene panel tests (MSK-IMPACT and F1CDx) for analyzing pathogenic variants in solid tumors; these tests can be used on formalin-fixed, paraffin-embedded (FFPE) tissue regardless of the primary organ from which the tumor arose [<a href="#rid38">38-40</a>]. These tests detect variations in the coding regions of over 400 genes (MSK-IMPACT) and over 300 genes (F1CDx), and can provide information about differences between tumor and adjacent noncancerous tissue and about genomic signatures such as microsatellite instability and tumor mutational burden. The indications for F1CDx include metastatic and recurrent cancer for a number of solid tumor types (eg, breast, colon, endometrial, lung, melanoma, pancreatic) as well as other solid tumors and tumors of unknown primary, for which genomic information would inform patient management [<a href="#rid41">41</a>].</p><p>The indications for MSK-IMPACT are not specified; they may include diagnostic or prognostic classification or therapy decisions (eg, use of a clinically approved therapy or for identification of candidates for participation in clinical trials).</p><p>The MSK-IMPACT study prospectively sequenced 10,000 tumors and matched normal tissue (in most cases, peripheral blood was the DNA source) [<a href="#rid42">42</a>]. This allowed the investigators to examine acquired genetic changes in tumors. The average depth of coverage (number of independent reads (see <a class="local">'Accuracy'</a> above)) was over 700-fold, and many of the alterations fell outside of the coverage regions of traditional "hot-spot" panels that focus on specific variants. Strong concordance was established with data from the Cancer Genome Atlas [<a href="#rid43">43</a>]. However, MSK-IMPACT identified more variants of potential clinical significance than would be expected from whole exome sequencing.</p><p>Examples of important findings from MSK-IMPACT and their clinical significance included the following:</p><p class="bulletIndent1"><span class="glyph">●</span>Actionable findings were identified in 3792 individuals (37 percent of the cohort).</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><em>TP53</em> mutations accounted for the highest percentage of pathogenic variants (over 10 percent of cases). MSK-IMPACT identified <em>TP53</em> mutations in 29 percent of patients with prostate cancer; the Cancer Genome Atlas only identified <em>TP53 </em>mutations in 7 percent.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The tumor types with the highest yields of actionable findings included gastrointestinal stromal tumors (in 76 percent); thyroid cancers (in 60 percent); breast cancers (in 57 percent); and melanomas (in 56 percent).</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Mismatch repair (MMR) abnormalities and microsatellite instability (MSI) were documented in approximately 100 individuals who had not been tested for MMR defects; some of these individuals received directed therapies that they may not have otherwise received.</p><p></p><p>Data from this study were used to enroll patients in therapeutic trials and in treatment decisions, highlighting the usefulness of the information.</p><p>This approach of determining therapy based on genetic abnormalities rather than tissue of origin is increasingly important as studies such as the NCI-MATCH attempt to advance precision medicine cancer treatment. In 2017, the monoclonal antibody <a class="drug drug_general" data-topicid="96957" href="/d/drug information/96957.html" rel="external">pembrolizumab</a> was approved by the FDA for the treatment of adult and pediatric patients with unresectable or metastatic solid tumors that have been identified as being MSI-high (MSI-H) or MMR deficient (dMMR) [<a href="#rid44">44</a>]. This is significant because it was the first time a chemotherapeutic agent was approved independent of the anatomical site of origin for a cancer.</p><p>Other available genetic tests in oncology use gene expression rather than gene sequencing to identify molecular signatures in tumors (eg, Oncotype Dx panels for breast and prostate cancer). Gene expression profiling determines the level to which a gene is transcribed, as opposed to variations in gene sequence. (See  <a class="medical medical_review" href="/d/html/94751.html" rel="external">"Molecular prognostic tests for prostate cancer", section on 'Tests based on molecular characteristics'</a> and  <a class="medical medical_review" href="/d/html/782.html" rel="external">"Prognostic and predictive factors in early, non-metastatic breast cancer", section on 'Receptor status'</a> and  <a class="medical medical_review" href="/d/html/126631.html" rel="external">"Deciding when to use adjuvant chemotherapy for hormone receptor-positive, HER2-negative breast cancer"</a>.)</p><p>Given the investment in precision medicine and personalized medicine initiatives in the treatment of cancer, tumor variant repositories and knowledge bases are increasingly becoming important. The Catalogue of Somatic Mutations in Cancer (<a class="external" href="/external-redirect?target_url=https%3A%2F%2Fcancer.sanger.ac.uk%2Fcosmic&amp;token=LwrTFAXVgPwLUgokgVwXDkYo%2Fg2izc9GWv0uCgpQaUU81qSiVy%2FL9f67517I9fsg&amp;TOPIC_ID=17039" target="_blank">COSMIC</a>) brings together data reviewed by expert curators from a number of sources including literature curation from genomic data uploaded for publication purposes, genomic profiles of cell lines used in cancer research, and data from other well-established databases such as The Cancer Genome Atlas (<a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.cancer.gov%2Fabout-nci%2Forganization%2Fccg%2Fresearch%2Fstructural-genomics%2Ftcga%2Fusing-tcga&amp;token=B5sIaYHZ5s5yCLnmiUgqXY%2FkGJ1FrZ8tNpf67MsR%2F0dxKLjvKxD7lF6oxJZ07hj3lAvYcKt8nkketNwrcB76ldOrXkp85tkYXXEJzqlApldZ3ucUXji9nalvMLU26aXk&amp;TOPIC_ID=17039" target="_blank">TCGA</a>) and the International Cancer Genome Consortium (<a class="external" href="/external-redirect?target_url=https%3A%2F%2Fdaco.icgc.org%2F&amp;token=TgsikILH6SYe4ay8ZWorl7minY%2FuuntFgNxOEOKUkXY%3D&amp;TOPIC_ID=17039" target="_blank">ICGC</a>). This combination of sources creates a rich reference resource that can aid both clinical applications and research in cancer genomics.</p><p class="headingAnchor" id="H16747088"><span class="h3">Diagnosis of infections</span><span class="headingEndMark"> — </span>In addition to diagnosing genetic disorders, NGS might be helpful in identifying an infectious pathogen when usual microbial or serologic testing is unrevealing or when faster diagnosis could improve outcomes [<a href="#rid45">45,46</a>]. This subject is discussed in detail in separate topic reviews. (See  <a class="medical medical_review" href="/d/html/1278.html" rel="external">"Molecular diagnosis of central nervous system infections"</a> and  <a class="medical medical_review" href="/d/html/2133.html" rel="external">"Detection of bacteremia: Blood cultures and other diagnostic tests", section on 'Organism identification'</a>.)</p><p>The role of NGS in identifying an elusive infection was demonstrated in a case report in which a 14-year-old boy developed unexplained fever and meningoencephalitis following travel to Puerto Rico; the boy had an underlying immunodeficiency syndrome [<a href="#rid47">47</a>]. An extensive infectious disease evaluation was unrevealing, and the patient's clinical status deteriorated. NGS identified a species of <em>Leptospira</em>, a pathogenic spirochetal organism acquired by exposure to contaminated water or soil, in the patient's cerebrospinal fluid (CSF) but not serum. Antibiotic therapy was initiated based on the NGS results, with clinical improvement. Subsequent testing confirmed the diagnosis. Additional studies have illustrated the advantages and disadvantages of genetic testing for infections, as discussed separately. (See  <a class="medical medical_review" href="/d/html/1278.html" rel="external">"Molecular diagnosis of central nervous system infections", section on 'Types of tests'</a>.) </p><p>NGS technology has played a critical role in several steps in vaccine development during the coronavirus disease 2019 (COVID-19) pandemic:</p><p class="bulletIndent1"><span class="glyph">●</span>Identification of viral sequence [<a href="#rid48">48,49</a>]</p><p class="bulletIndent1"><span class="glyph">●</span>Characterization of viral structure [<a href="#rid50">50</a>]</p><p class="bulletIndent1"><span class="glyph">●</span>Vaccine design [<a href="#rid51">51</a>] </p><p></p><p>Resulting vaccines are discussed separately. (See  <a class="medical medical_review" href="/d/html/129849.html" rel="external">"COVID-19: Vaccines"</a>.) </p><p class="headingAnchor" id="H1923136602"><span class="h3">Healthy people</span><span class="headingEndMark"> — </span>NGS technology is opening the door for proactive genetic screening and is being offered to healthy individuals to determine increased disease risks, pharmacogenomic variants, and nonmedical information (eg, ancestry). The Center for Disease Control and Prevention (CDC) has designated hereditary breast and ovarian cancer syndrome, Lynch Syndrome, and familial hypercholesterolemia as "Tier 1" genes, meaning that they have significant potential for positive impact on public health [<a href="#rid52">52</a>]. However, studies demonstrating improved outcomes from testing healthy people for these syndromes is lacking.</p><p>Clinical laboratories have started to offer genetic health screens in which a number of medically actionable genes are sequenced. The content of these panels typically include expanded versions of the conditions and genes listed in the American College of Medical Genetics and Genomics (ACMG) recommendations  (<a class="graphic graphic_table graphicRef97842" href="/d/graphic/97842.html" rel="external">table 1</a>) for the reporting of secondary findings in clinical exome and genome sequencing [<a href="#rid53">53</a>]. These genes and the rationale for their inclusion are discussed separately. (See  <a class="medical medical_review" href="/d/html/96539.html" rel="external">"Secondary findings from genetic testing", section on 'Decisions made by the laboratory'</a>.)</p><p>Research efforts are underway to determine the clinical value of screening healthy people with genomic sequencing. (See <a class="local">'Research'</a> below.)</p><p>It is important to distinguish whole genome- and whole exome-based sequencing from more targeted approaches such as SNP-based panels or targeted disease-specific NGS panels, which may have fewer secondary findings and may be associated with lower costs [<a href="#rid23">23</a>]. The costs associated with NGS panels continue to decrease, and this decrease has prompted the discussion of universal screening for assessing certain risk factors such as inherited cancer risk. As an example, the cost-effectiveness of such an approach using universal BRCA testing of all women older than 30 years was modeled in a 2015 analysis, which concluded that if the price of a BRCA test dropped below USD $250, this would lead to a cost of USD $53,000 per quality of life year (QALY) gained by preventing breast cancers, a cost on par with that of routine screening colonoscopy every 10 years beginning at age 50 years for colon cancer prevention [<a href="#rid54">54</a>]. NGS panels that include the <em>BRCA1</em> and <em>BRCA2</em> genes in addition to other inherited cancer genes (eg, <em>MSH</em> and <em>MLH</em> genes associated with Lynch syndrome) can be ordered clinically with self-pay prices in the range of $250 to $500. The cost-effectiveness of population screening using BRCA1/2-based NGS cancer panels continues to gain support [<a href="#rid55">55</a>]. Further explanation about SNPs and additional information about SNP-based panels are presented separately. (See  <a class="medical medical_review" href="/d/html/2893.html" rel="external">"Tools for genetics and genomics: Cytogenetics and molecular genetics", section on 'Detecting known mutations'</a> and  <a class="medical medical_review" href="/d/html/2900.html" rel="external">"Basic genetics concepts: DNA regulation and gene expression", section on 'Sequence variants'</a>.)</p><p class="headingAnchor" id="H130494"><span class="h2">Limitations</span><span class="headingEndMark"> — </span>NGS may not be as accurate as other methods for detecting specific types of mutations. As an example, detection of chromosomal copy number changes and/or large gains, losses, or translocations by NGS is problematic due to the short DNA sequence read lengths. These may result in failure to detect chromosomal deletions or insertions. Traditional Sanger sequencing shares some of the same limitations but, theoretically, to a lesser degree with its longer read lengths (approximately 1000 bases for Sanger sequencing versus approximately 100 to 200 bases for standard NGS, although some platforms are reporting longer read lengths). Research efforts to increase the sequencing read length in NGS platforms continues.</p><p>In general, when large chromosomal aberrations are suspected, alternative platforms are usually preferred over NGS such as comparative genomic hybridization (CGH) microarray, multiplex ligation-dependent probe amplification (MLPA), fluorescence in situ hybridization (FISH), or cytogenetics, with the ideal choice often dependent on the specific clinical condition being evaluated. These alternative methods are presented separately. (See  <a class="medical medical_review" href="/d/html/16647.html" rel="external">"Genomic disorders: An overview"</a>.)</p><p>Additional potential limitations such as cost, long turnaround time, and concerns about lack of insurance reimbursement, are gradually becoming less of a concern as clinical use of NGS increases. (See <a class="local">'Cost and turnaround time'</a> below and <a class="local">'Insurance reimbursement'</a> below.)</p><p class="headingAnchor" id="H27573643"><span class="h2">Genetic discrimination</span><span class="headingEndMark"> — </span>A common concern about genetic testing is the potential for inadequate protection of privacy of genetic information and associated impacts on employment and insurance coverage. This issue is discussed in detail separately. (See  <a class="medical medical_review" href="/d/html/2901.html" rel="external">"Genetic testing", section on 'Genetic discrimination'</a>.)</p><p class="headingAnchor" id="H176019301"><span class="h2">Disclosure of findings from genome sequencing</span><span class="headingEndMark"> — </span>Genome sequencing may lead to incidental findings with potential clinical importance. Issues of whether and how to disclose such findings, and which findings to report to the individual being tested, as well as recommendations from the American College of Medical Genetics (ACMG), revised in 2021 [<a href="#rid56">56</a>], regarding which results should be disclosed, are discussed in detail separately. (See  <a class="medical medical_review" href="/d/html/96539.html" rel="external">"Secondary findings from genetic testing"</a>.)</p><p>Historically, individual genomic reports provided directly from companies to consumers often included many variants unlikely to be clinically pathogenic. Because a genotyping chip rather than NGS platform was generally used, most variants were located in non-coding regions of DNA and occurred at high frequency in the general population; these variants were unlikely to have any significant predictive value. Often such variants were reported at higher frequencies than the lifetime disease risk observed in the population, further discounting their likelihood of clinical significance. Issues related to direct-to-consumer genetic testing are discussed in detail separately. (See  <a class="medical medical_review" href="/d/html/14599.html" rel="external">"Personalized medicine", section on 'Direct-to-consumer testing'</a>.)</p><p class="headingAnchor" id="H594754664"><span class="h1">RESEARCH</span><span class="headingEndMark"> — </span>In contrast to clinical indications, potential uses of NGS as a research tool are extensive. Examples include identification of new genes involved in unexplained syndromes (which may be reported in the clinical setting if they could possibly explain a patient's disease-associated findings) and determination of genetic changes during the development of acute myeloid leukemia (AML) and other cancers [<a href="#rid57">57</a>]. These findings may ultimately lead to breakthroughs in understanding disease pathogenesis, and to the development of additional diagnostic testing and/or management strategies for these diseases. (See  <a class="medical medical_review" href="/d/html/4502.html" rel="external">"Acute myeloid leukemia: Molecular genetics"</a>.)</p><p>Comparisons of genome sequencing across populations of individuals may inform research into disease mechanisms or might help to identify specific variants that might be of value for targeted testing or screening healthy people [<a href="#rid58">58</a>]. (See <a class="local">'Healthy people'</a> above.)</p><p class="bulletIndent1"><span class="glyph">●</span>The All of Us Research Program (<a class="external" href="/external-redirect?target_url=http%3A%2F%2Fallofus.nih.gov%2F&amp;token=QisoLZBn7YwpGy7PI9XPPLlEFkM36LxCB%2Fd%2FlFiy9as%3D&amp;TOPIC_ID=17039" target="_blank">https://allofus.nih.gov/</a>) is a major effort under the National Institutes of Health (NIH) Precision Medicine Initiative (PMI) to establish a national research cohort of over a million individuals that combines genomic data and patient surveys. The goal is to better understand how biology, environment, and lifestyle intersect to influence health. Genomic data from this cohort will be obtained using NGS technology. Enrollment began in mid-2017.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>In the United Kingdom, the Newborn Genomes Programme is evaluating whole genome sequencing in up to 200,000 newborn babies [<a href="#rid59">59</a>]. The focus will be on gene variants associated with rare diseases that can present in childhood. Genes such as <em>BRCA1 </em>and <em>BRCA2</em>, which are associated with adult-onset conditions, will not be evaluated. Concerns will need to be addressed related to the inability of newborns to provide informed consent and the risks and harms associated with false-positive or false-negative results. Rapid whole genome sequencing in acutely ill children when a genetic etiology is possible improves outcomes and is cost efficient, but whether whole genome sequencing (rapid or routine) is cost effective and improves outcomes when implemented for all newborns is uncertain [<a href="#rid30">30</a>].</p><p></p><p>Research is also evaluating the best way to provide genetic information to healthy people. As an example, in a 2017 trial, 100 healthy adults were randomly assigned to receive a family history report alone or in combination with a whole genome sequencing report [<a href="#rid60">60</a>]. Approximately 1 in 5 assigned to genome sequencing were found to have a monogenic disease risk finding, and approximately 1 in 25 had a new clinical diagnosis. All individuals assigned to genome sequencing also received findings related to carrier status, pharmacogenomic information related to five drugs, and risk predictions for eight cardiometabolic traits. Outcomes in both arms included new advice from the primary care physician, changes in health behaviors (typically in diet or exercise), and financial costs. Genome sequencing did not appear to increase overall anxiety, and an expert panel judged the majority of the primary care physicians' responses to be appropriate. These findings do not support the routine use of whole genome sequencing in healthy people. However, they do demonstrate how providers who are not geneticists may be able to manage results from genomics-based testing and challenge the idea that genetic information is too complex to handle at the primary care level, which will be important as genetic information becomes more ubiquitous in medical care and initial management decisions occur in the primary care setting.</p><p class="headingAnchor" id="H176019436"><span class="h1">PRACTICAL ISSUES</span></p><p class="headingAnchor" id="H176019403"><span class="h2">Where to order</span><span class="headingEndMark"> — </span>Considerations in selecting an NGS service include cost, turn-around time, quality of data interpretation, and method of results reporting. Many university and commercial laboratories in the United States offer Clinical Laboratory Improvement Amendments (CLIA)-certified NGS. The increased availability of CLIA-certified NGS tests has been spurred in part by continued decreased costs and the US Supreme Court's decision to overturn some of the patents held on <em>BRCA1</em> and <em>BRCA2</em> genetic testing [<a href="#rid61">61</a>]. Specific resources are generally available through the local academic institution, but direct access between commercial testing laboratories and healthcare providers through utilization of prepackaged test kits is common. Most clinical NGS testing uses targeted gene panels and exome sequencing. An exception is clinical whole genome sequencing offered through Illumina.</p><p class="headingAnchor" id="H176019409"><span class="h2">Cost and turnaround time</span><span class="headingEndMark"> — </span>The typical cost of clinical NGS ranges from several hundred to thousands of dollars. Factors that affect cost include the type of genome analyzed (eg, somatic tumor plus germline); whether a whole genome, exome, or gene panel is requested; the requested turnaround time (standard versus expedited); whether a single affected proband or additional family member(s) are tested; and the extent of sequence interpretation. The price barrier continues to drop, particularly where competition between laboratories is intensifying. Inherited cancer panels are now available for under USD $250 if a patient selects an option to pay the company directly, in advance. There is an ongoing shift towards more transparent pricing and out-of-pocket costs by the larger commercial NGS testing laboratories. (See <a class="local">'Insurance reimbursement'</a> below.)</p><p>Standard turnaround time for NGS is generally 3 to 12 weeks, but may be longer for more involved, complex cases.</p><p class="headingAnchor" id="H176019415"><span class="h2">Insurance reimbursement</span><span class="headingEndMark"> — </span>Insurance reimbursement for NGS is a complex issue. Many commercial laboratories offer services to patients and providers to investigate insurance coverage if a patent prefers to have the laboratory bill the insurance company directly (rather than pay out-of-pocket). In a study that reported on findings of whole exome sequencing for 250 patients (mostly children) with neurologic phenotypes found that insurance coverage was similar to other genetic testing (eg, reimbursement for the majority of tests) [<a href="#rid27">27</a>]. Of note, the individuals in this study had relatively major neurologic phenotypes, and all had undergone prior genetic testing, often extensive.</p><p>It is also common for insurance companies to accept appeals with letters of medical necessity justifying the need for NGS. These letters should provide an explanation for how the results of NGS will alter the clinical evaluation, diagnosis, treatment plan, or prognosis. For some genetic tests such as inherited cancer panels, national guidelines provide advice on the indications for genetic testing, which is typically covered by insurance when indicated. Many insurance companies and Medicare use the National Comprehensive Cancer Network (NCCN) guidelines as a basis for coverage criteria and decisions [<a href="#rid36">36</a>].</p><p class="headingAnchor" id="H997592"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Definitions</strong> – Next-generation sequencing (NGS) is a DNA sequencing technology that uses parallel sequencing of multiple small DNA fragments to determine sequence. This technology has allowed a dramatic increase in the speed (and a decrease in the cost) at which an individual's genome can be sequenced. (See <a class="local">'Terminology and evolution of technologies'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Starting material</strong> – NGS can be performed on double stranded DNA from a variety of sources. DNA extracted from leukocytes in whole blood is used for most clinical testing. (See <a class="local">'Source of DNA'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Genome, exome, or panel</strong> – NGS can be used to sequence an individual's whole genome or smaller portions of the genome such as the exome or a preselected subset of genes. The general trend is towards increased use of gene panels; there is also greater use of whole exome testing. Studies are examining the clinical usefulness and value to primary care providers. (See <a class="local">'Whole genome, exome, or gene panel'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Data quality</strong> – Sanger sequencing is used to confirm the presence of specific variants identified by NGS in clinical settings, because accuracy of traditional sequencing methods is greater, although this practice is being challenged. The interpretation of pathogenicity (eg, pathogenic versus variant of unknown significance [VUS]) can differ by laboratory. It may be helpful (and important) for the ordering clinician to understand how pathogenicity is assessed, and in some cases to critically reassess the interpretation. (See <a class="local">'Accuracy'</a> above and <a class="local">'Interpretation'</a> above and <a class="local">'Variant databases'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Uses</strong> – NGS may be appropriate for diagnosing suspected genetic disorders when sequencing of a single gene is unlikely to provide a diagnosis. Examples include the following settings (see <a class="local">'Indications for NGS'</a> above):</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>One of several potential genes may be responsible</p><p class="bulletIndent2"><span class="glyph">•</span>Obvious candidate genes have been tested and found to be normal</p><p class="bulletIndent2"><span class="glyph">•</span>The cost of NGS would be less than that of sequencing individual candidate genes sequentially</p><p></p><p class="bulletIndent1">NGS-based gene panels are used clinically in certain hematologic malignancies, and the first gene panel tests for solid tumors were approved in 2017. Other uses such as diagnosis of infections and screening of healthy people remain investigational.</p><p></p><p class="bulletIndent1">There is growing uptake of screening healthy individuals with NGS-based gene panels; however, evidence to support this practice is lacking. (See <a class="local">'Healthy people'</a> above.) </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Risks</strong> – Concerns regarding consent for NGS testing, genetic discrimination, and disclosure of incidental findings from NGS are discussed separately. (See  <a class="medical medical_review" href="/d/html/2901.html" rel="external">"Genetic testing"</a> and  <a class="medical medical_review" href="/d/html/96539.html" rel="external">"Secondary findings from genetic testing"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Cost and turnaround time</strong> – Many university and commercial laboratories in the United States offer NGS-based tests, the most common application being targeted gene panels for a specific clinical indication (eg, breast and ovarian cancer risk, cardiomyopathy, developmental delay). The cost may be as low as approximately USD $250 to up to several thousand dollars, depending on the type and amount of the genome analyzed as well as the condition being tested. Standard turnaround time is 3 to 12 weeks. Insurance reimbursement may be similar to other genetic testing in individuals with major clinical phenotypes for whom previous testing was unrevealing. (See <a class="local">'Practical issues'</a> above.)</p><p></p><p class="headingAnchor" id="H1180161306"><span class="h1">ACKNOWLEDGMENT</span><span class="headingEndMark"> — </span>The UpToDate editorial staff acknowledges Joseph V Thakuria, MD, MMSc, who contributed to earlier versions of this topic review.</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Rizzo JM, Buck MJ. Key principles and clinical applications of "next-generation" DNA sequencing. Cancer Prev Res (Phila) 2012; 5:887.</a></li><li><a class="nounderline abstract_t">Bennett ST, Barnes C, Cox A, et al. Toward the 1,000 dollars human genome. Pharmacogenomics 2005; 6:373.</a></li><li><a class="nounderline abstract_t">Logsdon GA, Vollger MR, Eichler EE. Long-read human genome sequencing and its applications. Nat Rev Genet 2020; 21:597.</a></li><li><a class="nounderline abstract_t">Hong YC, Liu HM, Chen PS, et al. Hair follicle: a reliable source of recipient origin after allogeneic hematopoietic stem cell transplantation. Bone Marrow Transplant 2007; 40:871.</a></li><li><a class="nounderline abstract_t">Thiede C, Prange-Krex G, Freiberg-Richter J, et al. Buccal swabs but not mouthwash samples can be used to obtain pretransplant DNA fingerprints from recipients of allogeneic bone marrow transplants. Bone Marrow Transplant 2000; 25:575.</a></li><li><a class="nounderline abstract_t">Rennert H, Leonard DG, Cushing M, et al. Avoiding pitfalls in bone marrow engraftment analysis: a case study highlighting the weakness of using buccal cells for determining a patient's constitutional genotype after hematopoietic stem cell transplantation. Cytotherapy 2013; 15:391.</a></li><li><a class="nounderline abstract_t">Walsh T, Casadei S, Lee MK, et al. Mutations in 12 genes for inherited ovarian, fallopian tube, and peritoneal carcinoma identified by massively parallel sequencing. Proc Natl Acad Sci U S A 2011; 108:18032.</a></li><li><a class="nounderline abstract_t">Selkirk CG, Vogel KJ, Newlin AC, et al. Cancer genetic testing panels for inherited cancer susceptibility: the clinical experience of a large adult genetics practice. Fam Cancer 2014; 13:527.</a></li><li><a class="nounderline abstract_t">Easton DF, Pharoah PD, Antoniou AC, et al. Gene-panel sequencing and the prediction of breast-cancer risk. N Engl J Med 2015; 372:2243.</a></li><li><a class="nounderline abstract_t">King MC, Levy-Lahad E, Lahad A. Population-based screening for BRCA1 and BRCA2: 2014 Lasker Award. JAMA 2014; 312:1091.</a></li><li class="breakAll">https://clinicalgenome.org/ (Accessed on August 20, 2019).</li><li><a class="nounderline abstract_t">Rehm HL, Bale SJ, Bayrak-Toydemir P, et al. ACMG clinical laboratory standards for next-generation sequencing. Genet Med 2013; 15:733.</a></li><li><a class="nounderline abstract_t">Strom SP, Lee H, Das K, et al. Assessing the necessity of confirmatory testing for exome-sequencing results in a clinical molecular diagnostic laboratory. Genet Med 2014; 16:510.</a></li><li><a class="nounderline abstract_t">Beck TF, Mullikin JC, NISC Comparative Sequencing Program, Biesecker LG. Systematic Evaluation of Sanger Validation of Next-Generation Sequencing Variants. Clin Chem 2016; 62:647.</a></li><li><a class="nounderline abstract_t">Taylor JC, Martin HC, Lise S, et al. Factors influencing success of clinical genome sequencing across a broad spectrum of disorders. Nat Genet 2015; 47:717.</a></li><li><a class="nounderline abstract_t">Bamshad MJ, Ng SB, Bigham AW, et al. Exome sequencing as a tool for Mendelian disease gene discovery. Nat Rev Genet 2011; 12:745.</a></li><li><a class="nounderline abstract_t">Lupski JR, Reid JG, Gonzaga-Jauregui C, et al. Whole-genome sequencing in a patient with Charcot-Marie-Tooth neuropathy. N Engl J Med 2010; 362:1181.</a></li><li><a class="nounderline abstract_t">Ashley EA, Butte AJ, Wheeler MT, et al. Clinical assessment incorporating a personal genome. Lancet 2010; 375:1525.</a></li><li><a class="nounderline abstract_t">Rehm HL, Berg JS, Brooks LD, et al. ClinGen--the Clinical Genome Resource. N Engl J Med 2015; 372:2235.</a></li><li><a class="nounderline abstract_t">Van Driest SL, Wells QS, Stallings S, et al. Association of Arrhythmia-Related Genetic Variants With Phenotypes Documented in Electronic Medical Records. JAMA 2016; 315:47.</a></li><li><a class="nounderline abstract_t">Amendola LM, Jarvik GP, Leo MC, et al. Performance of ACMG-AMP Variant-Interpretation Guidelines among Nine Laboratories in the Clinical Sequencing Exploratory Research Consortium. Am J Hum Genet 2016; 98:1067.</a></li><li><a class="nounderline abstract_t">Biesecker LG, Green RC. Diagnostic clinical genome and exome sequencing. N Engl J Med 2014; 370:2418.</a></li><li><a class="nounderline abstract_t">Adams DR, Eng CM. Next-Generation Sequencing to Diagnose Suspected Genetic Disorders. N Engl J Med 2018; 379:1353.</a></li><li><a class="nounderline abstract_t">100,000 Genomes Project Pilot Investigators, Smedley D, Smith KR, et al. 100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care - Preliminary Report. N Engl J Med 2021; 385:1868.</a></li><li><a class="nounderline abstract_t">Lee H, Deignan JL, Dorrani N, et al. Clinical exome sequencing for genetic identification of rare Mendelian disorders. JAMA 2014; 312:1880.</a></li><li><a class="nounderline abstract_t">Wright CF, Fitzgerald TW, Jones WD, et al. Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data. Lancet 2015; 385:1305.</a></li><li><a class="nounderline abstract_t">Yang Y, Muzny DM, Reid JG, et al. Clinical whole-exome sequencing for the diagnosis of mendelian disorders. N Engl J Med 2013; 369:1502.</a></li><li><a class="nounderline abstract_t">Yang Y, Muzny DM, Xia F, et al. Molecular findings among patients referred for clinical whole-exome sequencing. JAMA 2014; 312:1870.</a></li><li><a class="nounderline abstract_t">Gilissen C, Hehir-Kwa JY, Thung DT, et al. Genome sequencing identifies major causes of severe intellectual disability. Nature 2014; 511:344.</a></li><li><a class="nounderline abstract_t">Dimmock D, Caylor S, Waldman B, et al. Project Baby Bear: Rapid precision care incorporating rWGS in 5 California children's hospitals demonstrates improved clinical outcomes and reduced costs of care. Am J Hum Genet 2021; 108:1231.</a></li><li><a class="nounderline abstract_t">Wright CF, McRae JF, Clayton S, et al. Making new genetic diagnoses with old data: iterative reanalysis and reporting from genome-wide data in 1,133 families with developmental disorders. Genet Med 2018; 20:1216.</a></li><li><a class="nounderline abstract_t">Tarailo-Graovac M, Shyr C, Ross CJ, et al. Exome Sequencing and the Management of Neurometabolic Disorders. N Engl J Med 2016; 374:2246.</a></li><li><a class="nounderline abstract_t">Kuehn BM. NIH's Undiagnosed Diseases Program expands: 6 new sites offer potential answers to more patients. JAMA 2014; 312:587.</a></li><li><a class="nounderline abstract_t">Splinter K, Adams DR, Bacino CA, et al. Effect of Genetic Diagnosis on Patients with Previously Undiagnosed Disease. N Engl J Med 2018; 379:2131.</a></li><li><a class="nounderline abstract_t">Lazaridis KN, Schahl KA, Cousin MA, et al. Outcome of Whole Exome Sequencing for Diagnostic Odyssey Cases of an Individualized Medicine Clinic: The Mayo Clinic Experience. Mayo Clin Proc 2016; 91:297.</a></li><li class="breakAll">https://www.nccn.org/professionals/physician_gls/default.aspx (Accessed on May 10, 2019).</li><li><a class="nounderline abstract_t">Beitsch PD, Whitworth PW, Hughes K, et al. Underdiagnosis of Hereditary Breast Cancer: Are Genetic Testing Guidelines a Tool or an Obstacle? J Clin Oncol 2019; 37:453.</a></li><li class="breakAll">FDA unveils a streamlined path for the authorization of tumor profiling tests alongside its latest product action. Available at: https://www.fda.gov/NewsEvents/Newsroom/PressAnnouncements/ucm585347.htm (Accessed on November 16, 2017).</li><li class="breakAll">Evaluation of automatic class III designation for MSK-IMPACT (Integrated Mutation Profiling of Actionable Cancer Targets). Available at: https://www.accessdata.fda.gov/cdrh_docs/reviews/DEN170058.pdf (Accessed on November 16, 2017).</li><li class="breakAll">FDA announces approval, CMS proposes coverage of first breakthrough-designated test to detect extensive number of cancer biomarkers. Available at: https://www.cms.gov/Newsroom/MediaReleaseDatabase/Press-releases/2017-Press-releases-items/2017-11-30-2.html (Accessed on December 04, 2017).</li><li class="breakAll">https://www.cms.gov/Medicare/Coverage/DeterminationProcess/downloads/id290.pdf (Accessed on December 04, 2017).</li><li><a class="nounderline abstract_t">Zehir A, Benayed R, Shah RH, et al. Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients. Nat Med 2017; 23:703.</a></li><li class="breakAll">https://cancergenome.nih.gov/ (Accessed on February 20, 2018).</li><li class="breakAll">https://www.accessdata.fda.gov/drugsatfda_docs/label/2017/125514s024lbl.pdf (Accessed on November 27, 2017).</li><li><a class="nounderline abstract_t">Sintchenko V, Holmes EC. The role of pathogen genomics in assessing disease transmission. BMJ 2015; 350:h1314.</a></li><li><a class="nounderline abstract_t">Armstrong GL, MacCannell DR, Taylor J, et al. Pathogen Genomics in Public Health. N Engl J Med 2019; 381:2569.</a></li><li><a class="nounderline abstract_t">Wilson MR, Naccache SN, Samayoa E, et al. Actionable diagnosis of neuroleptospirosis by next-generation sequencing. N Engl J Med 2014; 370:2408.</a></li><li><a class="nounderline abstract_t">Zhu N, Zhang D, Wang W, et al. A Novel Coronavirus from Patients with Pneumonia in China, 2019. N Engl J Med 2020; 382:727.</a></li><li><a class="nounderline abstract_t">Corman VM, Landt O, Kaiser M, et al. Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR. Euro Surveill 2020; 25.</a></li><li><a class="nounderline abstract_t">Dai W, Zhang B, Jiang XM, et al. Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease. Science 2020; 368:1331.</a></li><li><a class="nounderline abstract_t">Kames J, Holcomb DD, Kimchi O, et al. Sequence analysis of SARS-CoV-2 genome reveals features important for vaccine design. Sci Rep 2020; 10:15643.</a></li><li class="breakAll">https://www.cdc.gov/genomics/implementation/toolkit/tier1.htm (Accessed on June 14, 2019).</li><li><a class="nounderline abstract_t">Kalia SS, Adelman K, Bale SJ, et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics. Genet Med 2017; 19:249.</a></li><li><a class="nounderline abstract_t">Long EF, Ganz PA. Cost-effectiveness of Universal BRCA1/2 Screening: Evidence-Based Decision Making. JAMA Oncol 2015; 1:1217.</a></li><li><a class="nounderline abstract_t">Manchanda R, Patel S, Gordeev VS, et al. Cost-effectiveness of Population-Based BRCA1, BRCA2, RAD51C, RAD51D, BRIP1, PALB2 Mutation Testing in Unselected General Population Women. J Natl Cancer Inst 2018; 110:714.</a></li><li><a class="nounderline abstract_t">Miller DT, Lee K, Chung WK, et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2021; 23:1381.</a></li><li><a class="nounderline abstract_t">Waddell N, Pajic M, Patch AM, et al. Whole genomes redefine the mutational landscape of pancreatic cancer. Nature 2015; 518:495.</a></li><li><a class="nounderline abstract_t">Goldfeder RL, Wall DP, Khoury MJ, et al. Human Genome Sequencing at the Population Scale: A Primer on High-Throughput DNA Sequencing and Analysis. Am J Epidemiol 2017; 186:1000.</a></li><li><a class="nounderline abstract_t">Burki T. UK explores whole-genome sequencing for newborn babies. Lancet 2022; 400:260.</a></li><li><a class="nounderline abstract_t">Vassy JL, Christensen KD, Schonman EF, et al. The Impact of Whole-Genome Sequencing on the Primary Care and Outcomes of Healthy Adult Patients: A Pilot Randomized Trial. Ann Intern Med 2017; 167:159.</a></li><li class="breakAll">Association for Molecular Pathology v. Myriad Genetics, Inc. No. 12-398. 569 U.S ___ (2013). Text available at: http://www.supremecourt.gov/opinions/12pdf/12-398_1b7d.pdf</li></ol></div><div id="topicVersionRevision">Topic 17039 Version 41.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22617168" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Key principles and clinical applications of "next-generation" DNA sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16004555" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Toward the 1,000 dollars human genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32504078" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Long-read human genome sequencing and its applications.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17704789" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Hair follicle: a reliable source of recipient origin after allogeneic hematopoietic stem cell transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10713640" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Buccal swabs but not mouthwash samples can be used to obtain pretransplant DNA fingerprints from recipients of allogeneic bone marrow transplants.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23253437" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Avoiding pitfalls in bone marrow engraftment analysis: a case study highlighting the weakness of using buccal cells for determining a patient's constitutional genotype after hematopoietic stem cell transplantation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22006311" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Mutations in 12 genes for inherited ovarian, fallopian tube, and peritoneal carcinoma identified by massively parallel sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25117502" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Cancer genetic testing panels for inherited cancer susceptibility: the clinical experience of a large adult genetics practice.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26014596" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Gene-panel sequencing and the prediction of breast-cancer risk.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25198398" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Population-based screening for BRCA1 and BRCA2: 2014 Lasker Award.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25198398" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Population-based screening for BRCA1 and BRCA2: 2014 Lasker Award.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23887774" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : ACMG clinical laboratory standards for next-generation sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24406459" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Assessing the necessity of confirmatory testing for exome-sequencing results in a clinical molecular diagnostic laboratory.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26847218" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Systematic Evaluation of Sanger Validation of Next-Generation Sequencing Variants.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25985138" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Factors influencing success of clinical genome sequencing across a broad spectrum of disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21946919" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Exome sequencing as a tool for Mendelian disease gene discovery.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20220177" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Whole-genome sequencing in a patient with Charcot-Marie-Tooth neuropathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20435227" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Clinical assessment incorporating a personal genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26014595" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : ClinGen--the Clinical Genome Resource.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26746457" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Association of Arrhythmia-Related Genetic Variants With Phenotypes Documented in Electronic Medical Records.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27181684" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Performance of ACMG-AMP Variant-Interpretation Guidelines among Nine Laboratories in the Clinical Sequencing Exploratory Research Consortium.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24941179" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Diagnostic clinical genome and exome sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30281996" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Next-Generation Sequencing to Diagnose Suspected Genetic Disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34758253" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : 100,000 Genomes Pilot on Rare-Disease Diagnosis in Health Care - Preliminary Report.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25326637" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Clinical exome sequencing for genetic identification of rare Mendelian disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25529582" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Genetic diagnosis of developmental disorders in the DDD study: a scalable analysis of genome-wide research data.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24088041" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Clinical whole-exome sequencing for the diagnosis of mendelian disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25326635" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Molecular findings among patients referred for clinical whole-exome sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24896178" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Genome sequencing identifies major causes of severe intellectual disability.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34089648" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Project Baby Bear: Rapid precision care incorporating rWGS in 5 California children's hospitals demonstrates improved clinical outcomes and reduced costs of care.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29323667" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Making new genetic diagnoses with old data: iterative reanalysis and reporting from genome-wide data in 1,133 families with developmental disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27276562" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Exome Sequencing and the Management of Neurometabolic Disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25117115" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : NIH's Undiagnosed Diseases Program expands: 6 new sites offer potential answers to more patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30304647" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Effect of Genetic Diagnosis on Patients with Previously Undiagnosed Disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26944241" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Outcome of Whole Exome Sequencing for Diagnostic Odyssey Cases of an Individualized Medicine Clinic: The Mayo Clinic Experience.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26944241" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Outcome of Whole Exome Sequencing for Diagnostic Odyssey Cases of an Individualized Medicine Clinic: The Mayo Clinic Experience.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30526229" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Underdiagnosis of Hereditary Breast Cancer: Are Genetic Testing Guidelines a Tool or an Obstacle?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30526229" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Underdiagnosis of Hereditary Breast Cancer: Are Genetic Testing Guidelines a Tool or an Obstacle?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30526229" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : Underdiagnosis of Hereditary Breast Cancer: Are Genetic Testing Guidelines a Tool or an Obstacle?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30526229" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Underdiagnosis of Hereditary Breast Cancer: Are Genetic Testing Guidelines a Tool or an Obstacle?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30526229" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Underdiagnosis of Hereditary Breast Cancer: Are Genetic Testing Guidelines a Tool or an Obstacle?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28481359" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28481359" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28481359" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25964672" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : The role of pathogen genomics in assessing disease transmission.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31881145" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : Pathogen Genomics in Public Health.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24896819" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Actionable diagnosis of neuroleptospirosis by next-generation sequencing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31978945" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : A Novel Coronavirus from Patients with Pneumonia in China, 2019.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31992387" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : Detection of 2019 novel coronavirus (2019-nCoV) by real-time RT-PCR.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32321856" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32973171" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : Sequence analysis of SARS-CoV-2 genome reveals features important for vaccine design.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32973171" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : Sequence analysis of SARS-CoV-2 genome reveals features important for vaccine design.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27854360" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26335890" id="rid53" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>54 : Cost-effectiveness of Universal BRCA1/2 Screening: Evidence-Based Decision Making.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29361001" id="rid54" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>55 : Cost-effectiveness of Population-Based BRCA1, BRCA2, RAD51C, RAD51D, BRIP1, PALB2 Mutation Testing in Unselected General Population Women.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34012068" id="rid55" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>56 : ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25719666" id="rid56" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>57 : Whole genomes redefine the mutational landscape of pancreatic cancer.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29040395" id="rid57" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>58 : Human Genome Sequencing at the Population Scale: A Primer on High-Throughput DNA Sequencing and Analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/35871805" id="rid58" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>59 : UK explores whole-genome sequencing for newborn babies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28654958" id="rid59" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>60 : The Impact of Whole-Genome Sequencing on the Primary Care and Outcomes of Healthy Adult Patients: A Pilot Randomized Trial.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
