<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Tools for genetics and genomics: Cytogenetics and molecular genetics
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Tools for genetics and genomics: Cytogenetics and molecular genetics
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Tools for genetics and genomics: Cytogenetics and molecular genetics
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Authors:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Iris Schrijver, MD
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            James L Zehnder, MD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Benjamin A Raby, MD, MPH
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Jennifer S Tirnauer, MD
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            Aug 10, 2023.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H1">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         Intense genetic research has exponentially increased our knowledge of the genetic code of humans and other organisms, leading to the development of numerous methods that facilitate our understanding of normal and abnormal genetic processes. Many of these methods and related techniques are now routinely used in the molecular diagnosis of both inherited disorders and diseases that result from somatic mutations, such as the hematologic malignancies. Molecular genetic and cytogenetic diagnostics are invaluable additions to laboratory testing and clinical evaluation, providing diagnostic, therapeutic, and prognostic information. (See
         <a class="medical medical_review" href="/z/d/html/4500.html" rel="external">
          "Genetic abnormalities in hematologic and lymphoid malignancies"
         </a>
         .)
        </p>
        <p>
         Although the advent of improved and faster molecular methods has transformed the traditional diagnostic process, keeping current with the most recent advances is daunting [
         <a href="#rid1">
          1
         </a>
         ]. A conceptual approach to a selection of the most common standard and novel diagnostic tools will be applied to this review. An outline of the advantages and limitations of each of the techniques is included, as well as some examples of their applications. A brief introduction to the terms required to properly understand these applications is provided separately. (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression"
         </a>
         .)
        </p>
        <p>
         Three general categories of testing can be distinguished.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Mutation detection
         </strong>
         of known sequence changes can be performed. This type of testing is targeted and typically limited to a predefined number of sequence changes, selected in advance. Selection is generally based on association with clinical phenotypes. Sequence changes may be located within a single gene or across multiple genes. Depending on the testing method used, the number of included sequence changes can range from a single mutation to thousands of mutations.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Cytogenetic studies
         </strong>
         of large structural variants are typically performed when the phenotype does not seem limited to point mutations and relatively small deletions and duplications. Such studies are helpful in syndromic phenotypes and for constellations of symptoms typically associated with abnormalities on the scale of chromosomes rather than single exons or genes.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Genotyping methods
         </strong>
         can be applied to identify mutations not selected in advance. These methods aim to discover mutations, and can target a gene with a known heterogeneous distribution of mutations, or can target larger segments of the genome to identify known or novel variations.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H5157832">
         <span class="h1">
          DETECTING KNOWN MUTATIONS
         </span>
         <span class="headingEndMark">
          —
         </span>
         There are many different approaches for the identification of selected, known mutations. Typically, these start with the polymerase chain reaction, after which additional assay steps are performed. The following section lists examples of some of the frequently used techniques, together with their advantages and disadvantages. General mutation detection methods such as DNA sequencing can also be applied to the identification of known mutations. Although the entire sequence in an amplified fragment would be read, mutations within that segment would be identified readily by this more comprehensive method.
        </p>
        <p class="headingAnchor" id="H2">
         <span class="h2">
          Polymerase chain reaction
         </span>
         <span class="headingEndMark">
          —
         </span>
         The automated polymerase chain reaction (PCR) is commonly the first step in the vast majority of DNA analyses because it increases the amount of DNA available for analysis. This important diagnostic tool is discussed in detail separately. (See
         <a class="medical medical_review" href="/z/d/html/2896.html" rel="external">
          "Polymerase chain reaction (PCR)"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H4">
         <span class="h3">
          Restriction enzyme digestion
         </span>
         <span class="headingEndMark">
          —
         </span>
         Restriction enzyme digestion can be used to detect mutations that create or destroy a restriction enzyme site. Specific restriction enzymes, usually isolated from bacteria, recognize unique short sequences within a DNA fragment. These enzymes can cleave the DNA strands at that exact site. If a mutation either changes the DNA code to a sequence that creates a new restriction enzyme site or obliterates an existing restriction enzyme site, the mutation can be detected based upon the presence or absence of the site  (
         <a class="graphic graphic_figure graphicRef81760" href="/z/d/graphic/81760.html" rel="external">
          figure 1
         </a>
         ).
        </p>
        <p>
         One example of such a test is a restriction digestion assay for Muenke syndrome, a type of craniosynostosis syndrome defined by a single mutation (c.749C&gt;G, p.Pro250Arg) in the FGFR3 gene that creates a new restriction site. In this assay, exon 7 is amplified by PCR, and the sample is subjected to a restriction digest with Ban I, the enzyme that recognizes the new site. DNA from patients who carry the mutation is cleaved by the enzyme, while control DNA lacking the mutation remains uncleaved.
        </p>
        <p>
         Advantages of this technique include the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It is technically easy and can be performed within one day.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Restriction enzyme analysis detects specific mutations and can be applied to many samples concurrently. These samples are then run side-by-side on a single gel.
        </p>
        <p>
        </p>
        <p>
         Disadvantages are the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         This method is impractical for disorders caused by a large number of different mutations and for the detection of mutations associated with nucleotide sequences that require the use of expensive restriction enzymes.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Only a small fraction of existing point mutations actually create or remove a restriction site. In some instances, this problem can be circumvented by the introduction of an artificial restriction site during PCR amplification [
         <a href="#rid2">
          2
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Incomplete digestion can produce erroneous results. This problem can be overcome by using adequate controls.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H5">
         <span class="h2">
          Amplification refractory mutation system
         </span>
         <span class="headingEndMark">
          —
         </span>
         The amplification refractory mutation system (ARMS) can be used to detect known point mutations. This technique requires a multiplex PCR reaction [
         <a href="#rid3">
          3
         </a>
         ]. In this reaction, two primer pairs are added to a single PCR tube and two separate sequences from one piece of DNA are amplified in the same reaction. One reaction (using the control primer pair) is an internal control to demonstrate that the PCR reaction itself has worked. The other reaction (using a primer pair specific for the mutation under study) will amplify the target sequence depending upon the presence or absence of a specific point mutation. A second tube contains DNA from the same patient, and includes the control primer pair and a primer pair that amplifies only the normal sequence. The only difference between the primers for the normal and mutant sequence is complementarity of one primer at the 3' end, where one is identical to the normal and one to the variant sequence. The other primer is the same for both reactions, and the product size will be the same. When run side by side on a gel, these samples will exhibit homozygosity for the normal sequence, homozygosity for the mutation, or heterozygosity  (
         <a class="graphic graphic_figure graphicRef55076" href="/z/d/graphic/55076.html" rel="external">
          figure 2
         </a>
         ).
        </p>
        <p>
         This assay is an actual modification of the PCR itself, rather than an additional step after the original amplification. ARMS can be performed for just one, or a small number of simultaneously tested mutations. An example is an ARMS test for BRAF mutation c.1799T&gt;A (p.V600E) which has been identified in colorectal carcinoma, melanoma, and papillary thyroid carcinoma. Because the mutation is a point mutation, changing one nucleotide to another, primers can be designed such that either the wild-type or the mutant sequence perfectly matches the very 3 prime end of the primer. In an optimized assay, a PCR product would then only be expected with the primer pair that entirely matches.
        </p>
        <p>
         Advantages of this technique include the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It is easy to perform and complete within one day.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It can detect specific point mutations and assess many samples concurrently. Modification of the method allows analysis of several mutations in one test tube.
        </p>
        <p>
        </p>
        <p>
         Disadvantages are the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It is impractical for disorders caused by a large number of mutations.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Primer pairs must be designed for all reactions. Amplification of these reactions under the same conditions is a prerequisite. If the conditions are suboptimal for one of the primer pairs, weak amplification or nonspecific amplification may result in ambiguous results.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Every patient sample requires multiple PCR tubes.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H6">
         <span class="h2">
          Allele-specific oligonucleotide hybridization
         </span>
         <span class="headingEndMark">
          —
         </span>
         Allele-specific oligonucleotide (ASO) hybridization involves the placing ("spotting") of denatured PCR-amplified DNA onto a membrane and subsequent hybridization with short allele-specific, labeled probes. Under optimal hybridization and washing conditions, hybridization will only occur if the probe sequence is perfectly complementary to the single-stranded sample DNA.
        </p>
        <p>
         Typically, PCR products from one patient sample are fixed onto two identical membranes (a "dot-blot"), one of which is hybridized with a probe that contains the normal sequence, while the other is hybridized with a probe for the mutant sequence. The two probes should differ by just one nucleotide, corresponding to the point mutation under investigation. After exposure to an autoradiographic film in the case of radioactive probe labeling, or after chemical treatment in the case of biotinylated oligomers, positive signals are scored and heterozygosity or homozygosity for the normal or mutant sequence can be determined  (
         <a class="graphic graphic_figure graphicRef80013" href="/z/d/graphic/80013.html" rel="external">
          figure 3
         </a>
         ).
        </p>
        <p>
         ASO hybridization can be modified to analyze a panel of mutations for a single patient. In "reverse" allele-specific hybridization, for example, sequence specific probes are spotted onto the membrane and only one membrane is used per patient. Reverse ASO is less cost-effective than regular ASO, but can decrease the turnaround time per sample. This method is frequently applied in molecular testing for cystic fibrosis. (See
         <a class="medical medical_review" href="/z/d/html/6367.html" rel="external">
          "Cystic fibrosis: Clinical manifestations and diagnosis", section on 'Molecular diagnosis'
         </a>
         .)
        </p>
        <p>
         Advantages of ASO hybridization include the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It is suitable for analysis of specific mutations or polymorphisms in numerous samples [
         <a href="#rid4">
          4
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It is highly sensitive and specific if properly optimized.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Adaptations for multiplex PCR analysis or automated microarray (DNA chip) analysis are possible. (See
         <a class="local">
          'Amplification refractory mutation system'
         </a>
         above and
         <a class="local">
          'Genotyping microarrays'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p>
         Disadvantages include:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Each ASO probe can only detect one specific sequence.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         ASO hybridization is amenable to small DNA mutations only.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         There is potential non-specificity if the hybridization and/or washing conditions are not fully optimized.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H2087689">
         <span class="h2">
          Genotyping microarrays
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genotyping microarrays are available on a variety of different molecular platforms. They can all interrogate a flexible number of mutations at the same time, which makes them attractive for high throughput analyses in one or multiple genes, for one or many different patients at the same time. These assays are typically automated, and in a high-throughput setting (ie, automated analysis of multiple samples simultaneously), do not require hands-on work after the initiation of the assay run.
        </p>
        <p>
         Advantages of this method include the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It is suitable for high-throughput analysis of specific mutations or polymorphisms in numerous samples.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Relatively less hands-on work is required per sample.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Interpretation of the data can also be highly automated.
        </p>
        <p>
        </p>
        <p>
         Disadvantages include:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The microarray analysis equipment as well as the individual arrays can be expensive.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Often this method is not suitable for low-volume testing, in particular when the microarrays can be used only one time, regardless of whether only one or many samples are tested.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H15538597">
         <span class="h1">
          DETECTING CYTOGENETIC ABNORMALITIES
         </span>
         <span class="headingEndMark">
          —
         </span>
         Cytogenetic abnormalities are genetic defects that involve large regions of chromosomes rather than small pieces of DNA (ie, translocations, large deletions, or aneuploidies). These defects can be detected by at least three methods – chromosomal (karyotypic) analysis, fluorescence in situ hybridization (FISH) using specific DNA probes on either metaphase chromosomes or interphase nuclei, and array comparative genomic hybridization (aCGH). The latter two methods are considered "molecular cytogenetics," because they can detect anomalies which are below the resolution of chromosomal analysis.
        </p>
        <p class="headingAnchor" id="H15538624">
         <span class="h2">
          Chromosomal analysis
         </span>
         <span class="headingEndMark">
          —
         </span>
         Chromosomal analysis (also called chromosome banding) is used to detect changes in large regions of chromosomes (translocations, large deletions, or aneuploidies).
        </p>
        <p>
         To perform chromosome analysis, lymphocytes, usually obtained from the peripheral blood, are cultured in vitro and stimulated to divide under the influence of mitogens. Other cell types, such as amniocytes, bone marrow cells, fibroblasts, and tumor cells can also be analyzed, often without a mitogenic stimulus. Once the cells divide readily, a chemical is added to arrest mitotic division in metaphase. In this phase of the cell cycle, the chromosomes are maximally contracted and hence their banding patterns are easier to recognize  (
         <a class="graphic graphic_figure graphicRef52419" href="/z/d/graphic/52419.html" rel="external">
          figure 4
         </a>
         ).
        </p>
        <p>
         Chromosomal banding is then used to identify each individual chromosome, for assessment of whether the correct number of each chromosome is present (two of each autosome plus the sex chromosomes) and whether there are structural abnormalities. This technique can be performed using various enzymes and dyes. The most frequently used banding technique is GTG (G-banding with trypsin and Giemsa-banding)  (
         <a class="graphic graphic_figure graphicRef81862" href="/z/d/graphic/81862.html" rel="external">
          figure 5
         </a>
         ). An identical banding pattern is seen in Q-banding, in which the chromosomes are stained with a fluorescent dye and viewed under ultraviolet illumination.
        </p>
        <p>
         The typical resolution of chromosome banding is about 400 bands in a haploid set of 23 chromosomes. A single chromosome band may contain 6 megabases (Mb) of DNA and approximately 150 genes. If higher banding resolution is desired to study relatively small chromosomal rearrangements, the cell cycles can be synchronized and cells arrested in prometaphase (550 bands) or even prophase (approximately 800 bands). The latter technique is called high resolution banding.
        </p>
        <p>
         Examples of the use of this method include evaluation of patients with acute myeloid leukemia (AML). (See
         <a class="medical medical_review" href="/z/d/html/4544.html" rel="external">
          "Acute myeloid leukemia: Cytogenetic abnormalities"
         </a>
         .)
        </p>
        <p>
         In many cases, chromosome banding has been replaced with alternative methods such as FISH and microarray analysis. (See
         <a class="local">
          'Fluorescence in situ hybridization'
         </a>
         below and
         <a class="local">
          'Array comparative genomic hybridization'
         </a>
         below.)
        </p>
        <p>
         Advantages of chromosomal banding include:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         In contrast to molecular genetic studies, chromosome banding techniques show the entire genome at one time.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         This method is suitable in diagnostic situations where a specific anomaly is suspected (eg, the Philadelphia chromosome in chronic myeloid leukemia, CML). It may also be useful to monitor disease; for example, to detect additional chromosomal abnormalities commonly seen in disease progression of CML. (See
         <a class="medical medical_review" href="/z/d/html/4487.html" rel="external">
          "Cellular and molecular biology of chronic myeloid leukemia", section on 'Progression to acute phase CML'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         In disorders having deletions of varying size within a specific chromosome, such as multiple myeloma [
         <a href="#rid5">
          5
         </a>
         ], karyotypes from many patients can be compared, in order to find the critical disease-associated region  (
         <a class="graphic graphic_figure graphicRef64638" href="/z/d/graphic/64638.html" rel="external">
          figure 6
         </a>
         ).
        </p>
        <p>
        </p>
        <p>
         Disadvantages are:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Most chromosome banding techniques can only detect major structural abnormalities and will not detect smaller regions of DNA gain or loss.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Interpretation is labor intensive and highly dependent upon operator experience and skill.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H15538631">
         <span class="h2">
          Fluorescence in situ hybridization
         </span>
         <span class="headingEndMark">
          —
         </span>
         Fluorescence in situ hybridization (FISH) is a technique that allows counting of the number and location of large pieces of chromosomes. This technique has greatly increased the sensitivity, specificity, and resolution of chromosome analyses [
         <a href="#rid6">
          6
         </a>
         ]. FISH can be performed on metaphase chromosomes or interphase nuclei; interphase FISH can be done on paraffin embedded tissue.
        </p>
        <p>
         Metaphase FISH allows identification of large chromosomal abnormalities, including deletions, duplications, and translocations, as well as smaller chromosomal microdeletions and duplications. For metaphase FISH, cells are arrested in mitosis as for chromosomal banding. They are then fixed using a mixture of acetic acid and methanol and then “dropped” on a glass microscope slide where they are affixed. DNA probes of a few hundred kilobases (kb) in length are used that match regions the chromosomes containing the DNA sequence in question. These probes are directly hybridized with the chromosomes on the slide (hence, the term "in situ" hybridization); immediate detection of the fluorescence signal is possible via fluorescence microscopy. Less commonly used isotopic and non-isotopic chemical labeling methods are also available  (
         <a class="graphic graphic_diagnosticimage graphicRef67440" href="/z/d/graphic/67440.html" rel="external">
          image 1
         </a>
         ).
        </p>
        <p>
         FISH probes produce a fluorescent dot on the chromosome to which they hybridize. Thus, every pair of chromosomes (or chromosome regions) produces two dots. These double dots sometimes fuse to form one signal. Cells that are monosomic for the chromosomal region in question would show only a single dot per nucleus, while trisomic cells would show three dots.
        </p>
        <p>
         FISH can also be modified for analysis of interphase nuclei. (See
         <a class="local">
          'Interphase FISH'
         </a>
         below.)
        </p>
        <p>
         Advantages of this technique are:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The resolution of FISH is much better than traditional chromosome banding (FISH can resolve 2 megabases (Mb) in length, compared to 6 Mb for chromosomal banding).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         FISH can be applied to both dividing (metaphase) and non-dividing (interphase) cells.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The protocol is technically fairly straightforward.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Hybridization with multiple probes enables detection of translocation products. An example is the BCR-ABL1 fusion of t(9;22) in chronic myeloid leukemia  (
         <a class="graphic graphic_figure graphicRef74102" href="/z/d/graphic/74102.html" rel="external">
          figure 7
         </a>
         ).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         FISH can identify a range of structural abnormalities including deletions, duplications, aneuploidy and the presence of derivative (structurally rearranged) chromosomes.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         FISH may be used to monitor recurrent or residual disease in bone marrow transplant patients. (See
         <a class="medical medical_review" href="/z/d/html/4482.html" rel="external">
          "General aspects of cytogenetic analysis in hematologic malignancies", section on 'FISH'
         </a>
         .)
        </p>
        <p>
        </p>
        <p>
         Disadvantages of the technique are:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Small mutations, including small deletions and insertions as well as point mutations, cannot be identified.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Uniparental disomy (inheritance of both copies of a chromosome from the same parent) will be missed because the probe merely detects the presence or absence of a locus or specific portion of a chromosome and not its source  (
         <a class="graphic graphic_figure graphicRef72197" href="/z/d/graphic/72197.html" rel="external">
          figure 8
         </a>
         ).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Chromosomal inversions will be missed since a probe can only detect the presence of a specific sequence but not its precise location within the chromosome.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Probes are not yet commercially available for all chromosomal regions.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The clinician has to choose the correct FISH probe in order to make an accurate diagnosis.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H15538638">
         <span class="h3">
          Interphase FISH
         </span>
         <span class="headingEndMark">
          —
         </span>
         Interphase FISH is used when dividing cells are not available (eg, in fully differentiated cells or in tissues that have been fixed and embedded in paraffin). It also improves the resolution of FISH probes. For interphase FISH, cells are tested with FISH probes without cell cycle synchronization. When using intact nuclei (eg, not from paraffin embedded tissue), the cells are harvested using a hypotonic solution, fixed, and again “dropped” on the slides. For FISH on paraffin embedded tissue, thin slices of pathological specimens are cut and affixed to the slides.
        </p>
        <p>
         Because chromosomes are only minimally condensed in interphase, this modification of FISH analysis provides the opportunity to hybridize probes at a high resolution (well under 1 Mb, compared to 2 Mb for metaphase FISH). In the interphase nucleus, chromosomal structures cannot be discerned and only the hybridized probe will light up  (
         <a class="graphic graphic_figure graphicRef60540" href="/z/d/graphic/60540.html" rel="external">
          figure 9
         </a>
         ). Hybridization with two different color probes that cross the breakpoint regions of genes involved in a translocation results in the expected signals from the normal chromosome as well as a fusion signal from the derived chromosome(s) on which the genes are juxtaposed by translocation [
         <a href="#rid7">
          7
         </a>
         ].
        </p>
        <p>
         Advantages of interphase FISH include the higher resolution than metaphase FISH; the ability to perform the test immediately without culturing the cells, which makes it faster; and the applicability to paraffin embedded sections. Interphase FISH is of special value in prenatal diagnosis of various aneuploidy states, such as trisomy 18 or trisomy 21, in which the ability to obtain results rapidly aids in decision making.
        </p>
        <p>
         The major disadvantage of interphase compared to metaphase FISH is that in interphase FISH the chromosomes themselves cannot be visualized. Thus, information cannot be provided regarding overall chromosome number and composition.
        </p>
        <p class="headingAnchor" id="H15538645">
         <span class="h3">
          Spectral karyotyping
         </span>
         <span class="headingEndMark">
          —
         </span>
         Spectral karyotyping (SKY, also called multicolor FISH) is a FISH technique that accurately identifies the chromosomal origin of all elements in a karyogram (complete chromosome set) using multiple wavelengths of light to generate signals of many colors. A combination of five fluorochromes can be used as probes to "paint" all 22 autosomes, as well as the X and the Y chromosomes, in different colors  (
         <a class="graphic graphic_figure graphicRef78864" href="/z/d/graphic/78864.html" rel="external">
          figure 10
         </a>
         ). The karyogram is analyzed by a computerized spectral imager and the chromosomes are classified based on their particular emission spectra [
         <a href="#rid8">
          8,9
         </a>
         ].
        </p>
        <p>
         Advantages of this technique are:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         This method allows complete karyotyping using automated analysis.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The origin of marker chromosomes, small insertions, and complex rearrangements can be inferred through the presence of color-coded chromosomal segments [
         <a href="#rid9">
          9,10
         </a>
         ].
        </p>
        <p>
        </p>
        <p>
         Disadvantages include:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The equipment required may be prohibitively expensive for small diagnostic laboratories.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Even though part of the analysis is computerized, the overall technique is labor intensive.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Structural rearrangements within a single chromosome will not be detected.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The resolution of SKY is relatively low (up to 15 Mb).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         SKY may be used when a specific abnormality is suspected, but is not applicable as a screening method.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H15538652">
         <span class="h2">
          Array comparative genomic hybridization
         </span>
         <span class="headingEndMark">
          —
         </span>
         Comparative genomic hybridization (CGH) allows detection of amplifications and deletions of smaller regions of DNA along the lengths of all of the chromosomes. The technique works by comparing the genomic content (or DNA) of a patient (or target) with a normal control individual (or individuals)  (
         <a class="graphic graphic_figure graphicRef58888" href="/z/d/graphic/58888.html" rel="external">
          figure 11
         </a>
         ). The resolution of “classical” or metaphase CGH is relatively low (approximately 15 Mb of DNA) [
         <a href="#rid11">
          11
         </a>
         ].
        </p>
        <p>
         Array CGH is a modification of CGH in which the comparator DNA, RNA, or tissue is arrayed on a glass slide or glass beads [
         <a href="#rid12">
          12
         </a>
         ]. There are three basic types of array: bacterial artificial chromosomes (BAC) arrays, oligonucleotide arrays (typically 60 base pairs in length), or single nucleotide polymorphism (SNP) arrays (typically a few nucleotides). Most SNP-based arrays now also include single locus probes as well as SNPs.
        </p>
        <p>
         There are at least two different types of arrays: “targeted” arrays and whole genomic arrays.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Targeted arrays “target” known microdeletion/microduplication syndromes, as well as other known loci of inherited Mendelian disorders (eg, tuberous sclerosis). The first targeted arrays were arrays of around 500 to 600 BACs.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Whole genomic arrays cover the entire genome at varying levels of resolution. The first whole genomic arrays were BAC arrays with around 2600 BACs spaced at about 1 Mb throughout the genome (ie, with about 1 Mb resolution). Oligonucleotide and SNP arrays have supplanted the use of BACs. Most laboratories use either oligonucleotide or SNP arrays with an average resolution of about 35 kb throughout the genome.
        </p>
        <p>
        </p>
        <p>
         Both SNP and oligonucleotide arrays can detect copy number variations, but only SNP arrays can be used to determine whether there is absence or loss of heterozygosity (AOH or LOH) for different regions, or even entire chromosomes in the presence of normal copy number [
         <a href="#rid13">
          13
         </a>
         ]. AOH refers to the inheritance of either paternal or maternal alleles alone  (
         <a class="graphic graphic_figure graphicRef72197" href="/z/d/graphic/72197.html" rel="external">
          figure 8
         </a>
         ). The absence of biparental inheritance can be seen in uniparental disomy (UPD) or in some cancer tissue samples. UPD can be the result of heterodisomy, in which two different homologous chromosomes from the same parent (either maternal or paternal) are present, instead of the normal biparental contribution (one chromosome from each parent). Alternatively, UPD can occur when there are two identical copies of a single parental chromosome (isodisomy). SNP arrays can only detect UPDs secondary to isodisomies [
         <a href="#rid13">
          13
         </a>
         ].
        </p>
        <p>
         Interpretation of array CGH can also be complicated by the presence of copy number variants (CNVs) that can be benign, pathogenic or unknown. (See
         <a class="medical medical_review" href="/z/d/html/16647.html" rel="external">
          "Genomic disorders: An overview", section on 'Copy number variations'
         </a>
         .)
        </p>
        <p>
         Advantages of array CGH are:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Typically, dividing cells and tissue culture are not necessary; the technique requires only good, high-quality DNA.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Resolution of the array is dependent upon the type of array used and the average spacing of the “probes” on the array.
        </p>
        <p>
        </p>
        <p>
         Disadvantages of this technique are:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Balanced structural rearrangements (ie, balanced translocations, inversions, insertions) will not be detected, because there is no change in copy number.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Levels of mosaicism (ie, copy number changes in some but not all cells) of 20 percent or less will not be detected.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Interpretation of copy number changes which have not been previously reported can be challenging, particularly if a phenotypically normal parent carries the same change.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The technology is still relatively expensive, even though costs have declined.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H9">
         <span class="h1">
          GENOTYPING NEW MUTATIONS
         </span>
         <span class="headingEndMark">
          —
         </span>
         In diseases with allelic heterogeneity, finding the pathogenic variant (disease-causing mutation) is not straightforward. In these disorders, genes need to be scanned for mutations before a specific pathogenic variant may be identified. After a new sequence variant has been detected and characterized, its pathogenicity must be established. This is commonly accomplished by screening a large number of normal controls, who are expected not to carry the same allelic variant. In addition, the DNA sequence of affected and unaffected individuals within the same family should be compared.
        </p>
        <p>
         Many methods of mutation screening are currently available. However, even the most common screening methods have limited application in clinical diagnostic laboratories, since the search for individual disease-causing mutations is a laborious and expensive process.
        </p>
        <p class="headingAnchor" id="H10">
         <span class="h2">
          Heteroduplex analysis
         </span>
         <span class="headingEndMark">
          —
         </span>
         Heteroduplex analysis is used to detect point mutations on one strand of a DNA helix. The technique uses denaturation and reannealing of the double stranded target DNA [
         <a href="#rid14">
          14
         </a>
         ]. If complementary single strands re-anneal, they form a perfectly aligned homoduplex. On the other hand, single strands that are not completely complementary because of the presence of a point-mutation in one strand form a heteroduplex. The failure to anneal at all base positions results in the formation of a "bubble" in the newly formed double strand. As compared to normal DNA, DNA with a bubble migrates more slowly during electrophoresis  (
         <a class="graphic graphic_figure graphicRef55618" href="/z/d/graphic/55618.html" rel="external">
          figure 12
         </a>
         ).
        </p>
        <p>
         This method is relatively easy to perform and requires little optimization. Some of the available gel matrices are less toxic than polyacrylamide gels. A disadvantage is that this scanning method does not identify the location of the mutation within the analyzed fragment.
        </p>
        <p class="headingAnchor" id="H11">
         <span class="h2">
          Single strand conformation analysis
         </span>
         <span class="headingEndMark">
          —
         </span>
         Single strand conformation analysis (SSCA) is based on the observation that single-stranded DNA fragments assume unique conformations that depend on their sequence composition when run in a non-denaturing polyacrylamide gel. Migration through the gel thus depends on chain length as well as strand conformation [
         <a href="#rid15">
          15
         </a>
         ]. Before interpretations can be made, the specific pattern of migration in wild-type control samples must be known. This can be achieved by the addition of normal controls in the run.
        </p>
        <p>
         Once optimized, this method is amenable to screening a fairly large number of samples at one time. However, this scanning method does not identify the location of the mutation within the analyzed fragment. SSCA also requires optimization, and reproducibility of the conditions in all analyses of the same DNA sequence is crucial.
        </p>
        <p class="headingAnchor" id="H12">
         <span class="h2">
          Sequencing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Sequencing methods are discussed in detail separately. (See
         <a class="medical medical_review" href="/z/d/html/2901.html" rel="external">
          "Genetic testing"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H13">
         <span class="h2">
          Southern and Northern blotting
         </span>
         <span class="headingEndMark">
          —
         </span>
         Southern blotting can be used to detect small mutations as well as large deletions, duplications and gene rearrangements that alter restriction enzyme cleavage sites or the sizes of the resulting pieces of DNA [
         <a href="#rid16">
          16
         </a>
         ]. In this technique, genomic DNA is digested with one or more restriction endonucleases and transferred to a membrane. This membrane is hybridized with a single-stranded radioactively labeled probe, under conditions that facilitate double-strand formation between the probe and those fragments on the gel containing the complementary sequence. When an autoradiographic film is exposed to the membrane and developed, the hybridized sequences become visible as bands. The location of each of these bands corresponds to the size of the fragment to which the probe is bound  (
         <a class="graphic graphic_figure graphicRef78900" href="/z/d/graphic/78900.html" rel="external">
          figure 13
         </a>
         ).
        </p>
        <p>
         Northern blotting, in which RNA rather than DNA is analyzed, is discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/14602.html" rel="external">
          "Tools for genetics and genomics: Gene expression profiling", section on 'Northern blot (mainly of historical interest)'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2103939">
         <span class="h1">
          SUMMARY
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Uses for cytogenetics
         </strong>
         – Cytogenetic and molecular diagnostic tools are applied for three major purposes in clinical genetics: detecting specific mutations, studying large chromosomal structural variants, and genotyping to find mutations that have not been previously identified. (See
         <a class="local">
          'Introduction'
         </a>
         above and
         <a class="medical medical_review" href="/z/d/html/4490.html" rel="external">
          "Chromosomal translocations, deletions, and inversions"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Known mutations
         </strong>
         – Most tools to identify selected mutations involve polymerase chain reaction (PCR) technology. Techniques involving specific restriction enzymes can be used to find mutations that might affect the enzyme’s target cleavage site, although only a small fraction of point mutations are amenable to such techniques. The amplification refractory mutation system (ARMS) involves a multiplex PCR reaction and can detect specific point mutations. Other techniques involve oligonucleotide hybridization and genotyping microarrays. Microarrays can allow for high throughput. (See
         <a class="medical medical_review" href="/z/d/html/2896.html" rel="external">
          "Polymerase chain reaction (PCR)"
         </a>
         and
         <a class="local">
          'Detecting known mutations'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Structural variation
         </strong>
         – Cytogenetic analysis to identify large structural variation may involve chromosomal (karyotype) analysis by chromosomal banding; fluorescence in situ hybridization (FISH) on metaphase or interphase nuclei; or array comparative genomic hybridization (CGH). The resolution increases, meaning smaller and smaller genetic defects can be detected, from karyotyping to interphase FISH to array CGH. (See
         <a class="local">
          'Detecting cytogenetic abnormalities'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          New variants
         </strong>
         – For diseases with allelic heterogeneity, new sequence variants continue to be discovered. Methods that allow analysis of entire genes or genomes improve this discovery. Most methods for mutation screening have limited applicability in clinical diagnostic laboratories. The most direct approach to mutation detection is automated sequencing of the target DNA. Whole genome sequencing is increasingly available in academic medical center molecular pathology laboratories and from commercial genomics establishments. (See
         <a class="local">
          'Genotyping new mutations'
         </a>
         above and
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H616263106">
         <span class="h1">
          ACKNOWLEDGMENT
         </span>
         <span class="headingEndMark">
          —
         </span>
         The UpToDate editorial staff acknowledges Athena M Cherry, PhD, who contributed to an earlier version of this topic review.
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li>
          <a class="nounderline abstract_t">
           Zamicnikova A. Genetic testing methods in myelodysplastic syndromes and leukemias: A review. Clin Leukemia 2007; 1:331.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Haliassos A, Chomel JC, Grandjouan S, et al. Detection of minority point mutations by modified PCR technique: a new approach for a sensitive diagnosis of tumor-progression markers. Nucleic Acids Res 1989; 17:8093.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Newton CR, Graham A, Heptinstall LE, et al. Analysis of any point mutation in DNA. The amplification refractory mutation system (ARMS). Nucleic Acids Res 1989; 17:2503.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Schrijver I, Liu W, Francke U. The pathogenicity of the Pro1148Ala substitution in the FBN1 gene: causing or predisposing to Marfan syndrome and aortic aneurysm, or clinically innocent? Hum Genet 1997; 99:607.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Shaughnessy J, Tian E, Sawyer J, et al. High incidence of chromosome 13 deletion in multiple myeloma detected by multiprobe interphase FISH. Blood 2000; 96:1505.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           van Ommen GJ, Breuning MH, Raap AK. FISH in genome research and molecular diagnostics. Curr Opin Genet Dev 1995; 5:304.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Haralambieva E, Banham AH, Bastard C, et al. Detection by the fluorescence in situ hybridization technique of MYC translocations in paraffin-embedded lymphoma biopsy samples. Br J Haematol 2003; 121:49.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Schröck E, Veldman T, Padilla-Nash H, et al. Spectral karyotyping refines cytogenetic diagnostics of constitutional chromosomal abnormalities. Hum Genet 1997; 101:255.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Jalal SM, Law ME, Stamberg J, et al. Detection of diagnostically critical, often hidden, anomalies in complex karyotypes of haematological disorders using multicolour fluorescence in situ hybridization. Br J Haematol 2001; 112:975.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hilgenfeld E, Padilla-Nash H, McNeil N, et al. Spectral karyotyping and fluorescence in situ hybridization detect novel chromosomal aberrations, a recurring involvement of chromosome 21 and amplification of the MYC oncogene in acute myeloid leukaemia M2. Br J Haematol 2001; 113:305.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kallioniemi A, Kallioniemi OP, Sudar D, et al. Comparative genomic hybridization for molecular cytogenetic analysis of solid tumors. Science 1992; 258:818.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Aradhya S, Cherry AM. Array-based comparative genomic hybridization: clinical contexts for targeted and whole-genome designs. Genet Med 2007; 9:553.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Papenhausen P, Schwartz S, Risheg H, et al. UPD detection using homozygosity profiling with a SNP genotyping microarray. Am J Med Genet A 2011; 155A:757.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Nagamine CM, Chan K, Lau YF. A PCR artifact: generation of heteroduplexes. Am J Hum Genet 1989; 45:337.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hayashi K. PCR-SSCP: a method for detection of mutations. Genet Anal Tech Appl 1992; 9:73.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Southern EM. Detection of specific sequences among DNA fragments separated by gel electrophoresis. J Mol Biol 1975; 98:503.
          </a>
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 2893 Version 29.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : Genetic testing methods in myelodysplastic syndromes and leukemias: A review
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2573037" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : Detection of minority point mutations by modified PCR technique: a new approach for a sensitive diagnosis of tumor-progression markers.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2785681" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : Analysis of any point mutation in DNA. The amplification refractory mutation system (ARMS).
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9150726" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : The pathogenicity of the Pro1148Ala substitution in the FBN1 gene: causing or predisposing to Marfan syndrome and aortic aneurysm, or clinically innocent?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10942398" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : High incidence of chromosome 13 deletion in multiple myeloma detected by multiprobe interphase FISH.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7549423" id="rid5" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          6 : FISH in genome research and molecular diagnostics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12670331" id="rid6" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          7 : Detection by the fluorescence in situ hybridization technique of MYC translocations in paraffin-embedded lymphoma biopsy samples.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9439652" id="rid7" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          8 : Spectral karyotyping refines cytogenetic diagnostics of constitutional chromosomal abnormalities.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11298595" id="rid8" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          9 : Detection of diagnostically critical, often hidden, anomalies in complex karyotypes of haematological disorders using multicolour fluorescence in situ hybridization.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11380393" id="rid9" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          10 : Spectral karyotyping and fluorescence in situ hybridization detect novel chromosomal aberrations, a recurring involvement of chromosome 21 and amplification of the MYC oncogene in acute myeloid leukaemia M2.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/1359641" id="rid10" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          11 : Comparative genomic hybridization for molecular cytogenetic analysis of solid tumors.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17873642" id="rid11" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          12 : Array-based comparative genomic hybridization: clinical contexts for targeted and whole-genome designs.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21594998" id="rid12" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          13 : UPD detection using homozygosity profiling with a SNP genotyping microarray.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2631699" id="rid13" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          14 : A PCR artifact: generation of heteroduplexes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/1476794" id="rid14" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          15 : PCR-SSCP: a method for detection of mutations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/1195397" id="rid15" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          16 : Detection of specific sequences among DNA fragments separated by gel electrophoresis.
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
