<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Polymerase chain reaction (PCR)</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Polymerase chain reaction (PCR)</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Polymerase chain reaction (PCR)</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Anne Slavotinek, MBBS, PhD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jennifer S Tirnauer, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Oct 11, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H8006600"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Polymerase chain reaction (PCR) underlies many molecular biology and molecular genetics techniques. In just a few hours, PCR can create a million copies of a single deoxyribonucleic acid (DNA) or ribonucleic acid (RNA) molecule, providing sufficient material for numerous clinical and research applications. </p><p>This topic reviews the PCR technique, modifications, and clinical uses. Other molecular tools, use of PCR to diagnose COVID-19, and the basic principles of DNA regulation are discussed separately.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>COVID-19 PCR testing </strong>– (See  <a class="medical medical_review" href="/d/html/128404.html" rel="external">"COVID-19: Diagnosis", section on 'NAAT (including RT-PCR)'</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>DNA regulation</strong> – (See  <a class="medical medical_review" href="/d/html/2900.html" rel="external">"Basic genetics concepts: DNA regulation and gene expression"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Genetic testing</strong> – (See  <a class="medical medical_review" href="/d/html/2901.html" rel="external">"Genetic testing"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>DNA sequencing</strong> – (See  <a class="medical medical_review" href="/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Cytogenetics </strong>– (See  <a class="medical medical_review" href="/d/html/2893.html" rel="external">"Tools for genetics and genomics: Cytogenetics and molecular genetics"</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Gene expression profiling</strong> – (See  <a class="medical medical_review" href="/d/html/14602.html" rel="external">"Tools for genetics and genomics: Gene expression profiling"</a>.)</p><p></p><p class="headingAnchor" id="H2834011273"><span class="h1">BACKGROUND INFORMATION</span></p><p class="headingAnchor" id="H3188475391"><span class="h2">Terminology</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Nucleic acid testing (NAT)</strong> – Refers broadly to any testing or diagnostic assay that analyzes nucleic acids (ie, DNA, RNA), as opposed to a serologic assay for an antigen. Typically applied to infectious disease testing. In practice, most NAT assays use PCR to amplify the nucleic acid of the target organism. (See <a class="local">'Infectious diseases'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>PCR</strong> – Polymerase chain reaction (PCR) is the general term that refers to the method of amplifying specified DNA segments by successive cycles of DNA replication using a thermostabile DNA polymerase and a set of primers that specify the DNA segment to be amplified  (<a class="graphic graphic_figure graphicRef64383" href="/d/graphic/64383.html" rel="external">figure 1</a>). (See <a class="local">'PCR process'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>RT-PCR</strong> – Reverse transcription PCR (RT-PCR) is used to convert a starting template of RNA into complementary DNA (cDNA), which can then be amplified by standard PCR methodologies. Many viral genomes are RNA-based rather than DNA-based, including human immunodeficiency virus (HIV), and RT-PCR is used to amplify these RNA-based organisms. (See <a class="local">'Monitoring viral load'</a> below and <a class="local">'Screening donated blood'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>qPCR</strong> – Real-time quantitative PCR (also called RT-qPCR or qPCR) is a highly sensitive method for quantifying the absolute or relative amount of a specific nucleic acid sequence. In qPCR, a fluorescent probe included in the PCR reaction is able to hybridize to the target sequence  (<a class="graphic graphic_figure graphicRef81663" href="/d/graphic/81663.html" rel="external">figure 2</a>), and the accumulation of fluorescently labeled PCR products is measured directly, without post-PCR modifications  (<a class="graphic graphic_figure graphicRef71655" href="/d/graphic/71655.html" rel="external">figure 3</a>). Details of the probe and a description of the methodology are provided below. (See <a class="local">'PCR reaction mixture'</a> below.)</p><p></p><p class="bulletIndent1">Due to the confusion caused by two meanings for RT (real-time and reverse transcriptase), it has been proposed that qPCR refer to the use of real time PCR for quantitative applications, and RT-PCR be used for more general use of reverse transcriptase PCR, but this convention has not been uniformly adopted [<a href="#rid1">1</a>].</p><p></p><p class="bulletIndent1">Common clinical applications of qPCR include quantification of viral load (eg, for SARS-CoV-2 RNA), determination of DNA copy number, measuring gene expression levels, and monitoring of measurable residual disease after treatment for hematologic malignancy. Supporting data are presented in separate reviews. (See  <a class="medical medical_review" href="/d/html/128404.html" rel="external">"COVID-19: Diagnosis", section on 'NAAT (including RT-PCR)'</a> and  <a class="medical medical_review" href="/d/html/4500.html" rel="external">"Genetic abnormalities in hematologic and lymphoid malignancies"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Cycle threshold </strong>– Cycle threshold (C<sub>t</sub>) is the qPCR-estimated value used to express the amount of DNA in a sample. C<sub>t</sub> is defined as the number of PCR cycles required for the reaction-generated fluorescent signal to exceed a prespecified threshold. </p><p></p><p class="bulletIndent1">Under normal conditions, fluorescence increases with each successive qPCR cycle, proportional to the starting abundance of DNA. The lower the starting concentration of DNA, the more cycles are needed to cross a given threshold of fluorescence  (<a class="graphic graphic_figure graphicRef141614" href="/d/graphic/141614.html" rel="external">figure 4</a>). Thus, a high C<sub>t</sub> indicates a low starting amount of DNA, and a low C<sub>t</sub> indicates a greater amount of DNA in the starting material (eg, greater viral load for HIV or SARS-CoV-2). The threshold is calibrated for individual qPCR assays, typically set at a multiple of the background fluorescent intensity (ie, the noise). (See <a class="local">'Clinical applications'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Multiplex PCR</strong> – This refers to PCR reactions assaying two or more distinct nucleic acid sequences simultaneously in the same sample. Multiplex PCR allows the same sample to be tested for multiple sequences (ie, different genes, different DNA variants, different strains of a microorganism). Multiplexing is often used for microbiologic screening of respiratory samples in a patient with suspected respiratory infection or cerebrospinal fluid (CSF) samples to test a panel of possible pathogens in a patient with suspected meningitis. (See <a class="local">'Infectious disease diagnosis'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Digital PCR</strong> – Digital PCR (dPCR), also known as digital drop PCR, is an adaptation of traditional PCR for the purposes of sample quantification without the need for multiple PCR cycles. </p><p></p><p class="bulletIndent1">The basic principle of dPCR is that a sample is separated (using complex microfluidics) into discrete aliquots (packets), each containing a very small number of nucleic acid sequences from the patient (on average, one). PCR is then performed in each packet, resulting in a binary result in which the target nucleic acid of interest is either present or absent. Subsequent modeling (assuming a Poisson distribution) enables estimation of the starting amount of target nucleic acid without the reliance on multiple amplification cycles as required in qPCR. The measured quantity in dPCR is thus considered an absolute measure, rather than the relative measure of traditional PCR (compared to the threshold value of a standard). dPCR is being used in several clinical settings, including testing related to infectious diseases and cancer [<a href="#rid2">2,3</a>]. (See <a class="local">'Infectious diseases'</a> below and <a class="local">'Cancer testing'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Genotyping</strong> – Genotyping is the process of characterizing an individual's genotype (the combination of alleles) at a particular locus (genomic location). Most types of genetic variation can be characterized using PCR techniques, including microsatellite repeats and single nucleotide polymorphism (SNP) markers, insertion/deletion variants (indels), and some structural variants, such as copy-number variants.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Sensitivity/limit of detection (LOD)</strong> – PCR has high analytical sensitivity (ability to detect extremely low levels of nucleic acids); the LOD is used to specify how many nucleic acid molecules can reliably be detected in a PCR reaction [<a href="#rid1">1</a>]. Analytical sensitivity should not be confused with clinical sensitivity, which refers to the number of patients with a disease who test positive. (See  <a class="medical medical_review" href="/d/html/2759.html" rel="external">"Glossary of common biostatistical and epidemiological terms", section on 'Sensitivity and specificity'</a>.)</p><p></p><p class="headingAnchor" id="H668606283"><span class="h2">Conceptual background</span></p><p class="headingAnchor" id="H2281834974"><span class="h3">How it works</span><span class="headingEndMark"> — </span>Similar to DNA replication that occurs during cell division, PCR rapidly amplifies targeted DNA regions by separating the two DNA strands that comprise the double helix and using DNA polymerase to replicate both strands. (See  <a class="medical medical_review" href="/d/html/129424.html" rel="external">"Basic genetics concepts: Chromosomes and cell division", section on 'DNA replication (S phase)'</a>.)</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Thermostable polymerase</strong> – What makes PCR so useful and different from prior uses of DNA polymerase is that the polymerase used in PCR is thermostable. First discovered in bacteria that live in thermal vents and hot springs, this polymerase is able to tolerate repeated cycles of exposure to extremely high temperatures (95°C) without becoming denatured [<a href="#rid4">4</a>]; it functions optimally near 70°C [<a href="#rid5">5</a>]. </p><p></p><p class="bulletIndent1">The high temperatures are needed to "melt" the DNA into separate sense and antisense strands for the polymerase to bind and elongate new DNA strands. Because the polymerase can tolerate the high temperatures needed to melt the DNA, multiple cycles of DNA melting followed by DNA polymerization can be repeated without the need to open the tube and replenish the reaction mix with fresh polymerase. As a result, DNA amplification can be automated. This advance allows for repeated brief rounds of DNA amplification (3 to 5 minutes) with high fidelity, allowing exponential amplification of any DNA sequence. </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Taq</strong> – The standard enzyme used in most PCR applications is derived from the bacterium <em>Thermus</em> (also <em>Thermophilus</em>) <em>aquaticus</em> (Taq). The error rate (number of incorrectly inserted nucleotides per nucleotide per cycle) for standard Taq is approximately 1 error per 100,000 bases (1 in 10<sup>5</sup>), which is considered acceptable for most clinical diagnostic applications as it is too low to impact assay accuracy.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Pfu</strong> – High-fidelity polymerases with in vitro proofreading ability, such as those derived from the archaebacterium <em>Pyrococcus furiosus</em> (Pfu), have error rates that are approximately 10 times lower (1 in 10<sup>6</sup>) and are used for applications that mandate more stringent DNA synthesis, such as gene cloning) [<a href="#rid6">6</a>]. </p><p></p><p class="bulletIndent1">Details of the regulation of DNA polymerase are presented separately. (See  <a class="medical medical_review" href="/d/html/129424.html" rel="external">"Basic genetics concepts: Chromosomes and cell division", section on 'DNA replication (S phase)'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>DNA template</strong> – The length of the DNA to be amplified can range from approximately 50 bases to 10 kilobases (kb). Due to the exponential amplification, the amount of starting DNA can be vanishingly small (as little as a single DNA molecule). (See <a class="local">'Sources and amount of template DNA or RNA'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Primers </strong>– The key to accurately amplifying a specific DNA target is the design of a pair of "PCR primers" – short DNA sequences that are complementary to the sense and antisense strands of the DNA and flank the target sequence on both sides, as illustrated by the blue and green arrows in the figure  (<a class="graphic graphic_figure graphicRef64383" href="/d/graphic/64383.html" rel="external">figure 1</a>). Binding of the primers to the complementary DNA sequence forms a duplex that the DNA polymerase can use to extend the primer using the DNA target as a template, resulting in the polymerase amplifying the DNA sequence. (See <a class="local">'Primer design'</a> below.)</p><p></p><p class="bulletIndent1">The main sources of error relate to poor primer design and DNA contamination. (See <a class="local">'Primer design'</a> below and <a class="local">'Controls'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Impact</strong> – Since its introduction, the impact of PCR on biomedical research and clinical testing has been immense [<a href="#rid7">7,8</a>]. PCR is relatively inexpensive and easy enough to perform for most molecular biology laboratories. For this advance, Kary Mullins was awarded the 1993 Nobel Prize in Chemistry [<a href="#rid9">9</a>].</p><p></p><p class="headingAnchor" id="H582040031"><span class="h3">Finding unique DNA sequences to amplify</span><span class="headingEndMark"> — </span>Successful PCR requires design of high-fidelity primers that target specific DNA sequences. Although the human genome sequence is very long (3 billion bases), a relatively short number of bases is sufficient to specify a unique sequence in the genome. The expected count of a specified sequence of length N bases in one human genome can be expressed by the following formula, based on probability theory:</p><div class="formulaContainer"><div class="formula"><p class="bulletIndent1">    (3 X 10<sup>9</sup>)<sup> </sup>x 1/4<sup>N</sup>   </p></div></div><p></p><p>In the above equation, 1/4<sup>N</sup> is the probability of a given sequence of length N bases, and the constant 3 X 10<sup>9</sup> is the number of bases in the human genome.</p><p>Thus, PCR primers of 20 and 25 nucleotides in length would be estimated to occur by chance at a frequency of 0.003 and 0.000003 per genome, respectively. Given that PCR requires both primers to colocalize to a relatively small genomic region with a specific orientation to each other (one on each DNA strand, 'facing' each other), the probability of amplifying more than one DNA region is exceedingly small.</p><p>Automated sequence alignment algorithms are available to enable rapid primer design based on a known genome sequence; these can also be used to verify that a primer pair will result in amplification of a unique sequence in the genome and to calculate the exact length of the expected amplicon [<a href="#rid10">10</a>].</p><p class="headingAnchor" id="H1396088842"><span class="h1">PCR PROCESS</span></p><p class="headingAnchor" id="H1975099256"><span class="h2">PCR reaction mixture</span><span class="headingEndMark"> — </span>The PCR reaction occurs in a single tube. For high-throughput PCR testing, multiple reactions can be run simultaneously using plates with multiple small wells.</p><p class="bulletIndent1"><span class="glyph">●</span><strong>What's in the tube</strong> – The PCR reaction mixture contains the following components:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>One or more DNA (or RNA) samples to be amplified (the "template"), often genomic DNA.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Short oligonucleotide primers (typically 21 to 23 bases long) that flank and specify the portion of DNA to be amplified.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Deoxynucleotide triphosphates (DNA bases) to use in chain elongation.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Thermostable DNA polymerase (the enzyme that elongates the primers when the appropriate template is present).</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>A buffer to maintain optimal pH for the DNA polymerase.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>In quantitative PCR (qPCR) reactions (see <a class="local">'Terminology'</a> above), a fluorescent probe.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Fluorescent probes for qPCR</strong> – In qPCR, a fluorescent probe is used to determine the amount of amplified DNA at each cycle. The amount of amplified DNA correlates with the amount of starting material in the sample. Fluorescence is very sensitive and can be used to detect small amounts of DNA in the mixture.</p><p></p><p class="bulletIndent1">Each probe is unique to each DNA target sequence. Probes are designed to bind to a region of the amplified DNA between (and not overlapping with) the two flanking PCR primers. The amount of probe bound is proportional to the amount of DNA that has been generated in each PCR cycle, which in turn is proportional to the amount of starting material in the sample.</p><p></p><p class="bulletIndent1">An ingenious design allows the probe to generate fluorescence when it is bound to the target DNA but not when it is free/unbound in the reaction tube  (<a class="graphic graphic_figure graphicRef81663" href="/d/graphic/81663.html" rel="external">figure 2</a>). The fluorescent reporter dye is located at the 5' end of the probe, and a quencher that blocks fluorescence is located at the 3' end of the probe. The quencher blocks fluorescence emission while it is in close proximity to the reporter. When the probe is free in solution, the quencher quenches fluorescence. When the probe binds to the target DNA, it is susceptible to digestion by the DNA polymerase participating in the PCR reaction. DNA polymerase has a 3' exonuclease function that can remove the quencher and dissociate the reporter, allowing the reporter to emit fluorescence. Due to the exponential nature of the PCR, the fluorescence signal increases proportionally to the amount of generated PCR product until a plateau is reached. This type of qPCR probe is also referred to as a "hydrolysis probe" [<a href="#rid8">8</a>]. </p><p></p><p class="bulletIndent1">Quantification is accomplished by comparing the cycle number at which the patient sample reaches a predetermined level of fluorescence to a standardized curve of a control sample, thus deriving copy number at the start of the reaction  (<a class="graphic graphic_figure graphicRef71655" href="/d/graphic/71655.html" rel="external">figure 3</a>). </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>The machine</strong> – This mixture can be placed in a PCR thermocycler that is preprogrammed with the optimal reaction cycle times and temperatures for amplifying the DNA sequence of interest. (See <a class="local">'Steps in the PCR process'</a> below.)</p><p></p><p class="bulletIndent1">For most clinical uses, these machines are very straightforward to operate and have a small footprint, taking up minimal space on a laboratory bench. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Controls</strong> – Appropriate positive and negative controls must be included for accuracy and troubleshooting. Because PCR is so sensitive and can amplify minute amounts of DNA, the negative control is especially important to ensure that the reagents in the reaction mixture are not contaminated with DNA from the individuals performing the PCR reaction, equipment used in pipetting or storing reagents, or environmental sources of contamination [<a href="#rid11">11</a>]. (See <a class="local">'Controls'</a> below.)</p><p></p><p class="headingAnchor" id="H659740221"><span class="h2">Steps in the PCR process</span><span class="headingEndMark"> — </span>Once the reaction mixture is assembled, three steps are repeated 30 to 40 times  (<a class="graphic graphic_figure graphicRef64383" href="/d/graphic/64383.html" rel="external">figure 1</a>). </p><p class="bulletIndent1"><span class="glyph">●</span><strong>Denaturing</strong> – The mixture of starting material (typically DNA from a patient sample) is heated to 95°C to denature the double-stranded template DNA into single strands. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Annealing</strong> – The mixture is cooled to a temperature just below the predicted melting temperatures of the primer pairs, resulting in primer binding to the single-stranded DNA template strands. The thermostable DNA polymerase can bind to the 3’ end of the primers.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Elongation</strong> – The temperature is raised to a temperature at which the thermostable DNA polymerase initiates chain elongation. Typical elongation temperature is approximately 70 to 72°C. The polymerase adds free nucleotides to the 3' ends of the primers, using the DNA template to determine which base to add next. Complementary bases to the template sequence (A with T, C with G) are added to the elongating PCR product. </p><p></p><p class="bulletIndent1">Elongation continues for about one minute and is terminated by raising the temperature to 95°C, resulting in strand separation (denaturing), to restart the cycle again. </p><p></p><p class="bulletIndent1">Amplification is exponential. The number of potential targets (templates) for primer annealing doubles with each PCR cycle since the amplified sequences produced in one cycle can serve as templates for subsequent cycles. </p><p></p><p>If the starting material is RNA (such as from a virus), the RNA must first be transcribed using reverse transcriptase to create a DNA template. This is referred to as reverse transcriptase PCR (RT-PCR). (See <a class="local">'Terminology'</a> above.)</p><p class="headingAnchor" id="H1831786029"><span class="h2">Primer design</span><span class="headingEndMark"> — </span>Primers must be carefully designed to ensure that they will amplify a unique sequence of DNA. This requires that primers:</p><p class="bulletIndent1"><span class="glyph">●</span>Are spaced a reasonable distance apart on sense and antisense strands</p><p class="bulletIndent1"><span class="glyph">●</span>Only recognize a single DNA sequence</p><p class="bulletIndent1"><span class="glyph">●</span>Bind with appropriate and similar characteristics (eg, similar GC content, which affects melting temperature)</p><p class="bulletIndent1"><span class="glyph">●</span>For infectious disease testing, primers cannot amplify any host DNA</p><p></p><p>Software that facilitates primer design and calculates annealing temperatures is freely distributed online [<a href="#rid12">12</a>].</p><p class="headingAnchor" id="H2649931666"><span class="h2">Sources and amount of template DNA or RNA</span><span class="headingEndMark"> — </span>Nearly any source of template DNA or RNA can be used for PCR.</p><p class="bulletIndent1"><span class="glyph">●</span>Body fluids and secretions, including saliva, sputum, nasal discharge, cerebrospinal fluid (CSF), and serum or plasma.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Tissues and cell collections, including blood cells, buccal swabs, bronchoalveolar lavage samples, fresh biopsy samples, and archived cell lines.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Archived tissue blocks, including tissue treated with formaldehyde or fixed in paraffin.</p><p></p><p>It is possible to start with as little as a single DNA molecule as the template and generate tens of millions of copies. If the reaction starts with two copies, then N cycles will theoretically yield 2<sup>N</sup> copies.</p><p>However, the quality of the sample DNA does impact the success of the reaction.</p><p class="bulletIndent1"><span class="glyph">●</span>Highly fragmented or degraded DNA can result in allelic dropout (selective amplification of only one allele at a heterozygous locus) or false-negative results. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Impurities in the sample could inhibit amplification. This is of particular concern in qPCR since inhibitors will lower reaction efficiency.</p><p></p><p class="headingAnchor" id="H651024205"><span class="h2">Controls</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Positive and negative controls</strong> – Technical positive and negative control reactions should be performed with all PCR assays to demonstrate:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Normal assay performance, in which template amplification occurs when the target DNA sequence is present (positive control).</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Assay reagents are not contaminated with extraneous DNA (negative control, also called no template control).</p><p></p><p class="bulletIndent1">Test conditions for control samples should mirror those of clinical samples. The positive control template should be prepared in the same manner as the test sample, with the positive control DNA typically extracted using the same extraction protocol and starting material as the sample. The negative control involves using all components of the PCR mixture except that water is added in lieu of the DNA template.</p><p></p><p class="bulletIndent1">Because of the extreme analytical sensitivity of PCR to detect DNA sequence, contamination of the sample with small amounts of extraneous DNA may result in a spurious or false-positive finding. To help decrease this problem, diagnostic PCR laboratories are often subdivided into separate, designated pre- and post-amplification areas in different rooms.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Assay-specific controls</strong> – Additional controls relevant to the specific assay may also be required. For example, when testing a parent for a sequence variant present in their offspring, the child's DNA may be required as a positive control for the PCR and sequencing with the parent's DNA.</p><p></p><p class="headingAnchor" id="H85281182"><span class="h2">Results</span><span class="headingEndMark"> — </span>PCR results can be reported as either binary (positive or negative) or quantitatively:</p><p class="bulletIndent1"><span class="glyph">●</span>For some applications (for example, DNA screening for the presence or absence of specific microorganisms), results are reported as either "positive/detected" or "negative/not detected."</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Quantitative PCR assays are used to measure nucleic acid abundance (for example, as surrogates of viral load). In this instance, the resulting cycle threshold (C<sub>t</sub>) values are typically converted to more meaningful clinical values (viral load, DNA copy number), by normalizing to C<sub>t </sub>values of quantitative standard controls of varying concentration that are run in parallel with the clinical assay. C<sub>t</sub> is defined above (see <a class="local">'Terminology'</a> above); examples are provided below. (See <a class="local">'Monitoring viral load'</a> below.)</p><p></p><p class="headingAnchor" id="H4247812164"><span class="h1">ADVANTAGES AND DISADVANTAGES</span></p><p class="headingAnchor" id="H4008922523"><span class="h2">Advantages of PCR</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Speed</strong> – The turnaround time for some applications is less than one hour [<a href="#rid8">8</a>]. Other applications such as large-scale commercial testing may take slightly longer (two to four hours), depending on the time required to prepare the sample. Single tube reactions enable more rapid sample processing and point-of-care testing.</p><p></p><p class="bulletIndent1">For infectious disease testing, PCR and other nucleic acid tests are significantly faster than culture-based assays, which can take days to weeks. (See <a class="local">'Infectious diseases'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Detection limit</strong> – PCR can amplify the DNA from a single cell or a single molecule of DNA and thus is extremely sensitive, sufficiently so that it can be used in preimplantation genetic testing in embryos or to detect a very low level of an infectious virus. This is also called high analytical sensitivity. (See <a class="local">'Terminology'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Low cost and ease of use</strong> – Cost is low because the components of the reaction mixture are inexpensive and human input during the process can be minimized, especially with automation. The main expenses are the cost of the PCR machine and the costs associated with sample collection and maintaining an appropriately clean facility for testing.</p><p></p><p class="headingAnchor" id="H2092657330"><span class="h2">Disadvantages of PCR</span><span class="headingEndMark"> — </span>PCR has disadvantages worth considering [<a href="#rid8">8</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Inability to identify new DNA variants or new microorganisms</strong> – To design appropriate PCR primers, the nucleotide sequence information of the region of interest must be known. This limits its use when attempting to isolate and amplify novel gene sequences. New organisms with previously undiscovered sequences may not be detected, or, if their sequences are similar to previously discovered organisms, they may be misclassified. Direct sequencing, including next generation DNA sequencing (NGS) assays, are required to identify unsuspected sequences. (See  <a class="medical medical_review" href="/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications", section on 'Diagnosis of infections'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Contamination</strong> – Because PCR is so sensitive, the possibility of introducing contaminating DNA is always a concern, especially with infectious disease testing when infectious organisms are present in the same room.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Error rates with longer target sequences</strong> – Although long-range PCR methods are available to amplify 5 to 10 kilobase (kb) fragments of DNA, they are susceptible to higher error rates and are less robust. Similarly, large copy number variants, such as deletions that encompass the amplified region, cannot be detected because only the unaffected allele will be amplified.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Inability to distinguish live from dead pathogens</strong> – PCR is highly sensitive for detecting pathogen DNA or RNA, but it cannot distinguish whether the nucleic acid is from a live organism or residual nucleic acid from a dead organism, since fragments of nucleic acid can persist for long periods of time after the organism has been killed. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Challenges in identifying rare sequences</strong> – It is possible for abundant templates to obscure detection of rare templates if both are recognized by the same primer pair. Modifications of PCR reactions have been developed to improve detection of rare sequences [<a href="#rid13">13</a>]. </p><p></p><p class="headingAnchor" id="H8006635"><span class="h1">CLINICAL APPLICATIONS</span><span class="headingEndMark"> — </span>Since its introduction, PCR has rapidly become indispensable for clinical care and research. </p><p>As with any clinical laboratory testing, laboratories that use PCR must be certified according to Clinical Laboratory Improvement Amendments (CLIA) regulations.</p><p class="headingAnchor" id="H685260379"><span class="h2">Infectious diseases</span><span class="headingEndMark"> — </span>Due to many favorable performance characteristics over culture-based methods (turnaround time of hours for PCR versus days for culture) and serologic-based tests (lower detection limit for PCR versus less diagnostic sensitivity for serologies), PCR is frequently used as both a first-line or confirmatory diagnostic assay for many pathogens. (See <a class="local">'Advantages of PCR'</a> above.)</p><p>PCR is also used to monitor response to therapy (for example, monitoring viral load). Provided some portion of the organism's DNA or RNA sequence is known, PCR can be used to identify the presence of any pathogen (viral, bacterial, fungal, protozoal). Robust nucleic acid extraction protocols have been developed so that PCR can be adapted to assay virtually any type of clinical sample, including blood, respiratory secretions, cerebrospinal fluid (CSF), and fresh tissue samples. It cannot be used to identify new (previously unknown) pathogens, and it cannot distinguish live pathogens from residual nucleic acids derived from organisms that have been killed. (See <a class="local">'Disadvantages of PCR'</a> above.)</p><p class="headingAnchor" id="H2840571500"><span class="h3">Infectious disease diagnosis</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>COVID-19</strong> – PCR assays to detect SARS-CoV-2 viral RNA from clinical samples were developed and deployed within weeks of publication of the SARS-CoV-2 sequence. They are routinely used to diagnose active COVID-19 infection, to confirm immunoassay-based results, and to identify asymptomatic SARS-CoV-2 infection. Details are discussed separately. (See  <a class="medical medical_review" href="/d/html/128404.html" rel="external">"COVID-19: Diagnosis", section on 'NAAT (including RT-PCR)'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Other viruses</strong> – Use of PCR assays to detect other common viral pathogens is commonplace. An example is herpes simplex virus (HSV) testing in CSF. (See  <a class="medical medical_review" href="/d/html/8286.html" rel="external">"PCR testing for the diagnosis of herpes simplex virus in patients with encephalitis or meningitis"</a>.)</p><p></p><p class="bulletIndent1">Multiplex assays that include probes targeting diverse viral sequences have been developed to distinguish between different viral infections in the same sample. An example is reverse transcription PCR (RT-PCR) for different influenza virus strains from the same patient specimen. (See  <a class="medical medical_review" href="/d/html/5973.html" rel="external">"Seasonal influenza in children: Clinical features and diagnosis", section on 'Seasonal influenza'</a> and  <a class="medical medical_review" href="/d/html/7004.html" rel="external">"Seasonal influenza in adults: Clinical manifestations and diagnosis", section on 'Choice of assay'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Bacteria and fungi</strong> – PCR-based assays have been proposed for various bacterial and fungal infections, both for diagnosis as well as testing for antimicrobial resistance. For bacteria, numerous multiplex and broad-range PCR assays targeting the 16S rRNA sequence have been developed to simultaneously screen for multiple pathogens. These assays can be used as first-line screens or as second-line tests when cultures are negative. </p><p></p><p class="bulletIndent1">A 2022 survey of 6130 clinical samples found that broad-range PCR assays were positive for at least one organism in 45 percent of culture-negative samples of pus, abscess, or empyema; positivity rates increased to 61 percent with additional testing using a targeted multiplex PCR panel of common bacterial pathogens [<a href="#rid14">14</a>].</p><p></p><p class="bulletIndent1">Other examples include:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>ESKAPE organisms</strong> – A widely adopted use of multiplex panels that screen for enteric infections is a PCR-based panel that detec<em>ts Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Pseudomonas aeruginosa</em>, a<em>nd Escherichia coli </em>(so-called ESKAPE organisms) [<a href="#rid15">15</a>]. Magnetic beads that bind to the PCR products are detected using magnetic resonance. In a series of hospitalized patients with suspected bacterial sepsis, this test was able to positively identify an infectious organism within four to eight hours, versus two to three days for most culture results [<a href="#rid16">16</a>]. Compared with culture, the sensitivity and specificity of the PCR-based test were both 90 percent. The PCR test was also able to identify infectious organisms that were not detected by blood cultures in a small number of patients. As noted by an editorialist, this type of testing is a potentially useful adjunct to culture in obtaining more rapid results but would not reduce the need for culture, because the PCR assay does not identify organisms other than those tested, and it does not provide antibiotic sensitivities [<a href="#rid17">17</a>].</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Bloodstream infections</strong> – Larger panels for detecting bacteria and fungi such as <em>Candida</em> have been developed and have been successful in clinical situations for rapid and accurate identification of pediatric bloodstream infections, even with very small blood volumes [<a href="#rid18">18</a>]. Further information on PCR-based testing for bacteremia is discussed separately. (See  <a class="medical medical_review" href="/d/html/2133.html" rel="external">"Detection of bacteremia: Blood cultures and other diagnostic tests"</a>.)</p><p></p><p class="headingAnchor" id="H1018052039"><span class="h3">Monitoring viral load</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>HIV </strong>– PCR of RNA isolated from blood is a standard tool in monitoring the viral load in individuals with human immunodeficiency virus (HIV) infection. Several available commercial real-time quantitative PCR (qPCR) assays provide absolute quantitation of HIV-1 RNA copies per mL of plasma. (See <a class="local">'Terminology'</a> above and  <a class="medical medical_review" href="/d/html/3770.html" rel="external">"Patient monitoring during HIV antiretroviral therapy"</a> and  <a class="medical medical_review" href="/d/html/3743.html" rel="external">"Techniques and interpretation of HIV-1 RNA quantitation", section on 'Available tests'</a>.) </p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>CMV</strong> – Quantification of viral load can be used in transplant recipients being treated for cytomegalovirus (CMV) infection. (See  <a class="medical medical_review" href="/d/html/8305.html" rel="external">"Approach to the diagnosis of cytomegalovirus infection", section on 'Monitoring response to treatment'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>EBV</strong> – Epstein-Barr virus (EBV) can cause post-transplant lymphoproliferative disease (PTLD). Monitoring of EBV viral load can help inform assessment of risk and strategies for preemptive treatment. (See  <a class="medical medical_review" href="/d/html/83230.html" rel="external">"Epidemiology, clinical manifestations, and diagnosis of post-transplant lymphoproliferative disorders", section on 'Measurement of EBV viral load'</a> and  <a class="medical medical_review" href="/d/html/7335.html" rel="external">"Treatment and prevention of post-transplant lymphoproliferative disorders", section on 'Preemptive treatment of viral reactivation'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Chronic hepatitis (HBV or HCV)</strong> – Viral load is sometimes used in people with chronic hepatitis B virus (HBV) infection or chronic hepatitis C virus (HCV) infection. (See  <a class="medical medical_review" href="/d/html/89950.html" rel="external">"Screening and diagnosis of chronic hepatitis C virus infection"</a> and  <a class="medical medical_review" href="/d/html/83476.html" rel="external">"Hepatitis B and pregnancy"</a> and  <a class="medical medical_review" href="/d/html/3680.html" rel="external">"Hepatitis B virus: Screening and diagnosis in adults", section on 'Serum HBV DNA assays'</a>.)</p><p></p><p class="headingAnchor" id="H2853055615"><span class="h3">Screening donated blood</span><span class="headingEndMark"> — </span>PCR is routinely used in nucleic acid testing of donated blood for pathogens. (See  <a class="medical medical_review" href="/d/html/7946.html" rel="external">"Blood donor screening: Laboratory testing", section on 'Infectious disease screening and surveillance'</a>.)</p><p class="headingAnchor" id="H772206773"><span class="h2">Cancer testing</span></p><p class="headingAnchor" id="H64851941"><span class="h3">Identifying pathogenic variants</span><span class="headingEndMark"> — </span>PCR can be performed on solid tumor biopsies and blood samples (liquid biopsy). It is used for various types of cancer testing, especially in tumors with known pathogenic variants that may influence treatment. Examples include specific missense variants in non-small cell lung cancer and metastatic melanoma. (See  <a class="medical medical_review" href="/d/html/16538.html" rel="external">"Personalized, genotype-directed therapy for advanced non-small cell lung cancer"</a> and  <a class="medical medical_review" href="/d/html/15408.html" rel="external">"Systemic treatment of metastatic melanoma with BRAF and other molecular alterations"</a>.)</p><p>As with infectious disease testing, other types of genetic testing, such as next generation sequencing (NGS), are likely to be more appropriate when the specific gene(s) and/or pathogenic variant(s) have not been identified, or when a large number of pathogenic variants are possible. (See  <a class="medical medical_review" href="/d/html/17039.html" rel="external">"Next-generation DNA sequencing (NGS): Principles and clinical applications"</a>.)</p><p class="headingAnchor" id="H3748124551"><span class="h3">Measurable residual disease</span><span class="headingEndMark"> — </span>PCR-based assays are routinely used for assessing measurable residual disease (MRD; also called minimal residual disease) in hematologic malignancies. (See  <a class="medical medical_review" href="/d/html/4519.html" rel="external">"Clinical use of measurable residual disease detection in acute lymphoblastic leukemia"</a> and  <a class="medical medical_review" href="/d/html/4525.html" rel="external">"Acute myeloid leukemia: Induction therapy in medically fit adults", section on 'Measurable residual disease'</a> and  <a class="medical medical_review" href="/d/html/4501.html" rel="external">"Evaluating response to treatment of chronic lymphocytic leukemia"</a> and  <a class="medical medical_review" href="/d/html/4494.html" rel="external">"Hematopoietic cell transplantation in chronic myeloid leukemia", section on 'Monitoring MRD post-HCT'</a>.)</p><p class="headingAnchor" id="H1182864396"><span class="h2">Obstetrics</span><span class="headingEndMark"> — </span>PCR is used for various indications including:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Preimplantation genetic testing</strong> – Preimplantation genetic testing (PGT) involves screening of early embryos in vitro (prior to uterine transfer) for specific pathogenic variants, such as disease variants present in one or both of the parents. PCR is done on a single cell that has been removed from the embryo. Technical considerations in single-cell PCR include preventing allele dropout and avoiding contamination with non-embryo DNA [<a href="#rid19">19</a>]. (See  <a class="medical medical_review" href="/d/html/6783.html" rel="external">"Preimplantation genetic testing", section on 'Genetic analysis'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Noninvasive prenatal testing</strong> – Noninvasive prenatal testing (NIPT) using free fetal DNA (ffDNA; also known as cell-free fetal DNA, or cfDNA) in maternal blood can be used to detect pathogenic variants or other clinically significant variants such as blood type in a fetus. Since ffDNA makes up only a small percentage of the circulating DNA, care must be taken to differentiate fetal from maternal sequences [<a href="#rid20">20</a>]. (See  <a class="medical medical_review" href="/d/html/458.html" rel="external">"Prenatal screening for common aneuploidies using cell-free DNA"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>CVS or amniocentesis</strong> – PCR can also be used to amplify DNA obtained from chorionic villus sampling (CVS) or amniocentesis. After PCR amplification, the PCR products are typically sequenced. (See  <a class="medical medical_review" href="/d/html/449.html" rel="external">"Chorionic villus sampling"</a> and  <a class="medical medical_review" href="/d/html/5390.html" rel="external">"Diagnostic amniocentesis"</a>.)</p><p></p><p class="headingAnchor" id="H1172066351"><span class="h2">Testing for other genetic variation</span><span class="headingEndMark"> — </span>PCR is used in other molecular tests, especially those for which a specific pathogenic variant(s) is known. For known variants, PCR primers can be designed that only amplify the variant sequence. The factor V Leiden thrombophilia variant is one such example. (See  <a class="medical medical_review" href="/d/html/122111.html" rel="external">"Gene test interpretation: Factor V Leiden"</a> and  <a class="medical medical_review" href="/d/html/1355.html" rel="external">"Factor V Leiden and activated protein C resistance", section on 'Genetic testing'</a>.)</p><p class="headingAnchor" id="H2387523049"><span class="h1">RESEARCH APPLICATIONS</span><span class="headingEndMark"> — </span>PCR is used extensively in clinical and basic science research. </p><p class="headingAnchor" id="H98300441"><span class="h2">Clinical research</span><span class="headingEndMark"> — </span>PCR is useful in hypothesis-generating research that asks questions such as:</p><p class="bulletIndent1"><span class="glyph">●</span>Do outcomes differ in patients whose tumors carry a specific pathogenic variant?</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>How likely is it for a patient with a specific tumor variant to carry the variant in the germline?</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Can diagnosis of infectious diseases be accelerated using DNA or RNA-based testing?</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Does treatment based on molecular testing provide better outcomes than treatment based on other types of testing such as bacterial culture or histopathologic diagnosis?</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Does a patient carry a variant that is diagnostic for a rare disease? </p><p></p><p class="headingAnchor" id="H2909801892"><span class="h2">Basic science research</span><span class="headingEndMark"> — </span>PCR is useful in numerous research applications; almost any molecular biology experiment can use PCR. Examples include:</p><p class="bulletIndent1"><span class="glyph">●</span>Assessing gene expression profiles in tumor DNA and adjacent normal cellular DNA to identify pathogenesis mechanisms or druggable therapeutic targets. An example is the discovery that a Herpes simplex-related virus is involved in the pathogenesis of Kaposi's sarcoma [<a href="#rid21">21-23</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Determining whether a class of tumors contains viral DNA that might suggest a pathogenic role.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Confirming rare variants identified by next-generation sequencing by amplifying specific fragments with PCR and then performing Sanger sequencing. After confirmation of rare variant(s), relatives can be tested using PCR and Sanger sequencing. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genotyping DNA samples from specific animal models.</p><p></p><p class="headingAnchor" id="H15008227"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Background </strong>– Polymerase chain reaction (PCR) is a technique for rapid and highly specific amplification of a region of DNA  (<a class="graphic graphic_figure graphicRef64383" href="/d/graphic/64383.html" rel="external">figure 1</a>). What makes PCR different from prior uses of DNA polymerase is that the polymerase in PCR can tolerate high temperatures needed to melt double-stranded DNA, allowing exponential amplification without the need to open the tube and add fresh polymerase. The technique was developed in the 1980s and led to a Nobel Prize in 1993. Terms for various types of PCR and quantification techniques are defined above. The figure illustrates the concept of cycle threshold for quantitative PCR  (<a class="graphic graphic_figure graphicRef141614" href="/d/graphic/141614.html" rel="external">figure 4</a>). (See <a class="local">'Background information'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Method</strong> – The reaction mixture contains a DNA template (patient sample), primers that specify the region of DNA to be amplified, free nucleotides, and a thermostable polymerase. Once the reaction mixture is assembled, three steps (denaturation, elongation, and annealing) are repeated 30 to 40 times  (<a class="graphic graphic_figure graphicRef64383" href="/d/graphic/64383.html" rel="external">figure 1</a>) in a preprogrammed PCR machine. An RNA template (from a virus) can also be used, and fluorescent probes can be incorporated for quantification. Software that facilitates primer design and calculates annealing temperatures is widely available. Positive and negative controls are essential. (See <a class="local">'PCR process'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Advantages and disadvantages </strong>– Advantages of PCR over other DNA amplification methods and other types of testing (bacterial culture, histopathology review) include rapid turnaround time, high analytical sensitivity, low cost, and ease of automation. PCR is not useful for identifying new DNA sequences or new microorganisms; next-generation DNA sequencing is used for these applications. False-positive results due to contaminating DNA are always a concern. Error rates increase with longer sequences, and PCR cannot distinguish live from dead organisms. (See <a class="local">'Advantages and disadvantages'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Clinical uses</strong> </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Infectious diseases</strong> – PCR can be used to identify the presence of any pathogen (viral, bacterial, fungal, protozoal) as long as some portion of the organism's DNA or RNA sequence is known. It is more sensitive than serologic assays and culture-based assays for most infectious diseases. It is routinely used to diagnose COVID-19 or asymptomatic SARS-CoV-2 infection. Clinical assays using multiplex PCR test for multiple possible pathogens in the same specimen such as respiratory secretions or cerebrospinal fluid (CSF). Other uses include monitoring viral load and screening donated blood. (See <a class="local">'Infectious diseases'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Cancer</strong> – PCR can be performed on solid tumor biopsies or blood samples. It is used frequently in genetic diagnosis and monitoring of measurable residual disease (MRD) after treatment. (See <a class="local">'Cancer testing'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Obstetrics</strong> – PCR is essential for preimplantation genetic testing (PGT) and can identify free fetal DNA in maternal blood. (See <a class="local">'Obstetrics'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Research uses</strong> – PCR is used extensively in clinical and basic science research. (See <a class="local">'Research applications'</a> above.)</p><p></p><p></p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Bustin SA, Benes V, Garson JA, et al. The MIQE guidelines: minimum information for publication of quantitative real-time PCR experiments. Clin Chem 2009; 55:611.</a></li><li><a class="nounderline abstract_t">Vogelstein B, Kinzler KW. Digital PCR. Proc Natl Acad Sci U S A 1999; 96:9236.</a></li><li><a class="nounderline abstract_t">Vynck M, Trypsteen W, Thas O, et al. The Future of Digital Polymerase Chain Reaction in Virology. Mol Diagn Ther 2016; 20:437.</a></li><li><a class="nounderline abstract_t">Chien A, Edgar DB, Trela JM. Deoxyribonucleic acid polymerase from the extreme thermophile Thermus aquaticus. J Bacteriol 1976; 127:1550.</a></li><li><a class="nounderline abstract_t">Templeton NS. The polymerase chain reaction. History, methods, and applications. Diagn Mol Pathol 1992; 1:58.</a></li><li><a class="nounderline abstract_t">Eckert KA, Kunkel TA. DNA polymerase fidelity and the polymerase chain reaction. PCR Methods Appl 1991; 1:17.</a></li><li><a class="nounderline abstract_t">Csako G. Present and future of rapid and/or high-throughput methods for nucleic acid testing. Clin Chim Acta 2006; 363:6.</a></li><li><a class="nounderline abstract_t">Mahanama A, Wilson-Davies E. Insight into PCR testing for surgeons. Surgery (Oxf) 2021; 39:759.</a></li><li><a class="nounderline abstract_t">Comfort N. Essay reviews. [Review of: Rabinow P. Making PCR: a story of biotechnology. University of Chicago Press, 1996; and Fujimura J. Crafting science: a sociohistory of the quest for the genetics of cancer. Harvard University Press, 1996]. Oral Hist Rev 1999; 26:181.</a></li><li><a class="nounderline abstract_t">Zweig AS, Karolchik D, Kuhn RM, et al. UCSC genome browser tutorial. Genomics 2008; 92:75.</a></li><li><a class="nounderline abstract_t">Hirschhorn JW, Avery A, Schandl CA. Managing a PCR Contamination Event in a Molecular Pathology Laboratory. Methods Mol Biol 2023; 2621:15.</a></li><li class="breakAll">Rozen S, Skaletsky HJ. Primer3 on the WWW for general users and for biologist programmers. In: Bioinformatics Methods and Protocols: Methods in Molecular Biology, Krawetz S, Misener S (Eds), Humana Press, Totowa, NJ 2000. p.365.</li><li><a class="nounderline abstract_t">Tanigawa N, Fujita T, Fujii H. Oligoribonucleotide (ORN) interference-PCR (ORNi-PCR): a simple method for suppressing PCR amplification of specific DNA sequences using ORNs. PLoS One 2014; 9:e113345.</a></li><li><a class="nounderline abstract_t">Harris KA, Brown JR. Diagnostic yield of broad-range 16s rRNA gene PCR varies by sample type and is improved by the addition of qPCR panels targeting the most common causative organisms. J Med Microbiol 2022; 71.</a></li><li><a class="nounderline abstract_t">De Angelis G, Posteraro B, De Carolis E, et al. T2Bacteria magnetic resonance assay for the rapid detection of ESKAPEc pathogens directly in whole blood. J Antimicrob Chemother 2018; 73:iv20.</a></li><li><a class="nounderline abstract_t">Nguyen MH, Clancy CJ, Pasculle AW, et al. Performance of the T2Bacteria Panel for Diagnosing Bloodstream Infections: A Diagnostic Accuracy Study. Ann Intern Med 2019; 170:845.</a></li><li><a class="nounderline abstract_t">Weinrib DA, Capraro GA. The Uncertain Clinical Benefit of the T2Bacteria Panel. Ann Intern Med 2019; 170:888.</a></li><li><a class="nounderline abstract_t">Lucignano B, Cento V, Agosta M, et al. Effective Rapid Diagnosis of Bacterial and Fungal Bloodstream Infections by T2 Magnetic Resonance Technology in the Pediatric Population. J Clin Microbiol 2022; 60:e0029222.</a></li><li><a class="nounderline abstract_t">Spits C, Sermon K. PGD for monogenic disorders: aspects of molecular biology. Prenat Diagn 2009; 29:50.</a></li><li><a class="nounderline abstract_t">Maddocks DG, Alberry MS, Attilakos G, et al. The SAFE project: towards non-invasive prenatal diagnosis. Biochem Soc Trans 2009; 37:460.</a></li><li><a class="nounderline abstract_t">Chang Y, Cesarman E, Pessin MS, et al. Identification of herpesvirus-like DNA sequences in AIDS-associated Kaposi's sarcoma. Science 1994; 266:1865.</a></li><li><a class="nounderline abstract_t">Moore PS, Chang Y. Detection of herpesvirus-like DNA sequences in Kaposi's sarcoma in patients with and those without HIV infection. N Engl J Med 1995; 332:1181.</a></li><li><a class="nounderline abstract_t">Mesri EA, Cesarman E, Arvanitakis L, et al. Human herpesvirus-8/Kaposi's sarcoma-associated herpesvirus is a new transmissible virus that infects B cells. J Exp Med 1996; 183:2385.</a></li></ol></div><div id="topicVersionRevision">Topic 2896 Version 30.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19246619" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : The MIQE guidelines: minimum information for publication of quantitative real-time PCR experiments.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10430926" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Digital PCR.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27351921" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : The Future of Digital Polymerase Chain Reaction in Virology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8432" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Deoxyribonucleic acid polymerase from the extreme thermophile Thermus aquaticus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1342955" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : The polymerase chain reaction. History, methods, and applications.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1842916" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : DNA polymerase fidelity and the polymerase chain reaction.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16102738" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Present and future of rapid and/or high-throughput methods for nucleic acid testing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34720325" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Insight into PCR testing for surgeons.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11623931" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Essay reviews. [Review of: Rabinow P. Making PCR: a story of biotechnology. University of Chicago Press, 1996; and Fujimura J. Crafting science: a sociohistory of the quest for the genetics of cancer. Harvard University Press, 1996].</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18514479" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : UCSC genome browser tutorial.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/37041437" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Managing a PCR Contamination Event in a Molecular Pathology Laboratory.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/37041437" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Managing a PCR Contamination Event in a Molecular Pathology Laboratory.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25405983" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Oligoribonucleotide (ORN) interference-PCR (ORNi-PCR): a simple method for suppressing PCR amplification of specific DNA sequences using ORNs.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/36748452" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Diagnostic yield of broad-range 16s rRNA gene PCR varies by sample type and is improved by the addition of qPCR panels targeting the most common causative organisms.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29608753" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : T2Bacteria magnetic resonance assay for the rapid detection of ESKAPEc pathogens directly in whole blood.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31083728" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Performance of the T2Bacteria Panel for Diagnosing Bloodstream Infections: A Diagnostic Accuracy Study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31083723" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : The Uncertain Clinical Benefit of the T2Bacteria Panel.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/36069557" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Effective Rapid Diagnosis of Bacterial and Fungal Bloodstream Infections by T2 Magnetic Resonance Technology in the Pediatric Population.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19101953" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : PGD for monogenic disorders: aspects of molecular biology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19290882" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : The SAFE project: towards non-invasive prenatal diagnosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7997879" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Identification of herpesvirus-like DNA sequences in AIDS-associated Kaposi's sarcoma.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7700310" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Detection of herpesvirus-like DNA sequences in Kaposi's sarcoma in patients with and those without HIV infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8642350" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Human herpesvirus-8/Kaposi's sarcoma-associated herpesvirus is a new transmissible virus that infects B cells.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
