<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Genetics: Glossary of terms
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Genetics: Glossary of terms
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Genetics: Glossary of terms
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Authors:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Benjamin A Raby, MD, MPH
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Robert D Blank, MD, PhD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Anne Slavotinek, MBBS, PhD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Jennifer S Tirnauer, MD
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            Apr 27, 2023.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H1">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         One of the greatest obstacles clinicians experience in reading about and understanding genetics is the extensive use of technical language and jargon. Genetic terms are frequently used imprecisely in published clinical literature.
        </p>
        <p>
         This topic defines some of the most important technical terms.
        </p>
        <p>
         A more extensive discussion of terms can be accessed in standard genetics reference texts [
         <a href="#rid1">
          1
         </a>
         ].
        </p>
        <p>
         A guide for the conventions regarding naming conventions for human genes and alleles is provided on a website from the HUGO Gene Nomenclature Committee (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.genenames.org%2Fabout%2Fguidelines%2F&amp;token=F%2F33Yq3Lpx4WKgz%2BcwHiE9PO3SgvOeTcoedqQCG0tBNKe%2BeKH0YkdEgqHtBC7PQT%2FFdlZnOY1AjDrsCvfvRMGA%3D%3D&amp;TOPIC_ID=2898" target="_blank">
          www.genenames.org/about/guidelines/
         </a>
         ). It is worth noting that preferred usage as established by professional societies is unevenly applied, even in carefully edited and reviewed documents. Furthermore, usage evolves over time, so it is important to consider shifts in terminology that reflect evolving standards as new technologies and findings alter the scientific and medical communities' communication needs. Readers should therefore expect that actual usage of terms may vary from the definitions provided here.
        </p>
        <p>
         Glossaries of epidemiological terms and terms that apply to systematic reviews and meta-analyses are presented separately in UpToDate. (See
         <a class="medical medical_review" href="/z/d/html/2759.html" rel="external">
          "Glossary of common biostatistical and epidemiological terms"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/16293.html" rel="external">
          "Systematic review and meta-analysis", section on 'Glossary of terms'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2">
         <span class="h1">
          DEFINITIONS
         </span>
        </p>
        <p class="headingAnchor" id="H3942702307">
         <span class="h2">
          Allele
         </span>
         <span class="headingEndMark">
          —
         </span>
         An allele is one of a series of alternative forms (genotypes) at locus, or a specific region of a chromosome. At the DNA level, different alleles have different base sequences.
        </p>
        <p class="headingAnchor" id="H1783841946">
         <span class="h2">
          Allelic fraction
         </span>
         <span class="headingEndMark">
          —
         </span>
         The allelic fraction can be defined as the number of times a variant base is observed, divided by the total number of times any base is observed at the locus [
         <a href="#rid2">
          2
         </a>
         ]. Allelic fraction is generally applied to a single variant in a tumor, and thus is distinct from allelic frequency, which examines the frequency of an allele in a population. (See
         <a class="local">
          'Allele frequency'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H1537087732">
         <span class="h2">
          Allele frequency
         </span>
         <span class="headingEndMark">
          —
         </span>
         The proportion of chromosomes, loci, or genes in a population harboring a specific allele. "Minor allele frequency" typically refers to the less common variant at a biallelic locus and is usually used to refer to the frequency of a single nucleotide polymorphism (SNP) (see
         <a class="local">
          'Single nucleotide polymorphism (SNP)'
         </a>
         below). This population frequency is distinguished from allelic ratio, which applies to a single person (eg, with a malignancy).
        </p>
        <p class="headingAnchor" id="H3789757172">
         <span class="h2">
          Allelic heterogeneity
         </span>
         <span class="headingEndMark">
          —
         </span>
         Allelic heterogeneity refers to the common occurrence of multiple pathogenic variants in one gene that all result in the same disease or syndrome. As an example, more than 1500 variants in the cystic fibrosis transmembrane conductance regulator (
         <em>
          CFTR
         </em>
         ) gene cause cystic fibrosis. Conversely, for some disorders, different pathogenic variants in a single gene can produce very different phenotypes.
        </p>
        <p>
         Allelic heterogeneity differs from genetic heterogeneity, in which variants in multiple genes can cause the same disease phenotype. (See
         <a class="local">
          'Genetic heterogeneity'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H2454124624">
         <span class="h2">
          Allelic ratio
         </span>
         <span class="headingEndMark">
          —
         </span>
         Allelic ratio measures the relative abundance of variant to wildtype alleles within a tumor. Higher allelic ratios (ie, a greater fraction of variant alleles) have been reported to be associated with poorer prognosis. Unlike allele frequency, which is a characteristic of a population (see
         <a class="local">
          'Allele frequency'
         </a>
         above), allelic ratio is a property of cells within a tumor in a single individual. Allelic ratio is of necessity an inexact concept because it is rare (for solid tumors at least) to avoid substantial contamination by non-tumor cells from blood, stroma, and vasculature. Amplification of variant sequences in a tumor can also have a large impact on allelic ratios.
        </p>
        <p class="headingAnchor" id="H4099420019">
         <span class="h2">
          Aneuploidy
         </span>
         <span class="headingEndMark">
          —
         </span>
         The state of having an abnormal number of chromosomes. A euploid human karyotype has 46 chromosomes  (
         <a class="graphic graphic_figure graphicRef72134" href="/z/d/graphic/72134.html" rel="external">
          figure 1
         </a>
         ). Aneuploidy can affect the entire somatic cell population, as in trisomy 21, or it can affect a subset of cells, as in a tumor.
        </p>
        <p class="headingAnchor" id="H3218144220">
         <span class="h2">
          Anticipation
         </span>
         <span class="headingEndMark">
          —
         </span>
         A phenomenon whereby the symptoms of a genetically-based condition appear at an earlier age, or with greater severity, in successive generations. Expansion of trinucleotide repeats is a known molecular cause for specific diseases (such as myotonic dystrophy, fragile X syndrome, Huntington’s chorea) that manifest anticipation.
        </p>
        <p class="headingAnchor" id="H1368106711">
         <span class="h2">
          Association
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genetic association is a property of alleles. It refers to the non-random relationship between an allele and a phenotype in a population. Genetic association between a marker allele and a phenotype can result either because the allele is a direct causal variant, because the allele is in linkage disequilibrium or segregating with a causal variant in close proximity, or because of stratification of the population. Association may be determined in a genome-wide association study. (See
         <a class="local">
          'Genome-wide association study (GWAS)'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H4245358129">
         <span class="h2">
          Autosome
         </span>
         <span class="headingEndMark">
          —
         </span>
         A chromosome other than X or Y. The human genome has 44 autosomes (22 pairs of autosomes)  (
         <a class="graphic graphic_figure graphicRef72134" href="/z/d/graphic/72134.html" rel="external">
          figure 1
         </a>
         ).
        </p>
        <p class="headingAnchor" id="H3829231885">
         <span class="h2">
          Autosomal
         </span>
         <span class="headingEndMark">
          —
         </span>
         A gene is autosomal if it is located on an autosome rather than a sex chromosome. A gene's inheritance pattern is also referred to as autosomal if the pattern corresponds to that of known autosomal genes (rather than sex-linked). (See
         <a class="local">
          'Sex-linked'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H2994976941">
         <span class="h2">
          Benign variant
         </span>
         <span class="headingEndMark">
          —
         </span>
         (See
         <a class="local">
          'Variant'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H3984329775">
         <span class="h2">
          Biome
         </span>
         <span class="headingEndMark">
          —
         </span>
         Humans are colonized by a multitude of microorganisms, which vary by age and location in the body. The biome (or microbiome) is the totality of colonizing microorganisms in a specific environmental milieu.
        </p>
        <p>
         Biomes may be studied genetically using metagenomics. (See
         <a class="local">
          'Metagenomics'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H103000573">
         <span class="h2">
          Carrier
         </span>
         <span class="headingEndMark">
          —
         </span>
         An individual who is heterozygous for a risk or disease allele. The term is typically used to describe someone who is heterozygous for a gene variant that causes autosomal recessive or X-linked recessive disease, but in clinical discussions, it is also used to describe heterozygotes for risk alleles or deleterious alleles that predispose to disease, regardless of inheritance type.
        </p>
        <p class="headingAnchor" id="H2161571447">
         <span class="h2">
          Carrier rate
         </span>
         <span class="headingEndMark">
          —
         </span>
         The frequency of carriers in a population.
        </p>
        <p class="headingAnchor" id="H3240578466">
         <span class="h2">
          Carrier testing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Clinical method of genotyping at-risk populations or relatives to identify individuals, usually asymptomatic, who have a pathogenic or likely pathogenic variant for an autosomal recessive or X-linked disorder. One example is prenatal screening for Tay-Sachs disease-associated variants in people of Ashkenazi Jewish ancestry. (See
         <a class="medical medical_review" href="/z/d/html/2901.html" rel="external">
          "Genetic testing"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/428.html" rel="external">
          "Preconception and prenatal carrier screening for genetic disorders more common in people of Ashkenazi Jewish descent and others with a family history of these disorders"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3993895047">
         <span class="h2">
          Cascade testing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Refers to a testing approach in which at-risk first-degree relatives are tested for a familial genetic variant; if these individuals test positive, then their first-degree relatives are tested. Cascade testing allows testing to be focused on a specific variant and reduces unnecessary testing of relatives who are not at risk.
        </p>
        <p class="headingAnchor" id="H2680527670">
         <span class="h2">
          Centromere
         </span>
         <span class="headingEndMark">
          —
         </span>
         The condensed region of a chromosome that mediates attachment of chromosomes to the microtubules of the mitotic or meiotic spindle. The centromere is important in preserving normal chromosome number.
        </p>
        <p class="headingAnchor" id="H2315986338">
         <span class="h2">
          Centrosome
         </span>
         <span class="headingEndMark">
          —
         </span>
         The specialized structure adjacent to the nucleus that nucleates microtubules. During cell division, replicated centrosomes move to opposite ends of the cell to form the poles of the mitotic spindle. (See
         <a class="local">
          'Mitosis'
         </a>
         below and
         <a class="medical medical_review" href="/z/d/html/129424.html" rel="external">
          "Basic genetics concepts: Chromosomes and cell division", section on 'Mitosis'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H1092899323">
         <span class="h2">
          Chimerism
         </span>
         <span class="headingEndMark">
          —
         </span>
         When referring to an individual, a state in which two or more populations of genetically distinct cells are present that arose from the fusion of two or more fertilized eggs. Contrasted with mosaicism. (See
         <a class="local">
          'Mosaicism'
         </a>
         below.)
        </p>
        <p>
         Also used in patients post-allogeneic hematopoietic stem cell transplant to refer to a state of two genetically distinct populations of hematopoietic cells (one from the donor and one from the recipient). (See
         <a class="medical medical_review" href="/z/d/html/3557.html" rel="external">
          "Preparative regimens for hematopoietic cell transplantation", section on 'Chimerism'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3724164370">
         <span class="h2">
          Chromatid
         </span>
         <span class="headingEndMark">
          —
         </span>
         One of two replications, or copies, of a chromosome formed prior to cell division and joined together at their centromeres. The centromere is the last portion of a chromosome to replicate during cell division. Sister chromatids are a pair of chromatids attached at the centromere. (See
         <a class="local">
          'Centromere'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H2759312189">
         <span class="h2">
          Chromatin
         </span>
         <span class="headingEndMark">
          —
         </span>
         A complex structure composed of DNA, RNA, and proteins that facilitates efficient packaging of DNA in cells. The primary structure of chromatin is the nucleosome, consisting of double-stranded DNA coiled around a core of histone proteins. Nucleosomes packed tightly together form a "bead-on-string" configuration, which in turn assembles in hierarchical looping structures to create densely-packaged chromatin. The regulation of gene transcription is governed by the uncoiling of packed chromatin (heterochromatin) into exposed DNA (euchromatin). (See
         <a class="medical medical_review" href="/z/d/html/14601.html" rel="external">
          "Principles of epigenetics"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2427059189">
         <span class="h2">
          Clonal
         </span>
         <span class="headingEndMark">
          —
         </span>
         Arising from a single clone, or cell. Examples include clonal selection of lymphocytes during immune development and clonal origin of leukemia cells or other tumor cells. (See
         <a class="medical medical_review" href="/z/d/html/3977.html" rel="external">
          "Immunoglobulin genetics"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4493.html" rel="external">
          "Acute myeloid leukemia: Pathogenesis"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H614101173">
         <span class="h2">
          Cloning
         </span>
         <span class="headingEndMark">
          —
         </span>
         Production of a genetically identical copy. Can refer to a single gene or to an entire organism.
        </p>
        <p class="headingAnchor" id="H1618877980">
         <span class="h2">
          Coding region
         </span>
         <span class="headingEndMark">
          —
         </span>
         Portion of a gene that encodes a protein.
        </p>
        <p class="headingAnchor" id="H4114138417">
         <span class="h2">
          Coding mutation, variant, or polymorphism
         </span>
         <span class="headingEndMark">
          —
         </span>
         A genetic variation in the open reading frame (protein-encoding region) of a gene.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Missense
         </strong>
         – Coding variants that alter amino acid composition of a protein are called non-synonymous or missense variants  (
         <a class="graphic graphic_figure graphicRef55798" href="/z/d/graphic/55798.html" rel="external">
          figure 2
         </a>
         ).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Synonymous
         </strong>
         – Variants that do not alter amino acid composition are called synonymous variants. These are sometimes called silent variants, although not all synonymous variants are truly silent, as they may affect other aspects of gene function besides protein coding.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Nonsense
         </strong>
         – Nonsense variants are coding variants that result in the introduction of a stop codon  (
         <a class="graphic graphic_figure graphicRef78399" href="/z/d/graphic/78399.html" rel="external">
          figure 3
         </a>
         ).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Frameshift
         </strong>
         – A frameshift mutation results from an insertion or deletion of a number of bases not divisible by three, resulting in shifting of the reading frame  (
         <a class="graphic graphic_figure graphicRef78767" href="/z/d/graphic/78767.html" rel="external">
          figure 4
         </a>
         ).
        </p>
        <p>
        </p>
        <p>
         Variants are also classified according to their pathogenicity. (See
         <a class="local">
          'Variant'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H3605286535">
         <span class="h2">
          Codon
         </span>
         <span class="headingEndMark">
          —
         </span>
         A three-nucleotide sequence that codes either for a specific amino acid or for chain initiation or termination during protein synthesis (translation of RNA to protein). The figures illustrate the codon sequences illustrated as a chart  (
         <a class="graphic graphic_figure graphicRef57764" href="/z/d/graphic/57764.html" rel="external">
          figure 5
         </a>
         ) or a wheel  (
         <a class="graphic graphic_figure graphicRef73711" href="/z/d/graphic/73711.html" rel="external">
          figure 6
         </a>
         ). (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression", section on 'Translation'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2828463269">
         <span class="h2">
          Complementation
         </span>
         <span class="headingEndMark">
          —
         </span>
         The restoration of normal phenotype by gene replacement. The replaced gene can either be an intact copy of a defective gene (direct replacement), or an alternate gene with function that can compensate for the defective gene's aberrant function.
        </p>
        <p class="headingAnchor" id="H3440247137">
         <span class="h2">
          Complex trait/complex disease
         </span>
         <span class="headingEndMark">
          —
         </span>
         Trait or disease for which interactions between more than one gene and/or environmental factors also play a role in the phenotype. (See
         <a class="medical medical_review" href="/z/d/html/14600.html" rel="external">
          "Principles of complex trait genetics"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H745843411">
         <span class="h2">
          Compound heterozygous
         </span>
         <span class="headingEndMark">
          —
         </span>
         Harboring two different pathogenic variants in the same gene that together are sufficient to manifest an autosomal recessive phenotype. This differs from "homozygous," which refers to harboring two copies of the same pathogenic variant, and from "double heterozygous," which refers to harboring two pathogenic variants at two separate genetic loci, which together manifest disease. (See
         <a class="local">
          'Digenic inheritance'
         </a>
         below and
         <a class="local">
          'Double heterozygous'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H2250302843">
         <span class="h2">
          Consanguinity
         </span>
         <span class="headingEndMark">
          —
         </span>
         Reproduction between two individuals from the same bloodline (eg, first cousin, second cousin). Consanguineous parentage increases the probability of a rare recessive disease, resulting from higher probability of both parents sharing the same rare deleterious sequence variant.
        </p>
        <p class="headingAnchor" id="H1375728442">
         <span class="h2">
          Copy number variation (CNV)
         </span>
         <span class="headingEndMark">
          —
         </span>
         The most prevalent type of chromosomal structural variation, in which the number of copies of a large chromosomal or DNA segment (usually measuring thousands to millions of bases) varies between individuals. (See
         <a class="medical medical_review" href="/z/d/html/16647.html" rel="external">
          "Genomic disorders: An overview", section on 'Copy number variations'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H444668741">
         <span class="h2">
          Coupling (in cis)
         </span>
         <span class="headingEndMark">
          —
         </span>
         The presence of two specified alleles at two linked loci on the same homologous chromosome (ie, "in cis"), and the two alternative alleles on the other chromosome. For illustration, in the case of dominant and recessive alleles, the coupling gametes formed are AB and ab  (
         <a class="graphic graphic_figure graphicRef58385" href="/z/d/graphic/58385.html" rel="external">
          figure 7
         </a>
         ).
        </p>
        <p>
         In contrast, repulsion refers to the presence of the specified alleles at two linked loci on different chromosomes (ie, in trans). (See
         <a class="local">
          'Repulsion (in trans)'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H1815524937">
         <span class="h2">
          CRISPR
         </span>
         <span class="headingEndMark">
          —
         </span>
         CRISPR (clustered regularly interspaced short palindromic repeats; pronounced "crisper"; sometimes referred to as CRISPR-Cas9) is a method used for gene or genome editing. It was adapted from a component of a bacterial defense system in combination with an endonuclease such as Cas9 or Cpf1. (See
         <a class="local">
          'Genome editing'
         </a>
         below and
         <a class="medical medical_review" href="/z/d/html/98724.html" rel="external">
          "Overview of gene therapy, gene editing, and gene silencing", section on 'Gene editing'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3305482665">
         <span class="h2">
          Crossing-over
         </span>
         <span class="headingEndMark">
          —
         </span>
         The exchange of chromosome segments through the process of recombination that occurs between two homologous chromosomes during meiosis. The site on the chromosome where the exchange occurs is called a crossover.
        </p>
        <p class="headingAnchor" id="H366724202">
         <span class="h2">
          De novo mutation
         </span>
         <span class="headingEndMark">
          —
         </span>
         A novel genetic sequence variant in the proband's DNA and absent in the parents' DNA. Often used to distinguish familial from sporadic cases of genetic disease.
        </p>
        <p class="headingAnchor" id="H3273258581">
         <span class="h2">
          Digenic inheritance
         </span>
         <span class="headingEndMark">
          —
         </span>
         Diseases caused by co-inheritance of variants at two distinct genetic loci (ie, in two different genes). Individuals with digenic inheritance may also be called "double heterozygous," which is distinct from compound heterozygous (having two different pathogenic variants in the same gene). (See
         <a class="local">
          'Compound heterozygous'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H4242890491">
         <span class="h2">
          Diploid
         </span>
         <span class="headingEndMark">
          —
         </span>
         Possessing two copies of each autosomal chromosome and two sex chromosomes. Most human cells are diploid. Hepatocytes are frequently polyploid (tetraploid or greater). Gametes are haploid (one copy of each autosome and one sex chromosome)  (
         <a class="graphic graphic_figure graphicRef72134" href="/z/d/graphic/72134.html" rel="external">
          figure 1
         </a>
         ). (See
         <a class="local">
          'Haploid'
         </a>
         below and
         <a class="local">
          'Ploidy'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H96690551">
         <span class="h2">
          DNA
         </span>
         <span class="headingEndMark">
          —
         </span>
         DNA (deoxyribonucleic acid) is the primary molecular constituent of chromosomes that stores the genetic information of most living organisms, including humans.
        </p>
        <p>
         The genetic information in DNA is encoded by the sequence of the four bases adenine, guanine, thymine, and cytosine.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Adenine and guanine are purines.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Thymine and cytosine are pyrimidines.
        </p>
        <p>
        </p>
        <p>
         DNA is usually present as a double-stranded antiparallel polymer composed of an outer phosphodeoxyribose backbone with central nucleotide side chains  (
         <a class="graphic graphic_figure graphicRef120004" href="/z/d/graphic/120004.html" rel="external">
          figure 8
         </a>
         ). (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression", section on 'DNA and RNA'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2977017041">
         <span class="h2">
          DNA barcoding
         </span>
         <span class="headingEndMark">
          —
         </span>
         A collection of methods developed to facilitate the analysis of complex mixtures of pooled samples, whereby short, unique DNA sequences (referred to as tags or barcodes) are added to each of the DNA samples (each isolated from a distinct individual) in the pool. Barcoding is used routinely in next-generation sequencing applications, including single-cell RNA sequencing and exome sequencing. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         and
         <a class="local">
          'Exome sequencing'
         </a>
         below.)
        </p>
        <p>
         Barcoding also refers to methods for determining the species of origin of a DNA sample on the basis of the DNA sequence itself. A clinical example is identification of the ingredients in an herbal preparation.
        </p>
        <p class="headingAnchor" id="H2335629778">
         <span class="h2">
          Dominant negative
         </span>
         <span class="headingEndMark">
          —
         </span>
         Dominant negative alleles are alleles that cause an abnormal phenotype or disease by a mechanism that depends on the abnormal gene product of a pathogenic allele interfering with the function of the normal gene product from the normal allele.
        </p>
        <p>
         In contrast with most loss-of-function variants that confer phenotype only when both alleles carry a pathogenic variant (recessive inheritance), dominant-negative variants act dominantly, meaning that only a single allele with the variant is sufficient to cause the disease phenotype.
        </p>
        <p class="headingAnchor" id="H3021797604">
         <span class="h2">
          Double heterozygous
         </span>
         <span class="headingEndMark">
          —
         </span>
         Heterozygous for two pathogenic variants at two separate genetic loci that together are sufficient to manifest a phenotype. Differs from compound heterozygous. (See
         <a class="local">
          'Compound heterozygous'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H2932917426">
         <span class="h2">
          Embryonic stem cell (ESC)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A pluripotential (pluripotent) cell derived from the inner cell mass of an early-stage embryo that is capable of differentiating into cells derived from all three germ layers. (See
         <a class="medical medical_review" href="/z/d/html/14598.html" rel="external">
          "Overview of stem cells", section on 'What defines a stem cell?'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2923278619">
         <span class="h2">
          Enhancer
         </span>
         <span class="headingEndMark">
          —
         </span>
         A region of DNA, upstream (5') or downstream (3") of a gene, that regulates gene expression. Enhancer function relies on binding of specific regulatory proteins (transcription factors). (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression", section on 'Gene expression'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H1633061071">
         <span class="h2">
          Enhancer hijacking
         </span>
         <span class="headingEndMark">
          —
         </span>
         Use by one gene of another gene's enhancer. Often due to changes in three-dimensional genome structure that place one region of DNA adjacent to another region. Can explain the mechanism of certain diseases involving aberrant gene expression.
        </p>
        <p class="headingAnchor" id="H1151712393">
         <span class="h2">
          Epigenetic change
         </span>
         <span class="headingEndMark">
          —
         </span>
         A modification of chromatin, often tissue-specific, that alters the expression of a gene without changing the nucleotide base sequence. Epigenetic changes may be stable in an individual but may be reversed during gametogenesis, development, certain environmental stresses such as starvation, or by certain medications such as histone deacetylase (HDAC) inhibitors. DNA methylation and histone acetylation are common epigenetic changes. Epigenetic changes form the mechanistic basis of imprinting. (See
         <a class="medical medical_review" href="/z/d/html/14601.html" rel="external">
          "Principles of epigenetics"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H1786466759">
         <span class="h2">
          Epistasis
         </span>
         <span class="headingEndMark">
          —
         </span>
         The process by which variations at two or more genetic loci interact to produce phenotypes different from the individual effects of each variant. This process is often referred to as either a gene-gene interaction or a genetic modifier effect.
        </p>
        <p class="headingAnchor" id="H2701193320">
         <span class="h2">
          Exome
         </span>
         <span class="headingEndMark">
          —
         </span>
         The portion of the genome that consists of exons. (See
         <a class="local">
          'Exon'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H2032140166">
         <span class="h2">
          Exome sequencing
         </span>
         <span class="headingEndMark">
          —
         </span>
         A sequencing strategy that provides the DNA sequence corresponding to all exons (which represent approximately 1 to 2 percent of the genome), excluding introns and other untranscribed genomic sequences. Though the complete exome includes noncoding 5' and 3' untranslated regions (UTRs), most exome sequencing assays are enriched for the coding exons and largely exclude the noncoding regions.
        </p>
        <p class="headingAnchor" id="H3520484613">
         <span class="h2">
          Exon
         </span>
         <span class="headingEndMark">
          —
         </span>
         A segment of DNA that is transcribed and present in mature messenger RNA (mRNA). Many exons encode a portion of a protein, but noncoding exons also exist. This is in contrast with an intron, the DNA sequence between exons that does not become part of mature mRNA. Exons constitute only a small percent of the genome (approximately 1 to 2 percent).
        </p>
        <p class="headingAnchor" id="H86594006">
         <span class="h2">
          Expressivity
         </span>
         <span class="headingEndMark">
          —
         </span>
         A parameter used in genetic models that quantifies the degree to which an inherited characteristic is expressed in an organism.
        </p>
        <p class="headingAnchor" id="H711473475">
         <span class="h2">
          Expression quantitative trait locus (eQTL)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A polymorphic genetic region that influences population-level variability of a target gene's transcript (RNA) abundance. The implicated regulatory variants can influence transcript abundance through a variety of mechanisms that either alter the timing or rate of transcription (RNA production) or transcript stability (RNA degradation). (See
         <a class="local">
          'Quantitative traits and quantitative trait loci (QTL)'
         </a>
         below.)
        </p>
        <p>
         Analogous terms that refer to loci that influence abundances of other molecule types include protein quantitative trait loci (pQTL), metabolomic quantitative trait loci (mQTL), and methylation quantitative trait loci (meQTL).
        </p>
        <p class="headingAnchor" id="H577558714">
         <span class="h2">
          Frameshift
         </span>
         <span class="headingEndMark">
          —
         </span>
         A frameshift is a DNA sequence variant that results from an insertion or deletion of a number of bases that is not divisible by three, resulting in a shift of the reading frame  (
         <a class="graphic graphic_figure graphicRef78767" href="/z/d/graphic/78767.html" rel="external">
          figure 4
         </a>
         ) and thus altering the entire downstream protein sequence.
        </p>
        <p class="headingAnchor" id="H2526278640">
         <span class="h2">
          Fusion gene
         </span>
         <span class="headingEndMark">
          —
         </span>
         A fusion gene is a functional gene product that results from the fusion of DNA segments from two physically distinct genes. The fusion occurs as a consequence of chromosomal rearrangements such as translocations, inversions, segmental deletions, or duplications. Examples include the
         <em>
          BCR-ABL
         </em>
         and the
         <em>
          FIP1L1-PDGFRA
         </em>
         oncogenes.
        </p>
        <p class="headingAnchor" id="H3419736662">
         <span class="h2">
          Gene
         </span>
         <span class="headingEndMark">
          —
         </span>
         A gene is a unit of DNA sequence that encodes specific function. Classical definitions limit genes to those elements that code for proteins. However, non-protein coding genes (such as noncoding RNAs or pseudogenes) are also genes.
        </p>
        <p class="headingAnchor" id="H3828842802">
         <span class="h2">
          Gene editing
         </span>
         <span class="headingEndMark">
          —
         </span>
         (See
         <a class="local">
          'Genome editing'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H3479936033">
         <span class="h2">
          Genetic heterogeneity
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genetic heterogeneity refers to a phenomenon in which variants in different genes result in the same phenotype or disease. Examples include the multiple genetic causes of sensorineural deafness. This differs from allelic heterogeneity, in which multiple variants in the same gene can lead to the same phenotype. (See
         <a class="local">
          'Allelic heterogeneity'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H3465061226">
         <span class="h2">
          Genetic polymorphism
         </span>
         <span class="headingEndMark">
          —
         </span>
         A genetic polymorphism is a DNA segment for which two or more alternate forms can be found above a minimum threshold frequency (usually &gt;1 percent) in a population. The common types of polymorphisms include single nucleotide variants (single base pair changes, also called single nucleotide polymorphisms [SNPs]), indels (insertion/deletion polymorphisms) or larger structural changes like copy number variants. Most commonly, genetic polymorphism refers a common single base-pair change or single nucleotide polymorphism (SNP). (See
         <a class="local">
          'Polymorphism'
         </a>
         below and
         <a class="local">
          'Single nucleotide polymorphism (SNP)'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H903417281">
         <span class="h2">
          Genetic risk score
         </span>
         <span class="headingEndMark">
          —
         </span>
         An estimate of an individual's genetic risk for a specific polygenic phenotype [
         <a href="#rid3">
          3
         </a>
         ]. Genetic risk scores are calculated using the cumulative contribution of all known risk alleles carried by the individual. This is in contrast to polygenic risk scores, which model genetic risk using a larger number of loci, including many that do not meet genome-wide significance criteria in association studies. (See
         <a class="medical medical_review" href="/z/d/html/14600.html" rel="external">
          "Principles of complex trait genetics"
         </a>
         and
         <a class="local">
          'Polygenic risk (PGR) score'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H4056953126">
         <span class="h2">
          Genome editing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genome editing (or gene editing) refers to the use of nucleases to introduce specific changes at defined sites in a gene or genome. Genome editing is used as a tool for genetic perturbation in research.
        </p>
        <p>
         Therapeutic applications for inherited genetic disorders are in early use or under investigation. (See
         <a class="medical medical_review" href="/z/d/html/98724.html" rel="external">
          "Overview of gene therapy, gene editing, and gene silencing", section on 'Gene editing'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3856285139">
         <span class="h2">
          Genome-wide association study (GWAS)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A GWAS (pronounced "gee-wass") study is a type of genetic mapping study that assesses for evidence of association between genetic variants and heritable traits across the entire genome. Typical studies consist of genotyping hundreds of thousands of common SNPs, using DNA microarrays or other methodologies, in large case-control populations, with the goal of identifying specific risk alleles that are more prevalent in cases than in controls.
        </p>
        <p class="headingAnchor" id="H4261387324">
         <span class="h2">
          Genotype
         </span>
         <span class="headingEndMark">
          —
         </span>
         A genotype is the combination of two alleles at one genomic location (locus) or base pair in an individual  (
         <a class="graphic graphic_figure graphicRef58385" href="/z/d/graphic/58385.html" rel="external">
          figure 7
         </a>
         ).
        </p>
        <p class="headingAnchor" id="H406115043">
         <span class="h2">
          Germline
         </span>
         <span class="headingEndMark">
          —
         </span>
         Germline refers to the gametes (ova and spermatozoa and their precursors) that have the capacity to give rise to offspring.
        </p>
        <p>
         In the context of pathogenic variants, germline refers to variation that arose in germline cells as opposed to somatic variants that were acquired in a specific tissue.
        </p>
        <p class="headingAnchor" id="H390400923">
         <span class="h2">
          Haploid
         </span>
         <span class="headingEndMark">
          —
         </span>
         Cells or organisms possessing one copy of each autosomal chromosome and one sex chromosome (and therefore effectively one copy of each gene). Gametes (ova and spermatozoa) are haploid. Fertilization of a haploid ovum by a haploid sperm results in formation of a diploid embryo. Many microorganisms are haploid. (See
         <a class="local">
          'Diploid'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H1042558214">
         <span class="h2">
          Haploinsufficiency
         </span>
         <span class="headingEndMark">
          —
         </span>
         Having an abnormal phenotype due to inactivation of one allele by a pathogenic (deleterious) variant. In a diploid cell, the single remaining functional copy of the gene does not produce sufficient protein, resulting in disease.
        </p>
        <p class="headingAnchor" id="H2732486409">
         <span class="h2">
          Haplotype
         </span>
         <span class="headingEndMark">
          —
         </span>
         The physical combination or sequence of alleles present on a single chromosome. By definition, alleles on one haplotype are in "cis"  (
         <a class="graphic graphic_figure graphicRef58385" href="/z/d/graphic/58385.html" rel="external">
          figure 7
         </a>
         ).
        </p>
        <p class="headingAnchor" id="H773063789">
         <span class="h2">
          Hemizygous
         </span>
         <span class="headingEndMark">
          —
         </span>
         The state of carrying only one copy of a genomic region due to deletion or altered function of the corresponding region on the other chromosome. Carriers of large-scale deletions are hemizygous. Hemizygosity can confer disease if having one normally functioning copy is insufficient for normal cellular function (haploinsufficiency), but if a single functional copy of the gene is sufficient for normal cellular function, the phenotype may not be abnormal. Hemizygosity can also confer disease if a pathogenic variant is present within the hemizygous region. (See
         <a class="local">
          'Haploinsufficiency'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H3157173185">
         <span class="h2">
          Heritability
         </span>
         <span class="headingEndMark">
          —
         </span>
         The proportion of phenotypic variation that is explained by genetic (or in some cases, epigenetic) factors.
        </p>
        <p class="headingAnchor" id="H1052502515">
         <span class="h2">
          Heteroplasmy
         </span>
         <span class="headingEndMark">
          —
         </span>
         The occurrence in a single cell of more than one population of mitochondrial genomes.
        </p>
        <p class="headingAnchor" id="H3344030654">
         <span class="h2">
          Identity by descent
         </span>
         <span class="headingEndMark">
          —
         </span>
         Alleles are identical by descent if they can be traced back to a common ancestor. Identity by descent is a more stringent classification than identity by state (see
         <a class="local">
          'Identity by state'
         </a>
         below). Identity by descent is the basis for establishing linkage.
        </p>
        <p class="headingAnchor" id="H158047186">
         <span class="h2">
          Identity by state
         </span>
         <span class="headingEndMark">
          —
         </span>
         Alleles are identical by state if the assay being used to distinguish alleles determines that they are identical.
        </p>
        <p class="headingAnchor" id="H1026529710">
         <span class="h2">
          Imprinting
         </span>
         <span class="headingEndMark">
          —
         </span>
         Gamete-specific gene silencing, in which only the allele from the mother or only the allele from the father is expressed, leading to observed parent-of-origin effects in offspring. Examples include the genetic locus responsible for Prader-Willi syndrome and Angelman Syndrome, and the
         <em>
          GNAS
         </em>
         gene, encoding the guanine nucleotide binding protein alpha activating subunit involved in pseudohyperparathyroidism. (See
         <a class="medical medical_review" href="/z/d/html/5875.html" rel="external">
          "Prader-Willi syndrome: Clinical features and diagnosis"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/452.html" rel="external">
          "Congenital cytogenetic abnormalities"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2065670240">
         <span class="h2">
          Indel
         </span>
         <span class="headingEndMark">
          —
         </span>
         A class of variants displaying an extra copy(ies) or a missing copy(ies) of a short genetic or chromosomal sequence  (
         <a class="graphic graphic_figure graphicRef69699" href="/z/d/graphic/69699.html" rel="external">
          figure 9
         </a>
         ). (See
         <a class="medical medical_review" href="/z/d/html/4490.html" rel="external">
          "Chromosomal translocations, deletions, and inversions"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H992606025">
         <span class="h2">
          Induced pluripotent stem cell (iPSC)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A pluripotent cell derived by in vitro reprogramming of a somatic cell. iPSCs are capable of both self-renewal and differentiation to mature lineages. (See
         <a class="medical medical_review" href="/z/d/html/14598.html" rel="external">
          "Overview of stem cells", section on 'Induced pluripotent stem (iPS) cells'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H648637710">
         <span class="h2">
          Intron
         </span>
         <span class="headingEndMark">
          —
         </span>
         A segment of DNA between two exons that is transcribed to pre-mRNA but is removed through the process of splicing and therefore is not part of mature mRNA. Introns may contain regulatory DNA or serve other functions.
        </p>
        <p class="headingAnchor" id="H1734657361">
         <span class="h2">
          Inversion
         </span>
         <span class="headingEndMark">
          —
         </span>
         A chromosomal rearrangement characterized by rotation and reintegration of a DNA segment, resulting in an inverted orientation of the segment relative to its typical state.
        </p>
        <p class="headingAnchor" id="H1747533387">
         <span class="h2">
          Karyotype
         </span>
         <span class="headingEndMark">
          —
         </span>
         Karyotype refers to the complete set of chromosomes in an organism or tumor. Karyotype is determined by visual examination and counting of condensed chromosomes from several representative cells to determine the number of copies of each chromosome as well as any translocations, sub-chromosomal deletions, or duplications. Determination of the karyotype of a tumor is also called "cytogenetic analysis." (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H4031196770">
         <span class="h2">
          Likely benign variant
         </span>
         <span class="headingEndMark">
          —
         </span>
         (See
         <a class="local">
          'Variant'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H999314707">
         <span class="h2">
          Likely pathogenic variant
         </span>
         <span class="headingEndMark">
          —
         </span>
         (See
         <a class="local">
          'Variant'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H3139062263">
         <span class="h2">
          Linkage
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genetic linkage refers to the relationship that exists between two loci that violate the Mendelian law of independent assortment and therefore segregate in kindreds in a non-random fashion. Non-independent assortment results because linked loci reside together on the same chromosome (ie, they are syntenic). However, many pairs of syntenic loci are not linked. Linkage therefore implies the linked loci are in close physical proximity to each other. The genetic linkage distance is expressed as the recombination fraction, which is measured in centiMorgans (cM). This is not necessarily proportional to the physical distance (base pairs) separating the loci.
        </p>
        <p class="headingAnchor" id="H692109156">
         <span class="h2">
          Linkage analysis
         </span>
         <span class="headingEndMark">
          —
         </span>
         Method of gene mapping that tests for the non-random segregation of a phenotype (trait, disease) with discrete chromosomal segments. Identification of linked regions implies the existence of genetic variants within the linked region. The process of disease-gene or trait-gene identification within this region is termed positional cloning.
        </p>
        <p class="headingAnchor" id="H4097651555">
         <span class="h2">
          Linkage disequilibrium
         </span>
         <span class="headingEndMark">
          —
         </span>
         The non-random association of alleles at two or more loci in a population. Linkage disequilibrium is present when the observed haplotype distribution of two or more markers in a population is significantly different from the expected haplotype distribution (which can be derived from the cross-product of observed allele frequencies)  (
         <a class="graphic graphic_figure graphicRef76528" href="/z/d/graphic/76528.html" rel="external">
          figure 10
         </a>
         ).
        </p>
        <p class="headingAnchor" id="H169768065">
         <span class="h2">
          Locus
         </span>
         <span class="headingEndMark">
          —
         </span>
         A locus (plural, loci) is a specific chromosomal or genomic location.
        </p>
        <p class="headingAnchor" id="H3229945422">
         <span class="h2">
          LOD score
         </span>
         <span class="headingEndMark">
          —
         </span>
         The "logarithm of the odds" (LOD) score is a quantitative measure of the statistical evidence of linkage between two genes. The LOD score depends on both the probability of co-segregation of the two genes during meiosis and the size and structure of the population in which the linkage analysis is performed.
        </p>
        <p>
         By convention, LOD scores &gt;3 are considered to be evidence of linkage in many human studies.
        </p>
        <p>
         In some studies, the threshold LOD scores for linkage at various statistical significance levels (typically genome-wide alpha ≤0.05 and ≤0.01) can be established empirically via permutation testing.
        </p>
        <p class="headingAnchor" id="H3964589819">
         <span class="h2">
          Lyonization
         </span>
         <span class="headingEndMark">
          —
         </span>
         (See
         <a class="local">
          'X-inactivation'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H1962140536">
         <span class="h2">
          Manhattan plot
         </span>
         <span class="headingEndMark">
          —
         </span>
         A type of plot used to display results of a GWAS study (see
         <a class="local">
          'Genome-wide association study (GWAS)'
         </a>
         above). Genomic coordinates are shown on the X-axis and the negative logarithm of the P-value for each SNP on the Y-axis. SNPs with the strongest association will have the lowest P-values, and hence the tallest profiles. Named for the appearance of the skyline in Manhattan in the United States  (
         <a class="graphic graphic_figure graphicRef90262" href="/z/d/graphic/90262.html" rel="external">
          figure 11
         </a>
         ).
        </p>
        <p class="headingAnchor" id="H2616546832">
         <span class="h2">
          Marker
         </span>
         <span class="headingEndMark">
          —
         </span>
         A locus with alternative alleles that can be used in genetic mapping experiments.
        </p>
        <p class="headingAnchor" id="H4053855015">
         <span class="h2">
          Meiosis
         </span>
         <span class="headingEndMark">
          —
         </span>
         The cell division process in germline cells by which the chromosomal complement is reduced from the diploid to the haploid number  (
         <a class="graphic graphic_figure graphicRef79125" href="/z/d/graphic/79125.html" rel="external">
          figure 12
         </a>
         ). (See
         <a class="medical medical_review" href="/z/d/html/129424.html" rel="external">
          "Basic genetics concepts: Chromosomes and cell division", section on 'Meiosis'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H42565124">
         <span class="h2">
          Mendelian inheritance
         </span>
         <span class="headingEndMark">
          —
         </span>
         A trait is said to have Mendelian inheritance if its genetic transmission can be explained by a Mendelian model of inheritance, such as autosomal dominant, autosomal recessive, or X-linked recessive or dominant inheritance. This is in contrast to non-Mendelian inheritance patterns such as digenic inheritance, or quantitative traits. (See
         <a class="local">
          'Digenic inheritance'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H1211297583">
         <span class="h2">
          Mendelian randomization
         </span>
         <span class="headingEndMark">
          —
         </span>
         A study design in which genotypes serve as a proxy for an epidemiologic exposure(s). The rationale for undertaking such investigations is that alleles segregate independently and are therefore immune to biases that cannot be overcome in observational studies.
        </p>
        <p>
         The alleles included in the study must be associated with the exposure being tested, and several additional stringent assumptions also must be satisfied, as discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/13514.html" rel="external">
          "Mendelian randomization"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2788605268">
         <span class="h2">
          Metagenomics
         </span>
         <span class="headingEndMark">
          —
         </span>
         The study of complex microbial populations (biomes) using genomic approaches. Human tissues such as the skin and gut have multiple heterogeneous populations of microorganisms that differ from each other with respect to phyla composition and abundance in a tissue- and site-specific manner. These abundances can be estimated by sequencing the mixed population of microorganisms, either through targeted sequencing of 16S ribosomes (for bacterial characterization) or whole-genome approaches (for bacteria, viruses, fungi, and other organisms).
        </p>
        <p class="headingAnchor" id="H1887986224">
         <span class="h2">
          Methylation
         </span>
         <span class="headingEndMark">
          —
         </span>
         The addition of methyl groups to cytosine bases in DNA or to lysine residues in the tails histones. Methylation followed by deamination is a major pathway for mutation of cytosine to thymine. Methylation is a form of epigenetic regulation that correlates with reduced gene transcription and is an important mechanism for gene imprinting and X-inactivation. (See
         <a class="local">
          'Epigenetic change'
         </a>
         above and
         <a class="local">
          'Imprinting'
         </a>
         above and
         <a class="local">
          'X-inactivation'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H1184179623">
         <span class="h2">
          Micro-RNA (miR)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A small noncoding RNA that regulates the stability or translation of a set of messenger RNAs (mRNAs).
        </p>
        <p class="headingAnchor" id="H1967539924">
         <span class="h2">
          Microsatellite
         </span>
         <span class="headingEndMark">
          —
         </span>
         A tandem array of short sequences of DNA (typically two to four bases). Microsatellites are numerous and widely distributed in the genome. There is often polymorphism in their length, making them useful markers in genetic studies, including genome mapping and pedigree-based linkage analysis. Microsatellites are also known as short tandem repeat markers (STRs) or short tandem repeat polymorphisms (STRPs). They constitute a prominent class of indel variants. (See
         <a class="local">
          'Indel'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H2663343461">
         <span class="h2">
          Mitochondrial genome
         </span>
         <span class="headingEndMark">
          —
         </span>
         The genetic material carried within mitochondria, known as mitochondrial DNA (mtDNA). At fertilization, all the mitochondria are derived from the egg, so mitochondrial genes display maternal inheritance. (See
         <a class="medical medical_review" href="/z/d/html/5148.html" rel="external">
          "Mitochondrial regulation and functions", section on 'Mitochondrial genetics'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H380597366">
         <span class="h2">
          Mitosis
         </span>
         <span class="headingEndMark">
          —
         </span>
         The process of cell division occurring in somatic cells, in which each daughter cell receives a full chromosome complement. (See
         <a class="medical medical_review" href="/z/d/html/129424.html" rel="external">
          "Basic genetics concepts: Chromosomes and cell division", section on 'Somatic cell division'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H838501563">
         <span class="h2">
          Monogenic trait/monogenic disease
         </span>
         <span class="headingEndMark">
          —
         </span>
         Trait or disease with inheritance that can be explained by a single gene. (See
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)"
         </a>
         .)
        </p>
        <p>
         Monogenic traits are contrasted with polygenic and complex diseases. (See
         <a class="local">
          'Polygenic trait/polygenic disease'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H3372859071">
         <span class="h2">
          Mosaicism
         </span>
         <span class="headingEndMark">
          —
         </span>
         When referring to an individual, a state in which two populations of genetically distinct cells are present that arose from a single fertilized egg. Mosaicism can arise through a variety of mechanisms including chromosome nondisjunction, anaphase lag, endoreplication, and post-fertilization mutation. (See
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)", section on 'Mosaicism'
         </a>
         .)
        </p>
        <p>
         A common instance occurs in Klinefelter syndrome, in which post-fertilization nondisjunction causes some but not all cells to harbor an XXY karyotype. (See
         <a class="local">
          'Karyotype'
         </a>
         above.)
        </p>
        <p>
         Contrasted with chimerism. (See
         <a class="local">
          'Chimerism'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H4081848015">
         <span class="h2">
          Mutation, mutant
         </span>
         <span class="headingEndMark">
          —
         </span>
         An alteration in a gene, or the altered version of a gene, typically in such a manner that affects function, but not always (eg, a "silent" mutation that changes the DNA sequence but not the protein sequence). These terms are used in several different senses, depending on context:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         In human genetics, a mutation is a genetic variant of low population frequency, in contrast with a polymorphism (often a single nuclear polymorphism [SNP]) with an allele frequency of 1 percent or greater. (See
         <a class="local">
          'Single nucleotide polymorphism (SNP)'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         Types of gene mutations include:
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Nonsense mutation
         </strong>
         – Creates premature stop codon  (
         <a class="graphic graphic_figure graphicRef78399" href="/z/d/graphic/78399.html" rel="external">
          figure 3
         </a>
         ).
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Missense mutation
         </strong>
         – Creates amino acid change  (
         <a class="graphic graphic_figure graphicRef55798" href="/z/d/graphic/55798.html" rel="external">
          figure 2
         </a>
         ).
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Synonymous mutation
         </strong>
         – Does not change protein sequence.
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Frameshift
         </strong>
         – Shifts the reading frame of the DNA, in turn altering the triplet codons for protein translation, creating an entirely new protein sequence downstream of the mutation  (
         <a class="graphic graphic_figure graphicRef78767" href="/z/d/graphic/78767.html" rel="external">
          figure 4
         </a>
         ).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         For human traits and diseases, there has been a shift in terminology to use the term variant rather than mutation; variants are further classified according to their pathogenicity. (See
         <a class="local">
          'Variant'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         This revision in terminology was based on a 2015 guideline [
         <a href="#rid4">
          4
         </a>
         ]; prior to this terminology change, mutation was commonly used to imply a change associated with abnormal function (eg, sickle cell mutation of the beta globin gene). Mutation remains appropriate in certain contexts such as when referring to a pathophysiologic process or to specific changes in a region of DNA (or less commonly, a protein). The rationale for preferring "variant" over "mutation" when discussing human genetic variation is discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/96539.html" rel="external">
          "Secondary findings from genetic testing", section on 'Definitions and classification of variants'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         When used in the context of inheritance, a de novo mutation implies a recent sequence change (either germline or somatic), in contrast with inheritance from a carrier parent. (See
         <a class="local">
          'De novo mutation'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         When used to refer to a non-human organism or population of non-human organisms, a mutant refers to a population that harbors a specific, atypical variant (eg, antibiotic-resistant mutants). This term should not be used for people.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H3120166530">
         <span class="h2">
          Mutation fraction
         </span>
         <span class="headingEndMark">
          —
         </span>
         Synonymous with allelic fraction or allelic ratio. (See
         <a class="local">
          'Allelic fraction'
         </a>
         above and
         <a class="local">
          'Allelic ratio'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H978729645">
         <span class="h2">
          Next-generation sequencing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Any of several high-throughput DNA sequencing methods that rely on parallel analysis of multiple DNA fragments. Common approaches include whole genome sequencing and exome sequencing. These methods have resulted in dramatic decreases in the cost and time needed for sequencing projects and are used in many clinical and research settings. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2012181996">
         <span class="h2">
          Noncoding variant
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genetic variation that does not map to gene regions that code for protein. These variants can be functional if they reside in and disrupt functional elements, such as noncoding RNA sequences or regulatory sites (promoters, enhancers, suppressors, or splice-sites).
        </p>
        <p class="headingAnchor" id="H1169292807">
         <span class="h2">
          Nucleic acid vaccine
         </span>
         <span class="headingEndMark">
          —
         </span>
         Refers to a synthetic nucleic acid sequence (either DNA or RNA) packaged in a lipid soluble particle that transfects human cells to produce antigenic viral proteins to induce an antiviral immune response.
        </p>
        <p>
         Examples include FDA-approved RNA vaccines for coronavirus disease 2019 (COVID-19). The use of nucleic acids in the vaccines facilitates modifications when new viral variants arise. (See
         <a class="medical medical_review" href="/z/d/html/129849.html" rel="external">
          "COVID-19: Vaccines"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3970454575">
         <span class="h2">
          Oncogene
         </span>
         <span class="headingEndMark">
          —
         </span>
         Gene that contributes to the production of cancer. Oncogenes typically act in a dominant manner (an oncogenic mutation at one allele is sufficient to promote tumorigenesis). In contrast, tumor suppressor genes typically act in a recessive manner. (See
         <a class="local">
          'Tumor suppressor gene'
         </a>
         below.)
        </p>
        <p>
         <strong>
          Pathogenic variant
         </strong>
         – Genetic change associated with disease or strongly suspected of being associated with disease. (See
         <a class="local">
          'Variant'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H1722947932">
         <span class="h2">
          Pedigree
         </span>
         <span class="headingEndMark">
          —
         </span>
         A diagram or other graphic representation of a kindred that shows the relationships among relatives, sex of each individual, and presence or absence of one or more genetic conditions in each individual  (
         <a class="graphic graphic_figure graphicRef55908" href="/z/d/graphic/55908.html" rel="external">
          figure 13
         </a>
         ).
        </p>
        <p class="headingAnchor" id="H1359853734">
         <span class="h2">
          Penetrance
         </span>
         <span class="headingEndMark">
          —
         </span>
         The probability that an individual harboring a pathogenic variant will develop the associated disease or condition. Incomplete (or variable) penetrance occurs when an individual with a pathogenic variant does not manifest features of the disorder. There are many causes of incomplete penetrance, including absence of environmental or genetic co-factors, epigenetic effects such as imprinting, sex-specific effects, or age-related expression differences. (See
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)", section on 'Penetrance and expressivity'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3765863976">
         <span class="h2">
          Phenotype
         </span>
         <span class="headingEndMark">
          —
         </span>
         A characteristic of an organism (as opposed to the organism's genotype). Phenotypes are sensitive to the assays used to assign or measure them. They may be categorical, such as presence or absence of a disease; or quantitative, such as systolic blood pressure. Further complexities in phenotypic description involve the physiological state of the organism at the time of measurement, age, or use of provocative stimuli. Most phenotypes are variable, and understanding sources of variability, both instrumental and biologic, is important for interpretation.
        </p>
        <p class="headingAnchor" id="H2978875878">
         <span class="h2">
          Pleiotropy
         </span>
         <span class="headingEndMark">
          —
         </span>
         The association of variant(s) in a single gene with multiple phenotypic effects, often in different tissues or organs. An example is Marfan syndrome, in which mutations in the fibrillin 1 (
         <em>
          FBN1
         </em>
         ) gene can cause cardiac, ocular, and connective tissue findings.
        </p>
        <p class="headingAnchor" id="H2921582365">
         <span class="h2">
          Ploidy
         </span>
         <span class="headingEndMark">
          —
         </span>
         The number of sets of chromosomes present in an organism or cell. Ploidy varies among different organisms, including those that are always haploid (bacteria), either haploid or diploid (eg,
         <em>
          Saccharomyces
         </em>
         species [yeast]), consistently diploid (eg, mammals) (see
         <a class="local">
          'Diploid'
         </a>
         above), or polyploid (eg, hexaploid wheat). Different tissues in multicellular organisms may have different ploidies (eg, mammalian hepatocytes are predominantly tetraploid). The gametes (ova and sperm) are haploid (see
         <a class="local">
          'Haploid'
         </a>
         above). Ploidy may change in illness, eg, cardiomyocytes display polyploidy in the setting of heart failure. The designation of ploidy is based on the predominant ploidy of cells in the organism. The designation of ploidy is based on the predominant ploidy of cells in the organism.
        </p>
        <p class="headingAnchor" id="H3924047198">
         <span class="h2">
          Polygenic risk (PGR) score
         </span>
         <span class="headingEndMark">
          —
         </span>
         An estimate of an individual's genetic risk for a specific polygenic phenotype that is derived from contributions of alleles at multiple loci, up to thousands. Allele-specific contributions are estimated using specialized linear regression methods. The scores are typically generated in a model-building population, then validated in additional independent test populations.
        </p>
        <p>
         PGR is synonymous with polygenic score and contrasts with genetic risk score, which calculates the contribution of the known risk alleles carried by an individual. (See
         <a class="local">
          'Genetic risk score'
         </a>
         above and
         <a class="medical medical_review" href="/z/d/html/14600.html" rel="external">
          "Principles of complex trait genetics", section on 'Polygenic risk scores'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H4202333271">
         <span class="h2">
          Polygenic trait/polygenic disease
         </span>
         <span class="headingEndMark">
          —
         </span>
         In contrast to monogenic diseases, polygenic diseases are those for which the inherited trait(s) is explained by more than one gene. (See
         <a class="local">
          'Monogenic trait/monogenic disease'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H1040708322">
         <span class="h2">
          Polymerase chain reaction (PCR)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A method of specifically amplifying a unique target sequence (DNA or RNA) in the laboratory. PCR uses specific primers and repeated cycles of heating and cooling with a heat-stable DNA polymerase to replicate the template material exponentially. (See
         <a class="medical medical_review" href="/z/d/html/2896.html" rel="external">
          "Polymerase chain reaction (PCR)"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H79497895">
         <span class="h2">
          Polymorphism
         </span>
         <span class="headingEndMark">
          —
         </span>
         Polymorphism can refer to a genetic polymorphism. (See
         <a class="local">
          'Genetic polymorphism'
         </a>
         above.)
        </p>
        <p>
         It can also refer to any biologic marker (DNA, RNA, or protein) with two or more states. Protein polymorphisms (varying amino acid sequence) can result from DNA polymorphisms or from differential RNA splicing (different isoforms), which in turn can result from sequence variation, epigenetic phenomena, or temporal/spatial/environmental differences.
        </p>
        <p class="headingAnchor" id="H3919422493">
         <span class="h2">
          Quantitative traits and quantitative trait loci (QTL)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A quantitative trait locus (QTL) is a genomic region linked or associated with a "quantitative" trait.
        </p>
        <p>
         In contrast to traits that have discrete values (such as male or female), quantitative traits refer to quantifiable phenotypes that vary along a continuum of values rather.
        </p>
        <p>
         Examples include clinical phenotypes (blood pressure or anthropomorphic measures such as height), intermediate disease phenotypes (serum immunoglobulin E levels), and molecular traits (serum biomarker levels, gene or protein expression levels). Each QTL explains a portion of the variance of a trait.
        </p>
        <p>
         Another example is the level of fetal hemoglobin (Hb F). (See
         <a class="medical medical_review" href="/z/d/html/90749.html" rel="external">
          "Fetal hemoglobin (Hb F) in health and disease", section on 'Quantitative trait loci associated with HBG expression'
         </a>
         .)
        </p>
        <p>
         Quantitative traits are sometimes referred to as "complex" traits, reflecting the fact that multiple genes, the environment, and gene-environment interactions all contribute to an individual's trait value. (See
         <a class="medical medical_review" href="/z/d/html/14600.html" rel="external">
          "Principles of complex trait genetics"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H92059740">
         <span class="h2">
          Read depth
         </span>
         <span class="headingEndMark">
          —
         </span>
         In genomic or gene sequencing, the number of independent times each base in a targeted region has been sequenced. Typically expressed as an average X coverage (for example, 20X = an average of 20 sequence reads per base). A minimum read depth of 30X is often required for clinical-grade sequencing. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2624951772">
         <span class="h2">
          Reading frame
         </span>
         <span class="headingEndMark">
          —
         </span>
         The starting point in translating the DNA sequence to protein. Since each codon includes three nucleotides, the reading frame can be initiated at one of three nucleotides. Offsetting the reading frame (such as by a frameshift) changes the amino acid composition of the encoded protein  (
         <a class="graphic graphic_figure graphicRef78767" href="/z/d/graphic/78767.html" rel="external">
          figure 4
         </a>
         ). (See
         <a class="local">
          'Expression quantitative trait locus (eQTL)'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H2322458728">
         <span class="h2">
          Recombinant
         </span>
         <span class="headingEndMark">
          —
         </span>
         Recombinant has different meanings in different contexts. For inheritance patterns, recombinant refers to offspring whose genotype and phenotype combinations differ from their parents, implying genetic recombination between the loci under study.
        </p>
        <p>
         For laboratory techniques, recombinant technologies (also called genetic engineering), are molecular genetic approaches that use the process of homologous recombination to manipulate genotypes for experimental purposes. Examples include transgenic models where specific genetic loci are either knocked-out (removed) or knocked-in (introduced) to enable study of the locus; recombinant inbred mouse strains; recombinant viral transfection for synthesis of protein.
        </p>
        <p class="headingAnchor" id="H217868057">
         <span class="h2">
          Recombination
         </span>
         <span class="headingEndMark">
          —
         </span>
         The process of exchanging DNA sequence between two homologous chromosome regions. Mandatory recombination occurs at least once per aligned chromosome pair during meiosis. The exchange results in the creation of novel haplotypes that are combinations of the grandparental haplotypes present in a diploid cell. (See
         <a class="local">
          'Meiosis'
         </a>
         above.)
        </p>
        <p>
         Exchange of unequal sequence content (non-homologous recombination) can introduce DNA gains and losses of thousands or millions of bases. These gains and losses result in structural genetic variation and copy number variants (CNVs). (See
         <a class="local">
          'Copy number variation (CNV)'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H1891457425">
         <span class="h2">
          Repulsion (in trans)
         </span>
         <span class="headingEndMark">
          —
         </span>
         The state in which alleles at two distinct loci are on physically distinct chromosomal strands (ie, in trans). By definition, these variants are not part of the same haplotype  (
         <a class="graphic graphic_figure graphicRef58385" href="/z/d/graphic/58385.html" rel="external">
          figure 7
         </a>
         ). In the example of dominant and recessive alleles, repulsion gametes formed are Ab and aB.
        </p>
        <p>
         The opposite relationship is coupling. (See
         <a class="local">
          'Coupling (in cis)'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H741224604">
         <span class="h2">
          Risk allele
         </span>
         <span class="headingEndMark">
          —
         </span>
         An allele associated with a disease phenotype that typically acts in combination with other genetic or environmental factors. Though a risk allele is often that which is least common (ie, the minor allele), risk alleles associated with some complex traits may be the more common allele.
        </p>
        <p class="headingAnchor" id="H2321139793">
         <span class="h2">
          RNA
         </span>
         <span class="headingEndMark">
          —
         </span>
         RNA (ribonucleic acid) is a polymer consisting of a phosphoribose backbone and the bases adenine, guanine, uracil, and cytosine as side chains. Many viruses use RNA rather than DNA as the principal form of genetic information (RNA viruses).
        </p>
        <p>
         There are several different types of RNAs that have diverse structures and functions.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          mRNA
         </strong>
         – Messenger RNA (mRNA) is transcribed from the coding strand of DNA and transmits the genetic information to the protein synthesis machinery, serving as an intermediary between a gene's DNA sequence and its encoded protein.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          rRNA
         </strong>
         – Ribosomal RNA (rRNA) is an integral component of ribosomes, the organelles responsible for protein synthesis.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          tRNA
         </strong>
         – Transfer RNAs (tRNAs) carry specific amino acids and recognize the corresponding codons of the mRNA during protein synthesis.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Regulatory RNAs
         </strong>
         – There are several types of regulatory RNAs such as micro RNAs (miRs), long noncoding RNAs (LNCs, the ribonucleoproteins involved in mRNA splicing), and PIWI-interacting RNAs (piRNAs). PIWI (P-element-induced wimpy testis) designates a class of proteins that may regulate stem cells and appear to be aberrantly expressed in some cancers [
         <a href="#rid5">
          5
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression", section on 'Transcription'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H3875175161">
         <span class="h2">
          RNA interference (RNAi)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A ubiquitous intracellular process mediated by small RNA species, whereby specific mRNAs are targeted for editing, degradation, or clearance. RNAi has important roles in the regulation of gene expression, developmental processes, cellular defense, and epigenetic effects.
        </p>
        <p>
         RNAi technology (also called antisense technology) has been used in the laboratory to test the function of a gene by preventing its expression. It has been used clinically as a means of posttranscriptional gene silencing to reduce the expression of viral or cancer genes, or to lower cholesterol. The specific therapies are sometimes referred to as antisense oligonucleotides (ASOs; AS-ODNs). Other therapeutic applications are in the fields of hematology, oncology, and neurodegenerative disease. (See
         <a class="medical medical_review" href="/z/d/html/98724.html" rel="external">
          "Overview of gene therapy, gene editing, and gene silencing", section on 'RNA interference'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H868338761">
         <span class="h2">
          Sequencing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Determination of the nucleotide base sequence of a gene or collection of genes that determines the amino acid sequence of a protein. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2542927585">
         <span class="h2">
          Sex chromosomes
         </span>
         <span class="headingEndMark">
          —
         </span>
         The X and Y chromosomes, which determine the sex of the individual (females are XX; males are XY).
        </p>
        <p class="headingAnchor" id="H817773338">
         <span class="h2">
          Sex-linked
         </span>
         <span class="headingEndMark">
          —
         </span>
         A gene is sex-linked if it is located on a sex chromosome rather than on an autosome. A gene's inheritance pattern is also referred to as sex-linked if the pattern corresponds to that of known sex-linked genes (rather than autosomal genes). (See
         <a class="local">
          'Autosomal'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H1888984689">
         <span class="h2">
          Silencing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Regulation that prevents the expression of a gene. Mechanisms of silencing include gene methylation (see
         <a class="local">
          'Methylation'
         </a>
         above), destruction of messenger RNA, or prevention of protein translation. (See
         <a class="local">
          'RNA interference (RNAi)'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H2407636019">
         <span class="h2">
          Single nucleotide polymorphism (SNP)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A single nucleotide polymorphism (SNP; pronounced "snip") is a polymorphism (difference in base sequence) that affects a single base pair. This terminology was previously used to refer to variation that had a population frequency of at least 1 percent. The term SNP is commonly used in research such as in GWAS studies (see
         <a class="local">
          'Genome-wide association study (GWAS)'
         </a>
         above). In clinical diagnostic testing, the term "variant" with a qualifier about pathogenicity is preferred (although use is inconsistent).
        </p>
        <p>
         SNP may also be used to refer to polymorphisms in a testing platform such as a SNP array. (See
         <a class="medical medical_review" href="/z/d/html/2902.html" rel="external">
          "Genetic association and GWAS studies: Principles and applications", section on 'Single nucleotide polymorphisms'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics", section on 'Allele-specific oligonucleotide hybridization'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics", section on 'Array comparative genomic hybridization'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H940268600">
         <span class="h2">
          Somatic
         </span>
         <span class="headingEndMark">
          —
         </span>
         Referring to tissues that are not within the germline. Somatic variants arise in somatic tissues and are therefore not passed from parent to offspring. Somatic variation is common in cancer.
        </p>
        <p class="headingAnchor" id="H2952035007">
         <span class="h2">
          Structural genetic variation
         </span>
         <span class="headingEndMark">
          —
         </span>
         A term that encompasses a variety of large-scale genomic aberrations, including segmental rearrangements, translocations, or inversions and copy-number variants (CNVs). (See
         <a class="local">
          'Copy number variation (CNV)'
         </a>
         above.)
        </p>
        <p>
         Large rearrangements or deletions can be visualized through karyotyping. Smaller variants, particularly CNVs, segmental duplications, and interchromosomal interstitial rearrangements, are assessed by array comparative genomic hybridization (array CGH) or SNP arrays. (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics", section on 'Genotyping microarrays'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2929531485">
         <span class="h2">
          Syntenic
         </span>
         <span class="headingEndMark">
          —
         </span>
         Describing genetic loci that reside on the same chromosome. As an example, the genes causing Birt-Hogg-Dubé syndrome (Folliculin [
         <em>
          FLCN
         </em>
         ], at chromosome 17p11) and early-onset breast cancer (
         <em>
          BRCA1
         </em>
         , at chromosome 17q21) are syntenic to each other on chromosome 17. However, because they are far apart from each other, they are not genetically linked. (See
         <a class="local">
          'Linkage'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H3656629141">
         <span class="h2">
          Telomere
         </span>
         <span class="headingEndMark">
          —
         </span>
         Region at the ends of a chromosome that prevents the loss of genetic material or the accidental fusion of two chromosomes together during cell division. Telomeres of chromosomes in most cells shorten as an individual ages. Telomere length is maintained by the enzyme telomerase. (See
         <a class="local">
          'Telomerase'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H987986146">
         <span class="h2">
          Telomerase
         </span>
         <span class="headingEndMark">
          —
         </span>
         Multicomponent enzyme that extends the length of telomeres. Telomerase variants are seen in some inherited telomere syndromes. (See
         <a class="medical medical_review" href="/z/d/html/100663.html" rel="external">
          "Dyskeratosis congenita and other telomere biology disorders"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3486029516">
         <span class="h2">
          Translocation
         </span>
         <span class="headingEndMark">
          —
         </span>
         A translocation is a structural chromosomal abnormality whereby chromosome segments are exchanged (swapped) between two non-homologous chromosomes.
        </p>
        <p>
         This form of rearrangement can be balanced, when the translocation does not result in any significant loss or gain of genetic material in the resultant gamete or cell; or unbalanced, when there is a gain or loss of genetic material in the resultant gamete or cell. (See
         <a class="medical medical_review" href="/z/d/html/4490.html" rel="external">
          "Chromosomal translocations, deletions, and inversions", section on 'Translocations'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2005622782">
         <span class="h2">
          Tumor suppressor gene
         </span>
         <span class="headingEndMark">
          —
         </span>
         A tumor suppressor gene is a gene that protects against the development or growth of tumors. Tumor suppressor genes typically act in a recessive manner (both normal copies must be lost for a tumor to develop). In contrast, oncogenes typically act in a dominant manner. (See
         <a class="local">
          'Oncogene'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H138467379">
         <span class="h2">
          Uniparental disomy
         </span>
         <span class="headingEndMark">
          —
         </span>
         The inheritance of two copies of a chromosome (or part of a chromosome) from one parent, and no copy from the other parent, due either to nondisjunction errors during either the first or second phases of meiosis, or to chromosomal alterations in early fetal development.
        </p>
        <p>
         Nondisjunction during the first phase of meiosis (meiosis I) will result in inheritance of each of the grandparental chromosomes from one parent, termed "heterodisomy." In contrast, nondisjunction during meiosis II results in inheritance of two identical copies of one grandparental chromosome, termed "isodisomy."
        </p>
        <p class="headingAnchor" id="H2504463718">
         <span class="h2">
          Variant
         </span>
         <span class="headingEndMark">
          —
         </span>
         Used to refer to a specific change in either DNA or protein sequence.
        </p>
        <p>
         In microbiology, a variant refers to an organismal isolate whose genetic sequence varies from that of its reference organism. (See
         <a class="local">
          'Viral variant'
         </a>
         below.)
        </p>
        <p>
         For germline variants, the American College of Medical Genetics and Genomics (ACMG) and the Association for Molecular Pathology have recommended use of a five-tier terminology system for the clinical classification of genetic variants  (
         <a class="graphic graphic_table graphicRef122646" href="/z/d/graphic/122646.html" rel="external">
          table 1
         </a>
         ), consisting of the following designations [
         <a href="#rid4">
          4
         </a>
         ]:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Pathogenic variant (PV)
         </strong>
         – A disease-causing variant, as determined by very strong genetic and experimental evidence, including consistent familial co-segregation with disease and definitive functional studies.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Likely pathogenic variant (LPV)
         </strong>
         – A variant with strong, but not definitive, evidence of pathogenicity based on its similarity to known pathogenic variants, co-segregation with disease in families or populations, and functional evidence.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Variant of uncertain significance (VUS)
         </strong>
         – A variant for which the specific criteria for the other four criteria are not met, or when contradictory lines of evidence in support of both benign or pathogenic classifications are present. Also called variant of unknown significance.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Likely benign variant
         </strong>
         – A variant with multiple supporting (but not conclusive) lines of evidence to suggest it is not disease-causing.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Benign variant
         </strong>
         – A variant with conclusive evidence as not disease-causing, as determined typically (but not only) by a high prevalence of the variant in the general (healthy) population, at a prevalence that exceeds that of the suspected disease.
        </p>
        <p>
        </p>
        <p>
         For clinical testing and management, these terms are preferred over "mutation." Mutation remains appropriate in certain contexts such as when referring to a pathophysiologic process or to specific changes in a region of DNA (or less commonly, a protein). (See
         <a class="local">
          'Mutation, mutant'
         </a>
         above.)
        </p>
        <p>
         Additional information about this classification and its application to genomic testing is presented separately. (See
         <a class="medical medical_review" href="/z/d/html/96539.html" rel="external">
          "Secondary findings from genetic testing", section on 'Definitions and classification of variants'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H439190776">
         <span class="h2">
          Variant allele frequency
         </span>
         <span class="headingEndMark">
          —
         </span>
         See allele frequency. (See
         <a class="local">
          'Allele frequency'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H3552163370">
         <span class="h2">
          Variant of uncertain significance (VUS)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A classification term used in clinical DNA sequencing reports to signify genetic polymorphisms (variants) for which the pathogenicity (likelihood of causing disease) cannot be determined easily and that cannot be readily classified as "pathogenic," "likely pathogenic," "benign, or "likely benign." Also called variant of unknown significance. (See
         <a class="local">
          'Variant'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H2017271020">
         <span class="h2">
          Viral variant
         </span>
         <span class="headingEndMark">
          —
         </span>
         A viral isolate with a genome sequence that differs from that of the reference virus, regardless of whether the sequence variant alters the virus's phenotype. A viral strain is a viral variant with a sequence change that confers a unique viral phenotype (eg, altered replication rate, infectivity, or lethality). In contrast, variants with an impact limited to antigenicity are referred to as having different serotypes rather than as different strains. (See
         <a class="medical medical_review" href="/z/d/html/126981.html" rel="external">
          "COVID-19: Epidemiology, virology, and prevention", section on 'Variants of concern'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3721925573">
         <span class="h2">
          Whole genome sequencing
         </span>
         <span class="headingEndMark">
          —
         </span>
         A sequencing strategy that provides the DNA sequence for the entire genome, including exons, introns, and other noncoding sequence. In contrast, exome sequencing only determines the sequence of gene-coding regions. (See
         <a class="local">
          'Exome sequencing'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H832339165">
         <span class="h2">
          X-inactivation
         </span>
         <span class="headingEndMark">
          —
         </span>
         An epigenetic process (see
         <a class="local">
          'Epigenetic change'
         </a>
         above) that occurs in all female mammalian cells, whereby one of the two X chromosomes are randomly rendered inactive, such that all subsequent gene expression is derived from the other (active) X chromosome. This is sometimes called lyonization, after Mary Lyon, who did important early work on this phenomenon. (See
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)", section on 'Sex-linked patterns'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/14601.html" rel="external">
          "Principles of epigenetics", section on 'Types of processes that are regulated'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3472001134">
         <span class="h1">
          SUMMARY
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Definitions
         </strong>
         – Commonly used genetics terms are defined above. (See
         <a class="local">
          'Definitions'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Genetics concepts
         </strong>
         – Basic genetics concepts are discussed in separate topic reviews. (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/129424.html" rel="external">
          "Basic genetics concepts: Chromosomes and cell division"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/16647.html" rel="external">
          "Genomic disorders: An overview"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/14600.html" rel="external">
          "Principles of complex trait genetics"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/14601.html" rel="external">
          "Principles of epigenetics"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Clinical applications
         </strong>
         – Use of genetic information in clinical care is also discussed in separate topic reviews. (See
         <a class="medical medical_review" href="/z/d/html/2901.html" rel="external">
          "Genetic testing"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/103304.html" rel="external">
          "Genetic counseling: Family history interpretation and risk assessment"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/14599.html" rel="external">
          "Personalized medicine"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/96539.html" rel="external">
          "Secondary findings from genetic testing"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Genetics tools
         </strong>
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          DNA sequencing
         </strong>
         – (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          PCR
         </strong>
         – (See
         <a class="medical medical_review" href="/z/d/html/8286.html" rel="external">
          "PCR testing for the diagnosis of herpes simplex virus in patients with encephalitis or meningitis"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Cytogenetics
         </strong>
         – (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Gene expression profiling/genome-wide association studies (GWAS)
         </strong>
         – (See
         <a class="medical medical_review" href="/z/d/html/14602.html" rel="external">
          "Tools for genetics and genomics: Gene expression profiling"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2902.html" rel="external">
          "Genetic association and GWAS studies: Principles and applications"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Animal models
         </strong>
         – (See
         <a class="medical medical_review" href="/z/d/html/2894.html" rel="external">
          "Tools for genetics and genomics: Specially bred and genetically engineered mice"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2903.html" rel="external">
          "Tools for genetics and genomics: Model systems"
         </a>
         .)
        </p>
        <p>
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li class="breakAll">
          Brenner's Encyclopedia of Genetics, 2nd ed, Maloy S, Hughes K (Eds), Elsevier, 2013.
         </li>
         <li>
          <a class="nounderline abstract_t">
           Van Loo P, Campbell PJ. ABSOLUTE cancer genomics. Nat Biotechnol 2012; 30:620.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cooke Bailey JN, Igo RP Jr. Genetic Risk Scores. Curr Protoc Hum Genet 2016; 91:1.29.1.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet Med 2015; 17:405.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Han YN, Li Y, Xia SQ, et al. PIWI Proteins and PIWI-Interacting RNA: Emerging Roles in Cancer. Cell Physiol Biochem 2017; 44:1.
          </a>
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 2898 Version 60.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="javascript:alert('بدون لینک');" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : Brenner's Encyclopedia of Genetics, 2nd ed, Maloy S, Hughes K (Eds), Elsevier, 2013.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22781683" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : ABSOLUTE cancer genomics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27727440" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : Genetic Risk Scores.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25741868" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29130960" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : PIWI Proteins and PIWI-Interacting RNA: Emerging Roles in Cancer.
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
