<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Genetic association and GWAS studies: Principles and applications
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Genetic association and GWAS studies: Principles and applications
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Genetic association and GWAS studies: Principles and applications
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Author:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            John Attia, MD, PhD, FRCPC, FRACP
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Benjamin A Raby, MD, MPH
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Jennifer S Tirnauer, MD
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            May 20, 2022.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H1">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         Interest in the genetic determinants of disease originated with Gregor Mendel's observations on the genetics of the pea in the 1860s. Subsequent studies have identified many of the genes responsible for "Mendelian" diseases, conditions that follow a clear familial pattern. However, diseases inherited in a Mendelian fashion (eg, Huntington disease and cystic fibrosis) are rare.
        </p>
        <p>
         The Human Genome Project has generated growing interest in genetic contribution to "complex" diseases. Such diseases combine some familial predisposition with a large environmental contribution. Examples include cardiovascular disease, diabetes, asthma, cancer, and obesity.
        </p>
        <p>
         This topic will discuss the principles and clinical applications of genetic association studies in the elucidation of the genetic basis for common diseases with complex genetic components. Additional discussions of modes of inheritance and a glossary of genetic terms are presented separately:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Monogenic disorders – (See
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Complex trait disorders – (See
         <a class="medical medical_review" href="/z/d/html/14600.html" rel="external">
          "Principles of complex trait genetics"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Glossary of terms – (See
         <a class="medical medical_review" href="/z/d/html/2898.html" rel="external">
          "Genetics: Glossary of terms"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H2">
         <span class="h1">
          TERMINOLOGY AND STUDY DESIGN
         </span>
        </p>
        <p class="headingAnchor" id="H3898615991">
         <span class="h2">
          Genetic association studies
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genetic association studies are analogous to traditional epidemiologic association studies. Instead of seeking association between traditional risk variables (eg, hypertension) and disease outcomes (eg, stroke), a genetic association study looks for an association between a genetic variable and a specified condition.
        </p>
        <p class="headingAnchor" id="H903496082">
         <span class="h2">
          Single nucleotide polymorphisms
         </span>
         <span class="headingEndMark">
          —
         </span>
         The genetic variable most commonly studied is a single nucleotide polymorphism (SNP, pronounced "snip"). A SNP is a base pair change that occurs in at least 1 percent of the population. SNPs are not clearly deleterious, in contrast to mutations, which occur less frequently and generally have an adverse effect on protein function. (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression", section on 'Genetic variation'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3229563817">
         <span class="h2">
          Study design
         </span>
         <span class="headingEndMark">
          —
         </span>
         The usual study design for association studies is case-control or nested case-control, where controls are selected from the general population. Other designs, using family-based data (eg, pedigrees, parent-offspring trios, or sibling pairs) are becoming less popular due to difficulties in obtaining sufficient sample size and the need for specialized statistical methods for analysis [
         <a href="#rid1">
          1
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H3">
         <span class="h2">
          Candidate gene approach
         </span>
         <span class="headingEndMark">
          —
         </span>
         Earlier gene association studies employed a "candidate" gene approach in which a genetic variant of interest was selected on the basis of the known biology of a disease. The genetic variant may be of interest because of its presumed biologic function, or association with a disease in previous studies. In this approach, one or a small number of known variants are genotyped, usually by PCR methods, in a number of cases and controls.
        </p>
        <p class="headingAnchor" id="H4">
         <span class="h2">
          Genome-wide approach (GWAS)
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genome-wide association studies (GWAS, pronounced "gee-wass") test hundreds of thousands of genes simultaneously  (
         <a class="graphic graphic_figure graphicRef54220" href="/z/d/graphic/54220.html" rel="external">
          figure 1
         </a>
         ). GWAS use microarray technology, which "arrays" a large series of test sequences on a solid surface. Technological advances with microarrays have enabled researchers to genotype an individual for between 500,000 and 4 million SNPs on a platform no bigger than a microscope slide.
        </p>
        <p>
         The principle behind microarrays is that DNA from an individual is hybridized against an array of short oligonucleotide probes (ie, DNA sequences) that are immobilized on a surface (the "array"). Sequences on the array are chosen to assay the regions of the genome with the most variation. If the individual being tested has the DNA sequence that matches a probe on the array, the DNA binds and is detected  (
         <a class="graphic graphic_figure graphicRef52164" href="/z/d/graphic/52164.html" rel="external">
          figure 2
         </a>
         ).
        </p>
        <p>
         Most GWAS are case-control studies, where a group of cases and a group of separate controls are gathered, DNA is isolated, and microarray data is obtained on each individual. The case-control design facilitates large sample sizes and therefore greater power, which is particularly important for detecting potentially small genetic effects. GWAS have also been performed using longitudinal cohorts, with cases and controls "nested" within the cohort.
        </p>
        <p>
         At the most basic level, the distribution of the three genotypes for each SNP (AA, Aa, aa) is compared between cases and controls using a Chi-square test. This is repeated over and over for each of the 500,000 or more SNPs on the array. As human genetic variation is increasingly catalogued, for example in databases such as "Hap Map" and "1000 Genomes," this information can be used to impute genetic variants that were not directly measured by the microarray. In other words, the 500,000 to 2 million SNPs that are directly assayed are used to make a best guess on the other SNPs that were not measured, often yielding up to 6 million SNPs or more to test for association. With reference panels such as the Haplotype Reference Consortium (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.sanger.ac.uk%2Fcollaboration%2Fhaplotype-reference-consortium%2F&amp;token=1ffrmHO3o2U9TnJFzjdGFuldlIMBxgRdcKcQCeQT2pJArbcREaiD7JgSja7b4QS0surUB3%2Bi97MxuCQPRvGBbY0h41lm4HBNBzemcHn4OMg%3D&amp;TOPIC_ID=2902" target="_blank">
          HRC
         </a>
         ), imputation can yield almost 40 million SNPs per person.
        </p>
        <p>
         The number of association studies in the medical literature has expanded dramatically. Between 2007 and 2009, GWAS identified nearly 1000 SNPs associated with a range of human traits and common diseases [
         <a href="#rid2">
          2
         </a>
         ]. By March 2011, that number had risen to almost 4000 SNPs for over 200 conditions [
         <a href="#rid3">
          3
         </a>
         ]. The GWAS explosion has continued, as described in a manually curated catalogue of GWAS studies to help readers navigate the growing body of data [
         <a href="#rid4">
          4
         </a>
         ].
        </p>
        <p>
         Despite the issue of "missing heritability" (see
         <a class="local">
          'Missing heritability'
         </a>
         below), GWAS have had some notable successes identifying genes of large effect. An example is the relationship between a complement factor H (CFH) polymorphism and age-related macular degeneration (AMD). This polymorphism is involved in regulation of the alternate complement pathway that results in increased inflammation. Individuals with one allele coding for a histidine substitution for tyrosine in position 402 of the CFH protein (CFH Y402H) are at increased risk for AMD. Meta-analysis indicates a possible multiplicative model in that each copy of the C allele at this locus (coding for histidine) increases the risk of AMD by approximately 2.5-fold compared to the T allele (ie, the TC heterozygote group has an odds ratio [OR] for AMD of 2.5 and the CC homozygote has an odds ratio of approximately 6 relative to the TT genotype group) [
         <a href="#rid5">
          5
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H626606">
         <span class="h1">
          GENETIC DETERMINANTS OF COMPLEX TRAITS
         </span>
        </p>
        <p class="headingAnchor" id="H5">
         <span class="h2">
          Common disease-common variant hypothesis
         </span>
         <span class="headingEndMark">
          —
         </span>
         The basic assumption regarding genetic determinants of complex disease is that common variants in many genes will each lead to a small rise (or fall) in the risk of disease, and that the overall risk of disease is determined by the combination of multiple variants and environmental exposures [
         <a href="#rid6">
          6
         </a>
         ]. The single nucleotide polymorphisms (SNPs) chosen for inclusion on microarrays are usually the more common variants in the population.
        </p>
        <p>
         A competing theory is "common disease-rare variant," in which it is anticipated that most common diseases will be due to multiple rare variants (&lt;1 percent), perhaps even unique mutations. If this is the case, GWAS would be less useful, because they are designed to reveal associations that occur commonly; and one would need to move to whole genome sequencing to detect rare variants associated with a disease. The technology for whole genome sequencing has advanced quickly and has become competitively priced, and hence it is becoming more common. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p>
         In reality, the dichotomy between common and rare variants is not absolute, and results from one type of association will likely complement the other. As an example, sequencing of the cystic fibrosis gene revealed that one major allele, delta F508, accounts for 67 percent of disease in Europe, but the remainder is due to hundreds of rarer disease alleles/mutations [
         <a href="#rid7">
          7
         </a>
         ]. A study from 2018 indicated that many of the signals seen in complex diseases overlapped with the signals seen for Mendelian diseases that share the same phenotype; it seems likely therefore that common variants with small effects in a particular gene could explain the disease risk for a complex disease and that rare mutations with very large effects in that same gene could explain the disease risk for a Mendelian version of that disease [
         <a href="#rid8">
          8
         </a>
         ].
        </p>
        <p>
         GWAS have led to the successful identification of many genes involved in complex disease, such as coronary artery disease, type 2 diabetes, stroke, multiple sclerosis, breast cancer, bipolar disorder, rheumatoid arthritis, and Alzheimer's disease [
         <a href="#rid9">
          9-17
         </a>
         ]. These studies largely support the hypothesis that multiple genes, each conferring small relative risks (ie, 1.2-1.8), collectively contribute to increased disease risk. Other GWAS results have shed light on the pathogenesis of a disease process (eg, the role of melatonin in type 2 diabetes) [
         <a href="#rid18">
          18,19
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H6">
         <span class="h2">
          Missing heritability
         </span>
         <span class="headingEndMark">
          —
         </span>
         Despite many successes, GWAS have not explained as much of the variance in complex traits as was originally expected. Heritability studies look at the variance in a trait and use family relationships (eg, twins and siblings) to partition how much of the variance is due to genetic factors and how much is due to environmental factors. As an example, studies have estimated that height is 80 to 90 percent "heritable," but GWAS and meta-analyses of GWAS on height have turned up more than 40 genes, accounting for less than 5 percent of the variance [
         <a href="#rid20">
          20,21
         </a>
         ].This discrepancy between the numerator (amount of heritability explained from GWAS SNPs) and denominator (amount of total heritability from family/population data) has been termed "missing heritability" [
         <a href="#rid2">
          2,22
         </a>
         ].
        </p>
        <p>
         There are several hypotheses about what may account for the "missing heritability":
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Some of the variation may be explained by rare alleles that will only be detected by whole genome sequencing, rather than SNPs. However, results from next-generation sequencing (NGS) studies (see
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         ) do not always find major new genetic associations or explain further heritability [
         <a href="#rid23">
          23
         </a>
         ]. In many cases the rare variants discovered have been "tagged" by common variants, and the extra sequencing has not revealed much additional information. Debate remains about whether rare variants are effectively tagged by common variants or whether they are independent. This debate is made more confusing by the finding that the answer depends on which measure of linkage disequilibrium is used, as different measures can give conflicting answers [
         <a href="#rid24">
          24
         </a>
         ]. In other cases, NGS does indeed identify multiple new variants that increase the explained heritability [
         <a href="#rid25">
          25
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Some of the variation may be explained by copy number variants (CNVs). CNVs are polymorphisms that consist of deletions or duplications of whole segments of DNA, spanning at least 1000 bases [
         <a href="#rid26">
          26
         </a>
         ]. Although CNVs can show association for some diseases where SNPs have not yielded any signal (eg, autism) [
         <a href="#rid27">
          27
         </a>
         ], it seems unlikely that common CNVs will explain most of the unaccounted heritability for the majority of complex traits [
         <a href="#rid28">
          28
         </a>
         ]. An investigation of the association of CNVs with eight complex diseases found that most CNV signals were in areas that had already been highlighted by SNP associations, and most CNVs were "tagged" by SNPs [
         <a href="#rid29">
          29
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/16647.html" rel="external">
          "Genomic disorders: An overview", section on 'Copy number variations'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Current methodology may not be adequate to detect gene/gene interactions (epistasis). This is particularly true if each genetic variant alone shows no effect but shows an effect in the presence of the other variant. Modelling indicates that gene/gene and higher order interactions could account for missing heritability, but would not have been detected due to insufficient sample size in the studies to date [
         <a href="#rid30">
          30
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Heritability may also be accounted for by epigenetic changes (eg, methylation of DNA, modification of histone proteins). There is increasing evidence that these non-DNA changes can also be passed on from generation to generation, potentially through various RNAs, challenging the traditional dogma that sperm carries only DNA [
         <a href="#rid31">
          31,32
         </a>
         ]. Alternatively, epigenetic changes influence gene expression and can explain some of the variation in human disease not captured purely by sequence-based methods [
         <a href="#rid33">
          33
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/14601.html" rel="external">
          "Principles of epigenetics"
         </a>
         .)
        </p>
        <p>
        </p>
        <p>
         Others, however, maintain that there is no "missing heritability," arguing that estimates of heritability are complex and may have been interpreted simplistically. High heritability does not necessarily mean genetic determination or genes of large effect; it may reflect a large component of shared environment as well [
         <a href="#rid34">
          34
         </a>
         ]. For example, a study looking at within- and between-family polygenic risk scores found that a substantial proportion of within-family heritability was explained by socioeconomic status; this raises the possibility that shared environment may be underestimated and shared genetics overestimated by current methods [
         <a href="#rid35">
          35
         </a>
         ].
        </p>
        <p>
         Overestimates of total heritability may have created "phantom heritability" (presumed genetic contributions that could not be identified even if all causative genetic variants were identified) [
         <a href="#rid30">
          30
         </a>
         ]. This line of argument is supported by the inability to explain all of the genetic variance in a trait even when thousands of SNPs are evaluated. For example, one model using nearly 30,000 SNPs could account for 45 percent of the variance in height [
         <a href="#rid36">
          36
         </a>
         ]. Fitting all genotyped SNPs explains a substantial, but not a majority, of the heritability for such complex phenotypes as height, body mass index (BMI), von Willebrand factor levels, and QT interval [
         <a href="#rid37">
          37
         </a>
         ].
        </p>
        <p>
         The dichotomy between the common variant versus rare variant hypotheses is increasingly moot. It is becoming clear that the many genetic variants responsible for a disease span the whole spectrum of frequency from common to rare, and the spectrum of effect sizes ranges from small to large; together, it is the combined effect of hundreds to millions of genetic variants that determines disease risk, not to mention the environmental variables that also influence risk [
         <a href="#rid38">
          38
         </a>
         ]. Chasing the "missing" heritability may end up detracting from the more fruitful work of identifying the etiologic pathways involved in a disease or identifying new "druggable" targets for therapeutic development.
        </p>
        <p class="headingAnchor" id="H8">
         <span class="h1">
          JUDGING STUDY VALIDITY
         </span>
         <span class="headingEndMark">
          —
         </span>
         It is increasingly necessary for physicians to become familiar with GWAS data and how to gauge validity of these studies.
        </p>
        <p class="headingAnchor" id="H9">
         <span class="h2">
          Potential biases
         </span>
         <span class="headingEndMark">
          —
         </span>
         There are multiple potential biases for case-control studies in traditional epidemiology. The potential for bias is less, though not absent, in genetic studies. In contrast to most environmental exposures, the genetic "exposure" is not chosen by the participant, doesn't vary with age or calendar year, is not subject to recall bias, and is not influenced by the disease or treatment.
        </p>
        <p>
         One potential bias that cannot be avoided, however, is survivor bias, in that those who are included in a case-control study are those who have survived the initial "insult"  (
         <a class="graphic graphic_figure graphicRef50171" href="/z/d/graphic/50171.html" rel="external">
          figure 3
         </a>
         ). As an example, in a study of stroke, results may differ between a case-control and cohort GWAS if certain genes are associated with initial stroke severity, causing early mortality before recruitment into a case-control study.
        </p>
        <p class="headingAnchor" id="H10">
         <span class="h2">
          Validity framework
         </span>
         <span class="headingEndMark">
          —
         </span>
         Many groups have published guidelines for critical appraisal of genetic association studies [
         <a href="#rid39">
          39,40
         </a>
         ]. One framework, from the "Users' Guides to the Medical Literature" series, asks the following questions [
         <a href="#rid41">
          41,42
         </a>
         ]:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Was the disease
         <strong>
          phenotype
         </strong>
         properly defined and accurately recorded by someone blind to the genetic information?
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Have any potential differences between disease and non-disease groups, particularly
         <strong>
          ancestry
         </strong>
         , been properly addressed?
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Was measurement of the
         <strong>
          genetic variants
         </strong>
         (ie, genotyping) unbiased and accurate?
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Do the genotype proportions observe
         <strong>
          Hardy-Weinberg equilibrium
         </strong>
         ?
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Have the investigators adjusted for
         <strong>
          multiple comparisons
         </strong>
         ?
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Are the results
         <strong>
          consistent with other studies
         </strong>
         ?
        </p>
        <p>
        </p>
        <p>
         The issues raised in these questions are as follows:
        </p>
        <p class="headingAnchor" id="H11">
         <span class="h3">
          Phenotyping
         </span>
         <span class="headingEndMark">
          —
         </span>
         What may seem like a unique disease (eg, ischemic stroke), may be a heterogeneous group of diseases (eg, large vessel stroke, lacunar stroke, and embolic stroke), or may be variously defined (eg, clinical signs only, or CT/MRI imaging confirmation). Investigators may selectively report only the disease definition or subgroup that yields a significant association [
         <a href="#rid43">
          43
         </a>
         ].
        </p>
        <p>
         Alternatively, an apparent single disease entity may represent genetically separate but clinically similar diseases ("genetic heterogeneity"), as seen with epilepsy [
         <a href="#rid44">
          44
         </a>
         ]. Including diseases with different genetic backgrounds may obscure a true genetic association.
        </p>
        <p>
         To avoid bias, those who are doing the phenotyping should be blind to the results of the genotyping and vice versa. The possibility of bias is greater for candidate gene studies where single nucleotide polymorphisms (SNPs) are typed manually in contrast to GWAS in which the genotype identification is automated.
        </p>
        <p class="headingAnchor" id="H12">
         <span class="h3">
          Ancestry and comorbidities
         </span>
         <span class="headingEndMark">
          —
         </span>
         Although genetic association studies are less prone to traditional confounding than other case control studies, there are two potential sources of bias:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Ethnicity or racial mix
         </strong>
         — This particular form of confounding, referred to as "population stratification," occurs when disease and non-disease populations include a different ethnic/racial mix. If the likelihood of developing the condition of interest varies with ancestry, then any SNP with a different allele frequency between ancestral groups will appear to be linked (spuriously) to the disease  (
         <a class="graphic graphic_figure graphicRef78339" href="/z/d/graphic/78339.html" rel="external">
          figure 4
         </a>
         ).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         As with traditional confounders, the way to correct for this is to measure the confounder and adjust for it in the analysis or to stratify the analysis. Ethnic subgroups may be identified by self-report or by statistical analysis of SNPs themselves [
         <a href="#rid45">
          45,46
         </a>
         ]. As an example, a spurious association between the CYP3A4-V polymorphism and prostate cancer in African-Americans disappeared when results were adjusted for additional genetic markers associated with ancestry in the population studied [
         <a href="#rid47">
          47
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Comorbidities/associated phenotypes
         </strong>
         — An apparent association between a genotype and condition of interest may represent a direct association between the genotype and another condition, with the disease of interest being linked to this other condition. As an example, two GWAS showed an association between type 2 diabetes and a SNP in the FTO (fat mass and obesity associated) gene [
         <a href="#rid10">
          10,48
         </a>
         ]. These studies selected diabetic patients and controls irrespective of their body mass index (BMI); another study that matched diabetic patients and controls on BMI showed no association. Thus, obesity is associated with diabetes and the initial SNP was thought to be linked with diabetes when in fact it was linked with the associated obesity.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H13">
         <span class="h3">
          Accuracy of genotyping
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genotyping error is a threat to the validity of genetic association studies. Genotyping errors may arise if there is a problem with the DNA samples or with the technology that is employed to identify alleles [
         <a href="#rid49">
          49
         </a>
         ].
        </p>
        <p>
         DNA samples may differ between diseased and non-diseased participants in ways that lead to inaccuracies in genotyping. As an example, in a GWAS for type 2 diabetes, blood used for the control population was stored from a cohort in 1958 while samples from patients with known diabetes were from the present day. The older blood resulted in genotyping errors that led to false positive SNP associations [
         <a href="#rid50">
          50
         </a>
         ].
        </p>
        <p>
         Genotyping error rates vary widely, from &lt;1 to 30 percent [
         <a href="#rid51">
          51
         </a>
         ]; rates up to a few percent are common in even the best studies [
         <a href="#rid52">
          52
         </a>
         ]. It is common practice for investigators to cull SNPs that do not have a high call rate (eg, 95 to 98 percent). Call rate refers to the percent of DNA samples that can be genotyped. This increases the validity of the data but does not ensure that the genotype information is correct. The genotype may be misidentified as, for example, when one of the alleles in a heterozygote is harder to identify than the other, resulting in a heterozygote being mislabeled a homozygote. Checking for Hardy-Weinberg equilibrium may be one way of detecting genotyping problems. (See
         <a class="local">
          'Hardy-Weinberg equilibrium'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H14">
         <span class="h3">
          Hardy-Weinberg equilibrium
         </span>
         <span class="headingEndMark">
          —
         </span>
         In the same way that most continuous variables in medicine observe a normal distribution, most allele distributions observe what is called Hardy-Weinberg equilibrium (HWE). This describes the steady state where there are no selective forces (eg, mutation, inbreeding, selective survival) acting on a particular locus or gene. The Hardy-Weinberg law states that if there are two alleles (named A and a) at a particular locus, with frequency p and q respectively, then after one generation of random mating, the genotype frequencies of the AA, Aa, and aa groups in the population will be p
         <sup>
          2,
         </sup>
         2pq, and q
         <sup>
          2,
         </sup>
         respectively. Given that there are only two alleles possible, A or a, then:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         p + q = 1,
        </p>
        <p>
        </p>
        <p>
         and
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         p
         <sup>
          2
         </sup>
         +2pq + q
         <sup>
          2
         </sup>
         = 1.
        </p>
        <p>
        </p>
        <p>
         HWE is commonly used as a quality measure. It has become general practice, in a genetic association study, to check whether the allele frequencies at a particular SNP observe HWE proportions. Results are considered to be consistent with HWE when the p-value is &lt;0.05. HWE calculators are available online [
         <a href="#rid53">
          53,54
         </a>
         ]. For a cohort study, HWE should be tested in the whole study population, whereas for a case-control study, it should be tested in the controls, since these are supposedly representative of the general population.
        </p>
        <p>
         However, HWE is non-specific and may be insensitive [
         <a href="#rid55">
          55
         </a>
         ]. Genotyping error and population stratification are two of many factors that may upset HWE proportions. Other factors that may affect HWE include new mutations, inbreeding, or a selective advantage of one allele over another.
        </p>
        <p class="headingAnchor" id="H15">
         <span class="h3">
          Multiple comparisons
         </span>
         <span class="headingEndMark">
          —
         </span>
         The usual p-value for judging statistical significance in traditional epidemiological studies (0.05) is intended for a single comparison. The 0.05 threshold means that a result as extreme or more extreme than the one seen will occur by chance once in 20 times; this is taken as low enough to not be chance and to indicate a significant association. However, if one is looking at 100 SNPs, then by chance alone one might expect 5 SNPs to reach this threshold; in fact there is over a 99 percent chance that at least one will reach this threshold.
        </p>
        <p>
         In GWAS, in which 1 to 40 million SNPs are tested simultaneously, the possibilities for false-positive are countless. The current consensus is that for such large scale studies, a p-value in the range of 5x10
         <sup>
          -8
         </sup>
         (in contrast to the usual 5x10
         <sup>
          -2
         </sup>
         ) should be considered the threshold [
         <a href="#rid56">
          56
         </a>
         ]. Some have argued that a simple Bonferroni correction for 10
         <sup>
          6
         </sup>
         comparisons is overly conservative given that the tests are not independent; the finding of linkage disequilibrium (LD) means that the tests are correlated. A review has summarized the various approaches to determining the correct threshold, including techniques that account for LD, false-positive report probability, false discovery rate, and Bayesian methods [
         <a href="#rid57">
          57
         </a>
         ]. These various methods all suggest a threshold between 10
         <sup>
          -7
         </sup>
         and 10
         <sup>
          -8
         </sup>
         , and hence the original threshold of 5 x 10
         <sup>
          -8
         </sup>
         remains widely used.
        </p>
        <p class="headingAnchor" id="H16">
         <span class="h3">
          Replication
         </span>
         <span class="headingEndMark">
          —
         </span>
         It is essential that studies replicate their results, given the large potential for false positive signals. This is commonly done by repeating the study in different populations.
        </p>
        <p>
         There is a growing movement to combine GWAS results using meta-analysis. Most of the genetic associations between SNPs and complex diseases are small in magnitude, and therefore even sizable studies may fail to detect underlying associations [
         <a href="#rid58">
          58
         </a>
         ]. Meta-analysis improves the identification of replicable associations and increases precision by increasing power. This is illustrated by studies in various conditions such as Crohn disease and asthma [
         <a href="#rid59">
          59,60
         </a>
         ].
        </p>
        <p>
         The HuGE Net website lists many of the meta-analyses performed to date and also hosts the HuGE Navigator, where one can find out what single studies, genome-wide association studies, meta-analyses, and synopses are available [
         <a href="#rid61">
          61,62
         </a>
         ].
        </p>
        <p>
         Replication can also take the form of functional studies in vitro or in animal models. Demonstration by cell culture or mouse mutagenesis that genetic variants can lead to difference in protein level or function can provide powerful support for the effect of genetic variants. Follow-up studies can also provide evidence to support an initial GWAS result, including:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Candidate gene association studies that focus on a few genetic variants that are near the site of the initial GWAS locus
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Fine mapping studies, involving a denser set of SNPs around the initial GWAS locus or involving resequencing of the entire locus to pick up rare variants
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H17">
         <span class="h1">
          INTERPRETING GWAS RESULTS
         </span>
         <span class="headingEndMark">
          —
         </span>
         Investigators usually report the magnitude of a genetic association using traditional measures of association: relative risks (RRs) in cohort studies, odds ratios (ORs) in case-control studies, and hazard ratios (HRs) in survival analyses that take account of the timing of events.
        </p>
        <p>
         Understanding the magnitude of the risk depends on the genetic model involved. For dominant variant alleles (producing a protein isoform that dominates function), the presence of even one copy (ie, heterozygosity) will result in maximal increase in risk. For recessive variant alleles, both alleles must be present to result in an increase in risk (ie, heterozygotes will not show an increase in risk). In both cases, a single RR, OR, or HR describes the magnitude of the association.
        </p>
        <p>
         If the effect of a variant allele is additive, then there is a "dose-response" effect: its presence in one gene will lead to an increase in risk, while its presence in both genes will lead to a further increase. There are two possible ways to calculate this further increase: one is to take the square of the risk (variably called the log-additive, per-allele, or multiplicative risk model), and the other is to take two times the risk (called the linear additive model). Recent work in diabetes indicates that most associations seem to follow the log-additive model [
         <a href="#rid63">
          63
         </a>
         ].
        </p>
        <p>
         It is important to understand that the magnitude of the effect does not necessarily translate into a causal relationship between the single nucleotide polymorphism (SNP) and the disease. It is possible that the SNP is only a marker for another SNP nearby that is linked (termed "linkage disequilibrium") and that is the true causal variant. While linkage disequilibrium does not detract from the potential use of the SNP as a clinical marker, it limits the potential to draw pathophysiologic conclusions from the identified SNP.
        </p>
        <p>
         Much of the work related to interpreting GWAS focuses on making sense of the hits that have been found and understanding how the variants affect function. In fact, over 85 percent of the hits in GWAS of complex diseases are in non-coding regions and presumably do not affect the coding sequence; emerging evidence indicates that these non-coding regions play a role in splicing, binding to proteins, and activating or repressing enhancers and promoters, among other key functions [
         <a href="#rid64">
          64,65
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H18">
         <span class="h1">
          CLINICAL APPLICATION
         </span>
         <span class="headingEndMark">
          —
         </span>
         A framework for translating GWAS results to clinical application has been developed and is presented here [
         <a href="#rid41">
          41,42
         </a>
         ]:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Does the genetic association improve predictive power beyond easily measured clinical variables?
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         What are the absolute versus relative genetic effects?
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Is the risk-associated allele likely to be present in my patient?
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Is the patient likely better off knowing the genetic information?
        </p>
        <p>
        </p>
        <p>
         These issues need to be addressed thoughtfully and quickly, particularly given the "direct-to-consumer" availability of genetic testing. (See
         <a class="medical medical_review" href="/z/d/html/14599.html" rel="external">
          "Personalized medicine", section on 'Direct-to-consumer testing'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H19">
         <span class="h2">
          Predictive power
         </span>
         <span class="headingEndMark">
          —
         </span>
         The effect of single SNPs in complex disease to date has been small (ie, odds ratios in the 1.1-1.6 range). Therefore, there has been interest in combining the effect of many genes into a genetic "profile" for greater clinical utility; these are called polygenic risk scores [
         <a href="#rid66">
          66
         </a>
         ].
        </p>
        <p>
         As an example, in one study, investigators created a profile of 5 SNPs associated with prostate cancer [
         <a href="#rid67">
          67
         </a>
         ]; they found an OR of 1.6 for those who were homozygous or heterozygous for the risk allele at one SNP, and up to 4.5 for those who were homozygous or heterozygous for the risk allele at 4 SNPs. The greater the number of SNPs in each profile, however, the lower the number of people with that combination; hence the increased magnitude of effect is offset by the small number of people to whom that effect is relevant [
         <a href="#rid68">
          68
         </a>
         ].
        </p>
        <p>
         For dichotomous (eg, yes/no) outcomes, there are a number of statistical tools to quantitate how much predictive power the genetic information adds to existing data. One is to calculate the area under the receiver operating characteristic (ROC) curve, an approach often used for diagnostic tests [
         <a href="#rid69">
          69
         </a>
         ]. An ROC curve plots the true positive rate (sensitivity) on the y-axis against the false positive rate (1-specificity) on the x-axis  (
         <a class="graphic graphic_figure graphicRef54239" href="/z/d/graphic/54239.html" rel="external">
          figure 5
         </a>
         ). An ROC curve with no greater predictive ability than chance would approximate a straight diagonal line from the origin (0, 0) to the upper right hand corner (1.0, 1.0). The area under the curve (AUC) would be 0.5. The visual representation of a perfectly predictive test would be a line that goes straight up the y-axis to 1.0 and then straight across the x-axis to 1.0 and would have an AUC of 1.
        </p>
        <p>
         Success in creating risk scores with a handful of SNPs has led some to try creating risk scores with tens or hundreds of thousands of SNPs, hoping to increase predictive power. However, predictive power levels off very quickly. As an example, one study looked at polygenic risk scores for five common complex diseases and found that increasing the number of SNPs in the risk score from tens to millions only increased the predictive power negligibly (increase in AUC of 0.01) and only explained 2 to 4 percent of the variance in disease risk [
         <a href="#rid70">
          70
         </a>
         ].
        </p>
        <p>
         There are a number of reasons why the addition of SNP information may not increase the predictive power of these models substantially:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The effect of genetic polymorphisms present from birth may genuinely be dwarfed by environmental factors over a lifetime, especially for late-onset diseases.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         SNP arrays are based on the most common polymorphisms (&gt;1 percent) and lose the ability to tag variants that are increasingly rare (&lt;0.1 percent). It appears that in some cases the sheer number of rare variants can contribute to heritability despite their individual rarity [
         <a href="#rid25">
          25
         </a>
         ].
        </p>
        <p>
        </p>
        <p>
         Some of heritability appears to be carried by copy number variants (CNVs), and CNVs are not routinely included in polygenic risk scores [
         <a href="#rid71">
          71
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/16647.html" rel="external">
          "Genomic disorders: An overview", section on 'Copy number variations'
         </a>
         .)
        </p>
        <p>
         There are rare examples, particularly in the area of pharmacogenomics, where genetic associations may have clinical applications because of their large magnitude. As an example, a SNP in the thiopurine methyltransferase (TPMT) gene identifies individuals who are at increased risk of life-threatening hematologic toxicity from the chemotherapeutic agent
         <a class="drug drug_general" data-topicid="9612" href="/z/d/drug information/9612.html" rel="external">
          6-mercaptopurine
         </a>
         [
         <a href="#rid72">
          72
         </a>
         ]. Genotyping this SNP can avoid substantial harm by additional monitoring or substituting an alternative chemotherapeutic agent in those with the high-risk genotype. (See
         <a class="medical medical_review" href="/z/d/html/2904.html" rel="external">
          "Overview of pharmacogenomics"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4058.html" rel="external">
          "Thiopurines: Pretreatment testing and approach to therapeutic drug monitoring for adults with inflammatory bowel disease"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/6245.html" rel="external">
          "Treatment of acute lymphoblastic leukemia/lymphoma in children and adolescents"
         </a>
         .)
        </p>
        <p>
         Another pharmacogenomics example is the use of CYP2C19 genetic testing for guiding the choice of P2Y12 inhibitors after primary percutaneous coronary intervention. In a trial using genotype-guided selection of antiplatelet agent in which individuals with loss-of-function alleles received
         <a class="drug drug_general" data-topicid="16784" href="/z/d/drug information/16784.html" rel="external">
          ticagrelor
         </a>
         or
         <a class="drug drug_general" data-topicid="9533" href="/z/d/drug information/9533.html" rel="external">
          prasugrel
         </a>
         and those without received
         <a class="drug drug_general" data-topicid="8921" href="/z/d/drug information/8921.html" rel="external">
          clopidogrel
         </a>
         , bleeding rates were significantly lower in the genotype-guided group compared with non-genotype-guided controls, with equivalent thrombotic event rates [
         <a href="#rid73">
          73
         </a>
         ].
        </p>
        <p>
         The most likely short-term clinical application of a genetic association is to provide prognostic information in the context of other known predictors. To be of value, the genetic marker must provide independent predictive power beyond traditional clinical predictive variables (eg, age, sex, smoking status) and beyond easily obtained surrogate measures of familial aggregation, such as family history. This is often not the case, particularly if the genetic polymorphism exerts its effect through a readily measured variable (eg, a gene controlling lipids exerts its effect through increases in LDL).
        </p>
        <p>
         As another example, the addition of genetic data to a model for prediction of type 2 diabetes did not improve the model fit and resulted in appropriate reclassification of, at most, 4 percent of people [
         <a href="#rid74">
          74
         </a>
         ].
        </p>
        <p>
         However, a high AUC or even an increment in AUC with the addition of genetic variables does not guarantee that the genetic information will be clinically useful. Points along the ROC curve may still correspond to sensitivities and specificities that are too low to be used for screening or prognosis. Furthermore, even if the sensitivity and specificity are high, the usefulness of the genetic variant for classification of risk may be limited due to low allele frequency (ie, only a small proportion of the risk group will be detected or stratified correctly) [
         <a href="#rid75">
          75
         </a>
         ].
        </p>
        <p>
         In summary:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         A very small p-value does not necessarily translate into a useful clinical marker because effect size (RR or OR) may be small.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         A high effect size does not necessarily translate into a clinical useful marker because the AUC may be low.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         A high AUC does not necessarily translate into a clinical useful marker because the combined sensitivity and specificity may be insufficiently high for screening or prognostication.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         High sensitivity and specificity may not necessarily translate into a clinical useful marker because the allele frequency may be low.
        </p>
        <p>
        </p>
        <p>
         These considerations show the difficulty in translating a statistically significant genetic predictor into a clinical useful and accurate clinical classifier.
        </p>
        <p class="headingAnchor" id="H20">
         <span class="h2">
          Absolute versus relative effects
         </span>
         <span class="headingEndMark">
          —
         </span>
         If the patient's risk of disease is low in the absence of a variant allele, even a 5- or 10-fold increase in risk in the presence of the allele may represent a small absolute increase in risk. Conversely, if the baseline risk is high, a modest increase in relative risk could impact clinical decision making.
        </p>
        <p>
         As an example, the Factor V Leiden mutation increases the risk of venous thrombosis by approximately sixfold [
         <a href="#rid76">
          76
         </a>
         ]. However, the baseline risk of thrombosis in the general population is sufficiently low (approximately 0.2 percent) that one would not use genotyping as a population screening test [
         <a href="#rid77">
          77
         </a>
         ]. However, the prevalence of Factor V Leiden in patients with venous thrombosis is 12 to 20 percent, and thus testing some patients with established venous thrombosis may be appropriate [
         <a href="#rid78">
          78,79
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/1363.html" rel="external">
          "Evaluating adult patients with established venous thromboembolism for acquired and inherited risk factors", section on 'Evaluation for hypercoagulable disorders'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H21">
         <span class="h2">
          Allele frequency in the relevant population
         </span>
         <span class="headingEndMark">
          —
         </span>
         In applying the results, clinicians must consider the likelihood that the particular allele is present in a particular patient. As an example, while factor V Leiden is relatively common in certain White populations (approximately 1 in 20 White individuals are heterozygous), it is virtually non-existent in populations from China. Hence, genotyping for factor V Leiden is unnecessary in an individual from China presenting with unprovoked deep vein thrombosis. (See
         <a class="medical medical_review" href="/z/d/html/1355.html" rel="external">
          "Factor V Leiden and activated protein C resistance", section on 'FVL genotypes'
         </a>
         .)
        </p>
        <p>
         Allele frequencies for various genes and populations of interest are available in the
         <strong>
          Al
         </strong>
         lele
         <strong>
          fre
         </strong>
         quency
         <strong>
          d
         </strong>
         atabase (ALFRED) or at the HapMap website [
         <a href="#rid80">
          80,81
         </a>
         ]. Some gene-disease associations may be restricted to a very select subgroup. As an example, variants in the
         <em>
          BRCA1
         </em>
         gene were identified in patients with early-onset breast cancer who had a strong family history [
         <a href="#rid82">
          82
         </a>
         ]. This group, however, only accounts for approximately 5 percent of all breast cancers. Hence, this genetic association is not worth testing for in those who present with typical late-onset breast cancer without a strong family history. However, in certain ancestry groups such as Ashkenazi Jews, who have a high prevalence of
         <em>
          BRCA1
         </em>
         mutations, testing may be appropriate in women with breast or ovarian cancer. (See
         <a class="medical medical_review" href="/z/d/html/753.html" rel="external">
          "Genetic testing and management of individuals at risk of hereditary breast and ovarian cancer syndromes"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H22">
         <span class="h2">
          Patient impact
         </span>
         <span class="headingEndMark">
          —
         </span>
         Even if the magnitude of the genetic effect is not sufficiently high to be clinically useful in prognostication, it may still be useful clinically in terms of changing risk behavior [
         <a href="#rid83">
          83
         </a>
         ]. Presenting personal genetic information may take advantage of the layperson's perception of DNA as their "life code" to prompt behavior change.
        </p>
        <p>
         As an example, early evidence suggests that providing information about glutathione-S-transferase (GST) genotypes, which affect nicotine metabolism, may influence smoking cessation rates [
         <a href="#rid84">
          84
         </a>
         ]. This potential benefit should be balanced against the potential worry of knowing that one's risk of health problems years in the future is increased, and the potential for increased insurance premiums or life insurance/disability denial.
        </p>
        <p class="headingAnchor" id="H2784355">
         <span class="h1">
          IMPLICATIONS FOR RESEARCH
         </span>
         <span class="headingEndMark">
          —
         </span>
         Apart from potential clinical uses, GWAS results can also shed light on new pathophysiological mechanisms. Principles have been proposed to move beyond identifying single nucleotide polymorphisms (SNPs) to defining their possible functional significance, which might include targeted resequencing, studying the SNP in other populations to further define and narrow linkage disequilibrium, exploring gene transcription functions and epigenetic regulation, and using model systems and cell models to further evaluate the proposed causal variants [
         <a href="#rid3">
          3
         </a>
         ]. The challenge of dissecting the correlation between genotype and phenotype will require rigorous evaluation.
        </p>
        <p class="headingAnchor" id="H23">
         <span class="h1">
          SUMMARY
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Definitions
         </strong>
         – Genetic association studies seek to identify the genetic component to risk of non-Mendelian complex disorders. The basic assumption regarding genetic determinants of complex disease is that the overall risk of disease is determined by the combination of multiple common genetic variants, each with small effect, and environmental exposures. The candidate gene approach selects a genetic variant of potential interest and looks for this gene in cases and controls. The genome-wide approach (genome-wide association studies [GWAS]) looks for an association between millions of variants and the condition of interest  (
         <a class="graphic graphic_figure graphicRef54220" href="/z/d/graphic/54220.html" rel="external">
          figure 1
         </a>
         ). (See
         <a class="local">
          'Terminology and study design'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Explaining the variance in complex traits
         </strong>
         – GWAS have identified some significant single nucleotide polymorphisms (SNPs) associated with disease, but there is still debate about the amount of variance explained by these, (ie, whether there is "missing heritability"). (See
         <a class="local">
          'Genetic determinants of complex traits'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Evaluating GWAS validity
         </strong>
         – GWAS should be evaluated for validity, considering potential errors related to phenotyping, genotyping, ancestral differences between cases and controls, and multiple comparators. Testing allele distributions for Hardy-Weinberg equilibrium and looking for consistency between studies are other important considerations. (See
         <a class="local">
          'Judging study validity'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Clinical implications
         </strong>
         – Understanding the magnitude of the risk from any given gene variant depends on the genetic model involved. Increasingly, genetic variants are being combined in polygenic risk scores to help predict risk of disease, prognosis, or likelihood of response; these risk scores can incorporate hundreds to millions of SNPs. Questions in applying results of genetic association studies to clinical practice include the effect on predictive power, the magnitude of absolute versus relative effects, generalizability to the given patient, and whether the genetic information will be beneficial to the patient. (See
         <a class="local">
          'Interpreting GWAS results'
         </a>
         above and
         <a class="local">
          'Clinical application'
         </a>
         above.)
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li>
          <a class="nounderline abstract_t">
           Dawn Teare M, Barrett JH. Genetic linkage studies. Lancet 2005; 366:1036.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Visscher PM, Montgomery GW. Genome-wide association studies and human disease: from trickle to flood. JAMA 2009; 302:2028.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Freedman ML, Monteiro AN, Gayther SA, et al. Principles for the post-GWAS functional characterization of cancer risk loci. Nat Genet 2011; 43:513.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           MacArthur J, Bowler E, Cerezo M, et al. The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog). Nucleic Acids Res 2017; 45:D896.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Thakkinstian A, Han P, McEvoy M, et al. Systematic review and meta-analysis of the association between complement factor H Y402H polymorphisms and age-related macular degeneration. Hum Mol Genet 2006; 15:2784.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Iyengar SK, Elston RC. The genetic basis of complex traits: rare variants or "common gene, common disease"? Methods Mol Biol 2007; 376:71.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Estivill X, Bancells C, Ramos C. Geographic distribution and regional origin of 272 cystic fibrosis mutations in European populations. The Biomed CF Mutation Analysis Consortium. Hum Mutat 1997; 10:135.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Freund MK, Burch KS, Shi H, et al. Phenotype-Specific Enrichment of Mendelian Disorder Genes near GWAS Regions across 62 Complex Traits. Am J Hum Genet 2018; 103:535.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Samani NJ, Erdmann J, Hall AS, et al. Genomewide association analysis of coronary artery disease. N Engl J Med 2007; 357:443.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zeggini E, Weedon MN, Lindgren CM, et al. Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. Science 2007; 316:1336.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Matarín M, Brown WM, Scholz S, et al. A genome-wide genotyping study in patients with ischaemic stroke: initial analysis and data release. Lancet Neurol 2007; 6:414.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           International Multiple Sclerosis Genetics Consortium, Hafler DA, Compston A, et al. Risk alleles for multiple sclerosis identified by a genomewide study. N Engl J Med 2007; 357:851.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Easton DF, Pooley KA, Dunning AM, et al. Genome-wide association study identifies novel breast cancer susceptibility loci. Nature 2007; 447:1087.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. Nature 2007; 447:661.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Plenge RM, Seielstad M, Padyukov L, et al. TRAF1-C5 as a risk locus for rheumatoid arthritis--a genomewide study. N Engl J Med 2007; 357:1199.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Coon KD, Myers AJ, Craig DW, et al. A high-density whole-genome association study reveals that APOE is the major susceptibility gene for sporadic late-onset Alzheimer's disease. J Clin Psychiatry 2007; 68:613.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Li H, Wetten S, Li L, et al. Candidate single-nucleotide polymorphisms from a genomewide association study of Alzheimer disease. Arch Neurol 2008; 65:45.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Prokopenko I, Langenberg C, Florez JC, et al. Variants in MTNR1B influence fasting glucose levels. Nat Genet 2009; 41:77.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Bouatia-Naji N, Bonnefond A, Cavalcanti-Proença C, et al. A variant near MTNR1B is associated with increased fasting plasma glucose levels and type 2 diabetes risk. Nat Genet 2009; 41:89.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Weedon MN, Lango H, Lindgren CM, et al. Genome-wide association analysis identifies 20 loci that influence adult height. Nat Genet 2008; 40:575.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lettre G, Jackson AU, Gieger C, et al. Identification of ten loci associated with height highlights new biological pathways in human growth. Nat Genet 2008; 40:584.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Maher B. Personal genomes: The case of the missing heritability. Nature 2008; 456:18.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hunt KA, Mistry V, Bockett NA, et al. Negligible impact of rare autoimmune-locus coding-region variants on missing heritability. Nature 2013; 498:232.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Turkmen A, Lin S. Are rare variants really independent? Genet Epidemiol 2017; 41:363.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wainschtein P, Jain D, Zheng Z, et al. Assessing the contribution of rare variants to complex trait heritability from whole-genome sequence data. Nat Genet 2022; 54:263.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wain LV, Armour JA, Tobin MD. Genomic copy number variation, human health, and disease. Lancet 2009; 374:340.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Weiss LA, Shen Y, Korn JM, et al. Association between microdeletion and microduplication at 16p11.2 and autism. N Engl J Med 2008; 358:667.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Conrad DF, Pinto D, Redon R, et al. Origins and functional impact of copy number variation in the human genome. Nature 2010; 464:704.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wellcome Trust Case Control Consortium, Craddock N, Hurles ME, et al. Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls. Nature 2010; 464:713.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zuk O, Hechter E, Sunyaev SR, Lander ES. The mystery of missing heritability: Genetic interactions create phantom heritability. Proc Natl Acad Sci U S A 2012; 109:1193.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Skinner MK, Manikkam M, Guerrero-Bosagna C. Epigenetic transgenerational actions of environmental factors in disease etiology. Trends Endocrinol Metab 2010; 21:214.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Chen Q, Yan W, Duan E. Epigenetic inheritance of acquired traits through sperm RNAs and sperm RNA modifications. Nat Rev Genet 2016; 17:733.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Garg P, Jadhav B, Rodriguez OL, et al. A Survey of Rare Epigenetic Variation in 23,116 Human Genomes Identifies Disease-Relevant Epivariations and CGG Expansions. Am J Hum Genet 2020; 107:654.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Visscher PM, Hill WG, Wray NR. Heritability in the genomics era--concepts and misconceptions. Nat Rev Genet 2008; 9:255.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Selzam S, Ritchie SJ, Pingault JB, et al. Comparing Within- and Between-Family Polygenic Score Prediction. Am J Hum Genet 2019; 105:351.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yang J, Benyamin B, McEvoy BP, et al. Common SNPs explain a large proportion of the heritability for human height. Nat Genet 2010; 42:565.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yang J, Manolio TA, Pasquale LR, et al. Genome partitioning of genetic variation for complex traits using common SNPs. Nat Genet 2011; 43:519.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zhang Y, Qi G, Park JH, Chatterjee N. Estimation of complex effect-size distributions using summary-level statistics from genome-wide association studies across 32 complex traits. Nat Genet 2018; 50:1318.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Little J, Bradley L, Bray MS, et al. Reporting, appraising, and integrating data on genotype prevalence and gene-disease associations. Am J Epidemiol 2002; 156:300.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Little J, Higgins JP, Ioannidis JP, et al. STrengthening the REporting of Genetic Association studies (STREGA): an extension of the STROBE Statement. Ann Intern Med 2009; 150:206.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Attia J, Ioannidis JP, Thakkinstian A, et al. How to use an article about genetic association: B: Are the results of the study valid? JAMA 2009; 301:191.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Attia J, Ioannidis JP, Thakkinstian A, et al. How to use an article about genetic association: C: What are the results and will they help me in caring for my patients? JAMA 2009; 301:304.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Contopoulos-Ioannidis DG, Alexiou GA, Gouvias TC, Ioannidis JP. An empirical evaluation of multifarious outcomes in pharmacogenetics: beta-2 adrenoceptor gene polymorphisms in asthma treatment. Pharmacogenet Genomics 2006; 16:705.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Berkovic SF, Scheffer IE. Genetics of the epilepsies. Epilepsia 2001; 42 Suppl 5:16.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Barnholtz-Sloan JS, McEvoy B, Shriver MD, Rebbeck TR. Ancestry estimation and correction for population stratification in molecular epidemiologic association studies. Cancer Epidemiol Biomarkers Prev 2008; 17:471.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Pritchard JK, Rosenberg NA. Use of unlinked genetic markers to detect population stratification in association studies. Am J Hum Genet 1999; 65:220.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kittles RA, Chen W, Panguluri RK, et al. CYP3A4-V and prostate cancer in African Americans: causal or confounding association because of population stratification? Hum Genet 2002; 110:553.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Frayling TM, Timpson NJ, Weedon MN, et al. A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity. Science 2007; 316:889.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Leek JT, Scharpf RB, Bravo HC, et al. Tackling the widespread and critical impact of batch effects in high-throughput data. Nat Rev Genet 2010; 11:733.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Clayton DG, Walker NM, Smyth DJ, et al. Population structure, differential bias and genomic control in a large-scale, case-control association study. Nat Genet 2005; 37:1243.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Akey JM, Zhang K, Xiong M, et al. The effect that genotyping errors have on the robustness of common linkage-disequilibrium measures. Am J Hum Genet 2001; 68:1447.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Pompanon F, Bonin A, Bellemain E, Taberlet P. Genotyping errors: causes, consequences and solutions. Nat Rev Genet 2005; 6:847.
          </a>
         </li>
         <li class="breakAll">
          www.tufts.edu/~mcourt01/Documents/Court%20lab%20-%20HW%20calculator.xls (Accessed on March 05, 2014).
         </li>
         <li class="breakAll">
          www.oege.org/software/hwe-mr-calc.shtml (Accessed on March 05, 2014).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cox DG, Kraft P. Quantification of the power of Hardy-Weinberg equilibrium testing to detect genotyping error. Hum Hered 2006; 61:10.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           McCarthy MI, Abecasis GR, Cardon LR, et al. Genome-wide association studies for complex traits: consensus, uncertainty and challenges. Nat Rev Genet 2008; 9:356.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sham PC, Purcell SM. Statistical power and significance testing in large-scale genetic studies. Nat Rev Genet 2014; 15:335.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moonesinghe R, Khoury MJ, Liu T, Ioannidis JP. Required sample size and nonreplicability thresholds for heterogeneous genetic associations. Proc Natl Acad Sci U S A 2008; 105:617.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Franke A, McGovern DP, Barrett JC, et al. Genome-wide meta-analysis increases to 71 the number of confirmed Crohn's disease susceptibility loci. Nat Genet 2010; 42:1118.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moffatt MF, Gut IG, Demenais F, et al. A large-scale, consortium-based genomewide association study of asthma. N Engl J Med 2010; 363:1211.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yu W, Wulf A, Yesupriya A, et al. HuGE Watch: tracking trends and patterns of published studies of genetic association and human genome epidemiology in near-real time. Eur J Hum Genet 2008; 16:1155.
          </a>
         </li>
         <li class="breakAll">
          www.hugenavigator.net (Accessed on August 04, 2015).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Salanti G, Southam L, Altshuler D, et al. Underlying genetic models of inheritance in established type 2 diabetes associations. Am J Epidemiol 2009; 170:537.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gallagher MD, Chen-Plotkin AS. The Post-GWAS Era: From Association to Function. Am J Hum Genet 2018; 102:717.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cannon ME, Mohlke KL. Deciphering the Emerging Complexities of Molecular Mechanisms at GWAS Loci. Am J Hum Genet 2018; 103:637.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Torkamani A, Wineinger NE, Topol EJ. The personal and clinical utility of polygenic risk scores. Nat Rev Genet 2018; 19:581.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zheng SL, Sun J, Wiklund F, et al. Cumulative association of five genetic variants with prostate cancer. N Engl J Med 2008; 358:910.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Janssens AC, van Duijn CM. Genome-based prediction of common diseases: advances and prospects. Hum Mol Genet 2008; 17:R166.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Irwig L, Bossuyt P, Glasziou P, et al. Designing studies to ensure that estimates of test accuracy are transferable. BMJ 2002; 324:669.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Khera AV, Chaffin M, Aragam KG, et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. Nat Genet 2018; 50:1219.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Li YR, Glessner JT, Coe BP, et al. Rare copy number variants in over 100,000 European ancestry subjects reveal multiple disease associations. Nat Commun 2020; 11:255.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           McLeod HL, Krynetski EY, Relling MV, Evans WE. Genetic polymorphism of thiopurine methyltransferase and its clinical relevance for childhood acute lymphoblastic leukemia. Leukemia 2000; 14:567.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Claassens DMF, Vos GJA, Bergmeijer TO, et al. A Genotype-Guided Strategy for Oral P2Y12 Inhibitors in Primary PCI. N Engl J Med 2019; 381:1621.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Meigs JB, Shrader P, Sullivan LM, et al. Genotype score in addition to common risk factors for prediction of type 2 diabetes. N Engl J Med 2008; 359:2208.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Jakobsdottir J, Gorin MB, Conley YP, et al. Interpretation of genetic association studies: markers with replicated highly significant odds ratios may be poor classifiers. PLoS Genet 2009; 5:e1000337.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Emmerich J, Rosendaal FR, Cattaneo M, et al. Combined effect of factor V Leiden and prothrombin 20210A on the risk of venous thromboembolism--pooled analysis of 8 case-control studies including 2310 cases and 3204 controls. Study Group for Pooled-Analysis in Venous Thromboembolism. Thromb Haemost 2001; 86:809.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cushman M, Tsai AW, White RH, et al. Deep vein thrombosis and pulmonary embolism in two cohorts: the longitudinal investigation of thromboembolism etiology. Am J Med 2004; 117:19.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ridker PM, Hennekens CH, Lindpaintner K, et al. Mutation in the gene coding for coagulation factor V and the risk of myocardial infarction, stroke, and venous thrombosis in apparently healthy men. N Engl J Med 1995; 332:912.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Koster T, Rosendaal FR, de Ronde H, et al. Venous thrombosis due to poor anticoagulant response to activated protein C: Leiden Thrombophilia Study. Lancet 1993; 342:1503.
          </a>
         </li>
         <li class="breakAll">
          The ALlele FREquency Database. Available at: http://alfred.med.yale.edu/alfred/index.asp (Accessed on March 05, 2014).
         </li>
         <li class="breakAll">
          International HapMap Project. Available at: www.hapmap.org (Accessed on March 05, 2014).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Miki Y, Swensen J, Shattuck-Eidens D, et al. A strong candidate for the breast and ovarian cancer susceptibility gene BRCA1. Science 1994; 266:66.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Marteau TM, Lerman C. Genetic risk and behavioural change. BMJ 2001; 322:1056.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hamajima N, Suzuki K, Ito Y, Kondo T. Genotype announcement to Japanese smokers who attended a health checkup examination. J Epidemiol 2006; 16:45.
          </a>
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 2902 Version 30.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16168786" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : Genetic linkage studies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19903925" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : Genome-wide association studies and human disease: from trickle to flood.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21614091" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : Principles for the post-GWAS functional characterization of cancer risk loci.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27899670" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog).
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16905558" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : Systematic review and meta-analysis of the association between complement factor H Y402H polymorphisms and age-related macular degeneration.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17984539" id="rid5" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          6 : The genetic basis of complex traits: rare variants or "common gene, common disease"?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9259197" id="rid6" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          7 : Geographic distribution and regional origin of 272 cystic fibrosis mutations in European populations. The Biomed CF Mutation Analysis Consortium.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30290150" id="rid7" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          8 : Phenotype-Specific Enrichment of Mendelian Disorder Genes near GWAS Regions across 62 Complex Traits.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17634449" id="rid8" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          9 : Genomewide association analysis of coronary artery disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17463249" id="rid9" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          10 : Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17434096" id="rid10" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          11 : A genome-wide genotyping study in patients with ischaemic stroke: initial analysis and data release.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17660530" id="rid11" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          12 : Risk alleles for multiple sclerosis identified by a genomewide study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17529967" id="rid12" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          13 : Genome-wide association study identifies novel breast cancer susceptibility loci.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17554300" id="rid13" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          14 : Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17804836" id="rid14" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          15 : TRAF1-C5 as a risk locus for rheumatoid arthritis--a genomewide study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17474819" id="rid15" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          16 : A high-density whole-genome association study reveals that APOE is the major susceptibility gene for sporadic late-onset Alzheimer's disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17998437" id="rid16" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          17 : Candidate single-nucleotide polymorphisms from a genomewide association study of Alzheimer disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19060907" id="rid17" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          18 : Variants in MTNR1B influence fasting glucose levels.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19060909" id="rid18" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          19 : A variant near MTNR1B is associated with increased fasting plasma glucose levels and type 2 diabetes risk.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18391952" id="rid19" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          20 : Genome-wide association analysis identifies 20 loci that influence adult height.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18391950" id="rid20" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          21 : Identification of ten loci associated with height highlights new biological pathways in human growth.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18987709" id="rid21" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          22 : Personal genomes: The case of the missing heritability.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23698362" id="rid22" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          23 : Negligible impact of rare autoimmune-locus coding-region variants on missing heritability.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28300291" id="rid23" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          24 : Are rare variants really independent?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/35256806" id="rid24" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          25 : Assessing the contribution of rare variants to complex trait heritability from whole-genome sequence data.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19535135" id="rid25" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          26 : Genomic copy number variation, human health, and disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18184952" id="rid26" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          27 : Association between microdeletion and microduplication at 16p11.2 and autism.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19812545" id="rid27" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          28 : Origins and functional impact of copy number variation in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20360734" id="rid28" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          29 : Genome-wide association study of CNVs in 16,000 cases of eight common diseases and 3,000 shared controls.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22223662" id="rid29" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          30 : The mystery of missing heritability: Genetic interactions create phantom heritability.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20074974" id="rid30" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          31 : Epigenetic transgenerational actions of environmental factors in disease etiology.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27694809" id="rid31" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          32 : Epigenetic inheritance of acquired traits through sperm RNAs and sperm RNA modifications.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/32937144" id="rid32" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          33 : A Survey of Rare Epigenetic Variation in 23,116 Human Genomes Identifies Disease-Relevant Epivariations and CGG Expansions.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18319743" id="rid33" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          34 : Heritability in the genomics era--concepts and misconceptions.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31303263" id="rid34" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          35 : Comparing Within- and Between-Family Polygenic Score Prediction.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20562875" id="rid35" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          36 : Common SNPs explain a large proportion of the heritability for human height.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21552263" id="rid36" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          37 : Genome partitioning of genetic variation for complex traits using common SNPs.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30104760" id="rid37" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          38 : Estimation of complex effect-size distributions using summary-level statistics from genome-wide association studies across 32 complex traits.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12181099" id="rid38" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          39 : Reporting, appraising, and integrating data on genotype prevalence and gene-disease associations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19189911" id="rid39" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          40 : STrengthening the REporting of Genetic Association studies (STREGA): an extension of the STROBE Statement.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19141767" id="rid40" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          41 : How to use an article about genetic association: B: Are the results of the study valid?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19155457" id="rid41" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          42 : How to use an article about genetic association: C: What are the results and will they help me in caring for my patients?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17001289" id="rid42" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          43 : An empirical evaluation of multifarious outcomes in pharmacogenetics: beta-2 adrenoceptor gene polymorphisms in asthma treatment.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11887962" id="rid43" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          44 : Genetics of the epilepsies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18349264" id="rid44" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          45 : Ancestry estimation and correction for population stratification in molecular epidemiologic association studies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10364535" id="rid45" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          46 : Use of unlinked genetic markers to detect population stratification in association studies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12107441" id="rid46" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          47 : CYP3A4-V and prostate cancer in African Americans: causal or confounding association because of population stratification?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17434869" id="rid47" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          48 : A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20838408" id="rid48" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          49 : Tackling the widespread and critical impact of batch effects in high-throughput data.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16228001" id="rid49" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          50 : Population structure, differential bias and genomic control in a large-scale, case-control association study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11359212" id="rid50" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          51 : The effect that genotyping errors have on the robustness of common linkage-disequilibrium measures.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16304600" id="rid51" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          52 : Genotyping errors: causes, consequences and solutions.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16304600" id="rid52" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          53 : Genotyping errors: causes, consequences and solutions.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16304600" id="rid53" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          54 : Genotyping errors: causes, consequences and solutions.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16514241" id="rid54" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          55 : Quantification of the power of Hardy-Weinberg equilibrium testing to detect genotyping error.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18398418" id="rid55" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          56 : Genome-wide association studies for complex traits: consensus, uncertainty and challenges.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24739678" id="rid56" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          57 : Statistical power and significance testing in large-scale genetic studies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18174335" id="rid57" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          58 : Required sample size and nonreplicability thresholds for heterogeneous genetic associations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21102463" id="rid58" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          59 : Genome-wide meta-analysis increases to 71 the number of confirmed Crohn's disease susceptibility loci.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20860503" id="rid59" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          60 : A large-scale, consortium-based genomewide association study of asthma.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18478035" id="rid60" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          61 : HuGE Watch: tracking trends and patterns of published studies of genetic association and human genome epidemiology in near-real time.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18478035" id="rid61" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          62 : HuGE Watch: tracking trends and patterns of published studies of genetic association and human genome epidemiology in near-real time.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19602701" id="rid62" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          63 : Underlying genetic models of inheritance in established type 2 diabetes associations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29727686" id="rid63" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          64 : The Post-GWAS Era: From Association to Function.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30388398" id="rid64" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          65 : Deciphering the Emerging Complexities of Molecular Mechanisms at GWAS Loci.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29789686" id="rid65" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          66 : The personal and clinical utility of polygenic risk scores.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18199855" id="rid66" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          67 : Cumulative association of five genetic variants with prostate cancer.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18852206" id="rid67" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          68 : Genome-based prediction of common diseases: advances and prospects.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11895830" id="rid68" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          69 : Designing studies to ensure that estimates of test accuracy are transferable.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30104762" id="rid69" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          70 : Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31937769" id="rid70" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          71 : Rare copy number variants in over 100,000 European ancestry subjects reveal multiple disease associations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10764140" id="rid71" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          72 : Genetic polymorphism of thiopurine methyltransferase and its clinical relevance for childhood acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31479209" id="rid72" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          73 : A Genotype-Guided Strategy for Oral P2Y12 Inhibitors in Primary PCI.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19020323" id="rid73" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          74 : Genotype score in addition to common risk factors for prediction of type 2 diabetes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19197355" id="rid74" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          75 : Interpretation of genetic association studies: markers with replicated highly significant odds ratios may be poor classifiers.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11583312" id="rid75" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          76 : Combined effect of factor V Leiden and prothrombin 20210A on the risk of venous thromboembolism--pooled analysis of 8 case-control studies including 2310 cases and 3204 controls. Study Group for Pooled-Analysis in Venous Thromboembolism.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15210384" id="rid76" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          77 : Deep vein thrombosis and pulmonary embolism in two cohorts: the longitudinal investigation of thromboembolism etiology.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7877648" id="rid77" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          78 : Mutation in the gene coding for coagulation factor V and the risk of myocardial infarction, stroke, and venous thrombosis in apparently healthy men.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7902898" id="rid78" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          79 : Venous thrombosis due to poor anticoagulant response to activated protein C: Leiden Thrombophilia Study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7902898" id="rid79" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          80 : Venous thrombosis due to poor anticoagulant response to activated protein C: Leiden Thrombophilia Study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7902898" id="rid80" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          81 : Venous thrombosis due to poor anticoagulant response to activated protein C: Leiden Thrombophilia Study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7545954" id="rid81" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          82 : A strong candidate for the breast and ovarian cancer susceptibility gene BRCA1.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11325776" id="rid82" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          83 : Genetic risk and behavioural change.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16369108" id="rid83" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          84 : Genotype announcement to Japanese smokers who attended a health checkup examination.
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
