<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Tools for genetics and genomics: Model systems
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Tools for genetics and genomics: Model systems
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Tools for genetics and genomics: Model systems
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Author:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Robert D Blank, MD, PhD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Benjamin A Raby, MD, MPH
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Jennifer S Tirnauer, MD
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            Jan 31, 2022.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H1">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         Even a complete knowledge of the entire three billion base pairs that make up the human genome (as provided by the Human Genome Project) is insufficient to understand the genetic basis of disease [
         <a href="#rid1">
          1
         </a>
         ]. The sequence data are far more informative if they can be correlated with functional information. This correlation is often accomplished through the use of model systems.
        </p>
        <p>
         This topic summarizes some of the most useful online sources of genomic information and analysis, and presents selected examples of the use of model systems to explore the relationship between genetic constitution and function. (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2">
         <span class="h1">
          BASIC CONCEPTS
         </span>
         <span class="headingEndMark">
          —
         </span>
         The genome is the full complement of genetic information encoded on a complete, haploid set of chromosomes. The human DNA sequence, as defined by the human genome project, represents a composite compiled by studying DNA obtained from many individuals. While these data include considerable information regarding sequence variation among individuals, inter-individual variation remains incompletely cataloged.
        </p>
        <p>
         Efforts to define variation between individuals and to relate such variation to disease risk remain central tasks for human geneticists. (See
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression", section on 'Genetic variation'
         </a>
         .)
        </p>
        <p>
         Much of this investigation is focused upon single nucleotide polymorphisms (SNPs), sequence variations which occur roughly once in several hundred bases [
         <a href="#rid2">
          2
         </a>
         ].
        </p>
        <p>
         The high density of SNPs and development of methods to efficiently screen for them allow use of SNPs to identify functional genes. Such genes contribute to a wide array of traits, including susceptibility to polygenic diseases and response to pharmaceutical agents [
         <a href="#rid3">
          3-5
         </a>
         ].
        </p>
        <p>
         Two complementary approaches are being pursued to accomplish the goal of relating gene composition to function, or genotype to phenotype:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Epidemiologic studies seek to establish associations between genes and traits.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Functional studies seek to elucidate the mechanisms by which sequence variation leads to phenotypic differences.
        </p>
        <p>
        </p>
        <p>
         Structural and functional conservation of the genome over the course of evolution makes genomics a fruitful approach to studying human physiology and pathology. Genes and their protein products are highly conserved. As an example, approximately 30 percent of human genes have an equivalent in yeast (ie, a homolog). The percentage of homologous genes increases as the evolutionary distance between humans and model organisms decreases. The genome sequence of man's closest relative, the chimpanzee, shares the highest degree of sequence conservation with the human genome, with only approximately 1.2 percent divergence in single nucleotide polymorphism.
        </p>
        <p>
         Another type of evolutionary relationship is evident in the organization of the genome: groups of genes physically clustered together on one chromosome in one organism are more often clustered together in other divergent species (so-called syntenic conservation) [
         <a href="#rid6">
          6-9
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H3">
         <span class="h1">
          GENOME-BASED EXPERIMENTAL DESIGNS
         </span>
        </p>
        <p class="headingAnchor" id="H4">
         <span class="h2">
          Genome-wide association studies (GWAS)
         </span>
         <span class="headingEndMark">
          —
         </span>
         A genome-wide association study (GWAS) is a gene mapping study that assesses evidence of association between genetic variants and heritable traits across the genome. The performance of GWAS is feasible because of the availability of dense SNP coverage across the human genome and genotyping platforms that allow one million genotypes to be assessed in a single experiment. Typical studies consist of genotyping hundreds of thousands of common SNPs, using DNA microarrays in large case-control populations, with the goal of identifying specific risk alleles that are more prevalent in cases than in controls. (See
         <a class="medical medical_review" href="/z/d/html/2902.html" rel="external">
          "Genetic association and GWAS studies: Principles and applications"
         </a>
         .)
        </p>
        <p>
         GWAS investigations of at least 50 traits or diseases have been reported. While these studies have succeeded in identifying variants in linkage disequilibrium with clinical disorders, the contributions of the allele-specific effects have been only modest [
         <a href="#rid10">
          10
         </a>
         ]. As an example, one study of stature identified three alleles affecting height, but the most powerful of these had an estimated effect size of less than half a centimeter [
         <a href="#rid11">
          11
         </a>
         ].
        </p>
        <p>
         Several databases of GWAS investigations are available. (See
         <a class="local">
          'Online sources of genetic information'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H4285893171">
         <span class="h2">
          Mendelian randomization
         </span>
         <span class="headingEndMark">
          —
         </span>
         GWAS has allowed identification of many SNPs associated with continuously varying traits, in the best cases accounting for a substantial fraction of the genetic trait variance.
        </p>
        <p>
         Mendelian randomization refers to another method of analyzing epidemiologic data to determine possible causal associations between gene variants and clinical phenotypes. As an example, suppose one wishes to determine whether a trait A contributes to a second trait B, and GWAS has been done for trait A. The Mendelian randomization study design tests whether there is a causal relationship between trait A and trait B [
         <a href="#rid12">
          12
         </a>
         ]. A subset of SNPs associated with trait A serve as the "genetic instrument" for trait A. Because meiosis segregates alleles at birth, prior to phenotype emergence, genotype serves as a means to randomize the population, mimicking the randomization of participants in a traditional randomized trial, with genotype serving as a proxy for trait A. (See
         <a class="medical medical_review" href="/z/d/html/13514.html" rel="external">
          "Mendelian randomization"
         </a>
         .)
        </p>
        <p>
         Mendelian randomization offers an attractive opportunity to make causal inferences in observational studies as in randomized trials, but it is essential to recognize that it relies on three strong assumptions, each of which can be questioned in most clinical settings [
         <a href="#rid13">
          13
         </a>
         ]:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The SNPs must be associated with trait A and the included SNPs should be representative of trait A's total genetic variation. In practice, the SNPs used as genetic instruments in a Mendelian randomization study typically account for a small percentage of trait A's variance.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The influence of the chosen SNPs must be mediated via trait A, rather than via any alternative pathway. This is commonly referred to as the "no pleiotropy" assumption.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The chosen SNPs must be independent of other potential confounding variables.
        </p>
        <p>
        </p>
        <p>
         In practice, verifying these assumptions is challenging and mandates that Mendelian randomization be interpreted with appropriate caution. Moreover, negative findings may be more reliable than positive findings. For example, Mendelian randomization findings are inconsistent with C-reactive protein (CRP) levels contributing to atherosclerotic disease risk directly, and instead suggest that the observed association between CRP and atherosclerotic disease are mediated by confounding factors such as excess body weight [
         <a href="#rid14">
          14
         </a>
         ]. In contrast, various putative causal links between CRP and cardiovascular disease outcomes have been questioned [
         <a href="#rid15">
          15
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/1484.html" rel="external">
          "C-reactive protein in cardiovascular disease"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H1335185423">
         <span class="h2">
          Expression quantitative trait loci (eQTL) analysis
         </span>
         <span class="headingEndMark">
          —
         </span>
         The pattern of gene expression within a tissue can be treated as a series of quantitative traits, analogous to stature. The expression of many genes can be measured simultaneously using microarrays. Either linkage mapping (expression quantitative trait loci mapping, eQTL mapping) or GWAS approaches can be used to map and identify sequence variants that account for differences in the levels of expression of specific genes [
         <a href="#rid16">
          16
         </a>
         ]. Results from such studies can be used to prioritize variants (ie, SNPs) for possible functional impact on human disease. Variants that confer changes in gene expression are more likely to lead to phenotypic variation at the cellular or clinical level.
        </p>
        <p class="headingAnchor" id="H342443302">
         <span class="h2">
          Deep sequencing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Technical advances in DNA sequencing, assembly of sequencing reads, and statistical analysis of sequence data have dramatically reduced the cost of sequencing. This has enabled "deep sequencing" (also called "next-generation sequencing") projects in which the whole exome (protein-encoding portion of the genome) or the whole genome is sequenced to seek biologically-significant sequence variation [
         <a href="#rid17">
          17-19
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p>
         Deep sequencing studies have revealed the contribution of rare variants to disease [
         <a href="#rid20">
          20,21
         </a>
         ]. In polygenic disorders, deep sequencing has the potential to account for the "missing heritability" that is encountered in GWAS designs [
         <a href="#rid22">
          22,23
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H2333009224">
         <span class="h2">
          RNA-seq
         </span>
         <span class="headingEndMark">
          —
         </span>
         Advances in sequencing technology and statistical analysis have made it possible for many laboratories to sequence the mRNA of a sample, hence the name RNA-seq [
         <a href="#rid24">
          24
         </a>
         ]. This approach is now often preferred to gene expression arrays as a means of characterizing the transcriptome (set of expressed genes) of a sample. RNA-seq offers two important advantages over array-based methods:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It is better able to characterize alternative splicing as a regulatory mechanism
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         It provides absolute as well as relative quantification of gene expression
        </p>
        <p>
        </p>
        <p>
         Technical advances have also made it possible to undertake RNA-seq using single cells as samples [
         <a href="#rid25">
          25
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H6">
         <span class="h2">
          Immunoprecipitation techniques (ChIP on Chip, ChIP-Seq)
         </span>
         <span class="headingEndMark">
          —
         </span>
         Immunoprecipitation (IP), commonly used to identify protein-protein interactions, has more recently been used to identify protein-DNA interactions, including genome-wide binding patterns of transcription factors and other DNA-binding elements. Results from these studies can be used to map transcription factor binding sites to each of the genes whose expression they control and to define the functional noncoding regions of the genome sequence. Many such annotations are available through the
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fgenome.ucsc.edu%2F&amp;token=ufMrPALdHsT7wurZejVHAmUjZNR9AdBHw41tISj6cis%3D&amp;TOPIC_ID=2903" target="_blank">
          UCSC Human Genome Browser
         </a>
         .
        </p>
        <p>
         Chromatin immunoprecipitation (ChIP) is performed by immunoprecipitation (IP) of fragmented but unpurified DNA, using a transcription factor or other DNA binding protein as the target of the IP. Following the precipitation, the DNA is purified and assayed. Assays are performed genome-wide through the use of microarrays ("gene chips," hence the term "ChIP on Chip") [
         <a href="#rid26">
          26
         </a>
         ]. ChIP-seq is a related methodology in which the immunoprecipitated DNA is sequenced instead of being hybridized to a microarray. This approach requires access to high-throughput sequencing and supporting bioinformatics resources. The resulting data constitute a map of the antigen DNA binding site. (See
         <a class="local">
          'Deep sequencing'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H1432856769">
         <span class="h2">
          Epigenomics
         </span>
         <span class="headingEndMark">
          —
         </span>
         An individual's genome is essentially constant across tissues, but the pattern of gene expression among tissues varies greatly. This observation points to the existence of regulatory mechanisms beyond DNA sequence variation. Transcriptional activity of a genomic region is determined in part by the methylation status of the DNA and how the DNA is packaged into chromatin, which in turn is dependent in part on the modification of histone proteins. These epigenetic modifications underlie tissue-specific differences in gene expression, X-chromosome inactivation, and imprinting. (See
         <a class="medical medical_review" href="/z/d/html/14601.html" rel="external">
          "Principles of epigenetics"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2898.html" rel="external">
          "Genetics: Glossary of terms", section on 'X-inactivation'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)", section on 'Parent-of-origin effects (imprinting)'
         </a>
         .)
        </p>
        <p>
         DNA methylation can be measured on a genome-wide scale, and such experiments are routine in many laboratories [
         <a href="#rid27">
          27
         </a>
         ]. Chromatin immunoprecipitation (ChIP) is a robust approach to identifying modified histones, enabling genome-wide assessment of specific histone modifications [
         <a href="#rid28">
          28
         </a>
         ]. (See
         <a class="local">
          'Immunoprecipitation techniques (ChIP on Chip, ChIP-Seq)'
         </a>
         above.)
        </p>
        <p class="headingAnchor" id="H2765638410">
         <span class="h2">
          Gene editing
         </span>
         <span class="headingEndMark">
          —
         </span>
         The value of model organisms has been enhanced by the development and widespread use of gene editing technology [
         <a href="#rid29">
          29
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/98724.html" rel="external">
          "Overview of gene therapy, gene editing, and gene silencing"
         </a>
         .)
        </p>
        <p>
         Gene editing has accelerated the accuracy and speed with which specific variants can be introduced into the genome. Just as gene editing is being used to correct molecular lesions in a therapeutic context, it can be used to engineer specific mutations for study in model organisms [
         <a href="#rid30">
          30-34
         </a>
         ]. Gene editing technology has had a greater impact on multicellular organisms, where isolating and maintaining germ cells is challenging. For unicellular organisms, targeted mutagenesis has long been a standard method.
        </p>
        <p class="headingAnchor" id="H7">
         <span class="h1">
          EXAMPLES OF MODEL SYSTEMS
         </span>
         <span class="headingEndMark">
          —
         </span>
         Although not comprehensive, a few examples of model systems are presented here to illustrate the use of appropriately chosen model systems to relate genetic and functional data. The value of model systems depends upon establishing a proper balance between the simplification they offer and the extent to which they mirror human physiology. These factors usually are inversely related, as summarized in the table comparing the different systems  (
         <a class="graphic graphic_table graphicRef127161" href="/z/d/graphic/127161.html" rel="external">
          table 1
         </a>
         ).
        </p>
        <p>
         Model systems allow investigators to relate raw DNA sequence data with functional information. Models are useful insofar as they offer the technical means to accomplish studies that could not be undertaken in human subjects; model organisms permit a host of functional studies that could never be undertaken in a clinical setting.
        </p>
        <p class="headingAnchor" id="H8">
         <span class="h2">
          Prokaryotes and yeast
         </span>
         <span class="headingEndMark">
          —
         </span>
         Prokaryotic and yeast models derive much of their value from the ability to investigate large experimental populations in search of rare individuals. Bacteriophages (phages) are a type of virus that infect bacteria. Yeast are single-celled eukaryotes that contain many well-conserved genes, many of which have human homologues.
        </p>
        <p>
         Standardized protocols can be used for selecting and/or screening for specific mutations. By selection, only individual organisms possessing a specific metabolic property survive to be examined. Screening employs identification of a specific metabolic property in some organisms that results in an easily scored difference in phenotype, allowing such organisms to be reliably identified, isolated, and recovered for further study.
        </p>
        <p class="headingAnchor" id="H9">
         <span class="h3">
          Bacteriophage lambda
         </span>
         <span class="headingEndMark">
          —
         </span>
         The bacteriophage lambda model has advanced general understanding of transcriptional changes in response to environmental stimuli. Lambda is a temperate phage (a type of virus that infects bacteria) that can grow in either a lytic or lysogenic life cycle. This is manifested morphologically by production of turbid plaques, in which surviving infected bacteria are present.
        </p>
        <p>
         There are various lambda mutants that produce clear plaques in which survival of infected bacteria is much reduced or that behave aberrantly with regard to superinfection. These properties are understood at a detailed molecular level.
        </p>
        <p>
         Interactions of human transcription factors with their cognate DNA response elements and the resulting modulation of transcription are more complex variations on the themes first defined in prokaryotes.
        </p>
        <p class="headingAnchor" id="H10">
         <span class="h3">
          Yeast
         </span>
         <span class="headingEndMark">
          —
         </span>
         <em>
          Saccharomyces cerevisiae
         </em>
         , a budding yeast, is one of the most fruitful model systems. Yeast have many properties that make them ideal model organisms: the possibility of targeted gene replacement, growth as both a haploid and a diploid organism, and an array of selection and screening schemes. The sequence of the entire
         <em>
          S. cerevisiae
         </em>
         genome has been available for decades.
        </p>
        <p>
         An example of the application of these features was the 1999 report of the effects of systematic deletion of over 2000 yeast genes under a variety of growth conditions [
         <a href="#rid35">
          35
         </a>
         ]. A significant insight arising from studies in yeast is the discovery of the sirtuin family of proteins, now recognized as critical determinants of longevity [
         <a href="#rid36">
          36
         </a>
         ].
        </p>
        <p>
         The ease of experimental manipulation in yeast makes this model system particularly useful for performing experiments in which many genotypes are screened for function. As an example, the yeast 2-hybrid system allows a systematic search for interactions among proteins [
         <a href="#rid37">
          37
         </a>
         ]. This strategy was applied in a study in which approximately 1000 interacting protein pairs were detected [
         <a href="#rid38">
          38
         </a>
         ]. Importantly, the method is not restricted to yeast proteins, so that interacting proteins from any species can be studied in yeast [
         <a href="#rid39">
          39
         </a>
         ].
        </p>
        <p>
         Protein-protein interactions are important features of virtually every cellular process. At the most basic level, it is important to recall that protein function depends on each protein assuming its proper conformation during synthesis; misfolded proteins lose function. So-called "chaperone" proteins interact with nascent peptides to assist in their correct folding during protein synthesis. While nascent protein-chaperone interactions can be studied individually and in other models, the yeast model system is particularly well-suited to global functional analysis of these interactions [
         <a href="#rid40">
          40
         </a>
         ].
        </p>
        <p>
         This experimental strategy has been adapted to seek interactions between proteins from other organisms. As noted above, approximately one-third of human genes have yeast homologs and understanding of human biology has been greatly advanced by study of these in the more easily manipulated yeast system.
        </p>
        <p>
         Data about yeast are readily available at the Saccharomyces genome database (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.yeastgenome.org%2F&amp;token=LMeIlcwjnxhT98hiiOz2AlCHfvBijwW0HWPa9L%2FJxJgA3esFS%2BKregwcg8oyZH86&amp;TOPIC_ID=2903" target="_blank">
          www.yeastgenome.org
         </a>
         ) [
         <a href="#rid41">
          41
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H11">
         <span class="h2">
          Nematode
         </span>
         <span class="headingEndMark">
          —
         </span>
         The nematode,
         <em>
          Caenorhabditis elegans
         </em>
         , is used extensively as a model system [
         <a href="#rid42">
          42-50
         </a>
         ]. Use in developmental biology derives from the ability to trace the cell lineages of every cell [
         <a href="#rid51">
          51,52
         </a>
         ]. C. elegans is a popular model in neurology, as its complete neural circuitry is known, and in studies of aging, as a number of lifespan mutants have been discovered [
         <a href="#rid42">
          42-50
         </a>
         ]. The C. elegans system has also provided important insights regarding the role of molecular chaperones in protecting against neurodegenerative diseases [
         <a href="#rid53">
          53
         </a>
         ].
        </p>
        <p>
         C. elegans data are available through
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwormbase.org%2F%2F&amp;token=Y6hdVMXH6GKmTQJWIyAZb5jSeNqyekSiB%2FOi2%2BLwnMQ%3D&amp;TOPIC_ID=2903" target="_blank">
          WormBase
         </a>
         and the Sanger Centre's
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.sanger.ac.uk%2Fdata%2Fworm-genome%2F&amp;token=vdCs7dqmABYrTYLl86e3ULKsMpE9KpJuThP8g3ElLofW0iNkVQYYlb8yMO9VKymkXZuxF%2BTb9f7kninHE0rVTA%3D%3D&amp;TOPIC_ID=2903" target="_blank">
          Worm Genome page
         </a>
         [
         <a href="#rid54">
          54,55
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H12">
         <span class="h2">
          Fruit fly
         </span>
         <span class="headingEndMark">
          —
         </span>
         The fruit fly,
         <em>
          Drosophila melanogaster
         </em>
         , the first organism in which detailed linkage maps were constructed, remains an extremely useful model.
        </p>
        <p>
         It has been particularly informative in studies of how spatial information is established during development. The Wnt signaling pathway occupies a central role in mediating multiple aspects of mammalian development and physiology, and it was first found through its homology to the Drosophila wingless gene. Wnt signaling is critical in establishing the body plan and regulating cell polarity and proliferation, including regulating skeletal mass [
         <a href="#rid56">
          56,57
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/2053.html" rel="external">
          "Normal skeletal development and regulation of bone formation and resorption"
         </a>
         .)
        </p>
        <p>
         An excellent online resource is
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fflybase.org%2F&amp;token=DKtiRaD%2BQDZ98lhQaaPMh7VAFgY8DfPsgP04ocFXYtA%3D&amp;TOPIC_ID=2903" target="_blank">
          Flybase
         </a>
         [
         <a href="#rid58">
          58
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H13">
         <span class="h2">
          Zebrafish
         </span>
         <span class="headingEndMark">
          —
         </span>
         The zebrafish is useful because its embryos develop freely, are transparent, and genetic manipulations can be readily performed. Zebrafish are particularly well-suited to studying environmental factors such as drugs and thermal stress because of their small size, aquatic environment, and poikilothermic metabolism. They are also easily amenable to in vivo RNA interference (RNAi), a molecular technique for targeted silencing of specific genes, allowing a wide variety of developmental features to be studied.
        </p>
        <p>
         As an example, RNAi-mediated suppression of HSP90 (90 kD heat shock protein) across several strains of zebrafish resulted in variable ocular development, with some strains demonstrating severe mutations and others more mild phenotype [
         <a href="#rid59">
          59
         </a>
         ]. The issues addressed in this study fully exploit the advantages of zebrafish as a model system.
        </p>
        <p>
         The
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fzfin.org%2F&amp;token=eTAtQmzjUbEdG1%2Fdb%2B6m%2FNiEZSPP4xJ5jsEo1WA2Ct4%3D&amp;TOPIC_ID=2903" target="_blank">
          Zebrafish Information Network
         </a>
         provides further information and additional links [
         <a href="#rid60">
          60
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H14">
         <span class="h2">
          Mouse and rat
         </span>
         <span class="headingEndMark">
          —
         </span>
         Mice are commonly used to study aspects of mammalian physiology that cannot be modeled in invertebrates or non-mammalian vertebrates. Mice are easy to breed, have been extensively studied genetically, and are readily available. There are hundreds of inbred strains, as well as congenic, recombinant inbred, and recombinant congenic strains. The numerous mice mutants available involve all aspects of physiology.
        </p>
        <p>
         Mice are also used to study the effects of mutating or complementing specific genes, through the use of transgenic technology. This allows study of the consequences of alterations in candidate genes. With the use of Cre-loxP systems, it is possible to engineer specific mutations and to restrict genetic lesions to specific tissues. Specific "reporter" mice are widely used to validate these manipulations [
         <a href="#rid61">
          61
         </a>
         ] and to assess the activity of specific pathways [
         <a href="#rid62">
          62
         </a>
         ]. These capabilities are being exploited by the
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.mousephenotype.org%2F&amp;token=kKXHxK%2Bt2cji8cnnyYI7JCXXTVWkUz73vHNiCHbd%2FjDoDK4X%2Fjrd2ZuMYZCmmM2q&amp;TOPIC_ID=2903" target="_blank">
          International Mouse Phenotyping Consortium
         </a>
         , which seeks to generate floxed knockout alleles for every mouse gene, phenotype these animals, annotate the findings, and make these resources widely available to investigators [
         <a href="#rid63">
          63
         </a>
         ].
        </p>
        <p>
         Humanized mice have allowed investigators to engraft human hematopoietic immune response cells into immunodeficient mice that can be manipulated experimentally. (See
         <a class="medical medical_review" href="/z/d/html/2894.html" rel="external">
          "Tools for genetics and genomics: Specially bred and genetically engineered mice"
         </a>
         .)
        </p>
        <p>
         The Jackson Laboratory's
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fwww.informatics.jax.org%2F&amp;token=tQI8lhw%2Fr0msgHkj%2FKEdZ4%2FuZ0LHPa1m12S3YsPXjX8ENJHUtyoBrdD4frqMNJna&amp;TOPIC_ID=2903" target="_blank">
          Mouse Genome Informatics site
         </a>
         provides access to information about genes, mutants, inbred strains, mapping and developmental data, and homology to genes in other organisms [
         <a href="#rid64">
          64-66
         </a>
         ]. Data on the rat genome can be accessed at the
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Frgd.mcw.edu%2Fwg%2Fciting-rgd&amp;token=4gCrmzFrUtu7XHAXhbRu4sSbSrs1HgrpF2vpI4jDADeEwd8aqTAab6Fkliqkpr6b&amp;TOPIC_ID=2903" target="_blank">
          Rat Genome Database
         </a>
         [
         <a href="#rid67">
          67
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H15">
         <span class="h1">
          ONLINE SOURCES OF GENETIC INFORMATION
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          OMIM
         </strong>
         – The most clinically oriented genome-based database is
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fomim.org%2F&amp;token=87jLzclRwWxNoI%2BQq5Gg719fkmhU1%2BHL5BgfqtERBlQ%3D&amp;TOPIC_ID=2903" target="_blank">
          OMIM
         </a>
         (Online Mendelian Inheritance in Man). OMIM is a manually curated compendium of genes and genetic phenotypes, regularly updated by reviewers who cull the published literature. Each entry is a full-text overview, with references hyperlinked to PubMed entries. OMIM's emphasis is the relationship between gen locus, genetic variation, and their phenotypic consequences. OMIM also provides the historical context in which understanding about various genes and diseases has emerged.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          GTR
         </strong>
         – For clinicians needing guidance in management of patients with genetic conditions, the
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgtr%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2Fy3%2BLj0shalBW%2BL1ZyL4tryNPgji1PaJ6Kbd%2Fo1hE9A6&amp;TOPIC_ID=2903" target="_blank">
          Genetic Testing Registry
         </a>
         (GTR) is a database that provides information on genetic conditions and laboratories that offer testing for pathogenic variants in disease genes.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          GeneCards
         </strong>
         – The Weizmann Institute's
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.genecards.org%2F&amp;token=F%2F33Yq3Lpx4WKgz%2BcwHiEwSM6W8ko6iO3Sc5Qwl781cb3Fl2YK8%2F0j6h0gKZbzoL&amp;TOPIC_ID=2903" target="_blank">
          GeneCards site
         </a>
         is a well-organized gene-centered database, providing genomic, proteomic, and functional information on all known and predicted human genes [
         <a href="#rid68">
          68
         </a>
         ]. The included information is primarily geared toward researchers, but includes links to diseases, drugs, and drug candidates.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          National Institutes of Health (NIH)
         </strong>
         – The National Center for Biotechnology Information (NCBI), operated by the National Library of Medicine, consists of a series of interconnected databases (the full list is given at
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fsearch%2F%3Fitool%3Dtoolbar&amp;token=wEsKVFvqlKzpT%2BqOtddW%2F9WyS2%2BmbISD1cPa9SCgRneqO2DxylW5RdUTQOnlrYcZWWPi%2FwID9m8TfRvLwcfG0g%3D%3D&amp;TOPIC_ID=2903" target="_blank">
          www.ncbi.nlm.nih.gov/sites/gquery?itool=toolbar
         </a>
         ), and serves as a repository for all publically submitted genomic data. The site includes nucleic acid sequence generated through federally-funded genome projects, genetic sequence variants, results from gene expression profiling studies and genome-wide association studies. The Entrez browser system is one possible entry point (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fsearch%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2F9WyS2%2BmbISD1cPa9SCgRnfdiqAhxfWINjjZusG4KKvX&amp;TOPIC_ID=2903" target="_blank">
          www.ncbi.nlm.nih.gov/gquery/gquery.fcgi
         </a>
         ). Many readers are already familiar with this site's medical literature search system, PubMed.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         An Entrez portal (the genotypes and phenotypes database, dbGaP
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgap%2F%3FDb%3Dgap&amp;token=wEsKVFvqlKzpT%2BqOtddW%2Fy3%2BLj0shalBW%2BL1ZyL4trzTBZr4tCUbIpWAu5hsJb%2BD&amp;TOPIC_ID=2903" target="_blank">
          www.ncbi.nlm.nih.gov/sites/entrez?Db=gap
         </a>
         ) allows users to search by study or disease. The National Human Genome Research Institute (NHGRI) and the European Molecular Biology (EMBL) also collaborate to produce a database of published GWAS at
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ebi.ac.uk%2Fgwas%2F&amp;token=q%2BtFDlZ3%2BJRQXZSWNuOsHFRyGx8wkrOWVXJ3evVCBbSMiwblIn6U%2Bf4ZvDW2Xssf&amp;TOPIC_ID=2903" target="_blank">
          http://www.ebi.ac.uk/gwas/
         </a>
         .
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         The Entrez system also allows users to conduct a variety of analyses. For example, the enables searching of all nucleotide (DNA) or protein databases for sequence homologies to a query sequence [
         <a href="#rid69">
          69-75
         </a>
         ]. BLAST reports include all sequence matches and provide metrics of the strength of alignment, confidence scores for the alignment, and graphical depictions of the alignments.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         While Entrez is extremely flexible and comprehensive, its interface is not as easy to use as that of several other resources that are more graphics-based. Two easily used graphics-based genome browsers are Ensembl (
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fwww.ensembl.org%2Findex.html&amp;token=%2B4AjuT4GfqDJcqrCW5u%2FeCP32t4Ql%2BlGPAwTtd%2BZQpw%2FTVGD1FJ0mrxj1cbcwdfi&amp;TOPIC_ID=2903" target="_blank">
          www.ensembl.org/index.html
         </a>
         ) [
         <a href="#rid76">
          76
         </a>
         ] and the
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fgenome.ucsc.edu%2F&amp;token=ufMrPALdHsT7wurZejVHAmUjZNR9AdBHw41tISj6cis%3D&amp;TOPIC_ID=2903" target="_blank">
          UCSC Genome Browser
         </a>
         [
         <a href="#rid77">
          77
         </a>
         ]. These browsers provide graphical representation of the genome with sequence annotation of genes, variants, functional elements, and other types of genomic data. Much of the information presented is derived from the NCBI database, and daily cross-references and updates of NCBI, Ensembl, and the UCSC browser ensure similar query retrievals regardless of which interface is used for the query.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Broad Institute
         </strong>
         – The Broad Institute hosts a suite of tools for analyzing sequence data, the Genetic Analysis Toolkit (GATK,
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fgatk.broadinstitute.org%2Fhc%2Fen-us&amp;token=e%2B5cRbbpiptpa2S%2FtiOvWZeBXZpfGX6H9fQwxibCTUVC8lmlZl53OwtX4VGdY49BrwC1JCwUsNIWoeWxLU0oqw%3D%3D&amp;TOPIC_ID=2903" target="_blank">
          https://gatk.broadinstitute.org/hc/en-us
         </a>
         ). In addition to software for undertaking analysis of next generation sequencing data, the site also features extensive documentation that helps users to follow best practices in seeking sequence variants.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         The genome aggregation database (gnomAD) is one such tool, integrating reference sequence data collected across various experimental platforms and facilitating variant interpretation [
         <a href="#rid78">
          78
         </a>
         ]. The Broad Institute also hosts cancer program scientific tools (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.broadinstitute.org%2Fcancer%2Fcancer-program-scientific-tools-and-resources&amp;token=UBpDO%2BJpegrPR8DNWEL62Co1rh7%2BD%2BwthLwwGitlrLXa5DBL6%2FDAZgvWw8%2FOpKohA7rRxxUIYcSQeSHmmEI9QoEjYJy2xycxVsjuHkZ9TPE%3D&amp;TOPIC_ID=2903" target="_blank">
          https://www.broadinstitute.org/cancer/cancer-program-scientific-tools-and-resources
         </a>
         ) and disorder-focused knowledge portals (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fhugeamp.org%2F&amp;token=vMq8sP0ycX6E1sT%2F89T6d4rsvIfqHFrqGxCKCA%2FF3S4%3D&amp;TOPIC_ID=2903" target="_blank">
          https://hugeamp.org
         </a>
         ) for several disease groups, including type 1 and type 2 diabetes, cardiovascular disease, cerebrovascular disease, sleep disorders, amyotrophic lateral sclerosis, lung disease, musculoskeletal disorders, and lipid droplet biology [
         <a href="#rid79">
          79
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H18">
         <span class="h1">
          SUMMARY
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Rationale
         </strong>
         – Understanding the genetic basis for human disease involves knowledge both of genome sequences and of gene function. Knowledge of variations in gene sequence is largely dependent on the use of single nucleotide polymorphisms (SNPs) in a variety of study designs. Gene function studies are enabled by the use of model systems. (See
         <a class="local">
          'Basic concepts'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Experimental approaches to genome analysis
         </strong>
         – Genome-based experimental designs include genome-wide association studies (GWAS), quantitative analysis of gene expression (eQTL analysis), and studies involving immunoprecipitation techniques. Although GWAS studies have identified multiple variants associated with clinical disorders, the physiologic effect of the GWAS-identified individual variant alleles has been relatively modest. (See
         <a class="local">
          'Genome-based experimental designs'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Animal model systems
         </strong>
         – Multiple model systems, chosen appropriately for their unique properties, can give insight to the relationship between genetic composition and function within the species. The value of model systems depends upon establishing a proper balance between the simplification they offer and the extent to which they mirror human physiology. Advantages and disadvantages of different model systems are summarized in the table  (
         <a class="graphic graphic_table graphicRef127161" href="/z/d/graphic/127161.html" rel="external">
          table 1
         </a>
         ). (See
         <a class="local">
          'Examples of model systems'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Online resources
         </strong>
         – Links to online resources including OMIM, GTR, NIH sites (National Library of Medicine, Entrez), and the Broad Institute, are listed above. (See
         <a class="local">
          'Online sources of genetic information'
         </a>
         above.)
        </p>
        <p>
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li>
          <a class="nounderline abstract_t">
           Venter JC, Adams MD, Myers EW, et al. The sequence of the human genome. Science 2001; 291:1304.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sherry ST, Ward MH, Kholodov M, et al. dbSNP: the NCBI database of genetic variation. Nucleic Acids Res 2001; 29:308.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wang DG, Fan JB, Siao CJ, et al. Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome. Science 1998; 280:1077.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Chakravarti A. Population genetics--making sense out of sequence. Nat Genet 1999; 21:56.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kwok PY, Gu Z. Single nucleotide polymorphism libraries: why and how are we building them? Mol Med Today 1999; 5:538.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Eppig JT, Nadeau JH. Comparative maps: the mammalian jigsaw puzzle. Curr Opin Genet Dev 1995; 5:709.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           DeBry RW, Seldin MF. Human/mouse homology relationships. Genomics 1996; 33:337.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Nadeau JH. Maps of linkage and synteny homologies between mouse and man. Trends Genet 1989; 5:82.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Seldin MF. Genome surfing: using internet-based informatic tools toward functional genetic studies in mouse and humans. Methods 1997; 13:445.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stein CM, Elston RC. Finding genes underlying human disease. Clin Genet 2009; 75:101.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Estrada K, Krawczak M, Schreiber S, et al. A genome-wide association study of northwestern Europeans involves the C-type natriuretic peptide signaling pathway in the etiology of human height variation. Hum Mol Genet 2009; 18:3516.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Katan MB. Apolipoprotein E isoforms, serum cholesterol, and cancer. Lancet 1986; 1:507.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Davey Smith G, Hemani G. Mendelian randomization: genetic anchors for causal inference in epidemiological studies. Hum Mol Genet 2014; 23:R89.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           C Reactive Protein Coronary Heart Disease Genetics Collaboration (CCGC), Wensley F, Gao P, et al. Association between C reactive protein and coronary heart disease: mendelian randomisation analysis based on individual participant data. BMJ 2011; 342:d548.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Markozannes G, Koutsioumpa C, Cividini S, et al. Global assessment of C-reactive protein and health-related outcomes: an umbrella review of evidence from observational studies and Mendelian randomization studies. Eur J Epidemiol 2021; 36:11.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cookson W, Liang L, Abecasis G, et al. Mapping complex disease traits with global gene expression. Nat Rev Genet 2009; 10:184.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ku CS, Naidoo N, Pawitan Y. Revisiting Mendelian disorders through exome sequencing. Hum Genet 2011; 129:351.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ng PC, Kirkness EF. Whole genome sequencing. Methods Mol Biol 2010; 628:215.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Mestan KK, Ilkhanoff L, Mouli S, Lin S. Genomic sequencing in clinical trials. J Transl Med 2011; 9:222.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ng SB, Turner EH, Robertson PD, et al. Targeted capture and massively parallel sequencing of 12 human exomes. Nature 2009; 461:272.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Choi M, Scholl UI, Ji W, et al. Genetic diagnosis by whole exome capture and massively parallel DNA sequencing. Proc Natl Acad Sci U S A 2009; 106:19096.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           1000 Genomes Project Consortium, Abecasis GR, Altshuler D, et al. A map of human genome variation from population-scale sequencing. Nature 2010; 467:1061.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zaboli G, Ameur A, Igl W, et al. Sequencing of high-complexity DNA pools for identification of nucleotide and structural variants in regions associated with complex traits. Eur J Hum Genet 2012; 20:77.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wang Z, Gerstein M, Snyder M. RNA-Seq: a revolutionary tool for transcriptomics. Nat Rev Genet 2009; 10:57.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stegle O, Teichmann SA, Marioni JC. Computational and analytical challenges in single-cell transcriptomics. Nat Rev Genet 2015; 16:133.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Acevedo LG, Iniguez AL, Holster HL, et al. Genome-scale ChIP-chip analysis using 10,000 human cells. Biotechniques 2007; 43:791.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Laird PW. Principles and challenges of genomewide DNA methylation analysis. Nat Rev Genet 2010; 11:191.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Schones DE, Zhao K. Genome-wide approaches to studying chromatin modifications. Nat Rev Genet 2008; 9:179.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Broeders M, Herrero-Hernandez P, Ernst MPT, et al. Sharpening the Molecular Scissors: Advances in Gene-Editing Technology. iScience 2020; 23:100789.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Nance J, Frøkjær-Jensen C. The Caenorhabditis elegans Transgenic Toolbox. Genetics 2019; 212:959.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Xu RG, Wang X, Shen D, et al. Perspectives on gene expression regulation techniques in Drosophila. J Genet Genomics 2019; 46:213.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Liu K, Petree C, Requena T, et al. Expanding the CRISPR Toolbox in Zebrafish for Studying Development and Disease. Front Cell Dev Biol 2019; 7:13.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Low BE, Kutny PM, Wiles MV. Simple, Efficient CRISPR-Cas9-Mediated Gene Editing in Mice: Strategies and Methods. Methods Mol Biol 2016; 1438:19.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Meek S, Mashimo T, Burdon T. From engineering to editing the rat genome. Mamm Genome 2017; 28:302.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Winzeler EA, Shoemaker DD, Astromoff A, et al. Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis. Science 1999; 285:901.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Michan S, Sinclair D. Sirtuins in mammals: insights into their biological function. Biochem J 2007; 404:1.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Brent R, Finley RL Jr. Understanding gene and allele function with two-hybrid methods. Annu Rev Genet 1997; 31:663.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Uetz P, Giot L, Cagney G, et al. A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae. Nature 2000; 403:623.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Parrish JR, Gulyas KD, Finley RL Jr. Yeast two-hybrid contributions to interactome mapping. Curr Opin Biotechnol 2006; 17:387.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gong Y, Kakihara Y, Krogan N, et al. An atlas of chaperone-protein interactions in Saccharomyces cerevisiae: implications to protein folding pathways in the cell. Mol Syst Biol 2009; 5:275.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Engel SR, Balakrishnan R, Binkley G, et al. Saccharomyces Genome Database provides mutant phenotype data. Nucleic Acids Res 2010; 38:D433.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hodgkin J, Horvitz HR, Jasny BR, Kimble JC. Elegans: Sequence to Biology. Science 1998; 282:2011.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           C. elegans Sequencing Consortium. Genome sequence of the nematode C. elegans: a platform for investigating biology. Science 1998; 282:2012.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Bargmann CI. Neurobiology of the Caenorhabditis elegans genome. Science 1998; 282:2028.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Pennisi E. Worming secrets from the C. elegans genome. Science 1998; 282:1972.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Blaxter M. Caenorhabditis elegans is a nematode. Science 1998; 282:2041.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ruvkun G, Hobert O. The taxonomy of developmental control in Caenorhabditis elegans. Science 1998; 282:2033.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Clarke ND, Berg JM. Zinc fingers in Caenorhabditis elegans: finding families and probing pathways. Science 1998; 282:2018.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Bloom FE. Staying afloat on the seas of data. Science 1998; 282:1989.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Chervitz SA, Aravind L, Sherlock G, et al. Comparison of the complete protein sets of worm and yeast: orthology and divergence. Science 1998; 282:2022.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Deppe U, Schierenberg E, Cole T, et al. Cell lineages of the embryo of the nematode Caenorhabditis elegans. Proc Natl Acad Sci U S A 1978; 75:376.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sulston JE, Schierenberg E, White JG, Thomson JN. The embryonic cell lineage of the nematode Caenorhabditis elegans. Dev Biol 1983; 100:64.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Prahlad V, Morimoto RI. Integrating the stress response: lessons for neurodegenerative diseases from C. elegans. Trends Cell Biol 2009; 19:52.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Harris TW, Antoshechkin I, Bieri T, et al. WormBase: a comprehensive resource for nematode research. Nucleic Acids Res 2010; 38:D463.
          </a>
         </li>
         <li class="breakAll">
          Worm Genome. Sanger Institute. Available at: https://www.sanger.ac.uk/data/worm-genome/ (Accessed on October 25, 2022).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Logan CY, Nusse R. The Wnt signaling pathway in development and disease. Annu Rev Cell Dev Biol 2004; 20:781.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           MacDonald BT, Tamai K, He X. Wnt/beta-catenin signaling: components, mechanisms, and diseases. Dev Cell 2009; 17:9.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Tweedie S, Ashburner M, Falls K, et al. FlyBase: enhancing Drosophila Gene Ontology annotations. Nucleic Acids Res 2009; 37:D555.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yeyati PL, Bancewicz RM, Maule J, van Heyningen V. Hsp90 selectively modulates phenotype in vertebrate development. PLoS Genet 2007; 3:e43.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sprague J, Bayraktaroglu L, Clements D, et al. The Zebrafish Information Network: the zebrafish model organism database. Nucleic Acids Res 2006; 34:D581.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Mao X, Fujiwara Y, Orkin SH. Improved reporter strain for monitoring Cre recombinase-mediated DNA excisions in mice. Proc Natl Acad Sci U S A 1999; 96:5037.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           DasGupta R, Fuchs E. Multiple roles for activated LEF/TCF transcription complexes during hair follicle development and differentiation. Development 1999; 126:4557.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ringwald M, Iyer V, Mason JC, et al. The IKMC web portal: a central point of entry to data and resources from the International Knockout Mouse Consortium. Nucleic Acids Res 2011; 39:D849.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Eppig JT, Blake JA, Bult CJ, et al. The Mouse Genome Database (MGD): comprehensive resource for genetics and genomics of the laboratory mouse. Nucleic Acids Res 2012; 40:D881.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Blake JA, Bult CJ, Kadin JA, et al. The Mouse Genome Database (MGD): premier model organism resource for mammalian genomics and genetics. Nucleic Acids Res 2011; 39:D842.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Smith CM, Finger JH, Hayamizu TF, et al. The mouse Gene Expression Database (GXD): 2007 update. Nucleic Acids Res 2007; 35:D618.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Laulederkind SJ, Hayman GT, Wang SJ, et al. The Rat Genome Database 2013--data, tools and users. Brief Bioinform 2013; 14:520.
          </a>
         </li>
         <li class="breakAll">
          Rebhan M, Chalifa-Caspi V, Prilusky J, Lancet D. GeneCards: Encyclopedia for genes, proteins and diseases., Weizmann Institute of Science, Rehovot, Israel 1997.
         </li>
         <li>
          <a class="nounderline abstract_t">
           Altschul SF, Gish W, Miller W, et al. Basic local alignment search tool. J Mol Biol 1990; 215:403.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Altschul SF, Madden TL, Schäffer AA, et al. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 1997; 25:3389.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gish W, States DJ. Identification of protein coding regions by database similarity search. Nat Genet 1993; 3:266.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Madden TL, Tatusov RL, Zhang J. Applications of network BLAST server. Methods Enzymol 1996; 266:131.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zhang J, Madden TL. PowerBLAST: a new network BLAST application for interactive or automated sequence analysis and annotation. Genome Res 1997; 7:649.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zhang Z, Schwartz S, Wagner L, Miller W. A greedy algorithm for aligning DNA sequences. J Comput Biol 2000; 7:203.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Morgulis A, Coulouris G, Raytselis Y, et al. Database indexing for production MegaBLAST searches. Bioinformatics 2008; 24:1757.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hubbard TJ, Aken BL, Beal K, et al. Ensembl 2007. Nucleic Acids Res 2007; 35:D610.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kent WJ, Sugnet CW, Furey TS, et al. The human genome browser at UCSC. Genome Res 2002; 12:996.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gudmundsson S, Singer-Berk M, Watts NA, et al. Variant interpretation using population databases: Lessons from gnomAD. Hum Mutat 2022; 43:1012.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lawrence MS, Stojanov P, Mermel CH, et al. Discovery and saturation analysis of cancer genes across 21 tumour types. Nature 2014; 505:495.
          </a>
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 2903 Version 31.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11181995" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : The sequence of the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11125122" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : dbSNP: the NCBI database of genetic variation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9582121" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9915503" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : Population genetics--making sense out of sequence.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10562720" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : Single nucleotide polymorphism libraries: why and how are we building them?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8745068" id="rid5" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          6 : Comparative maps: the mammalian jigsaw puzzle.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8660993" id="rid6" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          7 : Human/mouse homology relationships.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2660365" id="rid7" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          8 : Maps of linkage and synteny homologies between mouse and man.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9480788" id="rid8" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          9 : Genome surfing: using internet-based informatic tools toward functional genetic studies in mouse and humans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18783406" id="rid9" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          10 : Finding genes underlying human disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19570815" id="rid10" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          11 : A genome-wide association study of northwestern Europeans involves the C-type natriuretic peptide signaling pathway in the etiology of human height variation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2869248" id="rid11" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          12 : Apolipoprotein E isoforms, serum cholesterol, and cancer.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25064373" id="rid12" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          13 : Mendelian randomization: genetic anchors for causal inference in epidemiological studies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21325005" id="rid13" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          14 : Association between C reactive protein and coronary heart disease: mendelian randomisation analysis based on individual participant data.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/32978716" id="rid14" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          15 : Global assessment of C-reactive protein and health-related outcomes: an umbrella review of evidence from observational studies and Mendelian randomization studies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19223927" id="rid15" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          16 : Mapping complex disease traits with global gene expression.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21331778" id="rid16" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          17 : Revisiting Mendelian disorders through exome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20238084" id="rid17" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          18 : Whole genome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22206293" id="rid18" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          19 : Genomic sequencing in clinical trials.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19684571" id="rid19" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          20 : Targeted capture and massively parallel sequencing of 12 human exomes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19861545" id="rid20" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          21 : Genetic diagnosis by whole exome capture and massively parallel DNA sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20981092" id="rid21" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          22 : A map of human genome variation from population-scale sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21811304" id="rid22" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          23 : Sequencing of high-complexity DNA pools for identification of nucleotide and structural variants in regions associated with complex traits.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19015660" id="rid23" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          24 : RNA-Seq: a revolutionary tool for transcriptomics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25628217" id="rid24" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          25 : Computational and analytical challenges in single-cell transcriptomics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18251256" id="rid25" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          26 : Genome-scale ChIP-chip analysis using 10,000 human cells.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20125086" id="rid26" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          27 : Principles and challenges of genomewide DNA methylation analysis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18250624" id="rid27" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          28 : Genome-wide approaches to studying chromatin modifications.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31901636" id="rid28" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          29 : Sharpening the Molecular Scissors: Advances in Gene-Editing Technology.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31405997" id="rid29" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          30 : The Caenorhabditis elegans Transgenic Toolbox.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31060819" id="rid30" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          31 : Perspectives on gene expression regulation techniques in Drosophila.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30886848" id="rid31" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          32 : Expanding the CRISPR Toolbox in Zebrafish for Studying Development and Disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27150082" id="rid32" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          33 : Simple, Efficient CRISPR-Cas9-Mediated Gene Editing in Mice: Strategies and Methods.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28752194" id="rid33" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          34 : From engineering to editing the rat genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10436161" id="rid34" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          35 : Functional characterization of the S. cerevisiae genome by gene deletion and parallel analysis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17447894" id="rid35" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          36 : Sirtuins in mammals: insights into their biological function.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9442911" id="rid36" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          37 : Understanding gene and allele function with two-hybrid methods.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10688190" id="rid37" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          38 : A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16806892" id="rid38" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          39 : Yeast two-hybrid contributions to interactome mapping.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19536198" id="rid39" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          40 : An atlas of chaperone-protein interactions in Saccharomyces cerevisiae: implications to protein folding pathways in the cell.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19906697" id="rid40" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          41 : Saccharomyces Genome Database provides mutant phenotype data.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid41" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          42 : Elegans: Sequence to Biology
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9851916" id="rid42" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          43 : Genome sequence of the nematode C. elegans: a platform for investigating biology.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9851919" id="rid43" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          44 : Neurobiology of the Caenorhabditis elegans genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9874643" id="rid44" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          45 : Worming secrets from the C. elegans genome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9851921" id="rid45" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          46 : Caenorhabditis elegans is a nematode.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9851920" id="rid46" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          47 : The taxonomy of developmental control in Caenorhabditis elegans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9851917" id="rid47" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          48 : Zinc fingers in Caenorhabditis elegans: finding families and probing pathways.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9874645" id="rid48" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          49 : Staying afloat on the seas of data.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9851918" id="rid49" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          50 : Comparison of the complete protein sets of worm and yeast: orthology and divergence.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/272653" id="rid50" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          51 : Cell lineages of the embryo of the nematode Caenorhabditis elegans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/6684600" id="rid51" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          52 : The embryonic cell lineage of the nematode Caenorhabditis elegans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19112021" id="rid52" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          53 : Integrating the stress response: lessons for neurodegenerative diseases from C. elegans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19910365" id="rid53" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          54 : WormBase: a comprehensive resource for nematode research.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19910365" id="rid54" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          55 : WormBase: a comprehensive resource for nematode research.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15473860" id="rid55" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          56 : The Wnt signaling pathway in development and disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19619488" id="rid56" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          57 : Wnt/beta-catenin signaling: components, mechanisms, and diseases.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18948289" id="rid57" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          58 : FlyBase: enhancing Drosophila Gene Ontology annotations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17397257" id="rid58" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          59 : Hsp90 selectively modulates phenotype in vertebrate development.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16381936" id="rid59" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          60 : The Zebrafish Information Network: the zebrafish model organism database.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10220414" id="rid60" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          61 : Improved reporter strain for monitoring Cre recombinase-mediated DNA excisions in mice.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10498690" id="rid61" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          62 : Multiple roles for activated LEF/TCF transcription complexes during hair follicle development and differentiation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20929875" id="rid62" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          63 : The IKMC web portal: a central point of entry to data and resources from the International Knockout Mouse Consortium.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22075990" id="rid63" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          64 : The Mouse Genome Database (MGD): comprehensive resource for genetics and genomics of the laboratory mouse.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21051359" id="rid64" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          65 : The Mouse Genome Database (MGD): premier model organism resource for mammalian genomics and genetics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17130151" id="rid65" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          66 : The mouse Gene Expression Database (GXD): 2007 update.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23434633" id="rid66" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          67 : The Rat Genome Database 2013--data, tools and users.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23434633" id="rid67" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          68 : The Rat Genome Database 2013--data, tools and users.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2231712" id="rid68" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          69 : Basic local alignment search tool.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9254694" id="rid69" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          70 : Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8485583" id="rid70" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          71 : Identification of protein coding regions by database similarity search.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8743682" id="rid71" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          72 : Applications of network BLAST server.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9199938" id="rid72" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          73 : PowerBLAST: a new network BLAST application for interactive or automated sequence analysis and annotation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10890397" id="rid73" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          74 : A greedy algorithm for aligning DNA sequences.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18567917" id="rid74" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          75 : Database indexing for production MegaBLAST searches.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17148474" id="rid75" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          76 : Ensembl 2007.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12045153" id="rid76" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          77 : The human genome browser at UCSC.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/34859531" id="rid77" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          78 : Variant interpretation using population databases: Lessons from gnomAD.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24390350" id="rid78" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          79 : Discovery and saturation analysis of cancer genes across 21 tumour types.
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
