<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Microbiology of enterococci</title><link href="/z/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/z/d/css/icofont.min.css" rel="stylesheet"/><link href="/z/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/z/d/css/animate.css" rel="stylesheet"/><link href="/z/d/css/owl.carousel.css" rel="stylesheet"/><link href="/z/d/css/magnific-popup.css" rel="stylesheet"/><link href="/z/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/z/d/css/style.css" rel="stylesheet"/><link href="/z/d/css/rtl_edited.css" rel="stylesheet"/><link href="/z/d/css/responsive.css" rel="stylesheet"/><link href="https:///uptodate_assets/main.css" rel="stylesheet"/>
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>
<meta content="" name="description"/>
<link href="/z/d/img/favicon.png" rel="icon" type="image/png">
</link></head>
<body>
<!-- Start Main Menu Area -->
<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>
<section class="teacher-area ptb-10">
<div class="container">
</div>
<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Microbiology of enterococci</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Microbiology of enterococci</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Authors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Barbara E Murray, MD</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">William R Miller, MD</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Daniel J Sexton, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Milana Bogorodskaya, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Oct 23, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>A number of the characteristic features of enterococci (formerly called group D streptococci) have been recognized for at least 100 years, including their presence in feces and sewage, their ability to cause endocarditis and urinary tract infections (UTIs), and their ability to survive harsh environmental conditions including drying, high temperatures, and exposure to some antiseptics [<a href="#rid1">1,2</a>].</p><p>The common name "enterococcus," derived from the French "entérocoque" used in an 1899 publication, points out that these bacteria are cocci of intestinal (enteric) origin, and the 1906 use of <em>Streptococcus faecalis</em> and the 1919 use of <em>Streptococcus faecium</em> also emphasized their presence in feces.</p><p>Subsequent studies confirmed these early observations. Most humans and animals have enterococci in their intestinal tract; the counts in healthy humans often range from 10<sup>5</sup> to 10<sup>7</sup> bacteria per gram of stool.</p><p class="bulletIndent1"><span class="glyph">●</span>Enterococci, particularly <em>Enterococcus faecalis</em>, are a relatively common cause of endocarditis (third, after staphylococci and streptococci), causing 5 to 15 percent of community-acquired endocarditis and up to 30 percent of nosocomially acquired endocarditis), and have been recognized as the leading cause of infective endocarditis in patients who undergo transcatheter aortic valve implantation (TAVI) [<a href="#rid3">3,4</a>]. In addition, enterococci can be a common cause of nosocomial urinary tract infections (being recovered from up to 15 to 20 percent of UTIs in the hospital setting) [<a href="#rid5">5,6</a>].</p><p></p><p class="headingAnchor" id="H2"><span class="h1">MICROBIOLOGY CLASSIFICATION</span><span class="headingEndMark"> — </span>Enterococci are gram-positive, facultatively anaerobic cocci that form short- to medium-length chains. In the 1930s, Lancefield devised a serologic typing system that classified enterococci as group D streptococci, along with some organisms (eg, <em>Streptococcus gallolyticus</em>, formerly <em>S. bovis</em>) that differed from enterococci in a number of other features. In 1937, Sherman divided streptococci into the pyogenic, viridans, lactic, and enterococcal groups. Enterococci were characterized by the following features:</p><p class="bulletIndent1"><span class="glyph">●</span>Their reaction with group D antiserum</p><p class="bulletIndent1"><span class="glyph">●</span>Their ability to grow in high salt, at 10ºC and 40ºC, and at high pH</p><p class="bulletIndent1"><span class="glyph">●</span>Their ability to survive exposure to 60ºC for 30 minutes</p><p class="bulletIndent1"><span class="glyph">●</span>Their ability to hydrolyze esculin in the presence of bile</p><p></p><p>In the ensuing 45 to 50 years, enterococci continued to be taxonomically classified in the genus <em>Streptococcus</em>. In the mid-1980s, with advances in taxonomic techniques based upon nucleic acid analyses, it was proposed that <em>S. faecalis</em> and <em>S. faecium </em>be removed from the genus <em>Streptococcus</em>, transferred to a new genus, <em>Enterococcus</em>, and reclassified as <em>E. faecalis</em> and <em>E. faecium</em>, respectively [<a href="#rid7">7</a>]. Subsequently, other enterococcal species were also reclassified from the genus <em>Streptococcus</em> to the genus <em>Enterococcus</em>; dozens of new species have now been recognized.</p><p class="headingAnchor" id="H3"><span class="h2">Problems with classification</span><span class="headingEndMark"> — </span>The characteristic features of enterococci as being bile esculin positive, reactive with group D antiserum, and able to grow in 6.5 percent sodium chloride were used by clinical laboratories for decades to distinguish enterococci from other catalase-negative gram-positive cocci. However, some problems arise with the use of these criteria. As an example, the presence of group D antigen can be difficult to detect in some enterococcal isolates. In addition, other organisms such as lactococci also can grow in high salt and/or be bile esculin positive; however, lactococci do not grow (or grow very slowly) at 45ºC.</p><p>Two other genera that share a number of biochemical features with enterococci, including their ability to produce the group D antigen, are <em>Pediococcus </em>and <em>Leuconostoc</em>, which can also grow at 45ºC. These organisms are intrinsically <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a> resistant, which was a reliable distinguishing trait prior to the emergence of vancomycin-resistant enterococci (VRE). These genera are now best distinguished from VRE biochemically by the addition of tests for production of pyrrolidonyl arylamidase (PYR) and leucine aminopeptidase (LAP). Expected results are: +/+ with enterococci; -/+ for <em>Pediococcus</em>; and -/- for <em>Leuconostoc</em>; <em>Leuconostoc </em>also produce gas from glucose [<a href="#rid8">8</a>].</p><p class="headingAnchor" id="H4"><span class="h2">Identification at the species level</span><span class="headingEndMark"> — </span>Prior to the emergence of <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a> resistance, the majority of clinical enterococcal isolates were <em>E. faecalis</em> (85 to 90 percent), but the incidence of <em>E. faecium</em> in the United States has increased (to about 32 percent of health care-associated enterococcal isolates), coincident with emergence of <a class="drug drug_general" data-topicid="8737" href="/z/d/drug information/8737.html" rel="external">ampicillin</a> and vancomycin resistance [<a href="#rid9">9</a>]. Although commercial kits for <em>E. faecalis</em> are generally reliable, commercial strips and kits for non–<em>E. faecalis</em> have been less reliable.</p><p>With the increase in non-faecalis (primarily <em>E. faecium</em>) and the increased attention paid to vancomycin-resistant organisms, including <em>E. gallinarum</em> and <em>E. casseliflavus</em>, there is renewed interest in identifying these organisms. One reason for this interest is related to species-specific antimicrobial resistance differences that may not be apparent from routine antibiograms.</p><p>For example, revision of the Clinical and Laboratory Standards Institute (CLSI) <a class="drug drug_general" data-topicid="9304" href="/z/d/drug information/9304.html" rel="external">daptomycin</a> breakpoints for enterococci introduced species specific cutoffs based on different dosing regimens for <em>E. faecium</em> and all other enterococcal species (see  <a class="medical medical_review" href="/z/d/html/3163.html" rel="external">"Treatment of enterococcal infections", section on 'Susceptibility breakpoints'</a>). Pharmacokinetic-pharmacodynamic modeling and clinical data suggest higher daptomycin exposures are associated with increased probability of target attainment and better clinical outcomes in <em>E. faecium</em> infections, while data for increased dosing regimens in other enterococcal species is less robust [<a href="#rid10">10</a>]. Thus, the use of daptomycin for severe enterococcal infections such as endocarditis would require identification of the infecting organism to the species level. (See  <a class="medical medical_review" href="/z/d/html/2152.html" rel="external">"Antimicrobial therapy of left-sided native valve endocarditis", section on 'Enterococci'</a>.)</p><p>Another reason for species identification is that the appearance of a non-faecalis, non-faecium species in different patients may be a clue to an outbreak secondary to nosocomial spread. Repeated isolation of an unusual species from a single patient may also serve as a clue to relapse from an underlying focus (eg, renal abscess, endocarditis) rather than reinfection by a new organism.</p><p>Enterococcal species were traditionally distinguished based upon results from a complex set of biochemical reactions, including fermentation of carbohydrates, hydrolysis of arginine, tolerance to tellurite, and motility and pigmentation:</p><p class="bulletIndent1"><span class="glyph">●</span>Tellurite tolerance with production of black colonies and utilization of pyruvate are characteristic of <em>E. faecalis</em>.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The primary motile species are <em>E. casseliflavus</em> (<em>E. flavescens</em>) and <em>E. gallinarum</em>; yellow pigment is produced by <em>E. casseliflavus</em> and <em>E. mundtii</em>.</p><p></p><p>Motility and pigment are particularly useful for distinguishing <em>E. casseliflavus</em> and <em>E. gallinarum</em> from <em>E. faecium</em>, which can be misidentified by kits that do not include these tests. However, occasional isolates display atypical reactions.</p><p>There are a number of difficulties with identification of enterococci to the species level. These include the necessity to perform multiple reactions, the analysis of which is best performed by computer application, and the difficulty in identifying enterococci with atypical phenotypic characteristics [<a href="#rid11">11</a>]. In many locations, genetic methods have largely replaced these biochemical tests for species identification. These include gene probes, polymerase chain reaction (PCR) for specific genes, 16S rRNA sequencing, proprietary multiplexed nucleic acid amplification, and matrix-assisted laser desorption/ionization–time of flight (MALDI-TOF) [<a href="#rid12">12-17</a>].</p><p>On a practical level, almost all <em>E. faecalis</em> isolates are susceptible to <a class="drug drug_general" data-topicid="8737" href="/z/d/drug information/8737.html" rel="external">ampicillin</a> but resistant to <a class="drug drug_general" data-topicid="9515" href="/z/d/drug information/9515.html" rel="external">quinupristin-dalfopristin</a>, whereas most clinical <em>E. faecium</em> isolates are resistant to ampicillin yet susceptible to quinupristin-dalfopristin.</p><p class="headingAnchor" id="H5"><span class="h1">ANTIBIOTIC RESISTANCE</span><span class="headingEndMark"> — </span>Enterococci come well equipped with a variety of intrinsic (ie, naturally occurring species characteristics) antibiotic resistances. They are also capable of acquiring new resistance genes and/or mutations, particularly after exposure to antibiotics. (See  <a class="medical medical_review" href="/z/d/html/470.html" rel="external">"Mechanisms of antibiotic resistance in enterococci"</a>.)</p><p>In Europe, <em>E. faecium </em>strains resistant to <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a> have been identified frequently in food animals, although vancomycin-resistant enterococci belonging to clones common in food animal feces rarely have been found to infect humans. <em>E. faecium</em> isolates resistant to vancomycin have only rarely been reported in animals in the United States; isolates resistant to other antibiotics, including <a class="drug drug_general" data-topicid="9515" href="/z/d/drug information/9515.html" rel="external">quinupristin-dalfopristin</a>, have been recovered from commercial poultry and meat products in the United States [<a href="#rid18">18,19</a>].</p><p>It is unclear whether the ingestion of these organisms leads to persistent intestinal carriage in humans. This issue was addressed in a study in which 12 healthy volunteers were fed glycopeptide and streptogramin-resistant <em>E. faecium</em> strains recovered from chickens or pork [<a href="#rid20">20</a>]. The strains were detectable in the stool of 8 of the 12 volunteers at one week, 1 of 12 at two weeks, and none at five weeks. This observation illustrates the potential for spread of these organisms as well as their resistance genes, which have the potential to transfer to the existing human enterococcal flora.</p><p class="headingAnchor" id="H6"><span class="h2">Nosocomial infection</span><span class="headingEndMark"> — </span>Enterococci are the second to third most common organism recovered from hospital-acquired (nosocomial) infections, particularly in more severely ill patients who have been hospitalized for long periods of time and/or have received multiple antibiotics [<a href="#rid6">6,9</a>]. (See  <a class="medical medical_review" href="/z/d/html/4045.html" rel="external">"Vancomycin-resistant enterococci: Epidemiology, prevention, and control"</a>.)</p><p>Until the appearance of <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a> resistance, <em>E. faecalis</em> accounted for 85 to 90 percent of enterococcal isolates and <em>E. faecium</em> for ≤10 percent. However, <em>E. faecium</em> has been implicated in up to 40 percent of bloodstream isolates in high-risk populations such as liver or stem cell transplant recipients and up to 35 percent of enterococcal isolates from nosocomial infections [<a href="#rid21">21,22</a>].</p><p>The rise of <em>E. faecium</em> in the hospital setting has been attributed to its many antimicrobial resistances, which allow these organisms to survive and proliferate when antibiotic-susceptible organisms are eliminated [<a href="#rid23">23</a>]. </p><p>Initially, most clinical isolates of multiresistant <em>E. faecium</em> appeared to belong to a single clonal complex (CC17) of related multilocus sequence types. Subsequently, two distinct clades of <em>E. faecium</em>, an animal associated A (which also has most healthcare-associated strains) and human commensal clade B, were identified using whole genome sequencing techniques. The core genes of these two groups differ by an average of 3 to 5 percent and a molecular clock analysis suggests they diverged from each other thousands of years ago, leading some to propose that these differences warrant classification of clade B as a different species, <em>E. lactis</em> [<a href="#rid10">10,24,25</a>]. The timing and frequency of the emergence of a subclade designated "A1" and whether the animal-associated clade ("A2") is a truly distinct group is still debated, as the structure of this subclade is dependent on both differences in geographic sampling and recombination events [<a href="#rid26">26,27</a>]. Regardless, clade A, particularly the subgroup A1, contains almost all the clinical and multidrug-resistant <em>E. faecium</em> strains [<a href="#rid28">28</a>]. </p><p>The population structure of <em>E. faecalis</em> is much more variable than that of <em>E. faecium</em>; however, several hospital-adapted lineages have been identified. In one study of 515 <em>E. faecalis</em> from the United Kingdom and United States, isolates belonging to one of three hospital-adapted lineages were enriched for multiple antibiotic resistance determinants, including <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a> resistance [<a href="#rid29">29</a>]. Further studies are needed to determine if these <em>E. faecalis</em> lineages reflect a worldwide phenomenon.</p><p>In addition to having more antibiotic resistance genes conferring resistance to <a class="drug drug_general" data-topicid="8737" href="/z/d/drug information/8737.html" rel="external">ampicillin</a> and high levels of <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a>, the hospital-associated clade of <em>E. faecium</em> also has genes that animal and community human fecal isolates do not have, including proven and putative virulence and colonization factors, and a much higher rate of mutation. In addition, many multidrug-resistant <em>E. faecalis</em> and <em>E. faecium</em> lack a functional CRISPR-cas9 system, which usually serves to protect bacteria from phages, but can also prevent the acquisition of resistance determinants on mobile genetic elements [<a href="#rid30">30</a>]. This suggests that the hospital-associated clade is not just more antibiotic resistant but also more capable of adapting to antibiotics, the environment, and possibly to the human immune response [<a href="#rid28">28,31-34</a>].</p><p>Enterococci can often persist in the face of antibiotics use and therefore have enhanced opportunity to cause opportunistic infection [<a href="#rid35">35</a>], particularly in patients with altered immune defenses and/or multiple potential portals of entry. Enterococci also have intrinsic factors such as a number of potential adhesin genes, which may explain the propensity of enterococci to cause endocarditis.</p><p>Once enterococci cause infection, their antibiotic resistances can create even greater dilemmas. Our worst therapeutic problems arise when a multidrug-resistant hospital-acquired <em>E. faecium</em> causes endocarditis. (See  <a class="medical medical_review" href="/z/d/html/3163.html" rel="external">"Treatment of enterococcal infections"</a>.)</p><p class="headingAnchor" id="H90634767"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>General – </strong>Enterococci (formerly called group D streptococci) are gram-positive cocci of intestinal origin that usually form short chains and are a common cause of endocarditis and nosocomial infections. (See <a class="local">'Introduction'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Microbiology classification </strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Microbiologic characteristics of enterococci include reactivity with group D antiserum in approximately 80 percent; ability to grow in high salt, at 10ºC and 40ºC, and at high pH; and survival at 60ºC for 30 minutes. They hydrolyze esculin in the presence of bile and produce pyrrolidonyl arylamidase (PYR). (See <a class="local">'Microbiology classification'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Species differentiation, traditionally based on results from a complex set of biochemical reactions but more recently on DNA-based testing, is important, as species-specific antimicrobial resistance differences can affect therapeutic decisions. The identification of non-faecium, non-faecalis species can be helpful in detecting nosocomial outbreaks or relapsing infection. (See <a class="local">'Identification at the species level'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Almost all <em>E. faecalis</em> isolates are susceptible to <a class="drug drug_general" data-topicid="8737" href="/z/d/drug information/8737.html" rel="external">ampicillin</a> but resistant to <a class="drug drug_general" data-topicid="9515" href="/z/d/drug information/9515.html" rel="external">quinupristin-dalfopristin</a>. Most clinical <em>E. faecium</em> isolates are resistant to ampicillin, susceptible to quinupristin-dalfopristin, and have moderately high-level resistance to <a class="drug drug_general" data-topicid="10000" href="/z/d/drug information/10000.html" rel="external">tobramycin</a>; in the United States, most clinical <em>E. faecium</em> are resistant to moderate to high levels of <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a>. <em>E. gallinarum</em> and <em>E. casseliflavus</em> are intrinsically resistant to low levels of vancomycin. (See <a class="local">'Identification at the species level'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Antibiotic resistance </strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Enterococci resistant to antibiotics have been recovered from commercial poultry and meat products in the United States, but it is unclear whether the ingestion of these organisms leads to persistent intestinal carriage in humans. (See <a class="local">'Antibiotic resistance'</a> above and  <a class="medical medical_review" href="/z/d/html/470.html" rel="external">"Mechanisms of antibiotic resistance in enterococci"</a>.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Their many antimicrobial resistances, which allow them to survive, proliferate, and fill the void when antibiotic-susceptible organisms are eliminated, likely contribute to the frequency of enterococci in nosocomial infections. (See <a class="local">'Nosocomial infection'</a> above.)</p><p></p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Murray BE. The life and times of the Enterococcus. Clin Microbiol Rev 1990; 3:46.</a></li><li><a class="nounderline abstract_t">Wade JJ. Enterococcus faecium in hospitals. Eur J Clin Microbiol Infect Dis 1997; 16:113.</a></li><li><a class="nounderline abstract_t">Østergaard L, Lauridsen TK, Iversen K, et al. Infective endocarditis in patients who have undergone transcatheter aortic valve implantation: a review. Clin Microbiol Infect 2020; 26:999.</a></li><li><a class="nounderline abstract_t">Khan A, Aslam A, Satti KN, Ashiq S. Infective endocarditis post-transcatheter aortic valve implantation (TAVI), microbiological profile and clinical outcomes: A systematic review. PLoS One 2020; 15:e0225077.</a></li><li><a class="nounderline abstract_t">Gross PA, Harkavy LM, Barden GE, Flower MF. The epidemiology of nosocomial enterococcal urinary tract infection. Am J Med Sci 1976; 272:75.</a></li><li><a class="nounderline abstract_t">Emori TG, Gaynes RP. An overview of nosocomial infections, including the role of the microbiology laboratory. Clin Microbiol Rev 1993; 6:428.</a></li><li><a class="nounderline abstract_t">Schleifer KH, Kilpper-Balz R. Transfer of Streptococcus faecalis and Streptococcus faecium to the genus Enterococcus nom. rev. as Enterococcus faecalis comb. nov. and Enterococcus faecium comb. Int J Syst Bacteriol 1984; 34:31.</a></li><li class="breakAll">Teixeira LM, Carvalho MS, Facklam RR. Enterococcus. In: Manual of Clinical Microbiology, Murray PR, Baron EJ, Jorgensen JH, LLandry ML, Pfaller MA (Eds), ASM Press, Washington, DC 2007.</li><li><a class="nounderline abstract_t">Weiner-Lastinger LM, Abner S, Edwards JR, et al. Antimicrobial-resistant pathogens associated with adult healthcare-associated infections: Summary of data reported to the National Healthcare Safety Network, 2015-2017. Infect Control Hosp Epidemiol 2020; 41:1.</a></li><li><a class="nounderline abstract_t">Satlin MJ, Nicolau DP, Humphries RM, et al. Development of Daptomycin Susceptibility Breakpoints for Enterococcus faecium and Revision of the Breakpoints for Other Enterococcal Species by the Clinical and Laboratory Standards Institute. Clin Infect Dis 2020; 70:1240.</a></li><li><a class="nounderline abstract_t">Teixeira LM, Facklam RR, Steigerwalt AG, et al. Correlation between phenotypic characteristics and DNA relatedness within Enterococcus faecium strains. J Clin Microbiol 1995; 33:1520.</a></li><li><a class="nounderline abstract_t">Coque TM, Murray BE. Identification of Enterococcus faecalis strains by DNA hybridization and pulsed-field gel electrophoresis. J Clin Microbiol 1995; 33:3368.</a></li><li><a class="nounderline abstract_t">Tyrrell GJ, Bethune RN, Willey B, Low DE. Species identification of enterococci via intergenic ribosomal PCR. J Clin Microbiol 1997; 35:1054.</a></li><li><a class="nounderline abstract_t">Depardieu F, Perichon B, Courvalin P. Detection of the van alphabet and identification of enterococci and staphylococci at the species level by multiplex PCR. J Clin Microbiol 2004; 42:5857.</a></li><li><a class="nounderline abstract_t">Martinez RM, Bauerle ER, Fang FC, Butler-Wu SM. Evaluation of three rapid diagnostic methods for direct identification of microorganisms in positive blood cultures. J Clin Microbiol 2014; 52:2521.</a></li><li><a class="nounderline abstract_t">Deck MK, Anderson ES, Buckner RJ, et al. Rapid detection of Enterococcus spp. direct from blood culture bottles using Enterococcus QuickFISH method: a multicenter investigation. Diagn Microbiol Infect Dis 2014; 78:338.</a></li><li><a class="nounderline abstract_t">Fang H, Ohlsson AK, Ullberg M, Ozenci V. Evaluation of species-specific PCR, Bruker MS, VITEK MS and the VITEK 2 system for the identification of clinical Enterococcus isolates. Eur J Clin Microbiol Infect Dis 2012; 31:3073.</a></li><li><a class="nounderline abstract_t">Hayes JR, McIntosh AC, Qaiyumi S, et al. High-frequency recovery of quinupristin-dalfopristin-resistant Enterococcus faecium isolates from the poultry production environment. J Clin Microbiol 2001; 39:2298.</a></li><li><a class="nounderline abstract_t">McDonald LC, Rossiter S, Mackinson C, et al. Quinupristin-dalfopristin-resistant Enterococcus faecium on chicken and in human stool specimens. N Engl J Med 2001; 345:1155.</a></li><li><a class="nounderline abstract_t">Sørensen TL, Blom M, Monnet DL, et al. Transient intestinal carriage after ingestion of antibiotic-resistant Enterococcus faecium from chicken and pork. N Engl J Med 2001; 345:1161.</a></li><li><a class="nounderline abstract_t">Mikulska M, Del Bono V, Raiola AM, et al. Blood stream infections in allogeneic hematopoietic stem cell transplant recipients: reemergence of Gram-negative rods and increasing antibiotic resistance. Biol Blood Marrow Transplant 2009; 15:47.</a></li><li><a class="nounderline abstract_t">Bedini A, Codeluppi M, Cocchi S, et al. Gram-positive bloodstream infections in liver transplant recipients: incidence, risk factors, and impact on survival. Transplant Proc 2007; 39:1947.</a></li><li><a class="nounderline abstract_t">Suppola JP, Volin L, Valtonen VV, Vaara M. Overgrowth of Enterococcus faecium in the feces of patients with hematologic malignancies. Clin Infect Dis 1996; 23:694.</a></li><li><a class="nounderline abstract_t">Galloway-Peña J, Roh JH, Latorre M, et al. Genomic and SNP analyses demonstrate a distant separation of the hospital and community-associated clades of Enterococcus faecium. PLoS One 2012; 7:e30187.</a></li><li><a class="nounderline abstract_t">Palmer KL, Godfrey P, Griggs A, et al. Comparative genomics of enterococci: variation in Enterococcus faecalis, clade structure in E. faecium, and defining characteristics of E. gallinarum and E. casseliflavus. MBio 2012; 3:e00318.</a></li><li><a class="nounderline abstract_t">Rios R, Reyes J, Carvajal LP, et al. Genomic Epidemiology of Vancomycin-Resistant Enterococcus faecium (VREfm) in Latin America: Revisiting The Global VRE Population Structure. Sci Rep 2020; 10:5636.</a></li><li><a class="nounderline abstract_t">van Hal SJ, Willems RJL, Gouliouris T, et al. The interplay between community and hospital Enterococcus faecium clones within health-care settings: a genomic analysis. Lancet Microbe 2022; 3:e133.</a></li><li><a class="nounderline abstract_t">Lebreton F, van Schaik W, McGuire AM, et al. Emergence of epidemic multidrug-resistant Enterococcus faecium from animal and commensal strains. MBio 2013; 4.</a></li><li><a class="nounderline abstract_t">Raven KE, Reuter S, Gouliouris T, et al. Genome-based characterization of hospital-adapted Enterococcus faecalis lineages. Nat Microbiol 2016; 1:15033.</a></li><li><a class="nounderline abstract_t">Palmer KL, Gilmore MS. Multidrug-resistant enterococci lack CRISPR-cas. mBio 2010; 1.</a></li><li><a class="nounderline abstract_t">Rice LB, Carias L, Rudin S, et al. A potential virulence gene, hylEfm, predominates in Enterococcus faecium of clinical origin. J Infect Dis 2003; 187:508.</a></li><li><a class="nounderline abstract_t">Nallapareddy SR, Weinstock GM, Murray BE. Clinical isolates of Enterococcus faecium exhibit strain-specific collagen binding mediated by Acm, a new member of the MSCRAMM family. Mol Microbiol 2003; 47:1733.</a></li><li><a class="nounderline abstract_t">Leavis HL, Bonten MJ, Willems RJ. Identification of high-risk enterococcal clonal complexes: global dispersion and antibiotic resistance. Curr Opin Microbiol 2006; 9:454.</a></li><li><a class="nounderline abstract_t">Zhang X, Top J, de Been M, et al. Identification of a genetic determinant in clinical Enterococcus faecium strains that contributes to intestinal colonization during antibiotic treatment. J Infect Dis 2013; 207:1780.</a></li><li><a class="nounderline abstract_t">Sillanpää J, Prakash VP, Nallapareddy SR, Murray BE. Distribution of genes encoding MSCRAMMs and Pili in clinical and natural populations of Enterococcus faecium. J Clin Microbiol 2009; 47:896.</a></li></ol></div><div id="topicVersionRevision">Topic 3160 Version 21.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2404568" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : The life and times of the Enterococcus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9105837" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Enterococcus faecium in hospitals.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32036048" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Infective endocarditis in patients who have undergone transcatheter aortic valve implantation: a review.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31951610" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Infective endocarditis post-transcatheter aortic valve implantation (TAVI), microbiological profile and clinical outcomes: A systematic review.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/822713" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : The epidemiology of nosocomial enterococcal urinary tract infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8269394" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : An overview of nosocomial infections, including the role of the microbiology laboratory.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Transfer of Streptococcus faecalis and Streptococcus faecium to the genus Enterococcus nom. rev. as Enterococcus faecalis comb. nov. and Enterococcus faecium comb</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Transfer of Streptococcus faecalis and Streptococcus faecium to the genus Enterococcus nom. rev. as Enterococcus faecalis comb. nov. and Enterococcus faecium comb</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31767041" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Antimicrobial-resistant pathogens associated with adult healthcare-associated infections: Summary of data reported to the National Healthcare Safety Network, 2015-2017.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31504338" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Development of Daptomycin Susceptibility Breakpoints for Enterococcus faecium and Revision of the Breakpoints for Other Enterococcal Species by the Clinical and Laboratory Standards Institute.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7650178" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Correlation between phenotypic characteristics and DNA relatedness within Enterococcus faecium strains.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8586745" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Identification of Enterococcus faecalis strains by DNA hybridization and pulsed-field gel electrophoresis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9114380" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Species identification of enterococci via intergenic ribosomal PCR.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15583325" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Detection of the van alphabet and identification of enterococci and staphylococci at the species level by multiplex PCR.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24808235" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Evaluation of three rapid diagnostic methods for direct identification of microorganisms in positive blood cultures.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24439447" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Rapid detection of Enterococcus spp. direct from blood culture bottles using Enterococcus QuickFISH method: a multicenter investigation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22706514" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Evaluation of species-specific PCR, Bruker MS, VITEK MS and the VITEK 2 system for the identification of clinical Enterococcus isolates.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11376075" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : High-frequency recovery of quinupristin-dalfopristin-resistant Enterococcus faecium isolates from the poultry production environment.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11642231" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Quinupristin-dalfopristin-resistant Enterococcus faecium on chicken and in human stool specimens.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11642232" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Transient intestinal carriage after ingestion of antibiotic-resistant Enterococcus faecium from chicken and pork.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19135942" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Blood stream infections in allogeneic hematopoietic stem cell transplant recipients: reemergence of Gram-negative rods and increasing antibiotic resistance.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17692662" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Gram-positive bloodstream infections in liver transplant recipients: incidence, risk factors, and impact on survival.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8909828" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Overgrowth of Enterococcus faecium in the feces of patients with hematologic malignancies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22291916" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Genomic and SNP analyses demonstrate a distant separation of the hospital and community-associated clades of Enterococcus faecium.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22354958" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Comparative genomics of enterococci: variation in Enterococcus faecalis, clade structure in E. faecium, and defining characteristics of E. gallinarum and E. casseliflavus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32221315" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Genomic Epidemiology of Vancomycin-Resistant Enterococcus faecium (VREfm) in Latin America: Revisiting The Global VRE Population Structure.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/35146465" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : The interplay between community and hospital Enterococcus faecium clones within health-care settings: a genomic analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23963180" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Emergence of epidemic multidrug-resistant Enterococcus faecium from animal and commensal strains.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27572164" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Genome-based characterization of hospital-adapted Enterococcus faecalis lineages.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21060735" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Multidrug-resistant enterococci lack CRISPR-cas.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12552437" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : A potential virulence gene, hylEfm, predominates in Enterococcus faecium of clinical origin.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12622825" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Clinical isolates of Enterococcus faecium exhibit strain-specific collagen binding mediated by Acm, a new member of the MSCRAMM family.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16880002" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Identification of high-risk enterococcal clonal complexes: global dispersion and antibiotic resistance.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23447698" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Identification of a genetic determinant in clinical Enterococcus faecium strains that contributes to intestinal colonization during antibiotic treatment.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19193843" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Distribution of genes encoding MSCRAMMs and Pili in clinical and natural populations of Enterococcus faecium.</p>
</a>
</div>
</div>
</div>
</div>
</section>
</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->
<!-- End Footer Area -->
<!-- Back to top -->
<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->
<!-- Prpper JS -->
<!-- Bootstrap Min JS -->
<!-- Classy Nav Min Js -->
<!-- Owl Carousel Min Js -->
<!-- Magnific Popup JS -->
<!-- CounterUp JS -->
<!-- Waypoints JS -->
<!-- Form Validator Min JS -->
<!-- Contact Form Min JS -->
<!-- Main JS -->
<!-- Global site tag (gtag.js) - Google Analytics -->
</body>
</html>
