<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Clinical significance and molecular characteristics of common hepatitis B virus variants</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Clinical significance and molecular characteristics of common hepatitis B virus variants</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Clinical significance and molecular characteristics of common hepatitis B virus variants</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Anna SF Lok, MD</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Rafael Esteban, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jennifer Mitty, MD, MPH</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Jan 08, 2024.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Viral DNA and especially RNA genomes are inherently variable due to errors introduced during replication. These errors are in the range of 1 mutation in 1000 to 100,000 nucleotides per replication cycle for RNA viruses, and approximately 1 mutation in 100,000,000 nucleotides per replication cycle for DNA viruses [<a href="#rid1">1,2</a>]. The higher mutation rate among RNA viruses is related to the lack of proofreading functions of RNA polymerases and reverse transcriptases. As a result, mutations in viral genomic sequences are generated naturally during viral replication and should be viewed as a normal biological event.</p><p>Hepatitis B virus (HBV), a member of the Hepadnaviridae family, replicates asymmetrically via reverse transcription of an RNA intermediate, making it prone to mutations [<a href="#rid3">3</a>]. The estimated mutation rate of the hepadnavirus genome is about 2 x 10<sup>4</sup> base substitutions/site/year, about 100 times higher than that of other DNA viruses but about 100 to 1000 times lower than that of other RNA viruses [<a href="#rid4">4</a>]. Although mutations can occur randomly along the HBV genome, the overlapping open reading frames limit the number and location of viable mutations. Because chronic HBV infection frequently persists for decades, many variants may exist within the same host at any given time and each variant may have multiple base changes.</p><p>Selection by the host for the fittest variant occurs after the random mutation process. Fitness may be defined at the cellular level (viruses that replicate most efficiently in a cell type will predominate) or at the extra-cellular level (viruses that avoid immune elimination will become dominant) [<a href="#rid5">5</a>]. Thus, the accumulation of viral variants depends upon the rate at which the variants are generated and the advantage they confer to the virus.</p><p>The genetically different viral species concomitantly present in a single cell or in a single individual are termed quasi-species [<a href="#rid1">1</a>]. Interpretation of the clinical significance of HBV mutations is complicated by the lack of standardized nomenclature, differences in sensitivities of assays used in their detection, and the presence of mutations in more than one region of the HBV genome even in the same species [<a href="#rid6">6,7</a>]. Despite these difficulties, in vivo analyses of naturally occurring viral variants and in vitro mutagenesis studies have identified some mutations that have a role in viral latency, pathogenesis of liver disease, immune escape, and resistance to antiviral therapy. This topic review will summarize the clinical relevance and molecular characteristics of common HBV variants.</p><p class="headingAnchor" id="H2"><span class="h1">PRECORE AND CORE PROMOTER VARIANTS</span><span class="headingEndMark"> — </span>The precore/core region of the HBV genome encodes the nucleocapsid protein (HBcAg) and hepatitis B e-antigen (HBeAg) [<a href="#rid8">8,9</a>]. The core open reading frame has two transcripts with heterogeneous 5' ends and two in-phase initiation codons. HBeAg is translated from the precore mRNA, producing a precursor polypeptide comprising the precore and the entire core region. The precore polypeptide is translocated into the endoplasmic reticulum by a signal peptide. Cleavage of the amino and carboxy termini results in a secretory protein, HBeAg. HBcAg is translated from the pregenomic RNA. Both HBcAg and HBeAg contain B- and T-cell epitopes, of which some are shared. (See  <a class="medical medical_review" href="/d/html/3631.html" rel="external">"Characteristics of the hepatitis B virus and pathogenesis of infection"</a>.)</p><p>Secretion of HBeAg can be modulated at both transcriptional and translational levels. Transcription of the precore and pregenomic RNAs is under the control of regulatory elements including the basal core promoter and the core upstream regulatory sequence [<a href="#rid10">10</a>]. Mutations in these domains decrease precore mRNA transcription and HBeAg synthesis [<a href="#rid11">11,12</a>]. At the translational level, mutations within the precore region can block the translation of the precore protein and HBeAg production. The most common mutation in the precore region is a point mutation (G to A switch at nucleotide 1896), which changes codon 28 from tryptophan to a stop [<a href="#rid13">13-15</a>]. Because this mutation occurs upstream of the core start codon and because HBcAg is translated from a different transcript, HBcAg and nucleocapsid production is not interrupted.</p><p>The biological role of HBeAg in the HBV replication cycle is uncertain. Expression of HBeAg is nonessential for virus replication in animal models [<a href="#rid16">16</a>] and in humans [<a href="#rid17">17</a>]. HBeAg may act as a tolerogen or a target for immune response. In utero exposure to HBeAg can induce immune tolerance in newborn mice [<a href="#rid18">18</a>]. Perinatal transmission of HBV from HBeAg-positive mothers results in chronic HBV infection in the majority of babies [<a href="#rid19">19</a>]. In addition, HBeAg appears to modulate the host's immune response [<a href="#rid20">20-23</a>]. Precore variants that do not produce HBeAg may be selected because they can evade immune clearance [<a href="#rid24">24</a>].</p><p class="headingAnchor" id="H3"><span class="h2">Precore variants</span><span class="headingEndMark"> — </span>The most common precore variant has a point mutation at nucleotide 1896, creating a stop at codon 28 (G1896A, eW28X). This mutation is accompanied by another point mutation (G to A) at nucleotide 1899 in some patients. In vitro studies have demonstrated that precore protein can inhibit HBV replication, suggesting that precore stop codon variants may enhance viral replication [<a href="#rid25">25,26</a>]. However, patients with HBeAg-negative chronic hepatitis tend to have lower HBV DNA levels compared to those with HBeAg-positive chronic hepatitis [<a href="#rid15">15,27</a>].</p><p>Other naturally occurring mutations in the precore region include point mutations leading to initiation failure as well as deletions and insertions of nucleotides inducing frameshifts. In addition to the eW28X stop codon mutation, the most frequent precore mutations that can abrogate HBeAg production are point mutations in the start codon [<a href="#rid28">28,29</a>].</p><p>In a study involving 17 centers in the United States, precore variants were found in 27 percent of patients with chronic HBV infection, being significantly more common among the HBeAg-negative than the HBeAg-positive patients (38 versus 9 percent) [<a href="#rid30">30</a>]. A more recent study of 1036 patients (808 adults and 228 children) in the Hepatitis B Research Network found that the eW28X stop codon mutation was present as the dominant sequence in 29.4 percent of patients, and the prevalence increased with age; there was a higher prevalence in HBeAg-negative than in HBeAg-positive patients [<a href="#rid31">31</a>].</p><p class="headingAnchor" id="H4"><span class="h2">Core promoter variants</span><span class="headingEndMark"> — </span>The most common core promoter variant involves a dual mutation A1762T, G1764A (TA change). A number of mutations have been found to occur together with the TA change, including changes at nucleotide 1653, 1753 to 1757, and the dual mutation C1766T G1768A. The latter mutations are most often found in patients with fulminant hepatitis [<a href="#rid32">32</a>].</p><p>Deletions within the basal core promoter, which overlap with the X gene, have been found to occur in asymptomatic carriers, patients with chronic hepatitis B or hepatocellular carcinoma (HCC), and individuals with no HBV serologic markers [<a href="#rid33">33-35</a>]. Infection with these deletion variants is usually associated with low HBV DNA levels.</p><p>In the nationwide United States study, the dual core promoter variants were found in 44 percent of patients with chronic HBV infection, being significantly more common among the HBeAg-negative than the HBeAg-positive patients (51 versus 36 percent) [<a href="#rid30">30</a>].</p><p class="headingAnchor" id="H5"><span class="h2">Detection</span><span class="headingEndMark"> — </span>In addition to direct sequencing, several methods can be used to detect precore and core promoter variants including restriction fragment length polymorphism (RFLP) [<a href="#rid36">36,37</a>], line probe assay [<a href="#rid38">38,39</a>], ligase chain reaction [<a href="#rid40">40</a>], and colorimetric point mutation assay [<a href="#rid41">41</a>]. Some methods such as line probe assay and RFLP are more sensitive in detecting minor variants than direct sequencing [<a href="#rid39">39</a>].</p><p>The controversies surrounding the clinical significance of precore and core promoter variants is related to the heterogeneity of the study population, sensitivity of the methods used to detect these variants, and to the format in which results are reported. Some studies, particularly those that rely on less sensitive methods such as direct sequencing, only report the presence of variants when the variant is present as the predominant sequence, while other studies report the presence of variants when the variant is detected as a minor sequence in a mixed population.</p><p class="headingAnchor" id="H6"><span class="h2">Relationship to HBeAg status</span><span class="headingEndMark"> — </span>Precore variants have been detected mainly in patients who are HBeAg negative but they can also be found in some HBeAg-positive patients [<a href="#rid14">14,15,30,42</a>]. HBeAg-positive patients with precore variants are more likely to undergo spontaneous HBeAg seroconversion and do so earlier than those with wild-type precore sequence [<a href="#rid43">43,44</a>]. These findings are in accordance with the observation that precore variants that prevent HBeAg production are selected around the time of HBeAg clearance [<a href="#rid14">14,15,42</a>]. However, not all HBeAg seroconversion requires the selection of precore mutations.</p><p>Unlike precore variants, core promoter variants can be detected in similar proportions of HBeAg-negative and HBeAg-positive patients [<a href="#rid30">30,45,46</a>]. One study demonstrated that emergence of the dual core promoter (TA) change was more common among HBeAg-positive patients who subsequently seroconverted compared to those who remained HBeAg positive, indicating that the selection of core promoter variants may play a role in HBeAg clearance [<a href="#rid47">47</a>]. This study also found that core promoter variants could be detected earlier than precore variants in patients who cleared HBeAg.</p><p>In a study from the Hepatitis B Research Network, which included 1036 participants (808 adults, 228 children) in North America who were not receiving antiviral therapy, the prevalence of the precore G1896A variant as the dominant sequence was more likely in HBeAg-negative versus HBeAg-positive participants (47.5 and 7.4 percent, respectively) [<a href="#rid31">31</a>]. The prevalence of the core promoter TA variant was also more likely in HBeAg-negative compared with HBeAg-positive participants (42.1 and 24.9 percent, respectively). Detection of these variants in HBeAg-positive patients appears to be an indicator of impending HBeAg clearance as HBeAg clearance rates were significantly higher in HBeAg-positive participants with dominant precore or core promoter variants (24.4 and 15.0 per 100 person-years) compared to those with wild-type sequence (6.0 per 100 person-years).</p><p class="headingAnchor" id="H7"><span class="h2">Relationship to HBeAg-negative chronic hepatitis B</span><span class="headingEndMark"> — </span>Seroconversion from HBeAg to anti-HBe is usually accompanied by decreased levels of HBV replication and remission of liver disease. However, some patients who become HBeAg negative continue to have high HBV DNA levels and active liver disease, so-called HBeAg-negative chronic hepatitis B (HBeAg-CHB). Many of these patients harbor HBV variants in the precore or core promoter region [<a href="#rid13">13-15,42,48,49</a>], although some continue to have low levels of wild-type HBV. (See <a class="local">'Precore variants'</a> above and <a class="local">'Core promoter variants'</a> above.)</p><p>Patients have HBeAg-CHB if they meet the following criteria [<a href="#rid50">50</a>]: </p><p class="bulletIndent1"><span class="glyph">●</span>HBsAg positive and HBeAg negative</p><p class="bulletIndent1"><span class="glyph">●</span>Serum HBV DNA &gt;10<sup>4</sup> international units/mL</p><p class="bulletIndent1"><span class="glyph">●</span>Persistent or intermittent elevations in aminotransferase levels that cannot be attributed to other causes</p><p></p><p>There are clinical differences between patients with HBeAg-negative CHB and those who are HBeAg positive [<a href="#rid51">51</a>]. Patients with HBeAg-CHB tend to have lower serum HBV DNA levels and are more likely to have fluctuating serum aminotransferase levels [<a href="#rid15">15,27,51</a>]. Patients with HBeAg-CHB also tend to be older and have more advanced liver disease because they are in a later phase of chronic HBV infection. </p><p>Patients with HBeAg-negative CHB have been reported in all parts of the world but are more common in Mediterranean countries and Asia [<a href="#rid46">46,52-55</a>]. Such patients are estimated to account for 7 to 30 percent of those with chronic hepatitis B worldwide, based upon studies from the 1990s [<a href="#rid56">56</a>]. However, the exact prevalence of HBeAg-negative CHB is not known since large population-based studies have not been performed. The ratio of HBeAg-negative CHB to HBeAg-positive CHB has increased worldwide because of the decline in new cases of HBV infection secondary to implementation of HBV vaccination and aging of previously infected persons. In many countries including Asian countries, HBeAg-negative CHB is now more prevalent than HBeAg-positive CHB. </p><p>The geographical differences in the relative frequency of HBeAg-negative CHB and HBeAg-positive CHB are partly related to differences in the distribution of HBV genotypes [<a href="#rid57">57-59</a>]. (See  <a class="medical medical_review" href="/d/html/3644.html" rel="external">"Clinical significance of hepatitis B virus genotypes"</a>.) </p><p>As examples:</p><p class="bulletIndent1"><span class="glyph">●</span>The most common HBV variant associated with HBeAg-negative CHB is the precore stop codon variant G1896A, which is usually found in association with the HBV genotype D [<a href="#rid52">52</a>]. This genotype is prevalent in the Mediterranean basin. In some countries (eg, Italy and Greece) where HBV genotype D is dominant, &gt;90 percent of patients with chronic HBV infection are HBeAg negative. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>HBeAg-negative chronic hepatitis B (HBeAg-CHB) is rarely detected in association with genotype A (which is prevalent in the United States and North-West Europe). </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Studies in Hong Kong found that only 40 percent of patients with HBeAg-negative CHB had the precore variant. HBV genotypes B and C prevail in these areas [<a href="#rid46">46</a>].</p><p></p><p class="headingAnchor" id="H8"><span class="h2">Clinical outcomes</span><span class="headingEndMark"> — </span>Initial studies suggested that precore and core promoter variants were associated with more severe liver disease and HCC [<a href="#rid13">13,15,60-63</a>]. However, these variants have also been detected in inactive carriers and patients with minimal liver disease [<a href="#rid41">41,45,53,57</a>]. Furthermore, precore and core promoter variants had no impact on pre- and post-liver transplant outcomes in a study from the United States [<a href="#rid64">64</a>]. It is possible that host immune response and mutations in other regions of the HBV genome may be involved in the pathogenesis of liver disease [<a href="#rid65">65</a>].</p><p>There are increasing data to support an association between core promoter variants and more severe liver disease. A study from Hong Kong found patients with core promoter mutation tended to have higher hepatic necroinflammation scores [<a href="#rid66">66</a>]. The same group reported the prevalence of core promoter mutations was higher among chronic hepatitis B patients who developed complications of cirrhosis and HCC [<a href="#rid67">67,68</a>].</p><p>Another study from Taiwan involving 250 chronic hepatitis B-infected patients found that the prevalence of core promoter variants increased from 3 percent among inactive carriers to 64 percent among patients with HCC [<a href="#rid69">69</a>]. When controlling for other potential confounders such as sex, age, genotype, and presence of cirrhosis, the core promoter variants (A1762T, G1764A) were still significantly associated with the development of HCC (odds ratio [OR] 10.6). A longitudinal study from Taiwan demonstrated that persistence of core promoter mutations accumulated over time before the diagnosis of HCC [<a href="#rid70">70</a>]. Another study from the United States reported that patients with core promoter variants were more likely to have hepatic decompensation [<a href="#rid71">71</a>].</p><p>The major drawback of the above studies is that the conclusions were based on cross-sectional analyses. As a result, large-scale longitudinal studies with consideration of other host and viral confounders are needed to confirm the relationship between core promoter variants and advanced liver disease. Several studies have confirmed an association between basal core promoter mutations and hepatocellular carcinoma; in a few studies, there has been an association with cirrhosis as well [<a href="#rid72">72-75</a>]. One study included 2762 Taiwanese patients with chronic HBV followed between 1991 and 2004 (for a total of 33,747 person years), during which a total of 153 incident cases of HCC were detected [<a href="#rid76">76</a>]. After adjusting for baseline HBV DNA levels, sex, and age of recruitment, the basal core promoter variant A1762T/G1764A and genotype C were predictors of increased risk of HCC, while the precore variant G1896A was associated with a lower risk of HCC [<a href="#rid73">73</a>].</p><p class="headingAnchor" id="H9"><span class="h2">Fulminant hepatitis</span><span class="headingEndMark"> — </span>Precore and core promoter variants have been described in association with fulminant hepatitis [<a href="#rid7">7,77-80</a>]. It has been suggested that these variants result in a fulminant course because of enhanced HBV replication or a more aggressive immune response. However, in some studies, these variants were also detected in asymptomatic source patients who led to the outbreak of fulminant hepatitis B [<a href="#rid81">81</a>]. The mechanism by which precore variants cause fulminant hepatitis in the new host but an inactive liver disease in the original host is unclear. It is possible that de novo infection with precore variants elicits a more aggressive immune response while precore variants that are selected during the course of infection may escape immune clearance and are associated with less active disease.</p><p class="headingAnchor" id="H10"><span class="h2">Serum quantitative HBV DNA levels</span><span class="headingEndMark"> — </span>There is a paucity of data on the relationship between precore/core promoter variants and serum quantitative HBV DNA levels. The most comprehensive study was done in the United States and involved a total of 694 consecutive chronic HBV-infected patients [<a href="#rid71">71</a>]. HBeAg-positive patients with core promoter variant had significantly lower serum HBV DNA levels compared to those with wild-type sequence in the core promoter region. A possible explanation was the selection of core promoter variants during the process of immune clearance. By contrast, precore and/or core promoter variants were associated with higher serum HBV DNA levels than wild-type sequence in HBeAg-negative patients, which may be because precore/core promoter variants can evade the host immune response and can thus maintain higher levels of HBV replication. This observation has been confirmed by subsequent studies [<a href="#rid31">31,82,83</a>].</p><p class="headingAnchor" id="H11"><span class="h2">Interferon therapy</span><span class="headingEndMark"> — </span>Several studies have found that interferon (IFN) therapy is associated with a low rate of sustained response among patients with HBeAg-negative CHB [<a href="#rid84">84,85</a>]. These studies suggested that a longer duration of treatment (&gt;12 months) was necessary to decrease the rate of post-treatment relapse [<a href="#rid86">86</a>].</p><p>Relatively few studies have evaluated the relationship between precore and core promoter mutations and response to IFN therapy and results have been conflicting [<a href="#rid43">43,44,87-90</a>].</p><p class="bulletIndent1"><span class="glyph">●</span>HBeAg-positive patients: Some studies [<a href="#rid43">43,44</a>] found that patients who were HBeAg positive with precore variant were more likely to achieve HBeAg loss/seroconversion with IFN therapy while disparate results have been reported by others [<a href="#rid87">87-89,91,92</a>]. Whether the conflicting results could be accounted for by differences in HBV genotypes or baseline HBV DNA levels is unknown. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>HBeAg-negative patients: One study from Italy showed that patients with exclusively or predominantly precore variant had a higher rate of relapse after initial response to IFN [<a href="#rid84">84</a>]. A study from Germany found that responders to IFN therapy had significantly fewer mutations in the basic core promoter region compared with nonresponders [<a href="#rid88">88</a>].</p><p></p><p class="headingAnchor" id="H12"><span class="h2">Nucleoside analog therapy</span><span class="headingEndMark"> — </span>Most patients with HBeAg-negative CHB relapse after one year of nucleoside analog treatment (see appropriate topic reviews). As with IFN, longer duration of treatment is needed to maintain viral suppression.</p><p>There is very little information on the correlation between precore and core promoter mutations and response or resistance during nucleoside analog therapy. Clinical studies of <a class="drug drug_general" data-topicid="9544" href="/d/drug information/9544.html" rel="external">lamivudine</a>, <a class="drug drug_general" data-topicid="8690" href="/d/drug information/8690.html" rel="external">adefovir</a>, <a class="drug drug_general" data-topicid="8816" href="/d/drug information/8816.html" rel="external">entecavir</a>, <a class="drug drug_general" data-topicid="9488" href="/d/drug information/9488.html" rel="external">telbivudine</a>, and tenofovir show that nucleoside analogs have similar potency in HBeAg-positive and in HBeAg-negative patients. (See appropriate topic reviews.)</p><p class="headingAnchor" id="H13"><span class="h2">Summary</span><span class="headingEndMark"> — </span>Interpretation of the biologic and pathogenic significance of precore and core promoter variants should be made cautiously because many factors such as HBV genotype, viral quasispecies, mutations in other regions of the HBV genome, host immune response, and sensitivity of detection methods can influence the results. Testing precore and core promoter variants especially in patients suspected to have HBeAg-negative CHB can help further characterize the patient but these tests are not routinely indicated for clinical management because the precise role of these variants in predicting the severity of liver disease or response to antiviral therapy remain unclear.</p><p class="headingAnchor" id="H14"><span class="h1">PRE-S/S VARIANTS</span><span class="headingEndMark"> — </span>The major B-cell neutralizing epitopes of HBsAg reside in the "a" determinant region located at amino acid positions 124 to 149 [<a href="#rid93">93,94</a>]. Antibodies to the "a" determinant confer protection against all serotypes of HBV [<a href="#rid95">95</a>]. The pre-S and S regions of the HBV genome also contain putative HLA class I-restricted cytotoxic T lymphocyte (CTL) epitopes [<a href="#rid96">96</a>]. Mutations involving T- or B-cell epitopes in the pre-S and S regions may result in immune escape mutants [<a href="#rid97">97</a>]. In addition, mutations in the pre-S region may affect the secretion of small S protein. Clinically significant pre-S/S variants will be discussed below.</p><p class="headingAnchor" id="H15"><span class="h2">Vaccine-escape mutants</span><span class="headingEndMark"> — </span>HBV S gene mutants have been described in infants who were infected with hepatitis B despite an adequate anti-HBs response to hepatitis B vaccination. These mutations have been observed in many parts of the world, including Italy, Singapore, Taiwan, Japan, China, and Africa. The most common mutation involves glycine to arginine substitution at codon 145 (G145R) in the "a" determinant of HBsAg. In vitro studies confirmed that HBsAg with G145R mutation had decreased binding to hepatitis B surface antibodies [<a href="#rid98">98</a>]. Other mutations involving amino acids 126, 129, and 141 have also been reported among vaccinees but the clinical significance of these mutants is not well documented.</p><p>Most reports found that the vaccine-related HBV S mutants were not detected in the maternal carriers, suggesting that the mutations were selected by vaccine and/or <a class="drug drug_general" data-topicid="8520" href="/d/drug information/8520.html" rel="external">hepatitis B immune globulin</a> (HBIG). However, a few reports suggested that these mutants were transmitted from the mothers to the babies. One study reported horizontal transmission of G145R mutation to intrafamilial members [<a href="#rid99">99</a>].</p><p>A study from Taiwan [<a href="#rid100">100</a>] found that the prevalence of HBV S mutants among HBsAg-positive children increased after the introduction of universal vaccination, from 7.8 percent in 1984 to 25 percent in 1994, and about 23 percent in 1999. However, the carrier rate among Taiwanese children decreased from 9.8 to 0.7 percent in the same period. A subsequent update that included data from 2009 found no increase in breakthrough HBV infection of HBV S mutants over the 25-year period since the initiation of universal vaccination in Taiwan [<a href="#rid101">101</a>]. These data suggest the emergence of vaccine escape mutants is rare and has not diminished the efficacy of HBV vaccines. (See  <a class="medical medical_review" href="/d/html/3641.html" rel="external">"Hepatitis B virus immunization in adults"</a>.)</p><p class="headingAnchor" id="H16"><span class="h2">Immunoprophylaxis failure in orthotopic liver transplant recipients</span><span class="headingEndMark"> — </span>HBV reinfection in patients receiving maintenance HBIG immunoprophylaxis may be due to inadequate dosing or to the emergence of mutant virus that can escape neutralization. Several studies reported the detection of S mutants in patients who developed recurrent hepatitis B after liver transplantation despite HBIG prophylaxis. As with vaccine failure, G145R is the most common mutation. One study found significant correlation between these mutants and the duration of HBIG therapy and reversion of the mutation to wild-type sequence after withdrawal of HBIG. The use of nucleos(t)ide analogs with potent antiviral activity and a high barrier to resistance prior to and after liver transplant has greatly reduced the need for HBIG, the rate of HBV reinfection post-liver transplant, and the emergence of HBV S mutants. (See  <a class="medical medical_review" href="/d/html/4584.html" rel="external">"Liver transplantation in adults: Preventing hepatitis B virus infection in liver transplant recipients"</a>.)</p><p class="headingAnchor" id="H17"><span class="h2">Viral persistence in HBsAg-negative patients</span><span class="headingEndMark"> — </span>Loss of HBsAg usually indicates clearance of HBV. However, low levels of HBV DNA can be detected in serum and liver from some HBsAg-negative individuals using PCR assays [<a href="#rid102">102</a>]. In most instances of occult HBV infection (HBsAg negative but HBV DNA positive by PCR), HBV DNA is present in very low concentrations with sub-detectable amounts of HBsAg. However, in rare instances, mutations in the HBV genome down-regulate S gene expression or produce aberrant surface proteins. As a result, HBsAg is not detectable by conventional serologic assays [<a href="#rid103">103-106</a>]. Occult HBV infection may have an impact on blood transfusion, organ transplantation, diagnosis of cryptogenic liver disease, and hepatocellular carcinoma (HCC) development [<a href="#rid107">107,108</a>]. (See  <a class="medical medical_review" href="/d/html/3680.html" rel="external">"Hepatitis B virus: Screening and diagnosis in adults"</a>.)</p><p class="headingAnchor" id="H18"><span class="h1">POLYMERASE GENE MUTATIONS</span><span class="headingEndMark"> — </span>The HBV DNA polymerase gene consists of four distinct regions [<a href="#rid109">109</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>A primer involved in priming of reverse transcription</p><p class="bulletIndent1"><span class="glyph">●</span>A spacer with no known function</p><p class="bulletIndent1"><span class="glyph">●</span>A reverse transcriptase/DNA polymerase which is responsible for reverse transcription of the pregenomic RNA into the first (-) strand HBV DNA and for synthesis of the second (+) strand HBV DNA</p><p class="bulletIndent1"><span class="glyph">●</span>An RNAse H, which removes the RNA template</p><p></p><p>The reverse transcriptase/DNA polymerase region has five conserved domains: A, B, C, D, E. Domains A, C, and D are involved in nucleoside triphosphate binding and catalysis, while domains B and E participate in the positioning of the RNA template and the primer, relative to the catalytic site [<a href="#rid110">110,111</a>]. The putative catalytic domain is believed to reside in the tyrosine-methionine-aspartate-aspartate (YMDD) locus in domain C. This locus is conserved in all viral reverse transcriptases as well as in all isolates of hepadnaviruses [<a href="#rid109">109</a>]. Polymerase gene mutations that are associated with antiviral resistance can arise spontaneously but are rarely detected unless they have been selected by exposure to oral nucleos(t)ide analogs. (See  <a class="medical medical_review" href="/d/html/3642.html" rel="external">"Hepatitis B virus: Overview of management", section on 'Nucleos(t)ide analogs'</a>.)</p><p class="headingAnchor" id="H19"><span class="h1">X GENE VARIANTS</span><span class="headingEndMark"> — </span>The X region is crucial for the replication and expression of HBV because the X protein regulates HBV replication through activating and regulating viral and cellular genes [<a href="#rid112">112,113</a>]. The X region overlaps the core promoter, enhancer II, and two direct repeats that are involved in regulating precore and pregenomic mRNA transcription. HBx protein is believed to be important in hepatic carcinogenesis. Several studies reported that mutations affecting codons 130 and 131 of the X gene are more common in patients with hepatocellular carcinoma (HCC) than in patients with chronic hepatitis B [<a href="#rid62">62,63,114</a>]. These mutations correspond to the dual core promoter mutations A1762T and G1764A.</p><p>Although many studies have shown a convincing relationship between core promoter variants and HCC [<a href="#rid68">68-70</a>], it is unclear if core promoter variant is directly involved in the process of carcinogenesis or the increased risk of HCC is related to more active and rapidly progressive liver disease. Because of the association between core promoter variants and HBV genotype C, and more active hepatic necroinflammation, careful analysis is needed to determine if core promoter mutation is independently related to HCC development. A prospective cohort study from Taiwan found that core promoter but not precore mutation was associated with an increased risk of HCC and the association was independent of HBV genotype [<a href="#rid76">76</a>]. However, more studies are needed to clarify these associations. Whether X gene mutations that are unrelated to the common dual core promoter mutation increase the risk of HCC also needs further clarifications.</p><p class="headingAnchor" id="H97394206"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Impact of variants</strong> – Because chronic hepatitis B virus (HBV) infection frequently persists for decades, many variants may exist within the same host at any given time, and each variant may have multiple base changes. Some mutations have a role in viral latency, pathogenesis of liver disease, immune escape, and resistance to antiviral therapy. (See <a class="local">'Introduction'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Types of variants</strong> </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Precore/core region variants</strong> – The precore/core region of the HBV genome encodes the nucleocapsid protein (HBcAg) and hepatitis B e-antigen (HBeAg). Initial studies suggested that precore and core promoter variants were associated with more severe liver disease. However, these variants have also been detected in inactive carriers and patients with minimal liver disease. There are increasing data to support an association between core promoter variants and more severe liver disease, including hepatocellular carcinoma (HCC). (See <a class="local">'Precore and core promoter variants'</a> above.) </p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>The pre-S and S region variants</strong> – The pre-S and S regions of the HBV genome contain putative HLA class I-restricted cytotoxic T lymphocyte (CTL) epitope. Several studies reported the detection of S mutants in patients who developed recurrent hepatitis B after liver transplantation despite <a class="drug drug_general" data-topicid="8520" href="/d/drug information/8520.html" rel="external">hepatitis B immune globulin</a> (HBIG) prophylaxis, or in children born to HBsAg carrier mothers who developed HBV infection despite receiving HBIG and HBV vaccination. (See <a class="local">'Immunoprophylaxis failure in orthotopic liver transplant recipients'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Polymerase gene mutations</strong> – Polymerase gene mutations can lead to development of resistance against antiviral agents. They can arise spontaneously but are rarely detected unless they have been selected by exposure to oral nucleos(t)ide analogs. (See <a class="local">'Polymerase gene mutations'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>X gene variants</strong> – The X region is crucial for the replication and expression of HBV because the X protein regulates HBV replication through activating and regulating viral and cellular genes. Several studies reported that mutations affecting codons 130 and 131 of the X gene are more common in patients with HCC than in patients without HCC. These X gene mutations correspond to core promoter mutations that had been reported to be associated with an increased risk of HCC. (See <a class="local">'X gene variants'</a> above.)</p><p></p><p><strong>ACKNOWLEDGMENT</strong> — The UpToDate editorial staff acknowledges Dr. Chi-Jen Chu, who contributed to an earlier version of this topic review.</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Carman W, Thomas H, Domingo E. Viral genetic variation: hepatitis B virus as a clinical example. Lancet 1993; 341:349.</a></li><li><a class="nounderline abstract_t">Domingo E, Sabo D, Taniguchi T, Weissmann C. Nucleotide sequence heterogeneity of an RNA phage population. Cell 1978; 13:735.</a></li><li><a class="nounderline abstract_t">Summers J, Mason WS. Replication of the genome of a hepatitis B--like virus by reverse transcription of an RNA intermediate. Cell 1982; 29:403.</a></li><li><a class="nounderline abstract_t">Girones R, Miller RH. Mutation rate of the hepadnavirus genome. Virology 1989; 170:595.</a></li><li><a class="nounderline abstract_t">Oldstone MB. Molecular anatomy of viral persistence. J Virol 1991; 65:6381.</a></li><li><a class="nounderline abstract_t">Blum HE, Liang TJ, Galun E, Wands JR. Persistence of hepatitis B viral DNA after serological recovery from hepatitis B virus infection. Hepatology 1991; 14:56.</a></li><li><a class="nounderline abstract_t">Hasegawa K, Huang J, Rogers SA, et al. Enhanced replication of a hepatitis B virus mutant associated with an epidemic of fulminant hepatitis. J Virol 1994; 68:1651.</a></li><li><a class="nounderline abstract_t">Uy A, Bruss V, Gerlich WH, et al. Precore sequence of hepatitis B virus inducing e antigen and membrane association of the viral core protein. Virology 1986; 155:89.</a></li><li><a class="nounderline abstract_t">Ou JH, Laub O, Rutter WJ. Hepatitis B virus gene function: the precore region targets the core antigen to cellular membranes and causes the secretion of the e antigen. Proc Natl Acad Sci U S A 1986; 83:1578.</a></li><li><a class="nounderline abstract_t">Yuh CH, Chang YL, Ting LP. Transcriptional regulation of precore and pregenomic RNAs of hepatitis B virus. J Virol 1992; 66:4073.</a></li><li><a class="nounderline abstract_t">Buckwold VE, Xu Z, Chen M, et al. Effects of a naturally occurring mutation in the hepatitis B virus basal core promoter on precore gene expression and viral replication. J Virol 1996; 70:5845.</a></li><li><a class="nounderline abstract_t">Scaglioni PP, Melegari M, Wands JR. Biologic properties of hepatitis B viral genomes with mutations in the precore promoter and precore open reading frame. Virology 1997; 233:374.</a></li><li><a class="nounderline abstract_t">Carman WF, Jacyna MR, Hadziyannis S, et al. Mutation preventing formation of hepatitis B e antigen in patients with chronic hepatitis B infection. Lancet 1989; 2:588.</a></li><li><a class="nounderline abstract_t">Akahane Y, Yamanaka T, Suzuki H, et al. Chronic active hepatitis with hepatitis B virus DNA and antibody against e antigen in the serum. Disturbed synthesis and secretion of e antigen from hepatocytes due to a point mutation in the precore region. Gastroenterology 1990; 99:1113.</a></li><li><a class="nounderline abstract_t">Brunetto MR, Giarin MM, Oliveri F, et al. Wild-type and e antigen-minus hepatitis B viruses and course of chronic hepatitis. Proc Natl Acad Sci U S A 1991; 88:4186.</a></li><li><a class="nounderline abstract_t">Ganem D, Varmus HE. The molecular biology of the hepatitis B viruses. Annu Rev Biochem 1987; 56:651.</a></li><li><a class="nounderline abstract_t">Tong SP, Li JS, Vitvitski L, Trépo C. Replication capacities of natural and artificial precore stop codon mutants of hepatitis B virus: relevance of pregenome encapsidation signal. Virology 1992; 191:237.</a></li><li><a class="nounderline abstract_t">Milich DR, Jones JE, Hughes JL, et al. Is a function of the secreted hepatitis B e antigen to induce immunologic tolerance in utero? Proc Natl Acad Sci U S A 1990; 87:6599.</a></li><li><a class="nounderline abstract_t">Raimondo G, Tanzi E, Brancatelli S, et al. Is the course of perinatal hepatitis B virus infection influenced by genetic heterogeneity of the virus? J Med Virol 1993; 40:87.</a></li><li><a class="nounderline abstract_t">Twu JS, Schloemer RH. Transcription of the human beta interferon gene is inhibited by hepatitis B virus. J Virol 1989; 63:3065.</a></li><li><a class="nounderline abstract_t">Milich DR, Chen MK, Hughes JL, Jones JE. The secreted hepatitis B precore antigen can modulate the immune response to the nucleocapsid: a mechanism for persistence. J Immunol 1998; 160:2013.</a></li><li><a class="nounderline abstract_t">Jung MC, Diepolder HM, Spengler U, et al. Activation of a heterogeneous hepatitis B (HB) core and e antigen-specific CD4+ T-cell population during seroconversion to anti-HBe and anti-HBs in hepatitis B virus infection. J Virol 1995; 69:3358.</a></li><li><a class="nounderline abstract_t">Ferrari C, Penna A, Bertoletti A, et al. Cellular immune response to hepatitis B virus-encoded antigens in acute and chronic hepatitis B virus infection. J Immunol 1990; 145:3442.</a></li><li><a class="nounderline abstract_t">Frelin L, Wahlström T, Tucker AE, et al. A mechanism to explain the selection of the hepatitis e antigen-negative mutant during chronic hepatitis B virus infection. J Virol 2009; 83:1379.</a></li><li><a class="nounderline abstract_t">Guidotti LG, Matzke B, Pasquinelli C, et al. The hepatitis B virus (HBV) precore protein inhibits HBV replication in transgenic mice. J Virol 1996; 70:7056.</a></li><li><a class="nounderline abstract_t">Scaglioni PP, Melegari M, Wands JR. Posttranscriptional regulation of hepatitis B virus replication by the precore protein. J Virol 1997; 71:345.</a></li><li><a class="nounderline abstract_t">Hadziyannis S. Hepatitis E antigen negative chronic hepatitis B: From clinical recognition to pathogenesis and treatment. Viral Hepatitis Rev 1995; 1:7.</a></li><li><a class="nounderline abstract_t">Santantonio T, Jung MC, Miska S, et al. Prevalence and type of pre-C HBV mutants in anti-HBe positive carriers with chronic liver disease in a highly endemic area. Virology 1991; 183:840.</a></li><li><a class="nounderline abstract_t">Okamoto H, Yotsumoto S, Akahane Y, et al. Hepatitis B viruses with precore region defects prevail in persistently infected hosts along with seroconversion to the antibody against e antigen. J Virol 1990; 64:1298.</a></li><li><a class="nounderline abstract_t">Chu CJ, Keeffe EB, Han SH, et al. Hepatitis B virus genotypes in the United States: results of a nationwide study. Gastroenterology 2003; 125:444.</a></li><li><a class="nounderline abstract_t">Lau DTY, Ganova-Raeva L, Wang J, et al. Precore and Basal Core Promoter Hepatitis B Virus (HBV) Variants Are Present From a Young Age and Differ Across HBV Genotypes. Hepatology 2021; 73:1637.</a></li><li><a class="nounderline abstract_t">Ogata N, Miller RH, Ishak KG, Purcell RH. The complete nucleotide sequence of a pre-core mutant of hepatitis B virus implicated in fulminant hepatitis and its biological characterization in chimpanzees. Virology 1993; 194:263.</a></li><li><a class="nounderline abstract_t">Laskus T, Rakela J, Tong MJ, et al. Naturally occurring hepatitis B virus mutants with deletions in the core promoter region. J Hepatol 1994; 20:837.</a></li><li><a class="nounderline abstract_t">Fukuda R, Nguyen XT, Ishimura N, et al. X gene and precore region mutations in the hepatitis B virus genome in persons positive for antibody to hepatitis B e antigen: comparison between asymptomatic "healthy" carriers and patients with severe chronic active hepatitis. J Infect Dis 1995; 172:1191.</a></li><li><a class="nounderline abstract_t">Uchida T, Shimojima M, Gotoh K, et al. "Silent" hepatitis B virus mutants are responsible for non-A, non-B, non-C, non-D, non-E hepatitis. Microbiol Immunol 1994; 38:281.</a></li><li><a class="nounderline abstract_t">Lindh M, Furuta Y, Ljunggren KK, et al. Detection of hepatitis B virus precore TAG mutant by an amplification-created restriction site method. J Infect Dis 1995; 171:194.</a></li><li><a class="nounderline abstract_t">Takahashi K, Ohta Y, Kanai K, et al. Clinical implications of mutations C-to-T1653 and T-to-C/A/G1753 of hepatitis B virus genotype C genome in chronic liver disease. Arch Virol 1999; 144:1299.</a></li><li><a class="nounderline abstract_t">Grandjacques C, Pradat P, Stuyver L, et al. Rapid detection of genotypes and mutations in the pre-core promoter and the pre-core region of hepatitis B virus genome: correlation with viral persistence and disease severity. J Hepatol 2000; 33:430.</a></li><li><a class="nounderline abstract_t">Hussain M, Chu CJ, Sablon E, Lok AS. Rapid and sensitive assays for determination of hepatitis B virus (HBV) genotypes and detection of HBV precore and core promoter variants. J Clin Microbiol 2003; 41:3699.</a></li><li><a class="nounderline abstract_t">Minamitani S, Nishiguchi S, Kuroki T, et al. Detection by ligase chain reaction of precore mutant of hepatitis B virus. Hepatology 1997; 25:216.</a></li><li><a class="nounderline abstract_t">Ballard AL, Boxall EH. Colourimetric point mutation assay: for detection of precore mutants of hepatitis B. J Virol Methods 1997; 67:143.</a></li><li><a class="nounderline abstract_t">Okamoto H, Tsuda F, Akahane Y, et al. Hepatitis B virus with mutations in the core promoter for an e antigen-negative phenotype in carriers with antibody to e antigen. J Virol 1994; 68:8102.</a></li><li><a class="nounderline abstract_t">Lok AS, Akarca US, Greene S. Predictive value of precore hepatitis B virus mutations in spontaneous and interferon-induced hepatitis B e antigen clearance. Hepatology 1995; 21:19.</a></li><li><a class="nounderline abstract_t">Chu CM, Yeh CT, Lee CS, et al. Precore stop mutant in HBeAg-positive patients with chronic hepatitis B: clinical characteristics and correlation with the course of HBeAg-to-anti-HBe seroconversion. J Clin Microbiol 2002; 40:16.</a></li><li><a class="nounderline abstract_t">Kidd-Ljunggren K, Oberg M, Kidd AH. Hepatitis B virus X gene 1751 to 1764 mutations: implications for HBeAg status and disease. J Gen Virol 1997; 78 ( Pt 6):1469.</a></li><li><a class="nounderline abstract_t">Chan HL, Leung NW, Hussain M, et al. Hepatitis B e antigen-negative chronic hepatitis B in Hong Kong. Hepatology 2000; 31:763.</a></li><li><a class="nounderline abstract_t">Chan HL, Hussain M, Lok AS. Different hepatitis B virus genotypes are associated with different mutations in the core promoter and precore regions during hepatitis B e antigen seroconversion. Hepatology 1999; 29:976.</a></li><li><a class="nounderline abstract_t">Takahashi K, Aoyama K, Ohno N, et al. The precore/core promoter mutant (T1762A1764) of hepatitis B virus: clinical significance and an easy method for detection. J Gen Virol 1995; 76 ( Pt 12):3159.</a></li><li><a class="nounderline abstract_t">Kurosaki M, Enomoto N, Asahina Y, et al. Mutations in the core promoter region of hepatitis B virus in patients with chronic hepatitis B. J Med Virol 1996; 49:115.</a></li><li><a class="nounderline abstract_t">Lok AS, Heathcote EJ, Hoofnagle JH. Management of hepatitis B: 2000--summary of a workshop. Gastroenterology 2001; 120:1828.</a></li><li><a class="nounderline abstract_t">Zarski JP, Marcellin P, Cohard M, et al. Comparison of anti-HBe-positive and HBe-antigen-positive chronic hepatitis B in France. French Multicentre Group. J Hepatol 1994; 20:636.</a></li><li><a class="nounderline abstract_t">Brunetto MR, Oliveri F, Rocca G, et al. Natural course and response to interferon of chronic hepatitis B accompanied by antibody to hepatitis B e antigen. Hepatology 1989; 10:198.</a></li><li><a class="nounderline abstract_t">Fattovich G, Brollo L, Alberti A, et al. Long-term follow-up of anti-HBe-positive chronic active hepatitis B. Hepatology 1988; 8:1651.</a></li><li><a class="nounderline abstract_t">Naoumov NV, Schneider R, Grötzinger T, et al. Precore mutant hepatitis B virus infection and liver disease. Gastroenterology 1992; 102:538.</a></li><li><a class="nounderline abstract_t">Shindo M, Hamada K, Koya S, et al. The clinical significance of core promoter and precore mutations during the natural course and interferon therapy in patients with chronic hepatitis B. Am J Gastroenterol 1999; 94:2237.</a></li><li><a class="nounderline abstract_t">Schalm SW, Thomas HC, Hadziyannis SJ. Chronic hepatitis B. Prog Liver Dis 1990; 9:443.</a></li><li><a class="nounderline abstract_t">Lindh M, Andersson AS, Gusdal A. Genotypes, nt 1858 variants, and geographic origin of hepatitis B virus--large-scale analysis using a new genotyping method. J Infect Dis 1997; 175:1285.</a></li><li><a class="nounderline abstract_t">Lok AS, Akarca U, Greene S. Mutations in the pre-core region of hepatitis B virus serve to enhance the stability of the secondary structure of the pre-genome encapsidation signal. Proc Natl Acad Sci U S A 1994; 91:4077.</a></li><li><a class="nounderline abstract_t">Rodriguez-Frias F, Buti M, Jardi R, et al. Hepatitis B virus infection: precore mutants and its relation to viral genotypes and core mutations. Hepatology 1995; 22:1641.</a></li><li><a class="nounderline abstract_t">Lindh M, Hannoun C, Dhillon AP, et al. Core promoter mutations and genotypes in relation to viral replication and liver damage in East Asian hepatitis B virus carriers. J Infect Dis 1999; 179:775.</a></li><li><a class="nounderline abstract_t">Orito E, Mizokami M, Sakugawa H, et al. A case-control study for clinical and molecular biological differences between hepatitis B viruses of genotypes B and C. Japan HBV Genotype Research Group. Hepatology 2001; 33:218.</a></li><li><a class="nounderline abstract_t">Fang ZL, Ling R, Wang SS, et al. HBV core promoter mutations prevail in patients with hepatocellular carcinoma from Guangxi, China. J Med Virol 1998; 56:18.</a></li><li><a class="nounderline abstract_t">Baptista M, Kramvis A, Kew MC. High prevalence of 1762(T) 1764(A) mutations in the basic core promoter of hepatitis B virus isolated from black Africans with hepatocellular carcinoma compared with asymptomatic carriers. Hepatology 1999; 29:946.</a></li><li><a class="nounderline abstract_t">Gaglio P, Singh S, Degertekin B, et al. Impact of the hepatitis B virus genotype on pre- and post-liver transplantation outcomes. Liver Transpl 2008; 14:1420.</a></li><li><a class="nounderline abstract_t">Bonino F, Brunetto MR. Hepatitis B virus heterogeneity, one of many factors influencing the severity of hepatitis B. J Hepatol 1993; 18:5.</a></li><li><a class="nounderline abstract_t">Yuen MF, Tanaka Y, Ng IO, et al. Hepatic necroinflammation and fibrosis in patients with genotypes Ba and C, core-promoter and precore mutations. J Viral Hepat 2005; 12:513.</a></li><li><a class="nounderline abstract_t">Yuen MF, Sablon E, Yuan HJ, et al. Relationship between the development of precore and core promoter mutations and hepatitis B e antigen seroconversion in patients with chronic hepatitis B virus. J Infect Dis 2002; 186:1335.</a></li><li><a class="nounderline abstract_t">Yuen MF, Tanaka Y, Mizokami M, et al. Role of hepatitis B virus genotypes Ba and C, core promoter and precore mutations on hepatocellular carcinoma: a case control study. Carcinogenesis 2004; 25:1593.</a></li><li><a class="nounderline abstract_t">Kao JH, Chen PJ, Lai MY, Chen DS. Basal core promoter mutations of hepatitis B virus increase the risk of hepatocellular carcinoma in hepatitis B carriers. Gastroenterology 2003; 124:327.</a></li><li><a class="nounderline abstract_t">Chou YC, Yu MW, Wu CF, et al. Temporal relationship between hepatitis B virus enhancer II/basal core promoter sequence variation and risk of hepatocellular carcinoma. Gut 2008; 57:91.</a></li><li><a class="nounderline abstract_t">Chu CJ, Keeffe EB, Han SH, et al. Prevalence of HBV precore/core promoter variants in the United States. Hepatology 2003; 38:619.</a></li><li><a class="nounderline abstract_t">Tseng TC, Liu CJ, Yang HC, et al. Higher proportion of viral basal core promoter mutant increases the risk of liver cirrhosis in hepatitis B carriers. Gut 2015; 64:292.</a></li><li><a class="nounderline abstract_t">Qu LS, Zhu J, Liu TT, et al. Effect of combined mutations in the enhancer II and basal core promoter of hepatitis B virus on development of hepatocellular carcinoma in Qidong, China. Hepatol Res 2014; 44:1186.</a></li><li><a class="nounderline abstract_t">Lyu H, Lee D, Chung YH, et al. Synergistic effects of A1896, T1653 and T1762/A1764 mutations in genotype c2 hepatitis B virus on development of hepatocellular carcinoma. J Viral Hepat 2013; 20:219.</a></li><li><a class="nounderline abstract_t">Yin J, Xie J, Liu S, et al. Association between the various mutations in viral core promoter region to different stages of hepatitis B, ranging of asymptomatic carrier state to hepatocellular carcinoma. Am J Gastroenterol 2011; 106:81.</a></li><li><a class="nounderline abstract_t">Yang HI, Yeh SH, Chen PJ, et al. Associations between hepatitis B virus genotype and mutants and the risk of hepatocellular carcinoma. J Natl Cancer Inst 2008; 100:1134.</a></li><li><a class="nounderline abstract_t">Sato S, Suzuki K, Akahane Y, et al. Hepatitis B virus strains with mutations in the core promoter in patients with fulminant hepatitis. Ann Intern Med 1995; 122:241.</a></li><li><a class="nounderline abstract_t">Kosaka Y, Takase K, Kojima M, et al. Fulminant hepatitis B: induction by hepatitis B virus mutants defective in the precore region and incapable of encoding e antigen. Gastroenterology 1991; 100:1087.</a></li><li><a class="nounderline abstract_t">Omata M, Ehata T, Yokosuka O, et al. Mutations in the precore region of hepatitis B virus DNA in patients with fulminant and severe hepatitis. N Engl J Med 1991; 324:1699.</a></li><li><a class="nounderline abstract_t">Liang TJ, Hasegawa K, Rimon N, et al. A hepatitis B virus mutant associated with an epidemic of fulminant hepatitis. N Engl J Med 1991; 324:1705.</a></li><li><a class="nounderline abstract_t">Karayiannis P, Alexopoulou A, Hadziyannis S, et al. Fulminant hepatitis associated with hepatitis B virus e antigen-negative infection: importance of host factors. Hepatology 1995; 22:1628.</a></li><li><a class="nounderline abstract_t">Jardi R, Rodriguez F, Buti M, et al. Mutations in the basic core promoter region of hepatitis B virus. Relationship with precore variants and HBV genotypes in a Spanish population of HBV carriers. J Hepatol 2004; 40:507.</a></li><li><a class="nounderline abstract_t">Peng XM, Huang GM, Li JG, et al. High level of hepatitis B virus DNA after HBeAg-to-anti-HBe seroconversion is related to coexistence of mutations in its precore and basal core promoter. World J Gastroenterol 2005; 11:3131.</a></li><li><a class="nounderline abstract_t">Brunetto MR, Giarin M, Saracco G, et al. Hepatitis B virus unable to secrete e antigen and response to interferon in chronic hepatitis B. Gastroenterology 1993; 105:845.</a></li><li><a class="nounderline abstract_t">Oliveri F, Santantonio T, Bellati G, et al. Long term response to therapy of chronic anti-HBe-positive hepatitis B is poor independent of type and schedule of interferon. Am J Gastroenterol 1999; 94:1366.</a></li><li><a class="nounderline abstract_t">Lampertico P, Del Ninno E, Manzin A, et al. A randomized, controlled trial of a 24-month course of interferon alfa 2b in patients with chronic hepatitis B who had hepatitis B virus DNA without hepatitis B e antigen in serum. Hepatology 1997; 26:1621.</a></li><li><a class="nounderline abstract_t">Fattovich G, McIntyre G, Thursz M, et al. Hepatitis B virus precore/core variation and interferon therapy. Hepatology 1995; 22:1355.</a></li><li><a class="nounderline abstract_t">Erhardt A, Reineke U, Blondin D, et al. Mutations of the core promoter and response to interferon treatment in chronic replicative hepatitis B. Hepatology 2000; 31:716.</a></li><li><a class="nounderline abstract_t">Kao JH, Wu NH, Chen PJ, et al. Hepatitis B genotypes and the response to interferon therapy. J Hepatol 2000; 33:998.</a></li><li><a class="nounderline abstract_t">Marrone A, Zampino R, Luongo G, et al. Low HBeAg serum levels correlate with the presence of the double A1762T/G1764A core promoter mutation and a positive response to interferon in patients with chronic hepatitis B virus infection. Intervirology 2003; 46:222.</a></li><li><a class="nounderline abstract_t">Sonneveld MJ, Zoutendijk R, Hansen BE, Janssen HL. Pegylated interferon results in higher serological, but not virological, response rates when compared to continuous entecavir. Antivir Ther 2012; 17:1605.</a></li><li><a class="nounderline abstract_t">Sonneveld MJ, Rijckborst V, Zeuzem S, et al. Presence of precore and core promoter mutants limits the probability of response to peginterferon in hepatitis B e antigen-positive chronic hepatitis B. Hepatology 2012; 56:67.</a></li><li><a class="nounderline abstract_t">Bhatnagar PK, Papas E, Blum HE, et al. Immune response to synthetic peptide analogues of hepatitis B surface antigen specific for the a determinant. Proc Natl Acad Sci U S A 1982; 79:4400.</a></li><li><a class="nounderline abstract_t">Brown SE, Stanley C, Howard CR, et al. Antibody responses to recombinant and plasma derived hepatitis B vaccines. Br Med J (Clin Res Ed) 1986; 292:159.</a></li><li><a class="nounderline abstract_t">Szmuness W, Stevens CE, Harley EJ, et al. Hepatitis B vaccine in medical staff of hemodialysis units: efficacy and subtype cross-protection. N Engl J Med 1982; 307:1481.</a></li><li><a class="nounderline abstract_t">Chisari FV, Ferrari C. Hepatitis B virus immunopathogenesis. Annu Rev Immunol 1995; 13:29.</a></li><li><a class="nounderline abstract_t">Pircher H, Moskophidis D, Rohrer U, et al. Viral escape by selection of cytotoxic T cell-resistant virus variants in vivo. Nature 1990; 346:629.</a></li><li><a class="nounderline abstract_t">Waters JA, Kennedy M, Voet P, et al. Loss of the common "A" determinant of hepatitis B surface antigen by a vaccine-induced escape mutant. J Clin Invest 1992; 90:2543.</a></li><li><a class="nounderline abstract_t">Oon CJ, Chen WN, Goo KS, Goh KT. Intra-familial evidence of horizontal transmission of hepatitis B virus surface antigen mutant G145R. J Infect 2000; 41:260.</a></li><li><a class="nounderline abstract_t">Hsu HY, Chang MH, Ni YH, Chen HL. Survey of hepatitis B surface variant infection in children 15 years after a nationwide vaccination programme in Taiwan. Gut 2004; 53:1499.</a></li><li><a class="nounderline abstract_t">Hsu HY, Chang MH, Ni YH, et al. Chronologic changes in serum hepatitis B virus DNA, genotypes, surface antigen mutants and reverse transcriptase mutants during 25-year nationwide immunization in Taiwan. J Viral Hepat 2017; 24:645.</a></li><li><a class="nounderline abstract_t">Bréchot C, Thiers V, Kremsdorf D, et al. Persistent hepatitis B virus infection in subjects without hepatitis B surface antigen: clinically significant or purely "occult"? Hepatology 2001; 34:194.</a></li><li><a class="nounderline abstract_t">Carman WF, Van Deursen FJ, Mimms LT, et al. The prevalence of surface antigen variants of hepatitis B virus in Papua New Guinea, South Africa, and Sardinia. Hepatology 1997; 26:1658.</a></li><li><a class="nounderline abstract_t">Grethe S, Monazahian M, Böhme I, Thomssen R. Characterization of unusual escape variants of hepatitis B virus isolated from a hepatitis B surface antigen-negative subject. J Virol 1998; 72:7692.</a></li><li><a class="nounderline abstract_t">Hou J, Karayiannis P, Waters J, et al. A unique insertion in the S gene of surface antigen--negative hepatitis B virus Chinese carriers. Hepatology 1995; 21:273.</a></li><li><a class="nounderline abstract_t">Kato J, Hasegawa K, Torii N, et al. A molecular analysis of viral persistence in surface antigen-negative chronic hepatitis B. Hepatology 1996; 23:389.</a></li><li><a class="nounderline abstract_t">Conjeevaram HS, Lok AS. Occult hepatitis B virus infection: a hidden menace? Hepatology 2001; 34:204.</a></li><li><a class="nounderline abstract_t">Raimondo G, Locarnini S, Pollicino T, et al. Update of the statements on biology and clinical impact of occult hepatitis B virus infection. J Hepatol 2019; 71:397.</a></li><li><a class="nounderline abstract_t">Poch O, Sauvaget I, Delarue M, Tordo N. Identification of four conserved motifs among the RNA-dependent polymerase encoding elements. EMBO J 1989; 8:3867.</a></li><li><a class="nounderline abstract_t">Johnson MS, McClure MA, Feng DF, et al. Computer analysis of retroviral pol genes: assignment of enzymatic functions to specific sequences and homologies with nonviral enzymes. Proc Natl Acad Sci U S A 1986; 83:7648.</a></li><li><a class="nounderline abstract_t">Kohlstaedt LA, Wang J, Friedman JM, et al. Crystal structure at 3.5 A resolution of HIV-1 reverse transcriptase complexed with an inhibitor. Science 1992; 256:1783.</a></li><li><a class="nounderline abstract_t">Rossner MT. Review: hepatitis B virus X-gene product: a promiscuous transcriptional activator. J Med Virol 1992; 36:101.</a></li><li><a class="nounderline abstract_t">Yoo YD, Ueda H, Park K, et al. Regulation of transforming growth factor-beta 1 expression by the hepatitis B virus (HBV) X transactivator. Role in HBV pathogenesis. J Clin Invest 1996; 97:388.</a></li><li><a class="nounderline abstract_t">Hsia CC, Yuwen H, Tabor E. Hot-spot mutations in hepatitis B virus X gene in hepatocellular carcinoma. Lancet 1996; 348:625.</a></li></ol></div><div id="topicVersionRevision">Topic 3639 Version 20.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8094122" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Viral genetic variation: hepatitis B virus as a clinical example.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/657273" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Nucleotide sequence heterogeneity of an RNA phage population.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/6180831" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Replication of the genome of a hepatitis B--like virus by reverse transcription of an RNA intermediate.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2728351" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Mutation rate of the hepadnavirus genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1719227" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Molecular anatomy of viral persistence.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2066074" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Persistence of hepatitis B viral DNA after serological recovery from hepatitis B virus infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8107226" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Enhanced replication of a hepatitis B virus mutant associated with an epidemic of fulminant hepatitis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3535236" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Precore sequence of hepatitis B virus inducing e antigen and membrane association of the viral core protein.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3006057" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Hepatitis B virus gene function: the precore region targets the core antigen to cellular membranes and causes the secretion of the e antigen.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1602534" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Transcriptional regulation of precore and pregenomic RNAs of hepatitis B virus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8709203" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Effects of a naturally occurring mutation in the hepatitis B virus basal core promoter on precore gene expression and viral replication.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9217060" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Biologic properties of hepatitis B viral genomes with mutations in the precore promoter and precore open reading frame.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2570285" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Mutation preventing formation of hepatitis B e antigen in patients with chronic hepatitis B infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2394332" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Chronic active hepatitis with hepatitis B virus DNA and antibody against e antigen in the serum. Disturbed synthesis and secretion of e antigen from hepatocytes due to a point mutation in the precore region.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2034663" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Wild-type and e antigen-minus hepatitis B viruses and course of chronic hepatitis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3039907" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : The molecular biology of the hepatitis B viruses.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1413504" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Replication capacities of natural and artificial precore stop codon mutants of hepatitis B virus: relevance of pregenome encapsidation signal.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2395863" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Is a function of the secreted hepatitis B e antigen to induce immunologic tolerance in utero?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8360636" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Is the course of perinatal hepatitis B virus infection influenced by genetic heterogeneity of the virus?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2542615" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Transcription of the human beta interferon gene is inhibited by hepatitis B virus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9469465" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : The secreted hepatitis B precore antigen can modulate the immune response to the nucleocapsid: a mechanism for persistence.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7538172" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Activation of a heterogeneous hepatitis B (HB) core and e antigen-specific CD4+ T-cell population during seroconversion to anti-HBe and anti-HBs in hepatitis B virus infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2230128" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Cellular immune response to hepatitis B virus-encoded antigens in acute and chronic hepatitis B virus infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19004949" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : A mechanism to explain the selection of the hepatitis e antigen-negative mutant during chronic hepatitis B virus infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8794350" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : The hepatitis B virus (HBV) precore protein inhibits HBV replication in transgenic mice.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8985356" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Posttranscriptional regulation of hepatitis B virus replication by the precore protein.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Hepatitis E antigen negative chronic hepatitis B: From clinical recognition to pathogenesis and treatment</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1853582" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Prevalence and type of pre-C HBV mutants in anti-HBe positive carriers with chronic liver disease in a highly endemic area.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2304145" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Hepatitis B viruses with precore region defects prevail in persistently infected hosts along with seroconversion to the antibody against e antigen.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12891547" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Hepatitis B virus genotypes in the United States: results of a nationwide study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32860463" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Precore and Basal Core Promoter Hepatitis B Virus (HBV) Variants Are Present From a Young Age and Differ Across HBV Genotypes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8480422" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : The complete nucleotide sequence of a pre-core mutant of hepatitis B virus implicated in fulminant hepatitis and its biological characterization in chimpanzees.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7930487" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Naturally occurring hepatitis B virus mutants with deletions in the core promoter region.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7594653" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : X gene and precore region mutations in the hepatitis B virus genome in persons positive for antibody to hepatitis B e antigen: comparison between asymptomatic "healthy" carriers and patients with severe chronic active hepatitis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7935046" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : "Silent" hepatitis B virus mutants are responsible for non-A, non-B, non-C, non-D, non-E hepatitis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7798663" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Detection of hepatitis B virus precore TAG mutant by an amplification-created restriction site method.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10481738" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Clinical implications of mutations C-to-T1653 and T-to-C/A/G1753 of hepatitis B virus genotype C genome in chronic liver disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11019999" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Rapid detection of genotypes and mutations in the pre-core promoter and the pre-core region of hepatitis B virus genome: correlation with viral persistence and disease severity.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12904378" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : Rapid and sensitive assays for determination of hepatitis B virus (HBV) genotypes and detection of HBV precore and core promoter variants.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8985293" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Detection by ligase chain reaction of precore mutant of hepatitis B virus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9300379" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Colourimetric point mutation assay: for detection of precore mutants of hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7966600" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Hepatitis B virus with mutations in the core promoter for an e antigen-negative phenotype in carriers with antibody to e antigen.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7528709" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : Predictive value of precore hepatitis B virus mutations in spontaneous and interferon-induced hepatitis B e antigen clearance.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11773086" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : Precore stop mutant in HBeAg-positive patients with chronic hepatitis B: clinical characteristics and correlation with the course of HBeAg-to-anti-HBe seroconversion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9191945" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : Hepatitis B virus X gene 1751 to 1764 mutations: implications for HBeAg status and disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10706570" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : Hepatitis B e antigen-negative chronic hepatitis B in Hong Kong.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10051506" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Different hepatitis B virus genotypes are associated with different mutations in the core promoter and precore regions during hepatitis B e antigen seroconversion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8847524" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : The precore/core promoter mutant (T1762A1764) of hepatitis B virus: clinical significance and an easy method for detection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8991934" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : Mutations in the core promoter region of hepatitis B virus in patients with chronic hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11375963" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : Management of hepatitis B: 2000--summary of a workshop.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8071540" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : Comparison of anti-HBe-positive and HBe-antigen-positive chronic hepatitis B in France. French Multicentre Group.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2663695" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : Natural course and response to interferon of chronic hepatitis B accompanied by antibody to hepatitis B e antigen.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3192180" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : Long-term follow-up of anti-HBe-positive chronic active hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1732124" id="rid53" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>54 : Precore mutant hepatitis B virus infection and liver disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10445556" id="rid54" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>55 : The clinical significance of core promoter and precore mutations during the natural course and interferon therapy in patients with chronic hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2156307" id="rid55" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>56 : Chronic hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9180165" id="rid56" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>57 : Genotypes, nt 1858 variants, and geographic origin of hepatitis B virus--large-scale analysis using a new genotyping method.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8171038" id="rid57" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>58 : Mutations in the pre-core region of hepatitis B virus serve to enhance the stability of the secondary structure of the pre-genome encapsidation signal.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7489968" id="rid58" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>59 : Hepatitis B virus infection: precore mutants and its relation to viral genotypes and core mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10068571" id="rid59" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>60 : Core promoter mutations and genotypes in relation to viral replication and liver damage in East Asian hepatitis B virus carriers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11124839" id="rid60" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>61 : A case-control study for clinical and molecular biological differences between hepatitis B viruses of genotypes B and C. Japan HBV Genotype Research Group.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9700628" id="rid61" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>62 : HBV core promoter mutations prevail in patients with hepatocellular carcinoma from Guangxi, China.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10051502" id="rid62" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>63 : High prevalence of 1762(T) 1764(A) mutations in the basic core promoter of hepatitis B virus isolated from black Africans with hepatocellular carcinoma compared with asymptomatic carriers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18825703" id="rid63" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>64 : Impact of the hepatitis B virus genotype on pre- and post-liver transplantation outcomes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8340609" id="rid64" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>65 : Hepatitis B virus heterogeneity, one of many factors influencing the severity of hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16108767" id="rid65" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>66 : Hepatic necroinflammation and fibrosis in patients with genotypes Ba and C, core-promoter and precore mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12402204" id="rid66" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>67 : Relationship between the development of precore and core promoter mutations and hepatitis B e antigen seroconversion in patients with chronic hepatitis B virus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15090469" id="rid67" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>68 : Role of hepatitis B virus genotypes Ba and C, core promoter and precore mutations on hepatocellular carcinoma: a case control study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12557138" id="rid68" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>69 : Basal core promoter mutations of hepatitis B virus increase the risk of hepatocellular carcinoma in hepatitis B carriers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17502344" id="rid69" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>70 : Temporal relationship between hepatitis B virus enhancer II/basal core promoter sequence variation and risk of hepatocellular carcinoma.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12939588" id="rid70" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>71 : Prevalence of HBV precore/core promoter variants in the United States.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24763132" id="rid71" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>72 : Higher proportion of viral basal core promoter mutant increases the risk of liver cirrhosis in hepatitis B carriers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24341484" id="rid72" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>73 : Effect of combined mutations in the enhancer II and basal core promoter of hepatitis B virus on development of hepatocellular carcinoma in Qidong, China.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23383661" id="rid73" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>74 : Synergistic effects of A1896, T1653 and T1762/A1764 mutations in genotype c2 hepatitis B virus on development of hepatocellular carcinoma.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20959817" id="rid74" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>75 : Association between the various mutations in viral core promoter region to different stages of hepatitis B, ranging of asymptomatic carrier state to hepatocellular carcinoma.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18695135" id="rid75" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>76 : Associations between hepatitis B virus genotype and mutants and the risk of hepatocellular carcinoma.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7825758" id="rid76" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>77 : Hepatitis B virus strains with mutations in the core promoter in patients with fulminant hepatitis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2001807" id="rid77" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>78 : Fulminant hepatitis B: induction by hepatitis B virus mutants defective in the precore region and incapable of encoding e antigen.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2034246" id="rid78" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>79 : Mutations in the precore region of hepatitis B virus DNA in patients with fulminant and severe hepatitis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2034247" id="rid79" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>80 : A hepatitis B virus mutant associated with an epidemic of fulminant hepatitis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7489966" id="rid80" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>81 : Fulminant hepatitis associated with hepatitis B virus e antigen-negative infection: importance of host factors.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15123367" id="rid81" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>82 : Mutations in the basic core promoter region of hepatitis B virus. Relationship with precore variants and HBV genotypes in a Spanish population of HBV carriers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15918203" id="rid82" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>83 : High level of hepatitis B virus DNA after HBeAg-to-anti-HBe seroconversion is related to coexistence of mutations in its precore and basal core promoter.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7689519" id="rid83" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>84 : Hepatitis B virus unable to secrete e antigen and response to interferon in chronic hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10235220" id="rid84" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>85 : Long term response to therapy of chronic anti-HBe-positive hepatitis B is poor independent of type and schedule of interferon.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9398007" id="rid85" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>86 : A randomized, controlled trial of a 24-month course of interferon alfa 2b in patients with chronic hepatitis B who had hepatitis B virus DNA without hepatitis B e antigen in serum.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7590647" id="rid86" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>87 : Hepatitis B virus precore/core variation and interferon therapy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10706563" id="rid87" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>88 : Mutations of the core promoter and response to interferon treatment in chronic replicative hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11131465" id="rid88" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>89 : Hepatitis B genotypes and the response to interferon therapy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12931030" id="rid89" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>90 : Low HBeAg serum levels correlate with the presence of the double A1762T/G1764A core promoter mutation and a positive response to interferon in patients with chronic hepatitis B virus infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22898565" id="rid90" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>91 : Pegylated interferon results in higher serological, but not virological, response rates when compared to continuous entecavir.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22307831" id="rid91" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>92 : Presence of precore and core promoter mutants limits the probability of response to peginterferon in hepatitis B e antigen-positive chronic hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/6181506" id="rid92" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>93 : Immune response to synthetic peptide analogues of hepatitis B surface antigen specific for the a determinant.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3080114" id="rid93" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>94 : Antibody responses to recombinant and plasma derived hepatitis B vaccines.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/6755247" id="rid94" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>95 : Hepatitis B vaccine in medical staff of hemodialysis units: efficacy and subtype cross-protection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7612225" id="rid95" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>96 : Hepatitis B virus immunopathogenesis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1696684" id="rid96" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>97 : Viral escape by selection of cytotoxic T cell-resistant virus variants in vivo.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1281839" id="rid97" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>98 : Loss of the common "A" determinant of hepatitis B surface antigen by a vaccine-induced escape mutant.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11120616" id="rid98" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>99 : Intra-familial evidence of horizontal transmission of hepatitis B virus surface antigen mutant G145R.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15361503" id="rid99" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>100 : Survey of hepatitis B surface variant infection in children 15 years after a nationwide vaccination programme in Taiwan.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28182307" id="rid100" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>101 : Chronologic changes in serum hepatitis B virus DNA, genotypes, surface antigen mutants and reverse transcriptase mutants during 25-year nationwide immunization in Taiwan.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11431751" id="rid101" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>102 : Persistent hepatitis B virus infection in subjects without hepatitis B surface antigen: clinically significant or purely "occult"?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9398013" id="rid102" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>103 : The prevalence of surface antigen variants of hepatitis B virus in Papua New Guinea, South Africa, and Sardinia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9696878" id="rid103" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>104 : Characterization of unusual escape variants of hepatitis B virus isolated from a hepatitis B surface antigen-negative subject.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7843693" id="rid104" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>105 : A unique insertion in the S gene of surface antigen--negative hepatitis B virus Chinese carriers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8617416" id="rid105" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>106 : A molecular analysis of viral persistence in surface antigen-negative chronic hepatitis B.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11431752" id="rid106" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>107 : Occult hepatitis B virus infection: a hidden menace?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31004683" id="rid107" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>108 : Update of the statements on biology and clinical impact of occult hepatitis B virus infection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2555175" id="rid108" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>109 : Identification of four conserved motifs among the RNA-dependent polymerase encoding elements.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2429313" id="rid109" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>110 : Computer analysis of retroviral pol genes: assignment of enzymatic functions to specific sequences and homologies with nonviral enzymes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1377403" id="rid110" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>111 : Crystal structure at 3.5 A resolution of HIV-1 reverse transcriptase complexed with an inhibitor.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1583465" id="rid111" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>112 : Review: hepatitis B virus X-gene product: a promiscuous transcriptional activator.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8567959" id="rid112" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>113 : Regulation of transforming growth factor-beta 1 expression by the hepatitis B virus (HBV) X transactivator. Role in HBV pathogenesis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8774611" id="rid113" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>114 : Hot-spot mutations in hepatitis B virus X gene in hepatocellular carcinoma.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
