<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Characteristics of the hepatitis C virus</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Characteristics of the hepatitis C virus</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Characteristics of the hepatitis C virus</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Sanjiv Chopra, MD, MACP</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Adrian M Di Bisceglie, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Allyson Bloom, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Feb 17, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>It became apparent, after the discovery of the hepatitis A and B viruses in the late 1960s and early 1970s, that a large proportion of cases of acute and chronic hepatitis could not be explained by either of these agents. Another viral agent was suspected, and patients infected with this suspected agent were said to have non-A, non-B hepatitis. The agent was finally identified in 1989 when the genome of the virus was cloned and the agent was designated the hepatitis C virus (HCV) [<a href="#rid1">1</a>].</p><p>HCV is closely related to flaviviruses and pestiviruses. Its genetic organization and protein products classify it in the <em>Flaviviridae</em> family, although its diversity is great enough for it to be classified as a separate genus. HCV is not related to any of the other known hepatitis viruses; however, the recently described hepatitis G virus is a distant relative. (See  <a class="medical medical_review" href="/d/html/3660.html" rel="external">"Human pegivirus-1 infection"</a>.)</p><p class="headingAnchor" id="H2"><span class="h1">VIRAL GENOME AND REPLICATION</span><span class="headingEndMark"> — </span>The HCV genome is a positive-sense ribonucleic acid (RNA) molecule of approximately 9500 nucleotides. There are highly conserved 5’ and 3’ untranslated regions flanking an approximately 9000 nucleotide single open reading frame, which encodes a large polyprotein of about 3000 amino acids [<a href="#rid2">2</a>]. This protein undergoes posttranslational processing by host and viral enzymes to form the structural and nonstructural proteins and enzymes of the virus.</p><p>The 5' terminus of the viral RNA is an untranslated region (5' UTR) known to be essential for replication; it contains elements felt to coordinate viral protein synthesis. It is not surprising that this region is highly conserved and therefore serves as a useful target for amplification in diagnostic assays.</p><p class="headingAnchor" id="H3"><span class="h1">VIRAL HETEROGENEITY</span><span class="headingEndMark"> — </span>The polymerase enzyme of RNA viruses such as HCV lacks proofreading ability and is, therefore, unable to correct copying errors made during viral replication. Many of these nucleotide changes result in a nonfunctional genome or a replication-incompetent virus (lethal mutants). However, others persist and account for the tremendous viral diversity that is characteristic of HCV. This heterogeneity is extremely important in the diagnosis of infection, pathogenesis of disease, and the response to treatment. It prevents the development of conventional vaccines, allows the virus to escape eradication by the host's immune system, and affects the completeness of the response to antiviral therapies such as interferon [<a href="#rid3">3</a>]. </p><p>Viral heterogeneity takes several forms depending upon the degree of diversity:</p><p class="bulletIndent1"><span class="glyph">●</span>Quasispecies are families of different, but highly similar, strains that develop within an infected host over time. Nucleotide sequence homology is greater than 95 percent.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Over decades and centuries, the degree of HCV diversity has evolved into several distinct genotypes of the virus [<a href="#rid4">4</a>]. Sequence homology between genotypes is less than 80 percent. There are six genotypes and numerous subtypes of HCV.</p><p></p><p class="headingAnchor" id="H4"><span class="h2">Quasispecies</span><span class="headingEndMark"> — </span>Differences between quasispecies families are usually only apparent in the most rapidly changing parts of the genome (hypervariable regions). Typically, a single dominant sequence changes with time, being replaced by one or more minor populations due to external pressures (eg, the host immune system) on the quasispecies. One group of investigators generated more than 100 clones of the most genetically heterogeneous region of a single HCV isolate; they found 19 unique sequences, confined largely to the first hypervariable region (HVR1) of the E2 envelope protein [<a href="#rid5">5</a>].</p><p>The clinical implications of quasispecies are not yet entirely understood, although there is some evidence that the quasispecies may be important in the persistence of virus, natural history of infection, and the response to treatment. In one study of 59 patients with chronic HCV infection, increased quasispecies heterogeneity (eg, more than two predominant families of quasispecies) correlated with a longer estimated duration of HCV infection, presumed route of infection (transfusion), increased serum HCV-RNA level, and genotype 1 [<a href="#rid6">6</a>] (see <a class="local">'Genotypes'</a> below). Patients who had a sustained response to interferon had less pretreatment quasispecies heterogeneity than those who either relapsed or had no response to treatment.</p><p>Another report, in which HCV quasispecies were determined in liver samples, found that different quasispecies were compartmentalized within specific regions in the liver and that the degree of compartmentalization was greater in histologically advanced disease [<a href="#rid7">7</a>]. Another study found that patients with HIV had a more diversified HCV population compared with patients without HIV, possibly due to less selective pressure from the immune system [<a href="#rid8">8</a>].</p><p>This genetic diversity of HCV may allow it to escape host immune surveillance, thereby resulting in virus persistence and a lack of protective immunity [<a href="#rid3">3,5,9</a>]. In one study, for example, markers of viral replication and host immunity were studied in five chimpanzees sequentially inoculated over a period of three years with different HCV strains [<a href="#rid3">3</a>]. Each rechallenge of a convalescent chimpanzee with the same or a different HCV strain resulted in reinfection. Reinfection has also been observed in multiple transfused patients [<a href="#rid10">10</a>]. The genetic diversity of HCV has obvious implications for vaccine development.</p><p class="headingAnchor" id="H5"><span class="h2">Genotypes</span><span class="headingEndMark"> — </span>Six major genotypes of HCV have been defined, with more than 50 subtypes described; the most common subtypes are 1a, 1b, 2a, and 2b [<a href="#rid11">11,12</a>]. Sporadic viral isolates distinct from all others have been categorized as a seventh and eighth genotype [<a href="#rid13">13,14</a>]. </p><p>The evolution of genotypes has probably been influenced by several factors, including immune selection, infection patterns, replication efficiency, and population migration. Thus, there is a distinct geographic distribution of HCV genotypes  (<a class="graphic graphic_figure graphicRef99796" href="/d/graphic/99796.html" rel="external">figure 1</a>) [<a href="#rid15">15-18</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>Genotype 1 is most common (60 to 75 percent of isolates) in the United States, Latin America, and Europe.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genotype 3 is most common in South Asia (eg, India) and Australia.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genotype 4 is most common in Africa and the Middle East and appears to be emerging more frequently in Europe among people who inject drugs and men who have sex with men.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genotype 5 is most common in South Africa.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genotype 6 is most common in Hong Kong and Vietnam.</p><p></p><p>In the United States, genotypes 2 and 3 are less prevalent than genotype 1 infection (approximately 15 and 10 percent of infections, respectively) [<a href="#rid19">19</a>]. However, among younger individuals and people who use injection drugs, the prevalence of genotype 3 infection is higher. Genotypes 4, 5, and 6 are rare.</p><p>The impact of genotype on treatment regimen selection is discussed elsewhere. (See  <a class="medical medical_review" href="/d/html/16592.html" rel="external">"Management of chronic hepatitis C virus infection: Initial antiviral therapy in adults"</a> and  <a class="medical medical_review" href="/d/html/15795.html" rel="external">"Management of chronic hepatitis C virus infection: Antiviral retreatment following relapse in adults"</a>.)</p><p class="headingAnchor" id="H2476630649"><span class="h1">SOCIETY GUIDELINE LINKS</span><span class="headingEndMark"> — </span>Links to society and government-sponsored guidelines from selected countries and regions around the world are provided separately. (See  <a class="medical medical_society_guidelines" href="/d/html/113060.html" rel="external">"Society guideline links: Hepatitis C virus infection"</a>.)</p><p class="headingAnchor" id="H144223337"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span>The genetic organization and protein products of the hepatitis C virus (HCV) classify it in the <em>Flaviviridae</em> family, although its diversity is great enough for it to be classified as a separate genus. (See <a class="local">'Introduction'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The HCV genome is a positive-sense ribonucleic acid (RNA) molecule of approximately 9500 nucleotides. There are highly conserved 5' and 3' untranslated regions flanking an approximately 9000 nucleotide single open reading frame which encodes a large polyprotein of about 3000 amino acids. (See <a class="local">'Viral genome and replication'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The polymerase enzyme of RNA viruses such as HCV lacks proofreading ability and is therefore unable to correct copying errors made during viral replication. This leads to heterogeneity that is extremely important in the diagnosis of infection, pathogenesis of disease, and the response to treatment. (See <a class="local">'Viral genome and replication'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Viral heterogeneity takes several forms depending upon the degree of diversity:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Quasispecies are families of different but highly similar strains that develop within an infected host over time. Nucleotide sequence homology is greater than 95 percent. (See <a class="local">'Quasispecies'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Over decades and centuries, the degree of HCV diversity has evolved into several distinct genotypes of the virus. Sequence homology between genotypes is less than 80 percent. There are six major genotypes and numerous subtypes of HCV. (See <a class="local">'Genotypes'</a> above.)</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Choo QL, Kuo G, Weiner AJ, et al. Isolation of a cDNA clone derived from a blood-borne non-A, non-B viral hepatitis genome. Science 1989; 244:359.</a></li><li><a class="nounderline abstract_t">Major ME, Feinstone SM. The molecular virology of hepatitis C. Hepatology 1997; 25:1527.</a></li><li><a class="nounderline abstract_t">Farci P, Alter HJ, Govindarajan S, et al. Lack of protective immunity against reinfection with hepatitis C virus. Science 1992; 258:135.</a></li><li><a class="nounderline abstract_t">Simmonds P. Variability of hepatitis C virus. Hepatology 1995; 21:570.</a></li><li><a class="nounderline abstract_t">Farci P, Shimoda A, Wong D, et al. Prevention of hepatitis C virus infection in chimpanzees by hyperimmune serum against the hypervariable region 1 of the envelope 2 protein. Proc Natl Acad Sci U S A 1996; 93:15394.</a></li><li><a class="nounderline abstract_t">González-Peralta RP, Qian K, She JY, et al. Clinical implications of viral quasispecies heterogeneity in chronic hepatitis C. J Med Virol 1996; 49:242.</a></li><li><a class="nounderline abstract_t">Sakai A, Kaneko S, Honda M, et al. Quasispecies of hepatitis C virus in serum and in three different parts of the liver of patients with chronic hepatitis. Hepatology 1999; 30:556.</a></li><li><a class="nounderline abstract_t">Tanaka Y, Hanada K, Hanabusa H, et al. Increasing genetic diversity of hepatitis C virus in haemophiliacs with human immunodeficiency virus coinfection. J Gen Virol 2007; 88:2513.</a></li><li><a class="nounderline abstract_t">Weiner AJ, Geysen HM, Christopherson C, et al. Evidence for immune selection of hepatitis C virus (HCV) putative envelope glycoprotein variants: potential role in chronic HCV infections. Proc Natl Acad Sci U S A 1992; 89:3468.</a></li><li><a class="nounderline abstract_t">Lai ME, Mazzoleni AP, Argiolu F, et al. Hepatitis C virus in multiple episodes of acute hepatitis in polytransfused thalassaemic children. Lancet 1994; 343:388.</a></li><li><a class="nounderline abstract_t">Simmonds P, Alberti A, Alter HJ, et al. A proposed system for the nomenclature of hepatitis C viral genotypes. Hepatology 1994; 19:1321.</a></li><li><a class="nounderline abstract_t">Simmonds P, Bukh J, Combet C, et al. Consensus proposals for a unified system of nomenclature of hepatitis C virus genotypes. Hepatology 2005; 42:962.</a></li><li><a class="nounderline abstract_t">Smith DB, Bukh J, Kuiken C, et al. Expanded classification of hepatitis C virus into 7 genotypes and 67 subtypes: updated criteria and genotype assignment web resource. Hepatology 2014; 59:318.</a></li><li><a class="nounderline abstract_t">Borgia SM, Hedskog C, Parhy B, et al. Identification of a Novel Hepatitis C Virus Genotype From Punjab, India: Expanding Classification of Hepatitis C Virus Into 8 Genotypes. J Infect Dis 2018; 218:1722.</a></li><li><a class="nounderline abstract_t">Lau JY, Davis GL, Prescott LE, et al. Distribution of hepatitis C virus genotypes determined by line probe assay in patients with chronic hepatitis C seen at tertiary referral centers in the United States. Hepatitis Interventional Therapy Group. Ann Intern Med 1996; 124:868.</a></li><li><a class="nounderline abstract_t">Dusheiko G, Schmilovitz-Weiss H, Brown D, et al. Hepatitis C virus genotypes: an investigation of type-specific differences in geographic origin and disease. Hepatology 1994; 19:13.</a></li><li><a class="nounderline abstract_t">Messina JP, Humphreys I, Flaxman A, et al. Global distribution and prevalence of hepatitis C virus genotypes. Hepatology 2015; 61:77.</a></li><li><a class="nounderline abstract_t">Robaeys G, Bielen R, Azar DG, et al. Global genotype distribution of hepatitis C viral infection among people who inject drugs. J Hepatol 2016; 65:1094.</a></li><li><a class="nounderline abstract_t">Gordon SC, Trudeau S, Li J, et al. Race, Age, and Geography Impact Hepatitis C Genotype Distribution in the United States. J Clin Gastroenterol 2019; 53:40.</a></li></ol></div><div id="topicVersionRevision">Topic 3668 Version 19.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2523562" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Isolation of a cDNA clone derived from a blood-borne non-A, non-B viral hepatitis genome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9185778" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : The molecular virology of hepatitis C.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1279801" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Lack of protective immunity against reinfection with hepatitis C virus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7531173" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Variability of hepatitis C virus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8986822" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Prevention of hepatitis C virus infection in chimpanzees by hyperimmune serum against the hypervariable region 1 of the envelope 2 protein.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8818972" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Clinical implications of viral quasispecies heterogeneity in chronic hepatitis C.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10421668" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Quasispecies of hepatitis C virus in serum and in three different parts of the liver of patients with chronic hepatitis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17698661" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Increasing genetic diversity of hepatitis C virus in haemophiliacs with human immunodeficiency virus coinfection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1314389" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Evidence for immune selection of hepatitis C virus (HCV) putative envelope glycoprotein variants: potential role in chronic HCV infections.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7905553" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Hepatitis C virus in multiple episodes of acute hepatitis in polytransfused thalassaemic children.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8175159" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : A proposed system for the nomenclature of hepatitis C viral genotypes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16149085" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Consensus proposals for a unified system of nomenclature of hepatitis C virus genotypes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24115039" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Expanded classification of hepatitis C virus into 7 genotypes and 67 subtypes: updated criteria and genotype assignment web resource.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29982508" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Identification of a Novel Hepatitis C Virus Genotype From Punjab, India: Expanding Classification of Hepatitis C Virus Into 8 Genotypes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8610915" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Distribution of hepatitis C virus genotypes determined by line probe assay in patients with chronic hepatitis C seen at tertiary referral centers in the United States. Hepatitis Interventional Therapy Group.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8276349" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Hepatitis C virus genotypes: an investigation of type-specific differences in geographic origin and disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25069599" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Global distribution and prevalence of hepatitis C virus genotypes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27520879" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Global genotype distribution of hepatitis C viral infection among people who inject drugs.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28737649" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Race, Age, and Geography Impact Hepatitis C Genotype Distribution in the United States.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
