<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Overview of HIV drug resistance testing assays
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Overview of HIV drug resistance testing assays
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Overview of HIV drug resistance testing assays
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Author:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Michael J Kozal, MD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editors:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Martin S Hirsch, MD
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Paul E Sax, MD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Jennifer Mitty, MD, MPH
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            Sep 17, 2019.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H1">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         The use of drug resistance testing has become an integral part of HIV clinical care. The first clinical description of HIV resistance to an antiretroviral agent was published in 1989, when patients taking
         <a class="drug drug_general" data-topicid="10053" href="/z/d/drug information/10053.html" rel="external">
          zidovudine
         </a>
         monotherapy accumulated mutations within the reverse transcriptase gene, resulting in a marked increase in drug resistance [
         <a href="#rid1">
          1
         </a>
         ]. Subsequently, HIV variants resistant to every available antiretroviral agent have been identified. The evolution of drug resistance has significant clinical implications for choosing effective antiretroviral regimens.
        </p>
        <p>
         This topic will provide an overview of HIV drug resistance testing. The interpretation of these tests and the approach to selecting an antiretroviral therapy regimen for patients with drug resistance mutations are discussed elsewhere. (See
         <a class="medical medical_review" href="/z/d/html/3772.html" rel="external">
          "Interpretation of HIV drug resistance testing"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/13979.html" rel="external">
          "Selecting an antiretroviral regimen for treatment-experienced patients with HIV who are failing therapy"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/13975.html" rel="external">
          "Evaluation of the treatment-experienced patient failing HIV therapy"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H864487796">
         <span class="h1">
          RATIONALE FOR HIV SUSCEPTIBILITY TESTING
         </span>
         <span class="headingEndMark">
          —
         </span>
         The use of HIV susceptibility testing to guide therapy leads to better viral suppression and has been associated with improved survival [
         <a href="#rid2">
          2-5
         </a>
         ]. As an example, in an observational study of more than 2699 HIV-infected patients who were eligible for genotypic and phenotypic testing between 1999 and 2005, resistance testing was associated with improved survival (adjusted hazard ratio, 0.69 [95% CI 0.51-0.94]) after controlling for demographics, CD4 cell count, HIV RNA level, and intensity of clinical follow-up [
         <a href="#rid3">
          3
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H3184692256">
         <span class="h1">
          GENOTYPIC VERSUS PHENOTYPIC ASSAYS
         </span>
         <span class="headingEndMark">
          —
         </span>
         This section will provide an overview of the commonly used resistance assays. The use of resistance testing when selecting an antiretroviral regimen is discussed elsewhere. (See
         <a class="medical medical_review" href="/z/d/html/3773.html" rel="external">
          "Selecting antiretroviral regimens for treatment-naïve persons with HIV-1: General approach", section on 'Considerations prior to initiating treatment'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/13975.html" rel="external">
          "Evaluation of the treatment-experienced patient failing HIV therapy"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2999526046">
         <span class="h2">
          Overview of the assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         Resistance assays can be categorized as either genotypic or phenotypic. These assays detect resistance in fundamentally different ways, although the results generally correlate with each other.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Genotypic resistance assays detect the presence of specific drug resistance mutations in the regions of the HIV genome encoding protease, reverse transcriptase, and integrase. Results are reported as the individual mutations (eg, M184V, the signature mutation for
         <a class="drug drug_general" data-topicid="9544" href="/z/d/drug information/9544.html" rel="external">
          lamivudine
         </a>
         resistance) with comments such as "susceptible," "possibly resistant," or "resistant" for each antiretroviral agent. (See
         <a class="medical medical_review" href="/z/d/html/3772.html" rel="external">
          "Interpretation of HIV drug resistance testing"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Phenotypic resistance assays measure the extent to which an antiretroviral drug inhibits virus replication in vitro. The susceptibility that is measured with phenotypic assays is the aggregate of the acquired drug mutations in a patient's viral strain [
         <a href="#rid6">
          6
         </a>
         ]. Similar to bacteriologic methods, this is typically performed by demonstrating an increase in the inhibitory concentration (IC) that is required to inhibit in vitro growth by 50 percent (IC50) compared with virus replication in the absence of drug. Results are reported as a fold-change in drug susceptibility of the patient sample compared with a laboratory reference strain without resistance.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H3417418688">
         <span class="h2">
          Advantages and limitations
         </span>
         <span class="headingEndMark">
          —
         </span>
         In most patients, genotypic assays are preferred. Genotypic assays provide information on the specific resistance mutations present within the virus. Additional advantages of genotypic relative to phenotypic testing include lower cost and shorter turnaround time. These assays may also be more sensitive for detection of "mixtures" of resistant and wild-type viruses. HIV treatment guidelines recommend a genotypic assay be the preferred testing method in treatment-naïve patients [
         <a href="#rid4">
          4,7-10
         </a>
         ]. (See
         <a class="local">
          'Genotypic resistance assays'
         </a>
         below.)
        </p>
        <p>
         However, in treatment-experienced patients with multiple resistance mutations, interpreting complex genotypes can be challenging since it can be difficult to interpret the clinical significance of a specific mutation when many mutations are present. As an example, some mutations cause resistance to certain drugs but increase susceptibility to others, while other mutations may impact viral fitness (see
         <a class="medical medical_review" href="/z/d/html/3772.html" rel="external">
          "Interpretation of HIV drug resistance testing", section on 'Viral fitness'
         </a>
         ). For such patients, phenotypic resistance testing may have an advantage over genotypic testing, as it measures resistance more directly and can assess relative susceptibility and interactions among mutations. This includes mutations that may not have been recognized to date as causing resistance, or combinations of mutations that may have a different effect on susceptibility compared with each individual mutation alone.
        </p>
        <p>
         Both genotypic and phenotypic assays are limited by insensitivity to minority variants comprising 1 to 20 percent of the virus population [
         <a href="#rid11">
          11,12
         </a>
         ]. This limitation is important because once antiretroviral therapy (ART) is discontinued, a wild-type virus may reemerge to replace the drug-resistant virus as soon as four to six weeks after therapy is stopped. Thus, absence of a detectable resistance mutation must be interpreted with caution in a patient who has recently discontinued ART. Other techniques have been developed to improve the detection of these minority species compared with standard drug resistance assays. (See
         <a class="local">
          'Low-abundance drug-resistant variants'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H7">
         <span class="h2">
          Genotypic resistance assays
         </span>
        </p>
        <p class="headingAnchor" id="H8">
         <span class="h3">
          Available assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         Commercially available tests include United States Food and Drug Administration (FDA)-approved kits, as well as a variety of in-house assays performed by reference laboratories. These assays sequence regions of the HIV genome (eg protease, reverse transcriptase, and integrase genes) that have been polymerase chain reaction-amplified from the viral quasispecies circulating in a patient's plasma. Based on when genotype tests were first introduced, standard tests frequently include testing for nucleoside reverse transcriptase inhibitors (NRTI), non-nucleoside reverse transcriptase inhibitor (NNRTI), and protease inhibitor (PI) resistance. If a clinician is interested in identifying resistance to integrase inhibitors or the fusion inhibitor
         <a class="drug drug_general" data-topicid="8609" href="/z/d/drug information/8609.html" rel="external">
          enfuvirtide
         </a>
         , these tests usually must be requested separately.
        </p>
        <p class="headingAnchor" id="H1438246320">
         <span class="h3">
          Genotypic definitions of resistance
         </span>
         <span class="headingEndMark">
          —
         </span>
         Defining resistance using genotypic assays is generally more complex than for phenotypic assays. A mutation is generally identified as playing a role in resistance to a given drug if one or more of the following conditions are met:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The mutation confers phenotypic resistance when introduced into a drug-sensitive laboratory strain of HIV.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The mutation is selected for during serial in vitro passage of virus in the presence of drug.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The mutation is selected for during clinical therapy with that drug.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The presence of the mutation in clinical isolates is associated with phenotypic resistance and virologic failure.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H2667117535">
         <span class="h3">
          Interpreting genotypic assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         There are currently two general approaches to interpreting genotypic resistance testing:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Expert interpretation, in which data concerning the association of a mutation with drug resistance are evaluated and synthesized. Over 20 rules-based genotypic interpretation systems have been proposed [
         <a href="#rid13">
          13
         </a>
         ]; however, there is good correlation among the various genotypic interpretation systems [
         <a href="#rid14">
          14
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Databases, in which genotype is correlated with phenotype [
         <a href="#rid4">
          4,14
         </a>
         ].
        </p>
        <p>
        </p>
        <p>
         Collaborative groups have been formed to ensure interpretation of genotypic mutational patterns remains up to date as new data emerge and the information reflects agents used in the clinic [
         <a href="#rid14">
          14
         </a>
         ].
        </p>
        <p>
         There are insufficient data to recommend one type of interpretation over another. Evidence suggests that many of these systems have a similar ability to predict virologic outcome [
         <a href="#rid15">
          15,16
         </a>
         ]. The following summarizes some of the specific genotypic resistance testing interpretations that are available to clinicians and investigators. This summary provides only a few examples and is
         <strong>
          not
         </strong>
         intended to be a comprehensive list of the different approaches that are available.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Genotypic resistance assay reports typically list the specific drug resistance mutations detected in the test sample, compared with a reference sequence. In addition, each report summarizes which drugs may have reduced activity based upon the viral sequence. These interpretations are made using algorithms defined by experts in the field of resistance testing, based upon a synthesis of in vitro and clinical data. No direct prospective comparisons of the genotypic interpretations using these two methods have been made.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The International AIDS Society-USA provides an updated list of mutations associated with HIV drug resistance to all FDA-approved drugs, based on evaluations by a panel of experts (
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fwww.iasusa.org%2F&amp;token=WSG0bGDmo%2BZ5TV02z1kZCQaDbrJjjUgfDb8LY8qTtuM%3D&amp;TOPIC_ID=3769" target="_blank">
          www.iasusa.org
         </a>
         ) [
         <a href="#rid17">
          17
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The Stanford drug resistance database (
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fhivdb.stanford.edu%2F&amp;token=gbrKzpZMrcLn9ZCC9ag8fFlkWQOaaRu%2BeAYLQ0lynbflPVcXgyrKCDlyBDfr3i%2Bz&amp;TOPIC_ID=3769" target="_blank">
          hivdb.stanford.edu
         </a>
         ) provides access to a genotypic resistance algorithm, which assigns a drug penalty score for each resistance mutation, based on published studies of clinical outcome and expert opinion [
         <a href="#rid18">
          18
         </a>
         ]. Some information on expected fold-change in IC50 can be obtained by searching the database, which is drawn from published studies and some unpublished clinical trials data.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Agence Nationale de Recherches sur le SIDA (ANRS) [
         <a href="#rid19">
          19
         </a>
         ], and Rega [
         <a href="#rid20">
          20
         </a>
         ], in addition to the Stanford HIV database [
         <a href="#rid15">
          15,21
         </a>
         ], are the publicly available drug resistance interpretation systems that are most commonly used by HIV providers. The algorithms for these systems are transparent, updated regularly, and are free to the clinician. These systems are comparable in their ability to predict virologic outcome [
         <a href="#rid15">
          15,16
         </a>
         ].
        </p>
        <p>
        </p>
        <p>
         Caution should be used when interpreting resistance testing to investigational or recently approved antiretroviral agents. In addition, if a genotype was interpreted in the past using older algorithms, it should be reinterpreted using updated algorithms with new information as resistance interpretations can change over time [
         <a href="#rid22">
          22
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H757872852">
         <span class="h2">
          Phenotypic resistance assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         HIV phenotypic resistance testing is only offered through reference laboratories in the United States, and only the PhenoSense assay (LabCorp, Burlington, NC) is available. An older assay, entitled Antivirogram (Janssen Diagnostics, Belgium), was withdrawn from the market due to a decline in demand. However, a clinician may still encounter an old Antivirogram test result when reviewing a patient's prior drug resistance history.
        </p>
        <p class="headingAnchor" id="H3419726869">
         <span class="h3">
          Recombinant virus assay
         </span>
         <span class="headingEndMark">
          —
         </span>
         Phenotype testing is performed using recombinant virus assays (RVAs) [
         <a href="#rid23">
          23
         </a>
         ]. These replaced the slow, labor-intensive cell culture method initially used [
         <a href="#rid24">
          24
         </a>
         ] and has allowed phenotyping to be done on a larger scale.
        </p>
        <p>
         Available RVAs use recombinant (chimeric) viruses composed of protease, reverse transcriptase, and integrase gene sequences from viruses circulating in a patient's plasma, which are inserted into the genetic backbone of a laboratory reference strain of HIV. The resulting chimeric DNA clone is transfected into mammalian cells to produce an infectious virus that expresses the protease and reverse transcriptase enzymes from the patient HIV strain. The drug susceptibility of this chimeric virus can then be measured in a phenotypic resistance assay.
        </p>
        <p>
         RVAs accurately reflect the drug susceptibility of the original clinical isolate only if the region of the viral genome being amplified contains the genetic determinants (mutant sequences) of drug resistance. Modifications of the RVA are also available for fusion and CCR5 inhibitors [
         <a href="#rid25">
          25-29
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H13">
         <span class="h3">
          Phenotypic definitions of resistance
         </span>
         <span class="headingEndMark">
          —
         </span>
         Current phenotypic assays measure the ability of a laboratory virus that contains a specific viral gene from the patient sample to grow in the presence of different concentrations of an antiretroviral drug. The ability to define what constitutes the threshold for clinical resistance is a difficult process as it includes both assay and biologic factors.
        </p>
        <p>
         The cutoffs for clinical resistance would be best verified by clinical trials; however, this has not been performed for many agents. Thus, what constitutes significant clinical phenotypic resistance must often be inferred from decreases in susceptibility of the patient sample from that of a known wild-type HIV strain (usually the drug concentration it takes to inhibit viral replication by 50 percent, expressed in fold-increase in IC50). Clinical cutoffs are often calculated by examining the fold-change in susceptibility of a patient's virus and the clinical treatment response. When these studies are performed they also correlate the fold-change in susceptibility with the mutations present in the virus.
        </p>
        <p class="headingAnchor" id="H3568815171">
         <span class="h4">
          Assay variability
         </span>
         <span class="headingEndMark">
          —
         </span>
         When validating an assay, studies are done to define the inter-assay and inter-laboratory assessment of assay repeatability and robustness. Earlier definitions of resistance for phenotypic assays were based on the technical reproducibility of the assay and did not necessarily reflect clinically relevant definitions of resistance [
         <a href="#rid30">
          30
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H15">
         <span class="h4">
          Biologic variability
         </span>
         <span class="headingEndMark">
          —
         </span>
         Resistance thresholds based upon biologic variability are usually defined as a level that exceeds the mean fold-change in IC50 of HIV isolates obtained from treatment-naive patients by a certain factor. This definition of resistance is dependent upon the population chosen for study and assumes that all treatment-naive patients will respond equally well to that drug.
        </p>
        <p class="headingAnchor" id="H16">
         <span class="h4">
          Virologic outcome
         </span>
         <span class="headingEndMark">
          —
         </span>
         Resistance thresholds based upon virologic outcome are considered to be a "gold standard," but can be difficult to define. There is no consensus yet on the appropriate magnitude or timing of the virologic response that should be used to define these thresholds; such thresholds can be influenced by the activity of the background regimen (other antiretroviral agents used in the combination). The PhenoSense assay lists suggested clinical cutoffs for the antiretroviral agents based on clinical outcome data.
        </p>
        <p>
         Studies performed to date suggest that virologic response may vary continuously with baseline resistance. For example, in one study (JAGUAR study),
         <a class="drug drug_general" data-topicid="9351" href="/z/d/drug information/9351.html" rel="external">
          didanosine
         </a>
         (ddI) or placebo was added to a failing ART regimen, and virologic outcomes at four weeks were compared with phenotypic testing results using the PhenoSense assay [
         <a href="#rid31">
          31
         </a>
         ]. The proportion of virologic responders was 83 percent (15 of 18) for patients with a ddI fold-change &lt;1.3, 50 percent (33 of 66) for patients with a fold-change of 1.3 to 2.2, and 29 percent (4 of 14) for patients with a fold-change of &gt;2.2. Although these data need to be validated in larger populations, they suggest that a single resistance cutoff that sensitively and specifically predicts virologic failure in all patients may be difficult to define.
        </p>
        <p class="headingAnchor" id="H2423754949">
         <span class="h1">
          ADDITIONAL RESISTANCE ASSAYS
         </span>
        </p>
        <p class="headingAnchor" id="H20">
         <span class="h2">
          Low-abundance drug-resistant variants
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genotypic and phenotypic assays are typically unable to detect low-abundance drug-resistant variants that are present at less than 20 percent of the viral quasispecies in a given sample. Other techniques have been developed to improve the detection of these minority species compared with standard drug resistance assays [
         <a href="#rid12">
          12,32
         </a>
         ]. The presence of low frequency-resistant variants can be associated with inferior treatment responses [
         <a href="#rid12">
          12,33-35
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H21">
         <span class="h3">
          Point mutation assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         Point mutation assays depend on the differential hybridization of oligonucleotide probes to the wild-type virus and mutant variants [
         <a href="#rid36">
          36
         </a>
         ]. These assays must be individually tailored for each mutation they are designed to detect. False-positive or false-negative test results can occur due to binding site variability [
         <a href="#rid32">
          32
         </a>
         ]. However, the low cost of this type of assay makes it a good candidate for drug resistance epidemiology studies, where information may be needed only on the most common resistance mutations.
        </p>
        <p class="headingAnchor" id="H22">
         <span class="h3">
          Clonal sequencing assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         Clonal sequencing assays can detect minority species, but are labor intensive; approximately 100 to 300 clones must be sequenced in order to identify a viral variant present in only 1 or 2 percent of the viral quasispecies [
         <a href="#rid37">
          37
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H23">
         <span class="h3">
          Deep sequencing
         </span>
         <span class="headingEndMark">
          —
         </span>
         Deep sequencing (DS) can detect minority species present at ≥1 percent using a technique whereby hundreds of thousands of individual molecules are sequenced in a single assay run [
         <a href="#rid12">
          12,38
         </a>
         ]. The clonal sequences are aligned and analyzed to evaluate the prevalence of low-abundance drug-resistant HIV strains. Data suggest that using DS may lead to a lower risk of virologic failure due to improved detection of minority species, which may be missed by standard genotypic testing [
         <a href="#rid12">
          12,32
         </a>
         ]. However, the significance of finding low-level resistant variants depends upon the specific antiretroviral agent and its genetic barrier to resistance.
        </p>
        <p>
         DS techniques were retrospectively applied to a subset of 264 treatment-naive patients within a large clinical trial on baseline samples prior to initiating antiretroviral therapy (ART) [
         <a href="#rid12">
          12
         </a>
         ]. A significantly higher proportion of mutations were detected by DS than by standard genotypic sequencing (28 versus 14 percent, respectively). Among patients who initiated treatment with an ART regimen that combined nucleoside and non-nucleoside reverse-transcriptase inhibitors (NNRTI), all individuals who had an NNRTI-resistance mutation identified by DS experienced virologic failure over a prolonged follow-up period.
        </p>
        <p>
         Subsequent data have shown that low-level variants with isolated protease inhibitor (PI) mutations were not predictive of virologic failure; this is likely due to the higher genetic barrier to resistance for ritonavir-boosted PI-based regimens [
         <a href="#rid39">
          39
         </a>
         ]. In contrast, subjects with low-level resistant variants with extensive nucleoside analog mutations had a high rate of virologic failure, as would be expected.
        </p>
        <p>
         One study of DS found that the detection of low-level resistant variants was more closely correlated with the patient's prior treatment history than with results of DS. These results confirm the importance of obtaining a thorough medical history [
         <a href="#rid40">
          40
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/13975.html" rel="external">
          "Evaluation of the treatment-experienced patient failing HIV therapy"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H4107900689">
         <span class="h2">
          HIV-1 proviral DNA assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         Some genotypic resistance assays are designed to analyze HIV-1 proviral DNA located in host cells. These tests can detect drug resistance mutations within proviral HIV DNA archived within peripheral blood mononuclear cells (Genosure Archive, Monogram Biosciences, South San Francisco, California) or determine viral tropism (HIV-1 Coreceptor Tropism, Proviral DNA, QUEST Diagnostics). (See
         <a class="local">
          'Tropism assays'
         </a>
         below.)
        </p>
        <p>
         Proviral DNA resistance testing may be useful when switching a patient's ART regimen when the HIV viral load is nondetectable and there are no prior resistance results available, or if there is low-level viremia and a plasma HIV RNA genotypic assay is unlikely to be successful [
         <a href="#rid41">
          41
         </a>
         ]. (See
         <a class="local">
          'Genotypic resistance assays'
         </a>
         above.)
        </p>
        <p>
         However, the clinical utility of proviral DNA resistance testing has yet to be fully determined [
         <a href="#rid41">
          41
         </a>
         ], and the results must be interpreted within the clinical context.
        </p>
        <p class="headingAnchor" id="H24">
         <span class="h2">
          Tropism assays
         </span>
        </p>
        <p class="headingAnchor" id="H25">
         <span class="h3">
          General background
         </span>
         <span class="headingEndMark">
          —
         </span>
         HIV requires both the CD4 receptor and a coreceptor in order to enter CD4+ T-cells. This coreceptor can be CCR5 or CXCR4. Viruses that use CCR5 as their coreceptor are referred to as R5 viruses; those that use CXCR4 are X4 viruses. An individual may have R5 viruses, X4 viruses, or a combination of both [
         <a href="#rid42">
          42
         </a>
         ]. Most transmitted viruses are R5 viruses; X4 viruses usually emerge later in infection.
        </p>
        <p>
         The frequency of exclusive R5 virus varies depending on the patient's treatment history; in treatment-naive patients, exclusive R5 virus is found in approximately 80 to 85 percent [
         <a href="#rid43">
          43,44
         </a>
         ]; in treatment-experienced patients with late-stage disease it is found in only 50 to 56 percent [
         <a href="#rid45">
          45-47
         </a>
         ].
        </p>
        <p>
         Testing for viral tropism is recommended when considering the use of a CCR5 antagonist (
         <a class="drug drug_general" data-topicid="9445" href="/z/d/drug information/9445.html" rel="external">
          maraviroc
         </a>
         ), which only inhibits R5 virus [
         <a href="#rid7">
          7
         </a>
         ]. Coreceptor tropism testing may also be helpful for patients who demonstrate virologic failure on a CCR5 inhibitor [
         <a href="#rid7">
          7
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H26">
         <span class="h3">
          Types of tropism assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         Both phenotypic and genotypic based assays have been developed to determine viral tropism, although there are much more clinical trial data regarding the use of phenotypic tropism assays [
         <a href="#rid48">
          48,49
         </a>
         ]. The United States Department of Health and Human Services (DHHS) HIV treatment guidelines recommend phenotypic assays as the preferred method for tropism testing; a genotypic assay can be considered as an alternative [
         <a href="#rid7">
          7
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H27">
         <span class="h4">
          Phenotypic tropism assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         There are two types of phenotypic assays (Trofile assay, LabCorp, Burlington, NC; Phenoscript assay, VIRAlliance, Paris, France) [
         <a href="#rid48">
          48,50-55
         </a>
         ]. Similar to the previously mentioned phenotypic resistance methods, these assays use laboratory viruses that express patient-derived viral envelope proteins [
         <a href="#rid48">
          48
         </a>
         ]. The laboratory-generated pseudovirus is used to infect cell lines that express either CCR5 or CXCR4; results of these infectivity assays determine the tropism of the patient's isolate.
        </p>
        <p>
         Although there were problems with the sensitivity of earlier assays, subsequent tropism assays are able to detect CXCR4-utilizing viruses at very low levels (ie, 0.3 percent of the viral population) [
         <a href="#rid54">
          54,56
         </a>
         ]. These new phenotypic tropism assays compare well to older established methods that used cell lines (eg, MT-2 cells) that do not express CCR5 to determine tropism [
         <a href="#rid48">
          48,53
         </a>
         ].
        </p>
        <p>
         The main limitations of these assays are the turnaround time needed for results (approximately three to four weeks from specimen collection) and the requirement for a plasma HIV RNA level ≥1000 copies/mL. Even with adequate viremia, tropism cannot be determined on a small number of patient samples. (See
         <a class="medical medical_review" href="/z/d/html/13978.html" rel="external">
          "Overview of antiretroviral agents used to treat HIV", section on 'CCR5 antagonists'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H28">
         <span class="h4">
          Genotypic tropism assays
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genotypic assays sequence specific regions in the
         <em>
          env
         </em>
         gene [
         <a href="#rid49">
          49,57
         </a>
         ]. Env, a major protein of HIV, determines HIV tropism. As with other genotypic methods, sequence algorithms are used to predict tropism.
        </p>
        <p>
         When compared with early generation phenotypic assays, population-based genotypic assays are specific but have lower sensitivity for the detection of CXCR4-using viruses [
         <a href="#rid58">
          58
         </a>
         ]. However, when used retrospectively to determine tropism in patients enrolled in clinical trials of
         <a class="drug drug_general" data-topicid="9445" href="/z/d/drug information/9445.html" rel="external">
          maraviroc
         </a>
         , newer genotypic assays performed as well as or better than the original Trofile assay in predicting virologic response and clinical outcomes [
         <a href="#rid57">
          57-59
         </a>
         ].
        </p>
        <p>
         The use of deep sequencing of the envelope protein can detect and quantify low prevalence subpopulations of CXCR4-using HIV within a large set of clinical isolates [
         <a href="#rid60">
          60,61
         </a>
         ]. This method surpassed the ability of the original Trofile assay to predict viral suppression and the likelihood of switching tropism after
         <a class="drug drug_general" data-topicid="9445" href="/z/d/drug information/9445.html" rel="external">
          maraviroc
         </a>
         exposure.
        </p>
        <p>
         European guidelines favor genotypic testing for determining coreceptor usage. In the United States, DHHS HIV treatment guidelines still recommend that a phenotype be used to determine coreceptor tropism. Although there are more data correlating phenotypic tropism results to outcome than with genotypes, the latter are less expensive and results return much more rapidly. As a result, genotype tropism testing is now an alternative option for clinical practice [
         <a href="#rid7">
          7
         </a>
         ].
        </p>
        <p>
         In patients with an undetectable viral load or low-level viremia (eg, HIV RNA &lt;1,000 copies/mL), HIV-1 proviral DNA tropism testing can be used to help determine co-receptor usage [
         <a href="#rid7">
          7,27
         </a>
         ]. As an example, this assay may be helpful when changing a patient with undetectable HIV RNA to a new regimen that contains
         <a class="drug drug_general" data-topicid="9445" href="/z/d/drug information/9445.html" rel="external">
          maraviroc
         </a>
         [
         <a href="#rid62">
          62,63
         </a>
         ]. If only R5 virus is detected, maraviroc could be considered for use in the new regimen. However, caution should be used when interpreting the results from these assays, as the clinical utility of this approach is still under investigation [
         <a href="#rid7">
          7,64
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H29">
         <span class="h1">
          SUMMARY AND RECOMMENDATIONS
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Resistance assays can be categorized as either genotypic or phenotypic. Genotypic resistance assays detect the presence of specific drug resistance mutations in the regions of the HIV genome encoding protease, reverse transcriptase, and integrase. Phenotypic resistance assays measure the extent to which an antiretroviral drug inhibits virus replication in vitro. (See
         <a class="local">
          'Overview of the assays'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         In most settings, genotypic assays are preferred due to lower cost, faster turnaround time, and greater sensitivity for detecting mixtures of resistant and wild-type virus. However, in treatment-experienced patients with multiple resistance mutations, interpreting complex genotypes can be challenging since it can be difficult to interpret the clinical significance of a specific mutation when many mutations are present. For such patients, phenotypic resistance testing may have an advantage over genotypic testing, as it measures resistance more directly and can assess relative susceptibility and interactions among mutations. (See
         <a class="local">
          'Advantages and limitations'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Genotypic and phenotypic assays are typically unable to detect low-abundance drug-resistant variants that are present at less than 20 percent of the viral quasispecies in a given sample. Other techniques have been developed to improve the detection of these minority species compared with standard drug resistance assays. (See
         <a class="local">
          'Low-abundance drug-resistant variants'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Some genotypic resistance assays are designed to analyze HIV-1 proviral DNA located in host cells. These tests can detect drug resistance mutations within proviral HIV DNA archived within peripheral blood mononuclear cells or determine viral tropism. This test may be useful when switching a patient's antiretroviral regimen when the HIV viral load is nondetectable and/or there are no prior resistance results available. However, the clinical utility of proviral DNA resistance testing or proviral DNA tropism detection has yet to be fully determined. (See
         <a class="local">
          'HIV-1 proviral DNA assays'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Testing for viral tropism should be performed when considering the use of the CCR5 antagonist
         <a class="drug drug_general" data-topicid="9445" href="/z/d/drug information/9445.html" rel="external">
          maraviroc
         </a>
         , since this agent only inhibits virus that uses the CCR5 coreceptor to enter the CD4 cells. Coreceptor tropism testing may also be helpful for patients who demonstrate virologic failure on a CCR5 inhibitor. Both phenotypic- and genotypic-based assays have been developed to determine viral tropism, although there are more clinical trial data regarding the use of phenotypic tropism assays. (See
         <a class="local">
          'Tropism assays'
         </a>
         above.)
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li>
          <a class="nounderline abstract_t">
           Larder BA, Darby G, Richman DD. HIV with reduced sensitivity to zidovudine (AZT) isolated during prolonged therapy. Science 1989; 243:1731.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Baxter JD, Mayers DL, Wentworth DN, et al. A randomized study of antiretroviral management based on plasma genotypic antiretroviral resistance testing in patients failing therapy. CPCRA 046 Study Team for the Terry Beirn Community Programs for Clinical Research on AIDS. AIDS 2000; 14:F83.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Palella FJ Jr, Armon C, Buchacz K, et al. The association of HIV susceptibility testing with survival among HIV-infected patients receiving antiretroviral therapy: a cohort study. Ann Intern Med 2009; 151:73.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Günthard HF, Calvez V, Paredes R, et al. Human Immunodeficiency Virus Drug Resistance: 2018 Recommendations of the International Antiviral Society-USA Panel. Clin Infect Dis 2019; 68:177.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kantor R, Smeaton L, Vardhanabhuti S, et al. Pretreatment HIV Drug Resistance and HIV-1 Subtype C Are Independently Associated With Virologic Failure: Results From the Multinational PEARLS (ACTG A5175) Clinical Trial. Clin Infect Dis 2015; 60:1541.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kuritzkes DR. Preventing and managing antiretroviral drug resistance. AIDS Patient Care STDS 2004; 18:259.
          </a>
         </li>
         <li class="breakAll">
          Panel on Antiretroviral Guidelines for Adults and Adolescents. Guidelines for the use of antiretroviral agents in HIV-1-infected adults and adolescents. Department of Health and Human Services. Available at http://aidsinfo.nih.gov/contentfiles/lvguidelines/AdultandAdolescentGL.pdf (Accessed on November 27, 2019).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Saag MS, Benson CA, Gandhi RT, et al. Antiretroviral Drugs for Treatment and Prevention of HIV Infection in Adults: 2018 Recommendations of the International Antiviral Society-USA Panel. JAMA 2018; 320:379.
          </a>
         </li>
         <li class="breakAll">
          British HIV Association. British HIV Association guidelines for the routine
investigation and monitoring of adult HIV-1-positive
individuals, 2016. http://www.bhiva.org/documents/Guidelines/Monitoring/2016-BHIVA-Monitoring-Guidelines.pdf (Accessed on October 23, 2017).
         </li>
         <li class="breakAll">
          Panel on Antiretroviral Guidelines for Adults and Adolescents. Guidelines for the use of antiretroviral agents in HIV-1-infected adults and adolescents. Department of Health and Human Services. Available at https://aidsinfo.nih.gov/contentfiles/lvguidelines/adultandadolescentgl.pdf (Accessed on March 27, 2018).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Metzner KJ, Giulieri SG, Knoepfel SA, et al. Minority quasispecies of drug-resistant HIV-1 that lead to early therapy failure in treatment-naive and -adherent patients. Clin Infect Dis 2009; 48:239.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Simen BB, Simons JF, Hullsiek KH, et al. Low-abundance drug-resistant viral variants in chronically HIV-infected, antiretroviral treatment-naive patients significantly impact treatment outcomes. J Infect Dis 2009; 199:693.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Fox ZV, Geretti AM, Kjaer J, et al. The ability of four genotypic interpretation systems to predict virological response to ritonavir-boosted protease inhibitors. AIDS 2007; 21:2033.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Paredes R, Tzou PL, van Zyl G, et al. Collaborative update of a rule-based expert system for HIV-1 genotypic resistance test interpretation. PLoS One 2017; 12:e0181357.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rhee SY, Fessel WJ, Liu TF, et al. Predictive value of HIV-1 genotypic resistance test interpretation algorithms. J Infect Dis 2009; 200:453.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Frentz D, Boucher CA, Assel M, et al. Comparison of HIV-1 genotypic resistance test interpretation systems in predicting virological outcomes over time. PLoS One 2010; 5:e11505.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wensing AM, Calvez V, Günthard HF, et al. 2017 Update of the Drug Resistance Mutations in HIV-1. Top Antivir Med 2016; 24:132.
          </a>
         </li>
         <li class="breakAll">
          HIV drug resistance database. Stanford University. Available at https://hivdb.stanford.edu/ (Accessed on January 18, 2022).
         </li>
         <li>
          <a class="nounderline abstract_t">
           De Luca A, Cingolani A, Di Giambenedetto S, et al. Variable prediction of antiretroviral treatment outcome by different systems for interpreting genotypic human immunodeficiency virus type 1 drug resistance. J Infect Dis 2003; 187:1934.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Van Laethem K, De Luca A, Antinori A, et al. A genotypic drug resistance interpretation algorithm that significantly predicts therapy response in HIV-1-infected patients. Antivir Ther 2002; 7:123.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Liu TF, Shafer RW. Web resources for HIV type 1 genotypic-resistance test interpretation. Clin Infect Dis 2006; 42:1608.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Desai S, Kyriakides T, Holodniy M, et al. Evolution of genotypic resistance algorithms and their impact on the interpretation of clinical trials: an OPTIMA trial substudy. HIV Clin Trials 2007; 8:293.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kellam P, Larder BA. Recombinant virus assay: a rapid, phenotypic assay for assessment of drug susceptibility of human immunodeficiency virus type 1 isolates. Antimicrob Agents Chemother 1994; 38:23.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Japour AJ, Mayers DL, Johnson VA, et al. Standardized peripheral blood mononuclear cell culture assay for determination of drug susceptibilities of clinical human immunodeficiency virus type 1 isolates. The RV-43 Study Group, the AIDS Clinical Trials Group Virology Committee Resistance Working Group. Antimicrob Agents Chemother 1993; 37:1095.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wrin T, Huang W, Yap J, et al. Evaluating HIV-1 co-receptor usage and inhibitors of virus entry using recombinant virus assays. Antivir Ther 2001; 6:3.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cooper DA, Heera J, Goodrich J, et al. Maraviroc versus efavirenz, both in combination with zidovudine-lamivudine, for the treatment of antiretroviral-naive subjects with CCR5-tropic HIV-1 infection. J Infect Dis 2010; 201:803.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Fabeni L, Berno G, Svicher V, et al. Genotypic Tropism Testing in HIV-1 Proviral DNA Can Provide Useful Information at Low-Level Viremia. J Clin Microbiol 2015; 53:2935.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Abstracts of the 3rd International Workshop on Adverse Drug Reactions and Lipodystrophy in HIV. 23-26 October 2001. Athens, Greece. Antivir Ther 2001; 6 Suppl 4:3.
          </a>
         </li>
         <li class="breakAll">
          Limoli K, Huang W, Toma S, et al. Validation and performance characteristics of the PhenoSense HIV fusion inhibitor susceptibility assay. 45th Annual ICAAC, Washington DC, 2005. Abstract H-1076.
         </li>
         <li>
          <a class="nounderline abstract_t">
           Demeter L, Haubrich R. International perspectives on antiretroviral resistance. Phenotypic and genotypic resistance assays: methodology, reliability, and interpretations. J Acquir Immune Defic Syndr 2001; 26 Suppl 1:S3.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Flandre P, Chappey C, Marcelin AG, et al. Phenotypic susceptibility to didanosine is associated with antiviral activity in treatment-experienced patients with HIV-1 infection. J Infect Dis 2007; 195:392.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Shafer RW. Low-abundance drug-resistant HIV-1 variants: finding significance in an era of abundant diagnostic and therapeutic options. J Infect Dis 2009; 199:610.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Halvas EK, Wiegand A, Boltz VF, et al. Low frequency nonnucleoside reverse-transcriptase inhibitor-resistant variants contribute to failure of efavirenz-containing regimens in treatment- experienced patients. J Infect Dis 2010; 201:672.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Paredes R, Lalama CM, Ribaudo HJ, et al. Pre-existing minority drug-resistant HIV-1 variants, adherence, and risk of antiretroviral treatment failure. J Infect Dis 2010; 201:662.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kuritzkes DR, Lalama CM, Ribaudo HJ, et al. Preexisting resistance to nonnucleoside reverse-transcriptase inhibitors predicts virologic failure of an efavirenz-based regimen in treatment-naive HIV-1-infected subjects. J Infect Dis 2008; 197:867.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Johnson JA, Li JF, Wei X, et al. Minority HIV-1 drug resistance mutations are present in antiretroviral treatment-naïve populations and associate with reduced treatment efficacy. PLoS Med 2008; 5:e158.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Garcia-Diaz A, Lok CB, Madge S, et al. Detection of low-frequency K103N mutants after unstructured discontinuation of efavirenz in the presence of the CYP2B6 516 TT polymorphism. J Antimicrob Chemother 2008; 62:1188.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Chabria SB, Gupta S, Kozal MJ. Deep sequencing of HIV: clinical and research applications. Annu Rev Genomics Hum Genet 2014; 15:295.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lataillade M, Chiarella J, Yang R, et al. Prevalence and clinical significance of HIV drug resistance mutations by ultra-deep sequencing in antiretroviral-naïve subjects in the CASTLE study. PLoS One 2010; 5:e10952.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Le T, Chiarella J, Simen BB, et al. Low-abundance HIV drug-resistant viral variants in treatment-experienced persons correlate with historical antiretroviral use. PLoS One 2009; 4:e6079.
          </a>
         </li>
         <li class="breakAll">
          United States National Institutes of Health. Guidelines for the Use of Antiretroviral Agents in Adults and Adolescents with HIV: Laboratory Testing, Drug-Resistance Testing. http://aidsinfo.nih.gov/guidelines/html/1/adult-and-adolescent-arv/6/drug-resistance-testing (Accessed on August 27, 2019).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hammer SM, Eron JJ Jr, Reiss P, et al. Antiretroviral treatment of adult HIV infection: 2008 recommendations of the International AIDS Society-USA panel. JAMA 2008; 300:555.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moyle GJ, Wildfire A, Mandalia S, et al. Epidemiology and predictive factors for chemokine receptor use in HIV-1 infection. J Infect Dis 2005; 191:866.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Brumme ZL, Goodrich J, Mayer HB, et al. Molecular and clinical epidemiology of CXCR4-using HIV-1 in a large population of antiretroviral-naive individuals. J Infect Dis 2005; 192:466.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wilkin TJ, Su Z, Kuritzkes DR, et al. HIV type 1 chemokine coreceptor use among antiretroviral-experienced patients screened for a clinical trial of a CCR5 inhibitor: AIDS Clinical Trial Group A5211. Clin Infect Dis 2007; 44:591.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Coakley E. HIV tropism testing in the clinic. J Viral Entry 2007; 3:10.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Koot M, Keet IP, Vos AH, et al. Prognostic value of HIV-1 syncytium-inducing phenotype for rate of CD4+ cell depletion and progression to AIDS. Ann Intern Med 1993; 118:681.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hosoya N, Su Z, Wilkin T, et al. Assessing human immunodeficiency virus type 1 tropism: Comparison of assays using replication-competent virus versus plasma-derived pseudotyped virions. J Clin Microbiol 2009; 47:2604.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lin NH, Kuritzkes DR. Tropism testing in the clinical management of HIV-1 infection. Curr Opin HIV AIDS 2009; 4:481.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Whitcomb JM, Huang W, Fransen S, et al. Development and characterization of a novel single-cycle recombinant-virus assay to determine human immunodeficiency virus type 1 coreceptor tropism. Antimicrob Agents Chemother 2007; 51:566.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Reeves JD, Coakley E, Petropoulos CJ, Whitcomb JM. An enhanced sensitivity Trofile HIV coreceptor tropism assay for selecting patients for therapy with entry inhibitors targeting CCR5: a review of analytical and clinical studies. J Viral Entry 2009; 3:94.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Raymond S, Delobel P, Mavigner M, et al. Correlation between genotypic predictions based on V3 sequences and phenotypic determination of HIV-1 tropism. AIDS 2008; 22:F11.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Coakley E, Reeves JD, Huang W, et al. Comparison of human immunodeficiency virus type 1 tropism profiles in clinical samples by the Trofile and MT-2 assays. Antimicrob Agents Chemother 2009; 53:4686.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Su Z, Gulick RM, Krambrink A, et al. Response to vicriviroc in treatment-experienced subjects, as determined by an enhanced-sensitivity coreceptor tropism assay: reanalysis of AIDS clinical trials group A5211. J Infect Dis 2009; 200:1724.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Trouplin V, Salvatori F, Cappello F, et al. Determination of coreceptor usage of human immunodeficiency virus type 1 from patient plasma samples by using a recombinant phenotypic assay. J Virol 2001; 75:251.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Collin G, Descamps D, Telles F, et al. Differences in protease and reverse transcriptase sequences between the TruGene HIV-1 genotyping kit (Visible Genetics) and the ViroSeq genotyping system (PE Applied biosystems). Antivir Ther 2000; 5:53.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kagan RM, Johnson EP, Siaw M, et al. A genotypic test for HIV-1 tropism combining Sanger sequencing with ultradeep sequencing predicts virologic response in treatment-experienced patients. PLoS One 2012; 7:e46334.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           McGovern RA, Thielen A, Mo T, et al. Population-based V3 genotypic tropism assay: a retrospective analysis using screening samples from the A4001029 and MOTIVATE studies. AIDS 2010; 24:2517.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           McGovern RA, Thielen A, Portsmouth S, et al. Population-based sequencing of the V3-loop can predict the virological response to maraviroc in treatment-naive patients of the MERIT trial. J Acquir Immune Defic Syndr 2012; 61:279.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Swenson LC, Mo T, Dong WW, et al. Deep sequencing to infer HIV-1 co-receptor usage: application to three clinical trials of maraviroc in treatment-experienced patients. J Infect Dis 2011; 203:237.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Swenson LC, Mo T, Dong WW, et al. Deep V3 sequencing for HIV type 1 tropism in treatment-naive patients: a reanalysis of the MERIT trial of maraviroc. Clin Infect Dis 2011; 53:732.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Bonjoch A, Pou C, Pérez-Álvarez N, et al. Switching the third drug of antiretroviral therapy to maraviroc in aviraemic subjects: a pilot, prospective, randomized clinical trial. J Antimicrob Chemother 2013; 68:1382.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Vitiello P, Brudney D, MacCartney M, et al. Responses to switching to maraviroc-based antiretroviral therapy in treated patients with suppressed plasma HIV-1-RNA load. Intervirology 2012; 55:172.
          </a>
         </li>
         <li class="breakAll">
          United States National Institutes of Health. Guidelines for the Use of Antiretroviral Agents in Adults and Adolescents with HIV: Laboratory Testing, Co-receptor Tropism Assays. https://aidsinfo.nih.gov/guidelines/html/1/adult-and-adolescent-arv/8/co-receptor-tropism-assays (Accessed on August 27, 2019).
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 3769 Version 21.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2467383" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : HIV with reduced sensitivity to zidovudine (AZT) isolated during prolonged therapy.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10894268" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : A randomized study of antiretroviral management based on plasma genotypic antiretroviral resistance testing in patients failing therapy. CPCRA 046 Study Team for the Terry Beirn Community Programs for Clinical Research on AIDS.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19620160" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : The association of HIV susceptibility testing with survival among HIV-infected patients receiving antiretroviral therapy: a cohort study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30052811" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : Human Immunodeficiency Virus Drug Resistance: 2018 Recommendations of the International Antiviral Society-USA Panel.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25681380" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : Pretreatment HIV Drug Resistance and HIV-1 Subtype C Are Independently Associated With Virologic Failure: Results From the Multinational PEARLS (ACTG A5175) Clinical Trial.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15186710" id="rid5" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          6 : Preventing and managing antiretroviral drug resistance.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15186710" id="rid6" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          7 : Preventing and managing antiretroviral drug resistance.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30043070" id="rid7" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          8 : Antiretroviral Drugs for Treatment and Prevention of HIV Infection in Adults: 2018 Recommendations of the International Antiviral Society-USA Panel.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30043070" id="rid8" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          9 : Antiretroviral Drugs for Treatment and Prevention of HIV Infection in Adults: 2018 Recommendations of the International Antiviral Society-USA Panel.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30043070" id="rid9" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          10 : Antiretroviral Drugs for Treatment and Prevention of HIV Infection in Adults: 2018 Recommendations of the International Antiviral Society-USA Panel.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19086910" id="rid10" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          11 : Minority quasispecies of drug-resistant HIV-1 that lead to early therapy failure in treatment-naive and -adherent patients.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19210162" id="rid11" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          12 : Low-abundance drug-resistant viral variants in chronically HIV-infected, antiretroviral treatment-naive patients significantly impact treatment outcomes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17885293" id="rid12" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          13 : The ability of four genotypic interpretation systems to predict virological response to ritonavir-boosted protease inhibitors.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28753637" id="rid13" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          14 : Collaborative update of a rule-based expert system for HIV-1 genotypic resistance test interpretation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19552527" id="rid14" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          15 : Predictive value of HIV-1 genotypic resistance test interpretation algorithms.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20634893" id="rid15" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          16 : Comparison of HIV-1 genotypic resistance test interpretation systems in predicting virological outcomes over time.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28208121" id="rid16" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          17 : 2017 Update of the Drug Resistance Mutations in HIV-1.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28208121" id="rid17" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          18 : 2017 Update of the Drug Resistance Mutations in HIV-1.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12792870" id="rid18" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          19 : Variable prediction of antiretroviral treatment outcome by different systems for interpreting genotypic human immunodeficiency virus type 1 drug resistance.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12212924" id="rid19" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          20 : A genotypic drug resistance interpretation algorithm that significantly predicts therapy response in HIV-1-infected patients.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16652319" id="rid20" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          21 : Web resources for HIV type 1 genotypic-resistance test interpretation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17956830" id="rid21" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          22 : Evolution of genotypic resistance algorithms and their impact on the interpretation of clinical trials: an OPTIMA trial substudy.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8141575" id="rid22" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          23 : Recombinant virus assay: a rapid, phenotypic assay for assessment of drug susceptibility of human immunodeficiency virus type 1 isolates.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8517697" id="rid23" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          24 : Standardized peripheral blood mononuclear cell culture assay for determination of drug susceptibilities of clinical human immunodeficiency virus type 1 isolates. The RV-43 Study Group, the AIDS Clinical Trials Group Virology Committee Resistance Working Group.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid24" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          25 : Evaluating HIV-1 co-receptor usage and inhibitors of virus entry using recombinant virus assays
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20151839" id="rid25" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          26 : Maraviroc versus efavirenz, both in combination with zidovudine-lamivudine, for the treatment of antiretroviral-naive subjects with CCR5-tropic HIV-1 infection.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/26135872" id="rid26" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          27 : Genotypic Tropism Testing in HIV-1 Proviral DNA Can Provide Useful Information at Low-Level Viremia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11700643" id="rid27" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          28 : Abstracts of the 3rd International Workshop on Adverse Drug Reactions and Lipodystrophy in HIV. 23-26 October 2001. Athens, Greece.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11700643" id="rid28" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          29 : Abstracts of the 3rd International Workshop on Adverse Drug Reactions and Lipodystrophy in HIV. 23-26 October 2001. Athens, Greece.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11264997" id="rid29" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          30 : International perspectives on antiretroviral resistance. Phenotypic and genotypic resistance assays: methodology, reliability, and interpretations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17205478" id="rid30" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          31 : Phenotypic susceptibility to didanosine is associated with antiviral activity in treatment-experienced patients with HIV-1 infection.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19210160" id="rid31" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          32 : Low-abundance drug-resistant HIV-1 variants: finding significance in an era of abundant diagnostic and therapeutic options.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20102272" id="rid32" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          33 : Low frequency nonnucleoside reverse-transcriptase inhibitor-resistant variants contribute to failure of efavirenz-containing regimens in treatment- experienced patients.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20102271" id="rid33" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          34 : Pre-existing minority drug-resistant HIV-1 variants, adherence, and risk of antiretroviral treatment failure.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18269317" id="rid34" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          35 : Preexisting resistance to nonnucleoside reverse-transcriptase inhibitors predicts virologic failure of an efavirenz-based regimen in treatment-naive HIV-1-infected subjects.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18666824" id="rid35" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          36 : Minority HIV-1 drug resistance mutations are present in antiretroviral treatment-naïve populations and associate with reduced treatment efficacy.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18786939" id="rid36" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          37 : Detection of low-frequency K103N mutants after unstructured discontinuation of efavirenz in the presence of the CYP2B6 516 TT polymorphism.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24821496" id="rid37" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          38 : Deep sequencing of HIV: clinical and research applications.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20532178" id="rid38" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          39 : Prevalence and clinical significance of HIV drug resistance mutations by ultra-deep sequencing in antiretroviral-naïve subjects in the CASTLE study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19562031" id="rid39" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          40 : Low-abundance HIV drug-resistant viral variants in treatment-experienced persons correlate with historical antiretroviral use.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19562031" id="rid40" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          41 : Low-abundance HIV drug-resistant viral variants in treatment-experienced persons correlate with historical antiretroviral use.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18677028" id="rid41" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          42 : Antiretroviral treatment of adult HIV infection: 2008 recommendations of the International AIDS Society-USA panel.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15717260" id="rid42" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          43 : Epidemiology and predictive factors for chemokine receptor use in HIV-1 infection.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15995960" id="rid43" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          44 : Molecular and clinical epidemiology of CXCR4-using HIV-1 in a large population of antiretroviral-naive individuals.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17243065" id="rid44" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          45 : HIV type 1 chemokine coreceptor use among antiretroviral-experienced patients screened for a clinical trial of a CCR5 inhibitor: AIDS Clinical Trial Group A5211.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid45" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          46 : HIV tropism testing in the clinic
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8096374" id="rid46" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          47 : Prognostic value of HIV-1 syncytium-inducing phenotype for rate of CD4+ cell depletion and progression to AIDS.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19494074" id="rid47" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          48 : Assessing human immunodeficiency virus type 1 tropism: Comparison of assays using replication-competent virus versus plasma-derived pseudotyped virions.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20048714" id="rid48" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          49 : Tropism testing in the clinical management of HIV-1 infection.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17116663" id="rid49" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          50 : Development and characterization of a novel single-cycle recombinant-virus assay to determine human immunodeficiency virus type 1 coreceptor tropism.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid50" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          51 : An enhanced sensitivity Trofile HIV coreceptor tropism assay for selecting patients for therapy with entry inhibitors targeting CCR5: a review of analytical and clinical studies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18753930" id="rid51" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          52 : Correlation between genotypic predictions based on V3 sequences and phenotypic determination of HIV-1 tropism.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19687240" id="rid52" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          53 : Comparison of human immunodeficiency virus type 1 tropism profiles in clinical samples by the Trofile and MT-2 assays.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19874179" id="rid53" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          54 : Response to vicriviroc in treatment-experienced subjects, as determined by an enhanced-sensitivity coreceptor tropism assay: reanalysis of AIDS clinical trials group A5211.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11119595" id="rid54" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          55 : Determination of coreceptor usage of human immunodeficiency virus type 1 from patient plasma samples by using a recombinant phenotypic assay.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid55" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          56 : Differences in protease and reverse transcriptase sequences between the TruGene HIV-1 genotyping kit (Visible Genetics) and the ViroSeq genotyping system (PE Applied biosystems)
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23029482" id="rid56" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          57 : A genotypic test for HIV-1 tropism combining Sanger sequencing with ultradeep sequencing predicts virologic response in treatment-experienced patients.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20736814" id="rid57" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          58 : Population-based V3 genotypic tropism assay: a retrospective analysis using screening samples from the A4001029 and MOTIVATE studies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23095934" id="rid58" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          59 : Population-based sequencing of the V3-loop can predict the virological response to maraviroc in treatment-naive patients of the MERIT trial.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21288824" id="rid59" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          60 : Deep sequencing to infer HIV-1 co-receptor usage: application to three clinical trials of maraviroc in treatment-experienced patients.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21890778" id="rid60" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          61 : Deep V3 sequencing for HIV type 1 tropism in treatment-naive patients: a reanalysis of the MERIT trial of maraviroc.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23354282" id="rid61" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          62 : Switching the third drug of antiretroviral therapy to maraviroc in aviraemic subjects: a pilot, prospective, randomized clinical trial.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22286889" id="rid62" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          63 : Responses to switching to maraviroc-based antiretroviral therapy in treated patients with suppressed plasma HIV-1-RNA load.
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
