<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Immunoglobulin genetics</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Immunoglobulin genetics</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Immunoglobulin genetics</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jolan Walter, MD, PhD</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jennifer M Puck, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Anna M Feldweg, MD</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Feb 02, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>The genetics of immunoglobulin during both B cell development in the bone marrow and the induction of the humoral immune response are presented here. A <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.imgt.org%2F&amp;token=bJ98wEjwYbR6KcGnbD%2BLoUVAtYLGNzB3CISQE9PwLfQ%3D&amp;TOPIC_ID=3977" target="_blank">website</a> devoted to immunoglobulin genetics is also available [<a href="#rid1">1</a>]. Discussions of B cell development, the structure of immunoglobulins, and the humoral immune response are reviewed separately. (See  <a class="medical medical_review" href="/d/html/13933.html" rel="external">"Normal B and T lymphocyte development"</a> and  <a class="medical medical_review" href="/d/html/3971.html" rel="external">"Structure of immunoglobulins"</a> and  <a class="medical medical_review" href="/d/html/3975.html" rel="external">"The adaptive humoral immune response"</a>.)</p><p class="headingAnchor" id="H822078571"><span class="h2">Terminology</span><span class="headingEndMark"> — </span>Immunoglobulin molecules are Y-shaped molecules composed of four chains: two identical light chains and two identical heavy chains (<a class="graphic graphic_figure graphicRef119775" href="/d/graphic/119775.html" rel="external">figure 1</a>). The terms "immunoglobulin" (Ig) and "antibody" are generally used interchangeably. This topic review preferentially uses the term "immunoglobulin," which can refer to a single molecule or collectively to all immunoglobulins considered together.</p><p class="headingAnchor" id="H913134692"><span class="h1">FORMS OF IMMUNOGLOBULIN</span><span class="headingEndMark"> — </span>Immunoglobulin exists in two forms: secreted and membrane bound, which are identical except for differences in the C-terminal portion, where one has a transmembrane anchor region and the other does not. These two different forms are generated by differential splicing of messenger ribonucleic acid (mRNA) [<a href="#rid2">2</a>].</p><p class="headingAnchor" id="H3323059606"><span class="h2">Secreted</span><span class="headingEndMark"> — </span>Secreted immunoglobulin is made mainly by plasma cells and may bind to microbes and act to neutralize, opsonize, fix complement, or serve other functions.</p><p class="headingAnchor" id="H656966005"><span class="h2">Membrane bound</span><span class="headingEndMark"> — </span>Membrane-bound immunoglobulin, in association with signal-transducing molecules, Ig<sub>alpha</sub> and Ig<sub>beta</sub>, serves as a receptor for antigen on the surface of B cells (BCR) (<a class="graphic graphic_figure graphicRef122838" href="/d/graphic/122838.html" rel="external">figure 2</a>). The role of the BCR is to recognize, bind, and present antigen specific for the variable region. This also creates a tonic signal for B cell survival during B cell development.</p><p class="headingAnchor" id="H2"><span class="h1">IMMUNOGLOBULIN GENE ORGANIZATION</span><span class="headingEndMark"> — </span>There are several classes of immunoglobulin that are categorized by a unique heavy chain structure: IgG, IgM, IgA, IgD, and IgE. There are also subclasses of IgG (IgG1 to 4) and of IgA (IgA1 and IgA2). In addition, there are two different types of immunoglobulin light chains, kappa and lambda. Immunoglobulin heavy chains and each type of light chain are encoded by genes in different loci. The table shows the locations of these gene complexes on human chromosomes (<a class="graphic graphic_table graphicRef64329" href="/d/graphic/64329.html" rel="external">table 1</a>). (See  <a class="medical medical_review" href="/d/html/3971.html" rel="external">"Structure of immunoglobulins"</a>.)</p><p>Genes capable of encoding a complete immunoglobulin heavy or light chain do not exist as such within the deoxyribonucleic acid (DNA) of most cells. The complete genes are assembled by the union of separate gene segments, termed "rearrangement." These segments are widely separated in germ cells and in all somatic cells, except for B lymphocytes. Within B lymphocytes, these genes become rearranged to create a "mature" immunoglobulin gene that can encode a functional protein. This rearrangement process is the core of the immune system's ability to generate antibodies capable of recognizing the tremendous variety of antigenic structures in nature.</p><p>An immunoglobulin heavy chain gene is assembled from four types of gene segments (<a class="graphic graphic_figure graphicRef72581" href="/d/graphic/72581.html" rel="external">figure 3</a>):</p><p class="bulletIndent1"><span class="glyph">●</span>The heavy chain variable region (V<sub>H</sub>)</p><p class="bulletIndent1"><span class="glyph">●</span>The heavy chain joining region (J<sub>H</sub>)</p><p class="bulletIndent1"><span class="glyph">●</span>The heavy chain constant region (C<sub>H</sub>)</p><p class="bulletIndent1"><span class="glyph">●</span>The diversity gene segment (D)</p><p></p><p>An immunoglobulin light chain gene is assembled from three types of gene segments (<a class="graphic graphic_figure graphicRef76331" href="/d/graphic/76331.html" rel="external">figure 4</a>). These are:</p><p class="bulletIndent1"><span class="glyph">●</span>The light chain variable region (V<sub>L</sub>)</p><p class="bulletIndent1"><span class="glyph">●</span>The light chain joining region (J<sub>L</sub>)</p><p class="bulletIndent1"><span class="glyph">●</span>The light chain constant region (C<sub>L</sub>)</p><p></p><p>The nomenclature of the gene segments derives from names assigned to portions of immunoglobulin heavy or light chains based upon an analysis of their amino acid sequences (see  <a class="medical medical_review" href="/d/html/3971.html" rel="external">"Structure of immunoglobulins"</a>). Thus, the variable region shows great variation from one immunoglobulin chain to another. The constant region, as its name implies, is invariant within an immunoglobulin class or subclass. The joining and diversity regions were the names given to areas between variable and constant, each having its own characteristic sequence pattern. The variable regions of the heavy and light chains together form the antibody-combining site. This is the portion of the immunoglobulin molecule that makes contact with antigen. (See  <a class="medical medical_review" href="/d/html/3970.html" rel="external">"Overview of therapeutic monoclonal antibodies"</a>.)</p><p>The organization of immunoglobulin heavy and light chain genes is very similar.</p><p>The immunoglobulin heavy chain locus on chromosome 14 has been completely sequenced [<a href="#rid1">1,3,4</a>]. It spans 957 kilobase pairs of DNA and contains a total of 123 V<sub>H</sub> genes. Only 39 of these genes are functional. Note that not all V<sub>H</sub> loci in all individuals will contain the same number of V<sub>H</sub> genes due to deletions in the region that can be inherited allelically (the same is true of V<sub>L</sub> loci). This leads to some individual variation in the number of V genes that can be expressed at each locus. There are 26 D genes, 6 J<sub>H</sub> genes (along with 3 nonfunctional J<sub>H</sub> pseudogenes), and 11 C<sub>H</sub> region genes (<a class="graphic graphic_figure graphicRef72581" href="/d/graphic/72581.html" rel="external">figure 3</a>). Each of these C<sub>H</sub> genes corresponds to a particular immunoglobulin class or subclass. (See  <a class="medical medical_review" href="/d/html/3971.html" rel="external">"Structure of immunoglobulins"</a>.)</p><p>The kappa locus on chromosome 2 consists of 79 V<sub>kappa</sub> genes, only 49 of which are functional (ie, capable of encoding a complete V<sub>kappa</sub> light chain segment). The others are pseudogenes that contain lethal mutations that prevent expression or translation of functional protein. The V<sub>kappa</sub> genes are followed by five J<sub>kappa</sub> genes and one C<sub>kappa</sub> gene (<a class="graphic graphic_figure graphicRef76331" href="/d/graphic/76331.html" rel="external">figure 4</a>) [<a href="#rid1">1,5</a>]. The human lambda locus on chromosome 22 is somewhat different, being composed of 73 V<sub>lambda</sub> (of which 33 are functional), 7 J<sub>lambda</sub>, and 7 C<sub>lambda</sub> genes (some of which are pseudogenes) [<a href="#rid1">1,6</a>].</p><p>Immunoglobulin V genes may also be found in other chromosomal locations where they cannot be expressed. These loci are called orphons and are believed to have arisen via gene conversions that have been maintained during evolution. V<sub>H</sub> orphons can be found on chromosomes 15 and 16 [<a href="#rid7">7</a>]. A V<sub>kappa</sub> orphon is found near the centromere of chromosome 2 [<a href="#rid8">8</a>]. Most genes in orphon loci are pseudogenes. Since they are not used to create mature immunoglobulin genes, there is no selection pressure to limit the accumulation of mutations that destroy the ability of these genes to encode a functional protein.</p><p>V<sub>H</sub> and V<sub>L</sub> genes are grouped into V gene families based upon nucleotide sequence homology. V genes that are greater than 80 percent homologous are said to belong to the same family. Homology between families is generally less than 75 percent. Human V<sub>H</sub> genes have been divided into seven families designated V1 to V7 [<a href="#rid3">3</a>]. Similarly, human V<sub>kappa</sub> and V<sub>lambda</sub> genes have been divided into 4 and 10 families, respectively [<a href="#rid5">5,6</a>]. D genes are also grouped into six families [<a href="#rid3">3</a>].</p><p>A typical V gene consists of two exons separated by an intron. The first V gene exon encodes most of the leader sequence, or signal peptide, a structure analogous to those found in many secreted proteins. This is the first part of the protein synthesized (the amino terminus), and this peptide interacts with the endoplasmic reticulum, which is important in the intracellular transport and processing of the protein. The signal peptide is removed from the immunoglobulin molecule before it appears in the membrane or is secreted. The second V gene exon encodes the majority of the V region (approximately 95 amino acids). D genes are variable in length and may encode 1 to 15 amino acids. J genes encode 15 to 20 amino acids. (See  <a class="medical medical_review" href="/d/html/3971.html" rel="external">"Structure of immunoglobulins"</a>.)</p><p class="headingAnchor" id="H3"><span class="h1">IMMUNOGLOBULIN GENE REARRANGEMENT</span><span class="headingEndMark"> — </span>The creation of a mature, expressible immunoglobulin heavy or light chain gene requires the assembly of randomly selected gene "modules" taken from groups of similar elements scattered widely over many kilobases of DNA, and this process is termed "rearrangement" [<a href="#rid9">9</a>]. With heavy chain gene rearrangement, the 3' end of a diversity (D) gene is brought to the 5' end of a heavy chain joining region (J<sub>H</sub>) gene (<a class="graphic graphic_figure graphicRef72581" href="/d/graphic/72581.html" rel="external">figure 3</a>). This is called D-J<sub>H</sub> rearrangement. Next, the 3' end of a heavy chain variable region (V<sub>H</sub>) gene is juxtaposed to DJ<sub>H</sub> (V-DJ<sub>H</sub> rearrangement). Light chains undergo only one rearrangement event, light chain variable region (V<sub>L</sub>) to light chain joining region (J<sub>L</sub>) (<a class="graphic graphic_figure graphicRef76331" href="/d/graphic/76331.html" rel="external">figure 4</a>).</p><p class="headingAnchor" id="H4265242401"><span class="h2">V(D)J recombination</span><span class="headingEndMark"> — </span>Immunoglobulin gene rearrangement is regulated by complex interactions between special DNA sequences, DNA-binding factors, and DNA-modifying enzymes [<a href="#rid9">9</a>]. V, D, and J genes all have flanking regions called "recombination signal sequences." These are important in the mechanism of rearrangement. The heptamer (7 base pairs [bp]) and nonamer (9 bp) sequences are separated by a spacer of either 12 or 23 bp (<a class="graphic graphic_figure graphicRef62896" href="/d/graphic/62896.html" rel="external">figure 5</a>). A gene with a flanking sequence containing a 12-bp spacer may only join to a gene whose flanking sequence has a 23-bp spacer. This is called the 12- to 23-bp rule. These sequences are recognized by DNA-binding proteins, such as nucleases and ligases with specific activity of opening of the DNA hairpin and splicing with nonhomologous end joining mechanisms. The conservation of these flanking sequences in many species indicates their importance in the rearrangement mechanism.</p><p class="headingAnchor" id="H811442856"><span class="h3">Role of RAG proteins</span><span class="headingEndMark"> — </span>The process of gene rearrangement (V(D)J recombination) begins with complex regulatory mechanisms that affect the chromatin structure of the DNA regions to be cut and ligated. This regulation and selective unmasking or unpackaging of DNA help to ensure that the splicing mechanism does not go awry and lead to pathogenic translocations [<a href="#rid9">9</a>]. The gene splicing that occurs in immunoglobulin gene rearrangement is mediated via one or more complexes of proteins that cut the two DNA sites that will be joined and ligate the DNA, thereby holding the two segments in their newly spliced configuration. Recombination-activating gene 1 (RAG1) and RAG2, referred to collectively here as RAG genes, encode lymphoid-specific proteins that are expressed during the early stages of T cell and B cell development and initiate the process of V(D)J recombination by introducing DNA double-strand breaks at the junction between the heptamer and a coding element [<a href="#rid10">10</a>].</p><p>RAG1 binds directly to the recombination signal sequences and also binds RAG2<em>, </em>which functions as a reader of the histones surrounding the DNA and stabilizer of DNA-RAG1 interaction [<a href="#rid9">9</a>]. Two RAG1 and two RAG2 proteins combine to create a heterotetramer complex, which makes a double-stranded break in the DNA and recruits several additional molecules that complete the juxtaposition of the coding DNA ends and their ligation. RAG mutations have been linked to severe combined immunodeficiency but also a widening spectrum of other immune disorders in children and adults [<a href="#rid10">10,11</a>]. (See  <a class="medical medical_review" href="/d/html/3950.html" rel="external">"T-B-NK+ SCID: Pathogenesis, clinical manifestations, and diagnosis", section on 'RAG complex (initiation of recombination)'</a>.)</p><p>This second stage of the V(D)J recombination event is facilitated by several factors collectively called the "nonhomologous end joining" complex. Similarly to RAG mutations, pathogenic mutations in many of the members of this complex are linked to combined immunodeficiency phenotype (Ku70, Ku80, DNA-dependent protein kinase catalytic subunit, Xrcc4, DNA ligase IV, or Artemis). (See  <a class="medical medical_review" href="/d/html/3950.html" rel="external">"T-B-NK+ SCID: Pathogenesis, clinical manifestations, and diagnosis", section on 'Nonhomologous DNA end joining (recombination and DNA repair)'</a>.)</p><p>In the bone marrow, recognition of polyvalent self-antigens by immature B cells promotes reexpression of the RAG genes and continued rearrangement of kappa and, if necessary, lambda light chain genes to generate a novel light chain that is no longer self-reactive [<a href="#rid12">12-15</a>]. This process, called "receptor editing," is considered an important central B cell tolerance checkpoint. In fact, it is found to be reduced in patients and animal models of RAG deficiency and is proposed to contribute to autoreactive B cells and autoimmunity [<a href="#rid16">16,17</a>].</p><p class="headingAnchor" id="H543024180"><span class="h2">Formation of KRECs</span><span class="headingEndMark"> — </span>During gene rearrangement, the DNA that was between the two rearranging genes is formed into a closed DNA circle. In maturing B cells, after exhaustive rearrangement of the V<sub>kappa</sub> segments, a specific by-product is created, which contains the joined intronic recombination signal sequence (RSS) and K deleting element (Kde) and C<sub>kappa</sub>. These circular DNA byproducts are called kappa-deleting recombination excision circles (KRECs) (<a class="graphic graphic_figure graphicRef76331" href="/d/graphic/76331.html" rel="external">figure 4</a>),<strong> </strong>and the B cell is class-switching to C<sub>lambda </sub>[<a href="#rid18">18</a>]. A similar mechanism occurs in developing T cells, giving rise to T cell receptor excision circles (TRECs).</p><p>The KRECs produced during immunoglobulin gene rearrangement are retained in the nucleus of the B cell. They are not replicated during subsequent cell divisions, but they persist in the nucleus of one of the progeny cells after division. Thus, the fraction of the B cells bearing these circles gradually diminishes with successive rounds of cell division during development in the bone marrow. A small fraction of mature B cells in the circulation will have these DNA circles in their nuclei. The number of KRECs in peripheral blood lymphocyte DNA can be measured by quantitative polymerase chain reaction (PCR), and this can be used as a marker of the rate of B cell generation [<a href="#rid18">18</a>]. For example, the number of KRECs is very low in patients with absent or severely reduced B cells such as X-linked agammaglobulinemia or a subset of patients with common variable immunodeficiency [<a href="#rid19">19,20</a>]. KREC measurement is being studied as a clinical marker of B cell development and for newborn screening for defects of B cell development [<a href="#rid20">20,21</a>]. This type of assay is analogous to measurement of TRECs, which are used in newborn screening for T cell and combined immunodeficiencies [<a href="#rid22">22</a>]. (See  <a class="medical medical_review" href="/d/html/16363.html" rel="external">"Newborn screening for inborn errors of immunity", section on 'Screening for SCID and other T cell defects'</a> and  <a class="medical medical_review" href="/d/html/16363.html" rel="external">"Newborn screening for inborn errors of immunity", section on 'Screening for B cell defects'</a>.)</p><p class="headingAnchor" id="H4"><span class="h1">ANTIBODY DIVERSITY</span><span class="headingEndMark"> — </span>Several mechanisms generate immunoglobulin region (combining site) diversity in both secreted and membrane-bound immunoglobulin molecules:</p><p class="bulletIndent1"><span class="glyph">●</span>Multiple variable (V), diversity (D), and joining (J) genes (germline-encoded diversity)</p><p class="bulletIndent1"><span class="glyph">●</span>Rearrangement diversity with imprecise joining of gene segments</p><p class="bulletIndent1"><span class="glyph">●</span>Nongermline-encoded nucleotides (N regions)</p><p class="bulletIndent1"><span class="glyph">●</span>Pairing of heavy and light chains (combinatorial diversity)</p><p class="bulletIndent1"><span class="glyph">●</span>Somatic mutation [<a href="#rid23">23</a>]</p><p class="bulletIndent1"><span class="glyph">●</span>Clonal redemption [<a href="#rid24">24,25</a>]</p><p class="bulletIndent1"><span class="glyph">●</span>Gene replacements (receptor editing) [<a href="#rid26">26</a>]</p><p></p><p>These mechanisms allow for highly efficient use of a relatively small portion of the genome to generate population of antibody specificities so diverse that, no matter what antigens are encountered, a complementary antibody may be found. Some of these mechanisms (eg, receptor editing, clonal redemption) also ensure that autoreactive prone clones are purged during the diversification process and considered as "B cell tolerance checkpoints."</p><p>Note that all of the mechanisms that create antibody diversity also add significant cellular wastage. A series of gene rearrangements may create nonfunctional genes by altering the reading frame or by producing an early termination codon. In addition, several germline V genes are pseudogenes and encode nonfunctional proteins. In order to produce a first functional B cell receptor (BCR) and later immunoglobulin, two successful series of rearrangements are required, one for heavy and one for light chain genes. Each series of rearrangements is susceptible to error. Since signaling through the BCR is essential in the survival of B cells, developing B cells can be lost to aberrant immunoglobulin gene rearrangements. The exact fraction of cells that is lost is unknown, but it is probably significantly greater than 50 percent [<a href="#rid27">27</a>].</p><p class="headingAnchor" id="H5"><span class="h2">Germline-encoded diversity</span><span class="headingEndMark"> — </span>A large number of different heavy chain VDJ combinations may be assembled. The completely sequenced heavy chain variable region (V<sub>H</sub>) locus contains 39 usable V<sub>H</sub> genes, 26 D, and 6 heavy chain joining region (J<sub>H</sub>) genes (see <a class="local">'Immunoglobulin gene organization'</a> above). Assuming any V<sub>H</sub> can join to any D, and any D to any J<sub>H</sub>, 6084 different heavy chain VDJ units may therefore be assembled. The germline-encoded V<sub>H</sub> diversity is amplified by 156-fold, simply by combining V<sub>H</sub> gene segments with various D and J<sub>H</sub> segments. This may be an overestimate because it appears that some V<sub>H</sub> genes preferentially rearrange [<a href="#rid28">28</a>]. However, it is clear that the multiplicative nature of immunoglobulin gene rearrangement greatly extends the diversity that can be encoded in the genome.</p><p class="headingAnchor" id="H6"><span class="h2">Rearrangement diversity</span><span class="headingEndMark"> — </span>Since the joining of D to J<sub>H</sub> and V<sub>H</sub> to DJ<sub>H</sub> (similarly, light chain variable region [V<sub>L</sub>] to light chain joining region [J<sub>L</sub>]) is not precise, it does not always occur at the same position in the gene segments. This results in different codons at the joints and in different complementarity-determining region (CDR) 3 lengths (<a class="graphic graphic_figure graphicRef62896" href="/d/graphic/62896.html" rel="external">figure 5</a>).</p><p class="headingAnchor" id="H7"><span class="h2">Nongermline-encoded nucleotides</span><span class="headingEndMark"> — </span>Nucleotides that are not encoded by genomic DNA may exist at the gene segment junctions. These nucleotides are believed to be inserted at the junction by the enzyme terminal deoxyribonucleotidyl transferase and are designated the N region. This process contributes to diversity by changing the length of the resultant protein, and, if a nonmultiple of three nucleotides is added, the reading frame is also shifted.</p><p class="headingAnchor" id="H8"><span class="h2">Combinatorial diversity</span><span class="headingEndMark"> — </span>Additional diversity is created by the pairing of the heavy and light chains produced by the cell. Although a given V<sub>H</sub> cannot pair with all light chain V genes to form a functional antibody [<a href="#rid29">29</a>], the assembly of antibodies from two different component chains leads to another multiplicative increase in the possible repertoire of combining sites. The total number capable of being generated is equal to the product of the number of different heavy chains and the number of different light chains that can form functional pairs. As an example, 3198 different possible combinations may arise because there are 39 functional V<sub>H</sub> genes and 82 V<sub>kappa</sub> and V<sub>lambda</sub> genes. Even if not all of these pairs are functional, this represents an approximately 15- to 30-fold increase over germline-encoded diversity.</p><p>The parts of the immunoglobulin V region that contact antigen are called "complementarity-determining regions" (CDRs) (see  <a class="medical medical_review" href="/d/html/3971.html" rel="external">"Structure of immunoglobulins"</a>). The CDRs 1 and 2 are encoded by the 5' regions of V<sub>H</sub> genes, and they reflect predominantly germline-encoded sequence diversity. All of the rearrangement mechanisms described above create diversity within the most 3' CDR3, built up from the 3' end of a V<sub>H</sub>, a D, and the 5' end of a J<sub>H</sub> gene. One analysis estimates the overall number of potential heavy chain CDR3 sequences to be on the order of 10<sup>14</sup> [<a href="#rid23">23</a>].</p><p class="headingAnchor" id="H9"><span class="h2">Somatic hypermutation</span><span class="headingEndMark"> — </span>Somatic hypermutation (SHM) is another important mechanism for generating variation in V regions [<a href="#rid23">23</a>]. At some time after rearrangement is complete, the heavy and light chain V genes may accumulate point mutations in any CDR or framework region. One gene may collect up to 10 or more nucleotide changes, some fraction of which result in an amino acid substitution. These changes may alter antibody affinity or may even alter specificity. This process is important in the phenomenon of affinity maturation that occurs in germinal centers, although it may also be found outside these areas [<a href="#rid30">30</a>]. (See  <a class="medical medical_review" href="/d/html/3975.html" rel="external">"The adaptive humoral immune response"</a>.)</p><p>SHM is often induced in B cells after CD40-CD40L interaction with follicular helper T cells and executed by B cell intrinsic activation induced cytidine deaminase (AID) and uracil DNA glycosylase (UNG). Defects in this pathway are linked to hyperimmunoglobulin M syndrome [<a href="#rid31">31</a>]. (See  <a class="medical medical_review" href="/d/html/13567.html" rel="external">"Hyperimmunoglobulin M syndromes", section on 'Pathogenesis'</a>.)</p><p class="headingAnchor" id="H199060723"><span class="h2">Clonal redemption</span><span class="headingEndMark"> — </span>A variant of SHM is coined "clonal redemption." Anergic B cells with BCRs specific for both foreign and self-antigens can be rescued from anergy and autoreactivity in a secondary immune response via SHM and undergo class-switching to IgG in the process of clonal redemption [<a href="#rid24">24,25</a>]. This process has been examined in healthy individuals with viral infections [<a href="#rid24">24,25</a>] and among patients with Toll-like receptor (TLR) defects [<a href="#rid32">32</a>] and AID deficiency [<a href="#rid33">33</a>].</p><p class="headingAnchor" id="H11"><span class="h2">Gene replacements</span><span class="headingEndMark"> — </span>Gene replacements (receptor editing) occur during B cell development and antibody responses. These underlie selective processes critical for the formation of the primary and secondary antibody repertoires. (See  <a class="medical medical_review" href="/d/html/13933.html" rel="external">"Normal B and T lymphocyte development"</a> and  <a class="medical medical_review" href="/d/html/3975.html" rel="external">"The adaptive humoral immune response"</a>.)</p><p class="headingAnchor" id="H12"><span class="h1">ALLELIC EXCLUSION</span><span class="headingEndMark"> — </span>Since there are two copies of each chromosome within a cell, it is theoretically possible that a B cell may produce two different immunoglobulin heavy chains or two different light chains. This could result in the formation of several different antibodies in a single cell. However, this does not occur because of a phenomenon termed "allelic exclusion." During the null B cell stage, diversity-heavy chain joining region (D-J<sub>H</sub>) rearrangement usually occurs in both heavy chain loci before variable-diversity heavy chain joining region (V-DJ<sub>H</sub>) joining. If the first V-DJ<sub>H</sub> rearrangement on one chromosome is successful, a complete IgM (mu) heavy chain mRNA is made, IgM heavy chain protein is synthesized and complexed with surrogate light chains, then expressed on the cell surface as the pre-B cell receptor (pre-BCR). (See  <a class="medical medical_review" href="/d/html/13933.html" rel="external">"Normal B and T lymphocyte development"</a>.)</p><p>In addition to triggering rearrangement of light chain genes, the pre-BCR appears also to provide a negative signal inhibiting further heavy chain gene rearrangement on the other chromosome [<a href="#rid27">27,34</a>]. If the first heavy chain gene rearrangement is unsuccessful, no IgM heavy chain protein can be expressed, and the locus on the other chromosome is free to rearrange. If both rearrangements are unsuccessful, no immunoglobulin can be formed, and the cell soon dies by apoptosis (programmed cell death). Thus, only cells with one productive heavy chain rearrangement go on to rearrange light chain genes.</p><p>The kappa light chain genes usually rearrange before lambda genes. Successful kappa gene rearrangement allows formation of a complete immunoglobulin molecule, which seems to shut down further gene rearrangement, at least for a time (receptor editing may occur if the immunoglobulin is specific for self-antigen) (see  <a class="medical medical_review" href="/d/html/13933.html" rel="external">"Normal B and T lymphocyte development"</a>). If kappa rearrangement is unsuccessful (on both chromosomes), lambda genes then recombine. If no rearrangement produces a functional light chain, again no immunoglobulin can be formed, and the cell dies. Thus, this process of allelic exclusion ensures that a given B cell will produce only a single immunoglobulin specificity resulting from the combination of one heavy chain with one light chain. In an individual heterozygous with respect to a particular polymorphic heavy or light chain constant gene, a given B cell will produce antibodies of only one allotype. (See  <a class="medical medical_review" href="/d/html/3971.html" rel="external">"Structure of immunoglobulins"</a>.)</p><p class="headingAnchor" id="H13"><span class="h1">CLASS-SWITCHING</span><span class="headingEndMark"> — </span>After the mature B cell stage, a cell may change from production of IgM and IgD to synthesis of IgG, IgA, or IgE. This process is called "class-switching" (<a class="graphic graphic_figure graphicRef58440" href="/d/graphic/58440.html" rel="external">figure 6</a>) [<a href="#rid35">35</a>]. This is a stable change in the cell's genome and is transmitted to all progeny cells. Since class-switching yields the same variable diversity heavy chain joining region (VDJ<sub>H</sub>) in association with a different heavy chain constant region (C<sub>H</sub>) region, antigen specificity and idiotypic determinants remain unchanged. Only the heavy chain isotype is altered relative to the immunoglobulin that was produced before the switch. (See  <a class="medical medical_review" href="/d/html/3971.html" rel="external">"Structure of immunoglobulins"</a>.)</p><p>A single cell can produce both IgM and IgD by differential splicing of mRNA (<a class="graphic graphic_figure graphicRef50773" href="/d/graphic/50773.html" rel="external">figure 7</a>). The predominant mechanism of class-switching is somewhat similar to the process of VDJ<sub>H</sub> rearrangement in DNA. In the DNA, 5' to each of the C<sub>H</sub> genes (except C<sub>delta</sub>), there are 1 kb or more of a characteristic repetitive sequence, such as (GAGCT)n or (GGGT)n. These are called switch sequences (or switch regions or S regions). When a cell undergoes a class-switch, the S region 5' to C<sub>mu</sub> is brought to the S region 5' to a downstream C region (eg, g1) (<a class="graphic graphic_figure graphicRef58440" href="/d/graphic/58440.html" rel="external">figure 6</a>). The intervening DNA is either inverted or excised (looped out), just as in V gene recombination, although distinct cellular machinery is required to accomplish this recombination of single-stranded mRNA [<a href="#rid35">35</a>].</p><p>The fate of the DNA intervening between recombined sequences appears to be the same in V gene recombination and class-switching. However, the heptamer and nonamer V gene recombination sequences and S sequences are very different, and <em>RAG1</em> and <em>RAG2</em> are not involved in the class-switch. Factors of somatic mutation and class-switching are shared. A description of the cellular interactions and factors regulating class-switching can be found separately. (See  <a class="medical medical_review" href="/d/html/3975.html" rel="external">"The adaptive humoral immune response"</a>.)</p><p>The genetics of IgG subclasses are discussed separately. (See  <a class="medical medical_review" href="/d/html/3964.html" rel="external">"IgG subclasses: Physical properties, genetics, and biologic functions", section on 'Allotypes'</a>.)</p><p class="headingAnchor" id="H15"><span class="h1">CLONAL DISTRIBUTION OF B CELLS</span><span class="headingEndMark"> — </span>Each mature B cell expresses a more or less unique set of immunoglobulin heavy (H) and light (L) chain gene rearrangements and H + L combination. Each represents a genetically distinct clone [<a href="#rid27">27</a>]. All of the progeny B cells generated by mitosis of that cell (clonal expansion) are considered to belong to the same clone since they are genetically identical. Some divergence may occur during clonal expansion since an individual cell of a clone may class-switch or somatically mutate its variable genes differently from its siblings. Depending on one's point of view, one may designate the progeny of these genetic events as constituting new clones, yet they all remain clonally related since they originated from one cell. Thus, the circulating pool of B lymphocytes is comprised of many genetically distinct clones, each of which may have anywhere from one to thousands of members.</p><p class="headingAnchor" id="H98246844"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span>Genes capable of encoding a complete immunoglobulin (Ig) heavy or light chain do not exist as such within the deoxyribonucleic acid (DNA) of most cells. Instead, complete genes are assembled within B cells by the union of separate gene segments to create a "mature" immunoglobulin gene that can encode a functional protein. This rearrangement is critical to creating a diverse antibody repertoire capable of interacting with the tremendous variety of antigenic structures in nature. (See <a class="local">'Immunoglobulin gene organization'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>An immunoglobulin light chain gene is assembled from three types of gene segments. These are the heavy chain gene segments for the variable (V<sub>H</sub>), joining (J<sub>H</sub>), and constant (C<sub>H</sub>) regions, and another type of gene segment called D (diversity) (<a class="graphic graphic_figure graphicRef72581" href="/d/graphic/72581.html" rel="external">figure 3</a>). Similarly, light chain variable region (V<sub>L</sub>), joining region (J<sub>L</sub>), and constant region (C<sub>L</sub>) gene segments (<a class="graphic graphic_figure graphicRef76331" href="/d/graphic/76331.html" rel="external">figure 4</a>). (See <a class="local">'Immunoglobulin gene organization'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Immunoglobulin gene rearrangement is regulated by complex interactions between special DNA sequences, DNA-binding factors, and DNA-modifying enzymes. (See <a class="local">'Immunoglobulin gene rearrangement'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Several mechanisms of generating diversity contribute to the structural variations found within the combining site of an immunoglobulin molecule. These mechanisms include germline-encoded diversity, imprecise joining during rearrangement, nongermline-encoded nucleotides region diversity, combinatorial diversity, somatic hypermutation (SHM), and receptor editing. (See <a class="local">'Antibody diversity'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Once a mature B cell is producing IgM and IgD, it may "class-switch" to make IgG, IgA, or IgE. The cell then continues to make the isotype to which it has committed for the rest of its life, as does all of that cell's progeny. (See <a class="local">'Class-switching'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Each mature B cell expresses a unique set of immunoglobulin heavy (H) and light (L) chain gene rearrangements and H + L combination and represents a genetically distinct clone. Thus, the circulating pool of B lymphocytes is comprised of many genetically distinct clones, each of which may contain from one to thousands of members. (See <a class="local">'Clonal distribution of B cells'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The process for B cell receptor (BCR) expression and concomitant immunoglobulin secretion can be impaired in several variants of immune deficiencies. Pattern of changes in immunoglobulin levels and the BCR repertoire may give clues for the underlying genetic defect. (See <a class="local">'Role of RAG proteins'</a> above and <a class="local">'Somatic hypermutation'</a> above.)</p><p></p><p class="headingAnchor" id="H4014243761"><span class="h1">ACKNOWLEDGMENTS</span><span class="headingEndMark"> — </span>The UpToDate editorial staff acknowledges Francisco A Bonilla, MD, PhD, who contributed as an author, and E Richard Stiehm, MD, who contributed as a Section Editor, to earlier versions of this topic review.</p></div><div class="headingAnchor" id="references"><ol id="reference"><li class="breakAll">The International Immunogenetics Information System. www.imgt.org (Accessed on March 31, 2020).</li><li><a class="nounderline abstract_t">Borghesi L, Milcarek C. From B cell to plasma cell: regulation of V(D)J recombination and antibody secretion. Immunol Res 2006; 36:27.</a></li><li><a class="nounderline abstract_t">Matsuda F, Ishii K, Bourvagnet P, et al. The complete nucleotide sequence of the human immunoglobulin heavy chain variable region locus. J Exp Med 1998; 188:2151.</a></li><li class="breakAll">The International Immunogenetics Information System, IMGT Repertoire (IG and TR). http://www.imgt.org/IMGTrepertoire/LocusGenes/genetable/human/geneNumber.html#IGtotal (Accessed on February 24, 2020).</li><li><a class="nounderline abstract_t">Barbié V, Lefranc MP. The human immunoglobulin kappa variable (IGKV) genes and joining (IGKJ) segments. Exp Clin Immunogenet 1998; 15:171.</a></li><li><a class="nounderline abstract_t">Pallarès N, Frippiat JP, Giudicelli V, Lefranc MP. The human immunoglobulin lambda variable (IGLV) genes and joining (IGLJ) segments. Exp Clin Immunogenet 1998; 15:8.</a></li><li><a class="nounderline abstract_t">Nagaoka H, Ozawa K, Matsuda F, et al. Recent translocation of variable and diversity segments of the human immunoglobulin heavy chain from chromosome 14 to chromosomes 15 and 16. Genomics 1994; 22:189.</a></li><li><a class="nounderline abstract_t">Weichhold GM, Lautner-Rieske A, Zachau HG. Human immunoglobulin genes of the kappa type. The long-range map of an orphon V kappa gene region. Biol Chem Hoppe Seyler 1992; 373:1159.</a></li><li><a class="nounderline abstract_t">Cobb RM, Oestreich KJ, Osipovich OA, Oltz EM. Accessibility control of V(D)J recombination. Adv Immunol 2006; 91:45.</a></li><li><a class="nounderline abstract_t">Notarangelo LD, Kim MS, Walter JE, Lee YN. Human RAG mutations: biochemistry and clinical implications. Nat Rev Immunol 2016; 16:234.</a></li><li><a class="nounderline abstract_t">Lawless D, Geier CB, Farmer JR, et al. Prevalence and clinical challenges among adults with primary immunodeficiency and recombination-activating gene deficiency. J Allergy Clin Immunol 2018; 141:2303.</a></li><li><a class="nounderline abstract_t">Halverson R, Torres RM, Pelanda R. Receptor editing is the main mechanism of B cell tolerance toward membrane antigens. Nat Immunol 2004; 5:645.</a></li><li><a class="nounderline abstract_t">Gay D, Saunders T, Camper S, Weigert M. Receptor editing: an approach by autoreactive B cells to escape tolerance. J Exp Med 1993; 177:999.</a></li><li><a class="nounderline abstract_t">Tiegs SL, Russell DM, Nemazee D. Receptor editing in self-reactive bone marrow B cells. J Exp Med 1993; 177:1009.</a></li><li><a class="nounderline abstract_t">Radic MZ, Erikson J, Litwin S, Weigert M. B lymphocytes may escape tolerance by revising their antigen receptors. J Exp Med 1993; 177:1165.</a></li><li><a class="nounderline abstract_t">Cassani B, Poliani PL, Marrella V, et al. Homeostatic expansion of autoreactive immunoglobulin-secreting cells in the Rag2 mouse model of Omenn syndrome. J Exp Med 2010; 207:1525.</a></li><li><a class="nounderline abstract_t">Walter JE, Rucci F, Patrizi L, et al. Expansion of immunoglobulin-secreting cells and defects in B cell tolerance in Rag-dependent immunodeficiency. J Exp Med 2010; 207:1541.</a></li><li><a class="nounderline abstract_t">van Zelm MC, Szczepanski T, van der Burg M, van Dongen JJ. Replication history of B lymphocytes reveals homeostatic proliferation and extensive antigen-induced B cell expansion. J Exp Med 2007; 204:645.</a></li><li><a class="nounderline abstract_t">Kamae C, Nakagawa N, Sato H, et al. Common variable immunodeficiency classification by quantifying T-cell receptor and immunoglobulin κ-deleting recombination excision circles. J Allergy Clin Immunol 2013; 131:1437.</a></li><li><a class="nounderline abstract_t">van Zelm MC, van der Burg M, Langerak AW, van Dongen JJ. PID comes full circle: applications of V(D)J recombination excision circles in research, diagnostics and newborn screening of primary immunodeficiency disorders. Front Immunol 2011; 2:12.</a></li><li><a class="nounderline abstract_t">Nakagawa N, Imai K, Kanegane H, et al. Quantification of κ-deleting recombination excision circles in Guthrie cards for the identification of early B-cell maturation defects. J Allergy Clin Immunol 2011; 128:223.</a></li><li><a class="nounderline abstract_t">Serana F, Chiarini M, Zanotti C, et al. Use of V(D)J recombination excision circles to identify T- and B-cell defects and to monitor the treatment in primary and acquired immunodeficiencies. J Transl Med 2013; 11:119.</a></li><li><a class="nounderline abstract_t">Maizels N. Immunoglobulin gene diversification. Annu Rev Genet 2005; 39:23.</a></li><li><a class="nounderline abstract_t">Reed JH, Jackson J, Christ D, Goodnow CC. Clonal redemption of autoantibodies by somatic hypermutation away from self-reactivity during human immunization. J Exp Med 2016; 213:1255.</a></li><li><a class="nounderline abstract_t">Sabouri Z, Schofield P, Horikawa K, et al. Redemption of autoantibodies on anergic B cells by variable-region glycosylation and mutation away from self-reactivity. Proc Natl Acad Sci U S A 2014; 111:E2567.</a></li><li><a class="nounderline abstract_t">Azulay-Debby H, Melamed D. B cell receptor editing in tolerance and autoimmunity. Front Biosci 2007; 12:2136.</a></li><li><a class="nounderline abstract_t">Rajewsky K. Clonal selection and learning in the antibody system. Nature 1996; 381:751.</a></li><li><a class="nounderline abstract_t">Rao SP, Riggs JM, Friedman DF, et al. Biased VH gene usage in early lineage human B cells: evidence for preferential Ig gene rearrangement in the absence of selection. J Immunol 1999; 163:2732.</a></li><li><a class="nounderline abstract_t">Grey HM, Mannik M. Specificity of recombination of H and L chains from human gamma-G-myeloma proteins. J Exp Med 1965; 122:619.</a></li><li><a class="nounderline abstract_t">William J, Euler C, Christensen S, Shlomchik MJ. Evolution of autoantibody responses via somatic hypermutation outside of germinal centers. Science 2002; 297:2066.</a></li><li><a class="nounderline abstract_t">Davies EG, Thrasher AJ. Update on the hyper immunoglobulin M syndromes. Br J Haematol 2010; 149:167.</a></li><li><a class="nounderline abstract_t">Schickel JN, Glauzy S, Ng YS, et al. Self-reactive VH4-34-expressing IgG B cells recognize commensal bacteria. J Exp Med 2017; 214:1991.</a></li><li><a class="nounderline abstract_t">Cantaert T, Schickel JN, Bannock JM, et al. Decreased somatic hypermutation induces an impaired peripheral B cell tolerance checkpoint. J Clin Invest 2016; 126:4289.</a></li><li><a class="nounderline abstract_t">Cedar H, Bergman Y. Choreography of Ig allelic exclusion. Curr Opin Immunol 2008; 20:308.</a></li><li><a class="nounderline abstract_t">Stavnezer J, Guikema JE, Schrader CE. Mechanism and regulation of class switch recombination. Annu Rev Immunol 2008; 26:261.</a></li></ol></div><div id="topicVersionRevision">Topic 3977 Version 17.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="www.imgt.org" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : The International Immunogenetics Information System. www.imgt.org (Accessed on March 31, 2020).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17337763" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : From B cell to plasma cell: regulation of V(D)J recombination and antibody secretion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9841928" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : The complete nucleotide sequence of the human immunoglobulin heavy chain variable region locus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9841928" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : The complete nucleotide sequence of the human immunoglobulin heavy chain variable region locus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9813414" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : The human immunoglobulin kappa variable (IGKV) genes and joining (IGKJ) segments.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9619396" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : The human immunoglobulin lambda variable (IGLV) genes and joining (IGLJ) segments.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7959766" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Recent translocation of variable and diversity segments of the human immunoglobulin heavy chain from chromosome 14 to chromosomes 15 and 16.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1476663" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Human immunoglobulin genes of the kappa type. The long-range map of an orphon V kappa gene region.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16938538" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Accessibility control of V(D)J recombination.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26996199" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Human RAG mutations: biochemistry and clinical implications.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29477728" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Prevalence and clinical challenges among adults with primary immunodeficiency and recombination-activating gene deficiency.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15156139" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Receptor editing is the main mechanism of B cell tolerance toward membrane antigens.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8459227" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Receptor editing: an approach by autoreactive B cells to escape tolerance.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8459201" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Receptor editing in self-reactive bone marrow B cells.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8459210" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : B lymphocytes may escape tolerance by revising their antigen receptors.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20547828" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Homeostatic expansion of autoreactive immunoglobulin-secreting cells in the Rag2 mouse model of Omenn syndrome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20547827" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Expansion of immunoglobulin-secreting cells and defects in B cell tolerance in Rag-dependent immunodeficiency.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17312005" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Replication history of B lymphocytes reveals homeostatic proliferation and extensive antigen-induced B cell expansion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23273952" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Common variable immunodeficiency classification by quantifying T-cell receptor and immunoglobulinκ-deleting recombination excision circles.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22566803" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : PID comes full circle: applications of V(D)J recombination excision circles in research, diagnostics and newborn screening of primary immunodeficiency disorders.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21397315" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Quantification ofκ-deleting recombination excision circles in Guthrie cards for the identification of early B-cell maturation defects.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23656963" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Use of V(D)J recombination excision circles to identify T- and B-cell defects and to monitor the treatment in primary and acquired immunodeficiencies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16285851" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Immunoglobulin gene diversification.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27298445" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Clonal redemption of autoantibodies by somatic hypermutation away from self-reactivity during human immunization.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24821781" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Redemption of autoantibodies on anergic B cells by variable-region glycosylation and mutation away from self-reactivity.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17127450" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : B cell receptor editing in tolerance and autoimmunity.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8657279" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Clonal selection and learning in the antibody system.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10453015" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Biased VH gene usage in early lineage human B cells: evidence for preferential Ig gene rearrangement in the absence of selection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/4158438" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Specificity of recombination of H and L chains from human gamma-G-myeloma proteins.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12242446" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Evolution of autoantibody responses via somatic hypermutation outside of germinal centers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20180797" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Update on the hyper immunoglobulin M syndromes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28500047" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Self-reactive VH4-34-expressing IgG B cells recognize commensal bacteria.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27701145" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Decreased somatic hypermutation induces an impaired peripheral B cell tolerance checkpoint.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18400481" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Choreography of Ig allelic exclusion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18370922" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Mechanism and regulation of class switch recombination.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
