<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Classification, cytogenetics, and molecular genetics of acute lymphoblastic leukemia/lymphoma
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Classification, cytogenetics, and molecular genetics of acute lymphoblastic leukemia/lymphoma
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Classification, cytogenetics, and molecular genetics of acute lymphoblastic leukemia/lymphoma
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Authors:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Yanming Zhang, MD
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Michelle M Le Beau, PhD
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Jon C Aster, MD, PhD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Richard A Larson, MD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Alan G Rosmarin, MD
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            Jul 29, 2022.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H1">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         Acute lymphoblastic leukemia/lymphoma (ALL/LBL) refers to hematologic malignancies of lymphoid precursor cells. These entities are described as ALL/LBL because in this setting, leukemia and lymphoma are overlapping clinical presentations of the same disease; the systems for diagnosis and classification do not distinguish between leukemia and lymphoma. Broadly, ALL/LBL is divided into tumors of B cell and T cell lineage; rare tumors of natural killer (NK) cell lineage are also recognized. Immunophenotyping is required to determine the lineage of ALL/LBL because the different subtypes are morphologically indistinguishable.
        </p>
        <p>
         Most cases of ALL/LBL have cytogenetic and/or molecular abnormalities that are associated with distinctive phenotypes, prognostic features, and/or influence the choice of treatment. The World Health Organization (WHO) classification system uses immunophenotype and cytogenetic/molecular features to define specific categories of ALL/LBL [
         <a href="#rid1">
          1
         </a>
         ]. This topic will review the WHO classification of ALL/LBL and the associated cytogenetic and molecular abnormalities.
        </p>
        <p>
         Clinical manifestations, evaluation, and diagnosis of B cell ALL/LBL and T cell ALL/LBL in children and adults are discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/6244.html" rel="external">
          "Overview of the clinical presentation and diagnosis of acute lymphoblastic leukemia/lymphoma in children"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4686.html" rel="external">
          "Clinical manifestations, pathologic features, and diagnosis of B cell acute lymphoblastic leukemia/lymphoma"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/15773.html" rel="external">
          "Clinical manifestations, pathologic features, and diagnosis of precursor T cell acute lymphoblastic leukemia/lymphoma"
         </a>
         .)
        </p>
        <p>
         An overview of cytogenetic analysis in hematologic malignancies, including definitions, methods of detection, and genetic consequences of chromosomal translocations, is presented separately. (See
         <a class="medical medical_review" href="/z/d/html/4482.html" rel="external">
          "General aspects of cytogenetic analysis in hematologic malignancies"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3">
         <span class="h1">
          CYTOGENETIC AND MOLECULAR ABNORMALITIES
         </span>
         <span class="headingEndMark">
          —
         </span>
         Most cases of ALL/LBL have recurring cytogenetic and/or molecular abnormalities that contribute to the phenotype, natural history, and/or prognosis of the disease. Notably, B cell ALL/LBL (B-ALL/LBL) and T cell ALL/LBL (T-ALL/LBL) have different characteristic, recurring cytogenetic and molecular features.
        </p>
        <p class="headingAnchor" id="H2665264887">
         <span class="h2">
          Cytogenetic features
         </span>
         <span class="headingEndMark">
          —
         </span>
         Conventional karyotyping and fluorescence in situ hybridization (FISH) reveal recurring cytogenetic and/or molecular abnormalities in approximately 80 percent of cases of B-ALL/LBL and in 50 to 70 percent of T-ALL/LBL [
         <a href="#rid1">
          1,2
         </a>
         ]. Individual cases may manifest numerical abnormalities (eg, hyperdiploidy, hypodiploidy), structural changes (eg, translocations, inversions, deletions), or both. Most chromosomal findings in ALL/LBL are associated with specific molecular/genetic abnormalities [
         <a href="#rid3">
          3,4
         </a>
         ].
        </p>
        <p>
         The frequency of certain chromosomal abnormalities varies considerably between childhood and adult B-ALL/LBL; it is uncertain if there are karyotypic differences in childhood and adult T-ALL/LBL. Examples of differences in age-related cytogenetic findings in B-ALL/LBL include [
         <a href="#rid3">
          3,5-7
         </a>
         ]:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The t(9;22) is observed in 2 to 5 percent of children with ALL/LBL, compared with approximately one-third of adults .
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The t(12;21) is present in approximately 25 percent of children with B-ALL/LBL, compared with about 3 percent of adults .
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         A hyperdiploid karyotype is found in 30 to 40 percent of children, compared with 2 to 10 percent of adults.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The t(4;11) is present in up to 60 percent of infants younger than 12 months but in &lt;10 percent of adult ALL/LBL patients.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H2541923397">
         <span class="h2">
          Molecular features
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genes that are important in the regulation of lymphocyte growth and development are frequently mutated in ALL/LBL. However, except for those associated with specific chromosomal findings in B-ALL/LBL, specific molecular abnormalities are not used to categorize ALL/LBL.
        </p>
        <p>
         Mutations in genes encoding certain transcription factors (eg,
         <em>
          MYC
         </em>
         ,
         <em>
          IKZF1
         </em>
         ) or cell cycle regulators (eg,
         <em>
          TP53
         </em>
         ,
         <em>
          RB1, CDKN2A
         </em>
         ) are common, but they are generally associated with multiple World Health Organization (WHO) categories [
         <a href="#rid1">
          1
         </a>
         ]. As an example,
         <em>
          TP53
         </em>
         mutations were reported in 16 percent of B-ALL/LBL overall, but rates varied among disease categories (eg, the highest being 92 percent in B-ALL/LBL with low hypodiploidy) [
         <a href="#rid6">
          6
         </a>
         ]. The frequency of
         <em>
          TP53
         </em>
         abnormalities increased with age and was higher in B-ALL/LBL than in T-ALL/LBL (21 versus 8 percent, respectively). Abnormalities (eg, deletion, amplification, point mutation, or structural rearrangement) in
         <em>
          PAX5
         </em>
         (32 percent),
         <em>
          IKZF1
         </em>
         (28 percent), and other genes were reported in a study of 242 children with ALL/LBL [
         <a href="#rid8">
          8
         </a>
         ]. In another study, genomic analysis in 1988 children and adults identified 23 subtypes of B-ALL/LBL on the basis of recurring chromosome translocations, unique gene fusions, mutations, and/or distinctive gene expression profiles; only some of these subtypes correspond to the current WHO categories [
         <a href="#rid7">
          7
         </a>
         ].
        </p>
        <p>
         Gene expression profiles, microRNA expression, and epigenetic changes appear to have prognostic value in patients with ALL/LBL [
         <a href="#rid9">
          9-11
         </a>
         ]. However, with the exception of the distinctive gene expression profile of BCR-ABL1-like B cell ALL, these findings are not presently used to categorize ALL/LBL and their clinical value is not well-defined.
        </p>
        <p>
         Gene mutations, rearrangements, and other molecular abnormalities that contribute to the pathogenesis of T-ALL/LBL are described in greater detail below. (See
         <a class="local">
          'Non-TCR loci'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H2818364639">
         <span class="h1">
          WHO CLASSIFICATION OF ALL/LBL
         </span>
         <span class="headingEndMark">
          —
         </span>
         The World Health Organization (WHO) classification of tumors of hematopoietic and lymphoid tissues categorizes ALL/LBL according to lymphoid lineage, cytogenetic findings, and molecular features.
        </p>
        <p>
         The 2016 WHO classification of ALL/LBL  (
         <a class="graphic graphic_table graphicRef110239" href="/z/d/graphic/110239.html" rel="external">
          table 1
         </a>
         ) is based on immunophenotype, cytogenetic abnormalities, and molecular findings [
         <a href="#rid1">
          1,2
         </a>
         ]. ALL/LBL is broadly divided into tumors of B cell, T cell, and natural killer (NK) cell lineage, based on immunophenotype. In this system, cytogenetic and molecular features are relied on to subclassify B cell ALL/LBL (B-ALL/LBL), but not T cell ALL/LBL (T-ALL/LBL), early T cell precursor ALL/LBL, or NK cell ALL/LBL [
         <a href="#rid1">
          1,2
         </a>
         ]. The WHO categories of ALL/LBL are presented in the sections that follow.
        </p>
        <p>
         Details of the clinical evaluation, pathologic features, and diagnosis of ALL/LBL in children and adults are discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/6244.html" rel="external">
          "Overview of the clinical presentation and diagnosis of acute lymphoblastic leukemia/lymphoma in children"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4686.html" rel="external">
          "Clinical manifestations, pathologic features, and diagnosis of B cell acute lymphoblastic leukemia/lymphoma"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/15773.html" rel="external">
          "Clinical manifestations, pathologic features, and diagnosis of precursor T cell acute lymphoblastic leukemia/lymphoma"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H931242114">
         <span class="h1">
          B-ALL/LBL
         </span>
         <span class="headingEndMark">
          —
         </span>
         Most cases of B-ALL/LBL have a defining cytogenetic/molecular feature. Cytogenetic and molecular findings that are associated with phenotypic and/or prognostic features are fundamental to the WHO classification system  (
         <a class="graphic graphic_table graphicRef110239" href="/z/d/graphic/110239.html" rel="external">
          table 1
         </a>
         ) [
         <a href="#rid1">
          1,2,12,13
         </a>
         ]. Cases that do not have one of the defining features should be classified as B-ALL/LBL, not otherwise specified (NOS). (See
         <a class="local">
          'ALL/LBL, NOS'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H352544890">
         <span class="h2">
          B-ALL/LBL with recurrent genetic abnormalities
         </span>
        </p>
        <p class="headingAnchor" id="H3402544092">
         <span class="h3">
          t(9;22); BCR::ABL1
         </span>
         <span class="headingEndMark">
          —
         </span>
         The t(9;22)(q34.1;q11.2), called the Philadelphia (Ph) chromosome, generates the
         <em>
          BCR::ABL1
         </em>
         fusion gene. The resultant BCR::ABL1 protein is a constitutively active tyrosine kinase, and this subtype of ALL/LBL is treated with chemotherapeutic regimens that incorporate a BCR::ABL1 tyrosine kinase inhibitor (TKI). The Ph chromosome is also seen in chronic myeloid leukemia (CML); clinical manifestations of CML and the biology and molecular genetics of BCR::ABL1 are discussed in detail separately. (See
         <a class="medical medical_review" href="/z/d/html/4543.html" rel="external">
          "Clinical manifestations and diagnosis of chronic myeloid leukemia"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4537.html" rel="external">
          "Molecular genetics of chronic myeloid leukemia"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4487.html" rel="external">
          "Cellular and molecular biology of chronic myeloid leukemia"
         </a>
         .)
        </p>
        <p>
         The t(9;22) is present in one-third of all adult cases of B-ALL/LBL and in half of adult cases in individuals &gt;50 years; by contrast, only 5 percent of childhood B-ALL/LBL is
         <em>
          BCR::ABL1
         </em>
         positive [
         <a href="#rid3">
          3
         </a>
         ]. The clinical and morphologic features of t(9;22) are similar to other types of B-ALL/LBL. Although there may be organ involvement, purely lymphomatous presentations of B-ALL/LBL with t(9;22) are rare. The blasts in this subtype frequently express myeloid-associated markers, such as CD13 and CD33, and expression of CD25 is common in adults. Expression of both lymphoid and myeloid markers by this subtype may reflect transformation of a precursor cell that is less differentiated/more immature than other categories of ALL/LBL. In both adults and children, the presence of
         <em>
          BCR::ABL1
         </em>
         is associated with adverse outcomes, but treatment regimens that include a TKI have greatly improved outcomes. The higher frequency of t(9;22) in adults may account, in part, for the overall adverse prognosis of B-ALL/LBL in this age group.
        </p>
        <p>
         The t(9;22) creates a fusion of
         <em>
          BCR
         </em>
         at chromosome band 22q11.2 with
         <em>
          ABL1
         </em>
         (which encodes a cytoplasmic tyrosine kinase) at 9q34.1. Approximately half of tumors in adults with B-ALL/LBL with t(9;22) produce a 210 kilodalton (kD) fusion protein (p210) and the remainder produce a p190 fusion protein; most tumors in children produce the p190 isoform [
         <a href="#rid1">
          1
         </a>
         ]. The clinical behavior of tumors expressing p210 or p190 do not appear to differ.
        </p>
        <p>
         In addition to the Ph chromosome, many cases of B-ALL/LBL with t(9;22) have additional cytogenetic abnormalities (ACAs). In a large series of adult B-ALL/LBL with t(9;22), approximately two-thirds had ACAs; gain of a Ph chromosome, -7, +8, +X, and del(9p) were the most common secondary abnormalities [
         <a href="#rid14">
          14,15
         </a>
         ]. In the rare case when an additional subtype-defining cytogenetic finding is noted along with the t(9;22), the presence of t(9;22) appears to govern the clinical features of the disease and the disease should be subclassified as B-ALL/LBL with t(9;22). More than 80 percent of patients with B-ALL/LBL with t(9;22) have splicing abnormalities of
         <em>
          IKZF1
         </em>
         (which encodes a zinc finger protein required for normal lymphoid development); certain
         <em>
          IKZF1
         </em>
         isoforms may be associated with resistance to TKIs. However, abnormalities of
         <em>
          IKZF1
         </em>
         are also found in other categories of B-ALL/LBL and they may be associated with poor outcomes, independent of Ph chromosome status [
         <a href="#rid16">
          16,17
         </a>
         ]. As an example, a DNA copy number analysis of 221 patients with Ph chromosome-negative high-risk B-ALL/LBL and 258 unselected patients with B-ALL/LBL reported that patients with
         <em>
          IKZF1
         </em>
         deletions had an increased frequency of relapse at 5 and 10 years and resistance to chemotherapy [
         <a href="#rid18">
          18
         </a>
         ].
        </p>
        <p>
         Management of B-ALL/LBL with the t(9;22) includes treatment with a TKI, as discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/16661.html" rel="external">
          "Induction therapy for Philadelphia chromosome positive acute lymphoblastic leukemia in adults", section on 'Remission induction therapy'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H4051712815">
         <span class="h3">
          t(v;11q23.3)
         </span>
         <span class="headingEndMark">
          —
         </span>
         The category of B-ALL/LBL with t(v;11q23.3) is the most common category of B-ALL/LBL in infants. It is defined by the presence of a translocation between
         <em>
          KMT2A
         </em>
         (also called
         <em>
          MLL
         </em>
         ) at 11q23.3 and one of numerous fusion partners [
         <a href="#rid1">
          1
         </a>
         ]. KMT2A is a histone methyltransferase that regulates gene transcription via chromatin remodeling; target genes include
         <em>
          HOX
         </em>
         genes and
         <em>
          IKZF1
         </em>
         , both of which have critical roles in normal lymphocyte development [
         <a href="#rid19">
          19
         </a>
         ].
        </p>
        <p>
         Translocations involving 11q23.3 are seen in 5 to 7 percent of all B-ALL/LBL cases, but in 60 to 80 percent of cases in infants (&lt;1 year); it is less common in older children, but it again becomes slightly more common in adults [
         <a href="#rid1">
          1,19
         </a>
         ]. Clinically, it typically presents with very high white blood cell (WBC) counts (eg, &gt;100,000/microL) and frequent involvement of the central nervous system (CNS). Infant B-ALL/LBL with
         <em>
          KMT2A
         </em>
         translocations have a unique gene expression profile, suggesting that it constitutes a distinct disease entity. In a series of 1725 patients with pediatric B-ALL/LBL, children with tumors with
         <em>
          KMT2A
         </em>
         abnormalities had 50 and 60 percent five-year rates of event-free survival (EFS) and overall survival (OS), respectively, compared with 89 and 96 percent EFS and OS, respectively, in children with tumors belonging to the t(12;21) group [
         <a href="#rid5">
          5
         </a>
         ].
        </p>
        <p>
         There are no unique morphologic features that distinguish this category from other types of ALL/LBL. Leukemias with a deletion at 11q23.3 that do not have a
         <em>
          KMT2A
         </em>
         rearrangement are not included in this group. This category of B-ALL/LBL may show co-expression of myeloid and B cell markers and should be distinguished from leukemias that have distinct lymphoblastic and monoblastic populations (which can be confirmed by immunophenotyping); such cases should instead be classified as B/myeloid mixed phenotype leukemia (MPAL). (See
         <a class="medical medical_review" href="/z/d/html/116801.html" rel="external">
          "Mixed phenotype acute leukemia"
         </a>
         .)
        </p>
        <p>
         Notable rearrangement partners for
         <em>
          KMT2A
         </em>
         include:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          t(4;11)
         </strong>
         – The most common rearrangement is t(4;11)(q21.3;q23.3), which fuses
         <em>
          KMT2A
         </em>
         to
         <em>
          AFF1
         </em>
         [
         <a href="#rid19">
          19
         </a>
         ]. Patients characteristically have very high WBC counts, immature (progenitor B cell) immunophenotype (CD10-, CD19+, HLA-DR+), and co-expression of myeloid antigens (CD15+, CDw65+) [
         <a href="#rid5">
          5,19
         </a>
         ]. Reverse transcriptase-polymerase chain reaction (RT-PCR) for the
         <em>
          KMT2A-AFF1
         </em>
         transcript can be used for monitoring of measurable residual disease (MRD). (See
         <a class="medical medical_review" href="/z/d/html/4499.html" rel="external">
          "Detection of measurable residual disease in acute lymphoblastic leukemia/lymphoblastic lymphoma"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         B-ALL/LBL with the t(4;11) has a poor prognosis in both adults and children. As examples, adults achieved a 75 percent complete remission (CR) rate, but median EFS was only seven months; for children, the rate of CR was 88 percent, but median survival was 10 months; for infants, five-year EFS was only 29 percent [
         <a href="#rid5">
          5,7,8,20,21
         </a>
         ]. Among infants with
         <em>
          KMT2A
         </em>
         rearrangements, the t(4;11) may confer a particularly poor prognosis; as a result, intensified therapy, including transplantation, may be preferred in these patients [
         <a href="#rid5">
          5
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          t(11;19)
         </strong>
         – The second most common rearrangement is t(11;19)(q23.3;p13.3), which results in the
         <em>
          KMT2A::MLLT1
         </em>
         (also called
         <em>
          ENL
         </em>
         ) fusion, but approximately half of patients with this rearrangement have acute myeloid leukemia (AML; usually monoblastic). (See
         <a class="medical medical_review" href="/z/d/html/4544.html" rel="external">
          "Acute myeloid leukemia: Cytogenetic abnormalities"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H1280995514">
         <span class="h3">
          t(12;21)(p13.2;q22.1); ETV6::RUNX1
         </span>
         <span class="headingEndMark">
          —
         </span>
         The t(12;21)(p13.2;q22.1) is the most common genetic lesion in childhood ALL/LBL. It occurs in 15 to 25 percent of childhood precursor B-ALL/LBL, but it is less common in adult disease (3 to 4 percent) [
         <a href="#rid1">
          1
         </a>
         ].
        </p>
        <p>
         There are no distinctive clinical, morphologic, or cytochemical features that distinguish t(12;21) from other types of ALL/LBL [
         <a href="#rid1">
          1
         </a>
         ]. The t(12;21) is not easily detected by conventional karyotypic analysis because of the similarity in size and banding patterns of the 12p and 21q chromosome arms. As a result, fluorescence in situ hybridization (FISH) is generally required to detect this chromosomal rearrangement; its presence can also be inferred by using RT-PCR to detect the
         <em>
          ETV6::RUNX1
         </em>
         fusion transcript.
        </p>
        <p>
         Most patients with B-ALL/LBL with t(12;21) are 1 to 10 years old and have a favorable prognosis; &gt;90 percent of patients are cured [
         <a href="#rid1">
          1
         </a>
         ]. As examples, a study of 368 children with B-ALL/LBL reported that those with t(12;21) had 89 and 96 percent five-year EFS and OS, respectively, while another reported 91 percent five-year EFS [
         <a href="#rid5">
          5,22
         </a>
         ].
        </p>
        <p>
         The t(12;21) fuses
         <em>
          ETV6
         </em>
         (
         <em>
          TEL
         </em>
         ) at 12p13.2 with
         <em>
          RUNX1
         </em>
         at 21q22.1, creating a chimeric gene that encodes ETV6::RUNX1 fusion protein [
         <a href="#rid23">
          23
         </a>
         ]. Expression of ETV6::RUNX1 in marrow progenitor cells appears to induce a preleukemic stage of B-ALL/LBL development, but additional cooperating genetic mutations are required for the conversion into acute leukemia. The normal RUNX1 protein heterodimerizes with core binding factor beta (CBF beta) to form a functional transcription factor that regulates downstream target genes involved in numerous aspects of normal hematopoiesis. The t(12;21) fuses the region o
         <em>
          f ETV6
         </em>
         that encodes the 5' helix-loop-helix (HLH) dimerization domain with the region o
         <em>
          f RUNX1
         </em>
         that encodes the 3' DNA-binding and transactivation domain; the resulting fusion protein interferes with the transcriptional regulation of normal RUNX1 target genes. In many cases of B-ALL/LBL with the t(12;21), th
         <em>
          e ETV6
         </em>
         allele on the other chromosome 12 homolog is deleted, which suggests tha
         <em>
          t ETV6
         </em>
         functions as a tumor suppressor gene. Germline mutations in
         <em>
          ETV6
         </em>
         have been reported in association with germline predisposition to ALL [
         <a href="#rid24">
          24
         </a>
         ].
        </p>
        <p>
         <em>
          ETV6
         </em>
         is also involved in other leukemias and myeloproliferative neoplasms. As an example,
         <em>
          ETV6
         </em>
         is fused to the platelet-derived growth factor receptor (
         <em>
          PDGFR
         </em>
         )-beta gene in the t(5;12)(q32;p13.2), which is found in myeloid/lymphoid neoplasms with
         <em>
          PDGFRB
         </em>
         rearrangement (formerly classified as a subtype of chronic myelomonocytic leukemia) [
         <a href="#rid1">
          1
         </a>
         ]. Two other recurring
         <em>
          ETV6
         </em>
         fusions in myeloid neoplasms,
         <em>
          ETV6
         </em>
         ::
         <em>
          FLT3
         </em>
         in t(12;13) and
         <em>
          ETV6
         </em>
         ::
         <em>
          ABL1
         </em>
         in t(9;12), are recognized as additional causes of myeloid/lymphoid neoplasm with eosinophilia [
         <a href="#rid20">
          20,25
         </a>
         ]. In addition
         <em>
          ETV6::JAK2
         </em>
         and
         <em>
          ETV6::NTRK3
         </em>
         fusions have been reported in BCR::ABL1 like B-ALL [
         <a href="#rid26">
          26
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/2211.html" rel="external">
          "Hypereosinophilic syndromes: Clinical manifestations, pathophysiology, and diagnosis", section on 'Myeloproliferative HES variants'
         </a>
         .)
        </p>
        <p>
         Aberrations involving
         <em>
          RUNX1
         </em>
         are also found in other hematologic malignancies. These include
         <em>
          RUNX1
         </em>
         translocations, most commonly the t(8;21)(q22;q22.1) in a genetically distinct subset of AML, and pathogenic somatic mutations in a subset of myelodysplastic syndrome and AML. In addition, germline
         <em>
          RUNX1
         </em>
         mutations are the cause of familial platelet disorder with predisposition to AML, a rare autosomal dominant disorder associated with thrombocytopenia, platelet dysfunction leading to bleeding, and a high risk of myeloid neoplasia [
         <a href="#rid27">
          27
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/93183.html" rel="external">
          "Familial disorders of acute leukemia and myelodysplastic syndromes", section on 'Familial platelet disorder with propensity to myeloid malignancies (FPD)'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H9">
         <span class="h3">
          Hyperdiploidy
         </span>
         <span class="headingEndMark">
          —
         </span>
         B-ALL/LBL with hyperdiploidy is characterized by lymphoblasts that have &gt;50 chromosomes and lack translocations or other structural alterations.
        </p>
        <p>
         The clinical, morphologic, and cytochemical features of B-ALL/LBL with hyperdiploidy are similar to those of other categories [
         <a href="#rid1">
          1
         </a>
         ]. Hyperdiploid B-ALL/LBL accounts for up to one-quarter of B-ALL/LBL in children; it is not seen in infants, the frequency decreases in older children, and it accounts for &lt;10 percent of B-ALL/LBL in adults. Hyperdiploid ALL/LBL has a very favorable prognosis in children, with cure in &gt;90 percent of children; there are too few adult cases to assess its prognostic significance. Children with hyperdiploid ALL typically have other favorable clinical features, such as age one to nine years, low WBC count, and favorable immunophenotype (early pre-B or pre-B) [
         <a href="#rid5">
          5,11,28
         </a>
         ].
        </p>
        <p>
         The X chromosome and chromosomes 21, 14, and 4 are most commonly gained in hyperdiploid B-ALL/LBL; chromosomes 1, 2, and 3 are least often gained. Hyperdiploid B-ALL/LBL with trisomies of chromosomes 4 and 10 have the most favorable prognosis. There is controversy about whether the features of hyperdiploid ALL/LBL are related to gain of particular chromosomes or the total number of chromosomes gained. As described below, rare cases of apparent "pseudohyperdiploid" ALL/LBL may stem from tetraploidization of hypodiploid ALL/LBL.
        </p>
        <p class="headingAnchor" id="H10">
         <span class="h3">
          Hypodiploidy
         </span>
         <span class="headingEndMark">
          —
         </span>
         Hypodiploid B-ALL/LBL has lymphoblasts with &lt;46 chromosomes and is divided into subtypes based on the number of chromosomes; there may also be associated structural karyotypic abnormalities.
        </p>
        <p>
         Three distinct subgroups of hypodiploid ALL/LBL have been identified, which have different genetic findings and prognoses [
         <a href="#rid29">
          29,30
         </a>
         ]:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Near-haploid (23 to 29 chromosomes)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Low hypodiploid (33 to 39 chromosomes)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         High hypodiploid (40 to 45 chromosomes)
        </p>
        <p>
        </p>
        <p>
         There are no distinctive clinical, morphologic, or cytochemical features of this category. Hypodiploid B-ALL/LBL is uncommon, accounting for &lt;5 percent of cases [
         <a href="#rid31">
          31
         </a>
         ]. Although hypodiploid B-ALL/LBL occurs in both children and adults, the near-haploid category is limited to childhood. Importantly, the diagnosis of near-haploid or low hypodiploid B-ALL/LBL may be missed because the hypodiploid clone can undergo tetraploidization, which doubles the number of chromosomes, creating a near-diploid or hyperdiploid karyotype. Flow cytometric DNA index analysis or FISH assays to detect ploidy and copy number of selected chromosomes, and genomic microarray to detect allelic imbalances, such as loss of heterozygosity (LOH), can help clarify such ambiguous cases, which is important because of the distinctly different prognoses in these categories.
        </p>
        <p>
         Subgroups of hypodiploid B-ALL/LBL are associated with distinctive genetic abnormalities [
         <a href="#rid32">
          32
         </a>
         ]. Near-haploid B-ALL/LBL often has mutations of the RAS or PI3K signaling pathways, receptor tyrosine kinases, and/or
         <em>
          IKZF3
         </em>
         . The great majority of low hypodiploid cases have alterations of
         <em>
          IKZF2
         </em>
         , activation of RAS or PI3K signaling pathways, and/or loss-of-function mutations of
         <em>
          RB1
         </em>
         and/or
         <em>
          TP53
         </em>
         (some of which are germline). There are no specific gene alterations associated with high hypodiploid B-ALL/LBL.
        </p>
        <p>
         Hypodiploid B-ALL/LBL generally carries a poor prognosis, especially in patients with near-haploid and low-hypodiploid clones. As an example, one study showed that the three-year EFS of children with near-haploid and low-hypodiploid clones was 29 percent, compared with 66 percent for those with high hypodiploid clones (42 to 45 chromosomes) [
         <a href="#rid29">
          29
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H2690705618">
         <span class="h3">
          t(5;14)(q31.1;q32.1); IGH/IL3
         </span>
         <span class="headingEndMark">
          —
         </span>
         The category of B-ALL/LBL with t(5;14)(q31.1;q32.1) is exceedingly rare in both children and adults. The translocation generates a fusion of
         <em>
          IL3
         </em>
         and an
         <em>
          IGH
         </em>
         (immunoglobulin heavy chain) that causes overexpression of IL3 and is associated with variable degrees of eosinophilia.
        </p>
        <p>
         The clinical features can resemble other types of B-ALL/LBL, but some cases present with asymptomatic eosinophilia and few blasts in peripheral blood [
         <a href="#rid1">
          1
         </a>
         ]. The blasts have typical ALL/LBL morphology, but there may be an increase of eosinophils, which are not part of the leukemic clone (ie, they constitute a reactive population of eosinophils). There are too few reported cases to be certain of the prognosis of this subtype.
        </p>
        <p class="headingAnchor" id="H2551261987">
         <span class="h3">
          t(1;19)(q23;p13.3)/TCF3::PBX1
         </span>
         <span class="headingEndMark">
          —
         </span>
         The category of B-ALL/LBL with t(1;19)(q23;p13.3)/TCF3::PBX1 has blasts that harbor a translocation between
         <em>
          TCF3
         </em>
         (
         <em>
          E2A
         </em>
         ) on chromosome 19 and
         <em>
          PBX1
         </em>
         on chromosome 1.
        </p>
        <p>
         This category is relatively common in children (6 percent) and less common in adults [
         <a href="#rid1">
          1
         </a>
         ]. The morphologic and cytochemical features are similar to those of other categories of ALL/LBL. The blasts have a pre-B phenotype (ie, positive for CD19, CD10, and cytoplasmic mu heavy chain), and this rearrangement accounts for approximately one-third of cases of childhood pre-B cell ALL/LBL. Clinically, there is an increased risk for CNS relapse. Although the t(1;19) rearrangement was formerly considered to carry a poor prognosis, outcomes using contemporary intensive therapy are now comparable or more favorable than other categories of ALL/LBL [
         <a href="#rid5">
          5
         </a>
         ].
        </p>
        <p>
         This rearrangement creates a chimeric gene encoding a TCF3::PBX1 fusion protein, which contains the transcriptional activation domain of TCF3 and the DNA-binding and protein dimerization domains of PBX1 (a homeobox [HOX] protein). This fusion protein appears to interfere with functions of wild-type TCF3 and PBX1. The functional fusion gene is present on chromosome 19; in some cases, there is an unbalanced translocation with loss of the derivative chromosome 1. B-ALL/LBL with t(1;19) has a unique gene expression signature [
         <a href="#rid33">
          33
         </a>
         ].
        </p>
        <p>
         Some cases of B-ALL/LBL with a t(1;19) do not involve either
         <em>
          TCF3
         </em>
         or
         <em>
          PBX1
         </em>
         ; such cases should not be included in this category. Demonstration of a
         <em>
          TCF3
         </em>
         rearrangement alone is not sufficient to establish the presence of the t(1;19), as rare cases of B-ALL/LBL have a t(17;19) in which
         <em>
          TCF3
         </em>
         is translocated to
         <em>
          HLF
         </em>
         on chromosome 17; such cases are associated with a dismal prognosis.
        </p>
        <p class="headingAnchor" id="H610779371">
         <span class="h3">
          iAMP21
         </span>
         <span class="headingEndMark">
          —
         </span>
         B-ALL/LBL with iAMP21 is a provisional category in the WHO classification in which blasts have amplification of a portion of chromosome 21 [
         <a href="#rid1">
          1
         </a>
         ].
        </p>
        <p>
         This category accounts for 2 percent of cases of B-ALL/LBL and is usually found in older children who present with low WBC counts [
         <a href="#rid1">
          1
         </a>
         ]. There are no distinctive morphologic and cytochemical features. It is usually detected by FISH with a probe for
         <em>
          RUNX1
         </em>
         that reveals ≥5 copies of the gene or ≥3 extra copies on a single abnormal chromosome. Individuals with the rare constitutional Robertsonian translocation rob(15;21)(q10;q10)c have a nearly 3000-fold increased risk of developing this type of ALL/LBL [
         <a href="#rid34">
          34
         </a>
         ]. ALL/LBL with iAMP21 has been considered to carry a poor prognosis, although this is less clear in the era of intensified therapy [
         <a href="#rid35">
          35
         </a>
         ].
        </p>
        <p>
         Various structural abnormalities of 21q are found by cytogenetic analysis, including an increase in the length of 21q, consistent with gene amplification [
         <a href="#rid35">
          35
         </a>
         ]. Genomic microarray studies revealed complex genomic imbalances of 21q, with multiple gains of most of the long arm of the chromosome, along with deletion of the subtelomeric region [
         <a href="#rid36">
          36
         </a>
         ]. Detailed analysis revealed that breakage-fusion-bridge cycles occurred first, followed by chromothripsis, leading to the formation of tandem duplications of 21q [
         <a href="#rid34">
          34
         </a>
         ]. Even though
         <em>
          RUNX1
         </em>
         is part of the consistently amplified critical region of chromosome 21,
         <em>
          RUNX1
         </em>
         translocations or mutations are not observed and the pathogenesis does not appear to involve
         <em>
          RUNX1
         </em>
         . In one-fifth of cases, iAMP21 is the only cytogenetic abnormality, but in other cases there are other karyotypic or molecular findings, including gains of the X chromosome, abnormalities of chromosome 7, del(9p), del(12p), del(13q)/
         <em>
          RB1
         </em>
         , deletion of
         <em>
          IKZF1
         </em>
         ,
         <em>
          P2YB8::CRLF2
         </em>
         fusion, rearrangements of
         <em>
          CRLF2
         </em>
         , and deletions of
         <em>
          RB1
         </em>
         and
         <em>
          ETV6
         </em>
         ; it remains uncertain if these additional changes contribute to the pathogenesis of this leukemia [
         <a href="#rid35">
          35
         </a>
         ]. Cases that include iAMP21 plus another genetic lesion that might suggest another category (eg,
         <em>
          CRLF2
         </em>
         translocation) should be classified as ALL/LBL with iAMP21.
        </p>
        <p class="headingAnchor" id="H1713615124">
         <span class="h3">
          BCR::ABL1-like B cell ALL
         </span>
         <span class="headingEndMark">
          —
         </span>
         BCR::ABL1-like B cell ALL is a provisional category in the 2016 WHO classification that shares a similar gene expression profile with B-ALL/LBL with t(9;22)/
         <em>
          BCR::ABL1
         </em>
         , but lacks the
         <em>
          BCR::ABL1
         </em>
         fusion or t(9;22) by cytogenetic, FISH, or molecular analysis [
         <a href="#rid1">
          1,2,9
         </a>
         ]. This category is found in 10 to 13 percent of pediatric patients, 21 percent of adolescents, and up to 33 percent of adults with B-ALL/LBL; it is resistant to standard chemotherapy and associated with a poor prognosis [
         <a href="#rid16">
          16,21,22,37-40
         </a>
         ]. BCR::ABL1-like B-ALL/LBL is mutually exclusive of other well-defined recurring chromosome abnormalities in B-ALL/LBL, such as the t(12;21), t(4;11) and other 11q23.3/
         <em>
          KMT2A
         </em>
         translocations, t(1;19), and a hyperdiploid karyotype.
        </p>
        <p>
         The diagnosis can be made using RNA sequencing-based methods that directly detect fusion transcripts [
         <a href="#rid21">
          21
         </a>
         ]. It can often be inferred by flow cytometry (eg, expression of CRLF2), conventional karyotype, FISH (eg, rearrangement of frequently involved genes), and/or mutation and deletion analysis [
         <a href="#rid40">
          40
         </a>
         ]. Detection of BCR::ABL1-like B cell ALL is important for prognosis and treatment stratification. Compared with other cases of precursor B cell ALL, this subgroup has higher WBC count at presentation, higher levels of MRD at the end of induction therapy, and inferior five-year EFS and OS.
        </p>
        <p>
         In a study of 798 adult patients with B-ALL, BCR::ABL1-like ALL was reported in 28 percent of patients age 21 to 39, 20 percent of patients 40 to 59 years, and 24 percent of those age 60 to 89 [
         <a href="#rid21">
          21
         </a>
         ]. In a study of 1725 children, adolescents, or young adults with precursor B cell ALL enrolled on clinical trials, BCR::ABL1-like ALL was found in 15 percent [
         <a href="#rid37">
          37,39
         </a>
         ]. Another study reported BCR::ABL1-like ALL in 20 percent of nearly 1400 consecutively diagnosed patients with childhood B-ALL enrolled in Children's Oncology Group trials [
         <a href="#rid41">
          41
         </a>
         ].
        </p>
        <p>
         Genes that are mutated in BCR::ABL1-like B-ALL generally encode proteins involved in B cell development, proliferation and differentiation, cell cycle regulation, and cell signaling. These mutations result in constitutive kinase activation and signaling via activation of the ABL1 and JAK-STAT pathways, which are sensitive to TKIs or JAK kinase inhibitors [
         <a href="#rid37">
          37,41,42
         </a>
         ]. Genetic findings include
         <em>
          ABL1
         </em>
         -class fusions; rearrangements of
         <em>
          EPOR,
         </em>
         <em>
          JAK2
         </em>
         , or
         <em>
          CRLF2
         </em>
         ; and other changes that affect genes encoding components of the JAK/STAT or RAS pathways [
         <a href="#rid21">
          21,22,37,39,41,42
         </a>
         ].
        </p>
        <p>
         In approximately half of patients with the BCR::ABL1-like B cell ALL, the cytokine receptor-like factor 2 (
         <em>
          CRLF2
         </em>
         ) gene within the pseudo-autosomal region 1 (PAR1) of the short arm of the X or Y chromosome is involved in a cryptic translocation with the
         <em>
          IGH
         </em>
         gene at 14q32.3 (ie, t(X;14) or t(Y;14)), or is fused with the
         <em>
          P2RY8
         </em>
         gene via small interstitial deletions within PAR1 [
         <a href="#rid16">
          16,37
         </a>
         ]. Both chromosome translocations and the interstitial deletion lead to
         <em>
          CRLF2
         </em>
         overexpression due to juxtaposition with the
         <em>
          IGH
         </em>
         enhancer or the
         <em>
          P2RY8
         </em>
         promoter. Rarely, gain-of-function point mutations of the
         <em>
          CRLF2
         </em>
         gene or gain of extra copies of the
         <em>
          CRLF2
         </em>
         locus can result in its overexpression. In approximately 30 to 55 percent of these patients, mutations in
         <em>
          JAK2
         </em>
         or
         <em>
          JAK1
         </em>
         or in
         <em>
          IL7R, FLT3, SH2B3,
         </em>
         or
         <em>
          NRAS
         </em>
         are also detected, indicative of the cooperation of these kinase-activating lesions with
         <em>
          CRLF2
         </em>
         overexpression in leukemogenesis [
         <a href="#rid37">
          37,38
         </a>
         ]. In addition, deletion of part or all of
         <em>
          IKZF1
         </em>
         ,
         <em>
          PAX5
         </em>
         , and
         <em>
          EBF1
         </em>
         are frequently detected in these leukemias. In almost all patients with Down syndrome-related B cell ALL, the blasts have a deletion of
         <em>
          IKZF1
         </em>
         [
         <a href="#rid38">
          38
         </a>
         ].
        </p>
        <p>
         In the remaining BCR::ABL1-like B cell ALL cases with no
         <em>
          CRLF2
         </em>
         overexpression, other tyrosine kinase genes, such as
         <em>
          ABL1, ABL2, JAK2, EPOR,
         </em>
         and
         <em>
          PDGFRB,
         </em>
         are involved by translocations and fusions that produce chimeric genes encoding constitutively active kinases; these include
         <em>
          NUP214::ABL1, BCR::JAK2, STRB3::JAK2, IGH::EPOR, EBF1::PDGFRB,
         </em>
         and
         <em>
          AGGF1::PDGFRB
         </em>
         fusions [
         <a href="#rid37">
          37,39,43
         </a>
         ]. In addition, mutations of
         <em>
          FLT3, CREBBP, IL7R,
         </em>
         or
         <em>
          SH2B3
         </em>
         (
         <em>
          LNK
         </em>
         ) occur frequently in these patients.
        </p>
        <p class="headingAnchor" id="H2278747179">
         <span class="h2">
          ALL/LBL, NOS
         </span>
         <span class="headingEndMark">
          —
         </span>
         B-ALL/LBL, not otherwise specified (NOS) includes cases with chromosomal abnormalities that are not known to be associated with phenotypic or prognostic distinctions. Chromosomal abnormalities that are not clearly associated with phenotypic or prognostic distinctions include del(6q), del(9q), and del (12p). Other cases of B-ALL/LBL, NOS have sub-karyotypic abnormalities or novel fusion genes, without associated chromosomal abnormalities.
        </p>
        <p class="headingAnchor" id="H3411487346">
         <span class="h3">
          Chromosome 9p abnormalities
         </span>
         <span class="headingEndMark">
          —
         </span>
         Abnormalities of the short arm of chromosome 9 are found in approximately 5 to 10 percent of childhood and adult ALL cases [
         <a href="#rid1">
          1
         </a>
         ]. This can occur as a single abnormality in childhood and adult disease, but it often exists as a part of a complex clone with other numerical and structural abnormalities, in particular with a del(12p). Abnormalities involving 9p include del(9p), add(9p), der(9)t(V;9)(V;p), isochromosome 9q, and dicentric chromosome 9q.
        </p>
        <p>
         Deletion of 9p is an unfavorable risk factor and is associated with a high rate of relapse in precursor B cell ALL in children [
         <a href="#rid44">
          44
         </a>
         ]. Abnormalities in 9p were reported in 8 percent of 381 adult cases of precursor B cell ALL; OS was decreased compared with patients with normal karyotypes and was similar to those with poor prognosis t(9;22)/
         <em>
          BCR::ABL1
         </em>
         -positive ALL [
         <a href="#rid45">
          45
         </a>
         ].
        </p>
        <p>
         The pathophysiology of 9p abnormalities is unclear. In one study, monoallelic deletion of the
         <em>
          PAX5
         </em>
         gene at 9p13.2 was seen by microarray analysis and genomic DNA sequencing in 28 percent of patients with a cryptic or a large deletion involving chromosome 9 [
         <a href="#rid8">
          8
         </a>
         ]. Deletion or mutation of
         <em>
          PAX5
         </em>
         leads to loss-of-function by reducing PAX5 protein levels or creating a hypomorphic allele [
         <a href="#rid46">
          46
         </a>
         ]. Somatically acquired mutations in
         <em>
          JAK2
         </em>
         on 9p24 were reported in 18 percent of cases of Down syndrome-associated B-ALL/LBL, but are not commonly found in B-ALL/LBL without Down syndrome or in Down syndrome-associated acute megakaryoblastic leukemia [
         <a href="#rid47">
          47-49
         </a>
         ]. The
         <em>
          JAK2
         </em>
         R683 mutation in this setting differs from the
         <em>
          JAK2
         </em>
         V617 mutation associated with myeloproliferative neoplasms.
        </p>
        <p class="headingAnchor" id="H4205269779">
         <span class="h3">
          Other recurring gene fusions
         </span>
         <span class="headingEndMark">
          —
         </span>
         Up to 30 percent of B-ALL/LBL cases are not defined by current WHO genetic subgroups (ie, they do not have recurring chromosome abnormalities, kinase activating gene fusions, or Ph-like B-ALL). Next-generation sequencing of RNA and other genomic studies have revealed several recurring genetic abnormalities in this category of B-ALL/LBL, especially in adolescent and young adult patients. Examples include:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          <em>
           MEF2D::BCL9
          </em>
         </strong>
         <strong>
          fusion
         </strong>
         – The myocyte enhancer factor 2D (
         <em>
          MEF2D
         </em>
         ) gene encodes a transcription factor that participates in neuronal development and myogenesis. Fusion of
         <em>
          MEF2D
         </em>
         with
         <em>
          BCL9
         </em>
         or other genes was detected in 5 to 7 percent of B-ALL/LBL, mostly in cases occurring in adolescents [
         <a href="#rid50">
          50,51
         </a>
         ]. B-ALL/LBL with the
         <em>
          MEF2D::BCL9
         </em>
         fusion is characterized by a distinct immunophenotype, similar to that of B-ALL/LBL with the t(1;19), markedly high expression of
         <em>
          HDAC9
         </em>
         , resistance to chemotherapy, early relapse, and a poor outcome.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          <em>
           DUX4::ERG
          </em>
         </strong>
         <strong>
          or
         </strong>
         <strong>
          <em>
           DUX4::IGH
          </em>
         </strong>
         <strong>
          fusion
         </strong>
         – The double homeobox 4 gene (
         <em>
          DUX4
         </em>
         ) is located near the telomere of the long arm of chromosome 4 and encodes the transcriptional activator PITX1.
         <em>
          DUX4-IGH
         </em>
         or
         <em>
          DUX4-ERG
         </em>
         fusions were detected in about 4 to 7 percent of B-ALL/LBL in pediatric and young patients [
         <a href="#rid52">
          52-55
         </a>
         ]. The
         <em>
          DUX4-IGH
         </em>
         translocation leads to overexpression of an altered DUX4 protein with an aberrant carboxy terminus that induces B cell leukemia in mice. Nearly all patients with
         <em>
          DUX4
         </em>
         aberrations also show deletion of
         <em>
          ERG
         </em>
         , indicative of the cooperation of
         <em>
          DUX4
         </em>
         and
         <em>
          ERG
         </em>
         in mediating leukemogenesis.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          <em>
           EP300::ZNF384
          </em>
         </strong>
         <strong>
          and
         </strong>
         <strong>
          <em>
           CREBBP::ZNF384
          </em>
         </strong>
         <strong>
          fusions
         </strong>
         – The zinc finger protein ZNF384 regulates expression of genes encoding extracellular matrix proteins. An
         <em>
          EP300::ZNF384
         </em>
         or
         <em>
          CREBBP::ZNF384
         </em>
         fusion is detected in 7 to 12 percent of B-ALL/LBL in adolescent and young adult patients; the disease is CD10-negative and is characterized by high expression of
         <em>
          GATA3
         </em>
         ,
         <em>
          CEBPA
         </em>
         , and
         <em>
          CEBPB
         </em>
         , upregulated JAK-STAT signaling, and a relatively low frequency of mutations or deletions of B cell development-related genes, such as
         <em>
          IKZF1
         </em>
         ,
         <em>
          PAX5
         </em>
         ,
         <em>
          RUNX1
         </em>
         ,
         <em>
          ETV6
         </em>
         , or the cell cycle regulators
         <em>
          CDKN2A/CDKN2B
         </em>
         [
         <a href="#rid55">
          55,56
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          <em>
           UBTF
          </em>
         </strong>
         <strong>
          ::
         </strong>
         <strong>
          <em>
           ATXN7L3
          </em>
         </strong>
         <strong>
          fusion
         </strong>
         and
         <strong>
          <em>
           CDX2
          </em>
         </strong>
         <strong>
          overexpression:
         </strong>
         B-ALL/LBL with
         <em>
          UBTF
         </em>
         ::
         <em>
          ATXN7L3
         </em>
         <strong>
         </strong>
         fusion and
         <em>
          CDX2
         </em>
         overexpression are high-risk subgroups with unique biological features that are mostly seen in adolescent and young adult females.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         These abnormalities can be detected in about 1 to 2 percent of B-ALL/LBL, mostly in adolescent and adult female patients with median age of 31-35 years [
         <a href="#rid57">
          57-60
         </a>
         ]. Aberrant
         <em>
          CDX2
         </em>
         activation was seen in 17 of 3221 cases of primary B-ALL/LBL and in 5 of 177 cases of relapsed B-ALL/LBL among (among 177 cases) [
         <a href="#rid57">
          57
         </a>
         ].
         <em>
          CDX2
         </em>
         activation stems from a focal deletion downstream of
         <em>
          PAN3
         </em>
         at 13q12.2 that hijacks the
         <em>
          PAN3
         </em>
         enhancer. T
         <em>
          he UBTF::ATXN7L3
         </em>
         fusion results from a microdeletion of about 10 kb involving exon 18-21 of
         <em>
          UBTF
         </em>
         at 17q21.31 [
         <a href="#rid58">
          58
         </a>
         ]. This form of B-ALL/LBL generally lacks CD10 expression and is often IgM positive. Patients in these groups often have high-risk features, including high relapse rates and high residual disease burden at the end on intensive induction treatment.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H1401476666">
         <span class="h3">
          Burkitt lymphoma/leukemia
         </span>
         <span class="headingEndMark">
          —
         </span>
         The t(8;14)(q24.2;q32.3) reciprocal translocation, resulting in constitutive expression of the
         <em>
          MYC
         </em>
         gene as a result of the
         <em>
          IGH::MYC
         </em>
         translocation, is present in a high proportion of Burkitt lymphoma/leukemia of both African and non-African origin. Burkitt lymphoma/leukemia is derived from transformed germinal center B cells, and in the 2016 revision of the WHO classification system  (
         <a class="graphic graphic_table graphicRef110239" href="/z/d/graphic/110239.html" rel="external">
          table 1
         </a>
         ) is classified together with other non-Hodgkin lymphomas of B cell origin. (See
         <a class="medical medical_review" href="/z/d/html/4700.html" rel="external">
          "Epidemiology, clinical manifestations, pathologic features, and diagnosis of Burkitt lymphoma"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H14">
         <span class="h1">
          T-ALL/LBL
         </span>
         <span class="headingEndMark">
          —
         </span>
         T cell ALL/LBL (T-ALL/LBL) is a hematologic malignancy of precursor lymphoid cells committed to the T cell lineage. T-ALL/LBL accounts for approximately 15 percent of childhood ALL/LBL; it is more common in adolescents than in younger children, and more common in boys than in girls [
         <a href="#rid1">
          1
         </a>
         ]. It accounts for one-quarter of adult cases of ALL/LBL. Importantly, T-ALL/LBL accounts for 85 to 90 percent of lymphoblastic lymphoma presentations. Clinical manifestations, pathologic features, and diagnosis of T-ALL/LBL are described separately. (See
         <a class="medical medical_review" href="/z/d/html/15773.html" rel="external">
          "Clinical manifestations, pathologic features, and diagnosis of precursor T cell acute lymphoblastic leukemia/lymphoma"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H1147627093">
         <span class="h2">
          Cytogenetic abnormalities in T-ALL/LBL
         </span>
         <span class="headingEndMark">
          —
         </span>
         An abnormal karyotype is found in 50 to 70 percent of cases of T-ALL/LBL. However, in many cases, rearrangements are not detected by karyotyping but are only recognized by molecular genetic studies. As an example, rearrangements involving
         <em>
          TAL1
         </em>
         are present in 20 to 30 percent of cases of T-ALL/LBL, but a cytogenetic finding of t(1;14)(p33;q11.2) is detected in only 3 percent; in many cases,
         <em>
          TAL1
         </em>
         is fused to
         <em>
          STIL
         </em>
         as a result of a cryptic interstitial deletion at chromosome band 1p33. In contrast to the important prognostic impact of recurring cytogenetic abnormalities in B cell ALL/LBL, the prognostic implication of chromosomal translocations and gene mutations and fusions, described below, is less well-defined in T-ALL/LBL.
        </p>
        <p class="headingAnchor" id="H3869153733">
         <span class="h2">
          Molecular abnormalities in T-ALL/LBL
         </span>
        </p>
        <p class="headingAnchor" id="H3566580410">
         <span class="h3">
          TCR rearrangements
         </span>
         <span class="headingEndMark">
          —
         </span>
         The most common recurrent molecular/genetic abnormalities in T-ALL/LBL involve rearrangements of the
         <em>
          TCR
         </em>
         (T cell receptor) alpha and delta loci at 14q11.2, beta locus at 7q34, and gamma locus at 7p14.1 with various partner genes [
         <a href="#rid61">
          61-63
         </a>
         ]. Most partner genes encode a transcription factor or cell cycle regulator whose expression is dysregulated or activated by juxtaposition to transcriptional regulatory elements of the
         <em>
          TCR
         </em>
         loci.
         <em>
          TLX1
         </em>
         (
         <em>
          HOX11
         </em>
         ) at 10q24.3 is the partner in 7 percent of childhood and 30 percent of adult cases;
         <em>
          TLX3
         </em>
         (
         <em>
          HOX11L2
         </em>
         ) at 5q35.1 is rearranged in 20 percent of childhood and 10 to 15 percent of adult cases. Other partner genes include
         <em>
          MYC
         </em>
         at 8q24.2,
         <em>
          TAL1
         </em>
         at 1p33,
         <em>
          LMO1
         </em>
         (
         <em>
          RBTN1
         </em>
         ) at 11p15.4,
         <em>
          LMO2
         </em>
         (
         <em>
          RBTN2
         </em>
         ) at 11p13,
         <em>
          LYL1
         </em>
         at 19p13.1, and the tyrosine kinase
         <em>
          LCK
         </em>
         at 1p35.2.
        </p>
        <p class="headingAnchor" id="H17">
         <span class="h3">
          Non-TCR loci
         </span>
         <span class="headingEndMark">
          —
         </span>
         Other karyotypic and/or mutational abnormalities in T-ALL/LBL that do not involve the
         <em>
          TCR
         </em>
         loci, include [
         <a href="#rid1">
          1,61,64-68
         </a>
         ]:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         t(10;11)(p12.2;q14.2), which generates the
         <em>
          PICALM-MLLT10
         </em>
         fusion; 10 percent of cases.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <em>
          KMT2A
         </em>
         translocations, often with the
         <em>
          MLLT1
         </em>
         (
         <em>
          ENL
         </em>
         ) partner gene at 19p13.1; 8 percent of cases.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         del(9p), which in 30 percent of cases is associated with loss of expression of
         <em>
          CDKN2A
         </em>
         and
         <em>
          CDKN2B
         </em>
         , which encode p16 and p15 (inhibitors of CDK4/6) as well as ARF (also known as p14), a protein that stabilizes p53.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <em>
          NOTCH1
         </em>
         activating mutations, generally producing changes in the amino acid sequence of the extracellular heterodimerization domain and/or truncations of the C-terminal PEST domain; approximately half of cases.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <em>
          JAK1
         </em>
         or
         <em>
          JAK3
         </em>
         mutations.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <em>
          NUP214::ABL1
         </em>
         cryptic fusion (ie, not detectable by conventional chromosome analysis) is present in approximately 6 percent of T-ALL/LBL (and rare cases of B-ALL/LBL); it is detectable by fluorescence in situ hybridization (FISH) as episomal amplicons with variable numbers of copies of
         <em>
          ABL1
         </em>
         or as a homogeneous staining region (HSR).
        </p>
        <p>
        </p>
        <p>
         Disorders that affect regulators of T cell growth and development contribute to T-ALL/LBL leukemogenesis. It has been proposed that there are four functionally distinct subgroups of T-ALL/LBL, based on specific translocations that lead to aberrant expression of 1)
         <em>
          TAL1
         </em>
         or
         <em>
          LMO1/LMO2
         </em>
         , 2)
         <em>
          TLX1
         </em>
         , 3)
         <em>
          TLX3
         </em>
         , or 4)
         <em>
          HOXA
         </em>
         genes, all of which cause T cell maturation arrest at distinct stages of thymocyte development [
         <a href="#rid69">
          69,70
         </a>
         ].
         <em>
          PICALM::MLLT10
         </em>
         and
         <em>
          KMT2A::MLLT1
         </em>
         activat
         <em>
          e HOXA
         </em>
         genes
         <em>
          ; MYC
         </em>
         , which contributes to growth of the malignant cells, is a direct downstream target of NOTCH1; and mutations o
         <em>
          f FBXW7
         </em>
         increase the half-life of the NOTCH1 protein [
         <a href="#rid71">
          71,72
         </a>
         ]. Th
         <em>
          e NUP214::ABL1
         </em>
         fusion is often associated with abnormalities o
         <em>
          f TLX1 or TLX3
         </em>
         , and other recurring genetic aberrations in T-ALL/LBL, such as deletions o
         <em>
          f CDKN2A, CDKN2B,
         </em>
         a
         <em>
          nd IKZF1
         </em>
         .
        </p>
        <p>
         Several mechanisms have been proposed to explain the higher incidence of T-ALL/LBL in males. Recurrent mutations of loci on the X chromosome may contribute to the sex predilection. As an example, inactivating mutations or deletions in the
         <em>
          PHF6
         </em>
         locus on the X chromosome are found in 16 percent and 38 percent of primary T cell ALL/LBL in children and adults, respectively; loss of
         <em>
          PHF6
         </em>
         is associated with aberrant expression of
         <em>
          TLX1
         </em>
         [
         <a href="#rid73">
          73
         </a>
         ]. Similarly, a small percentage of cases demonstrate mutations in the ribosomal gene
         <em>
          RPL10
         </em>
         located on the X chromosome [
         <a href="#rid74">
          74
         </a>
         ]. However, mutations of these loci seemingly cannot account for the difference in incidence between males and females because they are located in regions that undergo X inactivation in females. Some of the sex predilection may be explained by mutations in
         <em>
          UTX
         </em>
         (
         <em>
          KDM6A
         </em>
         ), a tumor suppressor located in a region of the X chromosome that escapes X inactivation in females [
         <a href="#rid75">
          75,76
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/15773.html" rel="external">
          "Clinical manifestations, pathologic features, and diagnosis of precursor T cell acute lymphoblastic leukemia/lymphoma", section on 'Epidemiology'
         </a>
         .)
        </p>
        <p>
         MicroRNAs (miRNAs) are small noncoding RNAs that modulate the expression of genes at the post-transcriptional level and may have particular relevance in T-ALL. Expression profiles in human T-ALL have identified a small number of abundantly expressed miRNAs that target a set of tumor suppressor genes (
         <em>
          IKZF1, PTEN, BIM, NF1, FBXW7,
         </em>
         and
         <em>
          PHF
         </em>
         ) and promote T-ALL in mice [
         <a href="#rid77">
          77
         </a>
         ]. As an example,
         <em>
          mir-223
         </em>
         is differentially upregulated in T-ALL/LBL and appears to promote NOTCH1-driven leukemia.
        </p>
        <p class="headingAnchor" id="H3432760990">
         <span class="h1">
          EARLY T CELL PRECURSOR ALL/LBL
         </span>
         <span class="headingEndMark">
          —
         </span>
         Early T cell precursor (ETP)-ALL/LBL is a hematologic malignancy of cells committed to the T cell lineage but with a unique immunophenotype corresponding to very early stages of T cell differentiation [
         <a href="#rid1">
          1
         </a>
         ].
        </p>
        <p>
         ETP-ALL cells lack expression of CD1a and CD8 and have weak expression of CD5 [
         <a href="#rid78">
          78
         </a>
         ]. These cases account for up to 15 percent of T-ALL/LBL in children and 5 to 10 percent in adults and are generally associated with a higher rate of treatment failure. Whole genome sequencing of tumor cells from 12 pediatric patients with ETP-ALL/LBL demonstrated a high frequency of somatic mutations involving the pathways of normal hematopoietic development, RAS signaling, and histone modification that are similar to those commonly involved in acute myeloid leukemia [
         <a href="#rid79">
          79
         </a>
         ]. This study also failed to detect mutations in
         <em>
          NOTCH1
         </em>
         , the gene that is most commonly mutated in T-ALL/LBL in both children and adults. However, a subsequent study of 68 adult cases of ETP-ALL/LBL identified
         <em>
          NOTCH1
         </em>
         mutations in approximately 15 percent of cases, along with several other mutations that are more characteristic of acute myeloid leukemia, such as mutations in
         <em>
          DNMT3A
         </em>
         and
         <em>
          FLT3
         </em>
         [
         <a href="#rid80">
          80
         </a>
         ].
        </p>
        <p>
         Acute leukemia with
         <strong>
         </strong>
         <em>
          BCL11B
         </em>
         rearrangements has been identified as an entity among ETP-ALL and mixed lineage acute leukemia (MPAL); abnormalities include translocations, insertions, or focal tandem duplications/amplifications) [
         <a href="#rid81">
          81,82
         </a>
         ]. Whole genome sequencing detected several
         <em>
          BCL11B
         </em>
         -related rearrangements, including t(2;14), t(3;14), t(6;14), t(7;14), t(8;14), t(12;14) and t(14;21). These rearrangements involve diverse fusion partners, specifically
         <em>
          ZEB2
         </em>
         at 2q22.3,
         <em>
          SATB1
         </em>
         at 3p24.3,
         <em>
          ARID1
         </em>
         at 6q25.3,
         <em>
          CDK6
         </em>
         at 7q22.1,
         <em>
          CCDC26
         </em>
         /
         <em>
          MYC
         </em>
         at 8q24.2,
         <em>
          ETV6
         </em>
         at 12p13.2, and
         <em>
          RUNX1
         </em>
         at 21q22.1. All appear to relocate super-enhancers to positions near the
         <em>
          BCL11B
         </em>
         gene at 14q32.2, leading to increased
         <em>
          BCL11B
         </em>
         expression. The BCL11B protein is a critical transcription factor in regulating thymic T lineage commitment and specification, and its overexpression leads to the inhibition of the T cell differentiation and activation of the JAK/STAT transduction pathway.
        </p>
        <p class="headingAnchor" id="H30997234">
         <span class="h1">
          NK CELL ALL/LBL
         </span>
         <span class="headingEndMark">
          —
         </span>
         Natural killer (NK) cell ALL/LBL is a provisional category of ALL/LBL in the 2016 World Health Organization classification [
         <a href="#rid1">
          1
         </a>
         ]. This is a rare entity that must be distinguished from blastic plasmacytoid dendritic cell neoplasms (BPDCN), myeloid/NK acute leukemia, and other malignancies that express NK cell markers, such as CD56 [
         <a href="#rid1">
          1
         </a>
         ]. Chromosomal and genetic abnormalities of NK cell ALL/LBL are not defined.
        </p>
        <p class="headingAnchor" id="H12776475">
         <span class="h1">
          INFORMATION FOR PATIENTS
         </span>
         <span class="headingEndMark">
          —
         </span>
         UpToDate offers two types of patient education materials, "The Basics" and "Beyond the Basics." The Basics patient education pieces are written in plain language, at the 5
         <sup>
          th
         </sup>
         to 6
         <sup>
          th
         </sup>
         grade reading level, and they answer the four or five key questions a patient might have about a given condition. These articles are best for patients who want a general overview and who prefer short, easy-to-read materials. Beyond the Basics patient education pieces are longer, more sophisticated, and more detailed. These articles are written at the 10
         <sup>
          th
         </sup>
         to 12
         <sup>
          th
         </sup>
         grade reading level and are best for patients who want in-depth information and are comfortable with some medical jargon.
        </p>
        <p>
         Here are the patient education articles that are relevant to this topic. We encourage you to print or e-mail these topics to your patients. (You can also locate patient education articles on a variety of subjects by searching on "patient education" and the keyword(s) of interest.)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Beyond the Basics topics (see
         <a class="medical medical_patient" href="/z/d/html/685.html" rel="external">
          "Patient education: Acute lymphoblastic leukemia (ALL) treatment in adults (Beyond the Basics)"
         </a>
         )
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H1460654043">
         <span class="h1">
          SUMMARY
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Description
         </strong>
         – Acute lymphoblastic leukemia/lymphoma (ALL/LBL) refers to hematologic malignancies of lymphoid precursor cells. These entities are described as ALL/LBL because leukemia and lymphoma are overlapping clinical presentations of these diseases; diagnosis and classification systems do not distinguish between leukemia and lymphoma. (See
         <a class="local">
          'Introduction'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         Broadly, ALL/LBL is divided into B cell and T cell lineages; rare cases are associated with the natural killer (NK) cell lineage. Immunophenotyping is required to determine the lineage of ALL/LBL because the various lineages are morphologically indistinguishable.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Cytogenetic and molecular abnormalities in ALL/LBL
         </strong>
         – Conventional karyotyping and fluorescence in situ hybridization (FISH) reveal recurring cytogenetic and/or molecular abnormalities in most cases of ALL/LBL. Many of these findings contribute to the disease phenotype, natural history, and/or prognosis. Characteristic cytogenetic and molecular features differ between B cell ALL/LBL (B-ALL/LBL) and T cell ALL/LBL (T-ALL/LBL). (See
         <a class="local">
          'Cytogenetic and molecular abnormalities'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Genetic features and classification of ALL/LBL
         </strong>
         – In the World Health Organization (WHO) classification system of hematologic malignancies  (
         <a class="graphic graphic_table graphicRef110239" href="/z/d/graphic/110239.html" rel="external">
          table 1
         </a>
         ), cytogenetic and molecular features are fundamental to categorizing B-ALL/LBL; such features are not used to classify T-ALL/LBL, early T cell precursor ALL/LBL, or NK cell ALL/LBL. (See
         <a class="local">
          'WHO classification of ALL/LBL'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          B-ALL/LBL
         </strong>
         – The various WHO categories of B-ALL/LBL with recurrent genetic abnormalities are described above. (See
         <a class="local">
          'B-ALL/LBL with recurrent genetic abnormalities'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         B-ALL/LBL that does not include one of the specified recurrent genetic abnormalities is categorized as B-ALL/LBL, not otherwise specified (NOS). (See
         <a class="local">
          'ALL/LBL, NOS'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          T-ALL/LBL
         </strong>
         – Cytogenetic and molecular features of T-ALL/LBL are described above. (See
         <a class="local">
          'T-ALL/LBL'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Other ALL/LBL categories
         </strong>
         – Cytogenetic and molecular features are not well-defined for other WHO-defined categories of ALL/LBL. (See
         <a class="local">
          'Early T cell precursor ALL/LBL'
         </a>
         above and
         <a class="local">
          'NK cell ALL/LBL'
         </a>
         above.)
        </p>
        <p>
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li class="breakAll">
          WHO Classification of Tumours of Haematopoietic and Lymphoid Tissues, revised 4th edition, Swerdlow SH, Campo E, Harris NL, et al. (Eds), International Agency for Research on Cancer (IARC), Lyon 2017.
         </li>
         <li>
          <a class="nounderline abstract_t">
           Arber DA, Orazi A, Hasserjian R, et al. The 2016 revision to the World Health Organization classification of myeloid neoplasms and acute leukemia. Blood 2016; 127:2391.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moorman AV, Chilton L, Wilkinson J, et al. A population-based cytogenetic study of adults with acute lymphoblastic leukemia. Blood 2010; 115:206.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Mullighan CG. How advanced are we in targeting novel subtypes of ALL? Best Pract Res Clin Haematol 2019; 32:101095.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moorman AV, Ensor HM, Richards SM, et al. Prognostic effect of chromosomal abnormalities in childhood B-cell precursor acute lymphoblastic leukaemia: results from the UK Medical Research Council ALL97/99 randomised trial. Lancet Oncol 2010; 11:429.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stengel A, Schnittger S, Weissmann S, et al. TP53 mutations occur in 15.7% of ALL and are associated with MYC-rearrangement, low hypodiploidy, and a poor prognosis. Blood 2014; 124:251.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gu Z, Churchman ML, Roberts KG, et al. PAX5-driven subtypes of B-progenitor acute lymphoblastic leukemia. Nat Genet 2019; 51:296.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Mullighan CG, Goorha S, Radtke I, et al. Genome-wide analysis of genetic alterations in acute lymphoblastic leukaemia. Nature 2007; 446:758.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Harvey RC, Mullighan CG, Wang X, et al. Identification of novel cluster groups in pediatric high-risk B-precursor acute lymphoblastic leukemia with gene expression profiling: correlation with genome-wide DNA copy number alterations, clinical characteristics, and outcome. Blood 2010; 116:4874.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moorman AV, Enshaei A, Schwab C, et al. A novel integrated cytogenetic and genomic classification refines risk stratification in pediatric acute lymphoblastic leukemia. Blood 2014; 124:1434.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Roberts KG. Genetics and prognosis of ALL in children vs adults. Hematology Am Soc Hematol Educ Program 2018; 2018:137.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moorman AV, Barretta E, Butler ER, et al. Prognostic impact of chromosomal abnormalities and copy number alterations in adult B-cell precursor acute lymphoblastic leukaemia: a UKALL14 study. Leukemia 2022; 36:625.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Paietta E, Roberts KG, Wang V, et al. Molecular classification improves risk assessment in adult BCR-ABL1-negative B-ALL. Blood 2021; 138:948.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moorman AV, Harrison CJ, Buck GA, et al. Karyotype is an independent prognostic factor in adult acute lymphoblastic leukemia (ALL): analysis of cytogenetic data from patients treated on the Medical Research Council (MRC) UKALLXII/Eastern Cooperative Oncology Group (ECOG) 2993 trial. Blood 2007; 109:3189.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wetzler M, Dodge RK, Mrózek K, et al. Additional cytogenetic abnormalities in adults with Philadelphia chromosome-positive acute lymphoblastic leukaemia: a study of the Cancer and Leukaemia Group B. Br J Haematol 2004; 124:275.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Moorman AV, Schwab C, Ensor HM, et al. IGH@ translocations, CRLF2 deregulation, and microdeletions in adolescents and adults with acute lymphoblastic leukemia. J Clin Oncol 2012; 30:3100.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           van der Veer A, Zaliova M, Mottadelli F, et al. IKZF1 status as a prognostic feature in BCR-ABL1-positive childhood ALL. Blood 2014; 123:1691.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stanulla M, Cavé H, Moorman AV. IKZF1 deletions in pediatric acute lymphoblastic leukemia: still a poor prognostic marker? Blood 2020; 135:252.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Winters AC, Bernt KM. MLL-Rearranged Leukemias-An Update on Science and Clinical Approaches. Front Pediatr 2017; 5:4.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yao J, Xu L, Aypar U, et al. Myeloid/lymphoid neoplasms with eosinophilia/ basophilia and ETV6-ABL1 fusion: cell-of-origin and response to tyrosine kinase inhibition. Haematologica 2021; 106:614.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Roberts KG, Gu Z, Payne-Turner D, et al. High Frequency and Poor Outcome of Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia in Adults. J Clin Oncol 2017; 35:394.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Jain N, Roberts KG, Jabbour E, et al. Ph-like acute lymphoblastic leukemia: a high-risk subtype in adults. Blood 2017; 129:572.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sundaresh A, Williams O. Mechanism of ETV6-RUNX1 Leukemia. Adv Exp Med Biol 2017; 962:201.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Noetzli L, Lo RW, Lee-Sherick AB, et al. Germline mutations in ETV6 are associated with thrombocytopenia, red cell macrocytosis and predisposition to lymphoblastic leukemia. Nat Genet 2015; 47:535.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Tang G, Tam W, Short NJ, et al. Myeloid/lymphoid neoplasms with FLT3 rearrangement. Mod Pathol 2021; 34:1673.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Boer JM, den Boer ML. BCR-ABL1-like acute lymphoblastic leukaemia: From bench to bedside. Eur J Cancer 2017; 82:203.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sood R, Kamikubo Y, Liu P. Role of RUNX1 in hematological malignancies. Blood 2017; 129:2070.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Paulsson K, Lilljebjörn H, Biloglav A, et al. The genomic landscape of high hyperdiploid childhood acute lymphoblastic leukemia. Nat Genet 2015; 47:672.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Harrison CJ, Moorman AV, Broadfield ZJ, et al. Three distinct subgroups of hypodiploidy in acute lymphoblastic leukaemia. Br J Haematol 2004; 125:552.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Nachman JB, Heerema NA, Sather H, et al. Outcome of treatment in children with hypodiploid acute lymphoblastic leukemia. Blood 2007; 110:1112.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Safavi S, Paulsson K. Near-haploid and low-hypodiploid acute lymphoblastic leukemia: two distinct subtypes with consistently poor prognosis. Blood 2017; 129:420.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Holmfeldt L, Wei L, Diaz-Flores E, et al. The genomic landscape of hypodiploid acute lymphoblastic leukemia. Nat Genet 2013; 45:242.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yeoh EJ, Ross ME, Shurtleff SA, et al. Classification, subtype discovery, and prediction of outcome in pediatric acute lymphoblastic leukemia by gene expression profiling. Cancer Cell 2002; 1:133.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Li Y, Schwab C, Ryan SL, et al. Constitutional and somatic rearrangement of chromosome 21 in acute lymphoblastic leukaemia. Nature 2014; 508:98.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Harrison CJ, Moorman AV, Schwab C, et al. An international study of intrachromosomal amplification of chromosome 21 (iAMP21): cytogenetic characterization and outcome. Leukemia 2014; 28:1015.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Strefford JC, van Delft FW, Robinson HM, et al. Complex genomic alterations and gene expression in acute lymphoblastic leukemia with intrachromosomal amplification of chromosome 21. Proc Natl Acad Sci U S A 2006; 103:8167.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Roberts KG, Li Y, Payne-Turner D, et al. Targetable kinase-activating lesions in Ph-like acute lymphoblastic leukemia. N Engl J Med 2014; 371:1005.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hertzberg L, Vendramini E, Ganmore I, et al. Down syndrome acute lymphoblastic leukemia, a highly heterogeneous disease in which aberrant expression of CRLF2 is associated with mutated JAK2: a report from the International BFM Study Group. Blood 2010; 115:1006.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Roberts KG, Reshmi SC, Harvey RC, et al. Genomic and outcome analyses of Ph-like ALL in NCI standard-risk patients: a report from the Children's Oncology Group. Blood 2018; 132:815.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Conant JL, Czuchlewski DR. BCR-ABL1-like B-lymphoblastic leukemia/lymphoma: Review of the entity and detection methodologies. Int J Lab Hematol 2019; 41 Suppl 1:126.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Reshmi SC, Harvey RC, Roberts KG, et al. Targetable kinase gene fusions in high-risk B-ALL: a study from the Children's Oncology Group. Blood 2017; 129:3352.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Roberts KG, Pei D, Campana D, et al. Outcomes of children with BCR-ABL1–like acute lymphoblastic leukemia treated with risk-directed therapy based on the levels of minimal residual disease. J Clin Oncol 2014; 32:3012.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zhang Y, Gao Y, Zhang H, et al. PDGFRB mutation and tyrosine kinase inhibitor resistance in Ph-like acute lymphoblastic leukemia. Blood 2018; 131:2256.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kuchinskaya E, Heyman M, Nordgren A, et al. Interphase fluorescent in situ hybridization deletion analysis of the 9p21 region and prognosis in childhood acute lymphoblastic leukaemia (ALL): results from a prospective analysis of 519 Nordic patients treated according to the NOPHO-ALL 2000 protocol. Br J Haematol 2011; 152:615.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Nahi H, Hägglund H, Ahlgren T, et al. An investigation into whether deletions in 9p reflect prognosis in adult precursor B-cell acute lymphoblastic leukemia: a multi-center study of 381 patients. Haematologica 2008; 93:1734.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Coyaud E, Struski S, Prade N, et al. Wide diversity of PAX5 alterations in B-ALL: a Groupe Francophone de Cytogenetique Hematologique study. Blood 2010; 115:3089.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Bercovich D, Ganmore I, Scott LM, et al. Mutations of JAK2 in acute lymphoblastic leukaemias associated with Down's syndrome. Lancet 2008; 372:1484.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gaikwad A, Rye CL, Devidas M, et al. Prevalence and clinical correlates of JAK2 mutations in Down syndrome acute lymphoblastic leukaemia. Br J Haematol 2009; 144:930.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Mullighan CG, Collins-Underwood JR, Phillips LA, et al. Rearrangement of CRLF2 in B-progenitor- and Down syndrome-associated acute lymphoblastic leukemia. Nat Genet 2009; 41:1243.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gu Z, Churchman M, Roberts K, et al. Genomic analyses identify recurrent MEF2D fusions in acute lymphoblastic leukaemia. Nat Commun 2016; 7:13331.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Suzuki K, Okuno Y, Kawashima N, et al. MEF2D-BCL9 Fusion Gene Is Associated With High-Risk Acute B-Cell Precursor Lymphoblastic Leukemia in Adolescents. J Clin Oncol 2016; 34:3451.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yasuda T, Tsuzuki S, Kawazu M, et al. Recurrent DUX4 fusions in B cell acute lymphoblastic leukemia of adolescents and young adults. Nat Genet 2016; 48:569.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zhang J, McCastlain K, Yoshihara H, et al. Deregulation of DUX4 and ERG in acute lymphoblastic leukemia. Nat Genet 2016; 48:1481.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lilljebjörn H, Henningsson R, Hyrenius-Wittsten A, et al. Identification of ETV6-RUNX1-like and DUX4-rearranged subtypes in paediatric B-cell precursor acute lymphoblastic leukaemia. Nat Commun 2016; 7:11790.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Liu YF, Wang BY, Zhang WN, et al. Genomic Profiling of Adult and Pediatric B-cell Acute Lymphoblastic Leukemia. EBioMedicine 2016; 8:173.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Qian M, Zhang H, Kham SK, et al. Whole-transcriptome sequencing identifies a distinct subtype of acute lymphoblastic leukemia with predominant genomic abnormalities of EP300 and CREBBP. Genome Res 2017; 27:185.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kimura S, Montefiori L, Iacobucci I, et al. Enhancer retargeting of CDX2 and UBTF::ATXN7L3 define a subtype of high-risk B-progenitor acute lymphoblastic leukemia. Blood 2022; 139:3519.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Passet M, Kim R, Gachet S, et al. Concurrent CDX2 cis-deregulation and UBTF::ATXN7L3 fusion define a novel high-risk subtype of B-cell ALL. Blood 2022; 139:3505.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yasuda T, Sanada M, Kawazu M, et al. Two novel high-risk adult B-cell acute lymphoblastic leukemia subtypes with high expression of CDX2 and IDH1/2 mutations. Blood 2022; 139:1850.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Bastian L, Hartmann AM, Beder T, et al. UBTF::ATXN7L3 gene fusion defines novel B cell precursor ALL subtype with CDX2 expression and need for intensified treatment. Leukemia 2022; 36:1676.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Graux C, Cools J, Michaux L, et al. Cytogenetics and molecular genetics of T-cell acute lymphoblastic leukemia: from thymocyte to lymphoblast. Leukemia 2006; 20:1496.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Han X, Bueso-Ramos CE. Precursor T-cell acute lymphoblastic leukemia/lymphoblastic lymphoma and acute biphenotypic leukemias. Am J Clin Pathol 2007; 127:528.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           De Keersmaecker K, Marynen P, Cools J. Genetic insights in the pathogenesis of T-cell acute lymphoblastic leukemia. Haematologica 2005; 90:1116.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Belver L, Ferrando A. The genetics and mechanisms of T cell acute lymphoblastic leukaemia. Nat Rev Cancer 2016; 16:494.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Asnafi V, Radford-Weiss I, Dastugue N, et al. CALM-AF10 is a common fusion transcript in T-ALL and is specific to the TCRgammadelta lineage. Blood 2003; 102:1000.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Petit A, Trinquand A, Chevret S, et al. Oncogenetic mutations combined with MRD improve outcome prediction in pediatric T-cell acute lymphoblastic leukemia. Blood 2018; 131:289.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Liu Y, Easton J, Shao Y, et al. The genomic landscape of pediatric and young adult T-lineage acute lymphoblastic leukemia. Nat Genet 2017; 49:1211.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Tsurusaki Y, Nagai JI, Fujita S, et al. Whole-exome sequencing reveals the subclonal expression of NUP214-ABL1 fusion gene in T-cell acute lymphoblastic leukemia. Pediatr Blood Cancer 2020; 67:e28019.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Meijerink JP. Genetic rearrangements in relation to immunophenotype and outcome in T-cell acute lymphoblastic leukaemia. Best Pract Res Clin Haematol 2010; 23:307.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Van Vlierberghe P, Pieters R, Beverloo HB, Meijerink JP. Molecular-genetic insights in paediatric T-cell acute lymphoblastic leukaemia. Br J Haematol 2008; 143:153.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Weng AP, Millholland JM, Yashiro-Ohtani Y, et al. c-Myc is an important direct target of Notch1 in T-cell acute lymphoblastic leukemia/lymphoma. Genes Dev 2006; 20:2096.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Malyukova A, Dohda T, von der Lehr N, et al. The tumor suppressor gene hCDC4 is frequently mutated in human T-cell acute lymphoblastic leukemia with functional consequences for Notch signaling. Cancer Res 2007; 67:5611.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Van Vlierberghe P, Palomero T, Khiabanian H, et al. PHF6 mutations in T-cell acute lymphoblastic leukemia. Nat Genet 2010; 42:338.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           De Keersmaecker K, Atak ZK, Li N, et al. Exome sequencing identifies mutation in CNOT3 and ribosomal genes RPL5 and RPL10 in T-cell acute lymphoblastic leukemia. Nat Genet 2013; 45:186.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Van der Meulen J, Sanghvi V, Mavrakis K, et al. The H3K27me3 demethylase UTX is a gender-specific tumor suppressor in T-cell acute lymphoblastic leukemia. Blood 2015; 125:13.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ntziachristos P, Tsirigos A, Welstead GG, et al. Contrasting roles of histone 3 lysine 27 demethylases in acute lymphoblastic leukaemia. Nature 2014; 514:513.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Mavrakis KJ, Van Der Meulen J, Wolfe AL, et al. A cooperative microRNA-tumor suppressor gene network in acute T-cell lymphoblastic leukemia (T-ALL). Nat Genet 2011; 43:673.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Coustan-Smith E, Mullighan CG, Onciu M, et al. Early T-cell precursor leukaemia: a subtype of very high-risk acute lymphoblastic leukaemia. Lancet Oncol 2009; 10:147.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Zhang J, Ding L, Holmfeldt L, et al. The genetic basis of early T-cell precursor acute lymphoblastic leukaemia. Nature 2012; 481:157.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Neumann M, Heesch S, Schlee C, et al. Whole-exome sequencing in adult ETP-ALL reveals a high rate of DNMT3A mutations. Blood 2013; 121:4749.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Montefiori LE, Bendig S, Gu Z, et al. Enhancer Hijacking Drives Oncogenic BCL11B Expression in Lineage-Ambiguous Stem Cell Leukemia. Cancer Discov 2021; 11:2846.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Di Giacomo D, La Starza R, Gorello P, et al. 14q32 rearrangements deregulating BCL11B mark a distinct subgroup of T-lymphoid and myeloid immature acute leukemia. Blood 2021; 138:773.
          </a>
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 4484 Version 44.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="javascript:alert('بدون لینک');" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : WHO Classification of Tumours of Haematopoietic and Lymphoid Tissues, revised 4th edition, Swerdlow SH, Campo E, Harris NL, et al. (Eds), International Agency for Research on Cancer (IARC), Lyon 2017.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27069254" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : The 2016 revision to the World Health Organization classification of myeloid neoplasms and acute leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19897583" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : A population-based cytogenetic study of adults with acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31779973" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : How advanced are we in targeting novel subtypes of ALL?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20409752" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : Prognostic effect of chromosomal abnormalities in childhood B-cell precursor acute lymphoblastic leukaemia: results from the UK Medical Research Council ALL97/99 randomised trial.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24829203" id="rid5" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          6 : TP53 mutations occur in 15.7% of ALL and are associated with MYC-rearrangement, low hypodiploidy, and a poor prognosis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30643249" id="rid6" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          7 : PAX5-driven subtypes of B-progenitor acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17344859" id="rid7" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          8 : Genome-wide analysis of genetic alterations in acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20699438" id="rid8" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          9 : Identification of novel cluster groups in pediatric high-risk B-precursor acute lymphoblastic leukemia with gene expression profiling: correlation with genome-wide DNA copy number alterations, clinical characteristics, and outcome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24957142" id="rid9" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          10 : A novel integrated cytogenetic and genomic classification refines risk stratification in pediatric acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30504302" id="rid10" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          11 : Genetics and prognosis of ALL in children vs adults.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/34657128" id="rid11" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          12 : Prognostic impact of chromosomal abnormalities and copy number alterations in adult B-cell precursor acute lymphoblastic leukaemia: a UKALL14 study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/33895809" id="rid12" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          13 : Molecular classification improves risk assessment in adult BCR-ABL1-negative B-ALL.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17170120" id="rid13" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          14 : Karyotype is an independent prognostic factor in adult acute lymphoblastic leukemia (ALL): analysis of cytogenetic data from patients treated on the Medical Research Council (MRC) UKALLXII/Eastern Cooperative Oncology Group (ECOG) 2993 trial.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/14717774" id="rid14" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          15 : Additional cytogenetic abnormalities in adults with Philadelphia chromosome-positive acute lymphoblastic leukaemia: a study of the Cancer and Leukaemia Group B.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22851563" id="rid15" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          16 : IGH@ translocations, CRLF2 deregulation, and microdeletions in adolescents and adults with acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24366361" id="rid16" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          17 : IKZF1 status as a prognostic feature in BCR-ABL1-positive childhood ALL.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31821407" id="rid17" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          18 : IKZF1 deletions in pediatric acute lymphoblastic leukemia: still a poor prognostic marker?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28232907" id="rid18" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          19 : MLL-Rearranged Leukemias-An Update on Science and Clinical Approaches.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/32299902" id="rid19" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          20 : Myeloid/lymphoid neoplasms with eosinophilia/ basophilia and ETV6-ABL1 fusion: cell-of-origin and response to tyrosine kinase inhibition.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27870571" id="rid20" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          21 : High Frequency and Poor Outcome of Philadelphia Chromosome-Like Acute Lymphoblastic Leukemia in Adults.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27919910" id="rid21" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          22 : Ph-like acute lymphoblastic leukemia: a high-risk subtype in adults.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28299659" id="rid22" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          23 : Mechanism of ETV6-RUNX1 Leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25807284" id="rid23" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          24 : Germline mutations in ETV6 are associated with thrombocytopenia, red cell macrocytosis and predisposition to lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/33990705" id="rid24" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          25 : Myeloid/lymphoid neoplasms with FLT3 rearrangement.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28709134" id="rid25" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          26 : BCR-ABL1-like acute lymphoblastic leukaemia: From bench to bedside.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28179279" id="rid26" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          27 : Role of RUNX1 in hematological malignancies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25961940" id="rid27" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          28 : The genomic landscape of high hyperdiploid childhood acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15147369" id="rid28" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          29 : Three distinct subgroups of hypodiploidy in acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17473063" id="rid29" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          30 : Outcome of treatment in children with hypodiploid acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27903530" id="rid30" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          31 : Near-haploid and low-hypodiploid acute lymphoblastic leukemia: two distinct subtypes with consistently poor prognosis.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23334668" id="rid31" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          32 : The genomic landscape of hypodiploid acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12086872" id="rid32" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          33 : Classification, subtype discovery, and prediction of outcome in pediatric acute lymphoblastic leukemia by gene expression profiling.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24670643" id="rid33" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          34 : Constitutional and somatic rearrangement of chromosome 21 in acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24166298" id="rid34" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          35 : An international study of intrachromosomal amplification of chromosome 21 (iAMP21): cytogenetic characterization and outcome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16702559" id="rid35" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          36 : Complex genomic alterations and gene expression in acute lymphoblastic leukemia with intrachromosomal amplification of chromosome 21.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25207766" id="rid36" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          37 : Targetable kinase-activating lesions in Ph-like acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19965641" id="rid37" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          38 : Down syndrome acute lymphoblastic leukemia, a highly heterogeneous disease in which aberrant expression of CRLF2 is associated with mutated JAK2: a report from the International BFM Study Group.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29997224" id="rid38" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          39 : Genomic and outcome analyses of Ph-like ALL in NCI standard-risk patients: a report from the Children's Oncology Group.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31069976" id="rid39" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          40 : BCR-ABL1-like B-lymphoblastic leukemia/lymphoma: Review of the entity and detection methodologies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28408464" id="rid40" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          41 : Targetable kinase gene fusions in high-risk B-ALL: a study from the Children's Oncology Group.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25049327" id="rid41" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          42 : Outcomes of children with BCR-ABL1–like acute lymphoblastic leukemia treated with risk-directed therapy based on the levels of minimal residual disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29434033" id="rid42" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          43 : PDGFRB mutation and tyrosine kinase inhibitor resistance in Ph-like acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21241277" id="rid43" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          44 : Interphase fluorescent in situ hybridization deletion analysis of the 9p21 region and prognosis in childhood acute lymphoblastic leukaemia (ALL): results from a prospective analysis of 519 Nordic patients treated according to the NOPHO-ALL 2000 protocol.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18728022" id="rid44" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          45 : An investigation into whether deletions in 9p reflect prognosis in adult precursor B-cell acute lymphoblastic leukemia: a multi-center study of 381 patients.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20160164" id="rid45" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          46 : Wide diversity of PAX5 alterations in B-ALL: a Groupe Francophone de Cytogenetique Hematologique study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18805579" id="rid46" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          47 : Mutations of JAK2 in acute lymphoblastic leukaemias associated with Down's syndrome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19120350" id="rid47" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          48 : Prevalence and clinical correlates of JAK2 mutations in Down syndrome acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19838194" id="rid48" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          49 : Rearrangement of CRLF2 in B-progenitor- and Down syndrome-associated acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27824051" id="rid49" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          50 : Genomic analyses identify recurrent MEF2D fusions in acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27507882" id="rid50" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          51 : MEF2D-BCL9 Fusion Gene Is Associated With High-Risk Acute B-Cell Precursor Lymphoblastic Leukemia in Adolescents.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27019113" id="rid51" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          52 : Recurrent DUX4 fusions in B cell acute lymphoblastic leukemia of adolescents and young adults.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27776115" id="rid52" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          53 : Deregulation of DUX4 and ERG in acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27265895" id="rid53" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          54 : Identification of ETV6-RUNX1-like and DUX4-rearranged subtypes in paediatric B-cell precursor acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27428428" id="rid54" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          55 : Genomic Profiling of Adult and Pediatric B-cell Acute Lymphoblastic Leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27903646" id="rid55" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          56 : Whole-transcriptome sequencing identifies a distinct subtype of acute lymphoblastic leukemia with predominant genomic abnormalities of EP300 and CREBBP.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/35192684" id="rid56" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          57 : Enhancer retargeting of CDX2 and UBTF::ATXN7L3 define a subtype of high-risk B-progenitor acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/35316324" id="rid57" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          58 : Concurrent CDX2 cis-deregulation and UBTF::ATXN7L3 fusion define a novel high-risk subtype of B-cell ALL.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/34695176" id="rid58" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          59 : Two novel high-risk adult B-cell acute lymphoblastic leukemia subtypes with high expression of CDX2 and IDH1/2 mutations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/35397658" id="rid59" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          60 : UBTF::ATXN7L3 gene fusion defines novel B cell precursor ALL subtype with CDX2 expression and need for intensified treatment.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16826225" id="rid60" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          61 : Cytogenetics and molecular genetics of T-cell acute lymphoblastic leukemia: from thymocyte to lymphoblast.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17369128" id="rid61" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          62 : Precursor T-cell acute lymphoblastic leukemia/lymphoblastic lymphoma and acute biphenotypic leukemias.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16079112" id="rid62" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          63 : Genetic insights in the pathogenesis of T-cell acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27451956" id="rid63" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          64 : The genetics and mechanisms of T cell acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/12676784" id="rid64" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          65 : CALM-AF10 is a common fusion transcript in T-ALL and is specific to the TCRgammadelta lineage.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29051182" id="rid65" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          66 : Oncogenetic mutations combined with MRD improve outcome prediction in pediatric T-cell acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28671688" id="rid66" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          67 : The genomic landscape of pediatric and young adult T-lineage acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31556219" id="rid67" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          68 : Whole-exome sequencing reveals the subclonal expression of NUP214-ABL1 fusion gene in T-cell acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21112032" id="rid68" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          69 : Genetic rearrangements in relation to immunophenotype and outcome in T-cell acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18691165" id="rid69" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          70 : Molecular-genetic insights in paediatric T-cell acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16847353" id="rid70" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          71 : c-Myc is an important direct target of Notch1 in T-cell acute lymphoblastic leukemia/lymphoma.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17575125" id="rid71" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          72 : The tumor suppressor gene hCDC4 is frequently mutated in human T-cell acute lymphoblastic leukemia with functional consequences for Notch signaling.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20228800" id="rid72" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          73 : PHF6 mutations in T-cell acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23263491" id="rid73" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          74 : Exome sequencing identifies mutation in CNOT3 and ribosomal genes RPL5 and RPL10 in T-cell acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25320243" id="rid74" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          75 : The H3K27me3 demethylase UTX is a gender-specific tumor suppressor in T-cell acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25132549" id="rid75" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          76 : Contrasting roles of histone 3 lysine 27 demethylases in acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21642990" id="rid76" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          77 : A cooperative microRNA-tumor suppressor gene network in acute T-cell lymphoblastic leukemia (T-ALL).
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19147408" id="rid77" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          78 : Early T-cell precursor leukaemia: a subtype of very high-risk acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22237106" id="rid78" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          79 : The genetic basis of early T-cell precursor acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23603912" id="rid79" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          80 : Whole-exome sequencing in adult ETP-ALL reveals a high rate of DNMT3A mutations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/34103329" id="rid80" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          81 : Enhancer Hijacking Drives Oncogenic BCL11B Expression in Lineage-Ambiguous Stem Cell Leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/33876209" id="rid81" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          82 : 14q32 rearrangements deregulating BCL11B mark a distinct subgroup of T-lymphoid and myeloid immature acute leukemia.
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
