<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Genetic abnormalities in hematologic and lymphoid malignancies
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Genetic abnormalities in hematologic and lymphoid malignancies
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Genetic abnormalities in hematologic and lymphoid malignancies
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Authors:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Iris Schrijver, MD
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            James L Zehnder, MD
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editors:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Richard A Larson, MD
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Benjamin A Raby, MD, MPH
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Alan G Rosmarin, MD
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            Apr 15, 2022.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H1">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         Human malignancies can be caused by a variety of mechanisms, including the inactivation of tumor suppressor genes, activation of oncogenes, and general genomic instability. The vast majority of human hematologic neoplasms are caused by the clonal expansion of a single cell that has acquired a somatic mutation in one allele of an otherwise normal gene (proto-oncogene) governing cellular maturation and division. This mutated gene, now called an oncogene, stimulates inappropriate cellular proliferation, leading to the development of cancer.
        </p>
        <p>
         Whereas multiple, random cytogenetic abnormalities are a characteristic finding in advanced malignancies, tumor-specific chromosomal translocations contribute directly to malignant transformation. Such translocations, and other genetic abnormalities, have been described for a large number of hematopoietic and lymphoid malignancies, including the acute and chronic lymphoid and myeloid leukemias, other myeloproliferative disorders, myelodysplastic syndromes, multiple myeloma (plasma cell myeloma), and malignant lymphomas [
         <a href="#rid1">
          1,2
         </a>
         ].
        </p>
        <p>
         A classic example is the Philadelphia chromosome (Ph), the first cytogenetic abnormality identified in a human malignancy, which is present in over 90 percent of individuals with chronic myeloid leukemia (CML). Initially described in 1960 [
         <a href="#rid3">
          3
         </a>
         ], this small chromosome originates from the reciprocal translocation t(9;22)(q34;q11.2), which forms a transcriptionally active fusion gene between the
         <em>
          BCR
         </em>
         gene on chromosome 22 and the
         <em>
          ABL1
         </em>
         locus on chromosome 9  (
         <a class="graphic graphic_figure graphicRef62227" href="/z/d/graphic/62227.html" rel="external">
          figure 1
         </a>
         ). The chimeric protein that is encoded by the newly created
         <em>
          BCR::ABL1
         </em>
         gene on the Ph chromosome is involved in leukemogenesis through its interference with normal cell cycle events, such as signal transduction and the regulation of apoptosis and cell proliferation [
         <a href="#rid4">
          4
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4490.html" rel="external">
          "Chromosomal translocations, deletions, and inversions"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4487.html" rel="external">
          "Cellular and molecular biology of chronic myeloid leukemia"
         </a>
         .)
        </p>
        <p>
         The
         <em>
          BCR::ABL1
         </em>
         translocation is a diagnostic hallmark of CML and the only distinct chromosome abnormality seen in the chronic phase of this disorder. In addition, its presence is an important indicator of residual disease or relapse after treatment. Sensitive molecular genetic and cytogenetic methods to detect this translocation are now available for initial diagnosis and quantitative monitoring of disease status in patients with CML. (See
         <a class="medical medical_review" href="/z/d/html/4543.html" rel="external">
          "Clinical manifestations and diagnosis of chronic myeloid leukemia"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4537.html" rel="external">
          "Molecular genetics of chronic myeloid leukemia", section on 'Detecting the Philadelphia chromosome or its products'
         </a>
         .)
        </p>
        <p>
         Because
         <em>
          BCR::ABL1
         </em>
         is the most common and best studied tumor-specific translocation in the hematopoietic malignancies, we shall use this fusion gene as a model for the ability of cytogenetic and molecular genetic diagnostic tools to help us understand the pathophysiology, diagnosis, treatment, prognosis, and monitoring of disease activity in the various hematologic malignancies. Methods for molecular and cytogenetic diagnosis are summarized separately. (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2">
         <span class="h1">
          CHRONIC MYELOID LEUKEMIA AS A MODEL SYSTEM
         </span>
         <span class="headingEndMark">
          —
         </span>
         The incidence of chronic myeloid leukemia (CML) is approximately 1 to 2 cases per 100,000. It is slightly more common in males than in females [
         <a href="#rid5">
          5
         </a>
         ] and accounts for 15 to 20 percent of all leukemias [
         <a href="#rid4">
          4
         </a>
         ]. Because it is a disorder arising from a mutational event in a somatic cell, chromosomes in all tissues are normal except in affected cells within the hematopoietic system. In one study for example, the Philadelphia chromosome was found in cells from all hematopoietic lineages with the exception of mature T-lymphocytes, supporting the hypothesis that CML results from a single aberrant chromosomal event occurring in a pluripotent hematopoietic stem cell [
         <a href="#rid6">
          6
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4487.html" rel="external">
          "Cellular and molecular biology of chronic myeloid leukemia", section on 'Cell of origin'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3">
         <span class="h2">
          BCR and ABL1 genes
         </span>
         <span class="headingEndMark">
          —
         </span>
         The Philadelphia chromosome is a derivative chromosome 22 [der(22)] created by a reciprocal translocation between the region containing the
         <em>
          BCR
         </em>
         gene on the long arm of chromosome 22 (22q) and the segment that contains the
         <em>
          ABL1
         </em>
         gene on the long arm of chromosome 9 (9q)  (
         <a class="graphic graphic_figure graphicRef62227" href="/z/d/graphic/62227.html" rel="external">
          figure 1
         </a>
         ) [
         <a href="#rid7">
          7
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H4">
         <span class="h3">
          ABL1 gene
         </span>
         <span class="headingEndMark">
          —
         </span>
         The large
         <em>
          ABL1
         </em>
         gene spans approximately 225 kilobases (kb). After the intervening sequences (introns) are spliced out, the gene can be expressed as a 6.0 or 7.0 kb messenger RNA (mRNA). This range in sizes is due to the alternative splicing of exons 2 through 11 to either exon 1a or, less commonly, to exon 1b, which is larger, as shown in panel A  (
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ). The resulting protein is a tyrosine kinase with a molecular weight of 145 kilodalton (kD). (See
         <a class="medical medical_review" href="/z/d/html/4487.html" rel="external">
          "Cellular and molecular biology of chronic myeloid leukemia", section on 'ABL1'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H5">
         <span class="h3">
          BCR gene
         </span>
         <span class="headingEndMark">
          —
         </span>
         The
         <em>
          BCR
         </em>
         gene on chromosome 22 encompasses 134 kb and encodes 23 exons. It was named after the 5.8 kb major
         <strong>
          B
         </strong>
         reakpoint
         <strong>
          C
         </strong>
         luster
         <strong>
          R
         </strong>
         egion that includes 5 exons (b1 through b5) located in a central area of the gene, as shown in panel B  (
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ). The BCR mRNAs are 4.5 or 6.7 kb in size; both encode a 160 kD protein with serine/threonine kinase activity [
         <a href="#rid8">
          8
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4487.html" rel="external">
          "Cellular and molecular biology of chronic myeloid leukemia", section on 'BCR'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H6">
         <span class="h3">
          BCR::ABL1 fusion gene
         </span>
         <span class="headingEndMark">
          —
         </span>
         In t(9;22), the
         <em>
          BCR
         </em>
         gene on chromosome 22 fuses with
         <em>
          ABL1
         </em>
         from chromosome 9 in a head-to-tail manner, forming a novel 8.5 kb mRNA, as shown in panel C  (
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ). This mRNA comprises about half of the
         <em>
          BCR
         </em>
         exons at the centromeric 5' end, and
         <em>
          ABL1
         </em>
         exons 2 to 11 towards the 3' end [
         <a href="#rid7">
          7
         </a>
         ]. The breakpoints in the major breakpoint cluster region are located either between exons b2 and b3 (exons 13 and 14 of
         <em>
          BCR
         </em>
         ), or between exons b3 and b4 (
         <em>
          BCR
         </em>
         exons 14 and 15). (See
         <a class="medical medical_review" href="/z/d/html/4487.html" rel="external">
          "Cellular and molecular biology of chronic myeloid leukemia", section on 'BCR-ABL1'
         </a>
         .)
        </p>
        <p>
         The indication of exons within the breakpoint cluster region (ie, exons b1 through b5), rather than a numbering according to their position within
         <em>
          BCR
         </em>
         as a whole (ie, exons 12 through 16), dates back to the time when the
         <em>
          BCR
         </em>
         gene was not yet fully characterized and remains popular in the literature. To avoid confusion with other reports, we will adhere to this original
         <em>
          BCR
         </em>
         breakpoint indication. Whereas
         <em>
          ABL1
         </em>
         breakpoints can occur 5' to exon 1b, as well as in introns 1, 2, or 3, the great majority of patients demonstrate a chimeric transcript in which the
         <em>
          BCR
         </em>
         segment is spliced to exon a2 of
         <em>
          ABL1
         </em>
         [
         <a href="#rid9">
          9
         </a>
         ]. Therefore, the RNA transcripts in most patients consist of
         <em>
          ABL1
         </em>
         a2, spliced to
         <em>
          BCR
         </em>
         exons b2 or b3, denoted as b2a2 or b3a2, as shown in panel C  (
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ). In some patients with a confirmed b3a2 junction, coexpression of transcripts b3a2 and b2a2 can be detected, due to alternative splicing in which exon b3 is selectively spliced out. The encoded fusion protein, commonly described as p210 BCR::ABL1, because of its molecular weight of 210 kDa, possesses the oncogenic dysregulated kinase activity.
        </p>
        <p>
         Virtually all patients with CML exhibit the b2a2 or the b3a2 junction, generated by a breakpoint within the major breakpoint cluster BCR. Two other breakpoint cluster regions have been identified: the minor breakpoint cluster region is located upstream (ie, towards the centromeric, or 5' end) [
         <a href="#rid10">
          10
         </a>
         ] and the micro breakpoint cluster region is located downstream (ie, towards the telomeric, or 3' end), as shown in panel B  (
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ) [
         <a href="#rid11">
          11
         </a>
         ].
        </p>
        <p>
         The minor breakpoint is rarely affected in individuals with CML, but it serves as a breakpoint for two thirds of patients with acute lymphoblastic leukemia (ALL). The e1a2 fusion arises after alternative exons 1 and 2 are selectively spliced out, as shown in panel B  (
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ). The resulting protein is only 190 kD, smaller than the product of the major breakpoint [
         <a href="#rid12">
          12
         </a>
         ]. Only in rare cases is the breakpoint in the micro breakpoint identified. The junction between
         <em>
          BCR
         </em>
         exon 19 and
         <em>
          ABL1
         </em>
         exon 2 leads to a 230 kD protein product, as shown in panels A, B, and C  (
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ), which has been described in CML and, more frequently, in patients with chronic neutrophilic leukemia [
         <a href="#rid13">
          13
         </a>
         ]. (See
         <a class="local">
          'Chronic neutrophilic leukemia'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H7">
         <span class="h3">
          ABL1::BCR fusion gene
         </span>
         <span class="headingEndMark">
          —
         </span>
         In reciprocal translocations, two non-homologous chromosomes exchange parts. As a consequence of the
         <em>
          BCR::ABL1
         </em>
         translocation, chromosome 9 should exhibit fusion with the telomeric chromosome piece that was separated from chromosome 22  (
         <a class="graphic graphic_figure graphicRef62227" href="/z/d/graphic/62227.html" rel="external">
          figure 1
         </a>
         ). This rarely reported reciprocal product [der(9)] was demonstrated to be present in approximately 60 percent of patients with CML [
         <a href="#rid14">
          14
         </a>
         ]. In the
         <em>
          BCR::ABL1
         </em>
         fusion product, the
         <em>
          ABL1
         </em>
         breakpoint is observed at exon a2 because the splicing machinery selectively removes the exons upstream. When the
         <em>
          ABL1::BCR
         </em>
         fusion gene undergoes transcription, however, the
         <em>
          ABL1
         </em>
         breakpoint commonly occurs upstream of exon 1b, between exons 1b and 1a, or downstream of exon 1a, as shown in panel A  (
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ). This leads to a greater variety of fusion transcripts, consisting of a small portion of
         <em>
          ABL1
         </em>
         and a larger portion of
         <em>
          BCR
         </em>
         . (See
         <a class="medical medical_review" href="/z/d/html/4537.html" rel="external">
          "Molecular genetics of chronic myeloid leukemia", section on 'Reciprocal translocation product'
         </a>
         .)
        </p>
        <p>
         The observation of one (or occasionally more, when alternative splicing occurs within one individual) of the four 1b-
         <em>
          BCR
         </em>
         messages (1be2,1bb3,1bb4 and 1be20) in the majority of CML cases, has been ascribed to the sizable intron between
         <em>
          ABL1
         </em>
         exons 1b and 1a, which is likely to contain the preponderance of
         <em>
          ABL1
         </em>
         breakpoints. The newly formed junctions are in-frame and encode a message that could be translated into a functional protein sequence and expressed by the cell. The significance of the ABL1::BCR fusion protein, however, is still unclear. A major role in leukemogenesis is improbable, because it is not expressed in 40 percent of Ph positive CML patients. However, there are some reports of decreased survival rates in CML patients who have a deletion of
         <em>
          ABL1
         </em>
         on the derivative chromosome 9, as well as deletion of the proximal region, including the argininosuccinate synthase gene [
         <a href="#rid15">
          15,16
         </a>
         ]. Further prospective studies are necessary to clarify this issue.
        </p>
        <p class="headingAnchor" id="H69720132">
         <span class="h2">
          Infrequent breakpoints in BCR and ABL1
         </span>
         <span class="headingEndMark">
          —
         </span>
         Fewer than 10 percent of CML clones have breakpoints outside the major breakpoint cluster. Apart from the e1a2 junction (described above), and the infrequent e19a2 transcript in the micro breakpoint, there are a few rare outliers. These include fusion transcripts b2a3, e1a3, e19a3, e20a3 and e6a2 [
         <a href="#rid17">
          17
         </a>
         ]. Occasionally, translocation breakpoints can occur within coding exons, as exemplified in two patients with partial e8a2 and partial b2a2 fusions, respectively [
         <a href="#rid18">
          18
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H69720139">
         <span class="h3">
          Clinical implications of various breakpoints
         </span>
         <span class="headingEndMark">
          —
         </span>
         Considering the variety of
         <em>
          BCR
         </em>
         and
         <em>
          ABL1
         </em>
         breakpoints as well as RNA transcripts in CML, the question arises whether these molecular differences induce phenotypic variation in disease course and survival. Whereas some early reports suggested such correlations, other studies refute the early results. In one large-scale prospective study [
         <a href="#rid19">
          19
         </a>
         ], no prognostic association between the genomic breakpoint site or RNA message, and duration of the chronic phase or survival, was detected. The only possible correlation that was detected in a small number of patients whose white blood cell (WBC) count was lower than 100,000/µL, was one between platelet count and the b3a2 transcript.
        </p>
        <p>
         In contrast to the lack of genotype-phenotype variation within the major breakpoint in CML, distinct clinical features, resembling both CML and CMML, are exhibited by patients who express p190 BCR::ABL1 due to the minor breakpoint. There is an increase in peripheral immature granulocytes, considerable monocytosis, and a variable degree of basophilia. These features are consistent with a myeloproliferative disorder that includes expansion of the monocytic lineage, which is not characteristic of CML with p210 BCR::ABL1. This observation is supported by the fact that all cases of the minor breakpoint AML fell into the French-American-British (FAB) classification M4 or M5a, which are subgroups with myelomonocytic (M4) or monocytic (M5) proliferation [
         <a href="#rid20">
          20
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/86098.html" rel="external">
          "Acute myeloid leukemia: Classification"
         </a>
         .)
        </p>
        <p>
         Adults with ALL who express either p210 BCR::ABL1 or p190 BCR::ABL1 are phenotypically very similar, but in children there may be a distinction. Childhood ALL due to a breakpoint in the major breakpoint is uncommon, but, when present, tends to be associated with hyperleukocytic ALL  (
         <a class="graphic graphic_table graphicRef66337" href="/z/d/graphic/66337.html" rel="external">
          table 1
         </a>
         ). Firm conclusions, however, cannot be drawn, due to the small number of patients studied and the possibility that their malignancy may represent lymphoid blast crisis after an undiagnosed prior chronic phase of CML. However, in all types of ALL, the presence of Ph confers a poor prognosis.
        </p>
        <p>
         The largest fusion protein described for the
         <em>
          BCR::ABL1
         </em>
         translocation has a molecular weight of 230 kD and is most often associated with chronic neutrophilic leukemia (CNL). This rare disorder is closely related to CML, but in most instances, the disease course is more prolonged and symptoms are less severe. Compared with classic CML, the WBC count is lower and anemia is less pronounced. Moreover, the percentage of peripheral immature granulocytes is low and the spleen is only mildly enlarged [
         <a href="#rid13">
          13
         </a>
         ]. Only 2 of 10 CNL patients reported to date have progressed to the acute leukemic phase [
         <a href="#rid21">
          21
         </a>
         ]. A subset of patients has significant thrombocytosis, but the etiology of this finding is, as yet, unknown. (See
         <a class="local">
          'Chronic neutrophilic leukemia'
         </a>
         below.)
        </p>
        <p>
         In contrast to ALL, CML patients in whom no
         <em>
          BCR::ABL1
         </em>
         fusion product can be detected have a lower survival rate than individuals with the translocation. The patients who lack
         <em>
          BCR::ABL1
         </em>
         are of a higher age, have lower WBC counts, higher frequency of thrombocytopenia, less prominent basophilia, and an increasing leukemic burden, even though progression to overt blast crisis is less common [
         <a href="#rid22">
          22
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H69719826">
         <span class="h2">
          Detection of Ph and BCR::ABL1
         </span>
         <span class="headingEndMark">
          —
         </span>
         The most widely applied diagnostic method for CML is karyotyping of metaphase chromosomes by cytogenetic studies  (
         <a class="graphic graphic_figure graphicRef62227" href="/z/d/graphic/62227.html" rel="external">
          figure 1
         </a>
         ). The Ph chromosome is identified in approximately 90 percent of new patients with CML, and the progression to a more aggressive leukemic stage can be detected or confirmed by the emergence of additional structural or numerical chromosome abnormalities. (See
         <a class="medical medical_review" href="/z/d/html/4543.html" rel="external">
          "Clinical manifestations and diagnosis of chronic myeloid leukemia", section on 'Genetics'
         </a>
         .)
        </p>
        <p>
         DNA-based polymerase chain reaction (PCR) analysis can determine individual breakpoints precisely, but are less suitable for routine translocation detection, due to the broad distribution of possible translocation breakpoints and the prohibitive length of some introns. The intron-less mRNA transcript, on the other hand, is an excellent molecular template and reverse transcriptase polymerase chain reaction (RT-PCR) will detect a
         <em>
          BCR::ABL1
         </em>
         fusion in an additional one-half of the patients who are Ph negative by chromosome banding techniques [
         <a href="#rid23">
          23
         </a>
         ]. This highly sensitive method can detect the mutation in a single cell in a background of at least 10
         <sup>
          6
         </sup>
         unaffected cells. (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H69719942">
         <span class="h2">
          Ph chromosome negative CML
         </span>
         <span class="headingEndMark">
          —
         </span>
         In approximately 5 percent of clinically diagnosed patients with CML, no Ph is detected by standard cytogenetics. With the advent of molecular genetic studies such as FISH and PCR, however, a subgroup of patients with microscopically undetectable translocations has been uncovered [
         <a href="#rid24">
          24
         </a>
         ]. Some of these cases proved to be caused by complex gene rearrangements, involving one or more chromosomes in addition to 9 and 22  (
         <a class="graphic graphic_figure graphicRef74102" href="/z/d/graphic/74102.html" rel="external">
          figure 3
         </a>
         ). (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4543.html" rel="external">
          "Clinical manifestations and diagnosis of chronic myeloid leukemia", section on 'Genetics'
         </a>
         .)
        </p>
        <p>
         In other instances, the translocation of material from
         <em>
          ABL1
         </em>
         to chromosome 22 is non-reciprocal and more difficult to detect. A contributing factor to apparent Ph and
         <em>
          BCR::ABL1
         </em>
         negativity may be the choice of probes used in the detection assay. Most studies that have investigated
         <em>
          BCR::ABL1
         </em>
         -negative CML have selected probes that would be unable to detect breakpoints in regions outside the major breakpoint cluster.
        </p>
        <p>
         A small subset of patients may indeed lack the BCR::ABL1 fusion gene. Most cases have an atypical disease course with less frequent progression to blast crisis, and may need to be categorized under different diagnoses, such as chronic myelomonocytic leukemia (CMML), chronic neutrophilic leukemia, or "atypical CML" [
         <a href="#rid25">
          25,26
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4543.html" rel="external">
          "Clinical manifestations and diagnosis of chronic myeloid leukemia", section on 'Chronic neutrophilic leukemia'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4492.html" rel="external">
          "Clinical manifestations, diagnosis, and classification of myelodysplastic syndromes (MDS)", section on 'MDS/MPN syndromes'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/96031.html" rel="external">
          "Chronic myelomonocytic leukemia: Clinical features, evaluation, and diagnosis"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H69719770">
         <span class="h2">
          Monitoring of residual disease
         </span>
         <span class="headingEndMark">
          —
         </span>
         Monitoring of the reduction of
         <em>
          BCR-ABL1
         </em>
         transcripts is standard care, as discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/4496.html" rel="external">
          "Overview of the treatment of chronic myeloid leukemia", section on 'Monitoring response'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H8">
         <span class="h1">
          THE PH CHROMOSOME IN MALIGNANCIES OTHER THAN CML
         </span>
         <span class="headingEndMark">
          —
         </span>
         Whereas Ph is a recognized hallmark of CML, it is by no means exclusive to this disorder, and has been seen in acute lymphoblastic leukemia (ALL), acute myeloid leukemia (AML), and other hematologic malignancies  (
         <a class="graphic graphic_table graphicRef66337" href="/z/d/graphic/66337.html" rel="external">
          table 1
         </a>
         ).
        </p>
        <p class="headingAnchor" id="H9">
         <span class="h2">
          Acute lymphoblastic leukemia
         </span>
         <span class="headingEndMark">
          —
         </span>
         The Ph translocation is the most commonly detected chromosomal abnormality in ALL and has been identified in 5 percent of children [
         <a href="#rid27">
          27
         </a>
         ] and in almost one third of adults with this malignancy [
         <a href="#rid28">
          28
         </a>
         ]. In those instances in which pediatric ALL is Ph-positive, 85 percent express p190 BCR::ABL1 and the remaining 15 percent have a breakpoint in the major breakpoint  (
         <a class="graphic graphic_table graphicRef66337" href="/z/d/graphic/66337.html" rel="external">
          table 1
         </a>
         and
         <a class="graphic graphic_figure graphicRef66610" href="/z/d/graphic/66610.html" rel="external">
          figure 2
         </a>
         ).
        </p>
        <p>
         In Ph positive adults with ALL, the reverse transcription polymerase chain reaction technique (RT-PCR) (see
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         ) has discerned the e1a2 transcript with p190
         <em>
          BCR::ABL1
         </em>
         expression in 50 to 77 percent of cases, and b3a2 or b2a2 transcripts with p210
         <em>
          BCR::ABL1
         </em>
         expression in 23 to 50 percent [
         <a href="#rid25">
          25
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4484.html" rel="external">
          "Classification, cytogenetics, and molecular genetics of acute lymphoblastic leukemia/lymphoma", section on 't(9;22); BCR::ABL1'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H10">
         <span class="h2">
          Acute myeloid leukemia
         </span>
         <span class="headingEndMark">
          —
         </span>
         The Ph chromosome is identified in only a small minority of patients with acute myeloid leukemia (AML)  (
         <a class="graphic graphic_table graphicRef66337" href="/z/d/graphic/66337.html" rel="external">
          table 1
         </a>
         ) [
         <a href="#rid29">
          29
         </a>
         ], and has been the subject of debate. Initially, it was thought that Ph-positive AML was equivalent to CML in blast crisis, but several distinguishing features support the view that they are, in fact, separate entities:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Secondary chromosomal abnormalities are common in the acute (blastic) phase of CML, while these are not observed in newly diagnosed AML.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         All cells that originated from the initially mutated stem cell in CML carry the Ph translocation. The acute leukemias, on the other hand, generally exhibit both BCR-ABL1 positive and negative cells at the time of diagnosis.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The Philadelphia chromosome commonly persists when CML is in remission, while cytogenetic normalization is a frequent finding in acute leukemia in remission.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The frequency and type of additional chromosome aberrations can provide clues as to which diagnosis is most likely [
         <a href="#rid25">
          25
         </a>
         ]. In AML, breakpoints have been reported in M-bcr and m-bcr [
         <a href="#rid29">
          29,30
         </a>
         ], and putative AML has also been associated with a breakpoint in µ-bcr [
         <a href="#rid31">
          31
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H11">
         <span class="h2">
          Chronic neutrophilic leukemia
         </span>
         <span class="headingEndMark">
          —
         </span>
         Most patients with chronic neutrophilic leukemia have no detectable cytogenetic abnormalities, but when chromosomal studies demonstrate the presence of Ph, a breakpoint in the 3' µ-bcr (e19a2, p230) is typical. This disorder is characterized by a more benign course than classical CML, although the number of patients with a µ-bcr t(9;22) is small [
         <a href="#rid12">
          12
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4543.html" rel="external">
          "Clinical manifestations and diagnosis of chronic myeloid leukemia", section on 'Chronic neutrophilic leukemia'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H12">
         <span class="h2">
          Other hematologic malignancies
         </span>
         <span class="headingEndMark">
          —
         </span>
         In rare cases,
         <em>
          BCR::ABL1
         </em>
         has been described in essential thrombocytosis, myelodysplastic syndrome, multiple myeloma (plasma cell myeloma), and malignant lymphoma. The mere presence of
         <em>
          BCR::ABL1
         </em>
         in these disorders, however, does not prove that it is the primary cause. Since the number of chromosomal abnormalities tends to increase during the course of these chronic disorders, the appearance of the Ph chromosome may be interpreted as a secondary abnormality.
        </p>
        <p>
         An alternative explanation is that the patient has been incorrectly diagnosed. As an example, the Polycythemia Vera Study Group collected 6 patients who were originally diagnosed as having essential thrombocythemia, but were Philadelphia chromosome positive. Five of the six patients subsequently developed a clinical course typical of CML, including termination in blast crisis [
         <a href="#rid32">
          32
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H18">
         <span class="h1">
          ACUTE MYELOID LEUKEMIA
         </span>
         <span class="headingEndMark">
          —
         </span>
         In approximately 45 percent of patients with acute myeloid leukemia (AML), two (or more) structural or numerical cytogenetic anomalies occur, either as primary or secondary events. Exceptions exist for acute promyelocytic leukemia (FAB subtype M3), in which t(15;17) can be demonstrated in approximately 80 percent of cases and for FAB subgroup M4E, in which most patients exhibit t(16;16) or inv(16). (See
         <a class="medical medical_review" href="/z/d/html/4523.html" rel="external">
          "Molecular biology of acute promyelocytic leukemia"
         </a>
         .)
        </p>
        <p>
         The identification of cytogenetic anomalies affects initial diagnosis in ambiguous cases and becomes highly relevant for the determination of prognosis, appropriate clinical management, detection of relapse, and the elucidation of AML pathogenesis [
         <a href="#rid33">
          33
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4525.html" rel="external">
          "Acute myeloid leukemia: Induction therapy in medically fit adults", section on 'Pretreatment'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/86177.html" rel="external">
          "Pretreatment evaluation and prognosis of acute myeloid leukemia in older adults", section on 'Prognosis'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/4525.html" rel="external">
          "Acute myeloid leukemia: Induction therapy in medically fit adults", section on 'Introduction'
         </a>
         .)
        </p>
        <p>
         Numerical abnormalities that consist of the gain of a chromosome in the clonal cell population (hyperdiploidy) include trisomy 4, 8, 21 and 22. Of these, chromosome 8 is most frequently encountered, without preference for a particular FAB subtype. The most specific hyperdiploidy is trisomy 22 which often occurs in conjunction with inv(16)-positive subgroup FAB M4E. In contrast, hypodiploidy, and monosomy 7 or 7q- in particular, is most often identified in FAB subtypes M4 and M6. The table summarizes the most common cytogenetic changes in each FAB subgroup, most of which are structural rather than numerical  (
         <a class="graphic graphic_table graphicRef71750" href="/z/d/graphic/71750.html" rel="external">
          table 2
         </a>
         ) [
         <a href="#rid34">
          34
         </a>
         ].
        </p>
        <p>
         Many genes that are involved in AML chromosomal translocations and inversions have now been identified [
         <a href="#rid35">
          35
         </a>
         ]. Perhaps the best known example is
         <em>
          KMT2A
         </em>
         (previously known as
         <em>
          MLL
         </em>
         [mixed lineage leukemia]) gene on chromosome 11 band q23. This gene is implicated in AML (5 percent), ALL (10 percent), infant leukemias (up to 80 percent), and leukemias of mixed AML/ALL lineage. An exceptional feature of KMT2A translocations is that fusions with a wide variety of chromosomes have been reported. The most frequently identified reciprocal translocations are t(6;11), t(11;19) and t(9;11). The latter two carry a more favorable prognosis than other 11q23 translocations [
         <a href="#rid36">
          36
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H19">
         <span class="h1">
          MYELODYSPLASTIC SYNDROMES
         </span>
         <span class="headingEndMark">
          —
         </span>
         Both the etiology of the myelodysplastic syndromes (MDS) and their genotype-phenotype correlations are complex and, as yet, largely unknown. Whereas some patients may have a benign disease course without a terminal leukemic phase, the cytogenetic abnormalities that are often present in these disorders underscore their clonal nature. Chromosomal changes accumulate in the more malignant phases of MDS, which may reflect a general genomic instability. Not surprisingly, the vast majority of cytogenetic changes in MDS are also seen in AML, supporting the view that these disorders represent a continuum rather than strictly separate entities. Nevertheless, there are several distinct characteristics of MDS, including an occurrence at a higher age than most forms of AML, multi-lineage dysplasia, and the presence of unbalanced translocations.
        </p>
        <p>
         The International Prognostic Scoring System (IPSS), which has been devised for MDS, is based in part on the presence of chromosomal abnormalities. (See
         <a class="medical medical_review" href="/z/d/html/4483.html" rel="external">
          "Cytogenetics, molecular genetics, and pathophysiology of myelodysplastic syndromes/neoplasms (MDS)", section on 'del(5q)'
         </a>
         .)
        </p>
        <p>
         An unfavorable prognosis is associated with partial deletion of chromosome 7 [
         <a href="#rid37">
          37
         </a>
         ]. Ten percent of MDS/AML patients have a monosomy 7 or a deletion of 7q; patients with these karyotypes are prone to much faster progression to acute leukemia. Deletions of the long arms of chromosomes 5, 9, 11, 13, and 20, a deletion of the short arm of chromosome 12, inversions of chromosome 3, as well as translocations t(6;9), t(8;21) and t(9;11) are also fairly common.
        </p>
        <p>
         Thirty percent of MDS patients exhibit the 5q- syndrome, characterized by a region of variable chromosomal loss which always includes the region between 5q22 and 5q31. This critical region contains several growth factor genes, loss of which could disturb hematopoietic cell differentiation [
         <a href="#rid38">
          38
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4483.html" rel="external">
          "Cytogenetics, molecular genetics, and pathophysiology of myelodysplastic syndromes/neoplasms (MDS)", section on 'del(5q)'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H20">
         <span class="h1">
          MULTIPLE MYELOMA AND MONOCLONAL GAMMOPATHY
         </span>
         <span class="headingEndMark">
          —
         </span>
         Multiple myeloma is a malignancy of plasma cells, the most differentiated of the B-lymphocytes. As these cells have limited mitotic activity, cytogenetic abnormalities are not as easily detected as in other hematopoietic neoplasias. Sensitive techniques such as FISH, however, enable the detection of one or more karyotypic anomalies in approximately 80 percent of myeloma patients [
         <a href="#rid39">
          39
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/6649.html" rel="external">
          "Multiple myeloma: Clinical features, laboratory manifestations, and diagnosis", section on 'Cytogenetics'
         </a>
         .)
        </p>
        <p>
         A chromosome defect is identified in nearly one-half of patients with monoclonal gammopathy of undetermined significance (MGUS), while multiple anomalies are much more prevalent in MM. The cytogenetic deviations in MM are mostly non-specific, a fact that has hampered prognostic correlations. Patients with abnormal karyotypes are hyperdiploid in 65 percent and carry hypodiploidy in 20 percent. Partial chromosome gains or losses, as well as other structural abnormalities are present in 15 percent of patients. In this group, a 14q+ marker is most frequently encountered. A relatively poor prognosis is associated with partial or complete deletion of chromosome 13, or anomalies of the long arm of chromosome 11. When these abnormalities co-exist, the average event-free survival after autologous bone-marrow transplant is reduced by one-half, compared with patients with a single defect in chromosome 11 or 13 [
         <a href="#rid40">
          40
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H21">
         <span class="h1">
          B LYMPHOBLASTIC LEUKEMIA/LYMPHOMA
         </span>
         <span class="headingEndMark">
          —
         </span>
         Some B cell acute lymphoblastic leukemia/lymphoma (B-ALL) are associated with recurrent genetic abnormalities. The translocation t(9;22), which creates the Philadelphia chromosome (Ph), is present in approximately 25 percent of adults and 5 percent of children affected by B-ALL. In the presence of Ph, the likelihood of a favorable outcome in both children and adults is significantly reduced compared with other cytogenetic changes [
         <a href="#rid34">
          34
         </a>
         ]. In children, the t(12;21)(p13;q22) or ETV6-RUNX1 (formerly TEL-AML1) rearrangement occurs in approximately 25 percent of cases of B-ALL and is associated with a favorable prognosis [
         <a href="#rid41">
          41,42
         </a>
         ].
        </p>
        <p>
         Numerical chromosome changes are present in a minority of patients with B-ALL. Hypodiploidy carries a relatively poor prognosis, but is a feature in less than 8 percent of children with ALL. An increased chromosome number is seen in approximately 40 percent of children with ALL and is considered a favorable prognostic feature, except in the presence of i(17q). Hyperdiploidy is often accompanied by structural rearrangements, but recurrent translocations are rare and the isolation of specific breakpoint genes remains impractical. Hence, in order to monitor residual disease, interphase FISH is generally more useful than RT-PCR, provided that the appropriate probes are available. (See
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         .)
        </p>
        <p>
         Exceptions to this rule include translocations t(1;19)(q23;p13) and its variant t(17;19)(q21;p13). At least one quarter of childhood pre-B cell ALL and 5 percent of all pediatric ALLs are associated with this rearrangement. The hybrid transcript of the PBX and E2A genes or the HLF and E2A genes, respectively, can easily be detected by RT-PCR. Similarly, RT-PCR can be applied for the detection of the KMT2A-AF4 fusion hybrid in t(4;11), which is present in 70 percent of affected children younger than 12 months. The presence of this, or other KMT2A fusions, predicts a poor prognosis. (See
         <a class="medical medical_review" href="/z/d/html/4484.html" rel="external">
          "Classification, cytogenetics, and molecular genetics of acute lymphoblastic leukemia/lymphoma"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H2769079">
         <span class="h1">
          T LYMPHOBLASTIC LEUKEMIA/LYMPHOMA
         </span>
         <span class="headingEndMark">
          —
         </span>
         In the present World Health Organization (WHO) classification, T cell acute lymphoblastic leukemia (T-ALL) and T cell lymphoblastic lymphoma (T-LBL) are represented in a single category of T lymphoblastic leukemia/lymphoma. It is not yet clear, however, whether T-ALL and T-LBL are indeed reflective of the same conditions or should be regarded as two separate entities. A meta-analysis has demonstrated that translocations involving chromosome region 9q34 are significantly more common in T-LBL than in T-ALL. In this chromosomal region,
         <em>
          SET, ABL1, NUP214
         </em>
         , and
         <em>
          NOTCH1
         </em>
         play a role in leukemogenesis [
         <a href="#rid43">
          43
         </a>
         ]. Fifty to 70 percent of T-ALL are associated with cytogenetic abnormalities, most commonly involving the
         <em>
          TCR
         </em>
         gene loci (7p14-15, TCRG locus; 7q35, TCRB locus; 14q11, TCRA and TCRD) with a number of partner genes. Partners include TAL1 (1P32) in up to 30 percent of childhood T-ALL [
         <a href="#rid44">
          44
         </a>
         ],
         <em>
          HOX11
         </em>
         (10q24),
         <em>
          HOX11L2
         </em>
         (5q35),
         <em>
          LMO1
         </em>
         ,
         <em>
          LMO2
         </em>
         ,
         <em>
          NOTCH1
         </em>
         , and
         <em>
          LYL1
         </em>
         (19p13) [
         <a href="#rid45">
          45
         </a>
         ]. Deletions of
         <em>
          CDKN2A
         </em>
         occur in more than 30 percent of patients and lead to cell cycle activation. Mutations resulting in activation of
         <em>
          NOTCH1
         </em>
         occur in 50 percent of T-ALL and result in a proliferative signal via the
         <em>
          MYC
         </em>
         gene. Activating mutations in
         <em>
          FLT3
         </em>
         and
         <em>
          NRAS
         </em>
         have also been described, as have gene duplications of MYB and the formation of fusion genes [
         <a href="#rid45">
          45
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H22">
         <span class="h1">
          CHRONIC B CELL AND T CELL LEUKEMIAS
         </span>
         <span class="headingEndMark">
          —
         </span>
         Based on morphological, immunophenotypic, and genetic evidence, most patients with chronic lymphocytic leukemias (eg, chronic lymphocytic leukemia, B cell prolymphocytic leukemia, hairy cell leukemia) demonstrate clonal abnormalities in mature B cells. The subset of chronic T cell leukemias includes T cell prolymphocytic leukemia and large granular lymphocyte leukemia.
        </p>
        <p>
         As these chronic leukemias are primarily disorders of mature cells which may not readily divide in culture, cytogenetic abnormalities are not readily detectable by metaphase analysis. However, molecular cytogenetic methods such as FISH have greatly improved sensitivity [
         <a href="#rid46">
          46
         </a>
         ]. The cytogenetic abnormalities occurring with the highest incidence in chronic lymphocytic leukemia are trisomy 12 in approximately 20 percent of cases and rearrangements of chromosome 13 and 14. In half of the patients with trisomy 12, the cells exhibit atypical morphology, as opposed to fewer than 10 percent of patients without this aneuploidy. Survival, however, seems to be negatively influenced only in patients with low stage disease. Loss of heterozygosity in tumor suppressor genes, such as
         <em>
          RB1
         </em>
         in chromosome band 13q14, BRCA2 in 13q12 and P53 in 17p12 are frequent findings in accelerated CLL, but are highly non-specific. (See
         <a class="medical medical_review" href="/z/d/html/4539.html" rel="external">
          "Pathobiology of chronic lymphocytic leukemia"
         </a>
         .)
        </p>
        <p>
         Interphase fluorescence in situ hybridization (FISH) has shown abnormalities in at least 80 percent of CLL patients. In one study, 13q deletion was the most commonly observed anomaly (55 percent of patients), while 11q deletion and trisomy 12 were observed in approximately 20 percent [
         <a href="#rid47">
          47
         </a>
         ]. There were at least three categories of risk, as defined by the FISH anomalies – a poor risk group that consisted of 17p or p53 deletions, an intermediate risk group that consisted of trisomy 12q or a normal karyotype, and a good risk group that consisted of 13q deletions.
        </p>
        <p>
         T cell PLL is characterized by complex rearrangements, the most consistent of which are t(14;14)(q11;q32) or inv(14)(q11;q32). The T cell receptor (TCR) gene on 14q11 is thus merged with the TCL1 gene on 14q32, resulting in overexpression of the latter. Another frequent finding in T cell PLL is partial trisomy 8 with upregulation of the
         <em>
          MYC
         </em>
         oncogene. (See
         <a class="medical medical_review" href="/z/d/html/4745.html" rel="external">
          "Clinical manifestations, pathologic features, and diagnosis of T cell prolymphocytic leukemia"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H23">
         <span class="h2">
          Hodgkin lymphoma
         </span>
         <span class="headingEndMark">
          —
         </span>
         Despite major advances in cytogenetics, the etiology of Hodgkin lymphoma remains elusive. Many patients with Hodgkin lymphoma appear to have normal karyotypes, although some tumors exhibit complex chromosomal changes and gross aneuploidy. Hyperploidy and deletions of chromosomes 1p, 4q, 6q or 7q are regularly encountered in Hodgkin lymphoma. A specific rearrangement that could lead to the identification of affected genes, however, has not yet been found [
         <a href="#rid48">
          48
         </a>
         ].
        </p>
        <p>
         In contrast, in non-Hodgkin lymphoma, several disease-specific chromosomal changes have been reported [
         <a href="#rid49">
          49
         </a>
         ]. These are detailed below.
        </p>
        <p class="headingAnchor" id="H24">
         <span class="h2">
          Burkitt lymphoma
         </span>
         <span class="headingEndMark">
          —
         </span>
         The translocations in Burkitt lymphoma are imprecise events, compared with the exact gene fusions present in CML and ALL. Burkitt lymphoma arises when
         <em>
          MYC
         </em>
         on chromosome 8, band q24 is juxtaposed to one of the immunoglobulin loci on chromosome 14 (the immunoglobulin heavy chain), chromosome 22 (the lambda light chain), or chromosome 2 (the kappa light chain). The translocation breakpoints are variable and occur in a very large region in and around the MYC gene, but all lead to the aberrant overexpression of this otherwise tightly regulated gene. (See
         <a class="medical medical_review" href="/z/d/html/4700.html" rel="external">
          "Epidemiology, clinical manifestations, pathologic features, and diagnosis of Burkitt lymphoma", section on 'Translocations involving the MYC oncogene'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H25">
         <span class="h2">
          Follicular lymphoma
         </span>
         <span class="headingEndMark">
          —
         </span>
         The translocation t(14;18)(q32;q21), which is identified in approximately 80 percent of follicular B cell lymphomas and almost one third of diffuse large cell lymphomas, is often accompanied by additional chromosome aberrations. By this translocation, which is another example of an imprecise fusion, the
         <em>
          BCL2
         </em>
         gene on chromosome 18q is juxtaposed to the constitutively expressed Ig heavy chain gene on chromosome 14q;
         <em>
          BCL2
         </em>
         is upregulated as a consequence. (See
         <a class="medical medical_review" href="/z/d/html/4699.html" rel="external">
          "Clinical manifestations, pathologic features, diagnosis, and prognosis of follicular lymphoma", section on 'Cytogenetics'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H26">
         <span class="h2">
          Mantle cell lymphoma
         </span>
         <span class="headingEndMark">
          —
         </span>
         The cyclin D1 gene (
         <em>
          CCND1
         </em>
         ) is often overexpressed in mantle cell lymphoma, when t(11;14)(q13;q32), which is present in almost all cases of mantle cell lymphoma, places this gene adjacent to the immunoglobulin heavy chain gene. (See
         <a class="medical medical_review" href="/z/d/html/4702.html" rel="external">
          "Mantle cell lymphoma: Epidemiology, pathobiology, clinical manifestations, diagnosis, and prognosis", section on 'Genetic features'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H27">
         <span class="h2">
          Anaplastic lymphoma
         </span>
         <span class="headingEndMark">
          —
         </span>
         Thirty to 40 percent of pediatric large cell lymphomas are anaplastic. The typical rearrangement in these instances is t(2;5)(p23;q35). This translocation is an occasional manifestation in adult primary systemic anaplastic large cell lymphomas as well, and confers a good prognosis [
         <a href="#rid50">
          50
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/4705.html" rel="external">
          "Clinical manifestations, pathologic features, and diagnosis of systemic anaplastic large cell lymphoma (sALCL)", section on 'Pathogenesis'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H69718091">
         <span class="h1">
          SUMMARY
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Recurrent genetic abnormalities in hematologic malignancies
         </strong>
         – Cancer-specific chromosomal translocations contribute directly to malignant transformation in many hematologic malignancies. In addition, multiple, random cytogenetic abnormalities are a characteristic finding in advanced malignancies.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          CML as a model system
         </strong>
         – The myeloproliferative neoplasm, chronic myeloid leukemia (CML) provides a model for cancer-specific chromosomal and gene translocations. CML is derived from an abnormal pluripotent stem cell that has acquired the
         <em>
          BCR::ABL1
         </em>
         fusion gene. (See
         <a class="local">
          'Chronic myeloid leukemia as a model system'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          <em>
           BCR::ABL1
          </em>
         </strong>
         – The
         <em>
          BCR::ABL1
         </em>
         fusion gene is the product of a balanced translocation between chromosomes 9 and 22, denoted t(9;22) (q34.1;q11.21), which juxtaposes a 5’ segment of a breakpoint cluster region (
         <em>
          BCR
         </em>
         ) at 22q11 and the 3' segment of the
         <em>
          ABL1
         </em>
         at 9q34, resulting in an abnormally small chromosome known as the Philadelphia (Ph) chromosome. As a consequence of the translocation, chromosome 9 should also exhibit fusion with the telomeric chromosome piece that was separated from chromosome 22  (
         <a class="graphic graphic_figure graphicRef62227" href="/z/d/graphic/62227.html" rel="external">
          figure 1
         </a>
         ), reported as the reciprocal product der(9). (See
         <a class="local">
          'BCR and ABL1 genes'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         <strong>
          Genetic testing
         </strong>
         – Genetic testing for the Ph chromosome, BCR::ABL1 fusion gene, or the fusion mRNA gene product can be done by conventional cytogenetic analysis (karyotyping), fluorescence in situ hybridization (FISH) analysis, or by reverse transcription polymerase chain reaction (RT-PCR). Karyotyping is widely applied for the diagnosis of CML while quantitative RT-PCR is frequently employed to assess response to therapy. (See
         <a class="local">
          'Detection of Ph and BCR::ABL1'
         </a>
         above and
         <a class="local">
          'Monitoring of residual disease'
         </a>
         above and
         <a class="medical medical_review" href="/z/d/html/2893.html" rel="external">
          "Tools for genetics and genomics: Cytogenetics and molecular genetics"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          <em>
           BCR::ABL1
          </em>
         </strong>
         <strong>
          in other malignancies
         </strong>
         – The Ph chromosome is not limited to CML; it is also seen in acute lymphoblastic leukemia (ALL), acute myeloid leukemia (AML), and other hematologic malignancies. The number of chromosomal abnormalities tends to increase during the course of these disorders and the appearance of the Ph chromosome may be a secondary abnormality. (See
         <a class="local">
          'The Ph chromosome in malignancies other than CML'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Other recurrent genetic abnormalities
         </strong>
         – Other recurrent genetic abnormalities are commonly found in other hematologic and lymphoid malignancies. Assessment for genetic changes is routinely used as in the diagnostic evaluation, assessment of prognosis, and disease management.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H1935243401">
         <span class="h1">
          ACKNOWLEDGMENT
         </span>
         <span class="headingEndMark">
          —
         </span>
         The UpToDate editorial staff acknowledges Athena M Cherry, PhD, who contributed to an earlier version of this topic review.
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li>
          <a class="nounderline abstract_t">
           Mitelman F, Johansson B, Mertens F. The impact of translocations and gene fusions on cancer causation. Nat Rev Cancer 2007; 7:233.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rowley JD. Chromosomal translocations: revisited yet again. Blood 2008; 112:2183.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Nowell PC, Hungerford DA. A minute chromosome in human granulocytic leukemia. Science 1960; 132:1497.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Warmuth M, Danhauser-Riedl S, Hallek M. Molecular pathogenesis of chronic myeloid leukemia: implications for new therapeutic strategies. Ann Hematol 1999; 78:49.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Call TG, Noël P, Habermann TM, et al. Incidence of leukemia in Olmsted County, Minnesota, 1975 through 1989. Mayo Clin Proc 1994; 69:315.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Takahashi N, Miura I, Saitoh K, Miura AB. Lineage involvement of stem cells bearing the philadelphia chromosome in chronic myeloid leukemia in the chronic phase as shown by a combination of fluorescence-activated cell sorting and fluorescence in situ hybridization. Blood 1998; 92:4758.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rowley JD. The Philadelphia chromosome translocation. A paradigm for understanding leukemia. Cancer 1990; 65:2178.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Chissoe SL, Bodenteich A, Wang YF, et al. Sequence and analysis of the human ABL gene, the BCR gene, and regions involved in the Philadelphia chromosomal translocation. Genomics 1995; 27:67.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Groffen J, Heisterkamp N. The chimeric BCR-ABL gene. Baillieres Clin Haematol 1997; 10:187.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Chen SJ, Chen Z, Hillion J, et al. Ph1-positive, bcr-negative acute leukemias: clustering of breakpoints on chromosome 22 in the 3' end of the BCR gene first intron. Blood 1989; 73:1312.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Saglio G, Guerrasio A, Rosso C, et al. New type of Bcr/Abl junction in Philadelphia chromosome-positive chronic myelogenous leukemia. Blood 1990; 76:1819.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Melo JV. BCR-ABL gene variants. Baillieres Clin Haematol 1997; 10:203.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Pane F, Frigeri F, Sindona M, et al. Neutrophilic-chronic myeloid leukemia: a distinct disease with a specific molecular marker (BCR/ABL with C3/A2 junction). Blood 1996; 88:2410.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Melo JV, Gordon DE, Cross NC, Goldman JM. The ABL-BCR fusion gene is expressed in chronic myeloid leukemia. Blood 1993; 81:158.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sinclair PB, Nacheva EP, Leversha M, et al. Large deletions at the t(9;22) breakpoint are common and may identify a poor-prognosis subgroup of patients with chronic myeloid leukemia. Blood 2000; 95:738.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Huntly BJ, Reid AG, Bench AJ, et al. Deletions of the derivative chromosome 9 occur at the time of the Philadelphia translocation and provide a powerful and independent prognostic indicator in chronic myeloid leukemia. Blood 2001; 98:1732.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hochhaus A, Reiter A, Skladny H, et al. A novel BCR-ABL fusion gene (e6a2) in a patient with Philadelphia chromosome-negative chronic myelogenous leukemia. Blood 1996; 88:2236.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           How GF, Lim LC, Kulkarni S, et al. Two patients with novel BCR/ABL fusion transcripts (e8/a2 and e13/a2) resulting from translocation breakpoints within BCR exons. Br J Haematol 1999; 105:434.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Shepherd P, Suffolk R, Halsey J, Allan N. Analysis of molecular breakpoint and m-RNA transcripts in a prospective randomized trial of interferon in chronic myeloid leukaemia: no correlation with clinical features, cytogenetic response, duration of chronic phase, or survival. Br J Haematol 1995; 89:546.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Melo JV. The diversity of BCR-ABL fusion proteins and their relationship to leukemia phenotype. Blood 1996; 88:2375.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           How GF, Tan LT, Lim LC. Chronic myeloid leukemia with e19a2 (c3a2) BCR/ABL fusion junction--is it truly a benign disease? Leukemia 1998; 12:1166.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cortes JE, Talpaz M, Kantarjian H. Chronic myelogenous leukemia: a review. Am J Med 1996; 100:555.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Faderl S, Talpaz M, Kantarjian HM, Estrov Z. Should polymerase chain reaction analysis to detect minimal residual disease in patients with chronic myelogenous leukemia be used in clinical decision making? Blood 1999; 93:2755.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Reddy KS, Grove B. A Philadelphia-negative chronic myeloid leukemia with a BCR/ABL fusion gene on chromosome 9. Cancer Genet Cytogenet 1998; 107:48.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yee K, Anglin P, Keating A. Molecular approaches to the detection and monitoring of chronic myeloid leukemia: theory and practice. Blood Rev 1999; 13:105.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kurzrock R, Kantarjian HM, Shtalrid M, et al. Philadelphia chromosome-negative chronic myelogenous leukemia without breakpoint cluster region rearrangement: a chronic myeloid leukemia with a distinct clinical course. Blood 1990; 75:445.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Suryanarayan K, Hunger SP, Kohler S, et al. Consistent involvement of the bcr gene by 9;22 breakpoints in pediatric acute leukemias. Blood 1991; 77:324.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Westbrook CA, Hooberman AL, Spino C, et al. Clinical significance of the BCR-ABL fusion gene in adult acute lymphoblastic leukemia: a Cancer and Leukemia Group B Study (8762). Blood 1992; 80:2983.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Paietta E, Racevskis J, Bennett JM, et al. Biologic heterogeneity in Philadelphia chromosome-positive acute leukemia with myeloid morphology: the Eastern Cooperative Oncology Group experience. Leukemia 1998; 12:1881.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kantarjian HM, Talpaz M, Dhingra K, et al. Significance of the P210 versus P190 molecular abnormalities in adults with Philadelphia chromosome-positive acute leukemia. Blood 1991; 78:2411.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Haskovec C, Ponzetto C, Polák J, et al. P230 BCR/ABL protein may be associated with an acute leukaemia phenotype. Br J Haematol 1998; 103:1104.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Stoll DB, Peterson P, Exten R, et al. Clinical presentation and natural history of patients with essential thrombocythemia and the Philadelphia chromosome. Am J Hematol 1988; 27:77.
          </a>
         </li>
         <li class="breakAll">
          Heim S, Mitelman F. Acute myeloid leukemia. In: Cancer Cytogenetics, Wiley-Liss, New York 1995. p.69.
         </li>
         <li>
          <a class="nounderline abstract_t">
           Martinez-Climent JA. Molecular cytogenetics of childhood hematological malignancies. Leukemia 1997; 11:1999.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Martinez-Climent JA, García-Conde J. Chromosomal rearrangements in childhood acute myeloid leukemia and myelodysplastic syndromes. J Pediatr Hematol Oncol 1999; 21:91.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rowley JD. The critical role of chromosome translocations in human leukemias. Annu Rev Genet 1998; 32:495.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Greenberg P, Cox C, LeBeau MM, et al. International scoring system for evaluating prognosis in myelodysplastic syndromes. Blood 1997; 89:2079.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Dewald GW, Wright PI. Chromosome abnormalities in the myeloproliferative disorders. Semin Oncol 1995; 22:341.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Tabernero D, San Miguel JF, Garcia-Sanz M, et al. Incidence of chromosome numerical changes in multiple myeloma: fluorescence in situ hybridization analysis using 15 chromosome-specific probes. Am J Pathol 1996; 149:153.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Feinman R, Sawyer J, Hardin J, Tricot G. Cytogenetics and molecular genetics in multiple myeloma. Hematol Oncol Clin North Am 1997; 11:1.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Romana SP, Poirel H, Leconiat M, et al. High frequency of t(12;21) in childhood B-lineage acute lymphoblastic leukemia. Blood 1995; 86:4263.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lanza C, Volpe G, Basso G, et al. Outcome and lineage involvement in t(12;21) childhood acute lymphoblastic leukaemia. Br J Haematol 1997; 97:460.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sekimizu M, Sunami S, Nakazawa A, et al. Chromosome abnormalities in advanced stage T-cell lymphoblastic lymphoma of children and adolescents: a report from Japanese Paediatric Leukaemia/Lymphoma Study Group (JPLSG) and review of the literature. Br J Haematol 2011; 154:612.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Cazzaniga G, Rossi V, Biondi A. Monitoring minimal residual disease using chromosomal translocations in childhood ALL. Best Pract Res Clin Haematol 2002; 15:21.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Aifantis I, Raetz E, Buonamici S. Molecular pathogenesis of T-cell leukaemia and lymphoma. Nat Rev Immunol 2008; 8:380.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Brito-Babapulle V, Garcia-Marco J, Maljaie SH, et al. The impact of molecular cytogenetics on chronic lymphoid leukaemia. Acta Haematol 1997; 98:175.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Döhner H, Stilgenbauer S, Benner A, et al. Genomic aberrations and survival in chronic lymphocytic leukemia. N Engl J Med 2000; 343:1910.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Atkin NB. Cytogenetics of Hodgkin's disease. Cytogenet Cell Genet 1998; 80:23.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Willis TG, Dyer MJ. The role of immunoglobulin translocations in the pathogenesis of B-cell malignancies. Blood 2000; 96:808.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Heerema NA. Chromosomes in lymphomas and solid tumors. Cancer Invest 1998; 16:183.
          </a>
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 4500 Version 18.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17361217" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : The impact of translocations and gene fusions on cancer causation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18779403" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : Chromosomal translocations: revisited yet again.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : A minute chromosome in human granulocytic leukemia
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10089019" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : Molecular pathogenesis of chronic myeloid leukemia: implications for new therapeutic strategies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8170174" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : Incidence of leukemia in Olmsted County, Minnesota, 1975 through 1989.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9845542" id="rid5" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          6 : Lineage involvement of stem cells bearing the philadelphia chromosome in chronic myeloid leukemia in the chronic phase as shown by a combination of fluorescence-activated cell sorting and fluorescence in situ hybridization.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2189548" id="rid6" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          7 : The Philadelphia chromosome translocation. A paradigm for understanding leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7665185" id="rid7" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          8 : Sequence and analysis of the human ABL gene, the BCR gene, and regions involved in the Philadelphia chromosomal translocation.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9376659" id="rid8" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          9 : The chimeric BCR-ABL gene.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2930840" id="rid9" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          10 : Ph1-positive, bcr-negative acute leukemias: clustering of breakpoints on chromosome 22 in the 3' end of the BCR gene first intron.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2224129" id="rid10" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          11 : New type of Bcr/Abl junction in Philadelphia chromosome-positive chronic myelogenous leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9376660" id="rid11" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          12 : BCR-ABL gene variants.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8839830" id="rid12" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          13 : Neutrophilic-chronic myeloid leukemia: a distinct disease with a specific molecular marker (BCR/ABL with C3/A2 junction)
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8417787" id="rid13" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          14 : The ABL-BCR fusion gene is expressed in chronic myeloid leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10648381" id="rid14" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          15 : Large deletions at the t(9;22) breakpoint are common and may identify a poor-prognosis subgroup of patients with chronic myeloid leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11535505" id="rid15" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          16 : Deletions of the derivative chromosome 9 occur at the time of the Philadelphia translocation and provide a powerful and independent prognostic indicator in chronic myeloid leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8822944" id="rid16" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          17 : A novel BCR-ABL fusion gene (e6a2) in a patient with Philadelphia chromosome-negative chronic myelogenous leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10233416" id="rid17" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          18 : Two patients with novel BCR/ABL fusion transcripts (e8/a2 and e13/a2) resulting from translocation breakpoints within BCR exons.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7734353" id="rid18" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          19 : Analysis of molecular breakpoint and m-RNA transcripts in a prospective randomized trial of interferon in chronic myeloid leukaemia: no correlation with clinical features, cytogenetic response, duration of chronic phase, or survival.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8839828" id="rid19" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          20 : The diversity of BCR-ABL fusion proteins and their relationship to leukemia phenotype.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9665206" id="rid20" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          21 : Chronic myeloid leukemia with e19a2 (c3a2) BCR/ABL fusion junction--is it truly a benign disease?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8644769" id="rid21" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          22 : Chronic myelogenous leukemia: a review.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10216068" id="rid22" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          23 : Should polymerase chain reaction analysis to detect minimal residual disease in patients with chronic myelogenous leukemia be used in clinical decision making?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9809034" id="rid23" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          24 : A Philadelphia-negative chronic myeloid leukemia with a BCR/ABL fusion gene on chromosome 9.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10414946" id="rid24" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          25 : Molecular approaches to the detection and monitoring of chronic myeloid leukemia: theory and practice.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/2403827" id="rid25" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          26 : Philadelphia chromosome-negative chronic myelogenous leukemia without breakpoint cluster region rearrangement: a chronic myeloid leukemia with a distinct clinical course.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/1985699" id="rid26" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          27 : Consistent involvement of the bcr gene by 9;22 breakpoints in pediatric acute leukemias.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/1467514" id="rid27" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          28 : Clinical significance of the BCR-ABL fusion gene in adult acute lymphoblastic leukemia: a Cancer and Leukemia Group B Study (8762).
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9844918" id="rid28" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          29 : Biologic heterogeneity in Philadelphia chromosome-positive acute leukemia with myeloid morphology: the Eastern Cooperative Oncology Group experience.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/1932753" id="rid29" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          30 : Significance of the P210 versus P190 molecular abnormalities in adults with Philadelphia chromosome-positive acute leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9886327" id="rid30" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          31 : P230 BCR/ABL protein may be associated with an acute leukaemia phenotype.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/3422539" id="rid31" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          32 : Clinical presentation and natural history of patients with essential thrombocythemia and the Philadelphia chromosome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/3422539" id="rid32" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          33 : Clinical presentation and natural history of patients with essential thrombocythemia and the Philadelphia chromosome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9447814" id="rid33" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          34 : Molecular cytogenetics of childhood hematological malignancies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10206454" id="rid34" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          35 : Chromosomal rearrangements in childhood acute myeloid leukemia and myelodysplastic syndromes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9928489" id="rid35" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          36 : The critical role of chromosome translocations in human leukemias.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9058730" id="rid36" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          37 : International scoring system for evaluating prognosis in myelodysplastic syndromes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7638632" id="rid37" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          38 : Chromosome abnormalities in the myeloproliferative disorders.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/8686739" id="rid38" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          39 : Incidence of chromosome numerical changes in multiple myeloma: fluorescence in situ hybridization analysis using 15 chromosome-specific probes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9081201" id="rid39" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          40 : Cytogenetics and molecular genetics in multiple myeloma.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/7492786" id="rid40" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          41 : High frequency of t(12;21) in childhood B-lineage acute lymphoblastic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9163615" id="rid41" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          42 : Outcome and lineage involvement in t(12;21) childhood acute lymphoblastic leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21689091" id="rid42" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          43 : Chromosome abnormalities in advanced stage T-cell lymphoblastic lymphoma of children and adolescents: a report from Japanese Paediatric Leukaemia/Lymphoma Study Group (JPLSG) and review of the literature.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11987914" id="rid43" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          44 : Monitoring minimal residual disease using chromosomal translocations in childhood ALL.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18421304" id="rid44" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          45 : Molecular pathogenesis of T-cell leukaemia and lymphoma.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9401494" id="rid45" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          46 : The impact of molecular cytogenetics on chronic lymphoid leukaemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/11136261" id="rid46" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          47 : Genomic aberrations and survival in chronic lymphocytic leukemia.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9678329" id="rid47" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          48 : Cytogenetics of Hodgkin's disease.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/10910891" id="rid48" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          49 : The role of immunoglobulin translocations in the pathogenesis of B-cell malignancies.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/9541633" id="rid49" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          50 : Chromosomes in lymphomas and solid tumors.
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
