<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Overview of antibacterial susceptibility testing</title><link href="/z/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/z/d/css/icofont.min.css" rel="stylesheet"/><link href="/z/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/z/d/css/animate.css" rel="stylesheet"/><link href="/z/d/css/owl.carousel.css" rel="stylesheet"/><link href="/z/d/css/magnific-popup.css" rel="stylesheet"/><link href="/z/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/z/d/css/style.css" rel="stylesheet"/><link href="/z/d/css/rtl_edited.css" rel="stylesheet"/><link href="/z/d/css/responsive.css" rel="stylesheet"/><link href="https:///uptodate_assets/main.css" rel="stylesheet"/>
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>
<meta content="" name="description"/>
<link href="/z/d/img/favicon.png" rel="icon" type="image/png">
</link></head>
<body>
<!-- Start Main Menu Area -->
<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>
<section class="teacher-area ptb-10">
<div class="container">
</div>
<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Overview of antibacterial susceptibility testing</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Overview of antibacterial susceptibility testing</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Virginia M Pierce, MD</a></dd><dt><span> </span>Section Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Stephen B Calderwood, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Keri K Hall, MD, MS</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Apr 06, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H2281674909"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>The clinical microbiology laboratory serves as a valuable ally to clinicians in the diagnosis and treatment of infectious diseases. In particular, the isolation of bacteria from clinical samples yields information that can be used to guide the selection of appropriate antibiotic regimens based on knowledge of the most likely susceptibility profile of certain bacterial species. Through the use of in vitro antimicrobial susceptibility testing, the laboratory can specifically determine which antibiotics effectively inhibit the growth of a given bacterial isolate, allowing for targeted therapy. Antimicrobial resistance is a growing concern in both community and health care settings; as such, decisions surrounding empirical antibiotic treatment are becoming more complicated, and the importance of routine antimicrobial susceptibility testing to guide therapeutic decisions has increased.</p><p>Multiple different methods for antimicrobial susceptibility testing exist, including conventional methods, automated systems, and newer molecular techniques. Understanding these methods allows clinicians to correctly interpret susceptibility testing results reported by the clinical microbiology laboratory. In general, antimicrobial susceptibility testing methods used in clinical laboratories should:</p><p class="bulletIndent1"><span class="glyph">●</span>Provide rapid and accurate information to the clinician</p><p class="bulletIndent1"><span class="glyph">●</span>Be relatively inexpensive</p><p class="bulletIndent1"><span class="glyph">●</span>Be relatively easy to perform</p><p></p><p>This topic will provide an overview of antimicrobial susceptibility testing with a focus on bacterial isolates. Routine susceptibility testing of viruses and parasites is not performed in most clinical laboratories and is beyond the scope of this discussion. An overview of antimicrobial susceptibility testing in fungi is presented elsewhere. (See  <a class="medical medical_review" href="/z/d/html/465.html" rel="external">"Antifungal susceptibility testing"</a>.)</p><p class="headingAnchor" id="H253793423"><span class="h1">BASIC CONCEPTS OF ANTIMICROBIAL RESISTANCE</span></p><p class="headingAnchor" id="H390011865"><span class="h2">Intrinsic versus acquired resistance</span><span class="headingEndMark"> — </span>Bacteria can have either intrinsic or acquired resistance to antimicrobials. </p><p>Intrinsic resistance is the inherent resistance to an antimicrobial that all or almost all members of a species display, rendering susceptibility testing unnecessary [<a href="#rid1">1</a>]. As an example, <em>Klebsiella pneumoniae</em> is intrinsically resistant to the antimicrobial <a class="drug drug_general" data-topicid="8737" href="/z/d/drug information/8737.html" rel="external">ampicillin</a>. </p><p>In contrast, acquired resistance is the development of resistance to an antimicrobial to which members of the wild-type bacterial population are susceptible. Bacteria can acquire resistance through chromosomal mutations; through the horizontal transfer of genes via plasmids, integrons, transposons, or transformation; or through a combination of these mechanisms [<a href="#rid2">2</a>]. Unlike intrinsic resistance, acquired resistance in a specific bacterial isolate is not reliably predictable. The goal of antimicrobial susceptibility testing is to determine the degree of acquired resistance to antibiotics that might be employed therapeutically.</p><p class="headingAnchor" id="H2798790166"><span class="h2">Constitutive versus inducible resistance mechanisms</span><span class="headingEndMark"> — </span>The expression of some bacterial resistance mechanisms is variable, potentially complicating their detection in the microbiology laboratory and requiring special consideration. </p><p>Constitutively expressed resistance mechanisms are expressed continuously, while inducible expression occurs following exposure to a particular inciting agent [<a href="#rid3">3</a>]. For example, use of third generation cephalosporins for infections caused by certain Enterobacterales can induce production of a chromosomally encoded AmpC beta-lactamase,<em> </em>which results in resistance to this subgroup of beta-lactam antimicrobials [<a href="#rid4">4</a>]. (See  <a class="medical medical_review" href="/z/d/html/479.html" rel="external">"Beta-lactam antibiotics: Mechanisms of action and resistance and adverse effects", section on 'Chromosomal beta-lactamases'</a>.)</p><p class="headingAnchor" id="H1897544320"><span class="h2">Heteroresistance</span><span class="headingEndMark"> — </span>The phenotypic expression of an antimicrobial resistance mechanism within a bacterial population can be homogeneous or heterogeneous [<a href="#rid3">3</a>]. </p><p>Heterogeneous expression, or heteroresistance, can lead to bacterial subpopulations within a microbiologic sample that have varying degrees of phenotypic resistance, making the in vitro identification of resistance more difficult. Some conventional antimicrobial susceptibility testing methods can be insufficiently sensitive for the identification of heteroresistance, leading to the misclassification of certain bacterial strains as susceptible. Antimicrobial susceptibility testing methods that use higher inocula can overcome this barrier, facilitating the detection of small subpopulations with intermediate or resistant minimum inhibitory concentrations. Heteroresistant vancomycin-intermediate <em>Staphylococcus aureus</em> is an example of an organism with the capability of heterogenous expression [<a href="#rid5">5</a>].</p><p class="headingAnchor" id="H2964915127"><span class="h1">INDICATIONS FOR SUSCEPTIBILITY TESTING</span><span class="headingEndMark"> — </span>Antimicrobial susceptibility testing should be performed when clinically significant bacteria are isolated from patient specimens and the resulting information can be used to guide treatment. The microbiology laboratory is primarily responsible for making this determination [<a href="#rid6">6</a>]. </p><p>Susceptibility testing may not provide clinically useful information and thus may not be routinely performed in the following circumstances: </p><p class="bulletIndent1"><span class="glyph">●</span>When the antimicrobial susceptibility pattern of a particular organism is predictable. As an example, testing <em>Streptococcus pyogenes</em> for susceptibility to penicillin is not routinely performed because isolates that are not susceptible to penicillin have not been reported [<a href="#rid1">1</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>When the isolated organism is likely to represent the normal flora of the body site from which the specimen was collected (rather than a pathogen). As an example, <em>Lactobacillus </em>spp are considered part of the normal bacterial flora of the female genital tract, so their isolation in a vaginal culture would not trigger susceptibility testing.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>When insufficient numbers of bacterial colonies are grown in cultures of certain specimen types. As an example, organisms that grow in rare amounts in cultures of noninvasively collected urine specimens are likely to represent contamination and be of little clinical significance. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>When the results of in vitro antimicrobial susceptibility testing do not reliably predict in vivo<em> </em>effectiveness or a clinical therapeutic response. As an example, susceptibility testing of first- and second-generation cephalosporins against <em>Salmonella </em>and <em>Shigella </em>spp is not recommended due to the poor correlation between in vitro<em> </em>susceptibility testing results and clinical outcomes [<a href="#rid1">1</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>When guidance regarding the performance of standardized susceptibility testing for a particular organism is not available from professional groups such as the Clinical and Laboratory Standards Institute (CLSI) or the European Committee on Antimicrobial Susceptibility Testing (EUCAST), or from regulatory authorities such as the United States Food and Drug Administration (FDA). For certain infrequently isolated organisms (<em>Capnocytophaga </em>spp,<em> </em>for example), such guidance does not exist, making routine susceptibility testing and interpretation of these results difficult.</p><p></p><p>Host factors also inform the decision to perform susceptibility testing. In particular, clinicians should notify the microbiology laboratory if a patient is immunosuppressed, so that it can modify the approach to selecting appropriate bacterial isolates for susceptibility testing. Some bacteria thought to be nonpathogenic in immunocompetent hosts can cause serious infections in immunocompromised individuals, and their isolation (particularly from a normally sterile site) may warrant antimicrobial susceptibility testing. </p><p class="headingAnchor" id="H623588753"><span class="h1">CONVENTIONAL METHODS</span><span class="headingEndMark"> — </span>Conventional antimicrobial susceptibility testing methods are phenotypic in vitro tests that provide a direct measurement of antimicrobial activity. These techniques measure the activity of a specific antimicrobial agent against a clinical bacterial isolate by assessing bacterial growth in the presence of that agent. Much of the antimicrobial susceptibility testing performed in clinical laboratories relies on these conventional methods because they provide accurate and reproducible results. However, these methods have several limitations:</p><p class="bulletIndent1"><span class="glyph">●</span>They rely on bacterial growth. This requirement is often the rate-limiting step and can make susceptibility testing of more fastidious bacteria, such as <em>Granulicatella </em>spp (formerly known as nutritionally variant streptococci),<em> </em>more difficult to perform. The Clinical and Laboratory Standards Institute (CLSI) offers some guidance to microbiology laboratories on the testing of organisms that are fastidious or infrequently encountered in clinical practice (<a class="graphic graphic_table graphicRef114733" href="/z/d/graphic/114733.html" rel="external">table 1</a>) [<a href="#rid7">7</a>]. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Standardization of the testing methods is necessary to ensure accuracy as well as intra- and inter-laboratory reproducibility. This includes the selection of isolated colonies of the organism (taking care to avoid testing mixtures of different types of microorganisms), preparation of a standardized inoculum, and adherence to validated testing procedures. Performance standards for antimicrobial susceptibility testing are available from CLSI and the European Committee on Antimicrobial Susceptibility Testing (EUCAST) [<a href="#rid1">1,7-11</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>There are instances in which certain conventional testing methods are not recommended given their poor accuracy and reproducibility. As an example, disk diffusion is insensitive for the detection of vancomycin-intermediate <em>S. aureus</em>, and so this method should not be used when testing <em>S. aureus</em> isolates for susceptibility to <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a> [<a href="#rid1">1,12</a>].</p><p></p><p>Conventional antimicrobial susceptibility testing methods can be categorized into those that provide qualitative results and those that provide quantitative results, as below.</p><p class="headingAnchor" id="H1111325202"><span class="h2">Qualitative methods (disk diffusion/Kirby-Bauer method)</span><span class="headingEndMark"> — </span>Overall, qualitative conventional antimicrobial susceptibility testing methods tend to be easier to perform and interpret compared with quantitative methods. They classify the activity of an antimicrobial agent against a specific organism into one of four interpretive categories (susceptible, susceptible-dose dependent, intermediate, or resistant (see <a class="local">'Interpretation of results'</a> below)) [<a href="#rid1">1,8</a>]. The qualitative conventional testing method most commonly used by clinical microbiology laboratories is the disk diffusion, or Kirby-Bauer, method, due to its simplicity, reliability, and high degree of standardization [<a href="#rid13">13,14</a>]. </p><p class="headingAnchor" id="H1387906758"><span class="h3">Performance</span><span class="headingEndMark"> — </span>In brief, the procedure involves swabbing a standardized inoculum of bacteria (approximately 1 to 2 x 10<sup>8</sup> colony-forming units [CFU]/mL) onto a Mueller-Hinton (or other medium appropriate for the bacterial species) agar plate [<a href="#rid8">8</a>]. Commercially prepared paper disks, each embedded with a fixed concentration of a specific antimicrobial, are placed on the agar surface. During overnight incubation, the antimicrobials diffuse into the agar medium, creating a concentration gradient as each antimicrobial diffuses further away from the disk. Bacterial growth is inhibited at sites where the concentration of an antimicrobial is high enough to prevent proliferation of the organism. After incubation, the diameters of the complete growth inhibition zones around each disk are measured (<a class="graphic graphic_picture graphicRef79261" href="/z/d/graphic/79261.html" rel="external">picture 1</a>). Prespecified breakpoints are used to interpret the zone sizes and classify them as susceptible, susceptible-dose dependent, intermediate, or resistant. (See <a class="local">'Interpretation of results'</a> below.)</p><p>The diameter of the zone of inhibition is affected by both the susceptibility of the organism to a specific antibiotic and the rate at which the drug diffuses into the agar medium. Overall, there is a correlation between the size of the zone of inhibition and the minimum inhibitory concentration (MIC), but this relationship is not always linear [<a href="#rid15">15</a>]. As a result, simply measuring the zone of inhibition without knowledge of the breakpoints should not be performed [<a href="#rid1">1,7,8</a>].</p><p class="headingAnchor" id="H3314202072"><span class="h3">Advantages</span><span class="headingEndMark"> — </span>The disk diffusion method is a highly standardized method, with breakpoints established as correlates to reference broth microdilution MIC breakpoints for many organism-antimicrobial combinations [<a href="#rid16">16</a>]. In addition to its low cost and simplicity, an advantage of disk diffusion is the flexibility in the choice of tested antimicrobials; this choice can be customized in accordance with institutional formularies and local resistance patterns and can be modified easily when indicated. The qualitative results of disk diffusion testing (ie, susceptible, susceptible-dose dependent, intermediate, or resistant) are also readily interpreted by clinicians.</p><p class="headingAnchor" id="H2524354178"><span class="h3">Limitations</span><span class="headingEndMark"> — </span>Despite the numerous advantages of the disk diffusion method, there are also some important limitations. The lack of automation of the procedure makes its use in high-volume microbiology laboratories more difficult. </p><p>Furthermore, though this method is validated for most commonly encountered bacteria, its use with some more fastidious or slow-growing bacteria has not been as well studied or standardized. As an example, disk diffusion testing of HACEK group bacteria, such as <em>Aggregatibacter </em>spp,<em> Cardiobacterium </em>spp,<em> Eikenella corrodens, </em>and <em>Kingella </em>spp, is not recommended by CLSI for this reason [<a href="#rid7">7</a>]. </p><p>Finally, there are some circumstances in which the qualitative nature of disk diffusion susceptibility results is a limitation; for certain infections, a quantitative MIC is required to determine the best therapeutic approach. For example, recommended treatment regimens for endocarditis caused by viridans group streptococci are stratified by the penicillin MIC, necessitating the use of a quantitative antimicrobial susceptibility testing method [<a href="#rid1">1,17</a>].</p><p class="headingAnchor" id="H3356834294"><span class="h2">Quantitative methods</span><span class="headingEndMark"> — </span>Quantitative methods for antimicrobial susceptibility testing allow determination of the MIC, which is the lowest concentration of a specific agent that is needed to inhibit visible growth of an organism. Breakpoints published by the CLSI, EUCAST, or, in the United States, the Food and Drug Administration (FDA) [<a href="#rid18">18</a>], are used to classify MICs into susceptible, intermediate, or resistant interpretive categories. </p><p>Quantitative techniques are considered to be the most precise for the comparison of in vitro<em> </em>efficacy of different antibiotics against a specific organism. Furthermore, as noted above, there are certain infections for which a quantitative result is required in order to optimize the antimicrobial treatment regimen [<a href="#rid17">17</a>]. </p><p>Several quantitative antimicrobial susceptibility methods are currently available, including agar dilution, broth dilution (macrodilution and microdilution), and gradient methods, as detailed below.</p><p class="headingAnchor" id="H3592483893"><span class="h3">Agar dilution</span><span class="headingEndMark"> — </span>The agar dilution method is a well-studied and reproducible technique that is considered a reference standard for the susceptibility testing of many organism-antimicrobial combinations [<a href="#rid1">1,7,9,10</a>]. However, because of the high cost and labor-intensive nature of the method, the majority of clinical microbiology laboratories restrict its use to certain specific applications, such as screening for high-level <a class="drug drug_general" data-topicid="8489" href="/z/d/drug information/8489.html" rel="external">gentamicin</a> resistance in enterococci [<a href="#rid1">1</a>]. This technique is also recommended for use with fastidious bacteria that require special growth conditions, such as <em>Helicobacter pylori</em> and <em>Neisseria gonorrhoeae </em>[<a href="#rid7">7</a>]. </p><p>With this method, a fixed concentration of a specific antimicrobial is added into molten agar and allowed to solidify [<a href="#rid9">9</a>]. Agar plates containing serial twofold dilutions of the antimicrobial are created, with concentrations spanning the relevant range for testing the organism of interest. As many as 36 different bacterial isolates can be tested on a single agar plate in locations called spots, with a standardized inoculum (approximately 10<sup>4</sup> CFU) of an isolate inoculated onto a particular spot. Plates are incubated for 16 to 20 hours and subsequently examined for growth. The plate with the lowest antimicrobial concentration that visibly inhibits bacterial growth is designated as the MIC. (See <a class="local">'Interpretation of results'</a> below.)</p><p>A principle advantage of the agar dilution method is that MICs can be generated without using any special laboratory instrumentation. </p><p>As above, the main limitations of the agar dilution method are the high cost of reagents and the labor required to prepare the agar plates, which are generally not commercially available and cannot be batch-produced and stored for future use. </p><p class="headingAnchor" id="H265435726"><span class="h3">Broth dilution</span><span class="headingEndMark"> — </span>Broth dilution is performed by creating a series of twofold dilutions of a specific antimicrobial, including concentrations that cover the breakpoints for the organism of interest, and determining the lowest concentration of antimicrobial that inhibits growth (under standardized incubation conditions) of the bacterial isolate [<a href="#rid9">9</a>]. This concentration is the MIC. The principles of the broth dilution technique apply to both the broth macrodilution and broth microdilution methods. </p><p class="headingAnchor" id="H249937059"><span class="h4">Broth (tube) macrodilution</span><span class="headingEndMark"> — </span>The broth macrodilution method is mainly reserved for research settings, such as the evaluation of new antimicrobial compounds. </p><p>In the broth macrodilution procedure, serial antimicrobial dilutions are manually prepared in tubes containing 1 to 2 mL of broth medium [<a href="#rid9">9,14</a>]. Each tube is inoculated with an equal volume of a standardized bacterial suspension (containing approximately 5 x 10<sup>5</sup> CFU/mL). After overnight incubation, the tubes are visually inspected for the presence of turbidity, which is an indicator of bacterial growth. The tube with the lowest concentration of the specific antimicrobial in which turbidity is absent is the MIC (see <a class="local">'Interpretation of results'</a> below). The precision of the broth macrodilution method is considered to be plus or minus one twofold dilution.</p><p>Broth macrodilution is a time-honored technique that has been well studied and standardized, making it a reliable tool for the detection of antimicrobial resistance. However, limitations include its lack of automation, making it particularly labor intensive and subject to errors introduced during the manual preparation of the antibiotic dilutions. As a result, this method is not used for routine antimicrobial susceptibility testing in clinical microbiology laboratories.</p><p class="headingAnchor" id="H568549474"><span class="h4">Broth microdilution</span><span class="headingEndMark"> — </span>The broth microdilution method is a popular method for antimicrobial susceptibility testing in clinical microbiology laboratories. </p><p>It is considered a miniaturized and more automated version of the macrodilution method (see <a class="local">'Broth (tube) macrodilution'</a> above). It employs the same principles as the macrodilution procedure, but the antibiotic dilutions are prepared in small volumes of broth medium (usually 0.1 mL per well) in 96-well plates (<a class="graphic graphic_picture graphicRef78681" href="/z/d/graphic/78681.html" rel="external">picture 2</a>) [<a href="#rid9">9,13</a>]. Panels containing dilutions of commonly used antibiotics are commercially available (frozen or lyophilized), which saves labor and reduces the risk of error associated with preparing dilutions within the clinical laboratory. A standardized inoculum of bacteria (approximately 5 x 10<sup>5</sup> CFU/mL) is added into each well and the plates are incubated for 16 to 20 hours. After incubation, the wells can be examined either manually or through an automated tray reader for the presence of turbidity. As with the macrodilution method, the concentration of antibiotic that inhibits bacterial growth is defined as the MIC. (See <a class="local">'Interpretation of results'</a> below.)</p><p>Like agar dilution, broth microdilution testing is considered a reference method [<a href="#rid1">1</a>]. The results of broth microdilution testing are generally reproducible, and the 96-well plate format facilitates the concurrent testing of multiple antimicrobials within a relatively small footprint. Test panels are commercially available, and some such panels allow for simultaneous organism identification. As an additional advantage, broth microdilution can also be used for the susceptibility testing of many fastidious bacteria [<a href="#rid7">7</a>]. </p><p>Disadvantages include its high cost relative to some other methods, such as disk diffusion. Additionally, when using commercial test panels, the included antibiotics may not be well matched to an institution's formulary. Furthermore, because of the miniaturization of the microdilution method, one logarithm fewer bacteria are analyzed than with the macrodilution method; therefore, resistance mechanisms present in only a small subset of a bacterial population might not be represented within the tested inoculum, and the organism might falsely appear to be uniformly susceptible to a drug. (See <a class="local">'Heteroresistance'</a> above.)</p><p class="headingAnchor" id="H3576844999"><span class="h3">Antimicrobial gradient method</span><span class="headingEndMark"> — </span>The antimicrobial gradient method is an agar-based technique that relies on the creation of a concentration gradient of an antimicrobial to determine the susceptibility of a bacterial isolate to the antimicrobial (<a class="graphic graphic_picture graphicRef59934" href="/z/d/graphic/59934.html" rel="external">picture 3</a>). </p><p>As with the disk diffusion method, the gradient procedure involves the preparation of a standardized bacterial suspension containing approximately 1 to 2 x 10<sup>8 </sup>CFU/mL organisms and streaking it onto a Mueller-Hinton (or other medium appropriate for the bacterial species) agar plate. Commercially produced plastic strips, each impregnated with a graded concentration of a specific antimicrobial, are then placed on the inoculated plate. After overnight incubation, the MIC of each antimicrobial is determined by identifying the intersection of the elliptical zone of growth inhibition with the antimicrobial gradient on the strip.</p><p>Multiple reports have confirmed that results of gradient method testing correlate well with those of agar dilution and broth dilution for certain organism-antimicrobial combinations [<a href="#rid19">19-23</a>]. However, the gradient method should only be used for testing organism-antimicrobial combinations approved by the FDA or demonstrated in well-controlled studies to be equivalent in performance to CLSI reference methods, since the method is not robust for all applications; for example, the method may overstate the <a class="drug drug_general" data-topicid="10037" href="/z/d/drug information/10037.html" rel="external">vancomycin</a> MIC for <em>S. aureus</em> by one or more doubling dilutions.</p><p>One strength of the gradient technique is that it can be used to test fastidious organisms with special growth requirements [<a href="#rid13">13,14</a>]. Because each antimicrobial is tested individually (and not as part of a predetermined panel), the gradient method also allows for flexibility in the selection of which antimicrobials to test for a specific isolate. </p><p>Important limitations of the gradient method are its cost and lack of automation, both of which preclude its use for testing numerous antimicrobials against routine isolates.</p><p class="headingAnchor" id="H556591212"><span class="h2">Interpretation of results</span><span class="headingEndMark"> — </span>Conventional qualitative disk diffusion and quantitative susceptibility testing methods yield an inhibitory zone diameter and an MIC, respectively, for the given isolate-antibiotic pair. Breakpoints published by the CLSI, EUCAST, or, in the United States, the FDA, are used to interpret the zone sizes or MIC values and classify them into interpretive categories [<a href="#rid24">24,25</a>]. The categories used by CLSI are: </p><p class="bulletIndent1"><span class="glyph">●</span>Susceptible – Indicates that the antibiotic concentration that inhibits the isolate's growth is usually achieved with administration of the dose recommended for the type of infection and infecting organism. Clinical efficacy is expected.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Susceptible-dose dependent – Indicates that, to achieve the antibiotic concentration that inhibits the isolate’s growth, it is necessary to use a dosing regimen that results in higher drug exposure (by means of higher doses, more frequent doses, or both) than that achieved with the regimen used to establish the susceptible breakpoint. Like the intermediate category below, the susceptible-dose-dependent category includes a buffer zone to avoid major discrepancies in interpretation based upon small, uncontrolled technical factors, especially for drugs with a narrow margin between effective and toxic doses.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Intermediate – Indicates that the MIC of the antimicrobial approaches usually attainable blood and tissue levels and/or that response rates may be reduced compared with susceptible isolates. Clinical efficacy may be achieved when the antimicrobial is physiologically concentrated at the site of infection (eg, beta-lactams, aminoglycosides, and fluoroquinolones in the urine). The CLSI also defines the intermediate category to include a buffer zone to avoid major discrepancies in interpretation based upon small, uncontrolled technical factors, especially for drugs with a narrow margin between effective and toxic doses.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Resistant – Indicates that the concentrations usually achieved when the antibiotic is administered at conventional doses do not inhibit the isolate's growth, that the MIC or disk diffusion zone diameter falls into a range where specific microbial resistance mechanisms are likely, or that reliable clinical efficacy of the antimicrobial against the organism has not been established in clinical studies.</p><p></p><p>For commonly encountered bacteria, establishment of breakpoints involves extensive review and consideration of microbiologic, clinical, and pharmacodynamic data. For uncommonly encountered bacteria, breakpoints are largely informed by available wild-type MIC distributions. For each organism or group of organisms, CLSI also provides supplemental information regarding known resistance mechanisms or patterns, indications for antimicrobial susceptibility testing, how the interpretive criteria were derived, and special testing notes.</p><p class="headingAnchor" id="H3794159039"><span class="h1">AUTOMATED METHODS</span><span class="headingEndMark"> — </span>Most automated systems depend on the optical detection of bacterial growth in the presence of a specific antimicrobial. Because these optical systems can detect more subtle changes in growth than would be visible to the human eye, they can determine antimicrobial susceptibility patterns more rapidly than conventional methods [<a href="#rid13">13</a>]. </p><p>Multiple such automated systems are available; all use similar techniques and involve inoculation of antimicrobial microdilution trays or small cards with known quantities of bacteria, followed by incubation in an instrument that monitors growth in each well using photometric, fluorometric, or turbidimetric measurements (<a class="graphic graphic_picture graphicRef79132" href="/z/d/graphic/79132.html" rel="external">picture 4</a>) [<a href="#rid26">26</a>]. Using these automated systems, multiple antimicrobials can be tested against a specific isolate at a time, and many samples can be processed simultaneously.</p><p>The software of automated systems also offers certain advantages. It can include interpretive rules (which can be customized in some cases) to facilitate detection of unusual or important susceptibility patterns that warrant particular action, such as confirmation before reporting or expedited notification of clinicians or infection control personnel. Additionally, these automated systems can interface with the electronic medical record to quickly make results available to clinicians, which has been associated with improvements in clinical outcomes [<a href="#rid27">27,28</a>].</p><p>The principal limitation of automated antimicrobial susceptibility testing systems is cost, which is prohibitive for some clinical microbiology laboratories. Earlier automated systems also had difficulty detecting inducible antimicrobial resistance patterns, such as those seen with certain beta-lactamases; newer versions, however, have improved the ability to detect these specific resistance mechanisms, making them reliable tools for routine antimicrobial susceptibility testing of many commonly isolated bacteria [<a href="#rid13">13</a>]. There can be a lag between the publication of new interpretive criteria by the Clinical and Laboratory Standards Institute (CLSI) or the Food and Drug Administration (FDA) and the incorporation of the new breakpoints into the database of an automated system. In this setting, clinical microbiology laboratories must be aware of these discrepancies and perform validation studies to assure adequate performance of the automated system with the revised breakpoints before adopting them.</p><p>In the United States, FDA clearance of an automated system indicates that it provides results that are substantially equivalent to those generated using appropriate reference methods (eg, broth microdilution, agar dilution) for the organisms and antimicrobial agents claimed in the manufacturer's package insert [<a href="#rid1">1</a>].</p><p class="headingAnchor" id="H82584653"><span class="h1">METHODS TO DETECT SPECIFIC TYPES OF RESISTANCE</span><span class="headingEndMark"> — </span>Treatment regimens can often be devised on the basis of qualitative or quantitative antimicrobial susceptibility testing results for a given clinical isolate without knowledge of the specific acquired resistance mechanism or mechanisms present. Determination of specific resistance mechanisms (eg, extended-spectrum beta-lactamase production among Enterobacterales) is more commonly useful for infection control and epidemiologic purposes. However, there are some situations in which knowing that an isolate expresses a specific resistance mechanism can be clinically valuable.</p><p class="headingAnchor" id="H4153378657"><span class="h2">Beta-lactamase testing</span><span class="headingEndMark"> — </span>The beta-lactamases produced by many commonly encountered bacteria, including <em>Staphylococcus </em>spp, <em>Haemophilus influenzae</em>, <em>N. gonorrhoeae</em>, and <em>Enterococcus </em>spp, can be detected in the clinical microbiology laboratory. These enzymes confer resistance to penicillin and <a class="drug drug_general" data-topicid="8737" href="/z/d/drug information/8737.html" rel="external">ampicillin</a> through hydrolysis of the beta-lactam ring<em>.</em> In many cases, this enzymatic hydrolysis can be ascertained within minutes from colony growth (in contrast to the overnight incubation required by growth-dependent disk diffusion or minimum inhibitory concentration [MIC] tests), providing rapid information that can guide antimicrobial selection. In other cases, supplemental beta-lactamase testing should be performed to confirm the results of routine susceptibility testing before reporting results; for example, <em>Staphylococcus </em>spp<em> </em>that initially test susceptible to penicillin should only be reported as penicillin-susceptible if the results of beta-lactamase testing are negative [<a href="#rid1">1</a>].</p><p>The chromogenic cephalosporin beta-lactamase method can be used to detect beta-lactamases in staphylococci, <em>H. influenzae</em>, <em>N. gonorrhoeae</em>, <em>Enterococcus</em> spp, and other bacteria. It is performed by placing a small amount of test organism onto a commercially prepared paper disk embedded with nitrocefin, a chromogenic cephalosporin. If the test organism produces a beta-lactamase that hydrolyzes nitrocefin's beta-lactam ring, a pink color is produced (<a class="graphic graphic_picture graphicRef114730" href="/z/d/graphic/114730.html" rel="external">picture 5</a>) [<a href="#rid1">1</a>]. Some organisms, such as <em>H. influenzae</em> and <em>N. gonorrhoeae</em>, produce beta-lactamase constitutively. In contrast, staphylococci may produce detectable levels of beta-lactamase only after exposure to an inducing agent; therefore, the inoculum of staphylococci used for the chromogenic cephalosporin beta-lactamase test should be taken from growth in the zone margin surrounding a penicillin or <a class="drug drug_general" data-topicid="9223" href="/z/d/drug information/9223.html" rel="external">cefoxitin</a> disk. </p><p>For <em>S. aureus</em>, the penicillin zone edge test, or the "beach and cliff" test, can also be used to detect beta-lactamases [<a href="#rid1">1</a>]. In this procedure, a standard inoculum of the organism is plated onto a Mueller-Hinton agar plate. A commercially available penicillin disk is added to the agar plate, which is then incubated for 16 to 18 hours. After incubation, the zone edge of inhibition of growth around the penicillin disk is examined. A sharp zone edge (a "cliff") indicates the presence of a beta-lactamase, while a fuzzy zone edge (a "beach") indicates the absence of a beta-lactamase (<a class="graphic graphic_picture graphicRef114731" href="/z/d/graphic/114731.html" rel="external">picture 6</a>). The penicillin zone edge test is more sensitive than the nitrocefin-based method for the detection of beta-lactamase production by <em>S. aureus </em>[<a href="#rid29">29,30</a>]; as such, a negative nitrocefin-based beta-lactamase test must be confirmed with a penicillin zone edge test [<a href="#rid1">1</a>].</p><p class="headingAnchor" id="H3812729230"><span class="h2">Inducible clindamycin resistance testing</span><span class="headingEndMark"> — </span>The Clinical and Laboratory Standards Institute (CLSI) recommends that testing for inducible <a class="drug drug_general" data-topicid="9275" href="/z/d/drug information/9275.html" rel="external">clindamycin</a> resistance be performed on all <em>Staphylococcus </em>spp and<em> Streptococcus pneumoniae</em> isolates<em> </em>and for beta-hemolytic streptococci whenever antimicrobial susceptibility testing is clinically indicated (eg, group B streptococci isolated from recto-vaginal specimens obtained from pregnant women allergic to penicillin).</p><p>The genetic underpinning of this resistance mechanism is the <em>erm </em>gene, which mediates the methylation of 23S ribosomal ribonucleic acid (RNA), thereby modifying the binding target site for macrolides and <a class="drug drug_general" data-topicid="9275" href="/z/d/drug information/9275.html" rel="external">clindamycin</a> [<a href="#rid31">31</a>]. Clindamycin alone is a poor inducer of the methylase; as such, bacterial isolates can appear falsely susceptible to clindamycin when conventional antimicrobial susceptibility testing methods are used. The addition of <a class="drug drug_general" data-topicid="9418" href="/z/d/drug information/9418.html" rel="external">erythromycin</a>, however, allows for the induction of this resistance mechanism and accurate identification of the presence of the <em>erm</em> gene in <em>Staphylococcus </em>spp,<em> S. pneumoniae</em>,<em> </em>and beta-hemolytic streptococci [<a href="#rid1">1,31-34</a>]. </p><p>Standardized inducible <a class="drug drug_general" data-topicid="9275" href="/z/d/drug information/9275.html" rel="external">clindamycin</a> resistance testing can be performed using either the D-zone test (a disk diffusion method) or broth microdilution [<a href="#rid1">1</a>]. For the D-zone test, commercially available erythromycin- and clindamycin-impregnated disks are placed on an agar plate inoculated with a standard inoculum of the organism of interest; the distance between the disks is specified according to the organism. The plates are incubated for 16 to 24 hours and then visually inspected for the presence of a flattening of the clindamycin zone of inhibition adjacent to the <a class="drug drug_general" data-topicid="9418" href="/z/d/drug information/9418.html" rel="external">erythromycin</a> disk (D-zone) (<a class="graphic graphic_picture graphicRef114732" href="/z/d/graphic/114732.html" rel="external">picture 7</a>). An isolate with a positive D-zone should be reported as clindamycin-resistant. </p><p>To test for inducible <a class="drug drug_general" data-topicid="9275" href="/z/d/drug information/9275.html" rel="external">clindamycin</a> resistance using the broth microdilution method, specific concentrations of <a class="drug drug_general" data-topicid="9418" href="/z/d/drug information/9418.html" rel="external">erythromycin</a> and clindamycin are included in the same well, which is inoculated with a bacterial suspension in the usual manner for broth microdilution testing. After overnight incubation, the well is examined for bacterial growth; any visible growth indicates inducible clindamycin resistance. Manufacturers of automated antimicrobial susceptibility testing systems have integrated this test into their systems such that inducible clindamycin resistance can be determined concurrently with other antimicrobial susceptibility testing results.</p><p class="headingAnchor" id="H464547418"><span class="h2">High-level aminoglycoside resistance screening</span><span class="headingEndMark"> — </span>CLSI recommends screening for high-level aminoglycoside resistance for severe enterococcal infections, such as endocarditis [<a href="#rid1">1</a>]. Antimicrobial treatment of enterococcal endocarditis usually consists of a combination of a cell wall-active agent and an aminoglycoside, if the isolate is susceptible [<a href="#rid17">17</a>]. (See  <a class="medical medical_review" href="/z/d/html/2152.html" rel="external">"Antimicrobial therapy of left-sided native valve endocarditis", section on 'Enterococci'</a> and  <a class="medical medical_review" href="/z/d/html/2141.html" rel="external">"Antimicrobial therapy of prosthetic valve endocarditis", section on 'Enterococci'</a>.)</p><p>Disk diffusion, broth microdilution, and agar dilution methods can be used for this purpose [<a href="#rid1">1</a>]. All three methods have been shown to have good sensitivity and specificity for the detection of high-level aminoglycoside resistance when standard methods are applied [<a href="#rid35">35</a>]. Inconclusive results, however, can sometimes be seen with the disk diffusion method, requiring resolution with either broth microdilution or agar dilution [<a href="#rid1">1,35</a>]. </p><p class="bulletIndent1"><span class="glyph">●</span>The disk diffusion test is performed on Mueller-Hinton agar and follows the standard procedure (see <a class="local">'Performance'</a> above), but the disks are impregnated with high concentrations of <a class="drug drug_general" data-topicid="8489" href="/z/d/drug information/8489.html" rel="external">gentamicin</a> or <a class="drug drug_general" data-topicid="10238" href="/z/d/drug information/10238.html" rel="external">streptomycin</a> (120 mcg of gentamicin or 300 mcg of streptomycin). The plates are manually examined after 16 to 18 hours of incubation; zones of inhibition around each disk are measured and interpreted according to qualitative breakpoints. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>For the broth microdilution procedure, a standardized inoculum of the enterococcal isolate is added to a well containing brain-heart infusion broth and a high concentration of either <a class="drug drug_general" data-topicid="8489" href="/z/d/drug information/8489.html" rel="external">gentamicin</a> (500 mcg/mL) or <a class="drug drug_general" data-topicid="10238" href="/z/d/drug information/10238.html" rel="external">streptomycin</a> (1000 mcg/mL). The well is examined after 24 hours of incubation, and any visible growth is indicative of high-level resistance to the aminoglycoside. For streptomycin, if no growth is observed after 24 hours, a second read is performed after an additional 24 hours of incubation. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The agar dilution technique involves spotting a standardized inoculum of the bacterial isolate onto a brain-heart infusion agar plate containing a high concentration of either <a class="drug drug_general" data-topicid="8489" href="/z/d/drug information/8489.html" rel="external">gentamicin</a> (500 mcg/mL) or <a class="drug drug_general" data-topicid="10238" href="/z/d/drug information/10238.html" rel="external">streptomycin</a> (2000 mcg/mL); as with the broth microdilution method, growth (&gt;1 colony) after 24 hours of incubation (or 48 hours for streptomycin) indicates resistance. </p><p></p><p class="headingAnchor" id="H3216921140"><span class="h1">GENOTYPIC METHODS</span><span class="headingEndMark"> — </span>Although phenotypic methods remain the cornerstone of antimicrobial susceptibility testing in clinical microbiology laboratories, molecular assays that test for specific resistance genes have been increasingly incorporated into routine use. Techniques commonly used to detect bacterial nucleic acid sequences conferring antibiotic resistance include polymerase chain reaction (PCR) and deoxyribonucleic acid (DNA) hybridization. A number of such genotypic assays have been cleared by the United States Food and Drug Administration (FDA) [<a href="#rid36">36</a>], in some cases as screening tools for the identification of multidrug-resistant bacteria in hospital settings, and in others for diagnostic purposes [<a href="#rid37">37-40</a>]. A list of assays can be found on the <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.fda.gov%2Fmedical-devices%2Fin-vitro-diagnostics%2Fnucleic-acid-based-tests&amp;token=3cXBBB%2Fn7kKy4%2BeYBK93kquZqAxFJoPcszEA%2F1%2Bw%2BHFe0Jheb%2FvifY0i6%2Bx1kFKwKTjpXI3OkefTFe1YXzvxJ7AxNnLS6USxzdK%2FR4jCTjI%3D&amp;TOPIC_ID=467" target="_blank">FDA website</a>.</p><p>Genotypic antimicrobial susceptibility testing methods have several important strengths. Some molecular assays can be performed directly on clinical specimens or primary cultures, in contrast to conventional antimicrobial susceptibility testing methods that require initial bacterial growth, subculture, and pure colony isolation, resulting in substantially faster turnaround times. Furthermore, some molecular assays are more automated, objective, and simple to perform than conventional methods, which can reduce both labor and errors. Additionally, when a given phenotypic resistance pattern can occur as a consequence of more than one genetic determinant, molecular tests can provide more specific information about the presence of particular resistance genes that are of institutional epidemiological or infection control concern. In other cases, a genotypic test may provide potentially useful information about resistance difficult to detect using standard phenotypic approaches (eg, in the case of poor growth masking higher minimum inhibitory concentrations, low in vitro expression, or heteroresistance). </p><p>Molecular antimicrobial susceptibility testing methods also have important limitations that have thus far restricted their clinical application:</p><p class="bulletIndent1"><span class="glyph">●</span>Available FDA-cleared assays are designed to identify one or a few specific genetic resistance targets in a given bacterial species. This can be sufficient when antimicrobial resistance is mediated by one or two specific genes, as with the <em>mecA </em>gene, which regulates the vast majority of methicillin resistance in human isolates of <em>S. aureus</em>. For many other bacteria, however, the genetic underpinnings of antimicrobial resistance are much more complex, and reliance on molecular assays that target only a few known resistance genes to predict antimicrobial susceptibility would be misleading. This caveat is especially relevant among members of the family Enterobacterales,<em> </em>in which hundreds of resistance mechanisms have been described. In addition, emergence of genetic variants may interfere with detection of specific genetic resistance targets or lead to discrepancies between genotypic and phenotypic susceptibility results [<a href="#rid41">41</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genotypic results do not obviate the need for phenotypic antimicrobial susceptibility testing. While FDA-cleared molecular assays have good sensitivity and specificity for the detection of specific genetic resistance elements, phenotypic testing is still required to confirm those results and to provide information about other possible therapeutic options. As an example, a clinician treating a patient with an infection caused by methicillin-resistant <em>S. aureus</em> may not only need to know that the isolate is resistant to methicillin, but also whether the isolate is susceptible to other antimicrobials. Because molecular testing is performed in addition to (rather than in replacement of) phenotypic antimicrobial susceptibility testing, its use has the potential to increase both costs and labor in clinical microbiology laboratories. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The detection of a specific genetic resistance target is not always associated with the expression of this gene <em>in vivo</em>. In some circumstances, characterizing bacteria as resistant based on unexpressed genetic potential may result in over-reporting of resistance. </p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The majority of genotypic tests are significantly more expensive than phenotypic antimicrobial susceptibility tests and do not allow for high-throughput testing, both of which are important considerations in laboratories responsible for a large volume of clinical testing.</p><p></p><p>The Clinical and Laboratory Standards Institute (CLSI) provides a practical approach for how laboratories can use and report the results of molecular tests for detection of antibiotic resistance, including guidance for how to proceed when phenotypic and genotypic results are discordant [<a href="#rid1">1</a>].</p><p class="headingAnchor" id="H3628488622"><span class="h1">REPORTING SUSCEPTIBILITY RESULTS</span><span class="headingEndMark"> — </span>The selection of antimicrobials to use for susceptibility testing on a given isolate depends on several factors, including the type of organism, the site of infection, and the institutional formulary. </p><p>The Clinical and Laboratory Standards Institute provides guidance regarding the selection of appropriate antimicrobial agents for testing commonly encountered bacteria, as well as many infrequently isolated organisms [<a href="#rid1">1,7</a>]. These recommendations are based on clinical efficacy data when available, United States Food and Drug Administration clinical indications for use, consensus recommendations, prevalence of resistance, minimizing emergence of resistance, and cost [<a href="#rid1">1</a>]. </p><p>Once antimicrobial susceptibility testing is complete, the clinical microbiology laboratory also must decide which results should be released into the clinical information system. An institutional committee comprised of microbiologists, infectious diseases physicians, and pharmacists is often responsible for selecting the number and types of antimicrobials that should be reported for an individual organism under specific circumstances. The rationale for selective reporting is to help guide antimicrobial prescribing and reduce the inappropriate use of broad-spectrum antimicrobials when more targeted agents would suffice. As an example, a laboratory might test <em>S. pyogenes</em> isolates for susceptibility to carbapenems, but it would be inappropriate to routinely prescribe such broad-spectrum agents to treat infections caused by this pathogen, and so the carbapenem susceptibility results should generally be suppressed from reporting into the clinical information system.</p><p>Results that are not routinely reported should be released to the clinicians under specific circumstances, such as in cases of unexpected resistance or multidrug resistance (in which case, the susceptibility testing results for broader spectrum or second-line agents may be needed to guide the regimen selection). Direct communication between clinicians and clinical microbiology laboratory personnel is frequently helpful in such scenarios to establish whether additional testing should be undertaken. The clinical microbiology laboratory should save isolates from cultures for at least one week after the culture is reported as final, in the event that additional testing is required.</p><p class="headingAnchor" id="H85969693"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span>Antimicrobial susceptibility testing should be performed when clinically significant bacteria are isolated from patient specimens and the resulting information can be used to guide treatment. Testing is most useful when the antimicrobial susceptibility of an isolate is not predictable based on the genus and species, the organism is not part of the normal flora of the specimen site, the results of in vitro susceptibility testing predict clinical effectiveness, and guidance for the standardized performance of susceptibility testing is available. (See <a class="local">'Indications for susceptibility testing'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The disk diffusion test, or Kirby-Bauer method, is simple, reliable, and the most commonly used qualitative testing method. A standardized inoculum of bacteria is grown on a Mueller-Hinton agar plate, upon which commercially prepared paper disks, each containing a fixed concentration of an antimicrobial, are placed. The diameters of the zones of growth inhibition around each of the disks are measured and are qualitatively interpreted based on published breakpoints (susceptible, susceptible-dose dependent, intermediate, or resistant) (<a class="graphic graphic_picture graphicRef79261" href="/z/d/graphic/79261.html" rel="external">picture 1</a>). This method cannot be used for fastidious or slow-growing bacteria. (See <a class="local">'Qualitative methods (disk diffusion/Kirby-Bauer method)'</a> above and <a class="local">'Interpretation of results'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Agar dilution tests, in which standardized inocula of bacteria are grown on agar prepared with a fixed concentration of an antimicrobial, allow for multiple samples to be tested on a single set of plates. This technique is recommended for use with fastidious bacteria that require special growth conditions, such as <em>Helicobacter pylori</em> and <em>Neisseria gonorrhoeae</em>. Because the agar dilution test is costly and labor-intensive, the majority of clinical microbiology laboratories restrict its use to certain specific applications, such as screening for high-level <a class="drug drug_general" data-topicid="8489" href="/z/d/drug information/8489.html" rel="external">gentamicin</a> resistance in enterococci. (See <a class="local">'Agar dilution'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The broth microdilution method is commonly used in clinical microbiology laboratories. A standardized inoculum of bacteria is incubated in dilutions of antimicrobials commercially prepared in 96-well plates, and the lowest concentration of antimicrobial that inhibits visible growth of bacteria is the minimum inhibitory concentration (MIC) (<a class="graphic graphic_picture graphicRef78681" href="/z/d/graphic/78681.html" rel="external">picture 2</a>). (See <a class="local">'Broth microdilution'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The antimicrobial gradient method can be useful for testing fastidious organisms that require special growing conditions. Commercially produced plastic strips with graded antimicrobial concentrations are placed upon medium inoculated with a standardized inoculum of bacteria, and MICs are determined by the intersection of the elliptically shaped zone of growth inhibition and the strip (<a class="graphic graphic_picture graphicRef59934" href="/z/d/graphic/59934.html" rel="external">picture 3</a>). (See <a class="local">'Antimicrobial gradient method'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Automated susceptibility testing provides results more rapidly than traditional laboratory methods that require overnight incubation. Clinical microbiology laboratories must be aware that there can be a lag between the publication of new interpretive criteria and the incorporation of the new breakpoints into the database of an automated system. (See <a class="local">'Automated methods'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Treatment regimens can often be devised on the basis of antimicrobial susceptibility testing results without knowledge of the specific acquired resistance mechanisms present. However, such specific identification of beta-lactamase production in staphylococci and some other organisms, and inducible <a class="drug drug_general" data-topicid="9275" href="/z/d/drug information/9275.html" rel="external">clindamycin</a> resistance in staphylococci, pneumococci, and beta-hemolytic streptococci, can be clinically valuable. (See <a class="local">'Methods to detect specific types of resistance'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The incorporation of genotypic antimicrobial susceptibility testing into routine use in clinical laboratories has thus far been limited, in large part because of the high associated cost of genotypic testing, the fact that genotypic testing does not obviate the need for phenotypic testing, and the current limitation of US Food and Drug Administration (FDA)-cleared assays to the detection of only one or a few specific genetic resistance targets. (See <a class="local">'Genotypic methods'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Multiple factors, including the type of organism, the site of infection, and the institutional formulary, should inform the selection of antimicrobials for susceptibility testing and the selective reporting of susceptibility results. (See <a class="local">'Reporting susceptibility results'</a> above.)</p><p></p><p class="headingAnchor" id="H3591178917"><span class="h1">ACKNOWLEDGMENTS</span><span class="headingEndMark"> — </span>The UpToDate editorial staff acknowledges Mary Jane Ferraro, PhD, MPH, Jatin M Vyas, MD, PhD, and Sarah E Turbett, MD, who contributed to an earlier version of this topic review.</p></div><div class="headingAnchor" id="references"><ol id="reference"><li class="breakAll">Clinical and Laboratory Standards Institute (CLSI). Performance standards for antimicrobial susceptibility testing, 31st ed, CLSI supplement M100, Wayne, Pennsylvania 2021.</li><li><a class="nounderline abstract_t">Livermore DM. Bacterial resistance: origins, epidemiology, and impact. Clin Infect Dis 2003; 36:S11.</a></li><li class="breakAll">Procop GW, Church DL, Hall GS, et al. Antimicrobial susceptibility testing. In: Koneman's Color Atlas and Textbook of Diagnostic Microbiology, 7th ed, Wolters Kluwer, Philadelphia 2017. p.1074.</li><li><a class="nounderline abstract_t">Tamma PD, Doi Y, Bonomo RA, et al. A Primer on AmpC β-Lactamases: Necessary Knowledge for an Increasingly Multidrug-resistant World. Clin Infect Dis 2019; 69:1446.</a></li><li><a class="nounderline abstract_t">Holmes NE, Johnson PD, Howden BP. Relationship between vancomycin-resistant Staphylococcus aureus, vancomycin-intermediate S. aureus, high vancomycin MIC, and outcome in serious S. aureus infections. J Clin Microbiol 2012; 50:2548.</a></li><li><a class="nounderline abstract_t">Miller JM, Binnicker MJ, Campbell S, et al. A Guide to Utilization of the Microbiology Laboratory for Diagnosis of Infectious Diseases: 2018 Update by the Infectious Diseases Society of America and the American Society for Microbiology. Clin Infect Dis 2018; 67:e1.</a></li><li class="breakAll">Clinical and Laboratory Standards Institute (CLSI). Methods for antimicrobial dilution and disk susceptibility testing of infrequently isolated or fastidious bacteria, 3rd ed, CLSI guideline M45, Wayne, Pennsylvania 2015.</li><li class="breakAll">Clinical and Laboratory Standards Institute (CLSI). Performance standards for antimicrobial disk susceptibility tests; approved standard, 13th ed, CLSI standard M02, Wayne, Pennsylvania 2018.</li><li class="breakAll">Clinical and Laboratory Standards Institute (CLSI). Methods for dilution antimicrobial susceptibility tests for bacteria that grow aerobically; approved standard, 11th ed, CLSI standard M07, Wayne, Pennsylvania 2018.</li><li class="breakAll">Clinical and Laboratory Standards Institute (CLSI). Methods for antimicrobial susceptibility testing for anaerobic bacteria; approved standard, 9th ed, CLSI standard M11, Wayne, Pennsylvania 2018.</li><li class="breakAll">European Committee on Antimicrobial Susceptibility Testing (EUCAST). www.eucast.org (Accessed on December 13, 2021).</li><li><a class="nounderline abstract_t">Swenson JM, Anderson KF, Lonsway DR, et al. Accuracy of commercial and reference susceptibility testing methods for detecting vancomycin-intermediate Staphylococcus aureus. J Clin Microbiol 2009; 47:2013.</a></li><li><a class="nounderline abstract_t">Jorgensen JH, Ferraro MJ. Antimicrobial susceptibility testing: a review of general principles and contemporary practices. Clin Infect Dis 2009; 49:1749.</a></li><li><a class="nounderline abstract_t">Jenkins SG, Schuetz AN. Current concepts in laboratory testing to guide antimicrobial therapy. Mayo Clin Proc 2012; 87:290.</a></li><li><a class="nounderline abstract_t">Ringertz S, Kronvall G. On the theory of the disk diffusion test. Evidence for a non-linear relationship between critical concentration and MIC, and its practical implications for susceptibility testing of Haemophilus influenzae. APMIS 1988; 96:484.</a></li><li><a class="nounderline abstract_t">Humphries RM, Miller L, Zimmer B, et al. Contemporary Considerations for Establishing Reference Methods for Antibacterial Susceptibility Testing. J Clin Microbiol 2023; 61:e0188622.</a></li><li><a class="nounderline abstract_t">Baddour LM, Wilson WR, Bayer AS, et al. Infective Endocarditis in Adults: Diagnosis, Antimicrobial Therapy, and Management of Complications: A Scientific Statement for Healthcare Professionals From the American Heart Association. Circulation 2015; 132:1435.</a></li><li class="breakAll">US Food and Drug Administration. FDA-recognized antimicrobial susceptibility test interpretive criteria. www.fda.gov/drugs/development-resources/fda-recognized-antimicrobial-susceptibility-test-interpretive-criteria (Accessed on December 13, 2021).</li><li><a class="nounderline abstract_t">Baker CN, Stocker SA, Culver DH, Thornsberry C. Comparison of the E Test to agar dilution, broth microdilution, and agar diffusion susceptibility testing techniques by using a special challenge set of bacteria. J Clin Microbiol 1991; 29:533.</a></li><li><a class="nounderline abstract_t">Brown DF, Brown L. Evaluation of the E test, a novel method of quantifying antimicrobial activity. J Antimicrob Chemother 1991; 27:185.</a></li><li><a class="nounderline abstract_t">Pfaller MA, Jones RN, Microbiology Resource Committee, College of American Pathologists. Performance accuracy of antibacterial and antifungal susceptibility test methods: report from the College of American Pathologists Microbiology Surveys Program (2001-2003). Arch Pathol Lab Med 2006; 130:767.</a></li><li><a class="nounderline abstract_t">Deak E, Hindler JA, Skov R, et al. Performance of Etest and disk diffusion for detection of ciprofloxacin and levofloxacin resistance in Salmonella enterica. J Clin Microbiol 2015; 53:298.</a></li><li><a class="nounderline abstract_t">Tenover FC, Baker CN, Swenson JM. Evaluation of commercial methods for determining antimicrobial susceptibility of Streptococcus pneumoniae. J Clin Microbiol 1996; 34:10.</a></li><li><a class="nounderline abstract_t">Turnidge J, Paterson DL. Setting and revising antibacterial susceptibility breakpoints. Clin Microbiol Rev 2007; 20:391.</a></li><li><a class="nounderline abstract_t">Pierce VM, Mathers AJ. Setting Antimicrobial Susceptibility Testing Breakpoints: A Primer for Pediatric Infectious Diseases Specialists on the Clinical and Laboratory Standards Institute Approach. J Pediatric Infect Dis Soc 2022; 11:73.</a></li><li class="breakAll">Karlowsky JA, Richter SS. Antimicrobial susceptibility testing systems. In: Manual of Clinical Microbiology, Carroll KC, Pfaller MA (Eds), ASM Press, Washington DC 2020. p.1300.</li><li><a class="nounderline abstract_t">Doern GV, Vautour R, Gaudet M, Levy B. Clinical impact of rapid in vitro susceptibility testing and bacterial identification. J Clin Microbiol 1994; 32:1757.</a></li><li><a class="nounderline abstract_t">Kerremans JJ, Verboom P, Stijnen T, et al. Rapid identification and antimicrobial susceptibility testing reduce antibiotic use and accelerate pathogen-directed antibiotic use. J Antimicrob Chemother 2008; 61:428.</a></li><li><a class="nounderline abstract_t">Kaase M, Lenga S, Friedrich S, et al. Comparison of phenotypic methods for penicillinase detection in Staphylococcus aureus. Clin Microbiol Infect 2008; 14:614.</a></li><li><a class="nounderline abstract_t">Papanicolas LE, Bell JM, Bastian I. Performance of phenotypic tests for detection of penicillinase in Staphylococcus aureus isolates from Australia. J Clin Microbiol 2014; 52:1136.</a></li><li><a class="nounderline abstract_t">Lewis JS 2nd, Jorgensen JH. Inducible clindamycin resistance in Staphylococci: should clinicians and microbiologists be concerned? Clin Infect Dis 2005; 40:280.</a></li><li><a class="nounderline abstract_t">Bowling JE, Owens AE, McElmeel ML, et al. Detection of inducible clindamycin resistance in beta-hemolytic streptococci by using the CLSI broth microdilution test and erythromycin-clindamycin combinations. J Clin Microbiol 2010; 48:2275.</a></li><li><a class="nounderline abstract_t">Jorgensen JH, McElmeel ML, Fulcher LC, et al. Evaluation of disk approximation and single-well broth tests for detection of inducible clindamycin resistance in Streptococcus pneumoniae. J Clin Microbiol 2011; 49:3332.</a></li><li><a class="nounderline abstract_t">Swenson JM, Brasso WB, Ferraro MJ, et al. Detection of inducible clindamycin resistance in staphylococci by broth microdilution using erythromycin-clindamycin combination wells. J Clin Microbiol 2007; 45:3954.</a></li><li><a class="nounderline abstract_t">Swenson JM, Ferraro MJ, Sahm DF, et al. Multilaboratory evaluation of screening methods for detection of high-level aminoglycoside resistance in enterococci. National Committee for Clinical Laboratory Standards Study Group on Enterococci. J Clin Microbiol 1995; 33:3008.</a></li><li class="breakAll">US Food and Drug Administration. Nucleic acid based tests. www.fda.gov/MedicalDevices/ProductsandMedicalProcedures/InVitroDiagnostics/ucm330711.htm#microbial (Accessed on December 13, 2021).</li><li><a class="nounderline abstract_t">Ledeboer NA, Hodinka RL. Molecular Detection of Resistance Determinants. Clin Microbiol 2011; 49:S20.</a></li><li class="breakAll">Mangold KA, Peterson LR. Molecular detection of Staphylococcus aureus colonization and infection. In: Molecular Microbiology, Persing D, Tenover F, Hayden R, et al. (Eds), ASM Press, Washington DC 2016. p.169.</li><li class="breakAll">McGeer AJ, Willey BM. Detection of vancomycin-resistant enterococci. In: Molecular Microbiology, Persing D, Tenover F, Hayden R, et al. (Eds), ASM Press, Washington DC 2016. p.212.</li><li class="breakAll">Sullivan KV, Diekema DJ. Overview of molecular diagnostics in multiple-drug-resistant organism prevention: focus on multiple-drug-resistant gram-negative bacterial organisms. In: Molecular Microbiology, Persing D, Tenover F, Hayden R, et al. (Eds), ASM Press, Washington DC 2016. p.197.</li><li><a class="nounderline abstract_t">Yee R, Dien Bard J, Simner PJ. The Genotype-to-Phenotype Dilemma: How Should Laboratories Approach Discordant Susceptibility Results? J Clin Microbiol 2021; 59.</a></li></ol></div><div id="topicVersionRevision">Topic 467 Version 27.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="javascript:alert('بدون لینک');" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Clinical and Laboratory Standards Institute (CLSI). Performance standards for antimicrobial susceptibility testing, 31st ed, CLSI supplement M100, Wayne, Pennsylvania 2021.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12516026" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Bacterial resistance: origins, epidemiology, and impact.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12516026" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Bacterial resistance: origins, epidemiology, and impact.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30838380" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : A Primer on AmpCβ-Lactamases: Necessary Knowledge for an Increasingly Multidrug-resistant World.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22593595" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Relationship between vancomycin-resistant Staphylococcus aureus, vancomycin-intermediate S. aureus, high vancomycin MIC, and outcome in serious S. aureus infections.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29955859" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : A Guide to Utilization of the Microbiology Laboratory for Diagnosis of Infectious Diseases: 2018 Update by the Infectious Diseases Society of America and the American Society for Microbiology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29955859" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : A Guide to Utilization of the Microbiology Laboratory for Diagnosis of Infectious Diseases: 2018 Update by the Infectious Diseases Society of America and the American Society for Microbiology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29955859" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : A Guide to Utilization of the Microbiology Laboratory for Diagnosis of Infectious Diseases: 2018 Update by the Infectious Diseases Society of America and the American Society for Microbiology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29955859" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : A Guide to Utilization of the Microbiology Laboratory for Diagnosis of Infectious Diseases: 2018 Update by the Infectious Diseases Society of America and the American Society for Microbiology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29955859" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : A Guide to Utilization of the Microbiology Laboratory for Diagnosis of Infectious Diseases: 2018 Update by the Infectious Diseases Society of America and the American Society for Microbiology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29955859" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : A Guide to Utilization of the Microbiology Laboratory for Diagnosis of Infectious Diseases: 2018 Update by the Infectious Diseases Society of America and the American Society for Microbiology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19420170" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Accuracy of commercial and reference susceptibility testing methods for detecting vancomycin-intermediate Staphylococcus aureus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19857164" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Antimicrobial susceptibility testing: a review of general principles and contemporary practices.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22386185" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Current concepts in laboratory testing to guide antimicrobial therapy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/3260786" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : On the theory of the disk diffusion test. Evidence for a non-linear relationship between critical concentration and MIC, and its practical implications for susceptibility testing of Haemophilus influenzae.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/36971571" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Contemporary Considerations for Establishing Reference Methods for Antibacterial Susceptibility Testing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26373316" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Infective Endocarditis in Adults: Diagnosis, Antimicrobial Therapy, and Management of Complications: A Scientific Statement for Healthcare Professionals From the American Heart Association.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26373316" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Infective Endocarditis in Adults: Diagnosis, Antimicrobial Therapy, and Management of Complications: A Scientific Statement for Healthcare Professionals From the American Heart Association.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2037671" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Comparison of the E Test to agar dilution, broth microdilution, and agar diffusion susceptibility testing techniques by using a special challenge set of bacteria.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2055809" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Evaluation of the E test, a novel method of quantifying antimicrobial activity.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16740026" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Performance accuracy of antibacterial and antifungal susceptibility test methods: report from the College of American Pathologists Microbiology Surveys Program (2001-2003).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25355768" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Performance of Etest and disk diffusion for detection of ciprofloxacin and levofloxacin resistance in Salmonella enterica.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8748262" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Evaluation of commercial methods for determining antimicrobial susceptibility of Streptococcus pneumoniae.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17630331" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Setting and revising antibacterial susceptibility breakpoints.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34888640" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Setting Antimicrobial Susceptibility Testing Breakpoints: A Primer for Pediatric Infectious Diseases Specialists on the Clinical and Laboratory Standards Institute Approach.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34888640" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Setting Antimicrobial Susceptibility Testing Breakpoints: A Primer for Pediatric Infectious Diseases Specialists on the Clinical and Laboratory Standards Institute Approach.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7929770" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Clinical impact of rapid in vitro susceptibility testing and bacterial identification.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18156278" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Rapid identification and antimicrobial susceptibility testing reduce antibiotic use and accelerate pathogen-directed antibiotic use.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18397333" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Comparison of phenotypic methods for penicillinase detection in Staphylococcus aureus.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24452169" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Performance of phenotypic tests for detection of penicillinase in Staphylococcus aureus isolates from Australia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15655748" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Inducible clindamycin resistance in Staphylococci: should clinicians and microbiologists be concerned?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20392918" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Detection of inducible clindamycin resistance in beta-hemolytic streptococci by using the CLSI broth microdilution test and erythromycin-clindamycin combinations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21775547" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Evaluation of disk approximation and single-well broth tests for detection of inducible clindamycin resistance in Streptococcus pneumoniae.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17942655" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Detection of inducible clindamycin resistance in staphylococci by broth microdilution using erythromycin-clindamycin combination wells.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8576363" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Multilaboratory evaluation of screening methods for detection of high-level aminoglycoside resistance in enterococci. National Committee for Clinical Laboratory Standards Study Group on Enterococci.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8576363" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Multilaboratory evaluation of screening methods for detection of high-level aminoglycoside resistance in enterococci. National Committee for Clinical Laboratory Standards Study Group on Enterococci.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Molecular Detection of Resistance Determinants</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Molecular Detection of Resistance Determinants</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : Molecular Detection of Resistance Determinants</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Molecular Detection of Resistance Determinants</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/33441396" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : The Genotype-to-Phenotype Dilemma: How Should Laboratories Approach Discordant Susceptibility Results?</p>
</a>
</div>
</div>
</div>
</div>
</section>
</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->
<!-- End Footer Area -->
<!-- Back to top -->
<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->
<!-- Prpper JS -->
<!-- Bootstrap Min JS -->
<!-- Classy Nav Min Js -->
<!-- Owl Carousel Min Js -->
<!-- Magnific Popup JS -->
<!-- CounterUp JS -->
<!-- Waypoints JS -->
<!-- Form Validator Min JS -->
<!-- Contact Form Min JS -->
<!-- Main JS -->
<!-- Global site tag (gtag.js) - Google Analytics -->
</body>
</html>
