<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Hypertrophic cardiomyopathy: Gene mutations and clinical genetic testing</title><link href="/z/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/z/d/css/icofont.min.css" rel="stylesheet"/><link href="/z/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/z/d/css/animate.css" rel="stylesheet"/><link href="/z/d/css/owl.carousel.css" rel="stylesheet"/><link href="/z/d/css/magnific-popup.css" rel="stylesheet"/><link href="/z/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/z/d/css/style.css" rel="stylesheet"/><link href="/z/d/css/rtl_edited.css" rel="stylesheet"/><link href="/z/d/css/responsive.css" rel="stylesheet"/><link href="https:///uptodate_assets/main.css" rel="stylesheet"/>
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>
<meta content="" name="description"/>
<link href="/z/d/img/favicon.png" rel="icon" type="image/png">
</link></head>
<body>
<!-- Start Main Menu Area -->
<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>
<section class="teacher-area ptb-10">
<div class="container">
</div>
<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Hypertrophic cardiomyopathy: Gene mutations and clinical genetic testing</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Hypertrophic cardiomyopathy: Gene mutations and clinical genetic testing</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Author:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Martin S Maron, MD</a></dd><dt><span> </span>Section Editors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Benjamin A Raby, MD, MPH</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">William J McKenna, MD</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Todd F Dardas, MD, MS</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Nov 04, 2022.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Hypertrophic cardiomyopathy (HCM) is a genetically determined heart muscle disease most often (60 to 70 percent) caused by mutations in one of several sarcomere genes that encode components of the contractile apparatus.</p><p>HCM is characterized by left ventricular hypertrophy (LVH) of various morphologies, with a wide array of clinical manifestations and hemodynamic abnormalities  (<a class="graphic graphic_figure graphicRef58156" href="/z/d/graphic/58156.html" rel="external">figure 1</a>). Depending in part upon the site and extent of cardiac hypertrophy, patients with HCM can develop one or more of the following abnormalities:</p><p class="bulletIndent1"><span class="glyph">●</span>LV outflow obstruction</p><p class="bulletIndent1"><span class="glyph">●</span>Diastolic dysfunction</p><p class="bulletIndent1"><span class="glyph">●</span>Myocardial ischemia</p><p class="bulletIndent1"><span class="glyph">●</span>Mitral regurgitation</p><p></p><p>These structural and functional abnormalities can produce a variety of symptoms, including:</p><p class="bulletIndent1"><span class="glyph">●</span>Fatigue</p><p class="bulletIndent1"><span class="glyph">●</span>Dyspnea</p><p class="bulletIndent1"><span class="glyph">●</span>Chest pain</p><p class="bulletIndent1"><span class="glyph">●</span>Palpitations</p><p class="bulletIndent1"><span class="glyph">●</span>Presyncope or syncope</p><p></p><p>In broad terms, the symptoms related to HCM can be categorized as those related to heart failure (HF), chest pain, or arrhythmias. Patients with HCM have an increased incidence of both supraventricular and ventricular arrhythmias. A very small subgroup of predominantly younger HCM patients is at an increased risk for sudden cardiac death (SCD) due to ventricular tachyarrhythmias. (See  <a class="medical medical_review" href="/z/d/html/4926.html" rel="external">"Hypertrophic cardiomyopathy in adults: Supraventricular tachycardias including atrial fibrillation"</a> and  <a class="medical medical_review" href="/z/d/html/4952.html" rel="external">"Hypertrophic cardiomyopathy: Risk stratification for sudden cardiac death"</a>.)</p><p>Phenotypic development with LVH typically begins in puberty, and usually by early adulthood increases in wall thickness are complete with little change over time. For the majority of patients with HCM, clinical course is relatively benign. However, a subset of patients with LV outflow tract (LVOT) obstruction will develop progressive HF symptoms requiring invasive septal reduction therapy for relief of outflow gradients, while an even smaller number of patients without LVOT obstruction will progress to an end-stage form of the disease that is characterized by LV dilation and wall thinning and systolic dysfunction. Such patients are managed according to the standard approach to patients with HF due to systolic dysfunction. (See  <a class="medical medical_review" href="/z/d/html/4920.html" rel="external">"Hypertrophic cardiomyopathy: Management of patients with outflow tract obstruction"</a> and  <a class="medical medical_review" href="/z/d/html/121085.html" rel="external">"Overview of the management of heart failure with reduced ejection fraction in adults"</a>.)</p><p>The clinical diagnosis of HCM is based upon the finding of unexplained LVH, documented by echocardiography or magnetic resonance imaging, which can be focal or diffuse and most often asymmetric. LVH and other aspects of HCM phenotypic expression are secondary to consequences of sarcomere dysfunction [<a href="#rid1">1,2</a>]. In addition, given the availability of genetic testing, a greater number of HCM family members are now being identified as carrying a disease-causing mutation in the absence of LVH (ie, genotype positive/phenotype negative).</p><p>The genetics of HCM will be reviewed here, including a discussion of the role of genetic testing and screening. Other issues such as the clinical manifestations, diagnosis, and evaluation; natural history; and treatment of this disorder are discussed separately. (See  <a class="medical medical_review" href="/z/d/html/4948.html" rel="external">"Hypertrophic cardiomyopathy: Clinical manifestations, diagnosis, and evaluation"</a> and  <a class="medical medical_review" href="/z/d/html/4916.html" rel="external">"Hypertrophic cardiomyopathy: Natural history and prognosis"</a> and  <a class="medical medical_review" href="/z/d/html/4920.html" rel="external">"Hypertrophic cardiomyopathy: Management of patients with outflow tract obstruction"</a> and  <a class="medical medical_review" href="/z/d/html/4950.html" rel="external">"Hypertrophic cardiomyopathy: Management of patients without outflow tract obstruction"</a>.)</p><p class="headingAnchor" id="H22623272"><span class="h1">GENETIC TESTING</span><span class="headingEndMark"> — </span>Genetic testing for HCM is available as a clinical test. A number of institutional and commercial laboratories offer comprehensive genetic testing on a fee-for-service basis with panels that incorporate the most common mutations responsible for HCM. The development of DNA-based testing of patients with HCM can aid in diagnosis and management of patients, and permit cascade screening of families. However, genetic testing for HCM is not as straightforward as it might at first appear, as a number of issues, particularly related to the interpretation of findings, limit the usefulness of this test [<a href="#rid3">3,4</a>]. (See <a class="local">'Pathogenic mutations and variants of uncertain significance'</a> below.)</p><p>Prior to initiating genetic testing in family members, relatives of an affected individual should be clinically evaluated with history/physical examination, electrocardiography (ECG), and echocardiography in order to identify clinical evidence of HCM. If genetic testing identifies a pathogenic mutation in the proband, then mutation-specific genetic testing should be considered for all family members who have no clinical evidence for HCM [<a href="#rid3">3,4</a>]. (See  <a class="medical medical_review" href="/z/d/html/4948.html" rel="external">"Hypertrophic cardiomyopathy: Clinical manifestations, diagnosis, and evaluation", section on 'Screening of first-degree relatives'</a>.)</p><p class="headingAnchor" id="H22623485"><span class="h1">MUTATIONS IN SARCOMERIC PROTEIN GENES</span><span class="headingEndMark"> — </span>HCM is inherited in an autosomal dominant Mendelian pattern with variable expressivity and age-related penetrance. Variants in genes encoding thick or thin myofilament proteins of the cardiac sarcomere (or sarcomere-related structures) are frequently the cause of HCM; there are over 1500 mutations among these genes  (<a class="graphic graphic_figure graphicRef90769" href="/z/d/graphic/90769.html" rel="external">figure 2</a>) [<a href="#rid5">5</a>]. In an analysis of 33 genes frequently included in HCM testing panels, only eight genes were identified as definitively causative, three additional genes showed moderate evidence of causation, and 22 of 33 genes (67 percent) frequently included in HCM gene testing panels showed minimal or no evidence of association with HCM [<a href="#rid6">6</a>].</p><p class="headingAnchor" id="H22623491"><span class="h2">Types of mutations</span><span class="headingEndMark"> — </span>The vast majority of disease-causing mutations in HCM are missense, in which a single normal amino acid is replaced for another. This type of amino acid switch directly alters the fundamental functional properties of the protein. Other common mutations are frameshift, which produce a shortened truncated protein due to the insertion or deletion of one or more nucleic acids.</p><p class="headingAnchor" id="H22623355"><span class="h2">Genes</span><span class="headingEndMark"> — </span>Most of the genetic loci in familial HCM encode one of the myocardial contractile proteins of the cardiac sarcomere, including the following eight genes with definitive evidence supporting them as disease-causing:</p><p class="bulletIndent1"><span class="glyph">●</span>Cardiac troponin T (TNNT2 gene)</p><p class="bulletIndent1"><span class="glyph">●</span>Cardiac troponin I (TNNI3 gene)</p><p class="bulletIndent1"><span class="glyph">●</span>Myosin regulatory light chain (MYL2 gene)</p><p class="bulletIndent1"><span class="glyph">●</span>Myosin essential light chain (MYL3 gene)</p><p class="bulletIndent1"><span class="glyph">●</span>Cardiac myosin binding protein-C (MYBPC3 gene)</p><p class="bulletIndent1"><span class="glyph">●</span>Cardiac beta-myosin heavy chain (MYH7 gene)</p><p class="bulletIndent1"><span class="glyph">●</span>Alpha-cardiac actin (ACTC1 gene)</p><p class="bulletIndent1"><span class="glyph">●</span>Tropomyosin 1 (TPM1 gene)</p><p></p><p>In addition, three other genes have accumulated moderate evidence supporting HCM pathogenicity, including:</p><p class="bulletIndent1"><span class="glyph">●</span>Cardiac troponin C (TNNC1 gene)</p><p class="bulletIndent1"><span class="glyph">●</span>Junctophilin 2 (JPH2)</p><p class="bulletIndent1"><span class="glyph">●</span>Cysteine and glycine rich protein 3 (CSRP3)</p><p></p><p>Some patients with sporadic disease, in which the parents are clinically unaffected, have similar genetic abnormalities as those with familial disease. De novo mutations in cardiac myosin binding protein-C, cardiac beta-myosin heavy chain, cardiac troponin T, and alpha-tropomyosin genes have been found in isolated case reports of individuals with sporadic HCM [<a href="#rid7">7-10</a>].</p><p>A few genes that do not encode contractile proteins have been implicated in other diseases which can also manifest left ventricular hypertrophy (LVH). These include metabolic storage cardiomyopathies, which are diseases considered "phenocopies" of sarcomere HCM since the pattern and extent of LVH is similar to that of HCM. These diseases include:</p><p class="bulletIndent1"><span class="glyph">●</span>The GLA gene for Fabry disease which encodes for the enzyme alpha-galactosidase [<a href="#rid11">11,12</a>]. (See  <a class="medical medical_review" href="/z/d/html/4914.html" rel="external">"Fabry disease: Cardiovascular disease"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genes involved in the RAS MAP kinase pathway cause Noonan syndrome associated with LVH [<a href="#rid13">13,14</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The gene for muscle LIM protein (MLP), which is a regulator of myogenic differentiation and contributes to the linking of the contractile apparatus with the sarcolemma [<a href="#rid15">15</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The genes encoding the gamma-2 regulatory subunit of adenosine monophosphate-activated protein kinase (PRKAG2), an enzyme that modulates glucose uptake and glycolysis, and the gene encoding lysosome-associated membrane protein 2 (LAMP2), which may be involved in lysosomal enzyme targeting, autophagy, and lysosomal biogenesis [<a href="#rid16">16</a>]. (See <a class="local">'PRKAG2 and LAMP2 genes'</a> below.)</p><p></p><p class="headingAnchor" id="H112767659"><span class="h2">Pathogenic mutations and variants of uncertain significance</span><span class="headingEndMark"> — </span>Determining the pathogenicity of a mutation is often done on a probabilistic basis, rather than as a binary yes or no [<a href="#rid17">17</a>]. A number of points highlight this:</p><p class="bulletIndent1"><span class="glyph">●</span>A number of criteria have been established to determine if a mutation identified in an HCM proband is pathogenic (ie, causing disease) [<a href="#rid18">18</a>]. This would include the presence of one or more of the following:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The mutation cosegregates with the HCM phenotype in family members.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The mutation has previously been reported as a cause of HCM.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The mutation is absent from controls without HCM.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The mutation causes important alterations in protein structure that could reasonably cause disease (ie, frameshift mutations).</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>The identified amino acid change resulting from the mutation occurs in a region of the protein considered to be conservative and not one in which there has been variation over time.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Amino acid substitutions may be identified with genetic testing in which the relevance for causing disease remains uncertain, even after applying the above-mentioned criteria. These mutations are classified as variants of uncertain significance (VUS). Since it is not clear that a VUS is actually responsible for disease, these mutations cannot be used to make clinical decisions with respect to family screening or diagnosis [<a href="#rid19">19</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>A substantial proportion of patients with HCM will have a VUS identified using current genetic testing panels. Therefore, the difficulty in differentiating pathogenic mutations from VUS represents a major limitation to using genetic testing in HCM for clinical applications [<a href="#rid19">19</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The difficulty in reliably determining if gene variants are disease-causing remains a challenge. Using a publicly available exome database, investigators identified certain gene variants that were common among Black Americans [<a href="#rid20">20</a>]. These specific variants were previously evaluated for pathogenicity by comparing their frequency among cases with control populations. As a result, these variants were misclassified as pathogenic, resulting in false positive genetic testing results. Indeed, the authors demonstrated that with including even a small number of Black Americans in the control cohort, these variants would have been correctly characterized as benign, preventing the misclassification error. This investigation has underscored the need to determine variant pathogenicity in HCM (and other inheritable heart diseases) by incorporating control populations that include ancestry-matched controls.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Depending on the population, 30 to 70 percent of patients with a clinical diagnosis of HCM will have no sarcomere mutation identified [<a href="#rid17">17</a>]. Evidence that nonsarcomeric genes may be responsible for HCM in these patients has not been substantiated [<a href="#rid21">21</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Mutation status can even change over time. For example, a VUS can be reassigned to pathogenic if information arises which provides greater support for the mutation as disease-causing [<a href="#rid19">19</a>].</p><p></p><p class="headingAnchor" id="H112767698"><span class="h2">Frequency of identified mutations</span><span class="headingEndMark"> — </span>The diagnostic yield of mutation screening for the identification of pathogenic mutations in patients with HCM varies across studies, ranging from 30 to 63 percent, while in some populations with known founder variants, the yield of genetic testing can be as high as 72 percent [<a href="#rid7">7,22-29</a>]. In all study populations, mutations in the cardiac myosin binding protein-C gene are most common, accounting for up to half of the mutations identified [<a href="#rid7">7,22,24</a>]. Mutations in the cardiac beta-myosin heavy chain gene are second in frequency, being present in 25 to 40 percent of patients [<a href="#rid7">7,22,23</a>]. Mutations in the troponin I, troponin T, and alpha-tropomyosin genes account for only 5 to 10 percent of cases but may present with distinctive phenotypes [<a href="#rid7">7,25</a>]. Up to 5 percent of patients have multiple mutations [<a href="#rid7">7,22</a>].</p><p>In a study to assess the feasibility of genetic testing for HCM, 197 unrelated patients with either familial or sporadic HCM were screened by performing single-strand conformation polymorphism analysis of the coding sequence of nine genes, followed by DNA sequencing of abnormal patterns [<a href="#rid7">7</a>]. Disease-causing mutations were identified in 124 patients (63 percent). A total of 97 different mutations, including 60 novel ones, were identified. Mutations in the cardiac beta-myosin heavy chain and cardiac myosin binding protein-C genes accounted for more than 80 percent of the detected mutations.</p><p>In a study of 84 children diagnosed with isolated unexplained LVH before 15 years of age, mutations in genes associated with HCM in adults were identified in approximately half of presumed sporadic cases and in nearly two-thirds of familial cases [<a href="#rid26">26</a>]. Among the children with sarcomeric mutations &gt;75 percent had mutations in the cardiac beta-myosin heavy chain gene or the cardiac myosin binding protein-C gene.</p><p>In a study of 79 consecutive patients age 13 or younger who were diagnosed with HCM, 47 mutations were identified in 42 patients, with mutations involving the sarcomere protein genes MYBPC3 and MYH7 being the most common (49 and 36 percent of patients, respectively) [<a href="#rid27">27</a>]. (See <a class="local">'Cardiac myosin binding protein-C gene'</a> below and <a class="local">'Cardiac beta-myosin heavy chain gene'</a> below.)</p><p>The likelihood of identifying a mutation is lowest in patients ≥65 years of age. This finding is consistent with clinical studies showing cardiac differences in HCM in older patients [<a href="#rid30">30,31</a>] and suggests that older patients may have a different disease from younger patients. (See  <a class="medical medical_review" href="/z/d/html/4916.html" rel="external">"Hypertrophic cardiomyopathy: Natural history and prognosis", section on 'Late onset disease'</a>.)</p><p>Race may also impact the likelihood of identifying a sarcomeric mutation. Among 2467 patients enrolled in the Sarcomeric Human Cardiomyopathy Registry with hypertrophic cardiomyopathy, which included 205 Black patients (8 percent), sarcomeric gene mutations were identified in significantly fewer Black patients (26 versus 41 percent in non-Black patients).</p><p class="headingAnchor" id="H22623661"><span class="h1">PATHOGENESIS</span><span class="headingEndMark"> — </span>The consequences of HCM mutations in contractile protein genes appear to be directly related to effects on sarcomere function, leading to the characterization of HCM as a "disease of the sarcomere" [<a href="#rid32">32-38</a>]. However, a unifying explanation of the effect of these mutations on contraction, and thereby on hemodynamic function, has not been established. Some mutations appear to enhance contractility, while others have the opposite effect. (See  <a class="medical medical_review" href="/z/d/html/3451.html" rel="external">"Excitation-contraction coupling in myocardium"</a>.)</p><p>The effect of myosin mutations on the calcium sensitivity of myofilaments was studied in slow twitch skeletal muscle (soleus) from patients with HCM and selected myosin mutations [<a href="#rid39">39</a>]. Fibers from some of the mutations showed a decrease in mean calcium sensitivity, whereas others showed an increase, with higher active forces at low calcium concentrations and residual active force even under relaxing conditions. In addition, there was marked variability in calcium sensitivity between individual fibers carrying the same mutation; such effects were not observed in controls. The authors speculated that the variability in calcium sensitivity from fiber to fiber was likely to cause imbalances in force generation and be the final common pathway causing contractile dysfunction and development of disarray in the myocardium [<a href="#rid39">39</a>].</p><p>Others have suggested that the common abnormality is an increase in the energy cost of force production [<a href="#rid40">40</a>]. In support of this hypothesis, the ratio of phosphocreatine to ATP (a measure of myocardial energy reserve) is reduced by 30 percent in patients with HCM, regardless of genotype and even in the absence of overt hypertrophy [<a href="#rid41">41</a>].</p><p>Many genes are upregulated in the hearts of patients with HCM, including those producing secondary hypertrophy [<a href="#rid42">42</a>]. This suggests that the pathogenesis of the diverse cardiac phenotypes in part results from upregulation of the expression of various genes in response to the primary impetus provided by the mutant contractile protein.</p><p class="headingAnchor" id="H3987138"><span class="h1">SARCOMERIC GENE MUTATIONS CAUSING HCM</span><span class="headingEndMark"> — </span>Mutations in a variety of sarcomeric genes have been described in patients with HCM  (<a class="graphic graphic_figure graphicRef90768" href="/z/d/graphic/90768.html" rel="external">figure 3</a>), resulting in varying degrees of penetrance as well as differing phenotypic expressions of disease [<a href="#rid43">43,44</a>].</p><p class="headingAnchor" id="H3987144"><span class="h2">Cardiac myosin binding protein-C gene</span><span class="headingEndMark"> — </span>Mutations in the cardiac myosin binding protein-C gene (MYBPC3 gene) are the most common cause of HCM, being present in 14 to 26 percent of patients and, in the largest studies, accounting for 40 to 48 percent of mutations [<a href="#rid7">7,22,24,45</a>]. Mutations in cardiac myosin binding protein-C gene have also been identified in patients with dilated cardiomyopathy [<a href="#rid46">46</a>]. (See  <a class="medical medical_review" href="/z/d/html/4911.html" rel="external">"Genetics of dilated cardiomyopathy", section on 'Sarcomere genes'</a>.)</p><p>A broad range of gene defects in cardiac myosin binding protein-C can cause HCM, and numerous novel mutations, including missense, nonsense, splicing, and deletion and insertion mutations, have been identified [<a href="#rid47">47-53</a>]. Phenotypic expression of disease is heterogeneous, even in families with the same cardiac myosin binding protein-C mutation, and some patients with cardiac myosin binding protein-C mutations may not present until adult life [<a href="#rid46">46,47,54-57</a>]. One possible explanation for late onset disease is an interaction with genetic polymorphisms of the renin-angiotensin-aldosterone system (RAAS) [<a href="#rid58">58</a>]. (See <a class="local">'Renin-angiotensin system polymorphisms'</a> below.)</p><p>Approximately 40 percent of adults under the age of 50 with the cardiac myosin binding protein-C mutation do not have cardiac hypertrophy, and disease penetrance may remain incomplete through the age of 60 [<a href="#rid47">47</a>]. (See  <a class="medical medical_review" href="/z/d/html/4948.html" rel="external">"Hypertrophic cardiomyopathy: Clinical manifestations, diagnosis, and evaluation"</a>.)</p><p class="headingAnchor" id="H3987214"><span class="h2">Cardiac beta-myosin heavy chain gene</span><span class="headingEndMark"> — </span>Cardiac beta-myosin heavy chain gene (MYH7 gene) mutations, found in 13 to 25 percent of patients with HCM, are associated with a high penetrance of disease, younger age at diagnosis, and more severe hypertrophy [<a href="#rid7">7,22,23</a>]. As with other sarcomere gene mutations, there is substantial clinical heterogeneity in the phenotypic expression of HCM in patients with cardiac beta-myosin heavy chain gene mutations, likely due to the large number of different mutations reported (more than 50) or the coexistence of other genetic abnormalities [<a href="#rid59">59,60</a>]. In addition to variation in the phenotypic expression of the disease, survival in those with HCM due to a cardiac beta-myosin heavy chain mutation varies considerably despite nearly complete disease penetrance and significant hypertrophy.</p><p class="headingAnchor" id="H3987269"><span class="h2">Troponin T gene</span><span class="headingEndMark"> — </span>A variety of cardiac troponin T gene (TNNT2 gene) defects (missense mutations, small deletions and mutations in splice signals; chromosome 1q32) are responsible for 4 to 15 percent of cases of HCM, [<a href="#rid7">7,61,62</a>]. The cardiac phenotype produced by these gene defects is characterized by less hypertrophy than that observed with myosin gene mutations. Some adults with cardiac troponin gene mutations have normal cardiac wall thickness but have marked myocyte disarray and frequent episodes of sudden death at an early age [<a href="#rid63">63-65</a>].</p><p class="headingAnchor" id="H3987295"><span class="h2">Troponin I gene</span><span class="headingEndMark"> — </span>Mutations in the cardiac troponin I gene (TNNI3 gene; chromosome 19q13.4) have been identified in 2 to 7 percent of cases of HCM, with a disease penetrance of approximately 50 percent [<a href="#rid7">7,25,61</a>]. One of the reported mutations, a Lys183 deletion mutation in exon 7 of the TNNI3 gene, has been associated with a high disease penetrance, sudden death at any age, and dilated cardiomyopathy-like features in 44 percent in those over the age of 40 [<a href="#rid66">66</a>]. TNNI3 gene mutations are also associated with HCM with restrictive physiology [<a href="#rid67">67,68</a>].</p><p class="headingAnchor" id="H3987301"><span class="h2">Alpha tropomyosin gene</span><span class="headingEndMark"> — </span>Alpha tropomyosin gene (TPM1 gene) mutations, linked to chromosome 15q22.1, affect the thin filament of the sarcomere and account for less than 5 percent of cases of HCM [<a href="#rid7">7,61,62,69</a>]. The hypertrophic response to the alpha tropomyosin mutation Asp175Asn varies considerably among different families, suggesting that modifying genes and/or environment influences this cardiac phenotype. Survival is near normal in most of these patients, although this does not apply to all patients [<a href="#rid70">70,71</a>].</p><p class="headingAnchor" id="H3987307"><span class="h2">Myosin regulatory and essential light chain gene</span><span class="headingEndMark"> — </span>Mutations in the myosin regulatory light chain gene (MYL2 gene) or myosin essential light chain gene (MYL3 gene), which are located on chromosome 3p21.2-p21.3, are a rare cause of HCM [<a href="#rid72">72,73</a>]. The limited number of families with these gene defects has hindered correlation of genotype and phenotype, but there appears to be varied penetrance of at least some mutations [<a href="#rid73">73</a>].</p><p class="headingAnchor" id="H3987313"><span class="h2">Alpha-cardiac actin gene</span><span class="headingEndMark"> — </span>Sarcomeric mutations in the alpha-cardiac actin gene (ACTC1; chromosome 15q11-q14) are also a rare cause of HCM.</p><p class="headingAnchor" id="H3988165"><span class="h1">NONSARCOMERIC CAUSES OF LV HYPERTROPHY</span><span class="headingEndMark"> — </span>A number of uncommon metabolic myocardial storage cardiomyopathies demonstrate a clinical phenotype in which the pattern and extent of left ventricular hypertrophy (LVH) often mimics that of sarcomeric HCM. For this reason, these diseases are considered "phenocopies" of HCM [<a href="#rid45">45</a>]. Although these entities comprise only a small fraction of adult patients evaluated for HCM, differentiating diagnostically between these "HCM phenocopies" and sarcomeric HCM is crucial since the natural history, prognosis, and, in some cases, treatment strategies, are markedly different. Genetic testing can be performed to identify if a disease-causing mutation responsible for one of these diseases is present.</p><p class="headingAnchor" id="H3988172"><span class="h2">Alpha-galactosidase A and Fabry disease</span><span class="headingEndMark"> — </span>Mutations in the gene encoding alpha-galactosidase A cause Fabry disease, an X-linked recessive disorder that can be limited to the heart. Fabry disease may be responsible for as many as 12 percent of late-onset cases of LVH with no hypertension or aortic valve disease in women [<a href="#rid74">74</a>], though other series of unexplained LVH identify Fabry disease in 6.3 percent of men ≥40 and 1.4 percent &lt;40 years of age [<a href="#rid75">75</a>]. (See  <a class="medical medical_review" href="/z/d/html/4914.html" rel="external">"Fabry disease: Cardiovascular disease"</a>.)</p><p class="headingAnchor" id="H3988178"><span class="h2">RAS MAPK pathways genes and Noonan syndrome</span><span class="headingEndMark"> — </span>Approximately 20 percent of patients with Noonan syndrome, an autosomal dominant disorder characterized by facial dysmorphism, short stature, and congenital heart disease (pulmonary stenosis, septal defects), will also develop LVH. Several different gene mutations have been described in Noonan syndrome, but mutations in the genes coding for components of the RAS MAPK pathway have been associated with the development of LVH in this disorder [<a href="#rid13">13,14</a>]. (See <a class="local">'Mutations in sarcomeric protein genes'</a> above and  <a class="medical medical_review" href="/z/d/html/5832.html" rel="external">"Causes of short stature", section on 'Noonan syndrome'</a>.)</p><p class="headingAnchor" id="H3988184"><span class="h2">PRKAG2 and LAMP2 genes</span><span class="headingEndMark"> — </span>Mutations in the genes encoding the gamma-2 regulatory subunit of adenosine monophosphate (AMP)-activated protein kinase (PRKAG2) and lysosome-associated membrane protein 2 (LAMP2) have been associated with LVH in association with Wolff-Parkinson-White (WPW) syndrome [<a href="#rid16">16,76-81</a>]. These disorders are uncommon, with only 1 of 200 consecutive patients with unexplained LVH identified as having a PRKAG2 mutation [<a href="#rid80">80</a>]. These entities may be more common the setting of both LVH and ventricular preexcitation, as in another study of 24 patients with LVH and ventricular preexcitation, seven had PRKAG2 mutations, and four had LAMP2 mutations [<a href="#rid16">16</a>].</p><p>PRKAG2 is an enzyme that modulates glucose uptake and glycolysis. The defect in the mutated gene, which is inherited in an autosomal dominant pattern, may cause inappropriate activation of AMP-activated protein kinase, leading to an associated cardiomyopathy that results from vacuoles within the myocytes filled with glycogen-associated granules; the myocyte and myofibrillar disarray and cardiac fibrosis characteristic of mutations in genes encoding sarcomeric proteins are not seen [<a href="#rid79">79,81</a>]. In a review of 45 patients with cardiomyopathy due to PRKAG2 mutations, symptoms typical of HCM (palpitations, dyspnea, chest pain, and syncope) were present in 31 (69 percent), while an additional seven patients (15 percent) complained of myalgia and had a proximal myopathy [<a href="#rid82">82</a>]. Typical ECG abnormalities were seen in all patients by age 18, and LVH was present on echocardiography in 78 percent of adults and was progressive during follow-up. Progressive conduction system disease requiring pacemaker implantation occurred in 17 patients (38 percent) at a mean age of 38 years.</p><p>In a multicenter cohort study of 64 patients with cardiomyopathy and PRKAG2 mutations who were evaluated over a median of six years, average age at diagnosis was 36 years with a maximal wall thickness &lt;20 mm in the majority of patients and a preexcitation ECG pattern present in only one-third of patients. In this cohort with PRKAG2 cardiomyopathy, sudden death occurred in approximately 10 percent of patients, &gt;30 percent of patients required a permanent pacemaker due to complete heart block (average age, 37 years), early onset pf symptomatic atrial fibrillation/flutter was common, and more than 20 percent developed advanced HF symptoms, often with systolic dysfunction, including a subgroup requiring heart transplant.</p><p>Mutations in LAMP2 cause glycogen storage disease IIb (also called Danon disease), a semidominant X-linked disorder which is characterized by cardiomyopathy, skeletal myopathy, and variable intellectual disability. In those with LAMP2 mutations, the age of onset appears to be younger than in those with PRKAG2 mutations (mean of 15 versus 31 years), cardiomyopathy occurs earlier and more often in males, and most patients have elevations in serum alanine aminotransferase and creatine kinase [<a href="#rid16">16,83</a>]. The cardiomyopathy is typically hypertrophic in males, may be more often dilated in females, and is usually associated with preexcitation syndrome [<a href="#rid16">16,84</a>]. Some patients with LAMP2 mutations have predominantly cardiac manifestations without other features of Danon disease [<a href="#rid85">85</a>]. (See  <a class="medical medical_review" href="/z/d/html/2916.html" rel="external">"Lysosome-associated membrane protein 2 deficiency (glycogen storage disease IIb, Danon disease)"</a>.)</p><p class="headingAnchor" id="H3988232"><span class="h2">Renin-angiotensin system polymorphisms</span><span class="headingEndMark"> — </span>There is an interaction between several HCM gene mutations and genetic polymorphisms of the renin-angiotensin-aldosterone system (RAAS), which may account for variability in the presence and extent of hypertrophy among individuals, even those within the same family [<a href="#rid58">58,86-90</a>]. As an example, patients with the DD genotype of the angiotensin converting enzyme (ACE) gene generally have the greatest amount of septal hypertrophy compared with those with the ID or II genotypes [<a href="#rid86">86-88</a>]. Variations in the angiotensinogen gene may also contribute to different phenotypic expressions of the disease [<a href="#rid86">86,90</a>].</p><p>The influence of the RAAS is particularly evident with genetic mutations involving the myosin binding protein C [<a href="#rid86">86,87</a>]. In a study of 26 patients from one family who had cardiac myosin binding protein-C gene mutations, those with mutations resulting in genotypes associated with higher activation of the RAAS (pro-LVH genotype) had a significant increase in LV muscle mass, increased interventricular septal thickness, and pathologic ECG abnormalities [<a href="#rid58">58</a>]. Those without these genotypes did not have LVH or ECG abnormalities. In contrast, the ACE genotype appears to have no effect in mutations involving the cardiac beta-myosin heavy chain gene [<a href="#rid86">86</a>]. This suggests that RAAS genotypes may modify the clinical phenotype of HCM in a disease gene specific fashion. (See <a class="local">'Cardiac myosin binding protein-C gene'</a> above.)</p><p class="headingAnchor" id="H112767972"><span class="h1">CLINICAL APPLICATIONS OF GENETIC TESTING</span></p><p class="headingAnchor" id="H112767979"><span class="h2">Screening of family members for HCM</span><span class="headingEndMark"> — </span>Initial clinical screening of family members of a proband identified with HCM should include history, physical examination, ECG, and echocardiography  (<a class="graphic graphic_algorithm graphicRef126987" href="/z/d/graphic/126987.html" rel="external">algorithm 1</a>). There are differing expert opinions on the role and timing of genetic testing in first degree relatives of probands with a definitively identified gene-causing mutation [<a href="#rid4">4,91-93</a>]. While some experts recommend genetic testing in first-degree relatives for the pathogenic mutation identified in proband prior to performing clinical evaluation with ECG and echocardiography, UpToDate experts recommend only offering cascade genetic testing of family members after initial comprehensive clinical evaluation to determine if HCM is present. Screening of first-degree relatives of patients with HCM is discussed in greater detail separately. (See  <a class="medical medical_review" href="/z/d/html/4948.html" rel="external">"Hypertrophic cardiomyopathy: Clinical manifestations, diagnosis, and evaluation", section on 'Screening of first-degree relatives'</a>.)</p><p>A strategy of testing HCM family members to determine if they are at risk of developing disease cannot be pursued if a variant of uncertain significance (VUS; or no mutation) is identified in the proband  (<a class="graphic graphic_algorithm graphicRef126987" href="/z/d/graphic/126987.html" rel="external">algorithm 1</a>). However, a definitive pathogenic mutation result can be used to test other relatives for the same mutation [<a href="#rid17">17,19,94,95</a>].</p><p class="bulletIndent1"><span class="glyph">●</span>Relatives testing <strong>negative</strong> for the family mutation are considered unaffected. This result largely alleviates the psychologic and economic burden of further cardiovascular surveillance as well as the need for restrictions on lifestyle and competitive sports [<a href="#rid96">96</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Relatives testing <strong>positive</strong> for the same disease-causing mutation as the proband, but in whom there is no clinical evidence of left ventricular hypertrophy (LVH), are referred to as being genotype positive/phenotype negative (G+ P-). Such individuals have generated clinical decision-making dilemmas. However, at present there is no compelling evidence to suggest that G+ P- family members are at increased risk for sudden death [<a href="#rid97">97</a>]. Therefore, Bethesda Conference #36 consensus recommendations do not exclude G+ P- from competitive sports [<a href="#rid98">98</a>]. However, for G+ P- family members it is prudent to extend standard HCM surveillance with cardiac imaging at least through mid-life (ie, 40 years of age) to detect development of the phenotype [<a href="#rid99">99</a>].</p><p></p><p>Though the reported penetrance in HCM ranges from 50 to 100 percent, it is generally considered to be high, perhaps reflecting selection bias of published papers for more severe disease [<a href="#rid100">100-102</a>]. The penetrance of pathogenic sarcomere mutations is allele specific, and unless the mutation linked to pedigree analysis has been previously published, the precise likelihood of developing clinical expression of disease in an HCM family member identified as carrying a mutation is unknown [<a href="#rid103">103,104</a>]. In an observational study describing the experience at a single European center over 32 years, among 620 relatives of 149 probands with a definite pathogenic mutation who underwent genetic testing, 264 relatives (43 percent) were genotype positive for a pathogenic HCM mutation and 356 relatives (57 percent) were genotype negative [<a href="#rid29">29</a>]. Among the genotype positive patients, 16 percent developed clinical evidence of HCM over a seven-year period, including 44 percent &gt;50 years of age, supporting the concept that screening with imaging should extend to mid-life and possibly beyond in family members. On the other hand, the need for longitudinal screening was eliminated in over half of relatives in whom genetic testing demonstrated no pathogenic mutation.</p><p>Several issues complicate the use of genetic testing as a screening tool:</p><p class="bulletIndent1"><span class="glyph">●</span>Some of the genes responsible for HCM have not yet been identified; therefore, the likelihood of obtaining a positive test in the proband is only about 30 to 60 percent [<a href="#rid3">3,17,94,105,106</a>]. (See <a class="local">'Frequency of identified mutations'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Population studies have demonstrated that up to 5 percent of patients are compound heterozygotes (inheriting two different mutations within a single HCM gene), double heterozygotes (inheriting mutations in two HCM genes), or homozygotes (inheriting the same mutation from both parents). To be certain of detecting such genotypes, sequencing of candidate genes should continue in a given patient even after a single mutation has been identified [<a href="#rid7">7,107</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Many families, particularly with the two commonest gene abnormalities beta-myosin heavy chain and myosin binding protein C, have their own "private" mutations, and therefore knowledge of these gene mutations cannot be linked to data derived from other families. This often makes it difficult to reliably determine pathogenicity of these private mutations [<a href="#rid94">94</a>].</p><p></p><p class="headingAnchor" id="H112767993"><span class="h2">Genotype-phenotype relationships</span><span class="headingEndMark"> — </span>No definitive or reliable relationship has been established between individual sarcomere mutations and patterns of LVH in HCM, although some patterns have been suggested from large registry datasets [<a href="#rid17">17,95,108</a>]. Even within the same family, the morphology of LVH can be dramatically different among relatives with the same disease causing sarcomere mutation [<a href="#rid19">19</a>]. Therefore, mutations cannot be used to predict the type of phenotypically expressed HCM a patient will ultimately develop.</p><p class="headingAnchor" id="H112768001"><span class="h2">Predicting prognosis with mutations</span><span class="headingEndMark"> — </span>Although there was a tremendous amount of optimism that genetic testing could be used to help stratify patients for risk of sudden death, this hope has generally been unrealized. Indeed, given the substantial amount of genetic heterogeneity responsible for HCM, it has not been possible to classify mutations as being definitively "benign" or "malignant" [<a href="#rid109">109</a>].</p><p class="bulletIndent1"><span class="glyph">●</span>In a review of 293 unrelated patients with HCM who were genotyped for specific "benign" mutations in the genes for the cardiac beta- myosin heavy chain, troponin T, and alpha tropomyosin, only five patients (1.7 percent) possessed one of these benign mutations; all had severe manifestations of HCM, and three had a family history of sudden death [<a href="#rid71">71</a>]. Similarly, only three patients (1 percent) had "malignant" mutations [<a href="#rid110">110</a>]. Such a targeted strategy is unlikely to be effective given that most families have their own private mutation.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>In another series of 389 patients with HCM, 58 (15 percent) were identified as having mutations in beta-myosin heavy chain that were previously associated with a "malignant" course. Although the probands with a beta-myosin heavy chain mutation were diagnosed younger and had more LVH compared with other HCM patients, there was no difference in history of sudden cardiac death [<a href="#rid110">110</a>].</p><p></p><p>Therefore, a patient's clinical course cannot be predicted with any degree of certainty based on the type of mutation [<a href="#rid17">17,94,95,109,111</a>]. As a result, management decisions such as ICD therapy for primary prevention should not be predicated solely on the presence of a specific type of sarcomere mutation [<a href="#rid17">17</a>].</p><p>One possible exception to the above mentioned principle is in reference to the association between multiple mutations and outcome. Approximately 5 percent of patients with HCM have one or more sarcomere mutation [<a href="#rid109">109,112</a>]. Early observations have noted that double or triple mutations are associated with earlier and more severe disease expression [<a href="#rid107">107,112</a>]. However, additional studies are necessary before considering multiple mutations as an independent risk factor for adverse outcome in HCM.</p><p class="headingAnchor" id="H1875886902"><span class="h1">SOCIETY GUIDELINE LINKS</span><span class="headingEndMark"> — </span>Links to society and government-sponsored guidelines from selected countries and regions around the world are provided separately. (See  <a class="medical medical_society_guidelines" href="/z/d/html/115416.html" rel="external">"Society guideline links: Cardiomyopathy"</a>.)</p><p class="headingAnchor" id="H1381757"><span class="h1">SUMMARY AND RECOMMENDATIONS</span></p><p class="bulletIndent1"><span class="glyph">●</span>Genetic linkage studies in familial hypertrophic cardiomyopathy (HCM) have demonstrated autosomal dominant disease loci at a variety of chromosomal sites. Most of the genetic loci in familial HCM encode one of the myocardial contractile proteins of the cardiac sarcomere. Patients with sporadic disease, in which the parents do not carry a disease-causing mutation, have similar sarcomeric mutations as those with familial disease. Gene mutations which do not encode contractile proteins (PRKAG2, LAMP2, GLA in Fabry disease, RAS MAPK in Noonan syndrome) are responsible for diseases which can also manifest left ventricular hypertrophy and therefore can be considered "phenocopies" of sarcomeric HCM. Genetic testing can be used to confirm diagnosis of these nonsarcomeric metabolic cardiomyopathies. (See <a class="local">'Mutations in sarcomeric protein genes'</a> above and <a class="local">'Sarcomeric gene mutations causing HCM'</a> above and <a class="local">'Nonsarcomeric causes of LV hypertrophy'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The consequences of mutations in contractile protein genes appear to be directly related to effects on sarcomere function, leading to the characterization of HCM as a "disease of the sarcomere". However, a unifying explanation of the effect of these mutations on systolic and diastolic function has not been established. Some mutations appear to enhance contractility, while others have the opposite effect. (See <a class="local">'Pathogenesis'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Mutations in the cardiac myosin binding protein-C gene and cardiac beta-myosin heavy chain gene  (<a class="graphic graphic_figure graphicRef90769" href="/z/d/graphic/90769.html" rel="external">figure 2</a>) are the two most common mutations responsible for close to 70 percent of identifiable mutations in HCM. Mutations in the troponin I, troponin T, and alpha-tropomyosin genes account for only 5 to 10 percent of cases, while up to 5 percent of HCM patients have multiple mutations. (See <a class="local">'Frequency of identified mutations'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genetic testing for HCM is available as a clinical test. Genetic testing is limited by the fact that only 50 percent of patients have an identifiable mutation, and a substantial proportion have variants of uncertain significance (VUS) in which the pathogenicity of the mutation is uncertain. The greatest clinical applicability of genetic testing is in identifying at risk family members. However, prior to performing genetic testing in a family, first-degree relatives of an affected individual should be evaluated (by history, physical examination, electrocardiography, and echocardiography) for clinical evidence of HCM  (<a class="graphic graphic_algorithm graphicRef126987" href="/z/d/graphic/126987.html" rel="external">algorithm 1</a>). If a disease-causing mutation is identified in the proband, then other family members who have no clinical evidence of disease by echocardiography can be tested for that specific mutation to determine if they are at risk or not of developing HCM. (See <a class="local">'Screening of family members for HCM'</a> above and  <a class="medical medical_review" href="/z/d/html/4948.html" rel="external">"Hypertrophic cardiomyopathy: Clinical manifestations, diagnosis, and evaluation", section on 'Screening of first-degree relatives'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The availability of genetic testing in the clinical care of patients has resulted in the identification of a growing number of genotype positive/phenotype negative (G+ P-) patients. Although penetrance of sarcomere mutation is considered high, the precise likelihood of developing disease with a mutation is uncertain. There is no compelling evidence that G+ P- are at increased risk of sudden death, and therefore current recommendations do not definitively exclude G+ P- patients from participating in competitive sports. However, for G+ P- family members it is prudent to extend standard HCM surveillance with cardiac imaging at least through mid-life (40 years of age) to detect development of the phenotype. (See <a class="local">'Genotype-phenotype relationships'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>There is no clear relationship between genotype and phenotype in HCM. Therefore it is not possible to predict the type or extent of morphologic expression in an individual patient based on mutation. In addition, it is not possible to predict clinical outcome based on individual mutations, and therefore results of genetic testing do not impact on patient management strategies, including decision-making for ICD therapy. One possible exception to the principle that mutations do not predict clinical outcome relates to patients with double (or triple) mutations. Emerging observations suggest HCM patients with more than one disease-causing mutation are at greater risk of adverse disease related events, although more data are necessary before predicating management decisions based on the presence of multiple mutations. (See <a class="local">'Predicting prognosis with mutations'</a> above.)</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Marian AJ, Roberts R. The molecular genetic basis for hypertrophic cardiomyopathy. J Mol Cell Cardiol 2001; 33:655.</a></li><li><a class="nounderline abstract_t">Ashrafian H, Redwood C, Blair E, Watkins H. Hypertrophic cardiomyopathy:a paradigm for myocardial energy depletion. Trends Genet 2003; 19:263.</a></li><li><a class="nounderline abstract_t">Ackerman MJ, Priori SG, Willems S, et al. HRS/EHRA expert consensus statement on the state of genetic testing for the channelopathies and cardiomyopathies this document was developed as a partnership between the Heart Rhythm Society (HRS) and the European Heart Rhythm Association (EHRA). Heart Rhythm 2011; 8:1308.</a></li><li><a class="nounderline abstract_t">Ommen SR, Mital S, Burke MA, et al. 2020 AHA/ACC Guideline for the Diagnosis and Treatment of Patients With Hypertrophic Cardiomyopathy: A Report of the American College of Cardiology/American Heart Association Joint Committee on Clinical Practice Guidelines. Circulation 2020; 142:e558.</a></li><li><a class="nounderline abstract_t">Ingles J, Burns C, Barratt A, Semsarian C. Application of Genetic Testing in Hypertrophic Cardiomyopathy for Preclinical Disease Detection. Circ Cardiovasc Genet 2015; 8:852.</a></li><li><a class="nounderline abstract_t">Ingles J, Goldstein J, Thaxton C, et al. Evaluating the Clinical Validity of Hypertrophic Cardiomyopathy Genes. Circ Genom Precis Med 2019; 12:e002460.</a></li><li><a class="nounderline abstract_t">Richard P, Charron P, Carrier L, et al. Hypertrophic cardiomyopathy: distribution of disease genes, spectrum of mutations, and implications for a molecular diagnosis strategy. Circulation 2003; 107:2227.</a></li><li><a class="nounderline abstract_t">Watkins H, Thierfelder L, Anan R, et al. Independent origin of identical beta cardiac myosin heavy-chain mutations in hypertrophic cardiomyopathy. Am J Hum Genet 1993; 53:1180.</a></li><li><a class="nounderline abstract_t">Watkins H, Anan R, Coviello DA, et al. A de novo mutation in alpha-tropomyosin that causes hypertrophic cardiomyopathy. Circulation 1995; 91:2302.</a></li><li><a class="nounderline abstract_t">Anan R, Niimura H, Takenaka T, et al. Mutations in the genes for sarcomeric proteins in Japanese patients with onset sporadic hypertrophic cardiomyopathy after age 40 years. Am J Cardiol 2007; 99:1750.</a></li><li><a class="nounderline abstract_t">Monserrat L, Gimeno-Blanes JR, Marín F, et al. Prevalence of fabry disease in a cohort of 508 unrelated patients with hypertrophic cardiomyopathy. J Am Coll Cardiol 2007; 50:2399.</a></li><li><a class="nounderline abstract_t">Elliott P, Baker R, Pasquale F, et al. Prevalence of Anderson-Fabry disease in patients with hypertrophic cardiomyopathy: the European Anderson-Fabry Disease survey. Heart 2011; 97:1957.</a></li><li><a class="nounderline abstract_t">Wu X, Simpson J, Hong JH, et al. MEK-ERK pathway modulation ameliorates disease phenotypes in a mouse model of Noonan syndrome associated with the Raf1(L613V) mutation. J Clin Invest 2011; 121:1009.</a></li><li><a class="nounderline abstract_t">Marin TM, Keith K, Davies B, et al. Rapamycin reverses hypertrophic cardiomyopathy in a mouse model of LEOPARD syndrome-associated PTPN11 mutation. J Clin Invest 2011; 121:1026.</a></li><li><a class="nounderline abstract_t">Geier C, Perrot A, Ozcelik C, et al. Mutations in the human muscle LIM protein gene in families with hypertrophic cardiomyopathy. Circulation 2003; 107:1390.</a></li><li><a class="nounderline abstract_t">Arad M, Maron BJ, Gorham JM, et al. Glycogen storage diseases presenting as hypertrophic cardiomyopathy. N Engl J Med 2005; 352:362.</a></li><li><a class="nounderline abstract_t">Tester DJ, Ackerman MJ. Genetic testing for potentially lethal, highly treatable inherited cardiomyopathies/channelopathies in clinical practice. Circulation 2011; 123:1021.</a></li><li><a class="nounderline abstract_t">Richards CS, Bale S, Bellissimo DB, et al. ACMG recommendations for standards for interpretation and reporting of sequence variations: Revisions 2007. Genet Med 2008; 10:294.</a></li><li><a class="nounderline abstract_t">Maron BJ, Maron MS, Semsarian C. Genetics of hypertrophic cardiomyopathy after 20 years: clinical perspectives. J Am Coll Cardiol 2012; 60:705.</a></li><li><a class="nounderline abstract_t">Manrai AK, Funke BH, Rehm HL, et al. Genetic Misdiagnoses and the Potential for Health Disparities. N Engl J Med 2016; 375:655.</a></li><li><a class="nounderline abstract_t">Walsh R, Buchan R, Wilk A, et al. Defining the genetic architecture of hypertrophic cardiomyopathy: re-evaluating the role of non-sarcomeric genes. Eur Heart J 2017; 38:3461.</a></li><li><a class="nounderline abstract_t">Van Driest SL, Ommen SR, Tajik AJ, et al. Sarcomeric genotyping in hypertrophic cardiomyopathy. Mayo Clin Proc 2005; 80:463.</a></li><li><a class="nounderline abstract_t">Van Driest SL, Jaeger MA, Ommen SR, et al. Comprehensive analysis of the beta-myosin heavy chain gene in 389 unrelated patients with hypertrophic cardiomyopathy. J Am Coll Cardiol 2004; 44:602.</a></li><li><a class="nounderline abstract_t">Van Driest SL, Vasile VC, Ommen SR, et al. Myosin binding protein C mutations and compound heterozygosity in hypertrophic cardiomyopathy. J Am Coll Cardiol 2004; 44:1903.</a></li><li><a class="nounderline abstract_t">Mogensen J, Murphy RT, Kubo T, et al. Frequency and clinical expression of cardiac troponin I mutations in 748 consecutive families with hypertrophic cardiomyopathy. J Am Coll Cardiol 2004; 44:2315.</a></li><li><a class="nounderline abstract_t">Morita H, Rehm HL, Menesses A, et al. Shared genetic causes of cardiac hypertrophy in children and adults. N Engl J Med 2008; 358:1899.</a></li><li><a class="nounderline abstract_t">Kaski JP, Syrris P, Esteban MT, et al. Prevalence of sarcomere protein gene mutations in preadolescent children with hypertrophic cardiomyopathy. Circ Cardiovasc Genet 2009; 2:436.</a></li><li><a class="nounderline abstract_t">Van Driest SL, Ommen SR, Tajik AJ, et al. Yield of genetic testing in hypertrophic cardiomyopathy. Mayo Clin Proc 2005; 80:739.</a></li><li><a class="nounderline abstract_t">van Velzen HG, Schinkel AFL, Baart SJ, et al. Outcomes of Contemporary Family Screening in Hypertrophic Cardiomyopathy. Circ Genom Precis Med 2018; 11:e001896.</a></li><li><a class="nounderline abstract_t">Lever HM, Karam RF, Currie PJ, Healy BP. Hypertrophic cardiomyopathy in the elderly. Distinctions from the young based on cardiac shape. Circulation 1989; 79:580.</a></li><li><a class="nounderline abstract_t">Lewis JF, Maron BJ. Elderly patients with hypertrophic cardiomyopathy: a subset with distinctive left ventricular morphology and progressive clinical course late in life. J Am Coll Cardiol 1989; 13:36.</a></li><li><a class="nounderline abstract_t">Rayment I, Holden HM, Sellers JR, et al. Structural interpretation of the mutations in the beta-cardiac myosin that have been implicated in familial hypertrophic cardiomyopathy. Proc Natl Acad Sci U S A 1995; 92:3864.</a></li><li><a class="nounderline abstract_t">Watkins H, Seidman JG, Seidman CE. Familial hypertrophic cardiomyopathy: a genetic model of cardiac hypertrophy. Hum Mol Genet 1995; 4 Spec No:1721.</a></li><li><a class="nounderline abstract_t">Sweeney HL, Straceski AJ, Leinwand LA, et al. Heterologous expression of a cardiomyopathic myosin that is defective in its actin interaction. J Biol Chem 1994; 269:1603.</a></li><li><a class="nounderline abstract_t">Straceski AJ, Geisterfer-Lowrance A, Seidman CE, et al. Functional analysis of myosin missense mutations in familial hypertrophic cardiomyopathy. Proc Natl Acad Sci U S A 1994; 91:589.</a></li><li><a class="nounderline abstract_t">Sata M, Ikebe M. Functional analysis of the mutations in the human cardiac beta-myosin that are responsible for familial hypertrophic cardiomyopathy. Implication for the clinical outcome. J Clin Invest 1996; 98:2866.</a></li><li><a class="nounderline abstract_t">Lin D, Bobkova A, Homsher E, Tobacman LS. Altered cardiac troponin T in vitro function in the presence of a mutation implicated in familial hypertrophic cardiomyopathy. J Clin Invest 1996; 97:2842.</a></li><li><a class="nounderline abstract_t">Watkins H, Seidman CE, Seidman JG, et al. Expression and functional assessment of a truncated cardiac troponin T that causes hypertrophic cardiomyopathy. Evidence for a dominant negative action. J Clin Invest 1996; 98:2456.</a></li><li><a class="nounderline abstract_t">Kirschner SE, Becker E, Antognozzi M, et al. Hypertrophic cardiomyopathy-related beta-myosin mutations cause highly variable calcium sensitivity with functional imbalances among individual muscle cells. Am J Physiol Heart Circ Physiol 2005; 288:H1242.</a></li><li><a class="nounderline abstract_t">Watkins H. Genetic clues to disease pathways in hypertrophic and dilated cardiomyopathies. Circulation 2003; 107:1344.</a></li><li><a class="nounderline abstract_t">Crilley JG, Boehm EA, Blair E, et al. Hypertrophic cardiomyopathy due to sarcomeric gene mutations is characterized by impaired energy metabolism irrespective of the degree of hypertrophy. J Am Coll Cardiol 2003; 41:1776.</a></li><li><a class="nounderline abstract_t">Lim DS, Roberts R, Marian AJ. Expression profiling of cardiac genes in human hypertrophic cardiomyopathy: insight into the pathogenesis of phenotypes. J Am Coll Cardiol 2001; 38:1175.</a></li><li><a class="nounderline abstract_t">Maron BJ, Maron MS, Maron BA, Loscalzo J. Moving Beyond the Sarcomere to Explain Heterogeneity in Hypertrophic Cardiomyopathy: JACC Review Topic of the Week. J Am Coll Cardiol 2019; 73:1978.</a></li><li><a class="nounderline abstract_t">Geske JB, Ommen SR, Gersh BJ. Hypertrophic Cardiomyopathy: Clinical Update. JACC Heart Fail 2018; 6:364.</a></li><li><a class="nounderline abstract_t">Veselka J, Anavekar NS, Charron P. Hypertrophic obstructive cardiomyopathy. Lancet 2017; 389:1253.</a></li><li><a class="nounderline abstract_t">Ehlermann P, Weichenhan D, Zehelein J, et al. Adverse events in families with hypertrophic or dilated cardiomyopathy and mutations in the MYBPC3 gene. BMC Med Genet 2008; 9:95.</a></li><li><a class="nounderline abstract_t">Niimura H, Bachinski LL, Sangwatanaroj S, et al. Mutations in the gene for cardiac myosin-binding protein C and late-onset familial hypertrophic cardiomyopathy. N Engl J Med 1998; 338:1248.</a></li><li><a class="nounderline abstract_t">Bonne G, Carrier L, Bercovici J, et al. Cardiac myosin binding protein-C gene splice acceptor site mutation is associated with familial hypertrophic cardiomyopathy. Nat Genet 1995; 11:438.</a></li><li><a class="nounderline abstract_t">Carrier L, Bonne G, Bährend E, et al. Organization and sequence of human cardiac myosin binding protein C gene (MYBPC3) and identification of mutations predicted to produce truncated proteins in familial hypertrophic cardiomyopathy. Circ Res 1997; 80:427.</a></li><li><a class="nounderline abstract_t">Rottbauer W, Gautel M, Zehelein J, et al. Novel splice donor site mutation in the cardiac myosin-binding protein-C gene in familial hypertrophic cardiomyopathy. Characterization Of cardiac transcript and protein. J Clin Invest 1997; 100:475.</a></li><li><a class="nounderline abstract_t">Moolman JA, Reith S, Uhl K, et al. A newly created splice donor site in exon 25 of the MyBP-C gene is responsible for inherited hypertrophic cardiomyopathy with incomplete disease penetrance. Circulation 2000; 101:1396.</a></li><li><a class="nounderline abstract_t">Erdmann J, Raible J, Maki-Abadi J, et al. Spectrum of clinical phenotypes and gene variants in cardiac myosin-binding protein C mutation carriers with hypertrophic cardiomyopathy. J Am Coll Cardiol 2001; 38:322.</a></li><li><a class="nounderline abstract_t">Konno T, Shimizu M, Ino H, et al. A novel missense mutation in the myosin binding protein-C gene is responsible for hypertrophic cardiomyopathy with left ventricular dysfunction and dilation in elderly patients. J Am Coll Cardiol 2003; 41:781.</a></li><li><a class="nounderline abstract_t">Charron P, Dubourg O, Desnos M, et al. Clinical features and prognostic implications of familial hypertrophic cardiomyopathy related to the cardiac myosin-binding protein C gene. Circulation 1998; 97:2230.</a></li><li><a class="nounderline abstract_t">Niimura H, Patton KK, McKenna WJ, et al. Sarcomere protein gene mutations in hypertrophic cardiomyopathy of the elderly. Circulation 2002; 105:446.</a></li><li><a class="nounderline abstract_t">Maron BJ, Niimura H, Casey SA, et al. Development of left ventricular hypertrophy in adults in hypertrophic cardiomyopathy caused by cardiac myosin-binding protein C gene mutations. J Am Coll Cardiol 2001; 38:315.</a></li><li><a class="nounderline abstract_t">Dhandapany PS, Sadayappan S, Xue Y, et al. A common MYBPC3 (cardiac myosin binding protein C) variant associated with cardiomyopathies in South Asia. Nat Genet 2009; 41:187.</a></li><li><a class="nounderline abstract_t">Ortlepp JR, Vosberg HP, Reith S, et al. Genetic polymorphisms in the renin-angiotensin-aldosterone system associated with expression of left ventricular hypertrophy in hypertrophic cardiomyopathy: a study of five polymorphic genes in a family with a disease causing mutation in the myosin binding protein C gene. Heart 2002; 87:270.</a></li><li><a class="nounderline abstract_t">Solomon SD, Wolff S, Watkins H, et al. Left ventricular hypertrophy and morphology in familial hypertrophic cardiomyopathy associated with mutations of the beta-myosin heavy chain gene. J Am Coll Cardiol 1993; 22:498.</a></li><li><a class="nounderline abstract_t">Arbustini E, Fasani R, Morbini P, et al. Coexistence of mitochondrial DNA and beta myosin heavy chain mutations in hypertrophic cardiomyopathy with late congestive heart failure. Heart 1998; 80:548.</a></li><li><a class="nounderline abstract_t">Van Driest SL, Ellsworth EG, Ommen SR, et al. Prevalence and spectrum of thin filament mutations in an outpatient referral population with hypertrophic cardiomyopathy. Circulation 2003; 108:445.</a></li><li><a class="nounderline abstract_t">Watkins H, McKenna WJ, Thierfelder L, et al. Mutations in the genes for cardiac troponin T and alpha-tropomyosin in hypertrophic cardiomyopathy. N Engl J Med 1995; 332:1058.</a></li><li><a class="nounderline abstract_t">Varnava A, Baboonian C, Davison F, et al. A new mutation of the cardiac troponin T gene causing familial hypertrophic cardiomyopathy without left ventricular hypertrophy. Heart 1999; 82:621.</a></li><li><a class="nounderline abstract_t">Varnava AM, Elliott PM, Baboonian C, et al. Hypertrophic cardiomyopathy: histopathological features of sudden death in cardiac troponin T disease. Circulation 2001; 104:1380.</a></li><li><a class="nounderline abstract_t">Fujino N, Shimizu M, Ino H, et al. A novel mutation Lys273Glu in the cardiac troponin T gene shows high degree of penetrance and transition from hypertrophic to dilated cardiomyopathy. Am J Cardiol 2002; 89:29.</a></li><li><a class="nounderline abstract_t">Kokado H, Shimizu M, Yoshio H, et al. Clinical features of hypertrophic cardiomyopathy caused by a Lys183 deletion mutation in the cardiac troponin I gene. Circulation 2000; 102:663.</a></li><li><a class="nounderline abstract_t">Kubo T, Gimeno JR, Bahl A, et al. Prevalence, clinical significance, and genetic basis of hypertrophic cardiomyopathy with restrictive phenotype. J Am Coll Cardiol 2007; 49:2419.</a></li><li><a class="nounderline abstract_t">Mogensen J, Kubo T, Duque M, et al. Idiopathic restrictive cardiomyopathy is part of the clinical expression of cardiac troponin I mutations. J Clin Invest 2003; 111:209.</a></li><li><a class="nounderline abstract_t">Karibe A, Tobacman LS, Strand J, et al. Hypertrophic cardiomyopathy caused by a novel alpha-tropomyosin mutation (V95A) is associated with mild cardiac phenotype, abnormal calcium binding to troponin, abnormal myosin cycling, and poor prognosis. Circulation 2001; 103:65.</a></li><li><a class="nounderline abstract_t">Coviello DA, Maron BJ, Spirito P, et al. Clinical features of hypertrophic cardiomyopathy caused by mutation of a "hot spot" in the alpha-tropomyosin gene. J Am Coll Cardiol 1997; 29:635.</a></li><li><a class="nounderline abstract_t">Van Driest SL, Ackerman MJ, Ommen SR, et al. Prevalence and severity of "benign" mutations in the beta-myosin heavy chain, cardiac troponin T, and alpha-tropomyosin genes in hypertrophic cardiomyopathy. Circulation 2002; 106:3085.</a></li><li><a class="nounderline abstract_t">Poetter K, Jiang H, Hassanzadeh S, et al. Mutations in either the essential or regulatory light chains of myosin are associated with a rare myopathy in human heart and skeletal muscle. Nat Genet 1996; 13:63.</a></li><li><a class="nounderline abstract_t">Lee W, Hwang TH, Kimura A, et al. Different expressivity of a ventricular essential myosin light chain gene Ala57Gly mutation in familial hypertrophic cardiomyopathy. Am Heart J 2001; 141:184.</a></li><li><a class="nounderline abstract_t">Chimenti C, Pieroni M, Morgante E, et al. Prevalence of Fabry disease in female patients with late-onset hypertrophic cardiomyopathy. Circulation 2004; 110:1047.</a></li><li><a class="nounderline abstract_t">Sachdev B, Takenaka T, Teraguchi H, et al. Prevalence of Anderson-Fabry disease in male patients with late onset hypertrophic cardiomyopathy. Circulation 2002; 105:1407.</a></li><li><a class="nounderline abstract_t">Seidman C. Genetic causes of inherited cardiac hypertrophy: Robert L. Frye Lecture. Mayo Clin Proc 2002; 77:1315.</a></li><li><a class="nounderline abstract_t">MacRae CA, Ghaisas N, Kass S, et al. Familial Hypertrophic cardiomyopathy with Wolff-Parkinson-White syndrome maps to a locus on chromosome 7q3. J Clin Invest 1995; 96:1216.</a></li><li><a class="nounderline abstract_t">Blair E, Redwood C, Ashrafian H, et al. Mutations in the gamma(2) subunit of AMP-activated protein kinase cause familial hypertrophic cardiomyopathy: evidence for the central role of energy compromise in disease pathogenesis. Hum Mol Genet 2001; 10:1215.</a></li><li><a class="nounderline abstract_t">Arad M, Benson DW, Perez-Atayde AR, et al. Constitutively active AMP kinase mutations cause glycogen storage disease mimicking hypertrophic cardiomyopathy. J Clin Invest 2002; 109:357.</a></li><li><a class="nounderline abstract_t">Murphy RT, Mogensen J, McGarry K, et al. Adenosine monophosphate-activated protein kinase disease mimicks hypertrophic cardiomyopathy and Wolff-Parkinson-White syndrome: natural history. J Am Coll Cardiol 2005; 45:922.</a></li><li><a class="nounderline abstract_t">Ahmad F, Arad M, Musi N, et al. Increased alpha2 subunit-associated AMPK activity and PRKAG2 cardiomyopathy. Circulation 2005; 112:3140.</a></li><li><a class="nounderline abstract_t">Lopez-Sainz A, Dominguez F, Lopes LR, et al. Clinical Features and Natural History of PRKAG2 Variant Cardiac Glycogenosis. J Am Coll Cardiol 2020; 76:186.</a></li><li><a class="nounderline abstract_t">Yang Z, McMahon CJ, Smith LR, et al. Danon disease as an underrecognized cause of hypertrophic cardiomyopathy in children. Circulation 2005; 112:1612.</a></li><li><a class="nounderline abstract_t">Sugie K, Yamamoto A, Murayama K, et al. Clinicopathological features of genetically confirmed Danon disease. Neurology 2002; 58:1773.</a></li><li><a class="nounderline abstract_t">Maron BJ, Roberts WC, Arad M, et al. Clinical outcome and phenotypic expression in LAMP2 cardiomyopathy. JAMA 2009; 301:1253.</a></li><li><a class="nounderline abstract_t">Perkins MJ, Van Driest SL, Ellsworth EG, et al. Gene-specific modifying effects of pro-LVH polymorphisms involving the renin-angiotensin-aldosterone system among 389 unrelated patients with hypertrophic cardiomyopathy. Eur Heart J 2005; 26:2457.</a></li><li><a class="nounderline abstract_t">Tesson F, Dufour C, Moolman JC, et al. The influence of the angiotensin I converting enzyme genotype in familial hypertrophic cardiomyopathy varies with the disease gene mutation. J Mol Cell Cardiol 1997; 29:831.</a></li><li><a class="nounderline abstract_t">Lechin M, Quiñones MA, Omran A, et al. Angiotensin-I converting enzyme genotypes and left ventricular hypertrophy in patients with hypertrophic cardiomyopathy. Circulation 1995; 92:1808.</a></li><li><a class="nounderline abstract_t">Yoneya K, Okamoto H, Machida M, et al. Angiotensin-converting enzyme gene polymorphism in Japanese patients with hypertrophic cardiomyopathy. Am Heart J 1995; 130:1089.</a></li><li><a class="nounderline abstract_t">Ishanov A, Okamoto H, Yoneya K, et al. Angiotensinogen gene polymorphism in Japanese patients with hypertrophic cardiomyopathy. Am Heart J 1997; 133:184.</a></li><li><a class="nounderline abstract_t">Authors/Task Force members, Elliott PM, Anastasakis A, et al. 2014 ESC Guidelines on diagnosis and management of hypertrophic cardiomyopathy: the Task Force for the Diagnosis and Management of Hypertrophic Cardiomyopathy of the European Society of Cardiology (ESC). Eur Heart J 2014; 35:2733.</a></li><li><a class="nounderline abstract_t">Hershberger RE, Givertz MM, Ho CY, et al. Genetic Evaluation of Cardiomyopathy-A Heart Failure Society of America Practice Guideline. J Card Fail 2018; 24:281.</a></li><li><a class="nounderline abstract_t">Hershberger RE, Givertz MM, Ho CY, et al. Genetic evaluation of cardiomyopathy: a clinical practice resource of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2018; 20:899.</a></li><li><a class="nounderline abstract_t">Ho CY. Genetics and clinical destiny: improving care in hypertrophic cardiomyopathy. Circulation 2010; 122:2430.</a></li><li><a class="nounderline abstract_t">Landstrom AP, Ackerman MJ. Mutation type is not clinically useful in predicting prognosis in hypertrophic cardiomyopathy. Circulation 2010; 122:2441.</a></li><li><a class="nounderline abstract_t">Andersen PS, Havndrup O, Hougs L, et al. Diagnostic yield, interpretation, and clinical utility of mutation screening of sarcomere encoding genes in Danish hypertrophic cardiomyopathy patients and relatives. Hum Mutat 2009; 30:363.</a></li><li><a class="nounderline abstract_t">Maron BJ, Yeates L, Semsarian C. Clinical challenges of genotype positive (+)-phenotype negative (-) family members in hypertrophic cardiomyopathy. Am J Cardiol 2011; 107:604.</a></li><li><a class="nounderline abstract_t">Maron BJ, Zipes DP. Introduction: eligibility recommendations for competitive athletes with cardiovascular abnormalities-general considerations. J Am Coll Cardiol 2005; 45:1318.</a></li><li><a class="nounderline abstract_t">Maron BJ, Seidman JG, Seidman CE. Proposal for contemporary screening strategies in families with hypertrophic cardiomyopathy. J Am Coll Cardiol 2004; 44:2125.</a></li><li><a class="nounderline abstract_t">Fananapazir L, Epstein ND. Genotype-phenotype correlations in hypertrophic cardiomyopathy. Insights provided by comparisons of kindreds with distinct and identical beta-myosin heavy chain gene mutations. Circulation 1994; 89:22.</a></li><li><a class="nounderline abstract_t">Geisterfer-Lowrance AA, Kass S, Tanigawa G, et al. A molecular basis for familial hypertrophic cardiomyopathy: a beta cardiac myosin heavy chain gene missense mutation. Cell 1990; 62:999.</a></li><li><a class="nounderline abstract_t">Epstein ND, Cohn GM, Cyran F, Fananapazir L. Differences in clinical expression of hypertrophic cardiomyopathy associated with two distinct mutations in the beta-myosin heavy chain gene. A 908Leu----Val mutation and a 403Arg----Gln mutation. Circulation 1992; 86:345.</a></li><li><a class="nounderline abstract_t">Jensen MK, Havndrup O, Christiansen M, et al. Penetrance of hypertrophic cardiomyopathy in children and adolescents: a 12-year follow-up study of clinical screening and predictive genetic testing. Circulation 2013; 127:48.</a></li><li><a class="nounderline abstract_t">Maron MS. A paradigm shift in our understanding of the development of the hypertrophic cardiomyopathy phenotype?: not so fast! Circulation 2013; 127:10.</a></li><li><a class="nounderline abstract_t">Arad M, Seidman JG, Seidman CE. Phenotypic diversity in hypertrophic cardiomyopathy. Hum Mol Genet 2002; 11:2499.</a></li><li><a class="nounderline abstract_t">Marian AJ, Roberts R. To screen or not is not the question--it is when and how to screen. Circulation 2003; 107:2171.</a></li><li><a class="nounderline abstract_t">Kelly M, Semsarian C. Multiple mutations in genetic cardiovascular disease: a marker of disease severity? Circ Cardiovasc Genet 2009; 2:182.</a></li><li><a class="nounderline abstract_t">Neubauer S, Kolm P, Ho CY, et al. Distinct Subgroups in Hypertrophic Cardiomyopathy in the NHLBI HCM Registry. J Am Coll Cardiol 2019; 74:2333.</a></li><li><a class="nounderline abstract_t">Weissler-Snir A, Adler A, Williams L, et al. Prevention of sudden death in hypertrophic cardiomyopathy: bridging the gaps in knowledge. Eur Heart J 2017; 38:1728.</a></li><li><a class="nounderline abstract_t">Ackerman MJ, VanDriest SL, Ommen SR, et al. Prevalence and age-dependence of malignant mutations in the beta-myosin heavy chain and troponin T genes in hypertrophic cardiomyopathy: a comprehensive outpatient perspective. J Am Coll Cardiol 2002; 39:2042.</a></li><li><a class="nounderline abstract_t">Marian AJ. On genetic and phenotypic variability of hypertrophic cardiomyopathy: nature versus nurture. J Am Coll Cardiol 2001; 38:331.</a></li><li><a class="nounderline abstract_t">Maron BJ, Maron MS, Semsarian C. Double or compound sarcomere mutations in hypertrophic cardiomyopathy: a potential link to sudden death in the absence of conventional risk factors. Heart Rhythm 2012; 9:57.</a></li></ol></div><div id="topicVersionRevision">Topic 4956 Version 34.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11273720" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : The molecular genetic basis for hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12711218" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Hypertrophic cardiomyopathy:a paradigm for myocardial energy depletion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21787999" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : HRS/EHRA expert consensus statement on the state of genetic testing for the channelopathies and cardiomyopathies this document was developed as a partnership between the Heart Rhythm Society (HRS) and the European Heart Rhythm Association (EHRA).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/33215931" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : 2020 AHA/ACC Guideline for the Diagnosis and Treatment of Patients With Hypertrophic Cardiomyopathy: A Report of the American College of Cardiology/American Heart Association Joint Committee on Clinical Practice Guidelines.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26671970" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Application of Genetic Testing in Hypertrophic Cardiomyopathy for Preclinical Disease Detection.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30681346" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Evaluating the Clinical Validity of Hypertrophic Cardiomyopathy Genes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12707239" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Hypertrophic cardiomyopathy: distribution of disease genes, spectrum of mutations, and implications for a molecular diagnosis strategy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8250038" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Independent origin of identical beta cardiac myosin heavy-chain mutations in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7729014" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : A de novo mutation in alpha-tropomyosin that causes hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17560888" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Mutations in the genes for sarcomeric proteins in Japanese patients with onset sporadic hypertrophic cardiomyopathy after age 40 years.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18154965" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : Prevalence of fabry disease in a cohort of 508 unrelated patients with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21890869" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Prevalence of Anderson-Fabry disease in patients with hypertrophic cardiomyopathy: the European Anderson-Fabry Disease survey.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21339642" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : MEK-ERK pathway modulation ameliorates disease phenotypes in a mouse model of Noonan syndrome associated with the Raf1(L613V) mutation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21339643" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Rapamycin reverses hypertrophic cardiomyopathy in a mouse model of LEOPARD syndrome-associated PTPN11 mutation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12642359" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Mutations in the human muscle LIM protein gene in families with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15673802" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Glycogen storage diseases presenting as hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21382904" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Genetic testing for potentially lethal, highly treatable inherited cardiomyopathies/channelopathies in clinical practice.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18414213" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : ACMG recommendations for standards for interpretation and reporting of sequence variations: Revisions 2007.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22796258" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Genetics of hypertrophic cardiomyopathy after 20 years: clinical perspectives.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27532831" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Genetic Misdiagnoses and the Potential for Health Disparities.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28082330" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Defining the genetic architecture of hypertrophic cardiomyopathy: re-evaluating the role of non-sarcomeric genes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15819282" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Sarcomeric genotyping in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15358028" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Comprehensive analysis of the beta-myosin heavy chain gene in 389 unrelated patients with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15519027" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Myosin binding protein C mutations and compound heterozygosity in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15607392" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Frequency and clinical expression of cardiac troponin I mutations in 748 consecutive families with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18403758" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Shared genetic causes of cardiac hypertrophy in children and adults.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20031618" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Prevalence of sarcomere protein gene mutations in preadolescent children with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15945527" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Yield of genetic testing in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29661763" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Outcomes of Contemporary Family Screening in Hypertrophic Cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2917389" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Hypertrophic cardiomyopathy in the elderly. Distinctions from the young based on cardiac shape.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2909578" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Elderly patients with hypertrophic cardiomyopathy: a subset with distinctive left ventricular morphology and progressive clinical course late in life.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7731997" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Structural interpretation of the mutations in the beta-cardiac myosin that have been implicated in familial hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8541871" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Familial hypertrophic cardiomyopathy: a genetic model of cardiac hypertrophy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8294404" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Heterologous expression of a cardiomyopathic myosin that is defective in its actin interaction.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8290568" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Functional analysis of myosin missense mutations in familial hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8981935" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : Functional analysis of the mutations in the human cardiac beta-myosin that are responsible for familial hypertrophic cardiomyopathy. Implication for the clinical outcome.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8675696" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Altered cardiac troponin T in vitro function in the presence of a mutation implicated in familial hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8958207" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Expression and functional assessment of a truncated cardiac troponin T that causes hypertrophic cardiomyopathy. Evidence for a dominant negative action.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15550524" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : Hypertrophic cardiomyopathy-related beta-myosin mutations cause highly variable calcium sensitivity with functional imbalances among individual muscle cells.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12642349" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Genetic clues to disease pathways in hypertrophic and dilated cardiomyopathies.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12767664" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Hypertrophic cardiomyopathy due to sarcomeric gene mutations is characterized by impaired energy metabolism irrespective of the degree of hypertrophy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11583900" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Expression profiling of cardiac genes in human hypertrophic cardiomyopathy: insight into the pathogenesis of phenotypes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31000001" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : Moving Beyond the Sarcomere to Explain Heterogeneity in Hypertrophic Cardiomyopathy: JACC Review Topic of the Week.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29655825" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : Hypertrophic Cardiomyopathy: Clinical Update.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27912983" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : Hypertrophic obstructive cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18957093" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : Adverse events in families with hypertrophic or dilated cardiomyopathy and mutations in the MYBPC3 gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9562578" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Mutations in the gene for cardiac myosin-binding protein C and late-onset familial hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7493026" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : Cardiac myosin binding protein-C gene splice acceptor site mutation is associated with familial hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9048664" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : Organization and sequence of human cardiac myosin binding protein C gene (MYBPC3) and identification of mutations predicted to produce truncated proteins in familial hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9218526" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : Novel splice donor site mutation in the cardiac myosin-binding protein-C gene in familial hypertrophic cardiomyopathy. Characterization Of cardiac transcript and protein.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10736283" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : A newly created splice donor site in exon 25 of the MyBP-C gene is responsible for inherited hypertrophic cardiomyopathy with incomplete disease penetrance.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11499719" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : Spectrum of clinical phenotypes and gene variants in cardiac myosin-binding protein C mutation carriers with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12628722" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : A novel missense mutation in the myosin binding protein-C gene is responsible for hypertrophic cardiomyopathy with left ventricular dysfunction and dilation in elderly patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9631872" id="rid53" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>54 : Clinical features and prognostic implications of familial hypertrophic cardiomyopathy related to the cardiac myosin-binding protein C gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11815426" id="rid54" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>55 : Sarcomere protein gene mutations in hypertrophic cardiomyopathy of the elderly.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11499718" id="rid55" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>56 : Development of left ventricular hypertrophy in adults in hypertrophic cardiomyopathy caused by cardiac myosin-binding protein C gene mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19151713" id="rid56" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>57 : A common MYBPC3 (cardiac myosin binding protein C) variant associated with cardiomyopathies in South Asia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11847170" id="rid57" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>58 : Genetic polymorphisms in the renin-angiotensin-aldosterone system associated with expression of left ventricular hypertrophy in hypertrophic cardiomyopathy: a study of five polymorphic genes in a family with a disease causing mutation in the myosin binding protein C gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8335820" id="rid58" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>59 : Left ventricular hypertrophy and morphology in familial hypertrophic cardiomyopathy associated with mutations of the beta-myosin heavy chain gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10065021" id="rid59" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>60 : Coexistence of mitochondrial DNA and beta myosin heavy chain mutations in hypertrophic cardiomyopathy with late congestive heart failure.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12860912" id="rid60" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>61 : Prevalence and spectrum of thin filament mutations in an outpatient referral population with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7898523" id="rid61" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>62 : Mutations in the genes for cardiac troponin T and alpha-tropomyosin in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10525521" id="rid62" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>63 : A new mutation of the cardiac troponin T gene causing familial hypertrophic cardiomyopathy without left ventricular hypertrophy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11560853" id="rid63" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>64 : Hypertrophic cardiomyopathy: histopathological features of sudden death in cardiac troponin T disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11779518" id="rid64" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>65 : A novel mutation Lys273Glu in the cardiac troponin T gene shows high degree of penetrance and transition from hypertrophic to dilated cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10931807" id="rid65" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>66 : Clinical features of hypertrophic cardiomyopathy caused by a Lys183 deletion mutation in the cardiac troponin I gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17599605" id="rid66" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>67 : Prevalence, clinical significance, and genetic basis of hypertrophic cardiomyopathy with restrictive phenotype.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12531876" id="rid67" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>68 : Idiopathic restrictive cardiomyopathy is part of the clinical expression of cardiac troponin I mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11136687" id="rid68" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>69 : Hypertrophic cardiomyopathy caused by a novel alpha-tropomyosin mutation (V95A) is associated with mild cardiac phenotype, abnormal calcium binding to troponin, abnormal myosin cycling, and poor prognosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9060904" id="rid69" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>70 : Clinical features of hypertrophic cardiomyopathy caused by mutation of a "hot spot" in the alpha-tropomyosin gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12473556" id="rid70" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>71 : Prevalence and severity of "benign" mutations in the beta-myosin heavy chain, cardiac troponin T, and alpha-tropomyosin genes in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8673105" id="rid71" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>72 : Mutations in either the essential or regulatory light chains of myosin are associated with a rare myopathy in human heart and skeletal muscle.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11174330" id="rid72" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>73 : Different expressivity of a ventricular essential myosin light chain gene Ala57Gly mutation in familial hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15313943" id="rid73" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>74 : Prevalence of Fabry disease in female patients with late-onset hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11914245" id="rid74" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>75 : Prevalence of Anderson-Fabry disease in male patients with late onset hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12479519" id="rid75" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>76 : Genetic causes of inherited cardiac hypertrophy: Robert L. Frye Lecture.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7657794" id="rid76" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>77 : Familial Hypertrophic cardiomyopathy with Wolff-Parkinson-White syndrome maps to a locus on chromosome 7q3.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11371514" id="rid77" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>78 : Mutations in the gamma(2) subunit of AMP-activated protein kinase cause familial hypertrophic cardiomyopathy: evidence for the central role of energy compromise in disease pathogenesis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11827995" id="rid78" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>79 : Constitutively active AMP kinase mutations cause glycogen storage disease mimicking hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15766830" id="rid79" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>80 : Adenosine monophosphate-activated protein kinase disease mimicks hypertrophic cardiomyopathy and Wolff-Parkinson-White syndrome: natural history.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16275868" id="rid80" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>81 : Increased alpha2 subunit-associated AMPK activity and PRKAG2 cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32646569" id="rid81" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>82 : Clinical Features and Natural History of PRKAG2 Variant Cardiac Glycogenosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16144992" id="rid82" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>83 : Danon disease as an underrecognized cause of hypertrophic cardiomyopathy in children.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12084876" id="rid83" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>84 : Clinicopathological features of genetically confirmed Danon disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19318653" id="rid84" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>85 : Clinical outcome and phenotypic expression in LAMP2 cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16087648" id="rid85" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>86 : Gene-specific modifying effects of pro-LVH polymorphisms involving the renin-angiotensin-aldosterone system among 389 unrelated patients with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9140839" id="rid86" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>87 : The influence of the angiotensin I converting enzyme genotype in familial hypertrophic cardiomyopathy varies with the disease gene mutation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7671365" id="rid87" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>88 : Angiotensin-I converting enzyme genotypes and left ventricular hypertrophy in patients with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7484741" id="rid88" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>89 : Angiotensin-converting enzyme gene polymorphism in Japanese patients with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9023164" id="rid89" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>90 : Angiotensinogen gene polymorphism in Japanese patients with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25173338" id="rid90" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>91 : 2014 ESC Guidelines on diagnosis and management of hypertrophic cardiomyopathy: the Task Force for the Diagnosis and Management of Hypertrophic Cardiomyopathy of the European Society of Cardiology (ESC).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29567486" id="rid91" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>92 : Genetic Evaluation of Cardiomyopathy-A Heart Failure Society of America Practice Guideline.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29904160" id="rid92" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>93 : Genetic evaluation of cardiomyopathy: a clinical practice resource of the American College of Medical Genetics and Genomics (ACMG).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21135371" id="rid93" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>94 : Genetics and clinical destiny: improving care in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21135372" id="rid94" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>95 : Mutation type is not clinically useful in predicting prognosis in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19035361" id="rid95" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>96 : Diagnostic yield, interpretation, and clinical utility of mutation screening of sarcomere encoding genes in Danish hypertrophic cardiomyopathy patients and relatives.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21185001" id="rid96" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>97 : Clinical challenges of genotype positive (+)-phenotype negative (-) family members in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15837280" id="rid97" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>98 : Introduction: eligibility recommendations for competitive athletes with cardiovascular abnormalities-general considerations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15582308" id="rid98" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>99 : Proposal for contemporary screening strategies in families with hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8281650" id="rid99" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>100 : Genotype-phenotype correlations in hypertrophic cardiomyopathy. Insights provided by comparisons of kindreds with distinct and identical beta-myosin heavy chain gene mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1975517" id="rid100" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>101 : A molecular basis for familial hypertrophic cardiomyopathy: a beta cardiac myosin heavy chain gene missense mutation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/1638703" id="rid101" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>102 : Differences in clinical expression of hypertrophic cardiomyopathy associated with two distinct mutations in the beta-myosin heavy chain gene. A 908Leu----Val mutation and a 403Arg----Gln mutation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23197161" id="rid102" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>103 : Penetrance of hypertrophic cardiomyopathy in children and adolescents: a 12-year follow-up study of clinical screening and predictive genetic testing.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23283855" id="rid103" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>104 : A paradigm shift in our understanding of the development of the hypertrophic cardiomyopathy phenotype?: not so fast!</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12351586" id="rid104" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>105 : Phenotypic diversity in hypertrophic cardiomyopathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12732591" id="rid105" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>106 : To screen or not is not the question--it is when and how to screen.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20031583" id="rid106" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>107 : Multiple mutations in genetic cardiovascular disease: a marker of disease severity?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31699273" id="rid107" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>108 : Distinct Subgroups in Hypertrophic Cardiomyopathy in the NHLBI HCM Registry.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27371714" id="rid108" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>109 : Prevention of sudden death in hypertrophic cardiomyopathy: bridging the gaps in knowledge.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12084606" id="rid109" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>110 : Prevalence and age-dependence of malignant mutations in the beta-myosin heavy chain and troponin T genes in hypertrophic cardiomyopathy: a comprehensive outpatient perspective.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11499720" id="rid110" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>111 : On genetic and phenotypic variability of hypertrophic cardiomyopathy: nature versus nurture.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21839045" id="rid111" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>112 : Double or compound sarcomere mutations in hypertrophic cardiomyopathy: a potential link to sudden death in the absence of conventional risk factors.</p>
</a>
</div>
</div>
</div>
</div>
</section>
</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->
<!-- End Footer Area -->
<!-- Back to top -->
<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->
<!-- Prpper JS -->
<!-- Bootstrap Min JS -->
<!-- Classy Nav Min Js -->
<!-- Owl Carousel Min Js -->
<!-- Magnific Popup JS -->
<!-- CounterUp JS -->
<!-- Waypoints JS -->
<!-- Form Validator Min JS -->
<!-- Contact Form Min JS -->
<!-- Main JS -->
<!-- Global site tag (gtag.js) - Google Analytics -->
</body>
</html>
