<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Familial amyotrophic lateral sclerosis</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Familial amyotrophic lateral sclerosis</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Familial amyotrophic lateral sclerosis</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Authors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Leo McCluskey, MD, MBE</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Shafeeq S Ladha, MD</a></dd><dt><span> </span>Section Editors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Jeremy M Shefner, MD, PhD</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Benjamin A Raby, MD, MPH</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Richard P Goddeau, Jr, DO, FAHA</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Jul 05, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Amyotrophic lateral sclerosis (ALS) is one specific type of the more general group of motor neuron diseases. These disorders variably affect motor neurons located in the anterior (ventral) horn regions of the spinal cord, the cranial nerve motor nuclei in the pons and medulla, and the frontal cortex. Familial ALS accounts for 5 to 10 percent of all ALS cases.</p><p>ALS is a relentlessly progressive neurodegenerative disorder that causes muscle weakness, disability, and eventually death. The hallmark of ALS is the combination of upper motor neuron and lower motor neuron involvement. The lower motor neuron findings of weakness, atrophy, and fasciculations are a direct consequence of muscle denervation, hence the term "amyotrophic." The upper motor neuron findings of hyperreflexia and spasticity result from degeneration of the lateral corticospinal tracts in the spinal cord, which are gliotic and hardened to palpation at autopsy, hence the term "lateral sclerosis."</p><p>This topic review will discuss familial ALS. The epidemiology and clinical features of ALS are discussed separately. (See  <a class="medical medical_review" href="/d/html/5156.html" rel="external">"Epidemiology and pathogenesis of amyotrophic lateral sclerosis"</a> and  <a class="medical medical_review" href="/d/html/5136.html" rel="external">"Clinical features of amyotrophic lateral sclerosis and other forms of motor neuron disease"</a>.)</p><p class="headingAnchor" id="H136713437"><span class="h1">OVERVIEW</span><span class="headingEndMark"> — </span>Familial ALS accounts for approximately 5 to 10 percent of all ALS cases [<a href="#rid1">1,2</a>]. More than 20 causative genes have been identified. Among these, the two most common are <em>C9ORF72</em> and <em>SOD1</em>, which together make up nearly 50 percent of familial ALS.</p><p>Familial ALS is phenotypically and genetically heterogeneous. Although most familial ALS cases follow an autosomal dominant inheritance pattern, recessive and X-linked forms have been described. The nomenclature of ALS1 through ALS26 arises from the order of their discovery and not from any particular clinical classification. Many of the genetic variants have been described only in one or two families, and genotype-phenotype correlation is stronger for some forms of genetic ALS than for others.</p><p>In most individual cases, it may be difficult to determine on clinical grounds alone if ALS is familial or sporadic, especially at the onset of disease. A positive family history of ALS, other neurodegenerative disorders, and dementia or the presence of atypical features such as young age of onset or sensory loss, should alert the clinician to the possibility of familial ALS. However, the absence of a positive family history does not entirely exclude a diagnosis of familial ALS.</p><p class="headingAnchor" id="H1567067046"><span class="h1">GENETIC TESTING AND COUNSELING</span><span class="headingEndMark"> — </span>Genetic testing for specific ALS-causing pathogenic genetic variants is typically used for specific indications. These include the following  (<a class="graphic graphic_table graphicRef61384" href="/d/graphic/61384.html" rel="external">table 1</a>):</p><p class="bulletIndent1"><span class="glyph">●</span>Determining treatment eligibility with the antisense oligonucleotide <a class="drug drug_general" data-topicid="141342" href="/d/drug information/141342.html" rel="external">tofersen</a> for patients with <em>SOD1</em>-associated ALS. (See <a class="local">'ALS1 (SOD1 gene)'</a> below.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Providing risk information and subsequent counseling for unaffected at-risk family members of an affected relative with an established genetic cause  (<a class="graphic graphic_table graphicRef61384" href="/d/graphic/61384.html" rel="external">table 1</a>) is the most common use of genetic testing for ALS. (See  <a class="medical medical_review" href="/d/html/103304.html" rel="external">"Genetic counseling: Family history interpretation and risk assessment"</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Testing to confirm the diagnosis of ALS is used in rare cases of early-stage disease (ie, clinically suspected and clinically possible ALS by El Escorial criteria), with or without a family history of ALS. The discovery of an ALS-related gene can change the diagnosis to clinically definite laboratory-supported familial ALS. In cases where the diagnosis is clear, genetic testing has little or no additional value in making the diagnosis of ALS and will not impact medical management or treatment options. (See  <a class="medical medical_review" href="/d/html/5138.html" rel="external">"Diagnosis of amyotrophic lateral sclerosis and other forms of motor neuron disease", section on 'Diagnosis'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genetic testing may play a role in making an alternative diagnosis of Kennedy syndrome and late-onset Tay-Sachs disease, two genetic disorders that can mimic some of the clinical features of ALS. This is discussed separately. (See  <a class="medical medical_review" href="/d/html/5138.html" rel="external">"Diagnosis of amyotrophic lateral sclerosis and other forms of motor neuron disease", section on 'Genetic testing'</a> and  <a class="medical medical_review" href="/d/html/5138.html" rel="external">"Diagnosis of amyotrophic lateral sclerosis and other forms of motor neuron disease", section on 'Differential diagnosis'</a>.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genetic testing may be a prerequisite for participation in a clinical trial involving specific genetic subtypes of ALS, such as <em>C9ORF72</em>- or <em>SOD1</em>-associated ALS.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Genetic testing may provide limited prognostic information, such as predicting rapid progression in individuals with the A4V <em>SOD1</em> pathogenic genetic variant. However, genetic testing should <strong>not</strong> be used for this purpose, because the phenotypic variability seen in familial ALS makes such predictions unreliable at the level of individual cases. (See  <a class="medical medical_review" href="/d/html/5136.html" rel="external">"Clinical features of amyotrophic lateral sclerosis and other forms of motor neuron disease", section on 'Clinical patterns of progression'</a>.)</p><p></p><p>A glossary of genetic terms is available in a table  (<a class="graphic graphic_table graphicRef72267" href="/d/graphic/72267.html" rel="external">table 2</a>) and, in an expanded form, in a separate topic. (See  <a class="medical medical_review" href="/d/html/2898.html" rel="external">"Genetics: Glossary of terms"</a>.)</p><p class="headingAnchor" id="H3027894854"><span class="h2">Recurrence risk within a family</span><span class="headingEndMark"> — </span>For familial ALS, the recurrence risk of ALS in a given family (ie, whether or not another family member will develop ALS) is estimated primarily by determining the inheritance pattern. The mode of inheritance in familial ALS can be autosomal dominant, autosomal recessive, or (rarely) X-linked. Autosomal dominant inheritance is by far the most common pattern of inheritance observed in hereditary ALS.</p><p>The mode of inheritance is determined by analysis of the family history and the results of genetic testing, if available. Thus, a detailed pedigree covering at least three generations should be obtained that includes the affected individual's parents, siblings, children, grandparents, aunts and uncles, nieces and nephews, and first cousins. Information to be collected should include current age, health status, age at death, cause of death, medical diagnoses, history of dementia, and any possible symptoms of ALS that may have gone undiagnosed.</p><p>Of note, the family history may be falsely negative due to a number of factors, including failure to recognize the disorder in family members, death from other causes prior to ALS symptom onset, non-paternity, or late onset or reduced penetrance of the disease in affected family members.</p><p>With sporadic ALS, it is generally believed that the recurrence risk of ALS is not significantly higher than the risk in the general population when there is only one family member affected with ALS. The recurrence risk would increase should another family member be diagnosed with ALS. The recurrence risk may be as high as 50 percent if multiple family members are affected with ALS. However, the exact recurrence risk for ALS cannot be determined in the absence of a positive genetic test.</p><p class="headingAnchor" id="H2351135322"><span class="h2">Who should be tested?</span><span class="headingEndMark"> — </span>Genetic testing for at least <em>SOD1</em> and <em>C9ORF72</em> is encouraged in all patients, as genotype-specific therapies are in clinical trials and pathogenic genetic variants are occasionally identified in patients without a family history. An updated list of clinical trials available to patients with specific ALS genotypes can be found online at <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fclinicaltrials.gov%2F&amp;token=nbsXe3i4qfUea5RCPRHtOc5ko1zksgMm05jF47%2FUSB3l0tvt%2BGAm3dUe1JoUg9nw&amp;TOPIC_ID=5176" target="_blank">ClinicalTrials.gov</a>. (See <a class="local">'C9ORF72 gene'</a> below and  <a class="medical medical_review" href="/d/html/5128.html" rel="external">"Disease-modifying treatment of amyotrophic lateral sclerosis", section on 'Experimental therapy'</a>.)</p><p>While any willing individual with ALS can have testing, a genetic cause is most likely to be found in those who have a family history of autosomal dominant disease or juvenile-onset ALS with suspected autosomal recessive inheritance. Genetic testing may also be useful in individuals with familial ALS who lack a clear family history consistent with an autosomal dominant disease, a situation that may occur because of reduced penetrance or limited information about important family members. However, the likelihood of identifying a pathogenic variant in a gene associated with ALS is reduced in this setting. As noted earlier, the most common use of genetic testing for ALS is to provide risk information and subsequent counseling for unaffected at-risk family members.</p><p>In patients with a family history suggesting autosomal dominant inheritance, genetic testing can provide prognostic information. As examples, the finding of a <em>C9ORF72</em> pathogenic variant (see <a class="local">'C9ORF72 gene'</a> below) implies a high risk for the development of comorbid frontotemporal dementia (FTD) with ALS, while the finding of the A4V <em>SOD1</em> pathogenic variant (see <a class="local">'ALS1 (SOD1 gene)'</a> below) augurs a poor prognosis due to rapid progression and an average life expectancy of only 1.5 years after the onset of symptoms. However, the impact of a positive genetic test on clinical care in patients with ALS is limited, since the treatment options are few and management is mainly symptomatic.</p><p>Individuals with ALS who do not have a positive family history may also pursue genetic testing, as apparently sporadic cases of ALS have been reported to carry a pathogenic genetic variant in one of the genes currently associated with familial ALS, including <em>TARDBP</em>, <em>C9ORF72</em>, <em>SOD1</em>, <em>ANG</em>, <em>FUS</em>, <em>OPTN</em>, and <em>SETX</em>, or in one of the genes likely to be susceptibility factors for ALS, including <em>NEK1</em>, <em>ATXN2</em> and survival of motor neuron 1, telomeric (<em>SMN1</em>). In these cases, the lack of family history suggests either low penetrance or a de novo mutation of a gene associated with ALS. In addition, the absence of family history with apparently sporadic ALS makes it difficult to quantify the risk of ALS for unaffected family members with the same pathogenic variant. (See  <a class="medical medical_review" href="/d/html/5156.html" rel="external">"Epidemiology and pathogenesis of amyotrophic lateral sclerosis", section on 'Genetic susceptibility in sporadic ALS'</a>.)</p><p>Anyone diagnosed with ALS or any adult with a family history of ALS who is considering genetic testing should first meet with a genetic counselor or a physician knowledgeable about familial ALS to discuss the genetic basis of the disease as well as the medical, psychologic, economic, and familial implications. Those pursuing testing should be informed of a number of issues and limitations of genetic testing.</p><p class="bulletIndent1"><span class="glyph">●</span>The predictive value of a positive genetic test can be limited by a number of factors, including lack of population-based data on penetrance (ie, the likelihood of developing disease given inheritance of a disease-causing pathogenic variant) and variable expressivity (ie, phenotypic variations in the way the disease is expressed). Thus, the ability of the genetic test to predict phenotype (ie, the clinical validity) may be limited.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The impact of a positive genetic test for ALS for the purpose of clinical care is essentially nil, since the treatment options are limited and management is mainly symptomatic.</p><p></p><p>Additional concerns that should be addressed by genetic counseling include the following:</p><p class="bulletIndent1"><span class="glyph">●</span>The meaning of a positive or negative test, including the slight possibility of false-positive or -negative results</p><p class="bulletIndent1"><span class="glyph">●</span>The risk of variants of unknown significance</p><p class="bulletIndent1"><span class="glyph">●</span>Psychosocial issues related to family dynamics</p><p class="bulletIndent1"><span class="glyph">●</span>Disclosing genetic information to at-risk family members</p><p class="bulletIndent1"><span class="glyph">●</span>The possibility of genetic discrimination</p><p class="bulletIndent1"><span class="glyph">●</span>Financial risks impacting life, disability, and long-term care insurance</p><p></p><p>These issues are explored in detail separately. (See  <a class="medical medical_review" href="/d/html/2901.html" rel="external">"Genetic testing", section on 'Ethical, legal, and psychosocial issues'</a> and  <a class="medical medical_review" href="/d/html/103304.html" rel="external">"Genetic counseling: Family history interpretation and risk assessment"</a>.)</p><p>In families where a likely pathogenic variant in a gene associated with ALS has not been previously identified, testing is most likely to identify pathogenic variants in the affected family members with ALS, rather than at-risk members. Therefore, it is preferable to begin searching for variants by gene sequencing in affected members, and then subsequently perform targeted genotyping of identified pathogenic variants in potentially at-risk individuals. Once a disease-causing variant has been identified, other family members may choose to be tested for the same pathogenic variant regardless of whether or not they have symptoms. (See <a class="local">'Unaffected family members'</a> below.)</p><p>For ethical reasons, genetic testing is not appropriate for children and adolescents at risk for adult-onset ALS. There are a number of potential risks (eg, lack of autonomy, discrimination, stigmatization, anxiety) associated with the predictive testing of minors, and there is no compelling benefit because preventive treatment is unavailable [<a href="#rid3">3</a>].</p><p class="headingAnchor" id="H1499771311"><span class="h2">How the test is done</span><span class="headingEndMark"> — </span>Genetic testing can be done on a blood, saliva, or tissue sample. The sample is then analyzed for pathogenic variants in the specific gene(s) of interest.</p><p class="bulletIndent1"><span class="glyph">●</span>Full gene sequencing is considered the gold standard for identifying both known and novel variants in a gene and involves analyzing the entire coding region of the gene. Full gene sequencing will detect most pathogenic variants in the tested gene.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Targeted mutation analysis (genotyping) involves testing for a specific pathogenic variant or set of variants in the gene of interest and will only detect pathogenic variants in the tested region(s) of the gene but will not identify all variants in the gene of interest. Targeted mutation analysis is often used for predictive testing in at-risk family members to identify pathogenic variants already known to exist in affected family members (ie, as identified through sequencing).</p><p></p><p class="headingAnchor" id="H4176509920"><span class="h2">Genes to test</span><span class="headingEndMark"> — </span>Clinical genetic testing for inherited (familial) forms of ALS is available to look for causative pathogenic variants in several genes associated with ALS. Approximately 13 to 20 percent of familial ALS is linked to pathogenic variants in the <em>SOD1</em> gene, and approximately 40 percent of these are linked to hexanucleotide repeats in the <em>C9ORF72</em> gene [<a href="#rid4">4</a>]. In addition, pathogenic variants in the <em>FUS</em> gene, <em>ANG</em> gene, <em>TARDBP</em> gene, and <em>FIG4</em> gene may account for as much as 5, 2, 5, and 3 percent of familial ALS, respectively. (See <a class="local">'ALS1 (SOD1 gene)'</a> below and <a class="local">'C9ORF72 gene'</a> below and <a class="local">'ALS6 (FUS gene)'</a> below and <a class="local">'ALS9 (ANG gene)'</a> below and <a class="local">'ALS10 (TARDBP gene)'</a> below and <a class="local">'ALS11 (FIG4 gene)'</a> below.)</p><p>If the patient has juvenile-onset ALS, a multigene panel that includes genetic testing for <em>SPG11</em>, <em>SETX</em>, <em>ALS2, </em>and <em>SPTCL1</em> would be appropriate. (See <a class="local">'Autosomal dominant'</a> below and <a class="local">'Autosomal recessive'</a> below.)</p><p>Multigene genetic testing panels are commercially available for all the above listed genes as well as other less common genes. However, not all pathogenic genetic variants associated with ALS have been identified. Commercial testing for some patients with a genetic susceptibility to ALS may be negative. Additionally, multigene panels may also include other genetic variants not associated with ALS. These factors should be addressed when discussing genetic testing and when revealing the results of genetic testing to patients. (See <a class="local">'Interpretation'</a> below.)</p><p class="headingAnchor" id="H3134189348"><span class="h2">Laboratories offering testing</span><span class="headingEndMark"> — </span>Some laboratories offer free genetic testing panels for ALS and neurodegenerative disease, which has made obtaining this testing much easier and has largely removed the financial barrier to genetic testing in ALS. Information on laboratories offering genetic testing for ALS is available online at the <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgtr%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2Fy3%2BLj0shalBW%2BL1ZyL4tryNPgji1PaJ6Kbd%2Fo1hE9A6&amp;TOPIC_ID=5176" target="_blank">Genetic Testing Registry</a>. The Genetic Testing Registry website is a publicly funded central location for voluntary submission of medical genetics test information by providers.</p><p class="headingAnchor" id="H2525270934"><span class="h2">Interpretation</span><span class="headingEndMark"> — </span>To properly interpret genetic test results for ALS genes, it is necessary to know whether the person who is being tested has ALS and whether a pathogenic variant in the family has already been identified.</p><p class="headingAnchor" id="H1225639173"><span class="h3">Patients with ALS</span><span class="headingEndMark"> — </span>For individuals with ALS who do <strong>not</strong> already have a known pathogenic genetic variant, there are three possible types of test results as follows:</p><p class="bulletIndent1"><span class="glyph">●</span>A positive test result means that the individual has a disease-causing (pathogenic) variant or variants and that the cause of ALS observed in the family has been identified. A positive result also further defines the risk to develop ALS for other family members. Interpretation of the ALS risk to other family members depends on the mode of inheritance of the gene with the variant(s). (See <a class="local">'Recurrence risk within a family'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>A negative test result means that the individual does not have a pathogenic variant in the tested gene or genes. However, a negative result does not rule out entirely the possibility of a genetic cause for the following reasons:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Not all genes may have been tested. For example, if only testing for <em>SOD1</em> was ordered, a negative result would not rule out the possibility of a pathogenic variant in <em>TARDBP</em>, <em>FUS</em>, <em>FIG4</em>, or <em>ANG</em>.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Not all base positions of a targeted gene may have been adequately tested. For sequencing, this may occur if the assay was designed to target coding regions only, ignoring functional noncoding regions (such as regulatory or intronic regions). Also, gene coverage may be incomplete due to assay failure. In these instances, the individual may harbor pathogenic variants that are simply missed. In the context of targeted genotyping (testing of a limited number of variants) one can only confidently report a negative result for those specific variants tested. For example, a negative result from genotyping of the most common <em>SOD1</em> pathogenic variant in North America, A4V, only excludes that variant but does not rule out any of the other 139 known <em>SOD1</em> variants.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Not all genetic causes of ALS have been identified.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>A variant of unknown significance (VUS) result means that a change was found in the genetic sequence of the gene, but the laboratory cannot yet make a clear determination as to whether it is a pathogenic variant or a benign polymorphism. Benign polymorphisms are common variations in DNA that do not cause disease.</p><p></p><p>When a VUS is identified, it is suggested that the variant be tracked through the family, which involves testing multiple affected and unaffected family members. However, tracking a variant through a family for an adult-onset condition can be challenging; since ALS typically affects older individuals, family members from the preceding generation may already be deceased and unable to contribute a DNA sample. In addition, when an unaffected family member tests positive for the variant, it is not clear whether this individual will develop ALS in the future or if the variant is benign. It can take years to determine whether or not a VUS is pathogenic.</p><p>For individuals with a diagnosis of ALS who are part of a family with a known familial pathogenic variant, a positive test result confirms that the individual did inherit the known pathogenic genetic variant and supports that variant as the cause of ALS for the individual. A negative result means that the individual did not inherit the known pathogenic variant; therefore, the cause of ALS for the individual is unknown. In this scenario, the individual would be considered to have sporadic ALS. This is known as a phenocopy. A second sample may be sent for genetic testing to confirm the result.</p><p class="headingAnchor" id="H3296696424"><span class="h3">Unaffected family members</span><span class="headingEndMark"> — </span>Unaffected family members considering genetic testing for ALS are often concerned about possible symptoms of ALS. In such cases, a neurologic evaluation to look for evidence of ALS and a discussion with a trained genetic counselor can help alleviate concern.</p><p class="headingAnchor" id="H1162358001"><span class="h4">Known family genetic variant</span><span class="headingEndMark"> — </span>Testing an asymptomatic person who is at risk for inheriting a known pathogenic genetic variant is called presymptomatic (or predictive) testing. For presymptomatic testing, a variant must have been first identified in the family so that the clinician knows the appropriate test to order. The possible test results are as follows:</p><p class="bulletIndent1"><span class="glyph">●</span>A positive test result means that the individual has a pathogenic variant and is at risk to develop ALS. The probability that the individual will develop symptoms of ALS will depend on factors such as penetrance and expressivity. Penetrance and expressivity information is available for some genes, such as <em>C9ORF72</em>, <em>SOD1</em>, and <em>FUS</em>, but is limited for the other ALS genes. In general, the penetrance of most of the autosomal dominantly inherited ALS genes is above 90 percent.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>A negative test result means that the individual does not have a pathogenic variant in the tested gene and is not at risk for the familial form of ALS.</p><p></p><p>Family members may have different opinions about whether or not to pursue presymptomatic testing, as this is a very personal decision that can be confusing and emotionally difficult. Meeting with a trained genetic counselor and a psychologic evaluation are recommended to help the individual throughout the testing process; multiple counseling sessions are optimal, including pre-decision, pretest, and post-test counseling [<a href="#rid5">5</a>].</p><p class="headingAnchor" id="H2917191696"><span class="h4">No known family genetic variant</span><span class="headingEndMark"> — </span>Presymptomatic (or predictive) genetic testing for an unaffected individual with a family history of ALS but no known familial pathogenic variant is <strong>not</strong> recommended, because it is difficult to determine the appropriate test to order and interpret the meaning of negative or inconclusive genetic test results.</p><p class="bulletIndent1"><span class="glyph">●</span>A negative test result would mean that the individual did <strong>not</strong> have a pathogenic variant in the tested gene or genes. However, this finding would <strong>not</strong> rule out the possibility that they have inherited a genetic form of ALS. In this situation there are three possible scenarios to explain the history of ALS observed in the family:</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>It is due to a pathogenic variant in one of the tested genes, and the individual did not inherit the variant. The patient is not at an increased risk of developing ALS.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>It is due to an unknown gene, and it is unknown if the individual inherited the pathogenic variant. The patient may be at an increased risk of developing ALS.</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>It is due to environmental factors and is not genetic.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>A VUS means that a change was found in the genetic sequence of the gene, but the laboratory cannot make a clear determination as to whether it is a pathogenic variant or a benign polymorphism. This individual may or may not develop ALS in the future.</p><p></p><p class="headingAnchor" id="H1401328410"><span class="h2">Differences between clinical and research genetic testing</span><span class="headingEndMark"> — </span>The purpose of clinical genetic testing is primarily for diagnosis or presymptomatic genetic testing  (<a class="graphic graphic_table graphicRef61384" href="/d/graphic/61384.html" rel="external">table 1</a>). The testing is performed in an accredited laboratory. A report of the results is generated and is part of the patient's official medical record. There is a charge for clinical testing, which may or may not be covered by insurance.</p><p>The purpose of research genetic testing is to discover and to better understand the genetic factors of ALS. Patient identifiers and information remain confidential, and results do not become part of the patient's official medical record. There is no fee when participating in a research study, and generally results are not returned to participants. Many individuals choose to participate in research genetic testing to help advance the ALS field for the benefit of others with or without ALS now and for future generations.</p><p class="headingAnchor" id="H2"><span class="h1">AUTOSOMAL DOMINANT</span><span class="headingEndMark"> — </span>The majority of familial ALS cases follow an autosomal dominant inheritance pattern. In autosomal dominant inheritance, having a pathogenic variant in only one copy of a gene is sufficient for an individual to develop the condition. Typically, the copy of the gene with the pathogenic variant is inherited from a parent (as opposed to de novo mutations, which are uncommon), and most individuals with an autosomal dominant condition will have a positive family history. Each child of a parent with a dominant pathogenic variant has a 50 percent chance of inheriting the copy of the gene with the variant and being at risk to develop ALS.</p><p>Autosomal dominant pathogenic variants in apparently sporadic ALS can occur due to de novo (new) mutations, reduced penetrance, or insufficient family history information. The frequency of de novo mutations in ALS is unknown. The recurrence risk for siblings and children of an apparently sporadic ALS case with an autosomal dominant pathogenic variant depends on whether or not the affected individual has an inherited or a de novo mutation. If the individual with ALS inherited the pathogenic variant from a parent, there is a 50 percent chance the individual's siblings and children will inherit the variant and be at risk to develop ALS. In the case of a de novo mutation, the recurrence risk for siblings would be low, but the recurrence risk for children would be 50 percent.</p><p>The most common autosomal dominantly inherited forms of familial ALS are caused by pathogenic variants in the chromosome 9 open reading frame 72 (<em>C9ORF72</em>) gene (see <a class="local">'C9ORF72 gene'</a> below) and the superoxide dismutase type 1 (<em>SOD1</em>) gene (see <a class="local">'ALS1 (SOD1 gene)'</a> below) described in the following sections.</p><p class="headingAnchor" id="H1808661"><span class="h2">C9ORF72 gene</span><span class="headingEndMark"> — </span>Repeat expansions of the <em>C9ORF72</em> gene on chromosome 9p21 are causally linked to classic ALS, frontotemporal dementia (FTD), and ALS with FTD [<a href="#rid6">6-13</a>]. <em>C9ORF72</em> expansions are the most common cause of familial ALS, accounting for approximately 40 percent of cases in European populations and approximately 2 percent of familial cases in Chinese, Japanese, Korean, and Iranian populations [<a href="#rid10">10,11,14,15</a>]. They also account for approximately 5 percent of nonfamilial (sporadic, or singleton) ALS cases in European populations [<a href="#rid16">16</a>].</p><p>The <em>C9ORF72</em> gene encodes an uncharacterized protein that is highly conserved among species. The hexanucleotide repeat expansion seen in ALS-FTD cases tends to be large, usually in the range of 500 to 3500 repeats, and varies between different tissue types in the same individual. Repeat length does not show a consistent relationship with survival. A positive association has been observed between repeat length and age of onset in some but not all studies [<a href="#rid17">17-19</a>]. A variety of mechanisms are thought to underlie motor neuron death as a result of <em>C9ORF72</em> repeat expansion. These include both toxic gain of function and loss of function mechanisms [<a href="#rid20">20</a>].</p><p>ALS caused by <em>C9ORF72</em> (C9ALS) has a similar natural history compared with sporadic (singleton) ALS. In large cohorts of patients with C9ALS, the median age of onset is approximately 58 years, and median survival is approximately 30 to 36 months from diagnosis [<a href="#rid17">17</a>]. Approximately 50 percent of individuals with pathogenic variants are symptomatic by age 58, and 100 percent are symptomatic by age 80 [<a href="#rid21">21,22</a>]. Limb onset is more common than bulbar onset (54 versus 39 percent). Compared with sporadic ALS, C9ALS is more likely to manifest with comorbid FTD (50 versus 12 percent) [<a href="#rid16">16</a>]. Older age and bulbar onset are associated with worse overall survival [<a href="#rid23">23</a>].</p><p>The cognitive features of ALS and the manifestations of FTD are reviewed separately. (See  <a class="medical medical_review" href="/d/html/5136.html" rel="external">"Clinical features of amyotrophic lateral sclerosis and other forms of motor neuron disease", section on 'Cognitive symptoms'</a> and  <a class="medical medical_review" href="/d/html/5070.html" rel="external">"Frontotemporal dementia: Clinical features and diagnosis"</a>.)</p><p class="headingAnchor" id="H3"><span class="h2">ALS1 (SOD1 gene)</span><span class="headingEndMark"> — </span>Familial ALS1 is linked to pathogenic variants of the copper-zinc (Cu,Zn) <em>SOD1</em> gene localized in chromosome 21q22 [<a href="#rid24">24</a>]. Heterozygous genetic variants in <em>SOD1</em> account for approximately 15 percent of familial ALS cases in European populations and 30 percent of cases in Chinese, Japanese, Korean, and Iranian populations [<a href="#rid14">14</a>]. Most are inherited in an autosomal dominant manner. Approximately 50 percent of individuals with pathogenic variants are symptomatic by age 46, and 90 percent are symptomatic by age 70 [<a href="#rid25">25</a>]. Over 180 variants of the <em>SOD1</em> gene have been reported [<a href="#rid26">26,27</a>]. Variants in the gene are thought to exert their pathogenic effects primarily through a toxic gain of function [<a href="#rid28">28</a>]. The clinical features of ALS1 are variable, as illustrated by the following observations:</p><p class="bulletIndent1"><span class="glyph">●</span>The A4V missense variant is the most prevalent genetic variant in familial ALS1 and accounts for approximately 41 percent of patients with <em>SOD1</em> pathogenic variants [<a href="#rid29">29</a>]. It typically has a rapid progression with an average life expectancy of 1.5 years after the onset of symptoms [<a href="#rid30">30</a>]. Upper motor neuron involvement is either absent or mild, but pathologic abnormalities in systems other than the motor neurons are more frequent [<a href="#rid31">31</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The I113T pathogenic variant [<a href="#rid24">24</a>] is the second most common type, accounting for 16 percent of <em>SOD1</em> pathogenic variants. This variant has variable penetrance that can be low [<a href="#rid32">32</a>]. The mean age of onset is 59 years. The clinical course is variable, ranging from quite rapid progression and death within two years to very slow progression that can span 20 years [<a href="#rid32">32-34</a>]. Both upper and lower motor neuron findings occur. Posterior column involvement has been reported in one patient [<a href="#rid35">35</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Families with the A4T pathogenic variant in the <em>SOD1</em> gene have a variable disease onset and duration. In one large kindred, the age of onset ranged from 32 to 60 years, with a mean of 46 years [<a href="#rid36">36</a>]. Weakness in the legs was the most frequent early symptom, and there was a predominance of lower motor neuron signs. Bulbar and respiratory muscle involvement occurred in all individuals. The mean time from onset of symptoms to death was 14 months. One male with onset at age 37 had a more slowly developing form and was alive but severely affected 76 months after diagnosis. With this one exception, the severity was similar to that of the A4V pathogenic variant [<a href="#rid36">36</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Among 17 patients from three Japanese ALS1 families with the <em>SOD1</em> H46R heterozygous genetic variant, the age of disease onset ranged from 32 to 60 years (average 44.3), the age of death ranged from 46 to older than 88 years (average 59 years), and the mean disease duration was greater than 12 years [<a href="#rid37">37</a>]. The initial symptom was distal leg weakness in all patients. This relatively slow disease progression is in keeping with the clinical observation that bulbar involvement was minimal in these families; only 1 out of the 12 reported patients had tongue atrophy. Dementia was uniformly absent in these patients.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Another report described three members of a familial ALS1 pedigree with a rare A89V <em>SOD1</em> pathogenic variant [<a href="#rid38">38</a>]. This variant is characterized by incomplete penetrance, variable age of onset, and an associated painful sensory neuropathy. In these three patients, the disease onset was late in two and early in one. The early-onset patient was a 15-year-old male who presented with a one-year history of a progressive bilateral symmetric leg and arm weakness. The weakness was associated with sporadic cramps and fasciculations in the limbs and chest muscles but no bulbar involvement. On exam, he had brisk jaw reflexes, weak neck flexion and extension, bilateral weakness and atrophy of the upper and lower extremities, increased tone, hyperactive deep tendon reflexes, and a left Babinski response.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>In a report involving 20 patients, the G93C pathogenic variant was associated with a purely lower motor neuron phenotype and absence of bulbar involvement [<a href="#rid39">39</a>]. The mean age of onset was 46 years and disease progression was slow, with an average survival of nearly 13 years.</p><p></p><p>Rare patients with homozygous pathogenic variants in <em>SOD1</em> have been described with variable phenotypes [<a href="#rid40">40-45</a>]. Homozygous truncating pathogenic variants in <em>SOD1</em> causing complete absence of SOD1 enzyme have been associated with a particularly severe phenotype, including early-onset progressive spastic tetraparesis with little or no evidence of lower motor neuron involvement [<a href="#rid42">42,43</a>].</p><p><a class="drug drug_general" data-topicid="141342" href="/d/drug information/141342.html" rel="external">Tofersen</a> is an antisense oligonucleotide that targets <em>SOD1</em> to reduce pathologic translation and protein expression that was approved by the United States Food and Drug Administration in April 2023 for patients with ALS associated with <em>SOD1</em> variants. Tofersen for ALS is discussed in greater detail separately. (See  <a class="medical medical_review" href="/d/html/5128.html" rel="external">"Disease-modifying treatment of amyotrophic lateral sclerosis", section on 'Tofersen for SOD1-associated ALS'</a>.)</p><p class="headingAnchor" id="H4"><span class="h2">ALS3</span><span class="headingEndMark"> — </span>Familial ALS3 has been linked to the <em>WDR7</em> gene in the 18q21 region [<a href="#rid46">46,47</a>]. The average age of onset was 43 years. Leg onset was most common. Both upper and lower motor neuron signs occurred. Other central nervous system pathways were not involved. The mean life expectancy was five years.</p><p class="headingAnchor" id="H3873327568"><span class="h2">ALS4 (SETX gene)</span><span class="headingEndMark"> — </span>Familial ALS4 is a juvenile form with an average age of onset in the second decade [<a href="#rid48">48-50</a>]. It has been linked to the <em>SETX</em> gene on chromosome 9q34 coding for senataxin, a DNA and RNA helicase. It most commonly begins with fairly symmetric distal weakness and atrophy of the hands and feet. Proximal weakness occurs later. Upper motor neuron signs occur, but bulbar involvement is uncommon. The clinical course is extremely slow. Progression to a nonambulatory status does not occur until the fifth or sixth decades. Minor sensory changes can occur in older individuals.</p><p>Pathologically, the disorder is characterized by loss of upper and lower motor neurons as well as dorsal root ganglion neurons and posterior column fiber loss. Recessive pathogenic variants in <em>SETX</em> have been linked to inherited ataxia, oculomotor apraxia, and peripheral neuropathy [<a href="#rid51">51-54</a>].</p><p class="headingAnchor" id="H6"><span class="h2">ALS6 (FUS gene)</span><span class="headingEndMark"> — </span>Pathogenic variants in the fused in sarcoma (<em>FUS</em>) gene (also known as the <em>TLS</em> gene) on chromosome 16 have been linked to familial ALS6 in multiple families from multiple ethnic groups [<a href="#rid55">55-58</a>]. The phenotype is notable for onset at an average age of 45 years, with an average survival of 33 to 41 months [<a href="#rid56">56,57</a>]. However, there is a wide variation in the duration of symptoms, with one reported patient surviving for 18 years [<a href="#rid57">57</a>]. In a series of 18 patients, the onset was cervical more often than lumbar or bulbar [<a href="#rid56">56</a>]. <em>FUS</em> pathogenic variants have also been identified in a few patients with familial ALS and FTD or parkinsonism [<a href="#rid57">57</a>]. In cases in which pathologic specimens were obtained, there was severe lower motor neuron loss in the spinal cord and to a lesser degree in the brainstem [<a href="#rid56">56</a>].</p><p><em>FUS</em> pathogenic variants are estimated to occur in approximately 3 percent of familial ALS cases in European populations and 6 percent of familial ALS cases in Chinese, Japanese, Korean, and Iranian populations [<a href="#rid14">14</a>]. Both autosomal dominant and recessive inheritance patterns have been described [<a href="#rid55">55</a>].</p><p>The FUS protein is normally found in the nucleus, where it is involved in transcription regulation, RNA splicing, and RNA transport [<a href="#rid55">55,56</a>]. Thus, it is functionally similar to the TAR DNA binding protein (<em>TARDBP</em>) gene described below. Furthermore, cellular expression studies reveal that pathogenic variant forms of FUS protein have aberrant localization in the cytoplasm of neurons, similar to the pathology of <em>TARDBP</em> gene variants in familial ALS. These findings suggest that forms of ALS due to <em>FUS </em>and <em>TARDBP</em> pathogenic variants share a common pathogenic mechanism related to impaired RNA metabolism. (See <a class="local">'ALS10 (TARDBP gene)'</a> below.)</p><p class="headingAnchor" id="H7"><span class="h2">ALS7</span><span class="headingEndMark"> — </span>Familial ALS7 has been linked to chromosome 20p13 in a single family [<a href="#rid59">59</a>]. The mean onset was in the sixth decade, with a mean life expectancy of three years after onset. It is characterized by both upper and lower motor neuron signs.</p><p class="headingAnchor" id="H8"><span class="h2">ALS8 (VAPB gene)</span><span class="headingEndMark"> — </span>Familial ALS8 was initially linked to chromosome 20q13.33 in a single Brazilian family [<a href="#rid60">60</a>]. It is characterized by onset in the third to the fifth decade with manifestations that include cramps, fasciculations, and at times lower motor neuron weakness. Upper motor neuron findings are less common. Some individuals manifest sensory loss, a postural tremor, or both. The course is slowly progressive. The cause is a missense variant form of the VAMP associated protein B and C (<em>VAPB</em>) gene [<a href="#rid61">61</a>]. The same <em>VAPB</em> pathogenic variant has been identified in additional families with clinically heterogeneous types of motor neuron disease, including ALS8, typical severe ALS with rapid progression, and late-onset spinal muscular atrophy [<a href="#rid61">61-63</a>].</p><p class="headingAnchor" id="H9"><span class="h2">ALS9 (ANG gene)</span><span class="headingEndMark"> — </span>Missense variants in the <em>ANG</em> gene coding for the protein angiogenin have been identified in Irish, Scottish, and Scandinavian individuals, some with and some without a dominant family history of ALS [<a href="#rid64">64,65</a>]. Pathogenic forms of the <em>ANG</em> gene result in functional loss of angiogenic activity [<a href="#rid66">66</a>]. The angiogenin protein promotes vascularization of tissue and is required for the activity of vascular endothelial growth factor (VEGF).</p><p>The clinical course associated with <em>ANG</em> pathogenic variants is that of progressive upper and lower motor neuron disease with a wide range of onset age and survival [<a href="#rid64">64,65</a>]. The frequency of bulbar onset, approximately 50 percent, is greater than that of sporadic ALS. One patient with an <em>ANG</em> variant initially presented with parkinsonism, followed five years later by manifestations of upper and lower motor neuron disease and features characteristic of FTD [<a href="#rid67">67</a>].</p><p class="headingAnchor" id="H10"><span class="h2">ALS10 (TARDBP gene)</span><span class="headingEndMark"> — </span>The TAR DNA binding protein 43 (TDP-43) is a major component of ubiquitinated inclusions in sporadic ALS, FTD with ubiquitin-positive/tau-negative inclusions, ALS with FTD, and familial ALS (with the exception of <em>SOD1</em>-related and <em>FUS</em>-related familial ALS1). (See  <a class="medical medical_review" href="/d/html/5156.html" rel="external">"Epidemiology and pathogenesis of amyotrophic lateral sclerosis", section on 'Intracellular inclusions'</a>.)</p><p>Pathogenic variants in the <em>TARDBP</em> gene on chromosome 1p36.22 that encodes TDP-43 are the cause of autosomal dominant familial ALS10 and may be a rare cause of sporadic ALS [<a href="#rid68">68-72</a>]. These pathogenic variants account for approximately 4 percent of familial ALS cases in European populations and 2 percent of familial ALS cases in Chinese, Japanese, Korean, and Iranian populations [<a href="#rid14">14</a>].</p><p>Although the phenotype is variable, the clinical features of ALS in patients with <em>TARDBP</em> pathogenic variants compared with patients who have sporadic ALS include a greater likelihood for earlier onset, upper limb onset, and slower progression [<a href="#rid73">73</a>]. Some patients develop cognitive impairment or FTD. Other studies have identified <em>TARDBP</em> variants in apparently sporadic and familial forms of FTD with and without motor neuron disease [<a href="#rid74">74</a>].</p><p class="headingAnchor" id="H11"><span class="h2">ALS11 (FIG4 gene)</span><span class="headingEndMark"> — </span>Familial ALS11 is an autosomal dominant form of ALS caused by pathogenic variants in the FIG4 phosphoinositide 5-phosphatase (<em>FIG4</em>) gene on chromosome 6q21 [<a href="#rid75">75</a>].</p><p>In a study that screened DNA from 473 patients (364 with sporadic ALS and 109 with familial ALS), pathogenic variants in the <em>FIG4</em> gene were identified in nine patients (2 percent), including six with sporadic ALS and three with familial ALS [<a href="#rid75">75</a>]. Upper motor neuron features were prominent. The protein encoded by <em>FIG4</em> is a phosphoinositide 5-phosphatase that regulates a signaling lipid involved with membrane vesicle trafficking.</p><p>Charcot-Marie-Tooth disease type 4J (CMT4J) is an allelic disorder caused by <em>FIG4</em> pathogenic genetic variants. The phenotype of CMT4J is notable for autosomal recessive inheritance, early childhood to adult onset, and involvement of both motor and sensory neurons. (See  <a class="medical medical_review" href="/d/html/6220.html" rel="external">"Charcot-Marie-Tooth disease: Genetics, clinical features, and diagnosis", section on 'CMT4'</a>.)</p><p class="headingAnchor" id="H161409026"><span class="h2">ALS12 (OPTN gene)</span><span class="headingEndMark"> — </span>Pathogenic variants within the optineurin (<em>OPTN</em>) gene are the cause of ALS12, as first identified in two Japanese families [<a href="#rid76">76</a>]. Age of onset was 33 to 54 years, and progression was slow. Patients with ALS-FTD have also been described [<a href="#rid77">77</a>]. Later reports identified <em>OPTN</em> variants in 1 to 2 percent of familial ALS and 0 to 3.5 percent of sporadic ALS cases [<a href="#rid78">78,79</a>]. Although classified as an autosomal dominant disorder, autosomal recessive cases have also been reported [<a href="#rid76">76</a>].</p><p class="headingAnchor" id="H233524289"><span class="h2">ALS13 (ATXN2 gene)</span><span class="headingEndMark"> — </span>Mounting evidence suggests an association between intermediate length polyglutamine expansions in the ataxin 2 (<em>ATXN2</em>) gene and increased risk of ALS. This issue is discussed separately [<a href="#rid80">80</a>]. (See  <a class="medical medical_review" href="/d/html/5156.html" rel="external">"Epidemiology and pathogenesis of amyotrophic lateral sclerosis", section on 'Genetic susceptibility in sporadic ALS'</a>.)</p><p class="headingAnchor" id="H246948"><span class="h2">ALS14 (VCP gene)</span><span class="headingEndMark"> — </span>A pathogenic variant in the valosin-containing protein (<em>VCP</em>) gene was first identified in an Italian family as the cause of ALS14 [<a href="#rid81">81</a>]. Additional <em>VCP</em> pathogenic variants have since been identified in other familial ALS cases, suggesting that <em>VCP</em> pathogenic variants are responsible for 1 to 2 percent of familial ALS cases [<a href="#rid81">81,82</a>].</p><p>Pathogenic variants in <em>VCP</em> are also the cause of some but not all cases of an autosomal dominant familial syndrome characterized by inclusion body myopathy (IBM) associated with Paget disease of bone (PDB) and FTD (IBMPFD) [<a href="#rid83">83,84</a>]. There is mounting evidence that the spectrum of IBMPFD includes familial ALS [<a href="#rid82">82,84</a>]. The valosin-containing protein is implicated in numerous cellular functions, including pathways involved in ubiquitin-dependent protein degradation and autophagy. Variant valosin-containing protein may act by disrupting autophagy, resulting in the deposition of TDP-43 inclusions within the cytoplasm of affected cells [<a href="#rid85">85</a>]. (See  <a class="medical medical_review" href="/d/html/5156.html" rel="external">"Epidemiology and pathogenesis of amyotrophic lateral sclerosis", section on 'Intracellular inclusions'</a>.)</p><p class="headingAnchor" id="H2572382106"><span class="h2">ALS17 (CHMP2B gene)</span><span class="headingEndMark"> — </span>Case reports have associated pathogenic variants in the chromatin modifying protein 2b (<em>CHMP2B</em>) gene in both sporadic and dominantly inherited disorders referred to as chromosome 3-linked FTD (FTD3). While predominately associated with FTD3, late-onset (seventh to eighth decades) bulbar ALS with and without FTD has also been associated with <em>CHMP2B</em> pathogenic variants [<a href="#rid86">86,87</a>].</p><p class="headingAnchor" id="H10539001"><span class="h2">ALS18 (PFN1 gene)</span><span class="headingEndMark"> — </span>Pathogenic variants in the profilin 1 (<em>PFN1</em>) gene located on chromosome region 17p13.2 have been found in seven families with ALS displaying an autosomal dominant mode of inheritance [<a href="#rid88">88</a>]. The phenotype is notable for limb onset of ALS in middle age (average 45 years). The PFN1 protein regulates actin polymerization and has multiple other cellular functions. Pathogenic variants of the <em>PFN1</em> gene may play a role in ALS pathogenesis on the basis of reduced levels of bound actin, inhibition of axon growth, and a propensity to form intracellular aggregates compared with wild-type PFN1.</p><p class="headingAnchor" id="H1464542067"><span class="h2">ALS19 (ERBB4 gene)</span><span class="headingEndMark"> — </span>Pathogenic genetic variants in the <em>ERBB4</em> gene located on chromosome region 2p34 have been found in three families with onset of classic ALS. A heterozygous missense variant was first identified in one family that included three Japanese siblings who were diagnosed between the ages of 60 and 70. Subsequently, the same missense variant was identified in one Canadian individual with familial ALS, and a different missense variant was identified in a Japanese individual with sporadic ALS [<a href="#rid89">89</a>].</p><p class="headingAnchor" id="H2483497772"><span class="h2">ALS20 (HNRNPA1/A2B1 genes)</span><span class="headingEndMark"> — </span>The heterogeneous nuclear ribonucleoprotein A1 (<em>HNRNPA1</em>) and A2/B1 (<em>HNRNPA2B1</em>) genes encode for RNA binding proteins (RBPs), which play an important role in regulating RNA function and metabolism. The role of <em>HNRNPA1</em> and <em>HNRNPA2B1</em> in ALS was first suggested by a study that combined whole-exome sequencing with linkage analysis in families with autosomal dominant presentations of multisystem proteinopathy [<a href="#rid90">90</a>]. One of these families, with a pathogenic variant in <em>HNRNPA2B1</em>, had a rare inherited syndrome characterized by features of IBM associated with PDB, FTD, and ALS (also known as IBMPFD/ALS). Subsequently, exome screening of 212 familial ALS cases in which known ALS genes were excluded identified a causative pathogenic variant within the HNRNPA1 protein in one patient [<a href="#rid90">90</a>].</p><p>These genetic variants, which involve prion-like domains of the HNRNPA1 and HNRNPA2B1 proteins, appear to exacerbate the intrinsic tendency of RBPs to assemble into self-seeding fibrils, thereby leading to excess fibril formation and eventual neurodegeneration with cytoplasmic inclusions [<a href="#rid90">90</a>]. Pathogenic variants affecting a number of other RBPs are linked to ALS, including TDP-43, FUS, matrin 3 (MATR3), and TIA1<em> </em>[<a href="#rid91">91</a>]. In many of these cases, the affected RBPs also become depleted from the nucleus and deposited in cytoplasmic inclusions. (See <a class="local">'ALS10 (TARDBP gene)'</a> above and  <a class="medical medical_review" href="/d/html/5156.html" rel="external">"Epidemiology and pathogenesis of amyotrophic lateral sclerosis", section on 'Intracellular inclusions'</a> and  <a class="medical medical_review" href="/d/html/5156.html" rel="external">"Epidemiology and pathogenesis of amyotrophic lateral sclerosis", section on 'Altered RNA processing'</a>.)</p><p class="headingAnchor" id="H824091837"><span class="h2">ALS21 (MATR3 gene)</span><span class="headingEndMark"> — </span>Pathogenic variants in the <em>MATR3</em> gene on chromosome 5q31 coding for a nuclear matrix protein proposed to stabilize messenger RNA species were initially linked to a disorder manifesting as vocal cord and pharyngeal weakness with distal limb weakness ascribed to a myopathy. However, subsequent follow-up of these patients demonstrated progressive upper and lower motor neuron findings with respiratory failure ascribed to a progressive disorder of motor neurons [<a href="#rid92">92</a>].</p><p class="headingAnchor" id="H1100216188"><span class="h2">ALS22 (TUBA4A gene)</span><span class="headingEndMark"> — </span>Pathogenic variants in the microtubule network assembly gene, tubulin alpha 4a (<em>TUBA4A</em>), have been demonstrated to produce progressive upper and lower motor neuron signs, usually with spinal onset, and in some cases progressive cognitive decline consistent with frontotemporal dementia [<a href="#rid93">93</a>].</p><p class="headingAnchor" id="H2343971556"><span class="h2">ALS23 (ANXA11 gene)</span><span class="headingEndMark"> — </span>Located on chromosome 10q22.3, the annexin A11 (<em>ANXA11</em>) gene codes for a family of calcium-dependent phospholipid-binding proteins involved in vesicular trafficking. Pathogenic variants have been noted in both familial and sporadic ALS [<a href="#rid94">94</a>].</p><p class="headingAnchor" id="H1782605256"><span class="h2">ALS24 (NEK1 gene)</span><span class="headingEndMark"> — </span>Pathogenic variants in the NIMA-related kinase 1 (<em>NEK1</em>) gene that codes for a serine/threonine kinase involved in cell cycle regulation have been associated with a progressive upper and lower motor neuron disorder consistent with ALS. Cognitive dysfunction has not been described, although one patient was noted on magnetic resonance imaging to have hippocampal and temporal lobe atrophy [<a href="#rid95">95,96</a>].</p><p>Additionally, <em>NEK1</em> variants have been associated with 3 percent of sporadic ALS cases, suggesting that the gene can not only cause familial ALS but also increase the risk of development of sporadic ALS in carriers of these variants [<a href="#rid96">96</a>].</p><p class="headingAnchor" id="H1955211676"><span class="h2">ALS25 (KIF5A gene)</span><span class="headingEndMark"> — </span>Multiple lines of evidence implicate the kinesin family member 5A (<em>KIF5A</em>) gene in ALS [<a href="#rid97">97,98</a>]. Splice-site variants in the C-terminal domain of the <em>KIF5A</em> gene were identified in two families with ALS. A separate study found an excess of rare functional variants in familial ALS cases compared with controls, all of which localized to the C-terminal cargo-binding region of the gene [<a href="#rid97">97,98</a>]. This is in contrast with other pathogenic variants that localize to the N-terminal motor domain of <em>KIF5A</em> that have been previously described in patients with hereditary spastic paraplegia, Charcot-Marie-Tooth type 2, and neonatal intractable myoclonus.</p><p>Most patients manifest a classic ALS phenotype, with focal asymmetric onset of upper and lower motor neuron dysfunction in early to mid-adulthood. Limited available data support a highly penetrant loss-of-function mechanism related to dysfunction in intracellular transport [<a href="#rid97">97</a>].</p><p class="headingAnchor" id="H1824238611"><span class="h2">ALS26 (TIA1 gene)</span><span class="headingEndMark"> — </span>Pathogenic variants in the T cell-restricted intracellular antigen-1 (<em>TIA1</em>) gene were identified by whole genome sequencing of a multigenerational European family with ALS-FTD [<a href="#rid99">99</a>]. Most patients had the symptom onset occur between the ages of 51 and 74 years old. The mean duration of disease was 3 years (range 1 to 6 years). All had typical features of ALS, and several had evidence of FTD [<a href="#rid100">100</a>].</p><p>Neuropathologic studies in patients with variants in the <em>TIA1</em> gene have shown typical features of ALS but also the presence of structures resembling Lewy bodies. In vitro studies found that variant forms of the TIA1 protein undergo temperature- and concentration-dependent formation of amyloid-like fibrils, suggesting that protein aggregation plays a role in motor neuron death [<a href="#rid100">100</a>].</p><p class="headingAnchor" id="H303357690"><span class="h2">ALS27 (SPTLC1 gene)</span><span class="headingEndMark"> — </span>Variants in exon 2 of the serine palmitoyltransferase, long-chain base subunit-1 gene (<em>SPTLC1</em>) have been associated with an early childhood form of ALS in seven families [<a href="#rid101">101</a>]. These children typically present with lower extremity spasticity and upper motor neuron signs followed by eventual lower motor neuron involvement. Disease progression was significantly slower than sporadic ALS and mirrored other juvenile forms of ALS. Children in most families develop onset of symptoms from three to eight years old. Pathologic variants resulted from de novo mutations or autosomal dominance inheritance. </p><p><em>SPTLC1</em> is an important regulator of sphingolipid synthesis. Dysregulation of sphingolipids results in accumulation of sphingolipid species which are variably toxic to different types of neurons. Other variants in the <em>SPTLC1</em> gene have also been described in patients with hereditary sensory and autonomic neuropathy, type 1. Limited evidence suggests that ALS-associated variants lead to unregulated sphingolipid production whereas <em>HSAN1</em>-associated variants result in increased production of deoxysphingolipids and this may underlie the genotype-phenotype correlations [<a href="#rid101">101</a>]. (See  <a class="medical medical_review" href="/d/html/6227.html" rel="external">"Hereditary sensory and autonomic neuropathies", section on 'HSAN1'</a>.)</p><p class="headingAnchor" id="H3754378066"><span class="h2">Others</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong><em>DAO</em></strong><strong> gene</strong> – A variant in the D-amino acid oxidase (<em>DAO</em>) gene on chromosome region 12q24 has been associated with classic adult-onset familial ALS in a three-generational kindred from England [<a href="#rid102">102</a>]. Features included upper and lower motor neuron involvement and bulbar signs but no cognitive impairment. The mean age at time of death was 44 years (range 42 to 55 years).</p><p></p><p class="bulletIndent1">DAO is a flavoprotein that catalyzes the oxidative deamination of D-serine, a co-agonist in the activation of the N-methyl-D-aspartate (NMDA) glutamate receptor subtype. The R199W <em>DAO</em> variant identified in this study causes a loss of enzyme activity, which could lead to cell death via excitotoxicity from enhanced NMDA receptor activation.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong><em>NEFH </em></strong><strong>gene </strong>– Familial ALS linked to a variant of the neurofilament heavy chain (<em>NEFH)</em> gene located on chromosome 22q12.1-q13.1 has been noted in a Scandinavian kindred [<a href="#rid103">103</a>]. Variable features included monomelic amyotrophy, dementia, and bulbar dysfunction. One percent of sporadic ALS patients have been reported to have a genetic variant of the <em>NEFH </em>gene. Its role in the disease is uncertain.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong><em>SQSTM1</em></strong><strong> gene</strong> – Following upon the knowledge that protein degradation pathways are implicated in ALS and FTD, two groups screened large cohorts of patients with ALS and FTD and found variants in the sequestosome 1 (<em>SQSTM1</em>) gene [<a href="#rid104">104,105</a>]. The <em>SQSTM1</em> gene encodes for the p62 protein, which binds ubiquitin and is involved in protein degradation and autophagy. <em>SQSTM1</em> genetic variants were noted in both familial and sporadic cases of ALS and FTD. While one study reports heterozygous inheritance consistent with dominant genetics [<a href="#rid104">104</a>], the other is silent on this subject [<a href="#rid105">105</a>]. The precise role of this gene in the pathogenesis of ALS is not well defined.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong><em>CYLD</em></strong><strong> gene</strong> – A missense variant in CYLD lysine 63 deubiquitinase (<em>CYLD</em>) has been identified in a European family with ALS and FTD [<a href="#rid106">106</a>]. The variant leads to inhibition of nuclear factor kappa B (NF-kB) signaling and impaired autophagy. The autophagy pathway has also been implicated in other genetic forms of ALS, including <em>OPTN</em> and <em>SQSTM1</em>.</p><p></p><p class="headingAnchor" id="H16"><span class="h1">AUTOSOMAL RECESSIVE</span><span class="headingEndMark"> — </span>Recessively inherited familial ALS includes ALS2, ALS5, and ALS16. In autosomal recessive inheritance, a pathogenic variant of both copies of a gene is necessary in order for an individual to develop the condition. Typically, parents of an affected individual do not have the condition themselves but are obligate carriers for the pathogenic variant. The siblings of an individual with an autosomal recessive condition have the highest recurrence risk. Siblings have a 25 percent likelihood of inheriting both copies of the gene with the pathogenic variant and developing ALS, a 50 percent likelihood of being an unaffected carrier for the condition, and a 25 percent likelihood of being an unaffected noncarrier for the condition. Unless an individual's partner is also a carrier for the same disease gene, all children of an individual with autosomal recessive ALS will be unaffected carriers of the condition. In autosomal recessive inheritance, there may be no family history of the condition in previous generations because a single copy of a gene variant may be passed silently through a family.</p><p>The <em>ALS2</em>, SPG11, spatacsin vesicle trafficking associated (<em>SPG11</em>), and <em>SIGMAR1</em> genes are inherited in an autosomal recessive pattern. Additionally, certain pathogenic variants in some ALS genes that are usually associated with autosomal dominant inheritance can also be inherited in an autosomal recessive pattern. Examples include the <em>SOD1, FUS, and OPTN</em> genes. (See <a class="local">'ALS1 (SOD1 gene)'</a> above and <a class="local">'ALS6 (FUS gene)'</a> above and <a class="local">'ALS12 (OPTN gene)'</a> above.)</p><p class="headingAnchor" id="H17"><span class="h2">ALS2 (ALS2 gene)</span><span class="headingEndMark"> — </span>Familial ALS2 is caused by a pathogenic variant in exons 3 or 4 of the <em>ALS2</em> gene coding for Alsin, a putative GTPase regulatory protein, on chromosome 2q33 [<a href="#rid107">107,108</a>]. The mean age of onset is in the first decade. Clinical features involve the development of upper motor neuron signs in the face and lower limbs, greater than in the upper limbs. Mild lower motor neuron findings develop slowly in the distal legs, more so than the arms. The course is markedly slow with note of difficulty walking in the fifth decade. In addition, there are variant phenotypes and allelic disorders:</p><p class="bulletIndent1"><span class="glyph">●</span>One affected patient had ALS onset by age 2 and progression to anarthria and was nonambulatory by age 18 [<a href="#rid109">109</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Another report described juvenile onset of ALS associated with generalized dystonia and cerebellar signs in two consanguineous families [<a href="#rid110">110</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>A deletion in exon 9 of the <em>ALS2</em> gene is responsible for recessively inherited childhood-onset primary lateral sclerosis (PLS), at times associated with gaze paresis [<a href="#rid111">111</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Frame shift variants in various locations within the <em>ALS2</em> gene that produce a premature stop codon have been associated with recessively inherited infantile-onset familial spastic paraparesis [<a href="#rid112">112</a>].</p><p></p><p class="headingAnchor" id="H18"><span class="h2">ALS5 (SPG11 gene)</span><span class="headingEndMark"> — </span>Familial ALS5, the most common form of autosomal recessive familial ALS, has been linked to pathogenic variants of the <em>SPG11</em> gene on chromosome 15q15-q22 [<a href="#rid113">113,114</a>]. The onset is in the second decade and is characterized by the development of lower motor neuron signs that exceed the upper motor neuron findings. Distal involvement is usually greater than proximal, and arms are usually involved more than legs. Bulbar involvement occurs late.</p><p>Pathogenic variants involving <em>SPG11</em> are also the cause of hereditary spastic paraplegia with a thin or atrophied corpus callosum [<a href="#rid115">115</a>]. However, the clinical features of patients with ALS5 and <em>SPG11</em> variants differ from those associated with hereditary spastic paraplegia due to the presence of bulbar symptoms, upper motor neuron involvement, and pathologic manifestations of ALS [<a href="#rid114">114</a>]. In addition, the patients with familial ALS5 have not developed the corpus callosum thinning, ocular abnormalities, cognitive deficits, or psychiatric problems that are observed in <em>SPG11</em>-related hereditary spastic paraplegia. (See  <a class="medical medical_review" href="/d/html/5093.html" rel="external">"Disorders affecting the spinal cord", section on 'Hereditary spastic paraplegias'</a>.)</p><p class="headingAnchor" id="H3613063777"><span class="h2">ALS16 (SIGMAR1 gene)</span><span class="headingEndMark"> — </span>Familial ALS16 has been reported in one consanguineous Saudi Arabian family in which six family members developed an early-childhood onset condition consistent with juvenile ALS. A homozygous variant affecting a highly conserved amino acid in the transmembrane domain of the <em>SIGMAR1</em> gene was identified. Affected family members demonstrated lower limb spasticity with hyperreflexia and weakness at a very young age (1 to 2 years) that progressed to hand and forearm muscle weakness by age 9 to 10 years followed by paralysis of the forearm and triceps. Two affected family members were nonambulatory by the age of 20. Cognition was preserved, and none of the affected family members had respiratory or bulbar muscle weakness [<a href="#rid116">116</a>].</p><p class="headingAnchor" id="H633526471"><span class="h2">Others</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>SYNE1 gene</strong> – Variants in the synaptic nuclear envelope protein 1 (<em>SYNE1</em>) gene have been associated with cerebellar ataxia syndromes. The <em>SYNE1</em> gene is a ubiquitously expressed protein that participates in connecting the nuclear plasma membrane to the actin cytoskeleton [<a href="#rid117">117</a>]. It plays a role in maintaining neuronal connections, especially in the cerebellum. (See  <a class="medical medical_review" href="/d/html/14134.html" rel="external">"Overview of cerebellar ataxia in adults", section on 'Autosomal recessive ataxias'</a>.)</p><p></p><p class="bulletIndent1">Case reports of juvenile-onset ALS associated with heterozygous and homozygous variants in the <em>SYNE1</em> gene include one patient from Japan and two siblings from India [<a href="#rid118">118,119</a>]. The patients presented in the second decade of life with lower extremity weakness and wasting associated with spasticity. Cerebellar features developed later in the slowly progressive course. The genetic specific factors that result in ALS or cerebellar phenotypes are unknown.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>ERLIN1 gene</strong> – Variants in the endoplasmic reticulum lipid raft-associated 1 (<em>ERLIN1</em>) gene have been associated with recessive hereditary spastic paraplegia [<a href="#rid120">120</a>]. In addition, multiple members of a family from Turkey were reported as having early onset ALS characterized by lower extremity weakness and hyperreflexia as well as hand atrophy [<a href="#rid121">121</a>]. The disease course was very slow with progression over decades.</p><p></p><p class="headingAnchor" id="H29081274"><span class="h1">X-LINKED DOMINANT</span><span class="headingEndMark"> — </span>In X-linked dominant inheritance, the gene responsible for the condition is located on the X chromosome, and only one copy of the gene with the pathogenic variant is necessary to cause the disease. Therefore, unlike X-linked recessive inheritance, both males and females may be affected. If a father has an X-linked dominant condition, all of his female children will inherit the disease and none of his male children will inherit the disease. If a mother has an X-linked dominant disease, there is a 50 percent likelihood that either her female or male children will inherit the condition.</p><p class="headingAnchor" id="H233524747"><span class="h2">ALS15 (UBQLN2 gene)</span><span class="headingEndMark"> — </span>In unrelated families from North America and Europe, a form of dominantly inherited ALS with reduced penetrance in females has been linked to variants of the ubiquilin 2 (<em>UBQLN2</em>) gene located at Xp11.21 [<a href="#rid122">122,123</a>]. In the first report, the age of onset ranged from 16 to 71 years and was earlier in males, while mean disease duration was approximately four years and was similar in males and females [<a href="#rid122">122</a>]. A frontotemporal lobar type of dementia accompanied some cases of ALS.</p><p class="headingAnchor" id="H3230708558"><span class="h1">SOCIETY GUIDELINE LINKS</span><span class="headingEndMark"> — </span>Links to society and government-sponsored guidelines from selected countries and regions around the world are provided separately. (See  <a class="medical medical_society_guidelines" href="/d/html/118906.html" rel="external">"Society guideline links: Motor neuron disease"</a>.)</p><p class="headingAnchor" id="H29079459"><span class="h1">SUMMARY</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Clinical clues to genetic origin </strong>– Familial amyotrophic lateral sclerosis (ALS) is phenotypically and genetically heterogeneous. In most individual cases, it may be difficult to determine on clinical grounds alone if ALS is familial or sporadic, especially at the onset of disease. The presence of atypical features such as young age of onset, sensory loss, a positive family history of ALS, other neurodegenerative disorders, and dementia should alert the clinician to the possibility of familial ALS. (See <a class="local">'Introduction'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Genetic testing </strong>– Clinical genetic testing for inherited (familial) forms of ALS is available to look for causative pathogenic variants in several genes. Genetic testing for specific ALS-causing pathogenic genetic variants is typically used for specific indications  (<a class="graphic graphic_table graphicRef61384" href="/d/graphic/61384.html" rel="external">table 1</a>). The most common use of genetic testing for ALS is to provide risk information and subsequent counseling for unaffected at-risk family members. (See <a class="local">'Genetic testing and counseling'</a> above.)</p><p></p><p class="bulletIndent1">To properly interpret genetic test results for ALS genes, it is necessary to know whether the person being tested has ALS and whether a pathogenic variant in the family has already been identified. The recurrence risk of ALS in a given family (ie, whether or not another family member will develop ALS) is estimated primarily by determining the inheritance pattern. (See <a class="local">'Interpretation'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Autosomal dominant types </strong>– The majority of familial ALS cases follow an autosomal dominant inheritance pattern. (See <a class="local">'Autosomal dominant'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Repeat expansions in the <em>C9ORF72</em> gene and pathogenic variants in the <em>SOD1</em> gene are the most common causes of familial ALS. In addition, pathogenic variants in the <em>FUS</em> gene and the <em>TARDBP</em> gene are relatively common causes of familial ALS. (See <a class="local">'C9ORF72 gene'</a> above and <a class="local">'ALS1 (SOD1 gene)'</a> above and <a class="local">'ALS6 (FUS gene)'</a> above and <a class="local">'ALS10 (TARDBP gene)'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span>Less common forms of autosomal dominant familial ALS include ALS3, ALS4, ALS7, ALS8, ALS9, ALS11, ALS12, ALS14, ALS17, ALS18, and ALS caused by pathogenic variants of the <em>DAO</em>, <em>NEFH</em>, and <em>KIF5A</em> genes. (See <a class="local">'Autosomal dominant'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Autosomal recessive types </strong>– Autosomal recessive forms of familial ALS are associated with variants of the <em>ALS2,</em> <em>SPG11</em>, and <em>SIGMAR1</em> genes. (See <a class="local">'Autosomal recessive'</a> above and <a class="local">'ALS2 (ALS2 gene)'</a> above and <a class="local">'ALS5 (SPG11 gene)'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>X-linked dominant types </strong>– An X-linked dominant form of familial ALS is associated with <em>UBQLN2</em> gene pathogenic variants. (See <a class="local">'X-linked dominant'</a> above and <a class="local">'ALS15 (UBQLN2 gene)'</a> above.)</p><p></p><p class="headingAnchor" id="H1068848363"><span class="h1">ACKNOWLEDGMENT</span><span class="headingEndMark"> — </span>The UpToDate editorial staff acknowledges Dana Falcone, MS, LCGC, who contributed to earlier versions of this topic review.</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">Byrne S, Walsh C, Lynch C, et al. Rate of familial amyotrophic lateral sclerosis: a systematic review and meta-analysis. J Neurol Neurosurg Psychiatry 2011; 82:623.</a></li><li><a class="nounderline abstract_t">Taylor JP, Brown RH Jr, Cleveland DW. Decoding ALS: from genes to mechanism. Nature 2016; 539:197.</a></li><li><a class="nounderline abstract_t">Points to consider: ethical, legal, and psychosocial implications of genetic testing in children and adolescents. American Society of Human Genetics Board of Directors, American College of Medical Genetics Board of Directors. Am J Hum Genet 1995; 57:1233.</a></li><li><a class="nounderline abstract_t">Williams KL, Fifita JA, Vucic S, et al. Pathophysiological insights into ALS with C9ORF72 expansions. J Neurol Neurosurg Psychiatry 2013; 84:931.</a></li><li><a class="nounderline abstract_t">Benatar M, Stanislaw C, Reyes E, et al. Presymptomatic ALS genetic counseling and testing: Experience and recommendations. Neurology 2016; 86:2295.</a></li><li><a class="nounderline abstract_t">Morita M, Al-Chalabi A, Andersen PM, et al. A locus on chromosome 9p confers susceptibility to ALS and frontotemporal dementia. Neurology 2006; 66:839.</a></li><li><a class="nounderline abstract_t">Vance C, Al-Chalabi A, Ruddy D, et al. Familial amyotrophic lateral sclerosis with frontotemporal dementia is linked to a locus on chromosome 9p13.2-21.3. Brain 2006; 129:868.</a></li><li><a class="nounderline abstract_t">Valdmanis PN, Dupre N, Bouchard JP, et al. Three families with amyotrophic lateral sclerosis and frontotemporal dementia with evidence of linkage to chromosome 9p. Arch Neurol 2007; 64:240.</a></li><li><a class="nounderline abstract_t">Le Ber I, Camuzat A, Berger E, et al. Chromosome 9p-linked families with frontotemporal dementia associated with motor neuron disease. Neurology 2009; 72:1669.</a></li><li><a class="nounderline abstract_t">DeJesus-Hernandez M, Mackenzie IR, Boeve BF, et al. Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS. Neuron 2011; 72:245.</a></li><li><a class="nounderline abstract_t">Renton AE, Majounie E, Waite A, et al. A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD. Neuron 2011; 72:257.</a></li><li><a class="nounderline abstract_t">Gijselinck I, Van Langenhove T, van der Zee J, et al. A C9orf72 promoter repeat expansion in a Flanders-Belgian cohort with disorders of the frontotemporal lobar degeneration-amyotrophic lateral sclerosis spectrum: a gene identification study. Lancet Neurol 2012; 11:54.</a></li><li><a class="nounderline abstract_t">Hosler BA, Siddique T, Sapp PC, et al. Linkage of familial amyotrophic lateral sclerosis with frontotemporal dementia to chromosome 9q21-q22. JAMA 2000; 284:1664.</a></li><li><a class="nounderline abstract_t">Zou ZY, Zhou ZR, Che CH, et al. Genetic epidemiology of amyotrophic lateral sclerosis: a systematic review and meta-analysis. J Neurol Neurosurg Psychiatry 2017; 88:540.</a></li><li><a class="nounderline abstract_t">Majounie E, Renton AE, Mok K, et al. Frequency of the C9orf72 hexanucleotide repeat expansion in patients with amyotrophic lateral sclerosis and frontotemporal dementia: a cross-sectional study. Lancet Neurol 2012; 11:323.</a></li><li><a class="nounderline abstract_t">Byrne S, Elamin M, Bede P, et al. Cognitive and clinical characteristics of patients with amyotrophic lateral sclerosis carrying a C9orf72 repeat expansion: a population-based cohort study. Lancet Neurol 2012; 11:232.</a></li><li><a class="nounderline abstract_t">Cammack AJ, Atassi N, Hyman T, et al. Prospective natural history study of C9orf72 ALS clinical characteristics and biomarkers. Neurology 2019; 93:e1605.</a></li><li><a class="nounderline abstract_t">Chen Y, Lin Z, Chen X, et al. Large C9orf72 repeat expansions are seen in Chinese patients with sporadic amyotrophic lateral sclerosis. Neurobiol Aging 2016; 38:217.e15.</a></li><li><a class="nounderline abstract_t">Dols-Icardo O, García-Redondo A, Rojas-García R, et al. Characterization of the repeat expansion size in C9orf72 in amyotrophic lateral sclerosis and frontotemporal dementia. Hum Mol Genet 2014; 23:749.</a></li><li><a class="nounderline abstract_t">Mayl K, Shaw CE, Lee YB. Disease Mechanisms and Therapeutic Approaches in C9orf72 ALS-FTD. Biomedicines 2021; 9.</a></li><li><a class="nounderline abstract_t">Benussi L, Rossi G, Glionna M, et al. C9ORF72 hexanucleotide repeat number in frontotemporal lobar degeneration: a genotype-phenotype correlation study. J Alzheimers Dis 2015; 45:319.</a></li><li><a class="nounderline abstract_t">Murphy NA, Arthur KC, Tienari PJ, et al. Age-related penetrance of the C9orf72 repeat expansion. Sci Rep 2017; 7:2116.</a></li><li><a class="nounderline abstract_t">Glasmacher SA, Wong C, Pearson IE, Pal S. Survival and Prognostic Factors in C9orf72 Repeat Expansion Carriers: A Systematic Review and Meta-analysis. JAMA Neurol 2020; 77:367.</a></li><li><a class="nounderline abstract_t">Rosen DR, Siddique T, Patterson D, et al. Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis. Nature 1993; 362:59.</a></li><li><a class="nounderline abstract_t">Gaudette M, Hirano M, Siddique T. Current status of SOD1 mutations in familial amyotrophic lateral sclerosis. Amyotroph Lateral Scler Other Motor Neuron Disord 2000; 1:83.</a></li><li class="breakAll">The Amyotrophic Lateral Sclerosis Online Genetics Database. http://alsod.iop.kcl.ac.uk/ (Accessed on August 22, 2017).</li><li><a class="nounderline abstract_t">Abel O, Shatunov A, Jones AR, et al. Development of a Smartphone App for a Genetics Website: The Amyotrophic Lateral Sclerosis Online Genetics Database (ALSoD). JMIR Mhealth Uhealth 2013; 1:e18.</a></li><li><a class="nounderline abstract_t">Abati E, Bresolin N, Comi G, Corti S. Silence superoxide dismutase 1 (SOD1): a promising therapeutic target for amyotrophic lateral sclerosis (ALS). Expert Opin Ther Targets 2020; 24:295.</a></li><li><a class="nounderline abstract_t">Andersen PM, Sims KB, Xin WW, et al. Sixteen novel mutations in the Cu/Zn superoxide dismutase gene in amyotrophic lateral sclerosis: a decade of discoveries, defects and disputes. Amyotroph Lateral Scler Other Motor Neuron Disord 2003; 4:62.</a></li><li><a class="nounderline abstract_t">Ratovitski T, Corson LB, Strain J, et al. Variation in the biochemical/biophysical properties of mutant superoxide dismutase 1 enzymes and the rate of disease progression in familial amyotrophic lateral sclerosis kindreds. Hum Mol Genet 1999; 8:1451.</a></li><li><a class="nounderline abstract_t">Cudkowicz ME, McKenna-Yasek D, Chen C, et al. Limited corticospinal tract involvement in amyotrophic lateral sclerosis subjects with the A4V mutation in the copper/zinc superoxide dismutase gene. Ann Neurol 1998; 43:703.</a></li><li><a class="nounderline abstract_t">Andersen PM. Genetics of sporadic ALS. Amyotroph Lateral Scler Other Motor Neuron Disord 2001; 2 Suppl 1:S37.</a></li><li><a class="nounderline abstract_t">Gellera C. Genetics of ALS in Italian families. Amyotroph Lateral Scler Other Motor Neuron Disord 2001; 2 Suppl 1:S43.</a></li><li><a class="nounderline abstract_t">Gellera C, Castellotti B, Riggio MC, et al. Superoxide dismutase gene mutations in Italian patients with familial and sporadic amyotrophic lateral sclerosis: identification of three novel missense mutations. Neuromuscul Disord 2001; 11:404.</a></li><li><a class="nounderline abstract_t">Hays AP, Naini A, He CZ, et al. Sporadic amyotrophic lateral sclerosis and breast cancer: Hyaline conglomerate inclusions lead to identification of SOD1 mutation. J Neurol Sci 2006; 242:67.</a></li><li><a class="nounderline abstract_t">Aksoy H, Dean G, Elian M, et al. A4T mutation in the SOD1 gene causing familial amyotrophic lateral sclerosis. Neuroepidemiology 2003; 22:235.</a></li><li><a class="nounderline abstract_t">Arisato T, Okubo R, Arata H, et al. Clinical and pathological studies of familial amyotrophic lateral sclerosis (FALS) with SOD1 H46R mutation in large Japanese families. Acta Neuropathol 2003; 106:561.</a></li><li><a class="nounderline abstract_t">Rezania K, Yan J, Dellefave L, et al. A rare Cu/Zn superoxide dismutase mutation causing familial amyotrophic lateral sclerosis with variable age of onset, incomplete penetrance and a sensory neuropathy. Amyotroph Lateral Scler Other Motor Neuron Disord 2003; 4:162.</a></li><li><a class="nounderline abstract_t">Régal L, Vanopdenbosch L, Tilkin P, et al. The G93C mutation in superoxide dismutase 1: clinicopathologic phenotype and prognosis. Arch Neurol 2006; 63:262.</a></li><li><a class="nounderline abstract_t">Andersen PM, Nilsson P, Ala-Hurula V, et al. Amyotrophic lateral sclerosis associated with homozygosity for an Asp90Ala mutation in CuZn-superoxide dismutase. Nat Genet 1995; 10:61.</a></li><li><a class="nounderline abstract_t">Andersen PM, Forsgren L, Binzer M, et al. Autosomal recessive adult-onset amyotrophic lateral sclerosis associated with homozygosity for Asp90Ala CuZn-superoxide dismutase mutation. A clinical and genealogical study of 36 patients. Brain 1996; 119 ( Pt 4):1153.</a></li><li><a class="nounderline abstract_t">Andersen PM, Nordström U, Tsiakas K, et al. Phenotype in an Infant with SOD1 Homozygous Truncating Mutation. N Engl J Med 2019; 381:486.</a></li><li><a class="nounderline abstract_t">Park JH, Elpers C, Reunert J, et al. SOD1 deficiency: a novel syndrome distinct from amyotrophic lateral sclerosis. Brain 2019; 142:2230.</a></li><li><a class="nounderline abstract_t">Hayward C, Brock DJ, Minns RA, Swingler RJ. Homozygosity for Asn86Ser mutation in the CuZn-superoxide dismutase gene produces a severe clinical phenotype in a juvenile onset case of familial amyotrophic lateral sclerosis. J Med Genet 1998; 35:174.</a></li><li><a class="nounderline abstract_t">Ezer S, Daana M, Park JH, et al. Infantile SOD1 deficiency syndrome caused by a homozygous SOD1 variant with absence of enzyme activity. Brain 2022; 145:872.</a></li><li><a class="nounderline abstract_t">Hand CK, Khoris J, Salachas F, et al. A novel locus for familial amyotrophic lateral sclerosis, on chromosome 18q. Am J Hum Genet 2002; 70:251.</a></li><li><a class="nounderline abstract_t">Course MM, Gudsnuk K, Smukowski SN, et al. Evolution of a Human-Specific Tandem Repeat Associated with ALS. Am J Hum Genet 2020; 107:445.</a></li><li><a class="nounderline abstract_t">Chance PF, Rabin BA, Ryan SG, et al. Linkage of the gene for an autosomal dominant form of juvenile amyotrophic lateral sclerosis to chromosome 9q34. Am J Hum Genet 1998; 62:633.</a></li><li><a class="nounderline abstract_t">Chen YZ, Bennett CL, Huynh HM, et al. DNA/RNA helicase gene mutations in a form of juvenile amyotrophic lateral sclerosis (ALS4). Am J Hum Genet 2004; 74:1128.</a></li><li><a class="nounderline abstract_t">Grunseich C, Patankar A, Amaya J, et al. Clinical and Molecular Aspects of Senataxin Mutations in Amyotrophic Lateral Sclerosis 4. Ann Neurol 2020; 87:547.</a></li><li><a class="nounderline abstract_t">Asaka T, Yokoji H, Ito J, et al. Autosomal recessive ataxia with peripheral neuropathy and elevated AFP: novel mutations in SETX. Neurology 2006; 66:1580.</a></li><li><a class="nounderline abstract_t">Chen YZ, Hashemi SH, Anderson SK, et al. Senataxin, the yeast Sen1p orthologue: characterization of a unique protein in which recessive mutations cause ataxia and dominant mutations cause motor neuron disease. Neurobiol Dis 2006; 23:97.</a></li><li><a class="nounderline abstract_t">Tariq H, Imran R, Naz S. A Novel Homozygous Variant of SETX Causes Ataxia with Oculomotor Apraxia Type 2. J Clin Neurol 2018; 14:498.</a></li><li><a class="nounderline abstract_t">Szpisjak L, Obal I, Engelhardt JI, et al. A novel SETX gene mutation producing ataxia with oculomotor apraxia type 2. Acta Neurol Belg 2016; 116:405.</a></li><li><a class="nounderline abstract_t">Kwiatkowski TJ Jr, Bosco DA, Leclerc AL, et al. Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis. Science 2009; 323:1205.</a></li><li><a class="nounderline abstract_t">Vance C, Rogelj B, Hortobágyi T, et al. Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6. Science 2009; 323:1208.</a></li><li><a class="nounderline abstract_t">Yan J, Deng HX, Siddique N, et al. Frameshift and novel mutations in FUS in familial amyotrophic lateral sclerosis and ALS/dementia. Neurology 2010; 75:807.</a></li><li><a class="nounderline abstract_t">Akiyama T, Warita H, Kato M, et al. Genotype-phenotype relationships in familial amyotrophic lateral sclerosis with FUS/TLS mutations in Japan. Muscle Nerve 2016; 54:398.</a></li><li><a class="nounderline abstract_t">Sapp PC, Hosler BA, McKenna-Yasek D, et al. Identification of two novel loci for dominantly inherited familial amyotrophic lateral sclerosis. Am J Hum Genet 2003; 73:397.</a></li><li><a class="nounderline abstract_t">Nishimura AL, Mitne-Neto M, Silva HC, et al. A novel locus for late onset amyotrophic lateral sclerosis/motor neurone disease variant at 20q13. J Med Genet 2004; 41:315.</a></li><li><a class="nounderline abstract_t">Nishimura AL, Mitne-Neto M, Silva HC, et al. A mutation in the vesicle-trafficking protein VAPB causes late-onset spinal muscular atrophy and amyotrophic lateral sclerosis. Am J Hum Genet 2004; 75:822.</a></li><li><a class="nounderline abstract_t">Landers JE, Leclerc AL, Shi L, et al. New VAPB deletion variant and exclusion of VAPB mutations in familial ALS. Neurology 2008; 70:1179.</a></li><li><a class="nounderline abstract_t">Millecamps S, Salachas F, Cazeneuve C, et al. SOD1, ANG, VAPB, TARDBP, and FUS mutations in familial amyotrophic lateral sclerosis: genotype-phenotype correlations. J Med Genet 2010; 47:554.</a></li><li><a class="nounderline abstract_t">Greenway MJ, Alexander MD, Ennis S, et al. A novel candidate region for ALS on chromosome 14q11.2. Neurology 2004; 63:1936.</a></li><li><a class="nounderline abstract_t">Greenway MJ, Andersen PM, Russ C, et al. ANG mutations segregate with familial and 'sporadic' amyotrophic lateral sclerosis. Nat Genet 2006; 38:411.</a></li><li><a class="nounderline abstract_t">Wu D, Yu W, Kishikawa H, et al. Angiogenin loss-of-function mutations in amyotrophic lateral sclerosis. Ann Neurol 2007; 62:609.</a></li><li><a class="nounderline abstract_t">van Es MA, Diekstra FP, Veldink JH, et al. A case of ALS-FTD in a large FALS pedigree with a K17I ANG mutation. Neurology 2009; 72:287.</a></li><li><a class="nounderline abstract_t">Sreedharan J, Blair IP, Tripathi VB, et al. TDP-43 mutations in familial and sporadic amyotrophic lateral sclerosis. Science 2008; 319:1668.</a></li><li><a class="nounderline abstract_t">Yokoseki A, Shiga A, Tan CF, et al. TDP-43 mutation in familial amyotrophic lateral sclerosis. Ann Neurol 2008; 63:538.</a></li><li><a class="nounderline abstract_t">Van Deerlin VM, Leverenz JB, Bekris LM, et al. TARDBP mutations in amyotrophic lateral sclerosis with TDP-43 neuropathology: a genetic and histopathological analysis. Lancet Neurol 2008; 7:409.</a></li><li><a class="nounderline abstract_t">Kabashi E, Valdmanis PN, Dion P, et al. TARDBP mutations in individuals with sporadic and familial amyotrophic lateral sclerosis. Nat Genet 2008; 40:572.</a></li><li><a class="nounderline abstract_t">Kühnlein P, Sperfeld AD, Vanmassenhove B, et al. Two German kindreds with familial amyotrophic lateral sclerosis due to TARDBP mutations. Arch Neurol 2008; 65:1185.</a></li><li><a class="nounderline abstract_t">Corcia P, Valdmanis P, Millecamps S, et al. Phenotype and genotype analysis in amyotrophic lateral sclerosis with TARDBP gene mutations. Neurology 2012; 78:1519.</a></li><li><a class="nounderline abstract_t">Borroni B, Archetti S, Del Bo R, et al. TARDBP mutations in frontotemporal lobar degeneration: frequency, clinical features, and disease course. Rejuvenation Res 2010; 13:509.</a></li><li><a class="nounderline abstract_t">Chow CY, Landers JE, Bergren SK, et al. Deleterious variants of FIG4, a phosphoinositide phosphatase, in patients with ALS. Am J Hum Genet 2009; 84:85.</a></li><li><a class="nounderline abstract_t">Maruyama H, Morino H, Ito H, et al. Mutations of optineurin in amyotrophic lateral sclerosis. Nature 2010; 465:223.</a></li><li><a class="nounderline abstract_t">Feng SM, Che CH, Feng SY, et al. Novel mutation in optineurin causing aggressive ALS+/-frontotemporal dementia. Ann Clin Transl Neurol 2019; 6:2377.</a></li><li><a class="nounderline abstract_t">Del Bo R, Tiloca C, Pensato V, et al. Novel optineurin mutations in patients with familial and sporadic amyotrophic lateral sclerosis. J Neurol Neurosurg Psychiatry 2011; 82:1239.</a></li><li><a class="nounderline abstract_t">Belzil VV, Daoud H, Desjarlais A, et al. Analysis of OPTN as a causative gene for amyotrophic lateral sclerosis. Neurobiol Aging 2011; 32:555.e13.</a></li><li><a class="nounderline abstract_t">Chio A, Moglia C, Canosa A, et al. Exploring the phenotype of Italian patients with ALS with intermediate ATXN2 polyQ repeats. J Neurol Neurosurg Psychiatry 2022; 93:1216.</a></li><li><a class="nounderline abstract_t">Johnson JO, Mandrioli J, Benatar M, et al. Exome sequencing reveals VCP mutations as a cause of familial ALS. Neuron 2010; 68:857.</a></li><li><a class="nounderline abstract_t">González-Pérez P, Cirulli ET, Drory VE, et al. Novel mutation in VCP gene causes atypical amyotrophic lateral sclerosis. Neurology 2012; 79:2201.</a></li><li><a class="nounderline abstract_t">Watts GD, Wymer J, Kovach MJ, et al. Inclusion body myopathy associated with Paget disease of bone and frontotemporal dementia is caused by mutant valosin-containing protein. Nat Genet 2004; 36:377.</a></li><li><a class="nounderline abstract_t">Nalbandian A, Donkervoort S, Dec E, et al. The multiple faces of valosin-containing protein-associated diseases: inclusion body myopathy with Paget's disease of bone, frontotemporal dementia, and amyotrophic lateral sclerosis. J Mol Neurosci 2011; 45:522.</a></li><li><a class="nounderline abstract_t">Ju JS, Fuentealba RA, Miller SE, et al. Valosin-containing protein (VCP) is required for autophagy and is disrupted in VCP disease. J Cell Biol 2009; 187:875.</a></li><li><a class="nounderline abstract_t">Gydesen S, Brown JM, Brun A, et al. Chromosome 3 linked frontotemporal dementia (FTD-3). Neurology 2002; 59:1585.</a></li><li><a class="nounderline abstract_t">Parkinson N, Ince PG, Smith MO, et al. ALS phenotypes with mutations in CHMP2B (charged multivesicular body protein 2B). Neurology 2006; 67:1074.</a></li><li><a class="nounderline abstract_t">Wu CH, Fallini C, Ticozzi N, et al. Mutations in the profilin 1 gene cause familial amyotrophic lateral sclerosis. Nature 2012; 488:499.</a></li><li><a class="nounderline abstract_t">Takahashi Y, Fukuda Y, Yoshimura J, et al. ERBB4 mutations that disrupt the neuregulin-ErbB4 pathway cause amyotrophic lateral sclerosis type 19. Am J Hum Genet 2013; 93:900.</a></li><li><a class="nounderline abstract_t">Kim HJ, Kim NC, Wang YD, et al. Mutations in prion-like domains in hnRNPA2B1 and hnRNPA1 cause multisystem proteinopathy and ALS. Nature 2013; 495:467.</a></li><li><a class="nounderline abstract_t">Purice MD, Taylor JP. Linking hnRNP Function to ALS and FTD Pathology. Front Neurosci 2018; 12:326.</a></li><li><a class="nounderline abstract_t">Johnson JO, Pioro EP, Boehringer A, et al. Mutations in the Matrin 3 gene cause familial amyotrophic lateral sclerosis. Nat Neurosci 2014; 17:664.</a></li><li><a class="nounderline abstract_t">Smith BN, Ticozzi N, Fallini C, et al. Exome-wide rare variant analysis identifies TUBA4A mutations associated with familial ALS. Neuron 2014; 84:324.</a></li><li><a class="nounderline abstract_t">Smith BN, Topp SD, Fallini C, et al. Mutations in the vesicular trafficking protein annexin A11 are associated with amyotrophic lateral sclerosis. Sci Transl Med 2017; 9.</a></li><li><a class="nounderline abstract_t">Brenner D, Müller K, Wieland T, et al. NEK1 mutations in familial amyotrophic lateral sclerosis. Brain 2016; 139:e28.</a></li><li><a class="nounderline abstract_t">Kenna KP, van Doormaal PT, Dekker AM, et al. NEK1 variants confer susceptibility to amyotrophic lateral sclerosis. Nat Genet 2016; 48:1037.</a></li><li><a class="nounderline abstract_t">Brenner D, Yilmaz R, Müller K, et al. Hot-spot KIF5A mutations cause familial ALS. Brain 2018; 141:688.</a></li><li><a class="nounderline abstract_t">Nicolas A, Kenna KP, Renton AE, et al. Genome-wide Analyses Identify KIF5A as a Novel ALS Gene. Neuron 2018; 97:1268.</a></li><li><a class="nounderline abstract_t">Mackenzie IR, Nicholson AM, Sarkar M, et al. TIA1 Mutations in Amyotrophic Lateral Sclerosis and Frontotemporal Dementia Promote Phase Separation and Alter Stress Granule Dynamics. Neuron 2017; 95:808.</a></li><li><a class="nounderline abstract_t">Hirsch-Reinshagen V, Pottier C, Nicholson AM, et al. Clinical and neuropathological features of ALS/FTD with TIA1 mutations. Acta Neuropathol Commun 2017; 5:96.</a></li><li><a class="nounderline abstract_t">Mohassel P, Donkervoort S, Lone MA, et al. Childhood amyotrophic lateral sclerosis caused by excess sphingolipid synthesis. Nat Med 2021; 27:1197.</a></li><li><a class="nounderline abstract_t">Mitchell J, Paul P, Chen HJ, et al. Familial amyotrophic lateral sclerosis is associated with a mutation in D-amino acid oxidase. Proc Natl Acad Sci U S A 2010; 107:7556.</a></li><li><a class="nounderline abstract_t">Al-Chalabi A, Andersen PM, Nilsson P, et al. Deletions of the heavy neurofilament subunit tail in amyotrophic lateral sclerosis. Hum Mol Genet 1999; 8:157.</a></li><li><a class="nounderline abstract_t">Fecto F, Yan J, Vemula SP, et al. SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis. Arch Neurol 2011; 68:1440.</a></li><li><a class="nounderline abstract_t">Rubino E, Rainero I, Chiò A, et al. SQSTM1 mutations in frontotemporal lobar degeneration and amyotrophic lateral sclerosis. Neurology 2012; 79:1556.</a></li><li><a class="nounderline abstract_t">Dobson-Stone C, Hallupp M, Shahheydari H, et al. CYLD is a causative gene for frontotemporal dementia - amyotrophic lateral sclerosis. Brain 2020; 143:783.</a></li><li><a class="nounderline abstract_t">Gros-Louis F, Meijer IA, Hand CK, et al. An ALS2 gene mutation causes hereditary spastic paraplegia in a Pakistani kindred. Ann Neurol 2003; 53:144.</a></li><li><a class="nounderline abstract_t">Hadano S, Hand CK, Osuga H, et al. A gene encoding a putative GTPase regulator is mutated in familial amyotrophic lateral sclerosis 2. Nat Genet 2001; 29:166.</a></li><li><a class="nounderline abstract_t">Kress JA, Kühnlein P, Winter P, et al. Novel mutation in the ALS2 gene in juvenile amyotrophic lateral sclerosis. Ann Neurol 2005; 58:800.</a></li><li><a class="nounderline abstract_t">Sheerin UM, Schneider SA, Carr L, et al. ALS2 mutations: juvenile amyotrophic lateral sclerosis and generalized dystonia. Neurology 2014; 82:1065.</a></li><li><a class="nounderline abstract_t">Gascon GG, Chavis P, Yaghmour A, et al. Familial childhood primary lateral sclerosis with associated gaze paresis. Neuropediatrics 1995; 26:313.</a></li><li><a class="nounderline abstract_t">Eymard-Pierre E, Lesca G, Dollet S, et al. Infantile-onset ascending hereditary spastic paralysis is associated with mutations in the alsin gene. Am J Hum Genet 2002; 71:518.</a></li><li><a class="nounderline abstract_t">Hentati A, Ouahchi K, Pericak-Vance MA, et al. Linkage of a commoner form of recessive amyotrophic lateral sclerosis to chromosome 15q15-q22 markers. Neurogenetics 1998; 2:55.</a></li><li><a class="nounderline abstract_t">Orlacchio A, Babalini C, Borreca A, et al. SPATACSIN mutations cause autosomal recessive juvenile amyotrophic lateral sclerosis. Brain 2010; 133:591.</a></li><li><a class="nounderline abstract_t">Stevanin G, Santorelli FM, Azzedine H, et al. Mutations in SPG11, encoding spatacsin, are a major cause of spastic paraplegia with thin corpus callosum. Nat Genet 2007; 39:366.</a></li><li><a class="nounderline abstract_t">Al-Saif A, Al-Mohanna F, Bohlega S. A mutation in sigma-1 receptor causes juvenile amyotrophic lateral sclerosis. Ann Neurol 2011; 70:913.</a></li><li><a class="nounderline abstract_t">Puckelwartz MJ, Kessler E, Zhang Y, et al. Disruption of nesprin-1 produces an Emery Dreifuss muscular dystrophy-like phenotype in mice. Hum Mol Genet 2009; 18:607.</a></li><li><a class="nounderline abstract_t">Naruse H, Ishiura H, Mitsui J, et al. Juvenile amyotrophic lateral sclerosis with complex phenotypes associated with novel SYNE1 mutations. Amyotroph Lateral Scler Frontotemporal Degener 2021; 22:576.</a></li><li><a class="nounderline abstract_t">Nadaf SN, Chakor RT, Kothari KV, Mannan AU. Synaptic Nuclear Envelope Protein 1 (SYNE 1) Ataxia with Amyotrophic Lateral Sclerosis-like Presentation: A Novel Synaptic Nuclear Envelope Protein 1 (SYNE 1) Gene Deletion Mutation from India. Ann Indian Acad Neurol 2020; 23:539.</a></li><li><a class="nounderline abstract_t">Alazami AM, Adly N, Al Dhalaan H, Alkuraya FS. A nullimorphic ERLIN2 mutation defines a complicated hereditary spastic paraplegia locus (SPG18). Neurogenetics 2011; 12:333.</a></li><li><a class="nounderline abstract_t">Tunca C, Akçimen F, Coşkun C, et al. ERLIN1 mutations cause teenage-onset slowly progressive ALS in a large Turkish pedigree. Eur J Hum Genet 2018; 26:745.</a></li><li><a class="nounderline abstract_t">Deng HX, Chen W, Hong ST, et al. Mutations in UBQLN2 cause dominant X-linked juvenile and adult-onset ALS and ALS/dementia. Nature 2011; 477:211.</a></li><li><a class="nounderline abstract_t">Gellera C, Tiloca C, Del Bo R, et al. Ubiquilin 2 mutations in Italian patients with amyotrophic lateral sclerosis and frontotemporal dementia. J Neurol Neurosurg Psychiatry 2013; 84:183.</a></li></ol></div><div id="topicVersionRevision">Topic 5176 Version 54.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21047878" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : Rate of familial amyotrophic lateral sclerosis: a systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27830784" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Decoding ALS: from genes to mechanism.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7485175" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Points to consider: ethical, legal, and psychosocial implications of genetic testing in children and adolescents. American Society of Human Genetics Board of Directors, American College of Medical Genetics Board of Directors.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23463871" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Pathophysiological insights into ALS with C9ORF72 expansions.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27194384" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Presymptomatic ALS genetic counseling and testing: Experience and recommendations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16421333" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : A locus on chromosome 9p confers susceptibility to ALS and frontotemporal dementia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16495328" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : Familial amyotrophic lateral sclerosis with frontotemporal dementia is linked to a locus on chromosome 9p13.2-21.3.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17296840" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Three families with amyotrophic lateral sclerosis and frontotemporal dementia with evidence of linkage to chromosome 9p.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19433740" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : Chromosome 9p-linked families with frontotemporal dementia associated with motor neuron disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21944778" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Expanded GGGGCC hexanucleotide repeat in noncoding region of C9ORF72 causes chromosome 9p-linked FTD and ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21944779" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : A hexanucleotide repeat expansion in C9ORF72 is the cause of chromosome 9p21-linked ALS-FTD.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22154785" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : A C9orf72 promoter repeat expansion in a Flanders-Belgian cohort with disorders of the frontotemporal lobar degeneration-amyotrophic lateral sclerosis spectrum: a gene identification study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11015796" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : Linkage of familial amyotrophic lateral sclerosis with frontotemporal dementia to chromosome 9q21-q22.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28057713" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Genetic epidemiology of amyotrophic lateral sclerosis: a systematic review and meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22406228" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Frequency of the C9orf72 hexanucleotide repeat expansion in patients with amyotrophic lateral sclerosis and frontotemporal dementia: a cross-sectional study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22305801" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Cognitive and clinical characteristics of patients with amyotrophic lateral sclerosis carrying a C9orf72 repeat expansion: a population-based cohort study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31578300" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Prospective natural history study of C9orf72 ALS clinical characteristics and biomarkers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26725464" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : Large C9orf72 repeat expansions are seen in Chinese patients with sporadic amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24057670" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Characterization of the repeat expansion size in C9orf72 in amyotrophic lateral sclerosis and frontotemporal dementia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34070550" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Disease Mechanisms and Therapeutic Approaches in C9orf72 ALS-FTD.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25737153" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : C9ORF72 hexanucleotide repeat number in frontotemporal lobar degeneration: a genotype-phenotype correlation study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28522837" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Age-related penetrance of the C9orf72 repeat expansion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31738367" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Survival and Prognostic Factors in C9orf72 Repeat Expansion Carriers: A Systematic Review and Meta-analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8446170" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : Mutations in Cu/Zn superoxide dismutase gene are associated with familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11467054" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Current status of SOD1 mutations in familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11467054" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Current status of SOD1 mutations in familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25098641" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Development of a Smartphone App for a Genetics Website: The Amyotrophic Lateral Sclerosis Online Genetics Database (ALSoD).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32125907" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Silence superoxide dismutase 1 (SOD1): a promising therapeutic target for amyotrophic lateral sclerosis (ALS).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14506936" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Sixteen novel mutations in the Cu/Zn superoxide dismutase gene in amyotrophic lateral sclerosis: a decade of discoveries, defects and disputes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10400992" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Variation in the biochemical/biophysical properties of mutant superoxide dismutase 1 enzymes and the rate of disease progression in familial amyotrophic lateral sclerosis kindreds.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9629839" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Limited corticospinal tract involvement in amyotrophic lateral sclerosis subjects with the A4V mutation in the copper/zinc superoxide dismutase gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11465923" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Genetics of sporadic ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11465924" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Genetics of ALS in Italian families.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11369193" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Superoxide dismutase gene mutations in Italian patients with familial and sporadic amyotrophic lateral sclerosis: identification of three novel missense mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16423367" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Sporadic amyotrophic lateral sclerosis and breast cancer: Hyaline conglomerate inclusions lead to identification of SOD1 mutation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12792143" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : A4T mutation in the SOD1 gene causing familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14517684" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : Clinical and pathological studies of familial amyotrophic lateral sclerosis (FALS) with SOD1 H46R mutation in large Japanese families.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/13129803" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : A rare Cu/Zn superoxide dismutase mutation causing familial amyotrophic lateral sclerosis with variable age of onset, incomplete penetrance and a sensory neuropathy.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16476815" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : The G93C mutation in superoxide dismutase 1: clinicopathologic phenotype and prognosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7647793" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Amyotrophic lateral sclerosis associated with homozygosity for an Asp90Ala mutation in CuZn-superoxide dismutase.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8813280" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Autosomal recessive adult-onset amyotrophic lateral sclerosis associated with homozygosity for Asp90Ala CuZn-superoxide dismutase mutation. A clinical and genealogical study of 36 patients.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31314961" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Phenotype in an Infant with SOD1 Homozygous Truncating Mutation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31332433" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : SOD1 deficiency: a novel syndrome distinct from amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9556377" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : Homozygosity for Asn86Ser mutation in the CuZn-superoxide dismutase gene produces a severe clinical phenotype in a juvenile onset case of familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34788402" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : Infantile SOD1 deficiency syndrome caused by a homozygous SOD1 variant with absence of enzyme activity.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11706389" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : A novel locus for familial amyotrophic lateral sclerosis, on chromosome 18q.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32750315" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Evolution of a Human-Specific Tandem Repeat Associated with ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9497266" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : Linkage of the gene for an autosomal dominant form of juvenile amyotrophic lateral sclerosis to chromosome 9q34.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15106121" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : DNA/RNA helicase gene mutations in a form of juvenile amyotrophic lateral sclerosis (ALS4).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31957062" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : Clinical and Molecular Aspects of Senataxin Mutations in Amyotrophic Lateral Sclerosis 4.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16717225" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : Autosomal recessive ataxia with peripheral neuropathy and elevated AFP: novel mutations in SETX.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16644229" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : Senataxin, the yeast Sen1p orthologue: characterization of a unique protein in which recessive mutations cause ataxia and dominant mutations cause motor neuron disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/30198223" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : A Novel Homozygous Variant of SETX Causes Ataxia with Oculomotor Apraxia Type 2.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26811093" id="rid53" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>54 : A novel SETX gene mutation producing ataxia with oculomotor apraxia type 2.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19251627" id="rid54" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>55 : Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19251628" id="rid55" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>56 : Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20668259" id="rid56" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>57 : Frameshift and novel mutations in FUS in familial amyotrophic lateral sclerosis and ALS/dementia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26823199" id="rid57" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>58 : Genotype-phenotype relationships in familial amyotrophic lateral sclerosis with FUS/TLS mutations in Japan.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12858291" id="rid58" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>59 : Identification of two novel loci for dominantly inherited familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15060112" id="rid59" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>60 : A novel locus for late onset amyotrophic lateral sclerosis/motor neurone disease variant at 20q13.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15372378" id="rid60" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>61 : A mutation in the vesicle-trafficking protein VAPB causes late-onset spinal muscular atrophy and amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18322265" id="rid61" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>62 : New VAPB deletion variant and exclusion of VAPB mutations in familial ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20577002" id="rid62" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>63 : SOD1, ANG, VAPB, TARDBP, and FUS mutations in familial amyotrophic lateral sclerosis: genotype-phenotype correlations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15557516" id="rid63" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>64 : A novel candidate region for ALS on chromosome 14q11.2.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16501576" id="rid64" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>65 : ANG mutations segregate with familial and 'sporadic' amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17886298" id="rid65" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>66 : Angiogenin loss-of-function mutations in amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19153377" id="rid66" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>67 : A case of ALS-FTD in a large FALS pedigree with a K17I ANG mutation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18309045" id="rid67" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>68 : TDP-43 mutations in familial and sporadic amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18438952" id="rid68" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>69 : TDP-43 mutation in familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18396105" id="rid69" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>70 : TARDBP mutations in amyotrophic lateral sclerosis with TDP-43 neuropathology: a genetic and histopathological analysis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18372902" id="rid70" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>71 : TARDBP mutations in individuals with sporadic and familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18779421" id="rid71" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>72 : Two German kindreds with familial amyotrophic lateral sclerosis due to TARDBP mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22539580" id="rid72" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>73 : Phenotype and genotype analysis in amyotrophic lateral sclerosis with TARDBP gene mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20645878" id="rid73" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>74 : TARDBP mutations in frontotemporal lobar degeneration: frequency, clinical features, and disease course.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19118816" id="rid74" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>75 : Deleterious variants of FIG4, a phosphoinositide phosphatase, in patients with ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20428114" id="rid75" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>76 : Mutations of optineurin in amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31838784" id="rid76" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>77 : Novel mutation in optineurin causing aggressive ALS+/-frontotemporal dementia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21613650" id="rid77" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>78 : Novel optineurin mutations in patients with familial and sporadic amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21074290" id="rid78" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>79 : Analysis of OPTN as a causative gene for amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/36008116" id="rid79" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>80 : Exploring the phenotype of Italian patients with ALS with intermediate ATXN2 polyQ repeats.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21145000" id="rid80" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>81 : Exome sequencing reveals VCP mutations as a cause of familial ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23152587" id="rid81" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>82 : Novel mutation in VCP gene causes atypical amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15034582" id="rid82" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>83 : Inclusion body myopathy associated with Paget disease of bone and frontotemporal dementia is caused by mutant valosin-containing protein.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21892620" id="rid83" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>84 : The multiple faces of valosin-containing protein-associated diseases: inclusion body myopathy with Paget's disease of bone, frontotemporal dementia, and amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20008565" id="rid84" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>85 : Valosin-containing protein (VCP) is required for autophagy and is disrupted in VCP disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12451202" id="rid85" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>86 : Chromosome 3 linked frontotemporal dementia (FTD-3).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16807408" id="rid86" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>87 : ALS phenotypes with mutations in CHMP2B (charged multivesicular body protein 2B).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22801503" id="rid87" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>88 : Mutations in the profilin 1 gene cause familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24119685" id="rid88" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>89 : ERBB4 mutations that disrupt the neuregulin-ErbB4 pathway cause amyotrophic lateral sclerosis type 19.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23455423" id="rid89" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>90 : Mutations in prion-like domains in hnRNPA2B1 and hnRNPA1 cause multisystem proteinopathy and ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29867335" id="rid90" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>91 : Linking hnRNP Function to ALS and FTD Pathology.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24686783" id="rid91" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>92 : Mutations in the Matrin 3 gene cause familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25374358" id="rid92" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>93 : Exome-wide rare variant analysis identifies TUBA4A mutations associated with familial ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28469040" id="rid93" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>94 : Mutations in the vesicular trafficking protein annexin A11 are associated with amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26945885" id="rid94" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>95 : NEK1 mutations in familial amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27455347" id="rid95" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>96 : NEK1 variants confer susceptibility to amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29342275" id="rid96" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>97 : Hot-spot KIF5A mutations cause familial ALS.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29566793" id="rid97" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>98 : Genome-wide Analyses Identify KIF5A as a Novel ALS Gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28817800" id="rid98" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>99 : TIA1 Mutations in Amyotrophic Lateral Sclerosis and Frontotemporal Dementia Promote Phase Separation and Alter Stress Granule Dynamics.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29216908" id="rid99" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>100 : Clinical and neuropathological features of ALS/FTD with TIA1 mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/34059824" id="rid100" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>101 : Childhood amyotrophic lateral sclerosis caused by excess sphingolipid synthesis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20368421" id="rid101" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>102 : Familial amyotrophic lateral sclerosis is associated with a mutation in D-amino acid oxidase.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9931323" id="rid102" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>103 : Deletions of the heavy neurofilament subunit tail in amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22084127" id="rid103" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>104 : SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22972638" id="rid104" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>105 : SQSTM1 mutations in frontotemporal lobar degeneration and amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32185393" id="rid105" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>106 : CYLD is a causative gene for frontotemporal dementia - amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12509863" id="rid106" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>107 : An ALS2 gene mutation causes hereditary spastic paraplegia in a Pakistani kindred.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11586298" id="rid107" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>108 : A gene encoding a putative GTPase regulator is mutated in familial amyotrophic lateral sclerosis 2.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16240357" id="rid108" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>109 : Novel mutation in the ALS2 gene in juvenile amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24562058" id="rid109" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>110 : ALS2 mutations: juvenile amyotrophic lateral sclerosis and generalized dystonia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8719747" id="rid110" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>111 : Familial childhood primary lateral sclerosis with associated gaze paresis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12145748" id="rid111" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>112 : Infantile-onset ascending hereditary spastic paralysis is associated with mutations in the alsin gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9933301" id="rid112" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>113 : Linkage of a commoner form of recessive amyotrophic lateral sclerosis to chromosome 15q15-q22 markers.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20110243" id="rid113" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>114 : SPATACSIN mutations cause autosomal recessive juvenile amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17322883" id="rid114" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>115 : Mutations in SPG11, encoding spatacsin, are a major cause of spastic paraplegia with thin corpus callosum.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21842496" id="rid115" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>116 : A mutation in sigma-1 receptor causes juvenile amyotrophic lateral sclerosis.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19008300" id="rid116" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>117 : Disruption of nesprin-1 produces an Emery Dreifuss muscular dystrophy-like phenotype in mice.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32870032" id="rid117" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>118 : Juvenile amyotrophic lateral sclerosis with complex phenotypes associated with novel SYNE1 mutations.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/33223674" id="rid118" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>119 : Synaptic Nuclear Envelope Protein 1 (SYNE 1) Ataxia with Amyotrophic Lateral Sclerosis-like Presentation: A Novel Synaptic Nuclear Envelope Protein 1 (SYNE 1) Gene Deletion Mutation from India.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21796390" id="rid119" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>120 : A nullimorphic ERLIN2 mutation defines a complicated hereditary spastic paraplegia locus (SPG18).</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29453415" id="rid120" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>121 : ERLIN1 mutations cause teenage-onset slowly progressive ALS in a large Turkish pedigree.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21857683" id="rid121" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>122 : Mutations in UBQLN2 cause dominant X-linked juvenile and adult-onset ALS and ALS/dementia.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23138764" id="rid122" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>123 : Ubiquilin 2 mutations in Italian patients with amyotrophic lateral sclerosis and frontotemporal dementia.</p>
</a>
</div>
</div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
