<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Huntington disease: Genetics and pathogenesis</title><link href="/z/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/z/d/css/icofont.min.css" rel="stylesheet"/><link href="/z/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/z/d/css/animate.css" rel="stylesheet"/><link href="/z/d/css/owl.carousel.css" rel="stylesheet"/><link href="/z/d/css/magnific-popup.css" rel="stylesheet"/><link href="/z/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/z/d/css/style.css" rel="stylesheet"/><link href="/z/d/css/rtl_edited.css" rel="stylesheet"/><link href="/z/d/css/responsive.css" rel="stylesheet"/><link href="https:///uptodate_assets/main.css" rel="stylesheet"/>
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>
<meta content="" name="description"/>
<link href="/z/d/img/favicon.png" rel="icon" type="image/png">
</link></head>
<body>
<!-- Start Main Menu Area -->
<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>
<section class="teacher-area ptb-10">
<div class="container">
</div>
<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Huntington disease: Genetics and pathogenesis</h1>
<div class="utdArticleSection utdStyle" id="topicContent"><div id="topicTitle">Huntington disease: Genetics and pathogenesis</div> <div class="authorSectionElem"><div class="authorsElementsLeft"><dl id="topicContributors"><dt><span> </span>Authors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Huda Y Zoghbi, MD</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Harry T Orr, PhD</a></dd><dt><span> </span>Section Editors:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Marc C Patterson, MD, FRACP</a></dd><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">Helen V Firth, DM, FRCP, FMedSci</a></dd><dt><span> </span>Deputy Editor:</dt><dd><a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">April F Eichler, MD, MPH</a></dd></dl></div><div class="authorsElementsRight"><div id="literatureReviewDate"><div class="litReviewSingleLine"><bdi><span class="emphasis">Literature review current through:</span> Jan 2024.</bdi></div><div class="litReviewSingleLine"><bdi><span class="emphasis">This topic last updated:</span> Dec 08, 2023.</bdi></div></div></div></div><div id="topicWhatsNewContainer"></div><div id="topicText"><p class="headingAnchor" id="H1"><span class="h1">INTRODUCTION</span><span class="headingEndMark"> — </span>Unstable trinucleotide repeats are associated with a variety of neurodegenerative diseases. Nine of these disorders are associated with expansion of cytosine-adenine-guanine (CAG) repeats that encode for polyglutamine tracts in the protein products. Included in this group are Huntington disease (HD), spinobulbar muscular atrophy, dentatorubral pallidoluysian atrophy, and some of the spinocerebellar ataxias.</p><p>The most common presenting symptom of HD in adults is chorea (hence the name Huntington chorea). Other usual findings at presentation include memory deficits, affective disturbances, personality changes, and other manifestations of motor dysfunction such as parkinsonism and dystonia. Patients with juvenile-onset HD have minimal or no chorea, but develop myoclonus and seizures as well as cognitive and behavioral problems. Children also have a more rapidly progressive disease.</p><p>The genetics and pathogenesis of HD will be reviewed here. Clinical aspects and management of HD are discussed separately. (See  <a class="medical medical_review" href="/z/d/html/4907.html" rel="external">"Huntington disease: Clinical features and diagnosis"</a> and  <a class="medical medical_review" href="/z/d/html/4902.html" rel="external">"Huntington disease: Management"</a>.)</p><p class="headingAnchor" id="H2"><span class="h1">CLINICAL GENETICS</span><span class="headingEndMark"> — </span>HD is caused by expansion of the cytosine-adenine-guanine (CAG) trinucleotide repeats in the huntingtin (<em>HTT)</em> gene (also known as the HD or <em>IT15</em> gene) located on chromosome 4p16.3 that encodes the protein huntingtin [<a href="#rid1">1-3</a>]. Mutant huntingtin contains an expanded tract of glutamine residues, which is located near its amino terminal. The disease is transmitted in an autosomal dominant manner.</p><p>HD shares several clinical features with the other polyglutamine diseases:</p><p class="bulletIndent1"><span class="glyph">●</span><strong>Delayed-onset phenotype</strong> – They typically present in middle age, with progressive neuronal dysfunction and eventual neuronal loss over the ensuing 10 to 20 years.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Correlation with repeat length</strong> – The greater the number of CAG repeats on expanded alleles, the earlier the age of onset and more severe the disease.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Somatic and germline instability</strong> – The repeats show both somatic and germline instability. As a result of the latter, there can be expansion of the CAG repeat number over successive generations. This may cause earlier disease onset and a progressive worsening of the phenotype in subsequent generations, a phenomenon termed "anticipation." Anticipation is more common following paternal transmission of the disease allele.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Selective vulnerability</strong> – A certain subset of neurons is preferentially vulnerable to dysfunction, even though the relevant protein is widely expressed throughout the brain and other tissues [<a href="#rid4">4</a>].</p><p></p><p>Wildtype <em>HTT</em> alleles have 6 to 26 CAG repeat units. Intermediate alleles have 27 to 35 repeats and rarely cause disease [<a href="#rid5">5-7</a>]. However, alleles of ≥27 repeats are unstable and have a tendency to expand in future generations with paternal transmission.</p><p>The generally accepted threshold for the potential to develop HD is 36 repeats. Alleles with 36 to 39 CAG repeats have reduced penetrance and a phenotype more typically characterized by cognitive dysfunction before the onset of chorea, which may result in delayed diagnosis [<a href="#rid8">8</a>]. An individual with an allele in the 36 to 39 range is at risk for HD but may not develop symptoms; full penetrance occurs with ≥40 repeats.</p><p>Trinucleotide repeat size is estimated to account for 30 to 70 percent of the variance in age of HD symptom onset [<a href="#rid3">3,9-11</a>]. Alleles with 40 to 50 CAG repeats are found in most patients with the adult form of HD [<a href="#rid3">3</a>]. Juvenile HD is typically associated with alleles containing more than 60 CAG repeats, and some patients have more than 100 repeats. Additional factors predicting age of onset are thought to be environmental and other genetic determinants [<a href="#rid12">12,13</a>].</p><p>In addition to its effect on the age of presentation, it is generally believed that CAG repeat length is positively correlated with the rate of clinical disease progression [<a href="#rid14">14-18</a>]. There also appears to be a positive correlation between CAG repeat length and the severity of neuropathologic changes [<a href="#rid19">19</a>]. These findings suggest that large CAG expansions produce more widespread injury than smaller expansions and may affect neuronal subtypes that would otherwise be spared with smaller repeat sizes.</p><p class="headingAnchor" id="H2966144240"><span class="h1">PATHOGENESIS</span></p><p class="headingAnchor" id="H3"><span class="h2">Anticipation and transmitting parent effect</span><span class="headingEndMark"> — </span>Expansion of the cytosine-adenine-guanine (CAG) repeat number over successive generations causes an earlier and more severe phenotype, termed "anticipation."</p><p>Intergenerational transmissions are associated with either slight increases of one to four CAG units or slight decreases of one to two units [<a href="#rid1">1</a>]. However, paternal transmission can sometimes produce much larger increases, on the order of seven or more CAG repeats [<a href="#rid20">20,21</a>]. The high number of cellular divisions that occur during spermatogenesis may account for the pronounced paternal-repeat instability [<a href="#rid20">20,22</a>].</p><p>Because of the tendency for paternal transmission to result in greater expansion of the CAG repeat size, anticipation shows a major transmitting parent effect, as approximately 70 to 88 percent of symptomatic patients with juvenile HD inherit the mutant HD gene from their father [<a href="#rid23">23-25</a>].</p><p class="headingAnchor" id="H4"><span class="h2">Toxic gain of function</span><span class="headingEndMark"> — </span>Expansion of CAG repeats is thought to cause toxicity through a "gain-of-function" mechanism (ie, disease develops because the mutant form of the protein gains a function that is deleterious to the cell).</p><p>Three observations are compatible with HD being caused by a dominant gain-of-function rather than loss-of-function mechanism:</p><p class="bulletIndent1"><span class="glyph">●</span>Patients with homozygous disease have clinical manifestations and age of onset similar to those of heterozygous siblings [<a href="#rid26">26,27</a>]. By contrast, the data conflict on whether the clinical course is more severe with more rapid progression in those with homozygous disease [<a href="#rid26">26,27</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Reduction of normal huntingtin activity does <strong>not</strong> cause HD in both animal models and in humans. The clinical manifestations in mice with deletion or complete inactivation of wildtype <em>Htt</em>, the homologue of the HD gene, are different from those in humans with HD. These mice die in gestation between days 8.5 and 10.5, before the emergence of the nervous system [<a href="#rid28">28-30</a>]. Thus, huntingtin is critical for early embryonic development. Mice heterozygous for <em>Htt</em> inactivation are phenotypically normal [<a href="#rid28">28,29</a>].</p><p></p><p class="bulletIndent1">Similar findings have been noted in humans, providing further support for a gain-of-function mechanism. In addition, a patient with a breakpoint in the HD gene that led to reduced expression of huntingtin was phenotypically normal [<a href="#rid31">31</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>In mice, replacing wildtype <em>Htt</em> with a mutant <em>Htt</em> allele containing 50 glutamine repeats results in normal embryonic and brain development [<a href="#rid32">32</a>]. This finding indicates that the HD defect in humans does not mimic complete or partial <em>Htt</em> inactivation.</p><p></p><p>Of note, <em>HTT</em> residues outside of the polyglutamine tract impact disease severity, suggesting that the toxic gain of function is related to a normal function of huntingtin [<a href="#rid33">33,34</a>].</p><p class="headingAnchor" id="H1571537210"><span class="h2">Somatic expansion in striatum</span><span class="headingEndMark"> — </span>Considerable evidence supports the concept that somatic CAG/cytosine-thymine-guanine (CTG) expansion in the gene encoding huntingtin in the striatum of affected individuals occurs and plays a critical role in the heightened susceptibility of the striatum to HD [<a href="#rid35">35</a>]. In addition, deoxyribonucleic acid (DNA) repair genes significantly modify age at onset in HD [<a href="#rid36">36,37</a>], suggesting a mechanism that could contribute to somatic expansion of HTT CAG/CTG repeats [<a href="#rid38">38</a>].</p><p>A small molecule, naphthyridine-azaquinolone (NA), has been developed that selectively binds to hairpin structures formed by expanded HTT CAG/CTG repeats [<a href="#rid39">39</a>]. Injections of NA into the striatum of an HD mouse model induced contractions of the expanded repeat. Thus, somatic instability is a novel therapeutic target for HD [<a href="#rid40">40</a>].</p><p class="headingAnchor" id="H5"><span class="h2">Distribution and function of huntingtin</span><span class="headingEndMark"> — </span>Huntingtin is widely expressed throughout the brain and is present in a large number of tissues throughout the body [<a href="#rid41">41</a>]. However, the neuropathology of HD appears to be limited to the central nervous system, with neuronal cell loss and atrophy of the caudate and putamen (the neostriatum) being most prominent [<a href="#rid19">19</a>]. Thus, the distribution of huntingtin does not correlate with the pattern of neuronal cell loss seen in HD.</p><p>Wildtype huntingtin exists primarily as a soluble cytoplasmic protein in somatodendritic regions and in axons [<a href="#rid4">4,42</a>]. It is associated with cellular organelles including the Golgi apparatus, mitochondria, endoplasmic reticulum, cytoskeleton, and synaptic vesicles, and is also present to a lesser degree in the nucleus [<a href="#rid16">16</a>]. The full-length protein exists in a complex with HTT-associated protein 40 (HAP40), which supports the concept that the protein functions as a multivalent interaction hub [<a href="#rid43">43</a>].</p><p>Although it is essential for normal embryonic development, the role of wildtype huntingtin in the adult is poorly understood [<a href="#rid2">2</a>]. Huntingtin interacts with numerous proteins, raising the possibility that huntingtin is involved in multiple cellular events. Proposed functions for wildtype huntingtin include roles in the regulation of ciliogenesis, protein trafficking, vesicular transport and anchoring to the cytoskeleton, endocytosis, and postsynaptic signaling [<a href="#rid3">3,44-46</a>]. In addition, wildtype huntingtin may have a prosurvival (anti-apoptotic) function.</p><p class="headingAnchor" id="H6"><span class="h2">Mutant huntingtin</span><span class="headingEndMark"> — </span>Mutant huntingtin is widely expressed in the brain [<a href="#rid4">4,47,48</a>]. Yet, a key neuropathologic feature of HD is selective neuronal loss in the caudate and putamen (striatum).</p><p>In HD, electron microscopy reveals both cytoplasmic and nuclear abnormalities, including the presence of large neuronal intranuclear inclusions or aggregates similar to those in other polyglutamine disorders [<a href="#rid49">49-51</a>]. The aggregates are also found in dystrophic neurites. The aggregates consist of amino-terminus fragments of the expanded mutant huntingtin [<a href="#rid49">49,52</a>]. The degree of aggregation varies with the length of the polyglutamine expansion (ie, the number of CAG repeats) [<a href="#rid52">52</a>].</p><p>Formation of mutant huntingtin aggregates may occur by one or both of two proposed mechanisms [<a href="#rid3">3</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>The normal tertiary protein conformation of huntingtin may be destabilized by the presence of the expanded polyglutamine tract, leading to the formation of insoluble beta pleated sheets</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>The expanded polyglutamine tract may result in increased transglutaminase-mediated crosslinking with other polyglutamine-containing proteins, including mutant and wildtype huntingtin</p><p></p><p>While aggregation of mutant huntingtin is a pathologic hallmark of the disease process, the precise role of aggregates in the pathogenesis of HD is controversial [<a href="#rid3">3,53-55</a>]. As discussed below, the interaction of Rhes with mutant huntingtin results in both neurotoxicity and diminished aggregation of mutant huntingtin. This finding suggests that aggregation is a protective rather than a deleterious mechanism. (See <a class="local">'Rhes'</a> below.)</p><p>Experimental evidence consistently shows that the phenotypic severity of HD is directly related to levels of mutant huntingtin [<a href="#rid56">56</a>]. This concept has promoted therapeutic efforts to reduce and even reverse HD pathology and symptoms by inhibiting huntingtin expression or by altering its clearance. This approach is discussed elsewhere. (See  <a class="medical medical_review" href="/z/d/html/4902.html" rel="external">"Huntington disease: Management", section on 'Investigational therapies'</a>.)</p><p>Although the pathophysiology of neuronal loss is incompletely understood, there is evidence that mutant huntingtin may disrupt a number of intracellular pathways and thereby cause cellular demise by interfering with key components of these pathways and/or by sequestration of normal proteins into the huntingtin aggregates [<a href="#rid2">2,3</a>].</p><p>Proteins and pathways that are potentially disrupted by mutant huntingtin include the following [<a href="#rid2">2,3</a>]:</p><p class="bulletIndent1"><span class="glyph">●</span>Transcription disruption. One study showed that mutant huntingtin binds to and inhibits specificity protein 1 (Sp1), a transcription factor for cystathionine gamma-lyase (CSE), thereby leading to depletion of CSE, an enzyme necessary for the biosynthesis of the amino acid cysteine in the brain [<a href="#rid57">57</a>]. Furthermore, profoundly low levels of CSE were found in the striatum in mouse models of HD and in humans with HD; supplementation with cysteine reduced cytotoxicity in cell cultures of HD tissue and improved motor outcomes and survival in live mouse models of HD. These results suggest that mutant huntingtin causes cysteine depletion, which in turn mediates HD striatal neurodegeneration.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Activation of proteases, leading to proteolysis.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Inhibition of essential native proteins that contain polyglutamine repeats.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Reduction in protein degradation, perhaps by blocking or overloading the proteosome degradation system [<a href="#rid58">58</a>]; the importance of this mechanism is related to the central role of the ubiquitin protease system in the removal of damaged, mutated, mislocated, or misfolded proteins that could be toxic to the cell [<a href="#rid59">59</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Interference with axonal transport.</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Increased ciliogenesis [<a href="#rid44">44</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Disruption of synaptic transmission [<a href="#rid60">60,61</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Interference with the normal action of wildtype huntingtin [<a href="#rid62">62</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Alteration of tau splicing and promotion of tau hyperphosphorylation [<a href="#rid63">63-65</a>].</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Impaired nuclear-cytoplasmic transport, found during normal brain aging, has a role in HD pathogenesis [<a href="#rid66">66,67</a>].</p><p></p><p>Although not necessarily a direct consequence of mutant huntingtin, a variety of additional mechanisms may contribute to neurodegeneration and the clinical manifestations of HD [<a href="#rid3">3,68-70</a>]. These include:</p><p class="bulletIndent1"><span class="glyph">●</span>Excitotoxicity</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Metabolic dysfunction, mitochondrial dysfunction, and oxidative stress</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Promotion of apoptosis and/or autophagy</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Dysfunction of neuronal interaction and circuits, primarily involving the corticostriatal and nigrostriatal pathways</p><p></p><p class="bulletIndent1"><span class="glyph">●</span>Abnormal cerebrospinal fluid flow and impaired neuroblast migration [<a href="#rid44">44,45</a>]</p><p></p><p class="headingAnchor" id="H2414582668"><span class="h2">Role of autophagy</span><span class="headingEndMark"> — </span>Mutant huntingtin is degraded through autophagy [<a href="#rid71">71</a>]. Efforts are ongoing to explore whether the degradation pathway of mutant huntingtin can be altered therapeutically.</p><p>As an example, compounds that bind to both the mutant polyglutamine tract and to LC3B, which resides in the autophagosome, could lead to the engulfment and clearance of mutant huntingtin. Screens for small molecules that might potentially interact with mutant huntingtin and LC3B have identified compounds that enhance the clearance of mutant huntingtin but not wildtype HTT [<a href="#rid72">72</a>]. Further, testing in fruit fly and mouse models of HD demonstrated functional benefits. These results are encouraging in that they might lead to therapeutics that target the mutant protein while sparing the wildtype counterpart.</p><p class="headingAnchor" id="H7"><span class="h2">Rhes</span><span class="headingEndMark"> — </span>A possible explanation for the restriction of HD pathology mainly to the neostriatum involves the role of Rhes (Ras homolog enriched in striatum), a small guanine nucleotide-binding protein that is selectively localized to the striatum. Rhes interacts with both wildtype and mutant huntingtin, but binds more strongly with mutant huntingtin [<a href="#rid73">73</a>]. In cultured lines of human embryonic and murine brain cells, Rhes induces the small ubiquitin-like modifier (SUMO) protein to covalently attach to mutant huntingtin in a process known as sumoylation. This process reduces the aggregation of mutant huntingtin and elicits neurotoxicity.</p><p class="headingAnchor" id="H8"><span class="h1">NEUROPATHOLOGY</span><span class="headingEndMark"> — </span>The characteristic pathologic change in HD is diffuse, marked atrophy of the neostriatum that may be worse in the caudate than in the putamen [<a href="#rid19">19,74</a>]. The caudate atrophy can often be detected on brain imaging with computed tomography (CT) scan or magnetic resonance imaging (MRI). Caudate and putamen volume measurements by MRI suggest that striatal atrophy begins 9 to 20 years before the clinical diagnosis of HD [<a href="#rid75">75,76</a>].</p><p>Although a general correlation exists between clinical severity of motor impairment and the degree of neuronal loss [<a href="#rid77">77,78</a>], some symptomatic patients (5 of 163 in one postmortem series) with HD have no pathologic abnormalities [<a href="#rid74">74</a>]. Symptoms in such patients presumably are caused by cellular dysfunction related to mutant huntingtin and its associated biochemical changes.</p><p>The pathologic changes are more dramatic in early-onset HD. Affected patients typically show generalized brain atrophy and loss of cerebellar Purkinje cells [<a href="#rid19">19</a>].</p><p>At the cellular level, protein aggregates are seen both in the cytoplasm and nucleus. (See <a class="local">'Mutant huntingtin'</a> above.)</p><p class="headingAnchor" id="H2529747499"><span class="h1">SOCIETY GUIDELINE LINKS</span><span class="headingEndMark"> — </span>Links to society and government-sponsored guidelines from selected countries and regions around the world are provided separately. (See  <a class="medical medical_society_guidelines" href="/z/d/html/140090.html" rel="external">"Society guideline links: Huntington disease"</a>.)</p><p class="headingAnchor" id="H11671684"><span class="h1">INFORMATION FOR PATIENTS</span><span class="headingEndMark"> — </span>UpToDate offers two types of patient education materials, "The Basics" and "Beyond the Basics." The Basics patient education pieces are written in plain language, at the 5<sup>th</sup> to 6<sup>th</sup> grade reading level, and they answer the four or five key questions a patient might have about a given condition. These articles are best for patients who want a general overview and who prefer short, easy-to-read materials. Beyond the Basics patient education pieces are longer, more sophisticated, and more detailed. These articles are written at the 10<sup>th</sup> to 12<sup>th</sup> grade reading level and are best for patients who want in-depth information and are comfortable with some medical jargon.</p><p>Here are the patient education articles that are relevant to this topic. We encourage you to print or e-mail these topics to your patients. (You can also locate patient education articles on a variety of subjects by searching on "patient info" and the keyword(s) of interest.)</p><p class="bulletIndent1"><span class="glyph">●</span>Basics topics (see  <a class="medical medical_basics" href="/z/d/html/15820.html" rel="external">"Patient education: Huntington disease (The Basics)"</a>)</p><p></p><p class="headingAnchor" id="H9"><span class="h1">SUMMARY AND RECOMMENDATIONS</span></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Clinical genetics</strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Huntingtin (</strong><strong><em>HTT</em></strong><strong>) gene</strong> – Huntington disease (HD) is caused by expansion of the cytosine-adenine-guanine (CAG) trinucleotide repeats in the <em>HTT</em> gene (also known as the HD or <em>IT15</em> gene) located on chromosome 4p16.3 that encodes the protein huntingtin. The disease is transmitted in an autosomal dominant manner. (See <a class="local">'Clinical genetics'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Number of repeats</strong> – Wildtype <em>HTT</em> alleles have 6 to 26 CAG repeat units. Intermediate alleles have 27 to 35 repeats and rarely cause disease. The generally accepted threshold for developing HD is 36 repeats, but alleles with 36 to 39 CAG repeats have variable penetrance and more typically present with cognitive dysfunction, with chorea being less evident. Full penetrance occurs at repeat sizes ≥40. (See <a class="local">'Clinical genetics'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Genotype-phenotype correlation</strong> – Individuals with early-onset HD tend to have a large number of CAG repeats, while those developing HD late in life typically have a low repeat number. Alleles with 40 to 50 CAG repeats are found in most patients with the adult form of HD. In comparison, juvenile HD is typically associated with alleles containing more than 60 CAG repeats. (See <a class="local">'Clinical genetics'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Genetic anticipation</strong> – Expansion of the CAG repeat number over successive generations may cause an earlier and more severe phenotype, termed "anticipation." (See <a class="local">'Anticipation and transmitting parent effect'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Pathogenesis</strong></p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Wildtype huntingtin</strong> – Wildtype huntingtin is essential for normal embryonic development. While its role in adults is not completely understood, wildtype huntingtin participates in the regulation of ciliogenesis, protein trafficking, vesicular transport and anchoring to the cytoskeleton, endocytosis, and postsynaptic signaling. (See <a class="local">'Distribution and function of huntingtin'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Toxic gain of function</strong> – Expansion of CAG repeats is thought to produce a toxic "gain of function" (ie, disease develops because the mutant form of the protein gains a new function or has enhanced function that is deleterious to the cell). (See <a class="local">'Toxic gain of function'</a> above.)</p><p></p><p class="bulletIndent2">Aggregation of mutant huntingtin is a pathologic hallmark of the disease process. However, the precise role of aggregates in the pathogenesis of HD is controversial. Mutant huntingtin may disrupt a number of intracellular pathways and thereby cause cellular demise by interfering with key components of these pathways and/or by sequestration of normal proteins into the huntingtin aggregates. (See <a class="local">'Mutant huntingtin'</a> above.)</p><p></p><p class="bulletIndent2"><span class="glyph">•</span><strong>Localization to striatum</strong> – The predominant localization of HD neuropathology to the striatum may be explained by the interaction of mutant huntingtin with Rhes, a small guanine nucleotide-binding protein that is selectively localized to the striatum. (See <a class="local">'Rhes'</a> above.)</p><p></p><p class="bulletIndent1"><span class="glyph">●</span><strong>Neuropathology </strong>–<strong> </strong>The characteristic pathologic change in HD is diffuse, marked atrophy of the neostriatum that may be worse in the caudate than in the putamen. (See <a class="local">'Neuropathology'</a> above.)</p></div><div class="headingAnchor" id="references"><ol id="reference"><li><a class="nounderline abstract_t">A novel gene containing a trinucleotide repeat that is expanded and unstable on Huntington's disease chromosomes. The Huntington's Disease Collaborative Research Group. Cell 1993; 72:971.</a></li><li><a class="nounderline abstract_t">Walker FO. Huntington's disease. Lancet 2007; 369:218.</a></li><li><a class="nounderline abstract_t">Gil JM, Rego AC. Mechanisms of neurodegeneration in Huntington's disease. Eur J Neurosci 2008; 27:2803.</a></li><li><a class="nounderline abstract_t">Trottier Y, Devys D, Imbert G, et al. Cellular localization of the Huntington's disease protein and discrimination of the normal and mutated form. Nat Genet 1995; 10:104.</a></li><li><a class="nounderline abstract_t">Dayalu P, Albin RL. Huntington disease: pathogenesis and treatment. Neurol Clin 2015; 33:101.</a></li><li><a class="nounderline abstract_t">Ha AD, Jankovic J. Exploring the correlates of intermediate CAG repeats in Huntington disease. Postgrad Med 2011; 123:116.</a></li><li><a class="nounderline abstract_t">Semaka A, Kay C, Doty C, et al. CAG size-specific risk estimates for intermediate allele repeat instability in Huntington disease. J Med Genet 2013; 50:696.</a></li><li><a class="nounderline abstract_t">Heinzmann A, Sayah S, Lejeune FX, et al. Huntington's Disease with Small CAG Repeat Expansions. Mov Disord 2023; 38:1294.</a></li><li><a class="nounderline abstract_t">Andrew SE, Goldberg YP, Kremer B, et al. The relationship between trinucleotide (CAG) repeat length and clinical features of Huntington's disease. Nat Genet 1993; 4:398.</a></li><li><a class="nounderline abstract_t">Stine OC, Pleasant N, Franz ML, et al. Correlation between the onset age of Huntington's disease and length of the trinucleotide repeat in IT-15. Hum Mol Genet 1993; 2:1547.</a></li><li><a class="nounderline abstract_t">Lee JM, Ramos EM, Lee JH, et al. CAG repeat expansion in Huntington disease determines age at onset in a fully dominant fashion. Neurology 2012; 78:690.</a></li><li><a class="nounderline abstract_t">Wexler NS, Lorimer J, Porter J, et al. Venezuelan kindreds reveal that genetic and environmental factors modulate Huntington's disease age of onset. Proc Natl Acad Sci U S A 2004; 101:3498.</a></li><li><a class="nounderline abstract_t">Arning L, Kraus PH, Valentin S, et al. NR2A and NR2B receptor gene variations modify age at onset in Huntington disease. Neurogenetics 2005; 6:25.</a></li><li><a class="nounderline abstract_t">Furtado S, Suchowersky O, Rewcastle B, et al. Relationship between trinucleotide repeats and neuropathological changes in Huntington's disease. Ann Neurol 1996; 39:132.</a></li><li><a class="nounderline abstract_t">Brandt J, Bylsma FW, Gross R, et al. Trinucleotide repeat length and clinical progression in Huntington's disease. Neurology 1996; 46:527.</a></li><li><a class="nounderline abstract_t">Aylward EH, Li Q, Stine OC, et al. Longitudinal change in basal ganglia volume in patients with Huntington's disease. Neurology 1997; 48:394.</a></li><li><a class="nounderline abstract_t">Mahant N, McCusker EA, Byth K, et al. Huntington's disease: clinical correlates of disability and progression. Neurology 2003; 61:1085.</a></li><li><a class="nounderline abstract_t">Rosenblatt A, Liang KY, Zhou H, et al. The association of CAG repeat length with clinical progression in Huntington disease. Neurology 2006; 66:1016.</a></li><li><a class="nounderline abstract_t">Vonsattel JP, DiFiglia M. Huntington disease. J Neuropathol Exp Neurol 1998; 57:369.</a></li><li><a class="nounderline abstract_t">Duyao M, Ambrose C, Myers R, et al. Trinucleotide repeat length instability and age of onset in Huntington's disease. Nat Genet 1993; 4:387.</a></li><li><a class="nounderline abstract_t">Zühlke C, Riess O, Bockel B, et al. Mitotic stability and meiotic variability of the (CAG)n repeat in the Huntington disease gene. Hum Mol Genet 1993; 2:2063.</a></li><li><a class="nounderline abstract_t">Telenius H, Almqvist E, Kremer B, et al. Somatic mosaicism in sperm is associated with intergenerational (CAG)n changes in Huntington disease. Hum Mol Genet 1995; 4:189.</a></li><li><a class="nounderline abstract_t">Zühlke C, Riess O, Schröder K, et al. Expansion of the (CAG)n repeat causing Huntington's disease in 352 patients of German origin. Hum Mol Genet 1993; 2:1467.</a></li><li><a class="nounderline abstract_t">Brinkman RR, Mezei MM, Theilmann J, et al. The likelihood of being affected with Huntington disease by a particular age, for a specific CAG size. Am J Hum Genet 1997; 60:1202.</a></li><li><a class="nounderline abstract_t">Nahhas FA, Garbern J, Krajewski KM, et al. Juvenile onset Huntington disease resulting from a very large maternal expansion. Am J Med Genet A 2005; 137A:328.</a></li><li><a class="nounderline abstract_t">Wexler NS, Young AB, Tanzi RE, et al. Homozygotes for Huntington's disease. Nature 1987; 326:194.</a></li><li><a class="nounderline abstract_t">Squitieri F, Gellera C, Cannella M, et al. Homozygosity for CAG mutation in Huntington disease is associated with a more severe clinical course. Brain 2003; 126:946.</a></li><li><a class="nounderline abstract_t">Duyao MP, Auerbach AB, Ryan A, et al. Inactivation of the mouse Huntington's disease gene homolog Hdh. Science 1995; 269:407.</a></li><li><a class="nounderline abstract_t">Nasir J, Floresco SB, O'Kusky JR, et al. Targeted disruption of the Huntington's disease gene results in embryonic lethality and behavioral and morphological changes in heterozygotes. Cell 1995; 81:811.</a></li><li><a class="nounderline abstract_t">Zeitlin S, Liu JP, Chapman DL, et al. Increased apoptosis and early embryonic lethality in mice nullizygous for the Huntington's disease gene homologue. Nat Genet 1995; 11:155.</a></li><li><a class="nounderline abstract_t">Persichetti F, Carlee L, Faber PW, et al. Differential expression of normal and mutant Huntington's disease gene alleles. Neurobiol Dis 1996; 3:183.</a></li><li><a class="nounderline abstract_t">White JK, Auerbach W, Duyao MP, et al. Huntingtin is required for neurogenesis and is not impaired by the Huntington's disease CAG expansion. Nat Genet 1997; 17:404.</a></li><li><a class="nounderline abstract_t">Graham RK, Deng Y, Slow EJ, et al. Cleavage at the caspase-6 site is required for neuronal dysfunction and degeneration due to mutant huntingtin. Cell 2006; 125:1179.</a></li><li><a class="nounderline abstract_t">Gu X, Greiner ER, Mishra R, et al. Serines 13 and 16 are critical determinants of full-length human mutant huntingtin induced disease pathogenesis in HD mice. Neuron 2009; 64:828.</a></li><li><a class="nounderline abstract_t">Swami M, Hendricks AE, Gillis T, et al. Somatic expansion of the Huntington's disease CAG repeat in the brain is associated with an earlier age of disease onset. Hum Mol Genet 2009; 18:3039.</a></li><li><a class="nounderline abstract_t">Bettencourt C, Hensman-Moss D, Flower M, et al. DNA repair pathways underlie a common genetic mechanism modulating onset in polyglutamine diseases. Ann Neurol 2016; 79:983.</a></li><li><a class="nounderline abstract_t">Genetic Modifiers of Huntington’s Disease (GeM-HD) Consortium. Electronic address: <span class="__cf_email__" data-cfemail="a4c3d1d7c1c8c8c5e4ccc1c8cddc8ac9c3cc8accc5d6d2c5d6c08ac1c0d1">[email protected]</span>, Genetic Modifiers of Huntington’s Disease (GeM-HD) Consortium. CAG Repeat Not Polyglutamine Length Determines Timing of Huntington's Disease Onset. Cell 2019; 178:887.</a></li><li><a class="nounderline abstract_t">Pinto RM, Dragileva E, Kirby A, et al. Mismatch repair genes Mlh1 and Mlh3 modify CAG instability in Huntington's disease mice: genome-wide and candidate approaches. PLoS Genet 2013; 9:e1003930.</a></li><li><a class="nounderline abstract_t">Nakamori M, Panigrahi GB, Lanni S, et al. A slipped-CAG DNA-binding small molecule induces trinucleotide-repeat contractions in vivo. Nat Genet 2020; 52:146.</a></li><li><a class="nounderline abstract_t">Antonijevic IA, Bettencourt BR, Bialek P, et al. Suppression of somatic expression as a novel therapeutic approach for Huntington disease and other repeat expansion disorders. GEN Biotechnol 2022; 1:163.</a></li><li><a class="nounderline abstract_t">Carroll JB, Bates GP, Steffan J, et al. Treating the whole body in Huntington's disease. Lancet Neurol 2015; 14:1135.</a></li><li><a class="nounderline abstract_t">DiFiglia M, Sapp E, Chase K, et al. Huntingtin is a cytoplasmic protein associated with vesicles in human and rat brain neurons. Neuron 1995; 14:1075.</a></li><li><a class="nounderline abstract_t">Guo Q, Bin Huang, Cheng J, et al. The cryo-electron microscopy structure of huntingtin. Nature 2018; 555:117.</a></li><li><a class="nounderline abstract_t">Keryer G, Pineda JR, Liot G, et al. Ciliogenesis is regulated by a huntingtin-HAP1-PCM1 pathway and is altered in Huntington disease. J Clin Invest 2011; 121:4372.</a></li><li><a class="nounderline abstract_t">Liu JP, Zeitlin SO. The long and the short of aberrant ciliogenesis in Huntington disease. J Clin Invest 2011; 121:4237.</a></li><li><a class="nounderline abstract_t">Caviston JP, Holzbaur EL. Huntingtin as an essential integrator of intracellular vesicular trafficking. Trends Cell Biol 2009; 19:147.</a></li><li><a class="nounderline abstract_t">Trottier Y, Lutz Y, Stevanin G, et al. Polyglutamine expansion as a pathological epitope in Huntington's disease and four dominant cerebellar ataxias. Nature 1995; 378:403.</a></li><li><a class="nounderline abstract_t">Jou YS, Myers RM. Evidence from antibody studies that the CAG repeat in the Huntington disease gene is expressed in the protein. Hum Mol Genet 1995; 4:465.</a></li><li><a class="nounderline abstract_t">DiFiglia M, Sapp E, Chase KO, et al. Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain. Science 1997; 277:1990.</a></li><li><a class="nounderline abstract_t">Davies SW, Beardsall K, Turmaine M, et al. Are neuronal intranuclear inclusions the common neuropathology of triplet-repeat disorders with polyglutamine-repeat expansions? Lancet 1998; 351:131.</a></li><li><a class="nounderline abstract_t">Ross CA. Intranuclear neuronal inclusions: a common pathogenic mechanism for glutamine-repeat neurodegenerative diseases? Neuron 1997; 19:1147.</a></li><li><a class="nounderline abstract_t">Krobitsch S, Lindquist S. Aggregation of huntingtin in yeast varies with the length of the polyglutamine expansion and the expression of chaperone proteins. Proc Natl Acad Sci U S A 2000; 97:1589.</a></li><li><a class="nounderline abstract_t">Michalik A, Van Broeckhoven C. Pathogenesis of polyglutamine disorders: aggregation revisited. Hum Mol Genet 2003; 12 Spec No 2:R173.</a></li><li><a class="nounderline abstract_t">Saudou F, Finkbeiner S, Devys D, Greenberg ME. Huntingtin acts in the nucleus to induce apoptosis but death does not correlate with the formation of intranuclear inclusions. Cell 1998; 95:55.</a></li><li><a class="nounderline abstract_t">Okamoto S, Pouladi MA, Talantova M, et al. Balance between synaptic versus extrasynaptic NMDA receptor activity influences inclusions and neurotoxicity of mutant huntingtin. Nat Med 2009; 15:1407.</a></li><li><a class="nounderline abstract_t">Graham RK, Slow EJ, Deng Y, et al. Levels of mutant huntingtin influence the phenotypic severity of Huntington disease in YAC128 mouse models. Neurobiol Dis 2006; 21:444.</a></li><li><a class="nounderline abstract_t">Paul BD, Sbodio JI, Xu R, et al. Cystathionine γ-lyase deficiency mediates neurodegeneration in Huntington's disease. Nature 2014; 509:96.</a></li><li><a class="nounderline abstract_t">Seo H, Sonntag KC, Isacson O. Generalized brain and skin proteasome inhibition in Huntington's disease. Ann Neurol 2004; 56:319.</a></li><li><a class="nounderline abstract_t">Bence NF, Sampat RM, Kopito RR. Impairment of the ubiquitin-proteasome system by protein aggregation. Science 2001; 292:1552.</a></li><li><a class="nounderline abstract_t">Marco S, Giralt A, Petrovic MM, et al. Suppressing aberrant GluN3A expression rescues synaptic and behavioral impairments in Huntington's disease models. Nat Med 2013; 19:1030.</a></li><li><a class="nounderline abstract_t">Cummings DM, Cepeda C, Levine MS. Alterations in striatal synaptic transmission are consistent across genetic mouse models of Huntington's disease. ASN Neuro 2010; 2:e00036.</a></li><li><a class="nounderline abstract_t">Zuccato C, Ciammola A, Rigamonti D, et al. Loss of huntingtin-mediated BDNF gene transcription in Huntington's disease. Science 2001; 293:493.</a></li><li><a class="nounderline abstract_t">Fernández-Nogales M, Cabrera JR, Santos-Galindo M, et al. Huntington's disease is a four-repeat tauopathy with tau nuclear rods. Nat Med 2014; 20:881.</a></li><li><a class="nounderline abstract_t">Vuono R, Winder-Rhodes S, de Silva R, et al. The role of tau in the pathological process and clinical expression of Huntington's disease. Brain 2015; 138:1907.</a></li><li><a class="nounderline abstract_t">Gratuze M, Cisbani G, Cicchetti F, Planel E. Is Huntington's disease a tauopathy? Brain 2016; 139:1014.</a></li><li><a class="nounderline abstract_t">Gasset-Rosa F, Chillon-Marinas C, Goginashvili A, et al. Polyglutamine-Expanded Huntingtin Exacerbates Age-Related Disruption of Nuclear Integrity and Nucleocytoplasmic Transport. Neuron 2017; 94:48.</a></li><li><a class="nounderline abstract_t">Grima JC, Daigle JG, Arbez N, et al. Mutant Huntingtin Disrupts the Nuclear Pore Complex. Neuron 2017; 94:93.</a></li><li><a class="nounderline abstract_t">Mochel F, Haller RG. Energy deficit in Huntington disease: why it matters. J Clin Invest 2011; 121:493.</a></li><li><a class="nounderline abstract_t">Song W, Chen J, Petrilli A, et al. Mutant huntingtin binds the mitochondrial fission GTPase dynamin-related protein-1 and increases its enzymatic activity. Nat Med 2011; 17:377.</a></li><li><a class="nounderline abstract_t">Duan W, Jiang M, Jin J. Metabolism in HD: still a relevant mechanism? Mov Disord 2014; 29:1366.</a></li><li><a class="nounderline abstract_t">Ravikumar B, Rubinsztein DC. Role of autophagy in the clearance of mutant huntingtin: a step towards therapy? Mol Aspects Med 2006; 27:520.</a></li><li><a class="nounderline abstract_t">Li Z, Wang C, Wang Z, et al. Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds. Nature 2019; 575:203.</a></li><li><a class="nounderline abstract_t">Subramaniam S, Sixt KM, Barrow R, Snyder SH. Rhes, a striatal specific protein, mediates mutant-huntingtin cytotoxicity. Science 2009; 324:1327.</a></li><li><a class="nounderline abstract_t">Vonsattel JP, Myers RH, Stevens TJ, et al. Neuropathological classification of Huntington's disease. J Neuropathol Exp Neurol 1985; 44:559.</a></li><li><a class="nounderline abstract_t">Aylward EH, Sparks BF, Field KM, et al. Onset and rate of striatal atrophy in preclinical Huntington disease. Neurology 2004; 63:66.</a></li><li><a class="nounderline abstract_t">Kipps CM, Duggins AJ, Mahant N, et al. Progression of structural neuropathology in preclinical Huntington's disease: a tensor based morphometry study. J Neurol Neurosurg Psychiatry 2005; 76:650.</a></li><li><a class="nounderline abstract_t">Guo Z, Rudow G, Pletnikova O, et al. Striatal neuronal loss correlates with clinical motor impairment in Huntington's disease. Mov Disord 2012; 27:1379.</a></li><li><a class="nounderline abstract_t">Rosenblatt A, Abbott MH, Gourley LM, et al. Predictors of neuropathological severity in 100 patients with Huntington's disease. Ann Neurol 2003; 54:488.</a></li></ol></div><div id="topicVersionRevision">Topic 6182 Version 28.0</div></div>
<div class="row">
<div class="col-12">
<h4 style="text-align: center;padding: 20px;color: cadetblue;">References</h4>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8458085" id="rid0" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>1 : A novel gene containing a trinucleotide repeat that is expanded and unstable on Huntington's disease chromosomes. The Huntington's Disease Collaborative Research Group.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/17240289" id="rid1" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>2 : Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/18588526" id="rid2" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>3 : Mechanisms of neurodegeneration in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7647777" id="rid3" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>4 : Cellular localization of the Huntington's disease protein and discrimination of the normal and mutated form.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25432725" id="rid4" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>5 : Huntington disease: pathogenesis and treatment.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21904093" id="rid5" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>6 : Exploring the correlates of intermediate CAG repeats in Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23896435" id="rid6" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>7 : CAG size-specific risk estimates for intermediate allele repeat instability in Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/37288993" id="rid7" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>8 : Huntington's Disease with Small CAG Repeat Expansions.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8401589" id="rid8" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>9 : The relationship between trinucleotide (CAG) repeat length and clinical features of Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8268907" id="rid9" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>10 : Correlation between the onset age of Huntington's disease and length of the trinucleotide repeat in IT-15.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22323755" id="rid10" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>11 : CAG repeat expansion in Huntington disease determines age at onset in a fully dominant fashion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14993615" id="rid11" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>12 : Venezuelan kindreds reveal that genetic and environmental factors modulate Huntington's disease age of onset.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15742215" id="rid12" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>13 : NR2A and NR2B receptor gene variations modify age at onset in Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8572659" id="rid13" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>14 : Relationship between trinucleotide repeats and neuropathological changes in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8614526" id="rid14" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>15 : Trinucleotide repeat length and clinical progression in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9040728" id="rid15" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>16 : Longitudinal change in basal ganglia volume in patients with Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14581669" id="rid16" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>17 : Huntington's disease: clinical correlates of disability and progression.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16606912" id="rid17" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>18 : The association of CAG repeat length with clinical progression in Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9596408" id="rid18" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>19 : Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8401587" id="rid19" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>20 : Trinucleotide repeat length instability and age of onset in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8111374" id="rid20" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>21 : Mitotic stability and meiotic variability of the (CAG)n repeat in the Huntington disease gene.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7757066" id="rid21" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>22 : Somatic mosaicism in sperm is associated with intergenerational (CAG)n changes in Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8242072" id="rid22" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>23 : Expansion of the (CAG)n repeat causing Huntington's disease in 352 patients of German origin.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9150168" id="rid23" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>24 : The likelihood of being affected with Huntington disease by a particular age, for a specific CAG size.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16096998" id="rid24" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>25 : Juvenile onset Huntington disease resulting from a very large maternal expansion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2881213" id="rid25" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>26 : Homozygotes for Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/12615650" id="rid26" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>27 : Homozygosity for CAG mutation in Huntington disease is associated with a more severe clinical course.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7618107" id="rid27" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>28 : Inactivation of the mouse Huntington's disease gene homolog Hdh.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7774020" id="rid28" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>29 : Targeted disruption of the Huntington's disease gene results in embryonic lethality and behavioral and morphological changes in heterozygotes.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7550343" id="rid29" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>30 : Increased apoptosis and early embryonic lethality in mice nullizygous for the Huntington's disease gene homologue.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/8980018" id="rid30" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>31 : Differential expression of normal and mutant Huntington's disease gene alleles.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9398841" id="rid31" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>32 : Huntingtin is required for neurogenesis and is not impaired by the Huntington's disease CAG expansion.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16777606" id="rid32" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>33 : Cleavage at the caspase-6 site is required for neuronal dysfunction and degeneration due to mutant huntingtin.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20064390" id="rid33" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>34 : Serines 13 and 16 are critical determinants of full-length human mutant huntingtin induced disease pathogenesis in HD mice.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19465745" id="rid34" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>35 : Somatic expansion of the Huntington's disease CAG repeat in the brain is associated with an earlier age of disease onset.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/27044000" id="rid35" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>36 : DNA repair pathways underlie a common genetic mechanism modulating onset in polyglutamine diseases.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31398342" id="rid36" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>37 : CAG Repeat Not Polyglutamine Length Determines Timing of Huntington's Disease Onset.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24204323" id="rid37" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>38 : Mismatch repair genes Mlh1 and Mlh3 modify CAG instability in Huntington's disease mice: genome-wide and candidate approaches.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/32060489" id="rid38" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>39 : A slipped-CAG DNA-binding small molecule induces trinucleotide-repeat contractions in vivo.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid39" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>40 : Suppression of somatic expression as a novel therapeutic approach for Huntington disease and other repeat expansion disorders</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26466780" id="rid40" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>41 : Treating the whole body in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7748555" id="rid41" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>42 : Huntingtin is a cytoplasmic protein associated with vesicles in human and rat brain neurons.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/29466333" id="rid42" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>43 : The cryo-electron microscopy structure of huntingtin.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21985783" id="rid43" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>44 : Ciliogenesis is regulated by a huntingtin-HAP1-PCM1 pathway and is altered in Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21985782" id="rid44" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>45 : The long and the short of aberrant ciliogenesis in Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19269181" id="rid45" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>46 : Huntingtin as an essential integrator of intracellular vesicular trafficking.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7477379" id="rid46" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>47 : Polyglutamine expansion as a pathological epitope in Huntington's disease and four dominant cerebellar ataxias.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/7795604" id="rid47" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>48 : Evidence from antibody studies that the CAG repeat in the Huntington disease gene is expressed in the protein.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9302293" id="rid48" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>49 : Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9439509" id="rid49" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>50 : Are neuronal intranuclear inclusions the common neuropathology of triplet-repeat disorders with polyglutamine-repeat expansions?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9427237" id="rid50" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>51 : Intranuclear neuronal inclusions: a common pathogenic mechanism for glutamine-repeat neurodegenerative diseases?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/10677504" id="rid51" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>52 : Aggregation of huntingtin in yeast varies with the length of the polyglutamine expansion and the expression of chaperone proteins.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14504263" id="rid52" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>53 : Pathogenesis of polyglutamine disorders: aggregation revisited.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/9778247" id="rid53" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>54 : Huntingtin acts in the nucleus to induce apoptosis but death does not correlate with the formation of intranuclear inclusions.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19915593" id="rid54" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>55 : Balance between synaptic versus extrasynaptic NMDA receptor activity influences inclusions and neurotoxicity of mutant huntingtin.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16230019" id="rid55" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>56 : Levels of mutant huntingtin influence the phenotypic severity of Huntington disease in YAC128 mouse models.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/24670645" id="rid56" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>57 : Cystathionineγ-lyase deficiency mediates neurodegeneration in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15349858" id="rid57" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>58 : Generalized brain and skin proteasome inhibition in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11375494" id="rid58" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>59 : Impairment of the ubiquitin-proteasome system by protein aggregation.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/23852340" id="rid59" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>60 : Suppressing aberrant GluN3A expression rescues synaptic and behavioral impairments in Huntington's disease models.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/20585470" id="rid60" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>61 : Alterations in striatal synaptic transmission are consistent across genetic mouse models of Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/11408619" id="rid61" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>62 : Loss of huntingtin-mediated BDNF gene transcription in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25038828" id="rid62" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>63 : Huntington's disease is a four-repeat tauopathy with tau nuclear rods.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25953777" id="rid63" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>64 : The role of tau in the pathological process and clinical expression of Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/26969684" id="rid64" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>65 : Is Huntington's disease a tauopathy?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28384474" id="rid65" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>66 : Polyglutamine-Expanded Huntingtin Exacerbates Age-Related Disruption of Nuclear Integrity and Nucleocytoplasmic Transport.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/28384479" id="rid66" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>67 : Mutant Huntingtin Disrupts the Nuclear Pore Complex.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21285522" id="rid67" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>68 : Energy deficit in Huntington disease: why it matters.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/21336284" id="rid68" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>69 : Mutant huntingtin binds the mitochondrial fission GTPase dynamin-related protein-1 and increases its enzymatic activity.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/25124273" id="rid69" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>70 : Metabolism in HD: still a relevant mechanism?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/16973207" id="rid70" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>71 : Role of autophagy in the clearance of mutant huntingtin: a step towards therapy?</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/31666698" id="rid71" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>72 : Allele-selective lowering of mutant HTT protein by HTT-LC3 linker compounds.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/19498170" id="rid72" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>73 : Rhes, a striatal specific protein, mediates mutant-huntingtin cytotoxicity.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/2932539" id="rid73" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>74 : Neuropathological classification of Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15249612" id="rid74" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>75 : Onset and rate of striatal atrophy in preclinical Huntington disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/15834021" id="rid75" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>76 : Progression of structural neuropathology in preclinical Huntington's disease: a tensor based morphometry study.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/22975850" id="rid76" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>77 : Striatal neuronal loss correlates with clinical motor impairment in Huntington's disease.</p>
</a>
</div>
<div class="col-12" style="text-align: left;direction: ltr">
<a href="https://pubmed.ncbi.nlm.nih.gov/14520661" id="rid77" target="_blank">
<p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>78 : Predictors of neuropathological severity in 100 patients with Huntington's disease.</p>
</a>
</div>
</div>
</div>
</div>
</section>
</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->
<!-- End Footer Area -->
<!-- Back to top -->
<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->
<!-- Prpper JS -->
<!-- Bootstrap Min JS -->
<!-- Classy Nav Min Js -->
<!-- Owl Carousel Min Js -->
<!-- Magnific Popup JS -->
<!-- CounterUp JS -->
<!-- Waypoints JS -->
<!-- Form Validator Min JS -->
<!-- Contact Form Min JS -->
<!-- Main JS -->
<!-- Global site tag (gtag.js) - Google Analytics -->
</body>
</html>
