<!DOCTYPE html>
<html lang="fa">
 <meta content="text/html;charset=utf-8" http-equiv="content-type"/>
 <head>
  <title>
   Secondary findings from genetic testing
  </title>
  <link href="d/css/1.css" rel="stylesheet"/>
 </head>
 <body>
  <!-- Start Main Menu Area -->
  <br/>
  <br/>
  <br/>
  <br/>
  <div class="container">
   <div class="row">
   </div>
  </div>
  <section class="teacher-area ptb-10">
   <div class="container">
   </div>
   <section class="course-details-area ptb-60">
    <div class="container">
     <div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
      <h1 style="text-align:center">
       Secondary findings from genetic testing
      </h1>
      <div class="utdArticleSection utdStyle" id="topicContent">
       <div id="topicTitle">
        Secondary findings from genetic testing
       </div>
       <div class="authorSectionElem">
        <div class="authorsElementsLeft">
         <dl id="topicContributors">
          <dt>
           <span>
           </span>
           Authors:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Kurt Christensen, MPH, PhD
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Sarah S Kalia, ScM, CGC, PhD
           </a>
          </dd>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Robert C Green, MD, MPH
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Section Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Benjamin A Raby, MD, MPH
           </a>
          </dd>
          <dt>
           <span>
           </span>
           Deputy Editor:
          </dt>
          <dd>
           <a class="contributor contributor_credentials" rel="noopener noreferrer" target="_blank">
            Jennifer S Tirnauer, MD
           </a>
          </dd>
         </dl>
        </div>
        <div class="authorsElementsRight">
         <div id="literatureReviewDate">
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             Literature review current through:
            </span>
            Jan 2024.
           </bdi>
          </div>
          <div class="litReviewSingleLine">
           <bdi>
            <span class="emphasis">
             This topic last updated:
            </span>
            May 09, 2022.
           </bdi>
          </div>
         </div>
        </div>
       </div>
       <div id="topicWhatsNewContainer">
       </div>
       <div id="topicText">
        <p class="headingAnchor" id="H70123">
         <span class="h1">
          INTRODUCTION
         </span>
         <span class="headingEndMark">
          —
         </span>
         The increasing use and capabilities of genomic tools such as genome sequencing and exome sequencing raise important questions about how to handle health-related information that may inform prevention or treatment strategies, but are unrelated to the reasons testing was ordered. The questions of whether – and how – to disclose these secondary findings (also called incidental findings) from genetic testing have generated much debate, and the importance of how these questions are answered is expected to grow as laboratories and physicians transition to whole genome and whole exome sequencing rather than targeted gene panels.
        </p>
        <p>
         This topic review discusses an approach to the disclosure of secondary findings from genetic testing.
        </p>
        <p>
         Overviews of related subjects are presented separately:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Genomic disorders – (See
         <a class="medical medical_review" href="/z/d/html/16647.html" rel="external">
          "Genomic disorders: An overview"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Next generation DNA sequencing – (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Genetic testing – (See
         <a class="medical medical_review" href="/z/d/html/2901.html" rel="external">
          "Genetic testing"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Genetic counselling – (See
         <a class="medical medical_review" href="/z/d/html/103304.html" rel="external">
          "Genetic counseling: Family history interpretation and risk assessment"
         </a>
         .)
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Glossary of terms – (See
         <a class="medical medical_review" href="/z/d/html/2898.html" rel="external">
          "Genetics: Glossary of terms"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H70129">
         <span class="h1">
          DEFINITIONS AND CLASSIFICATION OF VARIANTS
         </span>
         <span class="headingEndMark">
          —
         </span>
         The term secondary finding (SF; also called "unexpected result" or "incidental finding") has been defined in several different ways [
         <a href="#rid1">
          1,2
         </a>
         ]. Narrowly defined, secondary findings are genetic variants that are unrelated to the original reason genetic testing was ordered. Here, we use a broad definition of secondary findings that includes health-informative variants that may have been identified through untargeted scans for any genetic findings of interest, including those that may have been obtained through direct-to-consumer (DTC) genetic testing services [
         <a href="#rid3">
          3
         </a>
         ].
        </p>
        <p>
         Secondary findings can be identified by any one of a number of genetic tests, including single gene tests and microarray analysis. However, secondary findings are more likely to be identified during testing with genomic sequencing methods such as genome sequencing or exome sequencing that assay all genes, as opposed to methods that assay only one or a few genes [
         <a href="#rid4">
          4
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications", section on 'Whole genome, exome, or gene panel'
         </a>
         .)
        </p>
        <p>
         When reported, secondary findings associated with disease risk are classified by the laboratory performing the testing according to their previously established or predicted pathogenicity, based on an interpretation of data from variant databases and the literature that correlates variants with clinical disease in reference populations [
         <a href="#rid5">
          5
         </a>
         ]. (See
         <a class="local">
          'Decisions made by the laboratory'
         </a>
         below.)
        </p>
        <p>
         Most laboratories classify variants with proven gene-disease associations into categories such as the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Pathogenic
         </strong>
         – Pathogenic variants are variants previously reported in patients with disease and/or are strongly suspected of being pathogenic based on preclinical studies.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Likely pathogenic
         </strong>
         – Likely pathogenic variants are those with sequence features that are likely to be implicated in disease pathogenesis but for which conclusive evidence of pathogenicity is not available.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Variant of unknown significance
         </strong>
         – Variants of unknown significance (VUS; also called "variant of uncertain significance" or "finding of unknown clinical significance") are variants that have some features suggestive of possible functional consequence, but for which there is insufficient evidence for either a pathogenic or benign role.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Likely benign
         </strong>
         – Likely benign variants are those for which weak data supporting pathogenicity may be available, but for which the majority of evidence suggests the effect of the variant is benign.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Benign
         </strong>
         – Benign findings are genetic variants not predicted to alter gene expression or function.
        </p>
        <p>
        </p>
        <p>
         These categories are not used for non-disease associations such as pharmacogenomic variants. Decisions about whether to report a variant as a secondary finding and how to classify these variants are made by individual laboratories, sometimes without input from ordering physicians or patients [
         <a href="#rid6">
          6,7
         </a>
         ].
        </p>
        <p>
         The term "pathogenic variant" is preferred over "mutation" since it provides greater clarity about whether a DNA change is disease-causing or benign and allows a uniform nomenclature to be used; a 2015 guidance document from the American College of Medical Genetics and Genomics (ACMG) and the Association for Molecular Pathology has recommended that this terminology be applied to all genetic testing (not just to secondary findings) [
         <a href="#rid5">
          5,8
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/2898.html" rel="external">
          "Genetics: Glossary of terms"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2900.html" rel="external">
          "Basic genetics concepts: DNA regulation and gene expression", section on 'Clinical classification of pathogenicity'
         </a>
         .)
        </p>
        <p>
         An often-cited list of genes published by ACMG (ie, the "ACMG list") represents genes recommended for examination for secondary findings based on criteria that emphasize consensus-based clinical validity (ie, the accuracy with which the finding predicts disease risk or presence) and utility (the ability to use the findings to impact clinical management, and risks and benefits resulting from test use)  (
         <a class="graphic graphic_table graphicRef97842" href="/z/d/graphic/97842.html" rel="external">
          table 1
         </a>
         ) [
         <a href="#rid6">
          6,9,10
         </a>
         ]. An ACMG working group updates this list periodically to add and remove genes [
         <a href="#rid11">
          11
         </a>
         ]; it has expanded in length since 2013 from 56 to 73 genes. ACMG has also emphasized that their recommendations are specific to secondary findings identified during indication-based testing rather than applications such as population screening or disclosure of individual research results [
         <a href="#rid12">
          12
         </a>
         ].
        </p>
        <p>
         Nevertheless, ACMG recommendations provide a "a minimum list of genes that should be evaluated" [
         <a href="#rid13">
          13
         </a>
         ]. In reality, most laboratories offer to query a much larger set of genes [
         <a href="#rid14">
          14
         </a>
         ]. These include conditions such as glucose-6-phosphate dehydrogenase deficiency (G6PD) and Fabry disease [
         <a href="#rid15">
          15
         </a>
         ].
        </p>
        <p>
         The French Society of Predictive and Personalized Medicine have published their own recommendations for cancer-related secondary findings, listing 36 genes rather than the 25 recommended by ACMG [
         <a href="#rid16">
          16
         </a>
         ].
        </p>
        <p>
         In addition, the Presidential Commission for the Study of Bioethical Issues in the United States has recommended that professional organizations deliberate and provide guidance for different fields and contexts [
         <a href="#rid3">
          3
         </a>
         ].
        </p>
        <p>
         Recommendations such as the ACMG list provide a list of genes to be examined for variants, rather than specific variants within those genes that should be reported, and specialists often disagree whether specific variants warrant reporting as secondary findings [
         <a href="#rid17">
          17,18
         </a>
         ]. Resources from the National Institutes of Health in the United States such as ClinGen (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fclinicalgenome.org%2F&amp;token=nbsXe3i4qfUea5RCPRHtOXAfmv8sxvNi1dzM5%2F2MCbBz%2Fjw073txTxlmmUrxvXA9&amp;TOPIC_ID=96539" target="_blank">
          clinicalgenome.org
         </a>
         ) have helped to develop standards for variant assessment and have curated the evidence base about the prevalence and penetrance of pathogenic variants in unselected populations. Importantly, these curations are tailored separately to adult and pediatric populations. Driving these efforts is the recognition that, even when a gene's association with disease risk is well established, penetrance estimates may have been developed from high-risk populations and may generalize poorly to groups such as healthy individuals in the general population. (See
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)", section on 'Penetrance and expressivity'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H70136">
         <span class="h1">
          LIKELIHOOD OF DETECTING A SECONDARY FINDING
         </span>
         <span class="headingEndMark">
          —
         </span>
         The prevalence of secondary findings depends on the population examined and the criteria used for identification and reporting; however, most studies find a prevalence between 1 and 6 percent. The following studies illustrate the range of findings:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Analyses of the United Kingdom Biobank found actionable secondary findings in 2 percent of participants when querying the 59 genes in the ACMG version 2.0 list [
         <a href="#rid19">
          19
         </a>
         ]. The third Electronic Medical Records and Genomics (eMERGE) Network identified secondary findings in the same list of genes in 2.5 percent of nearly 22,000 participants [
         <a href="#rid20">
          20
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Up to 1 percent of people in the general population have a pathogenic variant(s) in one of the original 56 genes recommended for analysis and reporting by the ACMG  (
         <a class="graphic graphic_table graphicRef97842" href="/z/d/graphic/97842.html" rel="external">
          table 1
         </a>
         ); if
         <strong>
          likely
         </strong>
         pathogenic variants are included, the percentage increases to between 1 and 3 percent [
         <a href="#rid15">
          15,21-24
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The first Clinical Sequencing Exploratory Research (CSER1) Consortium found that the prevalence of secondary findings associated with monogenic disease risks in individual studies could be as low as 0.5 percent and as high as 15 percent [
         <a href="#rid21">
          21
         </a>
         ]. Across consortium studies, 74 of 6240 individuals (1.2 percent) were identified with at least one pathogenic variant in the 56 genes recommended at the time by ACMG for disclosure of secondary findings [
         <a href="#rid25">
          25
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         An observational study in which 2000 exomes were sequenced consecutively at the Human Genome Sequencing Center at Baylor College of Medicine and analyzed using a more extensive list of medically actionable genes identified clinically significant secondary findings in 92 (4.6 percent) [
         <a href="#rid15">
          15
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Analysis of 572
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.genome.gov%2F20519355%2Fclinseq-a-largescale-medical-sequencing-clinical-research-pilot-study%2F&amp;token=pA%2FmDyfgW9gFQOhjAuc0T9zxs2kztjSRr%2FQfTNl3HGlb%2B7aPluFbNs6KdceoEQPJAbxA9DToEWcpZ3%2BHlwM39ZswaQF1NRn1GRUwYiVbHJIaYWPMdd1IpD7HN6fxHPGl&amp;TOPIC_ID=96539" target="_blank">
          ClinSeq Study
         </a>
         participants between 45 and 65 years of age, most of whom were generally healthy or had a personal history of cardiac disease, identified seven individuals (1.2 percent) with pathogenic BRCA1/2 variants and one individual with a pathogenic variant associated with paragangliomas [
         <a href="#rid26">
          26
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         In a study of 789 unaffected parents of children with developmental delays and intellectual disorders, pathogenic or likely pathogenic variants in genes on the ACMG list were identified in 11 participants (1.4 percent); when likely pathogenic variants from the
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fclinvar%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2Fy3%2BLj0shalBW%2BL1ZyL4trzGy12ipcVcj41059JRLkL3&amp;TOPIC_ID=96539" target="_blank">
          ClinVar
         </a>
         database and other recessive conditions were included, the number rose to 25 individuals (3.2 percent) included [
         <a href="#rid27">
          27
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Among 50 healthy adults who underwent whole genome sequencing in the
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.genomes2people.org%2Fresearch%2Fmedseq%2F&amp;token=o3Y2ihEPBeC5zzP5nYULCQtM2K6EJlQ3uLvU%2FhEtuYc0dxZgjdZr0JFH4%2FPwivduOGoOW06nVXMJAZUMarDcvw%3D%3D&amp;TOPIC_ID=96539" target="_blank">
          MedSeq Project
         </a>
         , one (2 percent) had a likely pathogenic variant in a gene on the ACMG list. When an additional 4500 genes were analyzed using expanded reporting criteria, variants associated with monogenic disease were identified in 13 (26 percent) [
         <a href="#rid28">
          28
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         In a study of 159 newborns who underwent exome sequencing in the BabySeq Project, pathogenic or likely pathogenic secondary monogenic disease variants associated with disorders that present or are clinically manageable during childhood were identified in 15 (9.4 percent) [
         <a href="#rid29">
          29
         </a>
         ]. An additional three of 85 newborns (3.5 percent) tested positive for pathogenic variants in five genes associated with actionable adult-onset conditions (
         <em>
          BRCA1
         </em>
         ,
         <em>
          BRCA2
         </em>
         ,
         <em>
          MLH1
         </em>
         ,
         <em>
          MSH2
         </em>
         , and
         <em>
          MSH6
         </em>
         ).
        </p>
        <p>
        </p>
        <p>
         Further, all individuals have pharmacogenomic variants that affect medication response, as well as hundreds of variants that provide information about disease susceptibility [
         <a href="#rid30">
          30,31
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/2904.html" rel="external">
          "Overview of pharmacogenomics"
         </a>
         .)
        </p>
        <p>
         In addition, between 25 and 50 percent of individuals are carriers for at least one severe recessive childhood disorder, although estimates as high as 2.8 carrier variants per person have been noted [
         <a href="#rid30">
          30,32-34
         </a>
         ]. The percentage of adults who are carriers for at least one recessive disorder is estimated at 90 to 95 percent when criteria are expanded to include mild and adult-onset disorders [
         <a href="#rid21">
          21,33-35
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H70143">
         <span class="h1">
          INFORMED CONSENT
         </span>
         <span class="headingEndMark">
          —
         </span>
         Consensus exists that patients should be informed about the potential for identifying secondary findings prior to undergoing genetic testing, including what kinds of findings could be detected, which findings would be disclosed, and how the information would be communicated [
         <a href="#rid36">
          36-39
         </a>
         ].
        </p>
        <p>
         Discussions between patients and the clinicians caring for them about genomic findings should culminate in joint decisions that account for the patient's clinical status, values, and stated preferences [
         <a href="#rid3">
          3,40
         </a>
         ]. In the cases where genomic sequencing is used, ACMG recommends counseling by a medical geneticist or genetic counselor prior to testing, as described below. This step would include notification that secondary findings may be identified and clarification about what kinds of findings would be disclosed [
         <a href="#rid41">
          41
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H3242041636">
         <span class="h2">
          Importance of pretest counseling
         </span>
         <span class="headingEndMark">
          —
         </span>
         The National Society of Genetic Counselors (NSGC) has released a position statement emphasizing the importance of pretest counseling to ensure patients make informed decisions about receiving results related to secondary findings [
         <a href="#rid42">
          42
         </a>
         ]. Recommended components of pretest counseling include anticipatory guidance for categories of results that will or will not be returned, limitations of analysis for secondary findings (see
         <a class="local">
          'Interpretation of 'negative' results'
         </a>
         below), discussion of patient preferences regarding secondary findings, and development of a plan for returning these results.
        </p>
        <p>
         Importantly, genomic sequencing does not substitute for family history analysis in assessing disease risk. This was illustrated in a study that compared family history–based risk assessment with single nucleotide polymorphism (SNP)-based testing for breast, prostate, and colon cancer risk [
         <a href="#rid43">
          43
         </a>
         ]. Family history–based assessment classified 22 of 88 participants as high-risk, while SNP-based testing only identified one high-risk patient. (See
         <a class="medical medical_review" href="/z/d/html/103304.html" rel="external">
          "Genetic counseling: Family history interpretation and risk assessment"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H3481946671">
         <span class="h2">
          Potential harms
         </span>
         <span class="headingEndMark">
          —
         </span>
         Informed consent for sequencing and the accompanying analysis of secondary findings should also include discussions of the likelihood of detecting additional variants, the types of findings that will and will not be disclosed, the patient's preferences for learning these types of information, and a plan for disclosing the results [
         <a href="#rid41">
          41
         </a>
         ].
        </p>
        <p>
         Potential harms of disclosure of secondary findings include psychological distress from learning of disease risk, financial and personal costs of additional testing that may be indicated, lifestyle changes to medications that are unwarranted or potentially dangerous (eg, patient-initiated medication changes in response to pharmacogenomic information), and adverse consequences of therapeutic interventions for which evidence of benefit is lacking in patients diagnosed incidentally:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Federal legislation protects patients from discrimination related to employment or health insurance based on genetic test results. Similar protections are lacking for other types of insurance (eg, life, disability, and long-term care insurance) where insurers may have the rights to ask for findings from prior genomic testing [
         <a href="#rid44">
          44,45
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         There may be additional psychological harms from learning about a risk of disease for which there is no effective intervention (eg, Huntington disease), or risks to privacy [
         <a href="#rid1">
          1,2
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         In addition, physicians' experiences with and understandings about genetics are often limited, raising risks that reports of secondary findings will be misinterpreted, particularly if the finding relates to a rare condition [
         <a href="#rid18">
          18,46-48
         </a>
         ]. Conversely, a report of no secondary findings detected may falsely reassure an individual that s/he is not at risk of disease and cause the individual to ignore personal symptoms or a strong family history of disease. Of note, the evidence threshold laboratories use for reporting a variant to physicians and patients tends to be much higher in the context of secondary findings compared with variants being analyzed for diagnostic purposes. (See
         <a class="local">
          'Interpretation of 'negative' results'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H3120625269">
         <span class="h2">
          Psychological distress
         </span>
         <span class="headingEndMark">
          —
         </span>
         The potential for psychological distress may vary according to disease risk, availability of prevention strategies, and whether the results were anticipated or not. An example of a setting in which distress was not increased comes from a trial involving 257 adults who were interested in learning about genetic risk for late onset Alzheimer disease (AD) [
         <a href="#rid49">
          49
         </a>
         ]. Participants had genotyping of their apolipoprotein E (
         <em>
          APOE
         </em>
         ) gene, which is associated with risk of late onset AD and risk of cardiac disease. Participants were randomly assigned to receive information only about their associated AD risk, or about their AD risk plus their risk for cardiac disease, based on
         <em>
          APOE
         </em>
         genotype. Compared with controls, those assigned to receive additional unexpected information about increased cardiac risk had lower scores for test-related distress; scores for anxiety and depression were well below clinically significant cutoffs in all participants. In addition, test-related distress was lower among those who learned they were at increased risk for both cardiac disease and AD compared with those who learned about an increased risk for AD only. An editorialist noted that low psychological distress may not be the most compelling reason for disclosure, and suggested a framework for population-based prioritization of results for disclosure that would be based on the predictive value of genetic risk information and the availability of highly effective prevention strategies [
         <a href="#rid50">
          50
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H2397465120">
         <span class="h2">
          Opt out option
         </span>
         <span class="headingEndMark">
          —
         </span>
         Patients should be provided the option to receive secondary findings or the option to "opt out" of screening for secondary findings. For patients who decline secondary findings, laboratories would ideally restrict bioinformatics analysis to genes relevant to the reasons testing was ordered. Individuals may also opt out of disclosure of secondary findings after analysis has occurred. Many experts believe that an individual who wishes to opt out of receiving certain findings is better served by opting out of genomic analysis, in order to preserve patient privacy. In two large studies that used exome sequencing for diagnosis in suspected genomic disorders, 1 to 8 percent of individuals opted out of receiving information on secondary findings, with the higher percentage of opt outs for information about recessive disorders and pharmacogenomic variants [
         <a href="#rid15">
          15,51
         </a>
         ]. Regardless of whether a patient has chosen to opt out of genetic analysis or disclosure, ordering physicians may inform patients that their preferences for receiving secondary findings will be re-assessed at the time when primary findings are disclosed.
        </p>
        <p class="headingAnchor" id="H755629553">
         <span class="h2">
          After death
         </span>
         <span class="headingEndMark">
          —
         </span>
         A final consideration for the consenting process is what to do with secondary findings after death. Patients and research participants often express wishes that any genetic risk information about them be shared with relatives after their death [
         <a href="#rid52">
          52
         </a>
         ]. However, in the United States, health privacy regulations in the Health Insurance Portability and Accountability Act (HIPAA) prohibit such disclosure without prior authorization from the decedent or a legal representative [
         <a href="#rid53">
          53
         </a>
         ]. Addressing this issue during the informed consent process may help patients initiate discussions with relatives about whether they would want to receive postmortem disclosure of secondary findings, along with any other medical information that might be critical to interpreting the clinical significance of a potentially disease-associated variant. (See
         <a class="local">
          'Postmortem disclosure'
         </a>
         below.)
        </p>
        <p>
         Additional ethical and psychosocial issues are discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/2901.html" rel="external">
          "Genetic testing", section on 'Ethical, legal, and psychosocial issues'
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H70150">
         <span class="h1">
          DISCLOSURE OF SECONDARY FINDINGS TO PATIENTS
         </span>
        </p>
        <p class="headingAnchor" id="H358242275">
         <span class="h2">
          Decisions made by the laboratory
         </span>
         <span class="headingEndMark">
          —
         </span>
         Each laboratory performing genomic testing makes decisions regarding which results are reported to the clinician. Unlike more straightforward clinical laboratory testing (eg, measurement of a serum protein concentration), genomic testing measures sequence variation at the nucleotide level in a very large portion of the genome; laboratories determining genetic sequence information are responsible for providing the interpretation of the clinical relevance of the variants identified.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Each laboratory typically has its own policies for identifying, analyzing, and reporting secondary findings. When genomic sequencing is used, a laboratory may have a pre-specified panel of genes it will examine for and report variants for unless instructed to do otherwise by the ordering physician; in some cases, laboratories may have a policy in place to allow clinicians or patients to opt out of receiving secondary findings or to allow them to receive an expanded panel of results upon request (eg, carrier status variants, pharmacogenomic variants). Laboratories have also been encouraged to develop policies about disclosing secondary findings to parents/caregivers or other third parties, although consensus is lacking about what these policies should be [
         <a href="#rid54">
          54
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         Laboratories are also encouraged to establish criteria and thresholds for reporting findings associated with consanguinity, such as multiple regions of extended homozygosity, because these findings can increase risks for autosomal recessive conditions [
         <a href="#rid55">
          55-57
         </a>
         ]. In circumstances where genetic testing results suggest possible conception between first- or second-degree relatives, laboratories are encouraged to discuss their findings with the clinician who ordered the test, because the clinician may have legal obligations for reporting potential rape or sexual assault [
         <a href="#rid57">
          57
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Many laboratories have also used Level A guidance for gene/drug pairs, as assessed by the Clinical Pharmacogenetics Implementation Consortium (CPIC), as the basis for offering pharmacogenomic secondary findings. (See
         <a class="medical medical_review" href="/z/d/html/2904.html" rel="external">
          "Overview of pharmacogenomics"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Each laboratory decides what risk assessment to assign to each disease-causing variant within the gene; this is generally done according to an accepted classification scheme based on the likely clinical significance of each variant (see
         <a class="local">
          'Definitions and classification of variants'
         </a>
         above). Interpretations of risk are generally made by reviewing published and/or proprietary epidemiologic, genetic, and biologic data, and using algorithms to predict the functional significance of certain variants. Patient information (eg, laboratory results, personal and family history of disease) may also inform how variants are classified.
        </p>
        <p>
        </p>
        <p>
         There may be relatively broad consensus in reporting a known pathogenic variant in a known cancer gene (eg, pathogenic BRCA1 mutation). However, laboratory assessments regarding the significance of a variant may diverge when the pathogenicity of the variant has not been established previously. This divergence in assessments was illustrated in a study that compared results from two genomics laboratories using a genotype-panel to test DNA from five individuals [
         <a href="#rid58">
          58
         </a>
         ]. The information reporting the genetic sequence was concordant for more than 99 percent of markers tested, but the risk prediction was concordant across all five subjects for only 4 of 13 medical conditions. Of note, this study evaluated susceptibility variants for common conditions as well as Mendelian diseases, which may have contributed to the high degree of disparity in risk predictions.
        </p>
        <p>
         Most laboratories will report a genomic finding if the sequence variant is "previously reported and is a recognized cause of the disorder" (ie, pathogenic), and many will report a variant that is "previously unreported and is of the type which is expected to cause the disorder" (ie, likely pathogenic) [
         <a href="#rid5">
          5
         </a>
         ]. However, estimates of the predictive value of testing, penetrance of specific variants, spectrum of possible phenotypes, and efficacy of interventions may have been developed from research on sick patients and may not generalize well to asymptomatic populations. Laboratories should consider the patient's baseline risk and ethnic background when they assess the pathogenicity of a particular variant [
         <a href="#rid59">
          59
         </a>
         ].
        </p>
        <p>
         Experts have called for the development of standards for the identification and reporting of secondary findings from genomic testing, and recommendations continue to evolve [
         <a href="#rid60">
          60
         </a>
         ]. Models and best practices have been emerging about how to integrate secondary findings into patient care [
         <a href="#rid61">
          61
         </a>
         ]. In some cases, the US Food and Drug Administration (FDA) has issued warnings about providing pharmacogenomic information where evidence of clinical validity and utility are limited [
         <a href="#rid62">
          62,63
         </a>
         ]. In the interim, individual laboratories are encouraged to develop clear policies based on their own assessment of existing evidence [
         <a href="#rid6">
          6,11
         </a>
         ]. In all situations, laboratories are strongly encouraged to set high thresholds for defining pathogenicity for patients who have no prior personal or family history suggesting a condition with a strong genetic component [
         <a href="#rid64">
          64
         </a>
         ]. Physicians will need to take these considerations into account when they decide whether and how to disclose secondary findings to their patients. (See
         <a class="local">
          'Review patient report'
         </a>
         below.)
        </p>
        <p class="headingAnchor" id="H1593125">
         <span class="h2">
          Review patient report
         </span>
         <span class="headingEndMark">
          —
         </span>
         Prior to meeting with the patient, the clinician should do the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Review patient preferences communicated at the time of consent for genomic testing (eg, did the patient opt out of receiving certain results?).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Weigh the potential harms and benefits of reporting the secondary finding.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Understand the clinical implications of the finding for the patient.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Reevaluate the patient's personal medical history, family history, and physical examination in light of the finding, if appropriate.
        </p>
        <p>
        </p>
        <p>
         Importantly, patient preferences for information and the clinical significance of a secondary finding from genetic testing may change over time. The decision about whether and when to disclose secondary findings will thus depend on the clinical context and the judgment of clinicians involved in the patient's care. Additionally, ethical obligations to respect patients' autonomy and right not to know information must be weighed against fiduciary duties to inform patients of potentially life-saving information [
         <a href="#rid65">
          65-71
         </a>
         ]. In the United States, a Presidential Commission has urged professional organizations to develop guidelines for responding to secondary findings from genomic testing that address these ethical and legal issues [
         <a href="#rid3">
          3
         </a>
         ]. For patients who have received genetic testing from a direct-to-consumer (DTC) source, clinicians may want to address how patients may have consented to allow the company to share their results with other parties, although there is no obligation to do so [
         <a href="#rid72">
          72
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/15076.html" rel="external">
          "Informed procedural consent", section on 'The duty to inform'
         </a>
         and
         <a class="local">
          'Liability concerns'
         </a>
         below.)
        </p>
        <p>
         In addition, physicians will need to consider the secondary finding in the context of the patient's primary reason for testing. As an example, patients for whom testing was done for a more immediate problem may be unreceptive to learning about additional results or poorly able to process the information until the primary indication for genomic testing has been addressed.
        </p>
        <p>
         Whether to disclose results indicating consanguinity is a question that may warrant special consideration, given varying norms about marriage of related individuals. Clinicians will need to consider how disclosure affects the physician-patient relationship, the psychosocial impact on patients, and/or any ethical or legal obligations for reporting that may exist [
         <a href="#rid73">
          73
         </a>
         ]. It may be helpful to disclose results with a team that includes a geneticist, genetic counselor, and patient advocate to minimize the potential for distress and to ensure the individuals understand the implications for disease [
         <a href="#rid73">
          73
         </a>
         ]. If either parent of a suspected consanguineous conception was a minor at the time of conception, a child-protection team should be consulted as necessary to understand legal obligations for reporting potential rape or sexual assault [
         <a href="#rid57">
          57
         </a>
         ].
        </p>
        <p>
         The clinical relevance of a variant for the patient should always be interpreted in the context of the patient's personal medical history and family history of disease, as well as physical examination and other relevant information (eg, overall health, behavioral risks, reproductive issues). Some genomic findings may help to explain prior diagnoses of the patients or their relatives. In some cases, repeating the physical examination for subtle manifestations associated with disease may be appropriate.
        </p>
        <p>
         The risk of disease is also influenced by the disease penetrance and expressivity, which varies by patient population:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Variants with well-established pathogenicity may have reduced penetrance (ie, an individual with such a variant may never develop disease) or variable expressivity (ie, differences in the severity of the phenotype or the specific disease manifestations that develop). (See
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)", section on 'Penetrance and expressivity'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The predictive value of testing, spectrum of phenotypes, and efficacy of interventions in asymptomatic populations may not be well-established, because clinical information about variants is often drawn from epidemiological research on individuals with classic disease phenotypes and/or families with a strong history of disease [
         <a href="#rid65">
          65
         </a>
         ]. As a consequence, penetrance estimates for disease variants are overstated in some databases and publications [
         <a href="#rid4">
          4,40,74,75
         </a>
         ]. Databases and resources that provide information more applicable to the general population are being developed. (See
         <a class="local">
          'Online information and resources'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Information regarding interpretation of disease variants may also be less relevant for members of ethnicities that are underrepresented in reference populations. (See
         <a class="local">
          'Underrepresented ethnicities'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p>
         A sample report that illustrates these concepts is provided in the figure  (
         <a class="graphic graphic_figure graphicRef122538" href="/z/d/graphic/122538.html" rel="external">
          figure 1
         </a>
         ).
        </p>
        <p>
         An example of a decision matrix regarding which secondary findings should be disclosed is presented in the table  (
         <a class="graphic graphic_table graphicRef97881" href="/z/d/graphic/97881.html" rel="external">
          table 2
         </a>
         ). This approach classifies variants into 'bins' based on their potential clinical utility to the patient, clinical validity (accuracy of predicting disease for that patient), and risks that disclosure may pose (eg, psychological distress, familial discord). It proposes routinely disclosing secondary findings of established utility and clear or presumed pathogenicity. Findings with clear validity but questionable utility to the patient could be disclosed depending on the potential harms of disclosure and patient preferences. Finally, findings without established validity and utility would be withheld altogether [
         <a href="#rid76">
          76
         </a>
         ]. The use of such a decision matrix may make genomic information less overwhelming [
         <a href="#rid77">
          77
         </a>
         ]. However, such algorithms may be difficult to implement consistently, given that consensus about how to classify variants is often lacking and patient preferences may change over time [
         <a href="#rid17">
          17
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H1838038385">
         <span class="h2">
          Understand implications of report
         </span>
         <span class="headingEndMark">
          —
         </span>
         There are several aspects of the report that should be understood by the clinician:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Sequence variant nomenclature
         </strong>
         – Specific genotype and allele terminology is used to describe the DNA sequence variant (and in some cases, the correlated changes to the protein sequence or structure). Examples are provided in the sample report  (
         <a class="graphic graphic_figure graphicRef122538" href="/z/d/graphic/122538.html" rel="external">
          figure 1
         </a>
         ). These may include specific nucleotide substitutions, insertions/deletions, and other changes. A listing of symbols that may be used in the report and their meanings is provided by the Human Genome Variation Society (eg,
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fvarnomen.hgvs.org%2F&amp;token=qOa16bczYaCEMoJAP%2BGLadSSPF3e3xCL8NBIPY%2B3obmmqIskz8Xd%2BhKjiqWWTOMi&amp;TOPIC_ID=96539" target="_blank">
          http://varnomen.hgvs.org/
         </a>
         and
         <a class="external" href="/external-redirect?target_url=http%3A%2F%2Fvarnomen.hgvs.org%2Frecommendations%2Fgeneral%2F&amp;token=sDDnR4qz7XFDzpMyVZx%2B8ct7NIjQmS%2BhV9TXrlHt19B5%2BXLewBK7YYYkoeIK2DgUOiZzcHkf61hjOOvtioSRJw%3D%3D&amp;TOPIC_ID=96539" target="_blank">
          http://varnomen.hgvs.org/recommendations/general/
         </a>
         ).
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Pathogenicity
         </strong>
         – Not all sequence variants have a well-established association with disease. Some reports may include variants of uncertain significance whereas others may only report variants known to be highly associated with a disease. The categories used to classify the pathogenicity of gene variants are summarized and discussed above. (See
         <a class="local">
          'Definitions and classification of variants'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Penetrance
         </strong>
         – Even if a variant is pathogenic, that does not necessarily mean the individual who carries that variant is affected by the associated disease or will develop the disease in the future. The risk of disease development depends on the penetrance of the genotype; the higher the penetrance, the greater the risk of developing disease symptoms. It is critical to base an individual's risk assessment on the pathogenicity
         <strong>
          and
         </strong>
         penetrance of the variant, as well as the individual's family history and other risk factors, in order to enable informed decisions about managing disease risk. (See
         <a class="medical medical_review" href="/z/d/html/2898.html" rel="external">
          "Genetics: Glossary of terms", section on 'Penetrance'
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)", section on 'Penetrance and expressivity'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Inheritance
         </strong>
         – The inheritance of monogenic (single gene) disorders typically depends on whether disease develops in the presence of a pathogenic variant in one or both copies (maternally and paternally derived copies) of the gene. The inheritance pattern determines the risk for other family members. The common inheritance patterns (autosomal dominant, autosomal recessive, and X-linked) and their implications for family members are discussed in more detail separately. (See
         <a class="medical medical_review" href="/z/d/html/2897.html" rel="external">
          "Inheritance patterns of monogenic disorders (Mendelian and non-Mendelian)"
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H98930240">
         <span class="h2">
          Informing and counseling the patient
         </span>
         <span class="headingEndMark">
          —
         </span>
         Disclosure of secondary findings involves informing the patient that the findings are present, counseling regarding the clinical implications of the findings, and discussing whether any additional intervention is needed. Patients appear to prefer the term "additional findings" over other terms including "secondary findings" [
         <a href="#rid78">
          78
         </a>
         ].
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Expectations
         </strong>
         – Disclosure can be complicated by unrealistic expectations among patients or their families/caregivers about the capabilities of genomic sequencing and disappointment when sequencing does
         <strong>
          not
         </strong>
         identify secondary findings [
         <a href="#rid79">
          79
         </a>
         ]. Clinicians may need to be particularly careful about patients' expectations if the individual had received testing through a direct-to-consumer (DTC) provider where pretest counseling may not have emphasized the limitations of testing as thoroughly as would be done in a genetic counseling session [
         <a href="#rid72">
          72
         </a>
         ]. Physicians who receive genomic reports must document whether and how they have acted upon the findings and their rationale, regardless of whether or not the patient has chosen to receive the information.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Additional testing
         </strong>
         – Importantly, detection of a pathogenic or likely pathogenic variant provides information about risk, not diagnosis of a disease, and the identification of a genetic variant is not a substitute for further diagnostic testing, if indicated. Conversely, negative reports from genomic testing may also require counseling to discuss the interpretation and further diagnostic steps. (See
         <a class="local">
          'Additional evaluation'
         </a>
         below and
         <a class="local">
          'Referrals'
         </a>
         below and
         <a class="local">
          'Interpretation of 'negative' results'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Potential responses
         </strong>
         – There are a variety of potential responses to a report of a secondary finding:
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         More intensive screening may be indicated. As an example, colonoscopy screening for colon cancer may be initiated at an earlier age than recommended for the general population in a patient with a hereditary nonpolyposis colon cancer (HNPCC) gene variant associated with colon cancer predisposition. (See
         <a class="local">
          'Additional evaluation'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         Prophylactic surgery may be considered. As an example, prophylactic hysterectomy and bilateral oophorectomy may be performed in a woman with a
         <em>
          BRCA1
         </em>
         mutation who has completed childbearing.
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         Carrier screening for a partner may be performed for a patient of reproductive age. As an example, if a patient has a recessive mutation for a disorder with a high carrier frequency in the population (eg, sickle cell disease in African Americans, cystic fibrosis in non-Hispanic White people of Northern European descent, or Gaucher disease Type 1 in Ashkenazi Jews), the partner may be screened and results incorporated into preconception counseling discussions.
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         The patient may undertake lifestyle modification for risk reduction. As an example, a patient with a gene variant associated with increased risk for melanoma may have increased adherence to screening recommendations for skin cancer.
        </p>
        <p>
        </p>
        <p class="bulletIndent2">
         <span class="glyph">
          •
         </span>
         Genetic counseling may be appropriate. Examples include addressing uncertainty about how to interpret a secondary finding and the impact of the variant on clinical management; assisting a patient in choosing among multiple options for which there is clinical equipoise; and helping a patient in planning discussions regarding the information with relatives who may share the genetic risk.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Surgery
         </strong>
         – One of the most challenging issues is the management of a patient with a known pathogenic variant for a potentially life-threatening disease (eg, pathogenic variant in
         <em>
          BRCA1
         </em>
         and risk of breast or ovarian cancer) for which the most effective intervention is major surgery. Evidence to guide management may exist for patients with a personal or family history of the disease, but comparable evidence is lacking for unselected populations [
         <a href="#rid65">
          65
         </a>
         ]. Clinicians may need to make critical decisions about how to respond to such findings despite a lack of consensus about how to interpret them [
         <a href="#rid74">
          74
         </a>
         ]. (See
         <a class="local">
          'Informed consent'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <strong>
          Time
         </strong>
         – The amount of time required to review secondary findings will depend on the context that warranted genetic testing in the first place and the content of the report. When testing was initiated for diagnostic or treatment purposes, it may be appropriate to postpone disclosure of nonurgent findings until primary concerns have been addressed and the patient is better able to understand and react to the information [
         <a href="#rid80">
          80
         </a>
         ]. Importantly, the nature of secondary findings means that the physician disclosing the result may have little or no prior knowledge about the condition. Furthermore, the extent to which health systems provide support to assist clinicians in their use of genomic information may vary from institution to institution (eg, the incorporation of findings into genomic indicators that adjust surveillance recommendations or the use of decision support to consider pharmacogenomic information when a relevant medication is ordered) [
         <a href="#rid45">
          45
         </a>
         ]. Physicians may need to allocate time to self-educate in preparation for disclosure of information.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Family members
         </strong>
         – Clinicians will need to consider how to enable patients to communicate information to family members. Genetic services often provide patients with written materials to facilitate sharing information with others. Primary care physicians can encourage patients to communicate with family members by checking in with patients to see how the information is flowing through the family and identifying possible barriers to communication [
         <a href="#rid46">
          46
         </a>
         ]. In some cases, patients may designate family members to receive their genomic information upon their death. (See
         <a class="local">
          'Electronic data storage'
         </a>
         below and
         <a class="local">
          'Postmortem disclosure'
         </a>
         below.)
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H70164">
         <span class="h2">
          Interpretation of 'negative' results
         </span>
         <span class="headingEndMark">
          —
         </span>
         A "negative" genomic sequencing report does
         <strong>
          not
         </strong>
         imply that an individual is not at increased risk of developing disease(s). Reasons for this include the following:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Most individuals carry dozens, if not hundreds, of variants that have not been characterized due to lack of evidence, but which may predispose them to disease. Such variants may be readily detected through genomic sequencing, but laboratories are discouraged from reporting variants that have limited evidence for pathogenicity [
         <a href="#rid59">
          59
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         Genome sequencing has a number of technical limitations. It reliably detects single nucleotide variants and small insertions or deletions but can be less reliable for detecting moderate-sized insertions or deletions. Sequencing, particularly whole exome sequencing, may fail to detect larger insertions, deletions, repeat expansions, and rearrangements [
         <a href="#rid4">
          4,74,81
         </a>
         ].These shortcomings may be far less prevalent as long-read sequencing platforms become more common [
         <a href="#rid82">
          82
         </a>
         ]. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications", section on 'Technical considerations'
         </a>
         .)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         The search for secondary findings during genomic sequencing usually is not as complete as a targeted gene query that may use Sanger sequencing to detect single nucleotide changes and insertions/deletions, or as array comparative genomic hybridization (CGH) to detect larger structural variation. Other differences between genomic sequencing and Sanger sequencing are discussed separately. (See
         <a class="medical medical_review" href="/z/d/html/17039.html" rel="external">
          "Next-generation DNA sequencing (NGS): Principles and clinical applications", section on 'Terminology and evolution of technologies'
         </a>
         .)
        </p>
        <p>
        </p>
        <p>
         Thus, if a patient has, or develops, a personal or family history that is consistent with a specific genetic condition, referral to a genetics specialist is recommended; and targeted gene analysis may be appropriate. (See
         <a class="local">
          'Referrals'
         </a>
         below and
         <a class="medical medical_review" href="/z/d/html/103304.html" rel="external">
          "Genetic counseling: Family history interpretation and risk assessment"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H1593304">
         <span class="h1">
          FOLLOW-UP CARE AND RECORD KEEPING
         </span>
         <span class="headingEndMark">
          —
         </span>
         Genomic data may be useful to the patient for current as well as future management, and the clinician should ensure that the patient understands how secondary findings from genomic testing might be used in the present and the future. As an example, a pharmacogenomic variant may not have immediate relevance but might affect future medication management. An ongoing challenge is how to integrate genomic data throughout the patient's life.
        </p>
        <p class="headingAnchor" id="H358243988">
         <span class="h2">
          Additional evaluation
         </span>
         <span class="headingEndMark">
          —
         </span>
         False-positive rates are relatively low in genomic sequencing when variants are substitutions (0.1 to 3 percent), although deletions and insertions have higher error rates ( 0.5 to 15 percent) [
         <a href="#rid83">
          83
         </a>
         ]. False-positive rates are much higher when variants are identified using array-based approaches.
        </p>
        <p>
         To minimize the potential for false-positive results, clinicians should address with the laboratory whether reported secondary findings had been confirmed through an orthogonal testing strategy (a strategy that uses an alternative approach and thus is not subject to the same technical and interpretation biases, such as using Sanger sequencing if the variant had originally been identified during whole exome sequencing) and should consider additional genetic testing if such confirmatory evaluation has not been performed. Some patients may also require additional evaluation after identification of genomic findings. Examples include electrocardiogram (ECG) and echocardiography after detecting a variant associated with hypertrophic cardiomyopathy, or imaging studies and blood pressure monitoring after detecting a variant associated with hereditary paraganglioma-pheochromocytoma syndrome.
        </p>
        <p>
         Searchable directories of laboratories that provide disease-specific testing are provided on the Genetic Testing Registry website (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fgtr%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2Fy3%2BLj0shalBW%2BL1ZyL4tryNPgji1PaJ6Kbd%2Fo1hE9A6&amp;TOPIC_ID=96539" target="_blank">
          www.ncbi.nlm.nih.gov/gtr/
         </a>
         ).
        </p>
        <p>
         By contrast, some information such as pharmacogenomic variants may not require additional testing but should be recorded in the medical record and communicated to the patient, so that appropriate dosing changes or closer monitoring can be instituted if implicated medications are considered in the future. An important consideration that should be communicated to patients is whether systems have been developed to automatically consider the pharmacogenomic information when a relevant medication is ordered, or whether patients will need to be proactive about informing clinicians about this information. (See
         <a class="medical medical_review" href="/z/d/html/2904.html" rel="external">
          "Overview of pharmacogenomics"
         </a>
         and
         <a class="medical medical_review" href="/z/d/html/14599.html" rel="external">
          "Personalized medicine"
         </a>
         .)
        </p>
        <p class="headingAnchor" id="H358244194">
         <span class="h2">
          Referrals
         </span>
         <span class="headingEndMark">
          —
         </span>
         Depending on which findings are reported and the physician's familiarity with interpreting and acting upon them, patients may benefit from referrals to specialists experienced with the condition(s) in question [
         <a href="#rid81">
          81
         </a>
         ].
        </p>
        <p>
         Clinicians should also be prepared to help patients locate a genetics professional to manage follow-up of secondary findings [
         <a href="#rid81">
          81
         </a>
         ]. Genetic counselors are master's-level healthcare professionals who work with patients through all aspects of the genetic testing process, from informed consent and test ordering through results return and medical management. A web-based tool for locating a genetic counselor is provided by the
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.nsgc.org%2F&amp;token=OJpa7WnHcNTDFJJhmVS3w6cJAXco8VpFrKoDhZ0I150%3D&amp;TOPIC_ID=96539" target="_blank">
          National Society of Genetic Counselors
         </a>
         (NSGC) [
         <a href="#rid84">
          84
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H358243995">
         <span class="h2">
          Electronic data storage
         </span>
         <span class="headingEndMark">
          —
         </span>
         Most health systems have limited capabilities for storing and retrieving genomic information. Efforts are underway to develop real-time decision support systems to notify clinicians of the availability of genomic information that would inform a specific treatment decision [
         <a href="#rid85">
          85
         </a>
         ]. In the United States, the National Institutes of Health (NIH) has created networks of institutions to facilitate the development of medical record systems better suited to storing and using genomic sequencing information [
         <a href="#rid85">
          85
         </a>
         ]. Barriers to the integration of genomic findings into electronic medical records and accompanying decision support include a lack of standardized nomenclature for genetic variants and a lack of consensus about what information to store [
         <a href="#rid86">
          86
         </a>
         ]. A more extensive discussion of the integration of genomic data into electronic medical records was addressed in a special issue of Genetics in Medicine in 2013 [
         <a href="#rid87">
          87
         </a>
         ].
        </p>
        <p>
         More generally, the knowledge base for genomics continues to rapidly expand. Patients may expect to be informed about updates regarding the effects of genomic variants on their health and well-being, and laboratories have the capability to re-examine data files, which are stored in widely used formats such as binary version of sequence alignment/map (BAM) or variant cell format (VCF), if additional analyses are needed in the future. Few health systems have the capability to provide automated updates to genomic test results, although software such as the GeneInsight Suite has been developed to store genomic sequencing information and automatically provide updated reports to clinicians as they relate to primary test findings [
         <a href="#rid88">
          88,89
         </a>
         ]. It is important for physicians to be clear with patients about whether and how they would be informed about any updated interpretations of variants that were detected. Patients often appreciate sharing the responsibility for tracking scientific developments and monitoring how they might affect interpretation of genetic testing results [
         <a href="#rid69">
          69
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H358243956">
         <span class="h2">
          Postmortem disclosure
         </span>
         <span class="headingEndMark">
          —
         </span>
         Clinicians have a commitment to uphold the privacy and autonomy of their patients, and direct disclosure by a clinician to a patient's relatives after the patient's death is generally avoided unless the patient or a legally authorized representative had explicitly given permission.
        </p>
        <p>
         When secondary findings from genomic testing are available for a deceased patient, the regulations of the Health Insurance Portability and Accountability Act (HIPAA) in the United States allow disclosure of the deceased patient's clinical results to a clinician for purposes of treating a relative of the decedent. However, when a deceased patient's clinical results would not be directly applied to a relative's clinical care, HIPAA guidelines prohibit disclosure of the decedent's results directly to relatives unless an authorization had been signed by the decedent or his/her legally designated administrator or executor [
         <a href="#rid53">
          53
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H358246782">
         <span class="h2">
          Liability concerns
         </span>
         <span class="headingEndMark">
          —
         </span>
         No federal or state statutes in the US directly address a clinician's duty to disclose secondary findings to patients [
         <a href="#rid3">
          3
         </a>
         ]. However, liability for failure to disclose such findings is a common concern among physicians, and some argue that statements such as the ACMG recommendations for secondary findings create fiduciary obligations for disclosure, as such professional standards may help define a legal standard of care [
         <a href="#rid3">
          3,70
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H70171">
         <span class="h1">
          SPECIAL POPULATIONS
         </span>
         <span class="headingEndMark">
          —
         </span>
         Additional ethical and practical considerations apply to certain populations discussed in the following sections.
        </p>
        <p class="headingAnchor" id="H70178">
         <span class="h2">
          Underrepresented ethnicities
         </span>
         <span class="headingEndMark">
          —
         </span>
         Classifying a genetic variant for individuals of some ethnicities can be challenging, because most informatics algorithms incorporate the prevalence of the variant in a reference population (eg, 1000 Genomes cohort), in which the patient's ethnicity may be poorly represented [
         <a href="#rid74">
          74
         </a>
         ]. This issue can be particularly challenging for individuals with African ancestry, given their greater genetic heterogeneity. Additionally, penetrance estimates for pathogenic variants have often been established from epidemiological research on primarily White individuals and may not apply to other populations.
        </p>
        <p>
         The issue of misclassification of benign variants as pathogenic in African Americans was demonstrated in a study that examined variants that had been identified as disease-causing for hypertrophic cardiomyopathy in the Human Gene Mutation Database, version 2012.2 [
         <a href="#rid90">
          90
         </a>
         ]. Based on epidemiological data from three large-scale genome sequencing projects in the United States that included more diverse samples (National Heart Lung and Blood Institute [NHLBI] Exome Sequencing Project, 1000 Genomes Project, Human Genome Diversity Project), the investigators identified five variants that are almost certainly benign based on their high population frequency (ie, present in &gt;1 percent of the population). Notably, these variants were disproportionately prevalent in Black Americans, with genotype frequencies of up to 27 percent (compared with up to 3 percent in White Americans). Simulations suggested that if even a small number of Black people had been included as controls in these cohorts, such variant misclassifications could have been avoided.
        </p>
        <p>
         Ethnicity has implications for the validity of other types of secondary genomic findings that may be provided. As an example, polygenic risk predictions consider multiple SNPs (sometimes thousands) that individually may have weak associations with disease risk but in aggregate may predict high risks for disease. Polygenic risk predictions often predict disease with high validity for populations of European descent but their ability to predict disease in underrepresented groups may be poor [
         <a href="#rid91">
          91
         </a>
         ].
        </p>
        <p>
         In addition, ethnic groups often vary in their beliefs about genetics and interpretation of genetic risk information [
         <a href="#rid92">
          92
         </a>
         ]. Physicians should be sensitive to the way ethnicity may affect both the technical interpretation of genetic data and the way patients interpret and respond to genetic test results.
        </p>
        <p class="headingAnchor" id="H70193">
         <span class="h2">
          Children
         </span>
         <span class="headingEndMark">
          —
         </span>
         Disclosure of secondary findings in children may be particularly challenging, because the initial testing of a child may be performed for the purpose of determining the genetic basis of a presumed genetic syndrome that has major impacts on health, and the family may be focused on that indication rather than secondary results. Additional ethical issues related to consent from minors are also of concern. Clinicians need to weigh the potential costs of disclosure, including harms from diagnostic testing, long-term surveillance, familial testing, psychosocial disruptions, and financial expenses, against the potential health benefits of disclosing findings to the child and family.
        </p>
        <p>
         Policy statements related to pediatric genetic testing often discourage the disclosure of adult-onset conditions in children until they have reached maturity and can decide for themselves whether or not to pursue testing [
         <a href="#rid70">
          70,93,94
         </a>
         ]. However, the ACMG Incidental Findings Working Group recommended seeking and reporting a minimum list of secondary findings to ordering physicians regardless of the patient's age, because another opportunity to identify these "hidden" risks may never arise in the future, and the information might impact the health of the parent carrying the same variant [
         <a href="#rid11">
          11
         </a>
         ]. The ACMG recommends that the same list of 73 genes be analyzed for secondary findings for all patients undergoing whole genome or whole exome sequencing, regardless of age  (
         <a class="graphic graphic_table graphicRef97842" href="/z/d/graphic/97842.html" rel="external">
          table 1
         </a>
         ) [
         <a href="#rid6">
          6,9,13
         </a>
         ]. (See
         <a class="local">
          'Definitions and classification of variants'
         </a>
         above.)
        </p>
        <p>
         Critics note that ACMG's stance conflicts with previous recommendations against testing children for adult-onset conditions, and that the recommendations undermine a child's right to decide whether to learn, in adulthood, whether they have inherited a particular genetic risk variant [
         <a href="#rid68">
          68,70
         </a>
         ]. Others have argued that secondary findings should be limited to genes associated with conditions where the majority of cases present in childhood and for which interventions are supported by evidence of at least moderate quality, namely, multiple endocrine neoplasia (MEN) type 2, retinoblastoma, tuberous sclerosis complex (TSC), Marfan syndrome, and Wilson disease [
         <a href="#rid95">
          95
         </a>
         ]. However, without a phenotype or family history suggestive of a disease, the patient's risk would not otherwise have come to clinical attention. In contrast to predictive genetic testing for a known familial risk, a genomic testing report may be the only chance for the patient and relatives to learn of their at-risk status, a scenario that ACMG categorizes as "opportunistic screening." This scenario presents a different calculus than those in which a patient is aware of his/her at-risk status due to a family history of the disease, and can choose in adulthood to pursue genetic testing for that condition [
         <a href="#rid6">
          6,11,60
         </a>
         ].
        </p>
        <p>
         A position statement of the American Society for Human Genetics (ASHG) released in 2015 emphasized that disclosing actionable secondary findings from sequencing in a child has significant potential benefits for the child and parents/caregivers. The ASHG statement supported an opt-out option for parents to decline analysis of their child's genome for secondary findings. However, it also recommended that clinicians should inform parents/caregivers when genomic findings have "urgent and serious implications for a child's health or welfare, and effective action can be taken to mitigate that threat," regardless of the parents' stated preferences for return of the findings [
         <a href="#rid96">
          96
         </a>
         ].
        </p>
        <p>
         Proponents note that a secondary finding discovered in a child who undergoes genome sequencing for another clinical indication may reveal a risk with immediate medical relevance to one of the child's parents (eg, a mutation in the
         <em>
          BRCA1
         </em>
         gene). If the family history is negative, the parent may be unaware of the increased risk. Early knowledge of and screening for a known genetic risk could improve the parent's prognosis and even save his or her life, which could indirectly benefit a child. For parents in their childbearing years, risk information about devastating childhood onset conditions could also be useful for reproductive planning [
         <a href="#rid6">
          6,11,60
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H1593204">
         <span class="h2">
          Research subjects
         </span>
         <span class="headingEndMark">
          —
         </span>
         Some individuals undergo genetic testing as part of a research study that may be unrelated to their personal or family health care. Further, this testing is often performed in a research laboratory rather than a Clinical Laboratory Improvement Amendments (CLIA)-certified laboratory, and thus may not have the accompanying analytic validity provided by clinical testing. These differences raise additional ethical issues for the research subject. Obligations for disclosure are unclear, with some noting that researchers are obligated to disclose actionable, high-impact secondary findings from any stage of research, and others preferring disclosure only in limited circumstances [
         <a href="#rid97">
          97-100
         </a>
         ]. Complicating the matter, patient consent for receiving these findings from a research study may have been solicited years before the research has been conducted or outside of a medical context, and participants may be difficult to reach. In the Geisinger MyCode project, no follow-up was noted in 27 of 542 participants (5 percent) who had CLIA-confirmed pathogenic or likely pathogenic variants in one or more of the 76 genes interrogated for secondary findings [
         <a href="#rid101">
          101
         </a>
         ].
        </p>
        <p>
         Increasingly, investigators are disclosing individual results of genomic testing from research studies if these results may affect participant healthcare [
         <a href="#rid38">
          38
         </a>
         ]. In general, researchers should consider the same principles for responding to secondary findings from genomic research as is used for clinical genomic testing. Consent for the research should emphasize that genetic testing is being conducted for research rather than clinical purposes [
         <a href="#rid41">
          41
         </a>
         ]. As translational genomics research advances, the boundaries between clinical and research testing may become increasingly blurred. Researchers and healthcare systems are in the process of developing policies, systems, and infrastructure to accommodate the responsible return of individual research results [
         <a href="#rid71">
          71
         </a>
         ].
        </p>
        <p class="headingAnchor" id="H70215">
         <span class="h1">
          ONLINE INFORMATION AND RESOURCES
         </span>
         <span class="headingEndMark">
          —
         </span>
         Many professional organizations for physicians have developed educational materials about genomics and/or guidelines for specific genetic conditions.
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          ACMG
         </strong>
         – The American College of Medical Genetics and Genomics (ACMG) website (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.acmg.net%2F&amp;token=IQxLzDq4doJGUgaZgeIY0xL5EakBx7JlWqf9qCS6Rck%3D&amp;TOPIC_ID=96539" target="_blank">
          www.acmg.net
         </a>
         ) provides resources such as policy statements, practice guidelines, and educational information.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         ACMG is also creating clinical decision support tools, known as "ACT Sheets," specific to genomic sequencing for physicians with limited experience in genomic medicine; ACMG plans to communicate updates to its recommendations through its newsletter and in its peer-reviewed monthly journal, Genetics in Medicine [
         <a href="#rid11">
          11
         </a>
         ].
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          NIH
         </strong>
         – The National Human Genome Research Institute (NHGRI) at the National Institutes of Health (NIH) in the United States hosts an Online Health and Support Resources portal (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.genome.gov%2F11510197%2Fonline-health-and-support-resources&amp;token=prCqmS1pg0UN8eQ68yHzODPzdsi6qSlEG1AXvNvPGBqD3FFZfRqFpbW8R1YobSGqIjkX0RJVhTnrMMkFtN%2Bh7ikqmEYEEcKhRT033u84Vh8%3D&amp;TOPIC_ID=96539" target="_blank">
          www.genome.gov/11510197
         </a>
         ) and provides articles summarizing support and educational materials [
         <a href="#rid102">
          102
         </a>
         ].
        </p>
        <p>
        </p>
        <p>
         A number of additional resources are available to provide information about genetic conditions. Sites such as
         <strong>
          ClinVar
         </strong>
         are preferred to some of the existing public databases of genomic variants, such as the database of Genotypes and Phenotypes (dbGaP) and the Human Gene Mutation Database (HGMD), which frequently exaggerate the association between genetic variants and disease [
         <a href="#rid103">
          103
         </a>
         ]. The following resources may be helpful:
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          MedGen
         </strong>
         – MedGen (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fmedgen%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2F1mEzbrra1EyPWuMKBWRSz1mKk7gocQCgBtxiwC4kZwT&amp;TOPIC_ID=96539" target="_blank">
          www.ncbi.nlm.nih.gov/medgen/
         </a>
         ) is the NCBI's portal to information about human disorders and other phenotypes having a genetic component, including professional guidelines.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          ClinGen
         </strong>
         – ClinGen (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fclinicalgenome.org%2F&amp;token=nbsXe3i4qfUea5RCPRHtOXAfmv8sxvNi1dzM5%2F2MCbBz%2Fjw073txTxlmmUrxvXA9&amp;TOPIC_ID=96539" target="_blank">
          https://clinicalgenome.org/
         </a>
         ) is a NIH-funded resource dedicated to building an authoritative central resource that defines the clinical relevance of genes and variants for use in precision medicine and research. The site includes evidence-based summaries about genes and conditions recommended for screening by ACMG, which can be accessed at
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fclinicalgenome.org%2Fcuration-activities%2Fclinical-actionability%2Ftraining-materials%2F&amp;token=2FE1Vc4TBHI6NtiyUzzjy3M%2FIVmSaJpXFMo6hkkV1DPkAwaQBZo25yhmqLGE1uZWNjkmjW8aprCLmnAV5hnMEYCs9O7x8HYNlP38G%2BZrXxA%3D&amp;TOPIC_ID=96539" target="_blank">
          https://clinicalgenome.org/working-groups/actionability/projects-initiatives/actionability-evidence-based-summaries/
         </a>
         .
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          GeneReviews
         </strong>
         – GeneReviews (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fbooks%2FNBK1116%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2Fy3%2BLj0shalBW%2BL1ZyL4trzEAfmGCQvouTQ5LbX9C3XVSHxjYdvs2mgciTuJx4IXLQ%3D%3D&amp;TOPIC_ID=96539" target="_blank">
          www.ncbi.nlm.nih.gov/books/NBK1116/
         </a>
         ) provides overviews and disease-specific discussions of counseling, diagnosis, and management for individuals and families with inherited disorders.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          ClinVar
         </strong>
         – ClinVar (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fclinvar%2F&amp;token=wEsKVFvqlKzpT%2BqOtddW%2Fy3%2BLj0shalBW%2BL1ZyL4trzGy12ipcVcj41059JRLkL3&amp;TOPIC_ID=96539" target="_blank">
          www.ncbi.nlm.nih.gov/clinvar/
         </a>
         ) is a freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          OMIM
         </strong>
         – Online Mendelian Inheritance in Man (OMIM,
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fomim&amp;token=wEsKVFvqlKzpT%2BqOtddW%2F1mEzbrra1EyPWuMKBWRSz2FZxizAt9vS6osbKh7UCRu&amp;TOPIC_ID=96539" target="_blank">
          www.ncbi.nlm.nih.gov/omim
         </a>
         ) provides a compendium of human genes and genetic phenotypes.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          NCI
         </strong>
         – The National Cancer Institute (NCI) website (
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.cancer.gov%2Fabout-cancer%2Fcauses-prevention%2Fgenetics%2Foverview-pdq&amp;token=MuwNBZAjpd%2BEalhMTq1%2Fsnnrn6h9ydbQsZRaw0HzqHFG9ApqEPk9NOO7GCnOGF2LMWYtx%2FeGkfqM8l4w3EEbQdERIljbnNcOWq491cDHy0I%3D&amp;TOPIC_ID=96539" target="_blank">
          www.cancer.gov/about-cancer/causes-prevention/genetics/overview-pdq
         </a>
         ) presents physician data query (PDQ) summaries of information about specific cancers, cancer genetics, and an overview of clinical genetics based on clinical evidence.
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          PharmGKB
         </strong>
         – PharmacoGenomics Knowledge Base (PharmGKB,
         <a class="external" href="/external-redirect?target_url=https%3A%2F%2Fwww.pharmgkb.org%2F&amp;token=xXI%2By9QNEPjV52%2B%2BDeHAeRSknjwZrSAAa8IM1A9AZ18sVzNz1i2SXggwyErj6gsH&amp;TOPIC_ID=96539" target="_blank">
          www.pharmgkb.org/
         </a>
         ) is a comprehensive resource of curated knowledge about the impact of genetic variation on drug response.
        </p>
        <p>
        </p>
        <p class="headingAnchor" id="H70236">
         <span class="h1">
          SUMMARY AND RECOMMENDATIONS
         </span>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Definitions
         </strong>
         – Secondary findings (also called incidental findings) are health-informative genetic variants identified during genomic analysis that are unrelated to the reason that testing was ordered or are part of a general search for findings of interest. (See
         <a class="local">
          'Definitions and classification of variants'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Prevalence
         </strong>
         – The likelihood of secondary findings depends on the genetic test and the criteria for identification and reporting. Actionable findings that might lead to a change in management are found in 1 to 6 percent of individuals. (See
         <a class="local">
          'Likelihood of detecting a secondary finding'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Consent
         </strong>
         – Patients should be informed about the potential for receiving secondary findings, including what kinds of variants would be disclosed and how they would be communicated; this should occur as part of the consent process before testing. Patients should also have an option to forego analysis for secondary findings (an "opt out"). Informed consent may also involve discussion of what to do with secondary findings after the patient's death, which may help patients initiate discussions with relatives about whether they would want to receive postmortem disclosure of this information, along with any other medical information that might be critical to their interpretation. In general, the negative impact of disclosing secondary findings appears to be relatively low. (See
         <a class="local">
          'Informed consent'
         </a>
         above and
         <a class="local">
          'Postmortem disclosure'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Laboratory standards
         </strong>
         – Each laboratory makes decisions regarding which genes will be analyzed for secondary findings, what pathogenicity to assign to each variant within the gene, and which results are reported. The American College of Medical Genetics and Genomics (ACMG) has identified 73 genes associated with 35 conditions that is considered a minimal set for which pathogenic or likely pathogenic variants should be reported  (
         <a class="graphic graphic_table graphicRef97842" href="/z/d/graphic/97842.html" rel="external">
          table 1
         </a>
         ). Numerous commissions have called for the refinement of professional standards. (See
         <a class="local">
          'Decisions made by the laboratory'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Disclosure of results
         </strong>
         – Disclosure of secondary findings depends on the clinical context and judgment of clinicians involved in the patient's care. Findings should always be interpreted in the context of the patient's medical and family histories of disease, physical examination, and other relevant information, as illustrated in the sample report  (
         <a class="graphic graphic_figure graphicRef122538" href="/z/d/graphic/122538.html" rel="external">
          figure 1
         </a>
         ). Ethical obligations to respect autonomy and right not to receive information must be weighed against fiduciary duties to share potentially life-saving results. A decision matrix is presented in the table  (
         <a class="graphic graphic_table graphicRef97881" href="/z/d/graphic/97881.html" rel="external">
          table 2
         </a>
         ). (See
         <a class="local">
          'Review patient report'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Counseling
         </strong>
         – After informing the patient that a finding is present, counseling should cover the clinical implications of the result and whether additional intervention is needed. A pathogenic variant is a marker of increased risk,
         <strong>
          not
         </strong>
         a diagnosis of disease. Conversely, a "negative" result does not imply that the patient's risk of disease is low. (See
         <a class="local">
          'Informing and counseling the patient'
         </a>
         above and
         <a class="local">
          'Interpretation of 'negative' results'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Diversity and equity
         </strong>
         – Technical limitations apply to identification and interpretation of secondary in individuals of underrepresented ethnicities for whom data on disease prevalence and penetrance are lacking. (See
         <a class="local">
          'Underrepresented ethnicities'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Children
         </strong>
         – Special ethical considerations apply to disclosure of secondary findings from genomic sequencing of children and research subjects. (See
         <a class="local">
          'Children'
         </a>
         above and
         <a class="local">
          'Research subjects'
         </a>
         above.)
        </p>
        <p>
        </p>
        <p class="bulletIndent1">
         <span class="glyph">
          ●
         </span>
         <strong>
          Resources
         </strong>
         – A variety of resources are available to assist clinicians, including policy statements, practice guidelines, and educational information. (See
         <a class="local">
          'Online information and resources'
         </a>
         above.)
        </p>
       </div>
       <div class="headingAnchor" id="references">
        <ol id="reference">
         <li>
          <a class="nounderline abstract_t">
           Christenhusz GM, Devriendt K, Dierickx K. Disclosing incidental findings in genetics contexts: a review of the empirical ethical research. Eur J Med Genet 2013; 56:529.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Shkedi-Rafid S, Dheensa S, Crawford G, et al. Defining and managing incidental findings in genetic and genomic practice. J Med Genet 2014; 51:715.
          </a>
         </li>
         <li class="breakAll">
          Presidential Commission for the Study of Bioethical Issues. Anticipate and communicate: Ethical management of incidental and secondary findings in the clinical, research, and direct-to-consumer contexts, U.S. Department of Health &amp; Human Services 2013. www.bioethics.gov/sites/default/files/FINALAnticipateCommunicate_PCSBI_0.pdf (Accessed on September 24, 2014).
         </li>
         <li class="breakAll">
          Green RC, Rehm HL, Kohane IS. Clinical genome sequencing. In: Genomic and Personalized Medicine, 2nd Ed, Ginsburg GS, Willard HF (Eds) (Eds), Academic Press, San Diego 2013. Vol 1, p.102.
         </li>
         <li>
          <a class="nounderline abstract_t">
           Richards S, Aziz N, Bale S, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet Med 2015; 17:405.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Green RC, Berg JS, Grody WW, et al. ACMG recommendations for reporting of incidental findings in clinical exome and genome sequencing. Genet Med 2013; 15:565.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Green RC, Lupski JR, Biesecker LG. Reporting genomic sequencing results to ordering clinicians: incidental, but not exceptional. JAMA 2013; 310:365.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Jarvik GP, Evans JP. Mastering genomic terminology. Genet Med 2017; 19:491.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kalia SS, Adelman K, Bale SJ, et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics. Genet Med 2017; 19:249.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Miller DT, Lee K, Chung WK, et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2021; 23:1381.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           American College of Medical Genetics and Genomics. Incidental findings in clinical genomics: a clarification. Genet Med 2013; 15:664.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           ACMG Board of Directors. The use of ACMG secondary findings recommendations for general population screening: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2019; 21:1467.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Miller DT, Lee K, Gordon AS, et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2021 update: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet Med 2021; 23:1391.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ormond KE, O'Daniel JM, Kalia SS. Secondary findings: How did we get here, and where are we going? J Genet Couns 2019; 28:326.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Yang Y, Muzny DM, Xia F, et al. Molecular findings among patients referred for clinical whole-exome sequencing. JAMA 2014; 312:1870.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Pujol P, Vande Perre P, Faivre L, et al. Guidelines for reporting secondary findings of genome sequencing in cancer genes: the SFMPP recommendations. Eur J Hum Genet 2018; 26:1732.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Green RC, Berg JS, Berry GT, et al. Exploring concordance and discordance for return of incidental findings from clinical sequencing. Genet Med 2012; 14:405.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Christensen KD, Bernhardt BA, Jarvik GP, et al. Anticipated responses of early adopter genetic specialists and nongenetic specialists to unsolicited genomic secondary findings. Genet Med 2018; 20:1186.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Van Hout CV, Tachmazidou I, Backman JD, et al. Exome sequencing and characterization of 49,960 individuals in the UK Biobank. Nature 2020; 586:749.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           eMERGE Clinical Annotation Working Group. Frequency of genomic secondary findings among 21,915 eMERGE network participants. Genet Med 2020; 22:1470.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Green RC, Goddard KA, Jarvik GP, et al. Clinical Sequencing Exploratory Research Consortium: Accelerating Evidence-Based Practice of Genomic Medicine. Am J Hum Genet 2016; 98:1051.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Amendola LM, Dorschner MO, Robertson PD, et al. Actionable exomic incidental findings in 6503 participants: challenges of variant classification. Genome Res 2015; 25:305.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Natarajan P, Gold NB, Bick AG, et al. Aggregate penetrance of genomic variants for actionable disorders in European and African Americans. Sci Transl Med 2016; 8:364ra151.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Olfson E, Cottrell CE, Davidson NO, et al. Identification of Medically Actionable Secondary Findings in the 1000 Genomes. PLoS One 2015; 10:e0135193.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hart MR, Biesecker BB, Blout CL, et al. Secondary findings from clinical genomic sequencing: prevalence, patient perspectives, family history assessment, and health-care costs from a multisite study. Genet Med 2019; 21:1100.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Johnston JJ, Rubinstein WS, Facio FM, et al. Secondary variants in individuals undergoing exome sequencing: screening of 572 individuals identifies high-penetrance mutations in cancer-susceptibility genes. Am J Hum Genet 2012; 91:97.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Thompson ML, Finnila CR, Bowling KM, et al. Genomic sequencing identifies secondary findings in a cohort of parent study participants. Genet Med 2018; 20:1635.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Vassy JL, Christensen KD, Schonman EF, et al. The Impact of Whole-Genome Sequencing on the Primary Care and Outcomes of Healthy Adult Patients: A Pilot Randomized Trial. Ann Intern Med 2017; 167:159.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ceyhan-Birsoy O, Murry JB, Machini K, et al. Interpretation of Genomic Sequencing Results in Healthy and Ill Newborns: Results from the BabySeq Project. Am J Hum Genet 2019; 104:76.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Tabor HK, Auer PL, Jamal SM, et al. Pathogenic variants for Mendelian and complex traits in exomes of 6,517 European and African Americans: implications for the return of incidental results. Am J Hum Genet 2014; 95:183.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Xue Y, Chen Y, Ayub Q, et al. Deleterious- and disease-allele prevalence in healthy individuals: insights from current predictions, mutation databases, and population-scale resequencing. Am J Hum Genet 2012; 91:1022.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lazarin GA, Haque IS, Nazareth S, et al. An empirical estimate of carrier frequencies for 400+ causal Mendelian variants: results from an ethnically diverse clinical sample of 23,453 individuals. Genet Med 2013; 15:178.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Bell CJ, Dinwiddie DL, Miller NA, et al. Carrier testing for severe childhood recessive diseases by next-generation sequencing. Sci Transl Med 2011; 3:65ra4.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gambin T, Jhangiani SN, Below JE, et al. Secondary findings and carrier test frequencies in a large multiethnic sample. Genome Med 2015; 7:54.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Blout CL, Robinson JO, McGuire AL, et al. Incorporation of Whole Genome Sequencing Results into the Electronic Medical Record: Attitudes of MedSeq Project Participants. Am J Hum Genet (ASHG Annual Meeting Abstract #258) 2015.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Reiff M, Ross K, Mulchandani S, et al. Physicians' perspectives on the uncertainties and implications of chromosomal microarray testing of children and families. Clin Genet 2013; 83:23.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Middleton A, Patch C, Wiggins J, et al. Position statement on opportunistic genomic screening from the Association of Genetic Nurses and Counsellors (UK and Ireland). Eur J Hum Genet 2014; 22:955.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wolf SM. The past, present, and future of the debate over return of research results and incidental findings. Genet Med 2012; 14:355.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Appelbaum PS, Stiles DF, Chung W. Cases in Precision Medicine: Should You Participate in a Study Involving Genomic Sequencing of Your Patients? Ann Intern Med 2019; 171:568.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ross LF, Rothstein MA, Clayton EW. Mandatory extended searches in all genome sequencing: "incidental findings," patient autonomy, and shared decision making. JAMA 2013; 310:367.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           ACMG Board of Directors. Points to consider for informed consent for genome/exome sequencing. Genet Med 2013; 15:748.
          </a>
         </li>
         <li class="breakAll">
          National Society of Genetic Counselors. Secondary and Incidental Findings in Genetic Testing, 3.23.2020. https://www.nsgc.org/Policy-Research-and-Publications/Position-Statements/Position-Statements/Post/secondary-and-incidental-findings-in-genetic-testing-1 (Accessed on July 23, 2021).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Heald B, Edelman E, Eng C. Prospective comparison of family medical history with personal genome screening for risk assessment of common cancers. Eur J Hum Genet 2012; 20:547.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hudson KL. Prohibiting genetic discrimination. N Engl J Med 2007; 356:2021.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kiryluk K, Goldstein DB, Rowe JW, et al. Precision Medicine in Internal Medicine. Ann Intern Med 2019; 170:635.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Guttmacher AE, Porteous ME, McInerney JD. Educating health-care professionals about genetics and genomics. Nat Rev Genet 2007; 8:151.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Shields AE, Blumenthal D, Weiss KB, et al. Barriers to translating emerging genetic research on smoking into clinical practice. Perspectives of primary care physicians. J Gen Intern Med 2005; 20:131.
          </a>
         </li>
         <li class="breakAll">
          Secretary's Advisory Committee on Genetics, Health, and Society. Genetics Education and Training. February 2011. http://www.genome.gov/Pages/Careers/HealthProfessionalEducation/SACGHS-EducationReport2011.pdf (Accessed on October 30, 2014).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Christensen KD, Roberts JS, Whitehouse PJ, et al. Disclosing Pleiotropic Effects During Genetic Risk Assessment for Alzheimer Disease: A Randomized Trial. Ann Intern Med 2016; 164:155.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Murray MF. Genomics: Prediction, Prevention, Priorities, and Punnett. Ann Intern Med 2016; 164:197.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lee H, Deignan JL, Dorrani N, et al. Clinical exome sequencing for genetic identification of rare Mendelian disorders. JAMA 2014; 312:1880.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Allen NL, Karlson EW, Malspeis S, et al. Biobank participants' preferences for disclosure of genetic research results: perspectives from the OurGenes, OurHealth, OurCommunity project. Mayo Clin Proc 2014; 89:738.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rothstein MA. Disclosing decedents' research results to relatives violates the HIPAA Privacy Rule. Am J Bioeth 2012; 12:16.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hegde M, Bale S, Bayrak-Toydemir P, et al. Reporting incidental findings in genomic scale clinical sequencing--a clinical laboratory perspective: a report of the Association for Molecular Pathology. J Mol Diagn 2015; 17:107.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hamamy H, Antonarakis SE, Cavalli-Sforza LL, et al. Consanguineous marriages, pearls and perils: Geneva International Consanguinity Workshop Report. Genet Med 2011; 13:841.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Sund KL, Zimmerman SL, Thomas C, et al. Regions of homozygosity identified by SNP microarray analysis aid in the diagnosis of autosomal recessive disease and incidentally detect parental blood relationships. Genet Med 2013; 15:70.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rehder CW, David KL, Hirsch B, et al. American College of Medical Genetics and Genomics: standards and guidelines for documenting suspected consanguinity as an incidental finding of genomic testing. Genet Med 2013; 15:150.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ng PC, Murray SS, Levy S, Venter JC. An agenda for personalized medicine. Nature 2009; 461:724.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Rehm HL, Bale SJ, Bayrak-Toydemir P, et al. ACMG clinical laboratory standards for next-generation sequencing. Genet Med 2013; 15:733.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           McGuire AL, Joffe S, Koenig BA, et al. Point-counterpoint. Ethics and genomic incidental findings. Science 2013; 340:1047.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Hicks JK, Shealy A, Schreiber A, et al. Patient Decisions to Receive Secondary Pharmacogenomic Findings and Development of a Multidisciplinary Practice Model to Integrate Results Into Patient Care. Clin Transl Sci 2018; 11:71.
          </a>
         </li>
         <li class="breakAll">
          https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/inova-genomics-laboratory-577422-04042019 (Accessed on September 12, 2019).
         </li>
         <li class="breakAll">
          Anticipate and Communication: Ethical Management of Incidental and Secondary Findings in the Clinical, Research, and Direct-to-Consumer Contexts. Presidential Commission for the Study of Bioethical Issues, 2013.
         </li>
         <li>
          <a class="nounderline abstract_t">
           ACMG Board of Directors. Points to consider in the clinical application of genomic sequencing. Genet Med 2012; 14:759.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Burke W, Antommaria AH, Bennett R, et al. Recommendations for returning genomic incidental findings? We need to talk! Genet Med 2013; 15:854.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Daack-Hirsch S, Driessnack M, Hanish A, et al. 'Information is information': a public perspective on incidental findings in clinical and research genome-based testing. Clin Genet 2013; 84:11.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Dal-Ré R, Katsanis N, Katsanis S, et al. Managing incidental genomic findings in clinical trials: fulfillment of the principle of justice. PLoS Med 2014; 11:e1001584.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wolf SM, Annas GJ, Elias S. Point-counterpoint. Patient autonomy and incidental findings in clinical genomics. Science 2013; 340:1049.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Townsend A, Adam S, Birch PH, et al. "I want to know what's in Pandora's Box": comparing stakeholder perspectives on incidental findings in clinical whole genomic sequencing. Am J Med Genet A 2012; 158A:2519.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Ross LF, Rothstein MA, Clayton EW. Premature guidance about whole-genome sequencing. Per Med 2013; 10.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wolf SM. Return of individual research results and incidental findings: facing the challenges of translational science. Annu Rev Genomics Hum Genet 2013; 14:557.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Artin MG, Stiles D, Kiryluk K, Chung WK. Cases in Precision Medicine: When Patients Present With Direct-to-Consumer Genetic Test Results. Ann Intern Med 2019; 170:643.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Delgado F, Tabor HK, Chow PM, et al. Single-nucleotide polymorphism arrays and unexpected consanguinity: considerations for clinicians when returning results to families. Genet Med 2015; 17:400.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Krier JB, Green RC. Management of incidental findings in clinical genomic sequencing. Curr Protoc Hum Genet 2013; Chapter 9:Unit9.23.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Berg JS, Adams M, Nassar N, et al. An informatics approach to analyzing the incidentalome. Genet Med 2013; 15:36.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Berg JS, Khoury MJ, Evans JP. Deploying whole genome sequencing in clinical practice and public health: meeting the challenge one bin at a time. Genet Med 2011; 13:499.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           McCormick JB, Sharp RR, Farrugia G, et al. Genomic medicine and incidental findings: balancing actionability and patient autonomy. Mayo Clin Proc 2014; 89:718.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Tan N, Amendola LM, O'Daniel JM, et al. Is "incidental finding" the best term?: a study of patients' preferences. Genet Med 2017; 19:176.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wynn J, Lewis K, Amendola LM, et al. Clinical providers' experiences with returning results from genomic sequencing: an interview study. BMC Med Genomics 2018; 11:45.
          </a>
         </li>
         <li class="breakAll">
          Petty RE, Cacioppo JT. The elaboration likelihood model of persuasion. In: Advances in Experimental Social Psychology, Berkowitz L (Ed), Academic Press, Orlando 1986. Vol 19, p.124.
         </li>
         <li>
          <a class="nounderline abstract_t">
           Biesecker LG, Green RC. Diagnostic clinical genome and exome sequencing. N Engl J Med 2014; 370:2418.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Mantere T, Kersten S, Hoischen A. Long-Read Sequencing Emerging in Medical Genetics. Front Genet 2019; 10:426.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Lincoln SE, Truty R, Lin CF, et al. A Rigorous Interlaboratory Examination of the Need to Confirm Next-Generation Sequencing-Detected Variants with an Orthogonal Method in Clinical Genetic Testing. J Mol Diagn 2019; 21:318.
          </a>
         </li>
         <li class="breakAll">
          National Society of Genetic Counselors. https://findageneticcounselor.nsgc.org/ (Accessed on September 02, 2021).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Gottesman O, Kuivaniemi H, Tromp G, et al. The Electronic Medical Records and Genomics (eMERGE) Network: past, present, and future. Genet Med 2013; 15:761.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kullo IJ, Haddad R, Prows CA, et al. Return of results in the genomic medicine projects of the eMERGE network. Front Genet 2014; 5:50.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Kannry JL, Williams MS. Integration of genomics into the electronic health record: mapping terra incognita. Genet Med 2013; 15:757.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Aronson SJ, Clark EH, Babb LJ, et al. The GeneInsight Suite: a platform to support laboratory and provider use of DNA-based genetic testing. Hum Mutat 2011; 32:532.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Wilcox AR, Neri PM, Volk LA, et al. A novel clinician interface to improve clinician access to up-to-date genetic results. J Am Med Inform Assoc 2014; 21:e117.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Manrai AK, Funke BH, Rehm HL, et al. Genetic Misdiagnoses and the Potential for Health Disparities. N Engl J Med 2016; 375:655.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Martin AR, Kanai M, Kamatani Y, et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Nat Genet 2019; 51:584.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Singer E, Antonucci T, Van Hoewyk J. Racial and ethnic variations in knowledge and attitudes about genetic testing. Genet Test 2004; 8:31.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Guidelines for genetic testing of healthy children. Paediatr Child Health 2003; 8:42.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           European Society of Human Genetics. Genetic testing in asymptomatic minors: Recommendations of the European Society of Human Genetics. Eur J Hum Genet 2009; 17:720.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Richer J, Laberge AM. Secondary findings from next-generation sequencing: what does actionable in childhood really mean? Genet Med 2019; 21:124.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Botkin JR, Belmont JW, Berg JS, et al. Points to Consider: Ethical, Legal, and Psychosocial Implications of Genetic Testing in Children and Adolescents. Am J Hum Genet 2015; 97:6.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Shalowitz DI, Miller FG. Disclosing individual results of clinical research: implications of respect for participants. JAMA 2005; 294:737.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Partridge AH, Winer EP. Sharing study results with trial participants: time for action. J Clin Oncol 2009; 27:838.
          </a>
         </li>
         <li class="breakAll">
          National Bioethics Advisory Commission. Research involving human biological materials: ethical Issues and policy guidance. Rockville, MD 1999. Vol 1. bioethicsarchive.georgetown.edu/nbac/hbm.pdf (Accessed on September 24, 2014).
         </li>
         <li>
          <a class="nounderline abstract_t">
           Clayton EW, Ross LF. Implications of disclosing individual results of clinical research. JAMA 2006; 295:37; author reply 37.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Schwartz MLB, McCormick CZ, Lazzeri AL, et al. A Model for Genome-First Care: Returning Secondary Genomic Findings to Participants and Their Healthcare Providers in a Large Research Cohort. Am J Hum Genet 2018; 103:328.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Uhlmann WR, Guttmacher AE. Key Internet genetics resources for the clinician. JAMA 2008; 299:1356.
          </a>
         </li>
         <li>
          <a class="nounderline abstract_t">
           Riggs ER, Wain KE, Riethmaier D, et al. Towards a Universal Clinical Genomics Database: the 2012 International Standards for Cytogenomic Arrays Consortium Meeting. Hum Mutat 2013; 34:915.
          </a>
         </li>
        </ol>
       </div>
       <div id="topicVersionRevision">
        Topic 96539 Version 27.0
       </div>
      </div>
      <div class="row">
       <div class="col-12">
        <h4 style="text-align: center;padding: 20px;color: cadetblue;">
         References
        </h4>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24036277" id="rid0" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          1 : Disclosing incidental findings in genetics contexts: a review of the empirical ethical research.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25228303" id="rid1" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          2 : Defining and managing incidental findings in genetic and genomic practice.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25228303" id="rid2" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          3 : Defining and managing incidental findings in genetic and genomic practice.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25228303" id="rid3" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          4 : Defining and managing incidental findings in genetic and genomic practice.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25741868" id="rid4" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          5 : Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23788249" id="rid5" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          6 : ACMG recommendations for reporting of incidental findings in clinical exome and genome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23917280" id="rid6" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          7 : Reporting genomic sequencing results to ordering clinicians: incidental, but not exceptional.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27657676" id="rid7" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          8 : Mastering genomic terminology.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27854360" id="rid8" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          9 : Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/34012068" id="rid9" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          10 : ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG).
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23828017" id="rid10" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          11 : Incidental findings in clinical genomics: a clarification.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31019278" id="rid11" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          12 : The use of ACMG secondary findings recommendations for general population screening: a policy statement of the American College of Medical Genetics and Genomics (ACMG).
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/34012069" id="rid12" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          13 : Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2021 update: a policy statement of the American College of Medical Genetics and Genomics (ACMG).
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30821867" id="rid13" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          14 : Secondary findings: How did we get here, and where are we going?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25326635" id="rid14" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          15 : Molecular findings among patients referred for clinical whole-exome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30089825" id="rid15" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          16 : Guidelines for reporting secondary findings of genome sequencing in cancer genes: the SFMPP recommendations.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22422049" id="rid16" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          17 : Exploring concordance and discordance for return of incidental findings from clinical sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29388940" id="rid17" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          18 : Anticipated responses of early adopter genetic specialists and nongenetic specialists to unsolicited genomic secondary findings.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/33087929" id="rid18" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          19 : Exome sequencing and characterization of 49,960 individuals in the UK Biobank.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/32546831" id="rid19" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          20 : Frequency of genomic secondary findings among 21,915 eMERGE network participants.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27181682" id="rid20" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          21 : Clinical Sequencing Exploratory Research Consortium: Accelerating Evidence-Based Practice of Genomic Medicine.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25637381" id="rid21" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          22 : Actionable exomic incidental findings in 6503 participants: challenges of variant classification.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27831900" id="rid22" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          23 : Aggregate penetrance of genomic variants for actionable disorders in European and African Americans.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/26332594" id="rid23" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          24 : Identification of Medically Actionable Secondary Findings in the 1000 Genomes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30287922" id="rid24" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          25 : Secondary findings from clinical genomic sequencing: prevalence, patient perspectives, family history assessment, and health-care costs from a multisite study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22703879" id="rid25" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          26 : Secondary variants in individuals undergoing exome sequencing: screening of 572 individuals identifies high-penetrance mutations in cancer-susceptibility genes.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29790872" id="rid26" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          27 : Genomic sequencing identifies secondary findings in a cohort of parent study participants.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28654958" id="rid27" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          28 : The Impact of Whole-Genome Sequencing on the Primary Care and Outcomes of Healthy Adult Patients: A Pilot Randomized Trial.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30609409" id="rid28" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          29 : Interpretation of Genomic Sequencing Results in Healthy and Ill Newborns: Results from the BabySeq Project.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25087612" id="rid29" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          30 : Pathogenic variants for Mendelian and complex traits in exomes of 6,517 European and African Americans: implications for the return of incidental results.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23217326" id="rid30" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          31 : Deleterious- and disease-allele prevalence in healthy individuals: insights from current predictions, mutation databases, and population-scale resequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22975760" id="rid31" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          32 : An empirical estimate of carrier frequencies for 400+ causal Mendelian variants: results from an ethnically diverse clinical sample of 23,453 individuals.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21228398" id="rid32" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          33 : Carrier testing for severe childhood recessive diseases by next-generation sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/26195989" id="rid33" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          34 : Secondary findings and carrier test frequencies in a large multiethnic sample.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/" id="rid34" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          35 : Incorporation of Whole Genome Sequencing Results into the Electronic Medical Record: Attitudes of MedSeq Project Participants
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22989118" id="rid35" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          36 : Physicians' perspectives on the uncertainties and implications of chromosomal microarray testing of children and families.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24398792" id="rid36" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          37 : Position statement on opportunistic genomic screening from the Association of Genetic Nurses and Counsellors (UK and Ireland).
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22481182" id="rid37" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          38 : The past, present, and future of the debate over return of research results and incidental findings.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31569215" id="rid38" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          39 : Cases in Precision Medicine: Should You Participate in a Study Involving Genomic Sequencing of Your Patients?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23917281" id="rid39" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          40 : Mandatory extended searches in all genome sequencing: "incidental findings," patient autonomy, and shared decision making.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23970068" id="rid40" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          41 : Points to consider for informed consent for genome/exome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23970068" id="rid41" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          42 : Points to consider for informed consent for genome/exome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22215421" id="rid42" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          43 : Prospective comparison of family medical history with personal genome screening for risk assessment of common cancers.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17507700" id="rid43" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          44 : Prohibiting genetic discrimination.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31035290" id="rid44" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          45 : Precision Medicine in Internal Medicine.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/17230201" id="rid45" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          46 : Educating health-care professionals about genetics and genomics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15836545" id="rid46" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          47 : Barriers to translating emerging genetic research on smoking into clinical practice. Perspectives of primary care physicians.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15836545" id="rid47" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          48 : Barriers to translating emerging genetic research on smoking into clinical practice. Perspectives of primary care physicians.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/26810768" id="rid48" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          49 : Disclosing Pleiotropic Effects During Genetic Risk Assessment for Alzheimer Disease: A Randomized Trial.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/26810850" id="rid49" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          50 : Genomics: Prediction, Prevention, Priorities, and Punnett.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25326637" id="rid50" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          51 : Clinical exome sequencing for genetic identification of rare Mendelian disorders.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24943692" id="rid51" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          52 : Biobank participants' preferences for disclosure of genetic research results: perspectives from the OurGenes, OurHealth, OurCommunity project.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22974022" id="rid52" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          53 : Disclosing decedents' research results to relatives violates the HIPAA Privacy Rule.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25684271" id="rid53" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          54 : Reporting incidental findings in genomic scale clinical sequencing--a clinical laboratory perspective: a report of the Association for Molecular Pathology.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21555946" id="rid54" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          55 : Consanguineous marriages, pearls and perils: Geneva International Consanguinity Workshop Report.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22858719" id="rid55" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          56 : Regions of homozygosity identified by SNP microarray analysis aid in the diagnosis of autosomal recessive disease and incidentally detect parental blood relationships.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23328890" id="rid56" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          57 : American College of Medical Genetics and Genomics: standards and guidelines for documenting suspected consanguinity as an incidental finding of genomic testing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19812653" id="rid57" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          58 : An agenda for personalized medicine.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23887774" id="rid58" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          59 : ACMG clinical laboratory standards for next-generation sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23686340" id="rid59" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          60 : Point-counterpoint. Ethics and genomic incidental findings.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28749586" id="rid60" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          61 : Patient Decisions to Receive Secondary Pharmacogenomic Findings and Development of a Multidisciplinary Practice Model to Integrate Results Into Patient Care.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28749586" id="rid61" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          62 : Patient Decisions to Receive Secondary Pharmacogenomic Findings and Development of a Multidisciplinary Practice Model to Integrate Results Into Patient Care.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/28749586" id="rid62" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          63 : Patient Decisions to Receive Secondary Pharmacogenomic Findings and Development of a Multidisciplinary Practice Model to Integrate Results Into Patient Care.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22863877" id="rid63" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          64 : Points to consider in the clinical application of genomic sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23907645" id="rid64" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          65 : Recommendations for returning genomic incidental findings? We need to talk!
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23590238" id="rid65" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          66 : 'Information is information': a public perspective on incidental findings in clinical and research genome-based testing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24453945" id="rid66" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          67 : Managing incidental genomic findings in clinical trials: fulfillment of the principle of justice.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23686341" id="rid67" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          68 : Point-counterpoint. Patient autonomy and incidental findings in clinical genomics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22903777" id="rid68" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          69 : "I want to know what's in Pandora's Box": comparing stakeholder perspectives on incidental findings in clinical whole genomic sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24348686" id="rid69" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          70 : Premature guidance about whole-genome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23875796" id="rid70" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          71 : Return of individual research results and incidental findings: facing the challenges of translational science.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31035287" id="rid71" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          72 : Cases in Precision Medicine: When Patients Present With Direct-to-Consumer Genetic Test Results.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/25232848" id="rid72" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          73 : Single-nucleotide polymorphism arrays and unexpected consanguinity: considerations for clinicians when returning results to families.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23595601" id="rid73" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          74 : Management of incidental findings in clinical genomic sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/22995991" id="rid74" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          75 : An informatics approach to analyzing the incidentalome.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21558861" id="rid75" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          76 : Deploying whole genome sequencing in clinical practice and public health: meeting the challenge one bin at a time.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24943691" id="rid76" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          77 : Genomic medicine and incidental findings: balancing actionability and patient autonomy.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27490114" id="rid77" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          78 : Is "incidental finding" the best term?: a study of patients' preferences.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29739461" id="rid78" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          79 : Clinical providers' experiences with returning results from genomic sequencing: an interview study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29739461" id="rid79" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          80 : Clinical providers' experiences with returning results from genomic sequencing: an interview study.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24941179" id="rid80" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          81 : Diagnostic clinical genome and exome sequencing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/31134132" id="rid81" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          82 : Long-Read Sequencing Emerging in Medical Genetics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30610921" id="rid82" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          83 : A Rigorous Interlaboratory Examination of the Need to Confirm Next-Generation Sequencing-Detected Variants with an Orthogonal Method in Clinical Genetic Testing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30610921" id="rid83" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          84 : A Rigorous Interlaboratory Examination of the Need to Confirm Next-Generation Sequencing-Detected Variants with an Orthogonal Method in Clinical Genetic Testing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23743551" id="rid84" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          85 : The Electronic Medical Records and Genomics (eMERGE) Network: past, present, and future.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24723935" id="rid85" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          86 : Return of results in the genomic medicine projects of the eMERGE network.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24097178" id="rid86" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          87 : Integration of genomics into the electronic health record: mapping terra incognita.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/21432942" id="rid87" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          88 : The GeneInsight Suite: a platform to support laboratory and provider use of DNA-based genetic testing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/24013137" id="rid88" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          89 : A novel clinician interface to improve clinician access to up-to-date genetic results.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/27532831" id="rid89" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          90 : Genetic Misdiagnoses and the Potential for Health Disparities.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30926966" id="rid90" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          91 : Clinical use of current polygenic risk scores may exacerbate health disparities.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/15140372" id="rid91" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          92 : Racial and ethnic variations in knowledge and attitudes about genetic testing.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/20011555" id="rid92" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          93 : Guidelines for genetic testing of healthy children.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19277059" id="rid93" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          94 : Genetic testing in asymptomatic minors: Recommendations of the European Society of Human Genetics.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/29875419" id="rid94" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          95 : Secondary findings from next-generation sequencing: what does actionable in childhood really mean?
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/26140447" id="rid95" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          96 : Points to Consider: Ethical, Legal, and Psychosocial Implications of Genetic Testing in Children and Adolescents.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16091577" id="rid96" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          97 : Disclosing individual results of clinical research: implications of respect for participants.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19164202" id="rid97" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          98 : Sharing study results with trial participants: time for action.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/19164202" id="rid98" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          99 : Sharing study results with trial participants: time for action.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/16391213" id="rid99" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          100 : Implications of disclosing individual results of clinical research.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/30100086" id="rid100" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          101 : A Model for Genome-First Care: Returning Secondary Genomic Findings to Participants and Their Healthcare Providers in a Large Research Cohort.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/18349098" id="rid101" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          102 : Key Internet genetics resources for the clinician.
         </p>
        </a>
       </div>
       <div class="col-12" style="text-align: left;direction: ltr">
        <a href="https://pubmed.ncbi.nlm.nih.gov/23463607" id="rid102" target="_blank">
         <p class="btn-info round" style='width: 100%;margin-bottom: 5px !important;padding: 10px;border-radius: 12px;;font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;'>
          103 : Towards a Universal Clinical Genomics Database: the 2012 International Standards for Cytogenomic Arrays Consortium Meeting.
         </p>
        </a>
       </div>
      </div>
     </div>
    </div>
   </section>
  </section>
  <!-- End Main Menu Area -->
  <!-- Start Search Popup Area -->
  <!-- End Footer Area -->
  <!-- Back to top -->
  <a class="scrolltop" href="#top">
   <i class="icofont-hand-drawn-up">
   </i>
  </a>
  <!-- End Back to top -->
  <!-- jQuery Min JS -->
  <!-- Prpper JS -->
  <!-- Bootstrap Min JS -->
  <!-- Classy Nav Min Js -->
  <!-- Owl Carousel Min Js -->
  <!-- Magnific Popup JS -->
  <!-- CounterUp JS -->
  <!-- Waypoints JS -->
  <!-- Form Validator Min JS -->
  <!-- Contact Form Min JS -->
  <!-- Main JS -->
  <!-- Global site tag (gtag.js) - Google Analytics -->
 </body>
</html>
