<!DOCTYPE html>

<html lang="fa">
<meta content="text/html;charset=utf-8" http-equiv="content-type"/>
<head><title>Interpretation of immunohistochemistry results for mismatch repair genes</title><link href="/d/css/bootstrap.min.css" rel="stylesheet"/><link href="/d/css/icofont.min.css" rel="stylesheet"/><link href="/d/css/classy-nav.min.css" rel="stylesheet"/><link href="/d/css/animate.css" rel="stylesheet"/><link href="/d/css/owl.carousel.css" rel="stylesheet"/><link href="/d/css/magnific-popup.css" rel="stylesheet"/><link href="/d/css/owl.theme.default.min.css" rel="stylesheet"/><link href="/d/css/style.css" rel="stylesheet"/><link href="/d/css/rtl_edited.css" rel="stylesheet"/><link href="/d/css/responsive.css" rel="stylesheet"/><link href="https://medilib.ir/uptodate_assets/main.css" rel="stylesheet">
<style>

#myUL {
  list-style-type: none;
  padding: 0;
  margin: 0;
}

#myUL li a {
  border: 1px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #f6f6f6;
  padding: 12px;
  text-decoration: none;
  font-size: 16px;
  color: black;
  display: block
}

#myUL li a:hover:not(.header) {
  background-color: #eee;
}

#searchResult {

  list-style-type: none;
  padding: 0;
  padding-right: 20px;
}

#searchResult li a {
  border: .5px solid #ddd;
  margin-top: -1px; /* Prevent double borders */
  background-color: #79bc21;
  padding: 7px;
  text-decoration: none;
  font-size: 14px;
  color: white;
  display: block
}

#searchResult li a:hover:not(.header) {
  background-color: #222;
}


#txtHint   {
  position: absolute;
  text-shadow: none;
  width: 100%;
  margin-top: -15px;
  padding-right: 10px;

  filter: alpha(opacity=95);
  -moz-opacity: 0.95;
  opacity: 1;
  z-index: 200;
}
.countBubl {
  background:#eb0029;
  color:#fff;
  padding:5px;
  margin-top: 50%;

}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(2) {
 width: 55%;
 padding-left: 20px;
}
.uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > span:nth-child(1) > img:nth-child(3) {
 width: 20%;
}
 #Uptodate_image {
      width: 50%;
  }#uptodate-area {
      /*z-index: 99;*/
  }
#uptodate-area > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) {
 margin-left: 10px;
 margin-bottom: 6%;
}
@media (max-width: @screen-xs) {
    .countBubl {
        font-size: 4px ;
    }
    #uptodate_version {
        font-size: 4px ;
    }
}

@media (max-width: @screen-sm) {
    .countBubl {
        font-size: 14px ;
    }
}
@media  all and(max-width: 820) {
    #Uptodate_image {
        width: 80% ;
    }#uptodate-area {
        z-index: 99;
    }
}@media  all and (min-width: 820) and(max-width: 88820){
    #Uptodate_image {
        width: 100% !important;
    }
    .countBubl {
        font-size: 4px ;
    }
}
div.col-md-3:nth-child(3) {
 margin-top: 6%;
}

div.col-md-3:nth-child(3) > img:nth-child(1) {
 width: 100%;
 margin-left: 11px;
}

</style>
<style media="screen" type="text/css">
  .goback {
    width: 200px;
    height: 40px;
    line-height: 40px;
    position: fixed;
    display: none;
    font-size: 15px;
    background: #71c0fa;
    z-index: 1000;
    text-align: center;
    color: #ffffff;
    -webkit-border-radius: 1px;
    -moz-border-radius: 1px;
    border-radius: 50px;
    bottom: 3.8%;
    right: 2%;
}
  .buyPackage {
  width: 100px;
  height: 100px;
  line-height: 40px;
  position: fixed;
  display: none;
  font-size: 10px;
  background: #fa759e;
  z-index: 1000;
  text-align: center;
  color: #ffffff;
  -webkit-border-radius: 1px;
  -moz-border-radius: 1px;
  border-radius: 10px;
  top: 150px;
  right: 2%;
  text-align: center;
}
.uptodate-area {
  /*background-image: url(../../assets/img/news-letter-bg.jpg);*/
  background-position: center center;
  background-size: cover;
  background-repeat: no-repeat;
    background-attachment: fixed;
  position: relative;
}
.uptodate-area::before {
  content: '';
  position: absolute;
  left: 0;
  padding-top: 100px;
  width: 100%;
}
#references{
  display: none;
}
</style>
<style>
section.sticky {
  position: sticky;
  top: 0;
  background-color: #fff !important;
  /*z-index: 100;*/
  /*padding-top: 80px;*/

}
.m4inb{
    border: solid !important;
  }

  .newsletter-area::before {
       background: unset !important;
  }
p {
    color: #555;
}
</style>

<meta content="" name="description"/>
<link href="/d/img/favicon.png" rel="icon" type="image/png"/>
</link></head>
<body>
<!-- Start Main Menu Area -->

<br/>
<br/>
<br/>
<br/>
<div class="container">
<div class="row">
</div>
</div>




<section class="teacher-area ptb-10">
<div class="container">
</div>

<section class="course-details-area ptb-60">
<div class="container">
<div class="uptodate2" style="padding:50px 0px 50px 0px;direction: ltr;text-align: left;">
<h1 style="text-align:center">Interpretation of immunohistochemistry results for mismatch repair genes</h1>
<div class="graphic"><div class="figure"><div class="ttl">Interpretation of immunohistochemistry results for mismatch repair genes</div><div class="cntnt"><table cellspacing="0"><colgroup width="16%"></colgroup><colgroup span="3" width="28%"></colgroup> <tbody> <tr> <td class="subtitle1">Result</td> <td class="subtitle1">Possible interpretation</td> <td class="subtitle1">Explanation/comments</td> <td class="subtitle1">Next steps to consider</td> </tr> <tr class="divider_bottom"> <td>Loss of MLH1 only (rare)</td> <td> <ol class="numbers_no_heading"> <li><em>MLH1 </em>germline mutation. </li> <li><em>MLH1</em> promoter hypermethylation. </li> <li>Bi-allelic (double) somatic <em>MLH1</em> or <em>PMS2</em> inactivation. </li> </ol> </td> <td> <ul> <li>Epigenetic silencing of the <em>MLH1 </em>gene can occur through <em>MLH1</em> promoter hypermethylation. </li> <li>Bi-allelic (double) inactivation of <em>MLH1</em> can rarely cause isolated MLH1 loss. </li> </ul> </td> <td> <ol class="numbers_no_heading"> <li><em>BRAF</em> V600E mutation testing (if negative, proceed to #2). </li> <li>Tumor testing for <em>MLH1</em> promoter hypermethylation testing.<sup>¶ </sup></li> <li>Germline panel testing for at least the MMR genes if family history is suggestive of Lynch syndrome.* </li> <li>Tumor sequencing to evaluate for bi-allelic (double) somatic <em>MLH1 </em>inactivation due to mutation and/or LOH. </li> </ol> </td> </tr> <tr class="divider_bottom"> <td>Loss of MLH1 and PMS2 (common)</td> <td> <ol class="numbers_no_heading"> <li><em>MLH1</em> germline mutation. </li> <li><em>PMS2</em> germline mutation. </li> <li><em>MLH1</em> promoter hypermethylation. </li> <li>Bi-allelic (double) somatic <em>MLH1</em> inactivation through mutation and/or LOH. </li> <li>Bi-allelic (double) somatic <em>PMS2</em> inactivation through mutation and/or LOH. </li> </ol> </td> <td> <ul> <li>Because the MLH1 and PMS2 proteins form a heterodimer, altered expression of either the MLH1 or PMS2 protein due to germline mutations frequently leads to loss of both MLH1 and PMS2 expression. </li> <li>Altered expression of either <em>MLH1 </em>or <em>PMS2 </em>due to bi-allelic (double) somatic mutation or LOH of the <em>MLH1</em> or <em>PMS2 </em>gene can also lead to this pattern. </li> </ul> </td> <td> <ol class="numbers_no_heading"> <li><em>BRAF </em>V600E mutation testing (if negative, proceed to #2). </li> <li>Tumor testing for <em>MLH1 </em>promoter hypermethylation.<sup>¶ </sup></li> <li>Tumor testing for bi-allelic (double) somatic <em>MLH1</em> or <em>PMS2</em> inactivation due to mutation and/or LOH. </li> <li>Germline panel testing for at least the MMR genes if family history is suggestive of Lynch syndrome.* </li> </ol> </td> </tr> <tr class="divider_bottom"> <td>Loss of PMS2 only (less common)</td> <td> <ol class="numbers_no_heading"> <li><em>PMS2</em> germline mutation. </li> <li>Bi-allelic (double) somatic PMS2 inactivation through mutation and/or LOH. </li> <li><em>MLH1</em> germline mutation. </li> <li>Bi-allelic (double) somatic <em>MLH1</em> inactivation through mutation and/or LOH. </li> </ol> </td> <td> <ul> <li>Because the MLH1 protein has heterodimer partner proteins other than the PMS2 protein, germline <em>PMS2</em> mutations may not cause loss of <em>MLH1</em> expression by IHC. </li> <li>PMS2 protein expression by IHC can be interpreted as equivocal—not clearly positive or negative. </li> <li><em>MLH1</em> germline mutations have rarely been identified when tumors show loss of staining of PMS2 only; bi-allelic double somatic <em>MLH1 </em>mutation or LOH could also mimic this. </li> </ul> </td> <td> <ol class="numbers_no_heading"> <li>Germline MMR panel testing.* </li> <li>If panel testing is negative consider tumor testing for the MMR genes to identify bi-allelic (double) somatic <em>PMS2</em> or <em>MLH1</em> inactivation due to mutation and/or LOH. </li> <li>If PMS2 expression by IHC is equivocal consider secondary MSI testing to confirm or rule out presence of dMMR. </li> </ol> </td> </tr> <tr class="divider_bottom"> <td>Loss of MSH2 only (rare)</td> <td> <ol class="numbers_no_heading"> <li><em>MSH2</em> germline mutation. </li> <li><em>EPCAM</em> germline mutation. </li> <li>Bi-allelic (double) somatic inactivation of <em>MSH2 </em>through mutation and/or LOH. </li> </ol> </td> <td> <ul> <li>Strong likelihood of germline <em>MSH2</em> or <em>EPCAM.</em> </li> <li>Bi-allelic (double) somatic inactivation of <em>MSH2 </em>is rare but has been reported. </li> </ul> </td> <td> <ol class="numbers_no_heading"> <li>Germline MMR panel testing.* </li> <li>If panel testing is negative consider tumor testing for the MMR genes to identify bi-allelic (double) somatic <em>MSH2 </em>inactivation due to mutation and/or LOH. </li> </ol> </td> </tr> <tr class="divider_bottom"> <td>Loss of MSH2 and MSH6 (common)</td> <td> <ol class="numbers_no_heading"> <li><em>MSH6</em> germline mutation. </li> <li><em>MSH2</em> germline mutation. </li> <li><em>EPCAM</em> germline mutation. </li> <li>Bi-allelic (double) somatic inactivation of <em>MSH2</em> through mutation and/or LOH. </li> <li>Bi-allelic (double) somatic inactivation of <em>MSH6</em> through mutation and/or LOH. </li> </ol> </td> <td> <ul> <li>Because the MSH2 and MSH6 proteins form a heterodimer, alterations of expression of either the MSH2 or MSH6 protein due to germline or bi-allelic (double) somatic mutations in either the <em>MSH2 </em>or <em>MSH6</em> genes could cause this pattern on IHC. </li> </ul> </td> <td> <ol class="numbers_no_heading"> <li>Germline MMR panel testing.* </li> <li>If panel testing is negative consider tumor testing for the MMR genes to identify bi-allelic (double) somatic <em>MSH2 </em>or<em> MSH6 </em>inactivation due to mutation and/or LOH. </li> </ol> </td> </tr> <tr class="divider_bottom"> <td>Loss of MSH6 (common)</td> <td> <ol class="numbers_no_heading"> <li><em>MSH6</em> germline mutation. </li> <li>Bi-allelic (double) somatic inactivation of <em>MSH6 </em>through mutation and/or LOH. </li> <li><em>MSH2</em> germline mutation. </li> <li>Bi-allelic (double) somatic inactivation of <em>MSH2 </em>through mutation and/or LOH. </li> </ol> </td> <td> <ul> <li>Because MSH2 has heterodimer partners other than MSH6, germline <em>MSH6 </em>mutations may not cause loss of MSH2 expression by IHC and frequently cause loss of staining of MSH6 only; similar reasoning holds for bi-allelic (double) somatic <em>MSH6 </em>inactivation through mutation and/or LOH. </li> </ul> </td> <td> <ol class="numbers_no_heading"> <li>Germline MMR panel testing.* </li> <li>If panel testing is negative consider tumor testing for the MMR genes to identify bi-allelic (double) somatic <em>MSH6</em> inactivation due to mutation and/or LOH. </li> </ol> </td> </tr> <tr class="divider_bottom"> <td>Loss of all 4 MMR proteins</td> <td> <ol class="numbers_no_heading"> <li>Likely an artifact of tissue fixation and IHC staining procedures. </li> <li>Germline mutation in any one of the Lynch syndrome genes </li> </ol> </td> <td> </td> <td> <ol class="numbers_no_heading"> <li>Consider MSI testing to confirm or rule out dMMR. </li> <li>Germline MMR panel testing if family history warrants testing.* </li> </ol> </td> </tr> <tr> <td>All proteins expressed</td> <td> <ol class="numbers_no_heading"> <li>Likely not Lynch syndrome if MSS. </li> <li>If MSI-H, rare missense germline mutations in <em>MLH1, MSH2, MSH6, </em>and <em>PMS2</em> can demonstrate normal IHC. </li> <li>If MSI-H, rare patients with bi-allelic (double) somatic alterations due to mutation and/or LOH. </li> </ol> </td> <td>Normal IHC is only approximately 85% sensitive for Lynch syndrome.</td> <td> <ol class="numbers_no_heading"> <li>Consider MSI testing to confirm or rule out dMMR. </li> <li>If MSI-H, germline MMR panel testing if family history warrants testing.* </li> <li>If MSI-H and germline panel testing are negative consider tumor testing for the MMR genes to identify bi-allelic (double) somatic inactivation due to mutation and/or LOH. </li> </ol> </td> </tr> </tbody></table></div><div class="graphic_footnotes">LOH: loss of heterozygosity; MMR: mismatch repair; dMMR: deficient mismatch repair; IHC: immunohistochemistry; MSI: microsatellite instability; MSI-H: high microsatellite instability.<br/>* A multi-syndrome gene panel is an option if there is a personal or family history of other cancers (not associated with Lynch syndrome).<br/>¶ Rarely, MLH1 methylated or BRAF V600E+ tumors can harbor germline MLH1 mutations. MLH1 methylation may also rarely be caused by constitutional MLH1 epimutation</div><div class="graphic_reference">References: 

<ol>
<li>Niessen RC, Kleibeuker JH, Westers H, Jager PO, Rozeveld D, Bos KK, et al. PMS2 involvement in patients suspected of Lynch syndrome. Genes, Chromosomes and Cancer 2009; 48:322.</li>
<li>Senter L, Clendenning M, Sotamaa K, Hampel H, Green J, Potter JD, et al. The clinical phenotype of Lynch syndrome due to germ-line PMS2 mutations. Gastroenterology, 2008; 135:419.</li>
<li>De Jong AE, van Puijenbroek M, Hendriks Y, Tops C, Wijnen J, Ausems MG, et al. Microsatellite instability, immunohistochemistry, and additional PMS2 staining in suspected hereditary nonpolyposis colorectal cancer. Clinical Cancer Research 2004; 10:972.</li>
<li>Zighelboim I, Powell M, Babb S, Whelan A, Schmidt A, Clendenning M, et al. Epitope-positive truncating MLH1 mutation and loss of PMS2: implications for IHC-directed genetic testing for lynch syndrome. Familial Cancer 2009; 8:501.</li>
<li>Hampel H, Pearlman R, Beightol M, et al. Assessment of tumor sequencing as a replacement for Lynch syndrome screening and current molecular tests for patients with colorectal cancer. JAMA Oncol 2018; 4:806.</li>
<li>Pearlman R, Haraldsdottir S, de la Chapelle A, et al. Clinical characteristics of colorectal Cancer Patients with Double Somatic Mismatch Repair Mutations Compared to Lynch Syndrome J Med Genet 2019; 56:462.</li>
<li>Mensenkamp AR, Vogelaar IP, van Zelst-Stams WA, et al. Somatic mutations in MLH1 and MSH2 are a frequent cause of mismatch-repair deficiency in Lynch syndrome-like tumors. Gastroenterology 2014; 146:643.</li>
<li>Haraldsdottir S, Hampel H, Tomsic J, et al. Colon and endometrial cancers with mismatch repair deficiency can arise from somatic, rather than germline, mutations. Gastroenterology 2014; 147:1308.</li>
<li>Sourrouille I, Coulet F, Lefevre JH, et al. Somatic mosaicism and double somatic hits can lead to MSI colorectal tumors. Fam Cancer 2013; 12:27.</li>
<li>Hitchins MP. Finding the needle in a haystack: identification of cases of Lynch syndrome with MLH1 epimutation. Fam Cancer 2016; 15:413.</li></ol>Adapted from: Weissman SM, Bellcross C, Bittner CC, et al. Genetic counseling considerations in the evaluation of families for lynch syndrome-a review. J Genet Couns 2011; 20:5.</div><div id="graphicVersion">Graphic 63906 Version 12.0</div></div></div>
</div>
</div>

</section>

</section>
<!-- End Main Menu Area -->
<!-- Start Search Popup Area -->


<!-- End Footer Area -->
<!-- Back to top -->

<a class="scrolltop" href="#top"><i class="icofont-hand-drawn-up"></i></a>
<!-- End Back to top -->
<!-- jQuery Min JS -->

<!-- Prpper JS -->

<!-- Bootstrap Min JS -->

<!-- Classy Nav Min Js -->

<!-- Owl Carousel Min Js -->

<!-- Magnific Popup JS -->

<!-- CounterUp JS -->

<!-- Waypoints JS -->

<!-- Form Validator Min JS -->

<!-- Contact Form Min JS -->

<!-- Main JS -->

<!-- Global site tag (gtag.js) - Google Analytics -->






</body>
</html>
